--- Area:MapWorldsScriptorium
+++ Export
@@ -11,5 +11,7 @@
 |entry_text                       = Knowledge balanced atop a vast pit. Hold on, or perish in the darkness of ignorance.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
+|flavour_text                     = Knowledge balanced atop a vast pit. Hold on,
+or perish in the darkness of ignorance.
 |main_page                        = Scriptorium Map
 }}