--- Area:MapWorldsConservatory
+++ Export
@@ -10,10 +10,7 @@
 |boss_monster_ids                 = Metadata/Monsters/Exiles/TituciusMapBoss
 |entry_text                       = Ornate, elaborate, ostentatious... and dripping with death.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea8_5
 |is_map_area                      = True
-|flavour_text                     = Ornate, elaborate, ostentatious...
-and dripping with death.
 |main_page                        = Conservatory Map
 }}