--- Area:1~2~6~2
+++ Export
@@ -9,8 +9,8 @@
 |connection_ids                   = 1_2_6_1
 |parent_area_id                   = 1_2_town
 |boss_monster_ids                 = Metadata/Monsters/DemonFourLegged/BossFourLeggedDemon
-|vaal_area_spawn_chance           = 12
 |vaal_area_ids                    = 1_SideArea2_6
 |flavour_text                     = Phantom whispers of maniacal passion.
 |main_page                        = The Chamber of Sins Level 2 (Act 2)
+|vaal_area_spawn_chance           = 12
 }}