--- Area:2~8~13
+++ Export
@@ -10,8 +10,8 @@
 |parent_area_id                   = 2_8_town
 |boss_monster_ids                 = Metadata/Monsters/LunarisSolaris/Lunaris, Metadata/Monsters/LunarisSolaris/Solaris
 |monster_ids                      = Metadata/Monsters/Statue/StoneStatueMaleSword, Metadata/Monsters/Statue/StoneStatueMaleMaul, Metadata/Monsters/Statue/StoneStatueMaleBow, Metadata/Monsters/Statue/StoneStatueFemaleUnarmed, Metadata/Monsters/Statue/StoneStatueFemaleBow, Metadata/Monsters/Statue/StoneStatueFemaleMaul, Metadata/Monsters/Statue/StoneStatueFemaleSword
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea3_6
 |flavour_text                     = A dark unrest sends ribbons to war.
 |main_page                        = The Harbour Bridge
+|vaal_area_spawn_chance           = 18
 }}