--- Area:MapWorldsOrchard
+++ Export
@@ -4,7 +4,7 @@
 |act                              = 11
 |area_level                       = 77
 |area_type_tags                   = forest
-|tags                             = map, no_echo, einharmaps, has_road_area, garden_area, labyrinth_tileset_area
+|tags                             = map, no_echo, einharmaps, has_road_area, garden_area, labyrinth_tileset_area, sun_nature_area
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/Labyrinth/GoddessOfJusticeMapBossSimplified