--- Area:MapWorldsLeyline
+++ Export
@@ -10,8 +10,9 @@
 |boss_monster_ids                 = Metadata/Monsters/SkeletonSoldier/SkeletonSoldierRangedMapBoss
 |entry_text                       = Amidst the barren wastes, a slender strand of life blooms.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea4_4
 |is_map_area                      = True
+|flavour_text                     = Amidst the barren wastes,
+a slender strand of life blooms.
 |main_page                        = Leyline Map
 }}