--- Area:MapWorldsBrambleValley
+++ Export
@@ -10,11 +10,7 @@
 |boss_monster_ids                 = Metadata/Monsters/LeagueHarvest/Red/HarvestBearBossMapBoss, Metadata/Monsters/LeagueHarvest/Green/HarvestCatBossMapBoss, Metadata/Monsters/LeagueHarvest/Blue/HarvestBirdBossMapBoss
 |entry_text                       = The trees grow twisted and gnarled, without a hint of light. A curse lingers here.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea2_5, MapSideArea2_8
 |is_map_area                      = True
-|flavour_text                     = The faintest traces of civilisation
-defiantly resist nature's scouring tendrils.
-An admirable, but futile, effort.
 |main_page                        = Bramble Valley Map
 }}