--- Area:2~8~7~1~
+++ Export
@@ -8,9 +8,9 @@
 |loading_screen                   = Act3
 |connection_ids                   = 2_8_6, 2_8_7_2
 |parent_area_id                   = 2_8_town
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea8_7
 |has_waypoint                     = True
 |flavour_text                     = Gibbous beauty eclipsed by a sisters' feud.
 |main_page                        = The Lunaris Temple Level 1 (Act 8)
+|vaal_area_spawn_chance           = 18
 }}