--- Area:MapAtlasAbyss
+++ Export
@@ -4,7 +4,7 @@
 |act                              = 11
 |area_level                       = 82
 |area_type_tags                   = cave, mountain
-|tags                             = map, no_echo, area_with_water, einharmaps, kaom_area, king_boss_area
+|tags                             = map, no_echo, area_with_water, einharmaps, kaom_area, king_boss_area, karui_map
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/Kaom/KaomMapBoss
