--- Area:1~1~8
+++ Export
@@ -8,9 +8,9 @@
 |loading_screen                   = Act1
 |connection_ids                   = 1_1_7_2, 1_1_9, 1_2_9
 |parent_area_id                   = 1_1_town
-|vaal_area_spawn_chance           = 12
 |vaal_area_ids                    = 1_SideArea1_6
 |has_waypoint                     = True
 |flavour_text                     = The cliffs watch with goatish eyes.
 |main_page                        = Prisoner's Gate (Act 1)
+|vaal_area_spawn_chance           = 12
 }}