--- Area:MapWorldsTerrace
+++ Export
@@ -4,12 +4,14 @@
 |act                              = 11
 |area_level                       = 80
 |area_type_tags                   = urban
-|tags                             = map, no_echo, cannot_be_twinned, einharmaps, garden_area, high_gardens_area, god_boss_map_area
+|tags                             = map, no_echo, cannot_be_twinned, einharmaps, garden_area, high_gardens_area, god_boss_map_area, sun_nature_area
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/Frog/FrogGod/SilverFrogMap
 |entry_text                       = A garden on a stone pedestal. An incredible, and foolish, feat.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
+|flavour_text                     = A garden on a stone pedestal.
+An incredible, and foolish, feat.
 |main_page                        = Terrace Map
 }}