--- Area:MapWorldsPrimordialPool
+++ Export
@@ -11,7 +11,7 @@
 |entry_text                       = A dough of bad blood. A yeast of ill will.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = A dough of bad blood.
+|flavour_text                     = A dough of bad blood.
 A yeast of ill will.
 |main_page                        = Primordial Pool Map
 }}