--- Area:1~2~4
+++ Export
@@ -9,9 +9,9 @@
 |connection_ids                   = 1_2_3
 |parent_area_id                   = 1_2_town
 |boss_monster_ids                 = Metadata/Monsters/BanditLeaderDexInt/BanditLeaderKraityn
-|vaal_area_spawn_chance           = 15
 |vaal_area_ids                    = 1_SideArea2_4
 |has_waypoint                     = True
 |flavour_text                     = A pure, soothing river once red and clogged with the victims of war.
 |main_page                        = The Broken Bridge (Act 2)
+|vaal_area_spawn_chance           = 15
 }}