--- Area:2~8~10
+++ Export
@@ -9,9 +9,9 @@
 |connection_ids                   = 2_8_9, 2_8_12_1
 |parent_area_id                   = 2_8_town
 |boss_monster_ids                 = Metadata/Monsters/Skeletons/Scarecrow/ScarecrowPitchforkBoss
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea3_8, 1_SideArea2_4
 |has_waypoint                     = True
 |flavour_text                     = The savagery of war stains both muck and mire.
 |main_page                        = The Imperial Fields
+|vaal_area_spawn_chance           = 18
 }}