--- Area:MapWorldsMineralPools
+++ Export
@@ -4,15 +4,15 @@
 |act                              = 11
 |area_level                       = 75
 |area_type_tags                   = swamp
-|tags                             = map, area_with_water, einharmaps, flooded_area, gold_pile_area, sea_witch_boss_area, crystalline_area, bridge_area
+|tags                             = map, area_with_water, einharmaps, flooded_area, gold_pile_area, sea_witch_boss_area, crystalline_area, bridge_area, sun_nature_area
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/Seawitch/MapSeaWitchBossSpark
 |entry_text                       = God's jar of raw materials. Life lying dormant, waiting to sprout.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = God's jar of raw materials.
-Life lying dormant,
+|flavour_text                     = God's jar of raw materials.
+Life lying dormant,
 waiting to sprout.
 |main_page                        = Mineral Pools Map
 }}