--- Area:1~1~4~1
+++ Export
@@ -10,6 +10,7 @@
 |parent_area_id                   = 1_1_town
 |boss_monster_ids                 = Metadata/Monsters/Seawitch/SeaWitchBossSubmerged
 |monster_ids                      = Metadata/Monsters/SandSpitters/SandSpitter2
+|vaal_area_ids                    = 1_SideArea1_2
 |has_waypoint                     = True
 |flavour_text                     = Slime, shadow, and sharp sharp teeth.
 |main_page                        = The Submerged Passage
