--- Area:2~8~2~1
+++ Export
@@ -10,8 +10,8 @@
 |parent_area_id                   = 2_8_town
 |modifier_ids                     = MapBossesSpawnZombieGiant
 |boss_monster_ids                 = Metadata/Monsters/Zombies/ZombieGiantSummoningPackMember, Metadata/Monsters/Zombies/ZombieGiantSummoningPackMember, Metadata/Monsters/Zombies/ZombieGiantSummoningPackMember, Metadata/Monsters/Zombies/ZombieGiantSummoningPackMember, Metadata/Monsters/Zombies/ZombieGiantSummoningPackMember
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea3_7
 |flavour_text                     = Spells of blood foul the waters below.
 |main_page                        = The Toxic Conduits
+|vaal_area_spawn_chance           = 18
 }}