--- Area:2~7~2
+++ Export
@@ -8,7 +8,6 @@
 |loading_screen                   = Act2
 |connection_ids                   = 2_7_1, 2_7_3, 2_7_5_1
 |parent_area_id                   = 2_7_town
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea2_4
 |has_waypoint                     = True
 |flavour_text                     = Ancient cruelty tears through the cobbled intersection.
