--- Area:MapWorldsBurialChambers
+++ Export
@@ -4,16 +4,14 @@
 |act                              = 11
 |area_level                       = 70
 |area_type_tags                   = dungeon
-|tags                             = map, indoors_area, einharmaps, bandit_area, alira_area, summoner_area, cemetery_map_area
+|tags                             = map, indoors_area, einharmaps, bandit_area, alira_area, summoner_area, cemetery_map_area, sun_nature_area
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/BanditLeaderInt/MapBanditLeaderAlira2
 |entry_text                       = Vile fiends hide beneath the peaceful plains.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea2_8
 |is_map_area                      = True
-|flavour_text                     = Vile fiends hide beneath
-the peaceful plains.
 |main_page                        = Burial Chambers Map
+|vaal_area_spawn_chance           = 8
 }}