--- Area:1~1~11~2
+++ Export
@@ -10,7 +10,6 @@
 |parent_area_id                   = 1_1_town
 |boss_monster_ids                 = Metadata/Monsters/Seawitch/SeaWitchBossFirestorm, Metadata/Monsters/seawitchillusion/BossMerveil
 |monster_ids                      = Metadata/Monsters/Seawitch/BossMerveil2
-|vaal_area_spawn_chance           = 12
 |vaal_area_ids                    = 1_SideArea1_1
 |flavour_text                     = The air needles the skin with its frigid fury.
 |main_page                        = The Cavern of Anger (Act 1)
