--- Area:2~10~2
+++ Export
@@ -9,7 +9,6 @@
 |connection_ids                   = 2_10_1, 2_10_3, 2_10_9, 2_10_8, 2_10_5, 2_10_7
 |parent_area_id                   = 2_10_town
 |boss_monster_ids                 = Metadata/Monsters/MotherOfFlames/MotherOfFlamesBossZombie2
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea5_3_1, 1_SideArea5_3_2
 |has_waypoint                     = True
 |spawn_weight1_tag                = high_level_map
@@ -18,4 +17,5 @@
 |spawn_weight2_value              = 1000
 |flavour_text                     = A flourishing place of trade, barely recognizable amidst the countless deaths.
 |main_page                        = The Ravaged Square
+|vaal_area_spawn_chance           = 18
 }}