--- Area:MapWorldsDungeon
+++ Export
@@ -10,10 +10,10 @@
 |boss_monster_ids                 = Metadata/Monsters/Brute/MapBossBrute
 |entry_text                       = A penitentiary for those who are not penitent is nothing more than an inn for the evil.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea4_1, MapSideArea1_5
 |is_map_area                      = True
 |flavour_text                     = A penitentiary for those who are not penitent
 is nothing more than an inn for the evil.
 |main_page                        = Dungeon Map
+|vaal_area_spawn_chance           = 8
 }}