--- Area:1~5~3b
+++ Export
@@ -9,12 +9,11 @@
 |connection_ids                   = 1_5_4b, 1_5_6, 1_5_8, 1_5_7
 |parent_area_id                   = 1_5_town
 |boss_monster_ids                 = Metadata/Monsters/BlacksmithGolem/BlacksmithGolem, Metadata/Monsters/MotherOfFlames/MotherOfFlamesBossZombie
-|entry_text                       = Cup your hands, children of Kitava.
-Drink of the hot, sweet wine of Oriath.
-And let us stay ourselves from the feast no longer.
+|entry_text                       = Cup your hands, children of Kitava.
+Drink of the hot, sweet wine of Oriath.
+And let us stay ourselves from the feast no longer.
 For our king is ravenous!
 |entry_npc                        = Utula
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea5_3_1, 1_SideArea5_3_2
 |has_waypoint                     = True
 |flavour_text                     = Oriath's bleeding heart is the main course in a terrible feast.
