--- Area:2~7~5~map
+++ Export
@@ -9,6 +9,7 @@
 |connection_ids                   = 2_7_5_1
 |parent_area_id                   = 2_7_town
 |boss_monster_ids                 = Metadata/Monsters/Maligaro/Maligaro2, Metadata/Monsters/Spiders/BlackDeath/BlackDeathMaligaro, Metadata/Monsters/DemonFourLegged/FourLeggedDemonMaligaro
+|vaal_area_ids                    = 1_SideArea2_6
 |flavour_text                     = The inner depths of decadent darkness.
 |main_page                        = Maligaro's Sanctum
 }}