--- Area:MapWorldsOvergrownShrineUnique
+++ Export
@@ -10,9 +10,9 @@
 |boss_monster_ids                 = Metadata/Monsters/Necromancer/NecromancerTwinUniqueMapBoss1, Metadata/Monsters/Necromancer/NecromancerTwinUniqueMapBoss2
 |is_map_area                      = True
 |is_unique_map_area               = True
-|flavour_text                     = Curious, she shook it then,
-And tiny screams emerged.
-Curiouser, she shook again,
+|flavour_text                     = Curious, she shook it then,
+And tiny screams emerged.
+Curiouser, she shook again,
 And an eerie silence purred.
 |main_page                        = Acton's Nightmare
 }}