--- Area:MapWorldsPlateau
+++ Export
@@ -11,5 +11,7 @@
 |entry_text                       = Steep cliffs divide where the trees may grow, and where they may not.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
+|flavour_text                     = Steep cliffs divide where the trees
+may grow, and where they may not.
 |main_page                        = Plateau Map
 }}