--- Area:1~2~5~2
+++ Export
@@ -9,8 +9,8 @@
 |connection_ids                   = 1_2_5_1
 |parent_area_id                   = 1_2_town
 |boss_monster_ids                 = Metadata/Monsters/Skeletons/SkeletonBossMelee
-|vaal_area_spawn_chance           = 12
 |vaal_area_ids                    = 1_SideArea2_5
 |flavour_text                     = Stones worn smooth by centuries of devotion.
 |main_page                        = The Crypt Level 2
+|vaal_area_spawn_chance           = 12
 }}