--- Area:MapWorldsCourtyard
+++ Export
@@ -11,7 +11,7 @@
 |entry_text                       = An elaborate, beautiful garden, hidden from the public. Selfish.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = An elaborate, beautiful garden,
+|flavour_text                     = An elaborate, beautiful garden,
 hidden from the public. Selfish.
 |main_page                        = Courtyard Map
 }}