--- Area:MapWorldsVaalTemple
+++ Export
@@ -11,5 +11,7 @@
 |entry_text                       = Voices. Visions. Violence. They will haunt these grounds for all of eternity.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
+|flavour_text                     = Voices. Visions. Violence.
+They will haunt these grounds for all of eternity.
 |main_page                        = Vaal Temple Map
 }}