--- Area:1~5~4
+++ Export
@@ -8,9 +8,9 @@
 |loading_screen                   = Act5
 |connection_ids                   = 1_5_3, 1_5_5
 |parent_area_id                   = 1_5_town
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea5_4
 |has_waypoint                     = True
 |flavour_text                     = Only those with faithful eyes may enter these solemn chambers unbidden.
 |main_page                        = The Templar Courts
+|vaal_area_spawn_chance           = 18
 }}