--- Area:MapWorldsExcavation
+++ Export
@@ -10,8 +10,10 @@
 |boss_monster_ids                 = Metadata/Monsters/MineBat/MineBatMapBoss, Metadata/Monsters/Handfinger/HandfingerMapBoss
 |entry_text                       = Two unexplored frontiers remain: The skies and the soil.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea4_3
 |is_map_area                      = True
+|flavour_text                     = Two unexplored frontiers remain:
+The skies and the soil.
 |main_page                        = Excavation Map
+|vaal_area_spawn_chance           = 8
 }}