--- Area:MapWorldsMuseum
+++ Export
@@ -11,7 +11,7 @@
 |entry_text                       = Ancient artefacts and beautiful baubles. All paid for in blood.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Ancient artifacts and beautiful baubles.
+|flavour_text                     = Ancient artifacts and beautiful baubles.
 All paid for in blood.
 |main_page                        = Museum Map
 }}