--- Area:MapWorldsSpiderForest
+++ Export
@@ -11,8 +11,8 @@
 |entry_text                       = Each branch, each leaf, an extension of a web that spans sanity.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Each branch, each leaf,
-an extension of a web
+|flavour_text                     = Each branch, each leaf,
+an extension of a web
 that spans sanity.
 |main_page                        = Spider Forest Map
 }}