--- Area:MapWorldsLair
+++ Export
@@ -11,7 +11,7 @@
 |entry_text                       = In darkness, the alpha rises. The companionship of beasts is fragile indeed.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = In darkness, the alpha rises. The companionship
+|flavour_text                     = In darkness, the alpha rises. The companionship
 of beasts is fragile indeed.
 |main_page                        = Lair Map
 }}