--- Area:2~8~5
+++ Export
@@ -9,9 +9,9 @@
 |connection_ids                   = 2_8_3, 2_8_6, 2_8_4
 |parent_area_id                   = 2_8_town
 |boss_monster_ids                 = Metadata/Monsters/Exiles/Titucius
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea8_5
 |has_waypoint                     = True
 |flavour_text                     = Hidden sin where revelry thrives.
 |main_page                        = The Bath House
+|vaal_area_spawn_chance           = 18
 }}