--- Area:MapWorldsCourthouse
+++ Export
@@ -10,8 +10,10 @@
 |boss_monster_ids                 = Metadata/Monsters/Exiles/ExileMarauder6Kitava, Metadata/Monsters/Exiles/ExileRanger2Kitava, Metadata/Monsters/Exiles/ExileShadow1Kitava
 |entry_text                       = Justice born of love is a surrender. Justice born of law is a punishment.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea5_4
 |is_map_area                      = True
+|flavour_text                     = Justice born of love is a surrender.
+Justice born of law is a punishment.
 |main_page                        = Courthouse Map
+|vaal_area_spawn_chance           = 8
 }}