--- Area:MapWorldsDesert
+++ Export
@@ -10,7 +10,6 @@
 |boss_monster_ids                 = Metadata/Monsters/Kira/KiraMapDesert
 |entry_text                       = Heat. Dust. Wind. These are what forge strong minds and bodies.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea4_2
 |is_map_area                      = True
 |main_page                        = Desert Map
