--- Area:MapWorldsShipyard
+++ Export
@@ -11,8 +11,8 @@
 |entry_text                       = The stone and lumber sleeps. The captains are long dead. Only the sea knows their end.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = The stone and lumber sleeps.
-The captains are long dead.
+|flavour_text                     = The stone and lumber sleeps.
+The captains are long dead.
 Only the sea knows their end.
 |main_page                        = Shipyard Map
 }}