--- Area:MapWorldsBoneCryptUnique
+++ Export
@@ -11,13 +11,13 @@
 |boss_monster_ids                 = Metadata/Monsters/Totems/olmec_totem/MapTotemBossLegendsMain
 |is_map_area                      = True
 |is_unique_map_area               = True
-|flavour_text                     = They flew, and leapt,
-and clambered over,
-They crawled, and swam,
-and slithered under.
-Still its ancient secrets
-await unclaimed
-And of this hidden temple,
+|flavour_text                     = They flew, and leapt,
+and clambered over,
+They crawled, and swam,
+and slithered under.
+Still its ancient secrets
+await unclaimed
+And of this hidden temple,
 only legends remain.
 |main_page                        = Olmec's Sanctum
 |screenshot_ext                   = png
