--- Area:MapWorldsCursedCryptUnique
+++ Export
@@ -11,13 +11,13 @@
 |boss_monster_ids                 = Metadata/Monsters/Necromancer/NecromancerUniqueMapBoss
 |is_map_area                      = True
 |is_unique_map_area               = True
-|flavour_text                     = For his sins
-he was hunted in his sleep,
-judgement wrought
-of twisted nightmare.
-The Coward bound
-his dreams to stone,
-awaiting another to walk
+|flavour_text                     = For his sins
+he was hunted in his sleep,
+judgement wrought
+of twisted nightmare.
+The Coward bound
+his dreams to stone,
+awaiting another to walk
 the path of true atonement.
 |main_page                        = The Coward's Trial
 }}