--- Area:MapWorldsCemeteryUnique
+++ Export
@@ -10,12 +10,12 @@
 |boss_monster_ids                 = Metadata/Monsters/Brute/BossBruteUniqueMap
 |is_map_area                      = True
 |is_unique_map_area               = True
-|flavour_text                     = Some dead have long
-envied the living.
-When given the chance,
-they take from us what they can.
-So we offer our suffrage.
-In return, they give
+|flavour_text                     = Some dead have long
+envied the living.
+When given the chance,
+they take from us what they can.
+So we offer our suffrage.
+In return, they give
 only suffering.
 |main_page                        = Hallowed Ground
 }}