--- Area:MapWorldsLeyline
+++ Export
@@ -4,14 +4,15 @@
 |act                              = 11
 |area_level                       = 70
 |area_type_tags                   = urban, dungeon
-|tags                             = indoors_area, map, area_with_water, einharmaps, rain_area, archer_boss_area
+|tags                             = indoors_area, map, area_with_water, einharmaps, rain_area, archer_boss_area, sun_nature_area
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/SkeletonSoldier/SkeletonSoldierRangedMapBoss
 |entry_text                       = Amidst the barren wastes, a slender strand of life blooms.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea4_4
 |is_map_area                      = True
+|flavour_text                     = Amidst the barren wastes,
+a slender strand of life blooms.
 |main_page                        = Leyline Map
 }}