--- Area:2~8~7~2
+++ Export
@@ -9,8 +9,8 @@
 |connection_ids                   = 2_8_7_1_
 |parent_area_id                   = 2_8_town
 |boss_monster_ids                 = Metadata/Monsters/GuardianOfLunaris/GuardianOfLunaris
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea8_7
 |flavour_text                     = Gibbous beauty eclipsed by a sisters' feud.
 |main_page                        = The Lunaris Temple Level 2 (Act 8)
+|vaal_area_spawn_chance           = 18
 }}