--- Area:1~3~17~2
+++ Export
@@ -9,7 +9,6 @@
 |connection_ids                   = 1_3_17_1
 |parent_area_id                   = 1_3_town
 |boss_monster_ids                 = Metadata/Monsters/Undying/CityStalkerMaleBoss
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea3_12
 |flavour_text                     = All the knowledge in the Empire could not save them.
 |main_page                        = The Archives
