--- Area:MapWorldsFields
+++ Export
@@ -4,14 +4,12 @@
 |act                              = 11
 |area_level                       = 73
 |area_type_tags                   = forest, urban
-|tags                             = map, area_with_water, cannot_be_twinned, einharmaps, fields_area, has_foxes_area, crossroad_area, god_boss_map_area
+|tags                             = map, area_with_water, cannot_be_twinned, einharmaps, fields_area, has_foxes_area, crossroad_area, god_boss_map_area, sun_nature_area
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/Ralakesh/Greust/GreustMap
 |entry_text                       = They pleaded for the gods to bless their land. Now they lie under the soil.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = They pleaded for the gods to bless their land.
-Now they lie under the soil.
 |main_page                        = Fields Map
 }}