--- Area:1~4~3~3
+++ Export
@@ -9,9 +9,9 @@
 |connection_ids                   = 1_4_3_2, 1_4_4_1, 1_4_5_1, 1_4_6_1
 |parent_area_id                   = 1_4_town
 |boss_monster_ids                 = Metadata/Monsters/Miner/MinerLanternBoss
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea4_3
 |has_waypoint                     = True
 |flavour_text                     = Gifts of the Beast, awaiting plunder.
 |main_page                        = The Crystal Veins
+|vaal_area_spawn_chance           = 18
 }}