--- Area:MapWorldsAtoll
+++ Export
@@ -11,7 +11,7 @@
 |entry_text                       = Time turns even the vicious volcanoes to hollow shells.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Time turns even the vicious
+|flavour_text                     = Time turns even the vicious
 volcanoes to hollow shells.
 |main_page                        = Atoll Map
 |screenshot_ext                   = png
