--- Area:MapWorldsCanyon
+++ Export
@@ -10,10 +10,9 @@
 |boss_monster_ids                 = Metadata/Monsters/Pitbull/PitbullMapBoss, Metadata/Monsters/Cockerel/CockerelBowMapBoss
 |entry_text                       = A deep wound in the land. One that will never heal.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea1_6
 |is_map_area                      = True
-|flavour_text                     = A deep wound in the land.
+|flavour_text                     = A deep wound in the land.
 One that will never heal.
 |main_page                        = Canyon Map
 |screenshot_ext                   = png
