--- Area:MapWorldsSunkenCity
+++ Export
@@ -4,12 +4,14 @@
 |act                              = 11
 |area_level                       = 82
 |area_type_tags                   = dungeon
-|tags                             = map, area_with_water, cannot_be_twinned, einharmaps, has_foxes_area, ruined_vaal_city_area, spider_boss_area, god_boss_map_area
+|tags                             = map, area_with_water, cannot_be_twinned, einharmaps, has_foxes_area, ruined_vaal_city_area, spider_boss_area, god_boss_map_area, city_home_map, arakaali_area
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/VaalSpiderGod/ArakaaliMap
 |entry_text                       = From the walls, water, and ferns... Death comes crawling on ancient steps.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
+|flavour_text                     = From the walls, water, and ferns...
+Death comes crawling on ancient steps.
 |main_page                        = Sunken City Map
 }}