--- Area:1~2~2a
+++ Export
@@ -9,8 +9,8 @@
 |connection_ids                   = 1_2_2
 |parent_area_id                   = 1_2_town
 |boss_monster_ids                 = Metadata/Monsters/Beasts/BeastBossWhite
-|vaal_area_spawn_chance           = 15
 |vaal_area_ids                    = 1_SideArea2_3
 |flavour_text                     = Hot and sour as the panting of a rabid bear.
 |main_page                        = The Den (Act 2)
+|vaal_area_spawn_chance           = 15
 }}