--- Area:2~7~12~2
+++ Export
@@ -4,12 +4,11 @@
 |act                              = 7
 |area_level                       = 55
 |area_type_tags                   = southernforest2
-|tags                             = indoors_area, area_with_water, no_tempests, act_boss_area, einharthornspiders, vaal_pyramid_area, temple_of_decay_area, vile_areas
+|tags                             = indoors_area, area_with_water, no_tempests, act_boss_area, einharthornspiders, vaal_pyramid_area, temple_of_decay_area, vile_areas, arakaali_area
 |loading_screen                   = Act2
 |connection_ids                   = 2_7_12_1, 2_8_1
 |parent_area_id                   = 2_7_town
 |boss_monster_ids                 = Metadata/Monsters/VaalSpiderGod/Arakaali
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea7_12
 |flavour_text                     = Arachnoid servants dribble and drool, coveting pilgrim flesh to feed abhorrent desires.
 |main_page                        = The Temple of Decay Level 2
