--- Area:2~6~7~1
+++ Export
@@ -8,9 +8,9 @@
 |loading_screen                   = Act1
 |connection_ids                   = 2_6_6, 2_6_7_2
 |parent_area_id                   = 2_6_town
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea1_5
 |has_waypoint                     = True
 |flavour_text                     = Echoes of the past bring new life to an old experiment.
 |main_page                        = The Lower Prison (Act 6)
+|vaal_area_spawn_chance           = 18
 }}