--- Area:MapWorldsShoreUnique
+++ Export
@@ -10,8 +10,8 @@
 |boss_monster_ids                 = Metadata/Monsters/Skeletons/SkeletonBossFairgravesTreasure
 |is_map_area                      = True
 |is_unique_map_area               = True
-|flavour_text                     = The Treasures of the Tides
-wait in agony
+|flavour_text                     = The Treasures of the Tides
+wait in agony
 at the End of the World.
 |main_page                        = Mao Kun
 }}