--- Area:MapWorldsAridLake
+++ Export
@@ -10,10 +10,9 @@
 |boss_monster_ids                 = Metadata/Monsters/Rhoas/MapRhoaBoss
 |entry_text                       = Water vanishes; an impromptu prison becomes a tomb.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea1_1, MapSideArea1_6
 |is_map_area                      = True
-|flavour_text                     = Water vanishes; an impromptu
+|flavour_text                     = Water vanishes; an impromptu
 prison becomes a tomb.
 |main_page                        = Arid Lake Map
 |screenshot_ext                   = png
