--- Area:1~2~14~3
+++ Export
@@ -8,8 +8,8 @@
 |loading_screen                   = Act2
 |connection_ids                   = 1_2_14_2, 1_3_1
 |parent_area_id                   = 1_2_town
-|vaal_area_spawn_chance           = 10
 |vaal_area_ids                    = 1_SideArea2_8
 |flavour_text                     = Two thousand years of regret.
 |main_page                        = The Ancient Pyramid
+|vaal_area_spawn_chance           = 10
 }}