--- Area:MapWorldsUndergroundRiver
+++ Export
@@ -11,7 +11,7 @@
 |entry_text                       = Life, like still water, becomes trapped. Stagnant. Foul.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Life, like still water, becomes trapped.
+|flavour_text                     = Life, like still water, becomes trapped.
 Stagnant. Foul.
 |main_page                        = Underground River Map
 }}