--- Area:MapWorldsOvergrownShrine
+++ Export
@@ -11,7 +11,7 @@
 |entry_text                       = A once great monument, now a forgotten memorial.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = A once great monument,
+|flavour_text                     = A once great monument,
 now a forgotten memorial.
 |main_page                        = Overgrown Shrine Map
 }}