--- Area:MapWorldsStrandUnique
+++ Export
@@ -4,14 +4,14 @@
 |act                              = 11
 |area_level                       = 71
 |area_type_tags                   = beach
-|tags                             = unique_map, map, limited_tempests, area_with_water, domination_leaguestone, coast_boat_area, map_area_with_open_water, ocean_area, sea_witch_boss_area, rogue_exile_boss_area
+|tags                             = unique_map, map, limited_tempests, area_with_water, domination_leaguestone, coast_boat_area, map_area_with_open_water, ocean_area, sea_witch_boss_area, rogue_exile_boss_area, ship_graveyard_area
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |modifier_ids                     = MapCaganHasHiddenMods
 |boss_monster_ids                 = Metadata/Monsters/Seawitch/DarkSeaWitch
 |is_map_area                      = True
 |is_unique_map_area               = True
-|flavour_text                     = We all began life in darkness,
+|flavour_text                     = We all began life in darkness,
 we shall all end it there.
 |main_page                        = Whakawairua Tuahu
 }}