--- Area:2~7~9
+++ Export
@@ -9,7 +9,6 @@
 |connection_ids                   = 2_7_8
 |parent_area_id                   = 2_7_town
 |boss_monster_ids                 = Metadata/Monsters/DespairBear/DespairBear, Metadata/Monsters/ChaosElemental/ChaosElementalWorldUnique
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea2_7
 |flavour_text                     = Dripping with anger and grief, the ancient thicket weeps.
 |main_page                        = The Dread Thicket (Act 7)
