--- Area:1~4~3~1
+++ Export
@@ -9,8 +9,8 @@
 |connection_ids                   = 1_4_town, 1_4_3_2
 |parent_area_id                   = 1_4_town
 |boss_monster_ids                 = Metadata/Monsters/Miner/MinerHammerBoss
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea4_3
 |flavour_text                     = Disease in the darkness.
 |main_page                        = The Mines Level 1
+|vaal_area_spawn_chance           = 18
 }}