--- Area:MapWorldsSpiderLair
+++ Export
@@ -10,7 +10,6 @@
 |boss_monster_ids                 = Metadata/Monsters/Revenant/RevenantMapBoss
 |entry_text                       = Panic only hastens the venom's passage to the heart.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea2_7
 |is_map_area                      = True
 |main_page                        = Spider Lair Map
