--- Area:MapWorldsTempleUnique
+++ Export
@@ -10,10 +10,10 @@
 |boss_monster_ids                 = Metadata/Monsters/DemonModular/DemonFemaleRangedMapBoss
 |is_map_area                      = True
 |is_unique_map_area               = True
-|flavour_text                     = Laughs of all the twisted
-Echo through these
-halls of gold.
-In this cold forgotten void
+|flavour_text                     = Laughs of all the twisted
+Echo through these
+halls of gold.
+In this cold forgotten void
 They wander, forever uncontrolled.
 |main_page                        = Poorjoy's Asylum
 }}