--- Area:MapWorldsCoralRuins
+++ Export
@@ -11,7 +11,7 @@
 |entry_text                       = Thrust above the waves, still teeming with horrors from the depths.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Thrust above the waves, still teeming
+|flavour_text                     = Thrust above the waves, still teeming
 with horrors from the depths.
 |main_page                        = Coral Ruins Map
 }}