--- Area:MapWorldsPier
+++ Export
@@ -11,5 +11,8 @@
 |entry_text                       = Man's reach extends into the ocean, clawing at its infinite depths.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
+|flavour_text                     = Man's reach extends into
+the ocean, clawing at
+its infinite depths.
 |main_page                        = Pier Map
 }}