--- Area:MapWorldsStagnation
+++ Export
@@ -8,12 +8,11 @@
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/BeehiveBehemoth/BeehiveBehemothBoss
-|entry_text                       = Aided by unrelenting rain, rot devours new life at the first fumble.
+|entry_text                       = The trees grow twisted and gnarled, without a hint of light. A curse lingers here.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea6_4
 |is_map_area                      = True
-|flavour_text                     = Aided by unrelenting rain,
+|flavour_text                     = Aided by unrelenting rain,
 rot devours new life at the first fumble.
 |main_page                        = Stagnation Map
 }}