--- Area:AfflictionTown4
+++ Export
@@ -5,7 +5,7 @@
 |area_level                       = 75
 |tags                             = map, no_leaguestone, map_not_on_atlas
 |loading_screen                   = Delirium
-|modifier_ids                     = MapUseChaosPacksWithVariety_, MapSimulacrumHiddenReducedPackSize_, MapDropNoItems, MapNoArrowAttachment
+|modifier_ids                     = MapUseChaosPacksWithVariety_, MapSimulacrumHiddenReducedPackSize_, MapDropNoItems, MapNoArrowAttachment, MapMaximumArchnemesisModsIsTwo
 |is_map_area                      = True
 |is_unique_map_area               = True
 |main_page                        = Hysteriagate