--- Area:MapWorldsPrecinct
+++ Export
@@ -11,7 +11,7 @@
 |entry_text                       = Crushed together and fenced in, to live out their lives like cattle.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Crushed together and fenced in,
+|flavour_text                     = Crushed together and fenced in,
 to live out their lives like cattle.
 |main_page                        = Precinct Map
 }}