--- Area:MapWorldsGorge
+++ Export
@@ -10,10 +10,7 @@
 |boss_monster_ids                 = Metadata/Monsters/Goatman/GoatmanLeapSlamMapBoss
 |entry_text                       = So much power in the hands of so few.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea4_7
 |is_map_area                      = True
-|flavour_text                     = We must all walk the narrow, winding path.
-The end is always the same.
 |main_page                        = Glacier Map
 }}