--- Area:2~6~13
+++ Export
@@ -9,8 +9,8 @@
 |connection_ids                   = 2_6_12, 2_6_14
 |parent_area_id                   = 2_6_town
 |boss_monster_ids                 = Metadata/Monsters/NessaCrab/NessaCrab
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea1_1
 |flavour_text                     = Stalactites drip in mourning for an ocean queen.
 |main_page                        = The Cavern of Anger (Act 6)
+|vaal_area_spawn_chance           = 18
 }}