--- Area:1~3~18~2
+++ Export
@@ -10,7 +10,6 @@
 |parent_area_id                   = 1_3_town
 |boss_monster_ids                 = Metadata/Monsters/GemMonster/IguanaBoss, Metadata/Monsters/Pope/Pope, Metadata/Monsters/Axis/AxisEliteSoldierBoss1, Metadata/Monsters/Axis/AxisEliteSoldierBoss2, Metadata/Monsters/Axis/AxisCasterBoss1
 |monster_ids                      = Metadata/Monsters/Dominusdemon/Dominusdemon
-|vaal_area_spawn_chance           = 12
 |vaal_area_ids                    = 1_SideArea3_11
 |flavour_text                     = The pinnacle of power. The height of hubris.
 |main_page                        = The Upper Sceptre of God
