--- Area:MapWorldsCarcass
+++ Export
@@ -11,8 +11,6 @@
 |entry_text                       = Life's wet, reproductive stink abates, relenting to death's chaste ritual of decay.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Life's wet, reproductive stink abates,
-relenting to death's chaste ritual of decay.
 |main_page                        = Carcass Map
 |screenshot_ext                   = png
 }}