--- Area:MapWorldsBoneCrypt
+++ Export
@@ -11,8 +11,8 @@
 |entry_text                       = The dead ask to be remembered, but such requests are seldom met.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = The dead ask to be remembered,
-but such requests
+|flavour_text                     = The dead ask to be remembered,
+but such requests
 are seldom met.
 |main_page                        = Bone Crypt Map
 }}