--- Area:MapWorldsMausoleum
+++ Export
@@ -10,10 +10,10 @@
 |boss_monster_ids                 = Metadata/Monsters/Zombies/ZombieBossTolmanMap
 |entry_text                       = A vault for the dead, as if they were valuable. Pathetic.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea5_6
 |is_map_area                      = True
 |flavour_text                     = A vault for the dead, as if
 they were valuable. Pathetic.
 |main_page                        = Mausoleum Map
+|vaal_area_spawn_chance           = 8
 }}