--- Area:MapWorldsAncientCity
+++ Export
@@ -4,18 +4,14 @@
 |act                              = 11
 |area_level                       = 72
 |area_type_tags                   = urban
-|tags                             = map, area_with_water, einharmaps, vaal_ruin_area, ruined_vaal_city_area, ruined_area
+|tags                             = map, area_with_water, einharmaps, vaal_ruin_area, ruined_vaal_city_area, ruined_area, city_home_map, sun_nature_area
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/Axis/AxisEliteGuardMapBoss
 |entry_text                       = This was not the first civilization to believe itself the greatest. And it will not be the last.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea2_8
 |is_map_area                      = True
-|flavour_text                     = This was not the first
-civilization to believe itself
-the greatest. And it will
-not be the last.
 |main_page                        = Ancient City Map
+|vaal_area_spawn_chance           = 8
 }}