--- Area:MapWorldsTortureChamber
+++ Export
@@ -11,5 +11,7 @@
 |entry_text                       = Who is the true criminal, the one behind bars, or the one who tortures him?
 |entry_npc                        = The Shaper
 |is_map_area                      = True
+|flavour_text                     = Who is the true criminal, the one behind
+bars, or the one who tortures him?
 |main_page                        = Primordial Blocks Map
 }}