--- Area:1~1~3a
+++ Export
@@ -9,6 +9,7 @@
 |connection_ids                   = 1_1_3
 |parent_area_id                   = 1_1_town
 |boss_monster_ids                 = Metadata/Monsters/Necromancer/NecromancerBossFetidPool
+|vaal_area_ids                    = 1_SideArea1_1
 |flavour_text                     = Sluggish waters simmering with decay.
 |main_page                        = The Fetid Pool
 }}