--- Area:MapWorldsCourtyardUnique
+++ Export
@@ -10,13 +10,13 @@
 |boss_monster_ids                 = Metadata/Monsters/Viking/VikingBossVinktarsSquare
 |is_map_area                      = True
 |is_unique_map_area               = True
-|flavour_text                     = To the east, 
-it cannot be seen.
-To the west, 
-it cannot be touched.
-To the south, 
-it cannot be remembered.
-And to the north, 
+|flavour_text                     = To the east, 
+it cannot be seen.
+To the west, 
+it cannot be touched.
+To the south, 
+it cannot be remembered.
+And to the north, 
 it cannot be contained.
 |main_page                        = The Vinktar Square
 }}