--- Area:MapWorldsRamparts
+++ Export
@@ -11,5 +11,7 @@
 |entry_text                       = A wall is the surest sign for raiders that there is something of value to take.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
+|flavour_text                     = A wall is the surest sign for raiders
+that there is something of value to take.
 |main_page                        = Ramparts Map
 }}