--- Area:MapWorldsNecropolis
+++ Export
@@ -11,7 +11,7 @@
 |entry_text                       = Even in death, we long to be close to one another.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Even in death, we long to
+|flavour_text                     = Even in death, we long to
 be close to one another.
 |main_page                        = Necropolis Map
 }}