--- Area:LakePrototype
+++ Export
@@ -0,0 +1,10 @@
+{{Area
+|id                               = LakePrototype
+|name                             = The Lake of Kalandra
+|act                              = 11
+|area_level                       = 1
+|loading_screen                   = LakeOfKalandra
+|modifier_ids                     = MapMonsterNoMapDrops
+|is_map_area                      = True
+|is_unique_map_area               = True
+}}