--- Area:MapWorldsGraveyard
+++ Export
@@ -11,10 +11,10 @@
 |boss_monster_ids                 = Metadata/Monsters/Skeletons/MapSkeletonBossCaster, Metadata/Monsters/Skeletons/MapSkeletonBossRanged, Metadata/Monsters/Skeletons/MapSkeletonBossPrisonMelee
 |entry_text                       = Unknown legacies. Untold stories.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea2_3
 |is_map_area                      = True
 |flavour_text                     = Unknown legacies.
 Untold stories.
 |main_page                        = Graveyard Map
+|vaal_area_spawn_chance           = 8
 }}