--- Area:MapWorldsWasteland
+++ Export
@@ -11,7 +11,7 @@
 |entry_text                       = A white bone stripped of its fertile flesh. Life spirals towards its end.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = A white bone stripped of its fertile flesh.
+|flavour_text                     = A white bone stripped of its fertile flesh.
 Life spirals towards its end.
 |main_page                        = Wasteland Map
 }}