--- Area:2~7~1
+++ Export
@@ -8,7 +8,6 @@
 |loading_screen                   = Act2
 |connection_ids                   = 2_7_town, 2_7_2
 |parent_area_id                   = 2_7_town
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea2_4
 |flavour_text                     = The stonework trembles atop this petrified land.
 |main_page                        = The Broken Bridge (Act 7)
