--- Area:MapAtlasEstuary
+++ Export
@@ -4,7 +4,7 @@
 |act                              = 11
 |area_level                       = 79
 |area_type_tags                   = beach
-|tags                             = area_with_water, map, no_echo, einharmaps, fire_area, map_area_with_open_water, ocean_area
+|tags                             = area_with_water, map, no_echo, einharmaps, fire_area, map_area_with_open_water, ocean_area, karui_map
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/Bandits/FrostfireMageMapBoss