--- Area:MapWorldsAridLake
+++ Export
@@ -4,17 +4,14 @@
 |act                              = 11
 |area_level                       = 69
 |area_type_tags                   = swamp
-|tags                             = map, area_with_water, einharmaps
+|tags                             = map, area_with_water, einharmaps, sun_nature_area
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/Rhoas/MapRhoaBoss
 |entry_text                       = Water vanishes; an impromptu prison becomes a tomb.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea1_1, MapSideArea1_6
 |is_map_area                      = True
-|flavour_text                     = Water vanishes; an impromptu
-prison becomes a tomb.
 |main_page                        = Arid Lake Map
 |screenshot_ext                   = png
 }}