--- Area:MapWorldsSummit
+++ Export
@@ -11,7 +11,7 @@
 |entry_text                       = We are swept ever upwards by our ambition, with no thought of what awaits us at the peak.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = We are swept ever upwards by our ambition,
+|flavour_text                     = We are swept ever upwards by our ambition,
 with no thought of what awaits us at the peak.
 |main_page                        = Summit Map
 }}