--- Area:MapWorldsArena
+++ Export
@@ -11,6 +11,8 @@
 |entry_text                       = The sound of a thousand calls for blood echo for eternity.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
+|flavour_text                     = The sound of a thousand calls
+for blood echo for eternity.
 |main_page                        = Arena Map
 |screenshot_ext                   = png
 }}