--- Area:2~8~12~2
+++ Export
@@ -10,8 +10,8 @@
 |parent_area_id                   = 2_8_town
 |boss_monster_ids                 = Metadata/Monsters/GuardianOfSolaris/GuardianOfSolaris
 |monster_ids                      = Metadata/Monsters/Statue/GoldStatueMaleSword, Metadata/Monsters/Statue/GoldStatueMaleMaul, Metadata/Monsters/Statue/GoldStatueMaleBow, Metadata/Monsters/Statue/GoldStatueFemaleUnarmed, Metadata/Monsters/Statue/GoldStatueFemaleBow, Metadata/Monsters/Statue/GoldStatueFemaleMaul, Metadata/Monsters/Statue/GoldStatueFemaleSword
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea3_5
 |flavour_text                     = Luminary flares extinguished by burning pride.
 |main_page                        = The Solaris Temple Level 2 (Act 8)
+|vaal_area_spawn_chance           = 18
 }}