--- Area:MapWorldsMoonTempleUnique
+++ Export
@@ -10,7 +10,7 @@
 |boss_monster_ids                 = Metadata/Monsters/LunarisSolaris/SolarisUniqueMap, Metadata/Monsters/LunarisSolaris/LunarisUniqueMap
 |is_map_area                      = True
 |is_unique_map_area               = True
-|flavour_text                     = In the shadow of war,
+|flavour_text                     = In the shadow of war,
 all are eclipsed.
 |main_page                        = The Twilight Temple
 }}