--- Area:Synthesis~MapBoss
+++ Export
@@ -4,6 +4,7 @@
 |act                              = 11
 |area_level                       = 79
 |area_type_tags                   = synthesis_area
+|tags                             = synthesis_area
 |loading_screen                   = Synthesis
 |modifier_ids                     = SynthesisDisallowFracturedItemsMod
 |boss_monster_ids                 = Metadata/Monsters/LeagueSynthesis/SynthesisVenariusBoss
