--- Area:MapWorldsCells
+++ Export
@@ -10,10 +10,9 @@
 |boss_monster_ids                 = Metadata/Monsters/Shavronne/ShavronneSoulMap
 |entry_text                       = Life crushed together. Left to crumble to dust.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea3_2
 |is_map_area                      = True
-|flavour_text                     = Life crushed together.
+|flavour_text                     = Life crushed together.
 Left to crumble to dust.
 |main_page                        = Cells Map
 }}