--- Area:2~9~4
+++ Export
@@ -9,8 +9,8 @@
 |connection_ids                   = 2_9_3
 |parent_area_id                   = 2_9_town
 |boss_monster_ids                 = Metadata/Monsters/Shakari/Shakari, Metadata/Monsters/Shakari/Shakari2
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea9_3, 1_SideArea9_9
 |flavour_text                     = A paradise hidden amongst sunbleached bones and blistering dunes.
 |main_page                        = The Oasis
+|vaal_area_spawn_chance           = 18
 }}