--- Area:MapWorldsCage
+++ Export
@@ -10,10 +10,10 @@
 |boss_monster_ids                 = Metadata/Monsters/BlackguardInquisitor/BloodwingMapBoss
 |entry_text                       = Treat a man like a wild animal, and he will soon learn to bite.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea5_2
 |is_map_area                      = True
-|flavour_text                     = Treat a man like a wild animal,
+|flavour_text                     = Treat a man like a wild animal,
 and he will soon learn to bite.
 |main_page                        = Cage Map
+|vaal_area_spawn_chance           = 8
 }}