--- Area:MapAtlasPhoenix
+++ Export
@@ -4,7 +4,7 @@
 |act                              = 11
 |area_level                       = 83
 |area_type_tags                   = dungeon, temple
-|tags                             = map, no_echo, cannot_be_twinned, indoors_area, einharmaps, fire_area, fire_prison_area
+|tags                             = map, no_echo, cannot_be_twinned, indoors_area, einharmaps, fire_area, fire_prison_area, Forge_Area
 |loading_screen                   = ShaperMaps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/AtlasBosses/PhoenixBoss
