--- Area:2~8~8
+++ Export
@@ -10,7 +10,6 @@
 |parent_area_id                   = 2_8_town
 |boss_monster_ids                 = Metadata/Monsters/Zombies/ZombieBossTolman_
 |monster_ids                      = Metadata/Monsters/Statue/StoneStatueMaleSword, Metadata/Monsters/Statue/StoneStatueMaleMaul, Metadata/Monsters/Statue/StoneStatueMaleBow, Metadata/Monsters/Statue/StoneStatueFemaleUnarmed, Metadata/Monsters/Statue/StoneStatueFemaleBow, Metadata/Monsters/Statue/StoneStatueFemaleMaul, Metadata/Monsters/Statue/StoneStatueFemaleSword, Metadata/Monsters/Statue/StoneStatueLargeFemaleSpear, Metadata/Monsters/Statue/StoneStatueLargeFemaleSword, Metadata/Monsters/Statue/StoneStatueLargeMaleSpear, Metadata/Monsters/Statue/StoneStatueLargeMaleSword
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea3_3
 |flavour_text                     = The restless dead shift 'neath darkened waters.
 |main_page                        = The Quay
