--- Area:MapWorldsReef
+++ Export
@@ -11,5 +11,8 @@
 |entry_text                       = Each plank of rotting wood, each dead sailor, feeds the island and makes it grow.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
+|flavour_text                     = Each plank of rotting wood,
+each dead sailor,
+feeds the island and makes it grow.
 |main_page                        = Reef Map
 }}