--- Area:1~4~7
+++ Export
@@ -9,8 +9,8 @@
 |connection_ids                   = 1_4_town, 1_5_1
 |parent_area_id                   = 1_4_town
 |boss_monster_ids                 = Metadata/Monsters/Vendigo/VendigoBoss
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea4_7
 |flavour_text                     = The greater the rise, the greater the fall.
 |main_page                        = The Ascent
+|vaal_area_spawn_chance           = 18
 }}