--- Area:MapWorldsTribunal
+++ Export
@@ -10,8 +10,8 @@
 |boss_monster_ids                 = Metadata/Monsters/Scavengers/MapScavengerBossWitch
 |entry_text                       = We must all walk the narrow, winding path. The end is always the same.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideAreaLavaTunnel
 |is_map_area                      = True
+|flavour_text                     = So much power in the hands of so few.
 |main_page                        = Crater Map
 }}