--- Area:1~1~5
+++ Export
@@ -9,6 +9,7 @@
 |connection_ids                   = 1_1_4_1, 1_1_6
 |parent_area_id                   = 1_1_town
 |boss_monster_ids                 = Metadata/Monsters/Totems/TotemBossSpark
+|vaal_area_ids                    = 1_SideArea1_4_
 |has_waypoint                     = True
 |flavour_text                     = Dusty winds laden with the scent of bone, old and otherwise.
 |main_page                        = The Ledge