--- Area:MapWorldsThicket
+++ Export
@@ -11,7 +11,7 @@
 |entry_text                       = Little light enters. No life escapes.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Little light enters.
+|flavour_text                     = Little light enters.
 No life escapes.
 |main_page                        = Thicket Map
 }}