--- Area:1~5~4b
+++ Export
@@ -9,7 +9,6 @@
 |connection_ids                   = 1_5_5, 1_5_3b
 |parent_area_id                   = 1_5_town
 |boss_monster_ids                 = Metadata/Monsters/Axis/Cleric/BlackguardClericPureBoss
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea5_4b
 |flavour_text                     = Fuelled by blood and chaos, a dark inferno engulfs the halls of justice.
 |main_page                        = The Torched Courts (Act 5)
