--- Area:MapWorldsFloodedMine
+++ Export
@@ -10,10 +10,10 @@
 |boss_monster_ids                 = Metadata/Monsters/RockGolem/RockGolemMapBoss
 |entry_text                       = In darkness, beast and man alike clamour to survive.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea1_2
 |is_map_area                      = True
-|flavour_text                     = In darkness, beast and man
+|flavour_text                     = In darkness, beast and man
 alike clamour to survive.
 |main_page                        = Flooded Mine Map
+|vaal_area_spawn_chance           = 8
 }}