--- Area:Programming~Lite
+++ Export
@@ -2,7 +2,7 @@
 |id                               = Programming_Lite
 |name                             = Programming World (Lite)
 |act                              = 3
-|area_level                       = 90
+|area_level                       = 83
 |tags                             = area_with_water
 |loading_screen                   = HelenaHideout
 |parent_area_id                   = 2_11_endgame_town