--- Area:MapWorldsArcade
+++ Export
@@ -4,17 +4,17 @@
 |act                              = 11
 |area_level                       = 70
 |area_type_tags                   = urban
-|tags                             = map, einharmaps, marketplace_area, ruined_area
+|tags                             = map, einharmaps, marketplace_area, ruined_area, city_home_map
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/Guardians/MapGuardianBossFire, Metadata/Monsters/Guardians/MapGuardianBossLightning
 |entry_text                       = We create so that some part of us may remain after death.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea3_8
 |is_map_area                      = True
 |flavour_text                     = We create so that some part
 of us may remain after death.
 |main_page                        = Arcade Map
 |screenshot_ext                   = png
+|vaal_area_spawn_chance           = 8
 }}