--- Area:2~7~3
+++ Export
@@ -6,11 +6,11 @@
 |area_type_tags                   = southernforest
 |tags                             = area_with_water, einharbonerhoa, has_foxes_area, has_road_area, graveyard_area
 |loading_screen                   = Act2
-|connection_ids                   = 2_7_2, 2_7_4
+|connection_ids                   = 2_7_2, SanctumFoyer_Fellshrine, 2_7_4
 |parent_area_id                   = 2_7_town
 |boss_monster_ids                 = Metadata/Monsters/MassSkeleton/MassSkeletonBoss2
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea2_3
 |flavour_text                     = A spirit of sadism blemishes the earth.
 |main_page                        = The Fellshrine Ruins (Act 7)
+|vaal_area_spawn_chance           = 18
 }}