--- Area:1~1~4~0
+++ Export
@@ -10,6 +10,7 @@
 |parent_area_id                   = 1_1_town
 |boss_monster_ids                 = Metadata/Monsters/ShieldCrabs/ShieldCrabBossSlowAura
 |monster_ids                      = Metadata/Monsters/SandSpitters/SandSpitter2, Metadata/Monsters/SandSpitters/SandSpitterBoss
+|vaal_area_ids                    = 1_SideArea1_1
 |flavour_text                     = Darkness chilled into stillness.
 |main_page                        = The Flooded Depths
 }}