--- Area:MapWorldsIvoryTemple
+++ Export
@@ -10,10 +10,9 @@
 |boss_monster_ids                 = Metadata/Monsters/MilitaryMapBoss/IvoryTempleMilitaryBoss1, Metadata/Monsters/MilitaryMapBoss/IvoryTempleMilitaryBoss2
 |entry_text                       = Man and nature intertwined, but only for the chosen few.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea3_5, MapSideArea3_10
 |is_map_area                      = True
-|flavour_text                     = Man and nature intertwined,
+|flavour_text                     = Man and nature intertwined,
 but only for the chosen few.
 |main_page                        = Ivory Temple Map
 |screenshot_ext                   = png
