--- Area:1~1~7~2
+++ Export
@@ -9,6 +9,7 @@
 |connection_ids                   = 1_1_7_1, 1_1_8
 |parent_area_id                   = 1_1_town
 |boss_monster_ids                 = Metadata/Monsters/Necromancer/NecromancerBossPrison, Metadata/Monsters/Brute/BossBrute
+|vaal_area_ids                    = 1_SideArea1_5
 |flavour_text                     = The air is rich with despair and apparition.
 |main_page                        = The Upper Prison
 }}