--- Area:MapWorldsMarshes
+++ Export
@@ -11,7 +11,7 @@
 |entry_text                       = Embraced by mud. Grasped by roots. The careless are silently swallowed.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Embraced by mud. Grasped by roots.
+|flavour_text                     = Embraced by mud. Grasped by roots.
 The careless are silently swallowed.
 |main_page                        = Marshes Map
 }}