--- Area:1~3~14~2
+++ Export
@@ -9,8 +9,8 @@
 |connection_ids                   = 1_3_14_1
 |parent_area_id                   = 1_3_town
 |boss_monster_ids                 = Metadata/Monsters/Axis/PietyBoss, Metadata/Monsters/DemonFemale/DemonBossFemale
-|vaal_area_spawn_chance           = 12
 |vaal_area_ids                    = 1_SideArea3_9_
 |flavour_text                     = Depravity is boundless.
 |main_page                        = The Lunaris Temple Level 2 (Act 3)
+|vaal_area_spawn_chance           = 12
 }}