--- Area:MapWorldsCemetery
+++ Export
@@ -4,7 +4,7 @@
 |act                              = 11
 |area_level                       = 73
 |area_type_tags                   = temple, swamp
-|tags                             = map, cannot_be_twinned, einharmaps, graveyard_area, rain_area, cemetery_map_area, god_boss_map_area
+|tags                             = map, cannot_be_twinned, einharmaps, graveyard_area, rain_area, cemetery_map_area, god_boss_map_area, bear_boss_map
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/DespairBear/DespairBearMap
