--- Area:MapAtlasSprings
+++ Export
@@ -4,7 +4,7 @@
 |act                              = 11
 |area_level                       = 81
 |area_type_tags                   = forest
-|tags                             = map, einharmaps, indoors_area, cave_area, bandit_area
+|tags                             = map, einharmaps, indoors_area, cave_area, bandit_area, peaceful_map
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/BanditLeaderStrDex/MapBanditLeaderOak2