--- Area:2~8~1
+++ Export
@@ -8,9 +8,9 @@
 |loading_screen                   = Act3
 |connection_ids                   = 2_7_12_2, 2_8_town
 |parent_area_id                   = 2_7_town
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea2_3
 |has_waypoint                     = True
 |flavour_text                     = A parapet shakes beneath a billowing sky.
 |main_page                        = The Sarn Ramparts
+|vaal_area_spawn_chance           = 18
 }}