--- Area:2~8~2~2
+++ Export
@@ -9,7 +9,6 @@
 |connection_ids                   = 2_8_2_1, 2_8_3, 2_8_8
 |parent_area_id                   = 2_8_town
 |boss_monster_ids                 = Metadata/Monsters/Doedre/DoedreSewers
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea3_7
 |has_waypoint                     = True
 |flavour_text                     = Spells of blood foul the waters below.
