--- Area:2~9~9
+++ Export
@@ -9,7 +9,6 @@
 |connection_ids                   = 2_9_8
 |parent_area_id                   = 2_9_town
 |boss_monster_ids                 = Metadata/Monsters/GeneralAdus/GeneralAdus
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea4_1
 |flavour_text                     = Madness was visited upon those who dug too deep.
 |main_page                        = The Refinery
