--- Area:2~7~6
+++ Export
@@ -9,7 +9,6 @@
 |connection_ids                   = 2_7_5_2, 2_7_7
 |parent_area_id                   = 2_7_town
 |boss_monster_ids                 = Metadata/Monsters/Beasts/BeastBossWhiteSkeletal, Metadata/Monsters/Necromancer/NecromancerSkeletalBeast
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea2_3
 |has_waypoint                     = True
 |flavour_text                     = Sodden spores flourish and feast on decaying flesh.
