--- Area:1~4~3~2
+++ Export
@@ -9,8 +9,8 @@
 |connection_ids                   = 1_4_3_1, 1_4_3_3
 |parent_area_id                   = 1_4_town
 |boss_monster_ids                 = Metadata/Monsters/Handfinger/HandfingerBoss, Metadata/Monsters/MineBat/MineBatBoss
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea4_3
 |flavour_text                     = Disease in the darkness.
 |main_page                        = The Mines Level 2
+|vaal_area_spawn_chance           = 18
 }}