--- Area:MapWorldsArsenal
+++ Export
@@ -11,7 +11,7 @@
 |entry_text                       = There is no war. There is no army. Only rage lingers.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = There is no war. There is no army.
+|flavour_text                     = There is no war. There is no army.
 Only rage lingers.
 |main_page                        = Arsenal Map
 }}