--- Area:2~9~8
+++ Export
@@ -9,7 +9,6 @@
 |connection_ids                   = 2_9_7, 2_9_9, 2_9_10_1
 |parent_area_id                   = 2_9_town
 |boss_monster_ids                 = Metadata/Monsters/Garukhan/Garukhan
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea4_3
 |has_waypoint                     = True
 |flavour_text                     = A monstrous terror stalks chthonic mines, yearning for the boundless sky.
