--- Area:1~2~11
+++ Export
@@ -8,8 +8,8 @@
 |loading_screen                   = Act2
 |connection_ids                   = 1_2_12, 1_2_8
 |parent_area_id                   = 1_2_town
-|vaal_area_spawn_chance           = 12
 |vaal_area_ids                    = 1_SideArea2_8
 |flavour_text                     = Shades scratching at imprisoning stone.
 |main_page                        = The Vaal Ruins
+|vaal_area_spawn_chance           = 12
 }}