--- Area:MapWorldsMinotaur
+++ Export
@@ -11,5 +11,8 @@
 |entry_text                       = One of my captors felt no emotion. He did not hesitate to inflict pain. Now all he feels is intense, unstoppable agony.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
+|flavour_text                     = One of my captors felt no emotion.
+He did not hesitate to inflict pain.
+Now all he feels is intense, unstoppable agony.
 |main_page                        = Maze of the Minotaur Map
 }}