--- Area:1~5~2
+++ Export
@@ -9,8 +9,8 @@
 |connection_ids                   = 1_5_town, 1_5_3
 |parent_area_id                   = 1_5_town
 |boss_monster_ids                 = Metadata/Monsters/BlackguardInquisitor/Bloodwing
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea5_2
 |flavour_text                     = The illusion of control is swiftly severed by the blade of reality.
 |main_page                        = The Control Blocks (Act 5)
+|vaal_area_spawn_chance           = 18
 }}