--- Area:MapAtlasPhantasmagoria
+++ Export
@@ -4,7 +4,7 @@
 |act                              = 11
 |area_level                       = 71
 |area_type_tags                   = temple, cave
-|tags                             = indoors_area, map, einharmaps, belly_area, vile_areas, doedre_area, unholy_trio_area
+|tags                             = indoors_area, map, einharmaps, belly_area, vile_areas, doedre_area, unholy_trio_area, doedre_map_area
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/Doedre/DoedreMapBoss2
