--- Area:MapWorldsTerrace
+++ Export
@@ -11,5 +11,7 @@
 |entry_text                       = A garden on a stone pedestal. An incredible, and foolish, feat.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
+|flavour_text                     = A garden on a stone pedestal.
+An incredible, and foolish, feat.
 |main_page                        = Terrace Map
 }}