--- Area:MapWorldsCastleRuins
+++ Export
@@ -11,6 +11,8 @@
 |entry_text                       = When the castle falls, it has failed its only purpose.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
+|flavour_text                     = When the castle falls,
+it has failed its only purpose.
 |main_page                        = Castle Ruins Map
 |screenshot_ext                   = png
 }}