--- Area:MapWorldsCrimsonTemple
+++ Export
@@ -4,7 +4,7 @@
 |act                              = 11
 |area_level                       = 81
 |area_type_tags                   = temple
-|tags                             = indoors_area, map, delve_chest, einharmaps, wealthy_area, church, kitava_cultist_area
+|tags                             = indoors_area, map, delve_chest, einharmaps, wealthy_area, church, kitava_cultist_area, blood_mage_boss_map
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/Banshee/BansheeMapBoss