--- Area:MapTier5~5
+++ Export
@@ -4,7 +4,7 @@
 |act                              = 11
 |area_level                       = 70
 |area_type_tags                   = beach, swamp
-|tags                             = map, area_with_water, einharmaps, reef_area, ship_graveyard_area, map_area_with_open_water, ocean_area, golem_boss_area
+|tags                             = map, area_with_water, einharmaps, reef_area, ship_graveyard_area, map_area_with_open_water, ocean_area, golem_boss_area, karui_map
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/GhostPirates/GhostPirateMapBoss2, Metadata/Monsters/RockGolem/TreasureGolemMapBossSimplified
