--- Area:MapWorldsAshenWood
+++ Export
@@ -11,9 +11,5 @@
 |entry_text                       = Desperately reaching skyward and digging downwards. Sustenance is an eternal struggle.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Desperately reaching skyward
-and digging downwards.
-Sustenance is an
-eternal struggle.
 |main_page                        = Ashen Wood Map
 }}