--- Area:1~3~6
+++ Export
@@ -9,7 +9,6 @@
 |connection_ids                   = 1_3_5
 |parent_area_id                   = 1_3_town
 |monster_ids                      = Metadata/Monsters/Statue/StoneStatueMaleSword, Metadata/Monsters/Statue/StoneStatueMaleMaul, Metadata/Monsters/Statue/StoneStatueMaleBow, Metadata/Monsters/Statue/StoneStatueFemaleUnarmed, Metadata/Monsters/Statue/StoneStatueFemaleBow, Metadata/Monsters/Statue/StoneStatueFemaleMaul, Metadata/Monsters/Statue/StoneStatueFemaleSword
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea3_4
 |flavour_text                     = Coughing up the dust of dead devotion.
 |main_page                        = The Catacombs
