--- Area:MapWorldsLighthouse
+++ Export
@@ -10,7 +10,6 @@
 |boss_monster_ids                 = Metadata/Monsters/Wb/WbFireLeader1, Metadata/Monsters/Wb/WbFireLeader2, Metadata/Monsters/Wb/WbFireLeader3, Metadata/Monsters/Wb/WbFireLeader4
 |entry_text                       = Fire ignites the sky. A signal to keep away. An invitation to invade.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea1_4
 |is_map_area                      = True
 |main_page                        = Lighthouse Map
