--- Area:2~9~3
+++ Export
@@ -8,7 +8,6 @@
 |loading_screen                   = Act4
 |connection_ids                   = 2_9_2, 2_9_4, 2_9_5
 |parent_area_id                   = 2_9_town
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea9_3
 |has_waypoint                     = True
 |flavour_text                     = Blustering sand, sharper than a bandit's blade.
