--- Area:MapWorldsMoonTemple
+++ Export
@@ -11,7 +11,7 @@
 |entry_text                       = Night's promises are dark, for even the stars must be harvested.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Night's promises are dark, for
+|flavour_text                     = Night's promises are dark, for
 even the stars must be harvested.
 |main_page                        = Moon Temple Map
 }}