--- Area:2~9~6
+++ Export
@@ -9,7 +9,6 @@
 |connection_ids                   = 2_9_5
 |parent_area_id                   = 2_9_town
 |boss_monster_ids                 = Metadata/Monsters/GemMonster/IguanaBossPhoenix
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea9_6
 |flavour_text                     = These corrosive waters devour both flesh and bone.
 |main_page                        = The Boiling Lake
