--- Area:MapWorldsSulphurVents
+++ Export
@@ -10,8 +10,9 @@
 |boss_monster_ids                 = Metadata/Monsters/GemMonster/IguanaBossPhoenixMap
 |entry_text                       = The creatures that prowl the wildest extremes, nature anoints with cruel efficacy to defend her.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea9_6
 |is_map_area                      = True
+|flavour_text                     = The creatures that prowl the wildest extremes,
+nature anoints with cruel efficacy to defend her.
 |main_page                        = Sulphur Vents Map
 }}