--- Area:MapWorldsBeach
+++ Export
@@ -4,17 +4,17 @@
 |act                              = 11
 |area_level                       = 68
 |area_type_tags                   = beach
-|tags                             = map, einharmaps, ocean_area
+|tags                             = map, einharmaps, ocean_area, peaceful_map, sun_nature_area
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/Scavengers/MapScavengerBossHailrake
 |entry_text                       = A paradise, forever replenished by endless waves teeming with life.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea1_3
 |is_map_area                      = True
-|flavour_text                     = A paradise, forever replenished by
+|flavour_text                     = A paradise, forever replenished by
 endless waves teeming with life.
 |main_page                        = Beach Map
 |screenshot_ext                   = png
+|vaal_area_spawn_chance           = 8
 }}