--- Area:MapAtlasTower
+++ Export
@@ -4,7 +4,7 @@
 |act                              = 11
 |area_level                       = 72
 |area_type_tags                   = dungeon
-|tags                             = indoors_area, map, cannot_be_twinned, delve_chest, einharmaps, tower_area, lightning_area, rain_area, shavronne_area, unholy_trio_area
+|tags                             = indoors_area, map, cannot_be_twinned, delve_chest, einharmaps, tower_area, lightning_area, rain_area, shavronne_area, unholy_trio_area, city_home_map
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/Shavronne/ShavronneTowerMap, Metadata/Monsters/Brute/BruteTowerMap_