--- Area:1~3~17~1
+++ Export
@@ -8,7 +8,6 @@
 |loading_screen                   = Act3
 |connection_ids                   = 1_3_15, 1_3_17_2
 |parent_area_id                   = 1_3_town
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea3_12
 |has_waypoint                     = True
 |flavour_text                     = All the knowledge in the Empire could not save them.
