--- Area:1~1~9a
+++ Export
@@ -9,6 +9,7 @@
 |connection_ids                   = 1_1_9
 |parent_area_id                   = 1_1_town
 |boss_monster_ids                 = Metadata/Monsters/WaterElemental/WaterElementaBossShipCave
+|vaal_area_ids                    = 1_SideArea1_1
 |flavour_text                     = A corpse's throat, exhaling death.
 |main_page                        = The Ship Graveyard Cave
 }}