--- Area:1~4~2
+++ Export
@@ -9,8 +9,8 @@
 |connection_ids                   = 1_4_town
 |parent_area_id                   = 1_4_town
 |boss_monster_ids                 = Metadata/Monsters/Voll/VollBoss, Metadata/Monsters/SkeletonSoldier/SkeletonSoldierRangedBoss
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea4_2, 1_SideArea4_4
 |flavour_text                     = Bones of betrayal, ashes of purity.
 |main_page                        = The Dried Lake
+|vaal_area_spawn_chance           = 18
 }}