--- Area:2~8~4
+++ Export
@@ -9,7 +9,6 @@
 |connection_ids                   = 2_8_5
 |parent_area_id                   = 2_8_town
 |boss_monster_ids                 = Metadata/Monsters/Frog/FrogGod/SilverFrog
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea3_12
 |flavour_text                     = Creeping dread chokes life from this thriving terrace.
 |main_page                        = The High Gardens
