--- Area:MapWorldsSilo
+++ Export
@@ -4,14 +4,14 @@
 |act                              = 11
 |area_level                       = 72
 |area_type_tags                   = urban, dungeon
-|tags                             = map, einharmaps, indoors_area, fire_area, cannot_be_twinned, kitava_cultist_area
+|tags                             = map, einharmaps, indoors_area, fire_area, cannot_be_twinned, kitava_cultist_area, Forge_Area
 |loading_screen                   = ShaperMaps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/Vilenta/VilentaMap_
 |entry_text                       = Fiend and fortune-hunter alike turned to ash. A sorry alliance of unending agony.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Grim blessings whispered in the deepest,
+|flavour_text                     = Grim blessings whispered in the deepest,
 most secret of places.
 |main_page                        = Silo Map
 }}