--- Area:MapAtlasTropicalIslandUnique
+++ Export
@@ -7,7 +7,7 @@
 |tags                             = unique_map, map, limited_tempests, area_with_water, no_strongboxes, anarchy_leaguestone, torment_leaguestone, invasion_leaguestone, warbands_leaguestone, ambush_leaguestone, domination_leaguestone, talisman_leaguestone, perandus_leaguestone, prophecy_leaguestone, essence_leaguestone, breach_leaguestone, map_area_with_open_water
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
-|modifier_ids                     = MapNoExiles
+|modifier_ids                     = MapNoExiles, MapMaximumArchnemesisModsIsTwo
 |boss_monster_ids                 = Metadata/Monsters/BloodChieftain/MonkeyChiefBloodEnrageUntaintedColossal, Metadata/Monsters/SandSpitters/SandSpitterUntaintedColossal, Metadata/Monsters/Rhoas/RhoaUntaintedColossal, Metadata/Monsters/Frog/FrogUntaintedColossal, Metadata/Monsters/GemMonster/IguanaUntaintedColossal, Metadata/Monsters/RootSpiders/RootSpiderHiddenUntaintedColossal
 |is_map_area                      = True
 |is_unique_map_area               = True
