--- Area:2~7~12~1
+++ Export
@@ -9,8 +9,8 @@
 |connection_ids                   = 2_7_11, 2_7_12_2
 |parent_area_id                   = 2_7_town
 |boss_monster_ids                 = Metadata/Monsters/Skeletons/SkeletonMeleeKnightElementalLargeSwordBoss
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea7_12
 |flavour_text                     = Arachnoid servants dribble and drool, coveting pilgrim flesh to feed abhorrent desires.
 |main_page                        = The Temple of Decay Level 1
+|vaal_area_spawn_chance           = 18
 }}