--- Area:MapWorldsTortureChamberUnique
+++ Export
@@ -10,9 +10,9 @@
 |modifier_ids                     = MapUniqueObasTrial
 |is_map_area                      = True
 |is_unique_map_area               = True
-|flavour_text                     = "Here the soil is more deadly
-than any swing of the axe."
-- Oba of the Karui,
+|flavour_text                     = "Here the soil is more deadly
+than any swing of the axe."
+- Oba of the Karui,
 Conqueror of Corruption.
 |main_page                        = Oba's Cursed Trove
 }}