--- Area:MapWorldsHydra
+++ Export
@@ -11,5 +11,8 @@
 |entry_text                       = One of my captors wore many faces, yet kept her true face hidden. Now she cannot escape her true face.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
+|flavour_text                     = One of my captors wore many faces,
+yet kept her true face hidden.
+Now she cannot escape her true face.
 |main_page                        = Lair of the Hydra Map
 }}