--- Area:1~3~2
+++ Export
@@ -9,8 +9,8 @@
 |connection_ids                   = 1_3_town, 1_3_3_1, 1_3_10_1
 |parent_area_id                   = 1_3_town
 |boss_monster_ids                 = Metadata/Monsters/Undying/UndyingBossOutcast
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea3_1
 |flavour_text                     = Only the wind whistles and whines here now.
 |main_page                        = The Slums
+|vaal_area_spawn_chance           = 18
 }}