--- Area:MapWorldsVault
+++ Export
@@ -4,14 +4,14 @@
 |act                              = 11
 |area_level                       = 76
 |area_type_tags                   = temple, dungeon
-|tags                             = indoors_area, map, area_with_water, einharmaps, wealthy_area, gold_pile_area, golem_boss_area
+|tags                             = indoors_area, map, area_with_water, einharmaps, wealthy_area, gold_pile_area, golem_boss_area, inventing_area
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/RockGolem/TreasureGolemMapBoss
 |entry_text                       = Wealth beyond imagining, locked beyond reach.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Wealth beyond imagining,
+|flavour_text                     = Wealth beyond imagining,
 locked beyond reach.
 |main_page                        = Vault Map
 }}