--- Area:MapWorldsForbiddenWoods
+++ Export
@@ -10,11 +10,7 @@
 |boss_monster_ids                 = Metadata/Monsters/SaplingMonster/IcedSaplingFiendBoss, Metadata/Monsters/SaplingMonster/IcedSaplingTricksterBoss, Metadata/Monsters/SaplingMonster/IcedSaplingHunterBoss
 |entry_text                       = Little light enters. No life escapes.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideAreaIceForest
 |is_map_area                      = True
-|flavour_text                     = Every instinct wills you to sleep through the bitter cold;
-a certain death. Those that yet survive must fight
-through every aching movement.
 |main_page                        = Forbidden Woods Map
 }}