--- Area:Map2Tier10~3
+++ Export
@@ -4,7 +4,7 @@
 |act                              = 11
 |area_level                       = 77
 |area_type_tags                   = urban
-|tags                             = map, einharmaps, marketplace_area, undying_boss_area, ruined_area
+|tags                             = map, einharmaps, marketplace_area, undying_boss_area, ruined_area, city_home_map
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/Undying/CityStalkerMaleMapBossBazaar