--- Area:MapWorldsMalformation
+++ Export
@@ -11,8 +11,8 @@
 |entry_text                       = Boils swell from the earth. Foul malignant flesh without purpose.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Boils swell from the earth.
-Foul malignant flesh
+|flavour_text                     = Boils swell from the earth.
+Foul malignant flesh
 without purpose.
 |main_page                        = Malformation Map
 }}