--- Area:MapWorldsInfestedValley
+++ Export
@@ -11,7 +11,7 @@
 |entry_text                       = Should we feel disgusted at how the weak predate the strong, or should we feel... admiration?
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Should we feel disgusted at how the weak predate
+|flavour_text                     = Should we feel disgusted at how the weak predate
 the strong, or should we feel... admiration?
 |main_page                        = Infested Valley Map
 }}