--- Area:2~9~2
+++ Export
@@ -8,8 +8,8 @@
 |loading_screen                   = Act4
 |connection_ids                   = 2_9_town, 2_9_3
 |parent_area_id                   = 2_9_town
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea4_7
 |flavour_text                     = A freezing drop to test one's resolve.
 |main_page                        = The Descent
+|vaal_area_spawn_chance           = 18
 }}