--- Area:MapWorldsChimera
+++ Export
@@ -11,5 +11,7 @@
 |entry_text                       = One of my captors hid a beast within his heart. Now his true self can be seen by all.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
+|flavour_text                     = One of my captors hid a beast within his heart.
+Now his true self can be seen by all.
 |main_page                        = Pit of the Chimera Map
 }}