--- Area:2~8~6
+++ Export
@@ -9,9 +9,9 @@
 |connection_ids                   = 2_8_5, 2_8_7_1_, 2_8_13
 |parent_area_id                   = 2_8_town
 |boss_monster_ids                 = Metadata/Monsters/Axis/AxisEliteLunarisBoss_
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea3_8
 |has_waypoint                     = True
 |flavour_text                     = Allegiances falter like the waning of the moon.
 |main_page                        = The Lunaris Concourse
+|vaal_area_spawn_chance           = 18
 }}