--- Area:SanctumNave
+++ Export
@@ -0,0 +1,12 @@
+{{Area
+|id                               = SanctumNave
+|name                             = Sanctum Cathedral
+|act                              = 11
+|area_level                       = 1
+|loading_screen                   = SanctumCathedral
+|modifier_ids                     = MapMonsterNoMapDrops, MapSanctumRareMonsterLifePenalty
+|spawn_weight1_tag                = sanctum_key
+|spawn_weight1_value              = 0
+|spawn_weight2_tag                = default
+|spawn_weight2_value              = 1000
+}}