--- Area:MapWorldsLavaChamber
+++ Export
@@ -11,7 +11,7 @@
 |entry_text                       = Fiend and fortune-hunter alike turned to ash. A sorry alliance of unending agony.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Fiend and fortune-hunter alike turned to ash.
+|flavour_text                     = Fiend and fortune-hunter alike turned to ash.
 A sorry alliance of unending agony.
 |main_page                        = Lava Chamber Map
 }}