--- Area:MapWorldsNecropolisUnique
+++ Export
@@ -10,8 +10,8 @@
 |boss_monster_ids                 = Metadata/Monsters/Skeletons/SkeletonBossMapWest4
 |is_map_area                      = True
 |is_unique_map_area               = True
-|flavour_text                     = So the raiders danced,
-leading their foe ever west,
+|flavour_text                     = So the raiders danced,
+leading their foe ever west,
 towards certain death.
 |main_page                        = Death and Taxes
 }}