--- Area:MapWorldsDefiledCathedral
+++ Export
@@ -4,14 +4,14 @@
 |act                              = 11
 |area_level                       = 79
 |area_type_tags                   = dungeon, temple, urban
-|tags                             = indoors_area, map, cannot_be_twinned, delve_chest, einharmaps, church, kitava_cultist_area
+|tags                             = indoors_area, map, cannot_be_twinned, delve_chest, einharmaps, church, kitava_cultist_area, blood_mage_boss_map
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/AvariusCasticus/AvariusCasticusWickermanMap
 |entry_text                       = Crafted with reverence. Defiled with ease.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Crafted with reverence.
+|flavour_text                     = Crafted with reverence.
 Defiled with ease.
 |main_page                        = Defiled Cathedral Map
 }}