--- Area:1~1~11~1
+++ Export
@@ -9,9 +9,9 @@
 |connection_ids                   = 1_1_9, 1_1_11_2
 |parent_area_id                   = 1_1_town
 |boss_monster_ids                 = Metadata/Monsters/Seawitch/SeaWitchBossSpark
-|vaal_area_spawn_chance           = 12
 |vaal_area_ids                    = 1_SideArea1_1
 |has_waypoint                     = True
 |flavour_text                     = The air needles the skin with its frigid fury.
 |main_page                        = The Cavern of Wrath
+|vaal_area_spawn_chance           = 12
 }}