--- Area:MapWorldsCore
+++ Export
@@ -4,7 +4,7 @@
 |act                              = 11
 |area_level                       = 80
 |area_type_tags                   = temple, cave
-|tags                             = indoors_area, map, no_echo, area_with_water, einharmaps, belly_area, harvest_area, core_map_area, vile_areas, doedre_area, shavronne_area, maligaro_area, unholy_trio_area
+|tags                             = indoors_area, map, no_echo, area_with_water, einharmaps, belly_area, harvest_area, core_map_area, vile_areas, doedre_area, shavronne_area, maligaro_area, unholy_trio_area, piety_area
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/Malachai/MalachaiMapBoss, Metadata/Monsters/Malachai/MalachaiMapBoss2