--- Area:MapWorldsIceberg
+++ Export
@@ -10,10 +10,10 @@
 |boss_monster_ids                 = Metadata/Monsters/Wb/WbColdLeader1, Metadata/Monsters/Wb/WbColdLeader2, Metadata/Monsters/Wb/WbColdLeader3
 |entry_text                       = Icy fangs seek tender hulls to sink themselves into.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideAreaIceValley
 |is_map_area                      = True
 |flavour_text                     = Icy fangs seek tender hulls
 to sink themselves into.
 |main_page                        = Iceberg Map
+|vaal_area_spawn_chance           = 8
 }}