--- Area:MapWorldsMineralPools
+++ Export
@@ -11,8 +11,8 @@
 |entry_text                       = God's jar of raw materials. Life lying dormant, waiting to sprout.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = God's jar of raw materials.
-Life lying dormant,
+|flavour_text                     = God's jar of raw materials.
+Life lying dormant,
 waiting to sprout.
 |main_page                        = Mineral Pools Map
 }}