--- Area:1~2~10
+++ Export
@@ -9,8 +9,8 @@
 |connection_ids                   = 1_2_9
 |parent_area_id                   = 1_2_town
 |boss_monster_ids                 = Metadata/Monsters/Spiders/SpiderBossWeaver
-|vaal_area_spawn_chance           = 15
 |vaal_area_ids                    = 1_SideArea2_7
 |flavour_text                     = Shrouded in mourning silks.
 |main_page                        = The Weaver's Chambers
+|vaal_area_spawn_chance           = 15
 }}