--- Area:MapWorldsSepulchre
+++ Export
@@ -4,7 +4,7 @@
 |act                              = 11
 |area_level                       = 76
 |area_type_tags                   = dungeon
-|tags                             = indoors_area, map, delve_chest, einharmaps, mausoleum_area, doedre_area, unholy_trio_area, trio_soul_area
+|tags                             = indoors_area, map, delve_chest, einharmaps, mausoleum_area, doedre_area, unholy_trio_area, trio_soul_area, doedre_map_area
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/Doedre/DoedreSoul2Map, Metadata/Monsters/Doedre/DoedreSoulMap