--- Area:MapWorldsVaalPyramidUnique
+++ Export
@@ -9,8 +9,8 @@
 |parent_area_id                   = 2_11_endgame_town
 |is_map_area                      = True
 |is_unique_map_area               = True
-|flavour_text                     = "Should I depart this mortal coil,
-so shall all of my treasures."
+|flavour_text                     = "Should I depart this mortal coil,
+so shall all of my treasures."
 - Atziri, Queen of the Vaal
 |main_page                        = Vaults of Atziri
 }}