--- Area:MapWorldsPalace
+++ Export
@@ -11,8 +11,8 @@
 |entry_text                       = Death forces even the most determined of despots to pass the crown.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Death forces even the most
-determined of despots to
+|flavour_text                     = Death forces even the most
+determined of despots to
 pass the crown.
 |main_page                        = Palace Map
 }}