--- Area:MapWorldsDarkForest
+++ Export
@@ -11,8 +11,5 @@
 |entry_text                       = The trees grow twisted and gnarled, without a hint of light. A curse lingers here.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = The trees grow twisted and gnarled,
-without a hint of light.
-A curse lingers here.
 |main_page                        = Dark Forest Map
 }}