--- Area:BreachBossChaos
+++ Export
@@ -7,7 +7,7 @@
 |tags                             = indoors_area, map, breach_map, no_leaguestone, map_not_on_atlas
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
-|modifier_ids                     = MapRevealedMap, MapIncreasedExperienceGainBreach__3, BreachDomainTiming3, MapNoExiles, BreachstoneCommanderRewards1
+|modifier_ids                     = MapRevealedMap, MapIncreasedExperienceGainBreach__3, BreachDomainTiming3, MapNoExiles, BreachstoneCommanderRewards1, BreachstoneExtraRandomMods1_
 |boss_monster_ids                 = Metadata/Monsters/BreachBosses/BreachBossChaosMap
 |is_map_area                      = True
 |is_unique_map_area               = True