--- Area:SanctumFoyer~1~2
+++ Export
@@ -0,0 +1,12 @@
+{{Area
+|id                               = SanctumFoyer_1_2
+|name                             = Sanctum Archives
+|act                              = 11
+|area_level                       = 1
+|loading_screen                   = SanctumArchives
+|modifier_ids                     = MapMonsterNoMapDrops, MapSanctumRareMonsterLifePenalty
+|spawn_weight1_tag                = sanctum_key
+|spawn_weight1_value              = 0
+|spawn_weight2_tag                = default
+|spawn_weight2_value              = 1000
+}}