--- Area:2~9~7
+++ Export
@@ -8,7 +8,6 @@
 |loading_screen                   = Act4
 |connection_ids                   = 2_9_5, 2_9_8
 |parent_area_id                   = 2_9_town
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea4_3
 |has_waypoint                     = True
 |flavour_text                     = Unspeakable horrors filigree and haunt the nameless dark.
