--- Area:MapWorldsLaboratory
+++ Export
@@ -4,14 +4,14 @@
 |act                              = 11
 |area_level                       = 75
 |area_type_tags                   = urban, temple
-|tags                             = indoors_area, map, delve_chest, einharmaps, wealthy_area, thaumaturgy_area
+|tags                             = indoors_area, map, delve_chest, einharmaps, wealthy_area, thaumaturgy_area, inventing_area
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/Undying/CityStalkerFemaleMapBossVoid
 |entry_text                       = My last waking moments looked very much like this place... I miss it. I miss her.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = My last waking moments looked very
+|flavour_text                     = My last waking moments looked very
 much like this place... I miss it. I miss her.
 |main_page                        = Laboratory Map
 }}