--- Area:MapWorldsCrystalOre
+++ Export
@@ -12,5 +12,7 @@
 |entry_text                       = As the torch flickers, brilliance emerges in boundless variance.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
+|flavour_text                     = As the torch flickers, brilliance
+emerges in boundless variance.
 |main_page                        = Crystal Ore Map
 }}