--- Area:1~2~15
+++ Export
@@ -8,8 +8,8 @@
 |loading_screen                   = Act2
 |connection_ids                   = 1_2_3, 1_2_5_1
 |parent_area_id                   = 1_2_town
-|vaal_area_spawn_chance           = 15
 |vaal_area_ids                    = 1_SideArea2_3
 |flavour_text                     = The ground aches with desecration. The rain tastes of blight.
 |main_page                        = The Fellshrine Ruins (Act 2)
+|vaal_area_spawn_chance           = 15
 }}