--- Area:MapWorldsRelicChambers
+++ Export
@@ -10,7 +10,6 @@
 |boss_monster_ids                 = Metadata/Monsters/Skeletons/MapSkeletonBossMeleeSimplified
 |entry_text                       = Relics of a forbidden past. Hidden away. Forgotten.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea5_7
 |is_map_area                      = True
 |main_page                        = Relic Chambers Map
