--- Area:MapWorldsColosseum
+++ Export
@@ -11,8 +11,8 @@
 |entry_text                       = Life means as little as a passing moment of entertainment. Let them rot.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Life means as little as a passing
-moment of entertainment.
+|flavour_text                     = Life means as little as a passing
+moment of entertainment.
 Let them rot.
 |main_page                        = Colosseum Map
 }}