--- Area:MapWorldsPromenade
+++ Export
@@ -11,7 +11,7 @@
 |entry_text                       = Even the greatest achievements will eventually be abandoned.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Even the greatest achievements
+|flavour_text                     = Even the greatest achievements
 will eventually be abandoned.
 |main_page                        = Promenade Map
 }}