--- Area:MapWorldsArachnidNest
+++ Export
@@ -11,7 +11,7 @@
 |entry_text                       = Man is not the only beast to bend nature to its will.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Man is not the only beast to
+|flavour_text                     = Man is not the only beast to
 bend nature to its will.
 |main_page                        = Arachnid Nest Map
 |screenshot_ext                   = png
