--- Area:MapWorldsChateauUnique
+++ Export
@@ -4,7 +4,7 @@
 |act                              = 11
 |area_level                       = 71
 |area_type_tags                   = urban, dungeon, temple
-|tags                             = indoors_area, map, no_strongboxes, unique_map, ambush_leaguestone, domination_leaguestone, perandus_leaguestone, prophecy_leaguestone, essence_leaguestone, breach_leaguestone, wealthy_area
+|tags                             = indoors_area, map, no_strongboxes, unique_map, ambush_leaguestone, domination_leaguestone, perandus_leaguestone, prophecy_leaguestone, essence_leaguestone, breach_leaguestone, wealthy_area, city_home_map
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |is_map_area                      = True