--- Area:1~2~2
+++ Export
@@ -9,7 +9,6 @@
 |connection_ids                   = 1_2_town, 1_2_2a, 1_2_3
 |parent_area_id                   = 1_2_town
 |boss_monster_ids                 = Metadata/Monsters/MossMonster/MossMonsterBossOldFields
-|vaal_area_spawn_chance           = 15
 |vaal_area_ids                    = 1_SideArea2_4
 |flavour_text                     = Sick pastures, feverish to the touch.
 |main_page                        = The Old Fields
