--- Area:MapWorldsFoundry
+++ Export
@@ -4,7 +4,7 @@
 |act                              = 11
 |area_level                       = 82
 |area_type_tags                   = urban, dungeon
-|tags                             = indoors_area, map, delve_chest, einharmaps, thaumaturgy_area
+|tags                             = indoors_area, map, delve_chest, einharmaps, thaumaturgy_area, inventing_area, Forge_Area
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/LeagueMetamorphosis/MetamorphosisMapBoss, Metadata/Monsters/Thaumaturgy/ThaumaturgistBoss
