--- Area:MapWorldsUndergroundSea
+++ Export
@@ -11,7 +11,7 @@
 |entry_text                       = Silent stillness. The waters are as cold as they are black.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Silent stillness. The waters are
+|flavour_text                     = Silent stillness. The waters are
 as cold as they are black.
 |main_page                        = Underground Sea Map
 }}