--- Area:2~10~6
+++ Export
@@ -9,7 +9,6 @@
 |connection_ids                   = 2_10_5, 2_10_town
 |parent_area_id                   = 2_10_town
 |boss_monster_ids                 = Metadata/Monsters/KitavaBoss/KitavaFinal
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea5_3_1, 1_SideArea5_3_2
 |spawn_weight1_tag                = high_level_map
 |spawn_weight1_value              = 0
