--- Area:MapWorldsArachnidTomb
+++ Export
@@ -11,6 +11,9 @@
 |entry_text                       = They feel your every move, your every breath, with a million little wires.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
+|flavour_text                     = They feel your every move,
+your every breath,
+with a million little wires.
 |main_page                        = Arachnid Tomb Map
 |screenshot_ext                   = png
 }}