--- Area:1~3~13
+++ Export
@@ -9,9 +9,9 @@
 |connection_ids                   = 1_3_10_1, 1_3_14_1, 1_3_15
 |parent_area_id                   = 1_3_town
 |boss_monster_ids                 = Metadata/Monsters/Axis/AxisGeneral
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea3_8
 |has_waypoint                     = True
 |flavour_text                     = Polished surfaces reflect brutal efficiency.
 |main_page                        = The Ebony Barracks
+|vaal_area_spawn_chance           = 18
 }}