--- Area:1~1~3
+++ Export
@@ -9,6 +9,7 @@
 |connection_ids                   = 1_1_2, 1_1_4_1, 1_1_3a
 |parent_area_id                   = 1_1_town
 |boss_monster_ids                 = Metadata/Monsters/Rhoas/RhoaBoss
+|vaal_area_ids                    = 1_SideArea1_1, 1_SideArea1_3
 |flavour_text                     = Mud and air seethes with warped life.
 |main_page                        = The Mud Flats (Act 1)
 }}