--- Area:MapWorldsPort
+++ Export
@@ -4,16 +4,16 @@
 |act                              = 11
 |area_level                       = 70
 |area_type_tags                   = urban
-|tags                             = map, einharmaps, marketplace_area, urban_sarn_area, dock_area, ocean_area, ruined_area
+|tags                             = map, einharmaps, marketplace_area, urban_sarn_area, dock_area, ocean_area, ruined_area, cannot_possess_boss
 |loading_screen                   = Maps
 |parent_area_id                   = 2_11_endgame_town
 |boss_monster_ids                 = Metadata/Monsters/MassSkeleton/MassSkeletonMapBoss, Metadata/Monsters/MassSkeleton/MassSkeletonMapBoss2, Metadata/Monsters/MassSkeleton/MassSkeletonMapBoss3, Metadata/Monsters/MassSkeleton/MassSkeletonMapBoss4, Metadata/Monsters/MassSkeleton/MassSkeletonMapBoss5
 |entry_text                       = The wealthy foolishly battle the ocean for domain.
 |entry_npc                        = The Shaper
-|vaal_area_spawn_chance           = 8
 |vaal_area_ids                    = MapSideArea8_3, MapSideArea8_5
 |is_map_area                      = True
 |flavour_text                     = The wealthy foolishly battle
 the ocean for domain.
 |main_page                        = Port Map
+|vaal_area_spawn_chance           = 8
 }}