--- Area:2~7~5~2
+++ Export
@@ -9,7 +9,6 @@
 |connection_ids                   = 2_7_5_1, 2_7_6
 |parent_area_id                   = 2_7_town
 |boss_monster_ids                 = Metadata/Monsters/ZombieDismember/ZombieBossDegenAura
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea2_6
 |flavour_text                     = Decrepit ruins born again. A lusting evil stalks its halls.
 |main_page                        = The Chamber of Sins Level 2 (Act 7)
