--- Area:MapWorldsDunes
+++ Export
@@ -11,7 +11,7 @@
 |entry_text                       = Shifting, morphing, expanding. A reminder of the virtues of an agile mind.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Shifting, morphing, expanding. A reminder
+|flavour_text                     = Shifting, morphing, expanding. A reminder
 of the virtues of an agile mind.
 |main_page                        = Dunes Map
 }}