--- Area:2~10~9
+++ Export
@@ -8,7 +8,6 @@
 |loading_screen                   = Act5
 |connection_ids                   = 2_10_2
 |parent_area_id                   = 2_10_town
-|vaal_area_spawn_chance           = 18
 |vaal_area_ids                    = 1_SideArea5_6
 |spawn_weight1_tag                = high_level_map
 |spawn_weight1_value              = 0
@@ -16,4 +15,5 @@
 |spawn_weight2_value              = 1000
 |flavour_text                     = The dead offer thanks, for they have been spared this final savagery.
 |main_page                        = The Ossuary (Act 10)
+|vaal_area_spawn_chance           = 18
 }}