--- Area:MapWorldsVaalPyramid
+++ Export
@@ -11,8 +11,8 @@
 |entry_text                       = A good slave is driven by only two thoughts: service and sacrifice.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = A good slave is driven by
-only two thoughts:
+|flavour_text                     = A good slave is driven by
+only two thoughts:
 service and sacrifice.
 |main_page                        = Vaal Pyramid Map
 }}