--- Area:MapWorldsOvergrownRuin
+++ Export
@@ -11,7 +11,7 @@
 |entry_text                       = Only fear remains unforgotten, where once magnificent rites drew important crowds.
 |entry_npc                        = The Shaper
 |is_map_area                      = True
-|flavour_text                     = Only fear remains unforgotten, where once
+|flavour_text                     = Only fear remains unforgotten, where once
 magnificent rites drew important crowds.
 |main_page                        = Overgrown Ruin Map
 }}