--- Area:MapWorldsAtollUnique
+++ Export
@@ -10,13 +10,13 @@
 |boss_monster_ids                 = Metadata/Monsters/seawitchillusion/MapBossMerveilMaelstrom, Metadata/Monsters/Seawitch/MapBossMerveilMaelstrom
 |is_map_area                      = True
 |is_unique_map_area               = True
-|flavour_text                     = Whispers from
-a world apart
-Speak my name
-beyond the tomb;
-Bound within
-the Maelström's heart,
-Will they grant me
+|flavour_text                     = Whispers from
+a world apart
+Speak my name
+beyond the tomb;
+Bound within
+the Maelström's heart,
+Will they grant me
 strength or doom?
 |main_page                        = Maelström of Chaos
 }}