--- Locus of Corruption
+++ Export
@@ -5,7 +5,7 @@
 |flavour_text            = Darkness coalesces, imbues, and blesses.
 |tier                    = 3
 |icon                    = CorruptionRoom3
-|modifier_ids            = MapPlayerMaxResists3__
+|modifier_ids            = MapPlayerMaxResists3__, IncursionTempleAmuletVialChance3__
 |architect_metadata_id   = Metadata/Monsters/VaalArchitect/VaalArchitect8
 |architect_name          = Paquate, Architect of Corruption
 }}
