--- Sparring Room
+++ Export
@@ -6,7 +6,7 @@
 |tier                    = 1
 |upgrade_room_id         = WeaponRoomII
 |icon                    = WeaponRoom
-|modifier_ids            = 
+|modifier_ids            = IncursionMonsterCrits1, IncursionTempleRoomBonus1_
 |architect_metadata_id   = Metadata/Monsters/VaalArchitect/VaalArchitect11
 |architect_name          = Ticaba, Architect of the Arena
 }}