--- Hall of Champions
+++ Export
@@ -1,11 +1,11 @@
 {{Incursion room
 |id                      = WeaponRoomIII
 |name                    = Hall of Champions
-|description             = Increases monster damage throughout the Temple.<br />Contains chests full of weapons.
+|description             = Increases monster criticals throughout the Temple.<br />Contains chests full of weapons.
 |flavour_text            = Blood spills for cheap thrills.
 |tier                    = 3
 |icon                    = WeaponRoom3
-|modifier_ids            = IncursionMonsterDamageMap3
+|modifier_ids            = IncursionMonsterCrits3, IncursionTempleRoomBonus3
 |architect_metadata_id   = Metadata/Monsters/VaalArchitect/VaalArchitect11
 |architect_name          = Ticaba, Architect of the Arena
 }}
