--- Hall of Lords
+++ Export
@@ -8,7 +8,7 @@
 |tier                    = 2
 |upgrade_room_id         = QueensChambersIII
 |icon                    = QueensChambers2
-|modifier_ids            = IncursionMonsterMagicPacks2
+|modifier_ids            = IncursionMonsterMagicPacks2, IncursionTempleFlaskVialChance2
 |architect_metadata_id   = Metadata/Monsters/VaalArchitect/VaalArchitect6
 |architect_name          = Xipocado, Royal Architect
 }}