--- Torture Cages
+++ Export
@@ -3,11 +3,12 @@
 {{Incursion room
 |id                      = TormentII
 |name                    = Torture Cages
-|description             = Contains Tormented Spirits.
+|description             = Contains Tormented Spirits.<br />Contains additional roaming Tormented Spirits.<br />Architects and the Omnitect are possessed by a Tormented Spirit.
 |flavour_text            = Iron bars tell of the criminal swell.
 |tier                    = 2
 |upgrade_room_id         = TormentIII
 |icon                    = TormentRoom2
+|modifier_ids            = IncursionMonsterTormentedSpirit1, IncursionTempleAdditionalTormentedSpirit2
 |architect_metadata_id   = Metadata/Monsters/VaalArchitect/VaalArchitect23
 |architect_name          = Tzamoto, Architect of Torments
 }}
