--- Demolition Lab
+++ Export
@@ -1,11 +1,12 @@
 {{Incursion room
 |id                      = ExplosivesRoomII
 |name                    = Demolition Lab
-|description             = Contains Explosive Charges.<br/>Contains two Sealed Coffers.
+|description             = Contains Explosive Charges.<br />Contains two Sealed Coffers.
 |flavour_text            = Stone walls shaking with the might of unmaking.
 |tier                    = 2
 |upgrade_room_id         = ExplosivesRoomIII
 |icon                    = ExplosivesRoom2
+|modifier_ids            = 
 |architect_metadata_id   = Metadata/Monsters/VaalArchitect/VaalArchitect9
 |architect_name          = Hayoxi, Architect of Destruction
 }}
