--- Crucible of Flame
+++ Export
@@ -1,10 +1,11 @@
 {{Incursion room
 |id                      = BossFireIII
 |name                    = Crucible of Flame
-|description             = Augments the Omnitect with fire.<br />Contains a valuable item.
+|description             = Monsters throughout the Temple have additional fire damage.<br />Adds additional Fire monster packs throughout the Temple.<br />Augments the Omnitect with fire.<br />Contains a valuable item.
 |flavour_text            = Earth torn asunder births a mechanical wonder.
 |tier                    = 3
 |icon                    = BossFireRoom3
+|modifier_ids            = IncursionMonsterAddedAsFire3, IncursionTempleRoomBonus3, IncursionTempleFirePacks3, IncursionTempleFireVialChance3
 |architect_metadata_id   = Metadata/Monsters/VaalArchitect/VaalArchitect21
 |architect_name          = Puhuarte, Architect of the Forge
 }}