--- Cultivar Chamber
+++ Export
@@ -6,7 +6,7 @@
 |tier                    = 2
 |upgrade_room_id         = PoisonRoomIII
 |icon                    = PoisonRoom2
-|modifier_ids            = IncursionTemplePoisonClusters2
+|modifier_ids            = IncursionTemplePoisonClusters2, IncursionTemplePoisonVialChance2
 |architect_metadata_id   = Metadata/Monsters/VaalArchitect/VaalArchitect1
 |architect_name          = Tacati, Architect of Toxins
 }}
