--- Arena of Valour
+++ Export
@@ -6,7 +6,7 @@
 |tier                    = 2
 |upgrade_room_id         = WeaponRoomIII
 |icon                    = WeaponRoom2
-|modifier_ids            = IncursionMonsterCrits2
+|modifier_ids            = IncursionMonsterCrits2, IncursionTempleRoomBonus2_
 |architect_metadata_id   = Metadata/Monsters/VaalArchitect/VaalArchitect11
 |architect_name          = Ticaba, Architect of the Arena
 }}
