--- Chamber of Iron
+++ Export
@@ -5,7 +5,7 @@
 |flavour_text            = A bacchanalia of the finest regalia.
 |tier                    = 3
 |icon                    = ArmourRoom3
-|modifier_ids            = IncursionMonsterResists3
+|modifier_ids            = IncursionMonsterResists3, IncursionTempleRoomBonus3
 |architect_metadata_id   = Metadata/Monsters/VaalArchitect/VaalArchitect10
 |architect_name          = Atmohua, Architect of Iron
 }}
