--- Catalyst of Corruption
+++ Export
@@ -6,7 +6,7 @@
 |tier                    = 2
 |upgrade_room_id         = CorruptionRoomIII
 |icon                    = CorruptionRoom2
-|modifier_ids            = MapPlayerMaxResists2
+|modifier_ids            = MapPlayerMaxResists2, IncursionTempleAmuletVialChance2
 |architect_metadata_id   = Metadata/Monsters/VaalArchitect/VaalArchitect8
 |architect_name          = Paquate, Architect of Corruption
 }}