--- Workshop
+++ Export
@@ -6,7 +6,7 @@
 |tier                    = 1
 |upgrade_room_id         = WorkshopII
 |icon                    = Workshop
-|modifier_ids            = MapMassiveBossIncursionTemple1
+|modifier_ids            = IncursionMonsterLife1, MapMassiveBossIncursionTemple1, IncursionTempleRoomBonus1_
 |architect_metadata_id   = Metadata/Monsters/VaalArchitect/VaalArchitect17
 |architect_name          = Jiquani, Architect of Industry
 }}
