--- Sacrificial Chamber
+++ Export
@@ -1,11 +1,12 @@
 {{Incursion room
 |id                      = SacrificeRoomI
 |name                    = Sacrificial Chamber
-|description             = Allows you to sacrifice a unique item.<br/>Ahuana drops an additional Unique Item.
+|description             = Allows you to sacrifice a unique item.<br />Ahuana drops an additional Unique Item.
 |flavour_text            = Sacred words read to bless the dead.
 |tier                    = 1
 |upgrade_room_id         = SacrificeRoomII
 |icon                    = SacrificeRoom
+|modifier_ids            = 
 |architect_metadata_id   = Metadata/Monsters/VaalArchitect/VaalArchitect2
 |architect_name          = Ahuana, Architect of Ceremonies
 }}