--- House of the Others
+++ Export
@@ -1,10 +1,11 @@
 {{Incursion room
 |id                      = BreachRoomIII
 |name                    = House of the Others
-|description             = Contains three Breaches and Breach Splinters.<br/>Breaches open and close faster.
+|description             = Contains three Breaches and Breach Splinters.<br />Breaches open and close faster.
 |flavour_text            = The Dreamer's reach extends through the breach.
 |tier                    = 3
 |icon                    = BreachRoom3
+|modifier_ids            = IncursionTempleBreachSpeed2
 |architect_metadata_id   = Metadata/Monsters/VaalArchitect/VaalArchitect13
 |architect_name          = Zilquapa, Architect of the Breach
 }}