--- Apex of Ascension
+++ Export
@@ -1,10 +1,11 @@
 {{Incursion room
 |id                      = SacrificeRoomIII
 |name                    = Apex of Ascension
-|description             = Allows you to sacrifice a unique item.<br/>Ahuana drops an additional Unique Item.
+|description             = Allows you to sacrifice a unique item.<br />Ahuana drops an additional Unique Item.
 |flavour_text            = Form sliced through, but born anew.
 |tier                    = 3
 |icon                    = SacrificeRoom3
+|modifier_ids            = 
 |architect_metadata_id   = Metadata/Monsters/VaalArchitect/VaalArchitect2
 |architect_name          = Ahuana, Architect of Ceremonies
 }}
