--- Poison Garden
+++ Export
@@ -6,7 +6,7 @@
 |tier                    = 1
 |upgrade_room_id         = PoisonRoomII
 |icon                    = PoisonRoom
-|modifier_ids            = IncursionTemplePoisonClusters1
+|modifier_ids            = IncursionTemplePoisonClusters1, IncursionTemplePoisonVialChance1
 |architect_metadata_id   = Metadata/Monsters/VaalArchitect/VaalArchitect1
 |architect_name          = Tacati, Architect of Toxins
 }}
