--- Conduit of Lightning
+++ Export
@@ -1,11 +1,11 @@
 {{Incursion room
 |id                      = BossLightningIII
 |name                    = Conduit of Lightning
-|description             = Monsters throughout the Temple have additional lightning damage.<br/>Adds additional Lightning monster packs throughout the Temple<br/>Augments the Omnitect with lightning.<br />Contains a valuable item.
+|description             = Monsters throughout the Temple have additional lightning damage.<br />Adds additional Lightning monster packs throughout the Temple.<br />Augments the Omnitect with lightning.<br />Contains a valuable item.
 |flavour_text            = Arcs emerge to power the scourge.
 |tier                    = 3
 |icon                    = BossLightningRoom3
-|modifier_ids            = IncursionMonsterAddedAsLightning3
+|modifier_ids            = IncursionMonsterAddedAsLightning3, IncursionTempleRoomBonus3, IncursionTempleLightningPacks3, IncursionTempleLightningVialChance3
 |architect_metadata_id   = Metadata/Monsters/VaalArchitect/VaalArchitect22
 |architect_name          = Xopec, Architect of Power
 }}
