--- Pools of Restoration
+++ Export
@@ -6,7 +6,7 @@
 |tier                    = 1
 |upgrade_room_id         = HealingRoomII
 |icon                    = HealingRoom
-|modifier_ids            = IncursionMonsterRegen1
+|modifier_ids            = IncursionMonsterRegen1, IncursionTempleRoomBonus1_, IncursionTempleHealingVialChance1
 |architect_metadata_id   = Metadata/Monsters/VaalArchitect/VaalArchitect19
 |architect_name          = Guatelitzi, Architect of Flesh
 }}