--- Hall of Offerings
+++ Export
@@ -1,11 +1,12 @@
 {{Incursion room
 |id                      = SacrificeRoomII
 |name                    = Hall of Offerings
-|description             = Allows you to sacrifice a unique equipment.<br/>Ahuana drops an additional Unique Item.
+|description             = Allows you to sacrifice a unique item.<br />Ahuana drops an additional Unique Item.
 |flavour_text            = Red gifts flow to the pits below.
 |tier                    = 2
 |upgrade_room_id         = SacrificeRoomIII
 |icon                    = SacrificeRoom2
+|modifier_ids            = 
 |architect_metadata_id   = Metadata/Monsters/VaalArchitect/VaalArchitect2
 |architect_name          = Ahuana, Architect of Ceremonies
 }}