--- Automaton Lab
+++ Export
@@ -1,12 +1,12 @@
 {{Incursion room
 |id                      = BossMinionsII
 |name                    = Automaton Lab
-|description             = Increase monster damage throughout the Temple.<br/>Adds additional Minion monster packs throughout the Temple.<br/>Augments the Omnitect with minions.
+|description             = Increases monster damage throughout the Temple.<br />Adds additional Minion monster packs throughout the Temple.<br />Augments the Omnitect with minions.
 |flavour_text            = Forged forms assemble in swarms.
 |tier                    = 2
 |upgrade_room_id         = BossMinionsIII
 |icon                    = BossMinionRoom2
-|modifier_ids            = IncursionMonsterDamageMap2
+|modifier_ids            = IncursionMonsterDamageMap2, IncursionTempleRoomBonus2_, IncursionTempleMinionPacks2, IncursionTempleMinionVialChance2
 |architect_metadata_id   = Metadata/Monsters/VaalArchitect/VaalArchitect20
 |architect_name          = Citaqualotl, Architect of the Swarm
 }}
