--- Sanctum of Vitality
+++ Export
@@ -6,7 +6,7 @@
 |tier                    = 2
 |upgrade_room_id         = HealingRoomIII
 |icon                    = HealingRoom2
-|modifier_ids            = IncursionMonsterRegen2
+|modifier_ids            = IncursionMonsterRegen2, IncursionTempleRoomBonus2_, IncursionTempleHealingVialChance2_
 |architect_metadata_id   = Metadata/Monsters/VaalArchitect/VaalArchitect19
 |architect_name          = Guatelitzi, Architect of Flesh
 }}
