--- Storm of Corruption
+++ Export
@@ -5,9 +5,9 @@
 |flavour_text            = Darkness flies across the skies.
 |tier                    = 3
 |icon                    = StormRoom3
+|modifier_ids            = IncursionTempestWeightTier3, IncursionTempleTempestBuffDuration3
 |architect_metadata_id   = Metadata/Monsters/VaalArchitect/VaalArchitect3
 |architect_name          = Topotante, Architect of Storms
-|modifier_ids            = IncursionTempestWeightTier3
 }}
 
 {{Incursion room progression}}