--- Throat Stabber
+++ Export
@@ -16,7 +16,7 @@
 |attack_speed                            = 1.5
 |physical_damage_min                     = 21
 |physical_damage_max                     = 73
-|weapon_range                            = 11
+|weapon_range                            = 1.1
 |quest_reward1_type                      = quest
 |quest_reward1_quest                     = The Storm Blade
 |quest_reward1_quest_id                  = a9q3