--- Herald of Purity
+++ Export
@@ -49,14 +49,14 @@
 |level1                                  = True
 |level1_level_requirement                = 16
 |level1_stat_text                        = Buff grants 9% more Physical Damage
-|level1_stat1_id                         = display_minion_monster_level
+|level1_stat1_id                         = base_display_minion_actor_level
 |level1_stat1_value                      = 16
 |level1_stat2_id                         = herald_of_purity_physical_damage_+%_final
 |level1_stat2_value                      = 9
 |level2                                  = True
 |level2_level_requirement                = 20
 |level2_stat_text                        = Minions have 2% more Maximum Life<br>Buff grants 9% more Physical Damage<br>Minions deal 5% more Physical Damage
-|level2_stat1_id                         = display_minion_monster_level
+|level2_stat1_id                         = base_display_minion_actor_level
 |level2_stat1_value                      = 20
 |level2_stat2_id                         = active_skill_minion_life_+%_final
 |level2_stat2_value                      = 2
@@ -67,7 +67,7 @@
 |level3                                  = True
 |level3_level_requirement                = 24
 |level3_stat_text                        = Minions have 4% more Maximum Life<br>Buff grants 9% more Physical Damage<br>Minions deal 10% more Physical Damage
-|level3_stat1_id                         = display_minion_monster_level
+|level3_stat1_id                         = base_display_minion_actor_level
 |level3_stat1_value                      = 24
 |level3_stat2_id                         = active_skill_minion_life_+%_final
 |level3_stat2_value                      = 4
@@ -78,7 +78,7 @@
 |level4                                  = True
 |level4_level_requirement                = 28
 |level4_stat_text                        = Minions have 6% more Maximum Life<br>Buff grants 9% more Physical Damage<br>Minions deal 15% more Physical Damage
-|level4_stat1_id                         = display_minion_monster_level
+|level4_stat1_id                         = base_display_minion_actor_level
 |level4_stat1_value                      = 28
 |level4_stat2_id                         = active_skill_minion_life_+%_final
 |level4_stat2_value                      = 6
@@ -89,7 +89,7 @@
 |level5                                  = True
 |level5_level_requirement                = 31
 |level5_stat_text                        = Minions have 8% more Maximum Life<br>Buff grants 9% more Physical Damage<br>Minions deal 20% more Physical Damage
-|level5_stat1_id                         = display_minion_monster_level
+|level5_stat1_id                         = base_display_minion_actor_level
 |level5_stat1_value                      = 31
 |level5_stat2_id                         = active_skill_minion_life_+%_final
 |level5_stat2_value                      = 8
@@ -100,7 +100,7 @@
 |level6                                  = True
 |level6_level_requirement                = 34
 |level6_stat_text                        = Minions have 10% more Maximum Life<br>Buff grants 10% more Physical Damage<br>Minions deal 25% more Physical Damage
-|level6_stat1_id                         = display_minion_monster_level
+|level6_stat1_id                         = base_display_minion_actor_level
 |level6_stat1_value                      = 34
 |level6_stat2_id                         = active_skill_minion_life_+%_final
 |level6_stat2_value                      = 10
@@ -111,7 +111,7 @@
 |level7                                  = True
 |level7_level_requirement                = 37
 |level7_stat_text                        = Minions have 12% more Maximum Life<br>Buff grants 10% more Physical Damage<br>Minions deal 30% more Physical Damage
-|level7_stat1_id                         = display_minion_monster_level
+|level7_stat1_id                         = base_display_minion_actor_level
 |level7_stat1_value                      = 37
 |level7_stat2_id                         = active_skill_minion_life_+%_final
 |level7_stat2_value                      = 12
@@ -122,7 +122,7 @@
 |level8                                  = True
 |level8_level_requirement                = 40
 |level8_stat_text                        = Minions have 14% more Maximum Life<br>Buff grants 10% more Physical Damage<br>Minions deal 35% more Physical Damage
-|level8_stat1_id                         = display_minion_monster_level
+|level8_stat1_id                         = base_display_minion_actor_level
 |level8_stat1_value                      = 40
 |level8_stat2_id                         = active_skill_minion_life_+%_final
 |level8_stat2_value                      = 14
@@ -133,7 +133,7 @@
 |level9                                  = True
 |level9_level_requirement                = 43
 |level9_stat_text                        = Minions have 16% more Maximum Life<br>Buff grants 10% more Physical Damage<br>Minions deal 40% more Physical Damage
-|level9_stat1_id                         = display_minion_monster_level
+|level9_stat1_id                         = base_display_minion_actor_level
 |level9_stat1_value                      = 43
 |level9_stat2_id                         = active_skill_minion_life_+%_final
 |level9_stat2_value                      = 16
@@ -144,7 +144,7 @@
 |level10                                 = True
 |level10_level_requirement               = 46
 |level10_stat_text                       = Minions have 18% more Maximum Life<br>Buff grants 10% more Physical Damage<br>Minions deal 45% more Physical Damage
-|level10_stat1_id                        = display_minion_monster_level
+|level10_stat1_id                        = base_display_minion_actor_level
 |level10_stat1_value                     = 46
 |level10_stat2_id                        = active_skill_minion_life_+%_final
 |level10_stat2_value                     = 18
@@ -155,7 +155,7 @@
 |level11                                 = True
 |level11_level_requirement               = 49
 |level11_stat_text                       = Minions have 20% more Maximum Life<br>Buff grants 11% more Physical Damage<br>Minions deal 50% more Physical Damage
-|level11_stat1_id                        = display_minion_monster_level
+|level11_stat1_id                        = base_display_minion_actor_level
 |level11_stat1_value                     = 49
 |level11_stat2_id                        = active_skill_minion_life_+%_final
 |level11_stat2_value                     = 20
@@ -166,7 +166,7 @@
 |level12                                 = True
 |level12_level_requirement               = 52
 |level12_stat_text                       = Minions have 22% more Maximum Life<br>Buff grants 11% more Physical Damage<br>Minions deal 55% more Physical Damage
-|level12_stat1_id                        = display_minion_monster_level
+|level12_stat1_id                        = base_display_minion_actor_level
 |level12_stat1_value                     = 52
 |level12_stat2_id                        = active_skill_minion_life_+%_final
 |level12_stat2_value                     = 22
@@ -177,7 +177,7 @@
 |level13                                 = True
 |level13_level_requirement               = 55
 |level13_stat_text                       = Minions have 24% more Maximum Life<br>Buff grants 11% more Physical Damage<br>Minions deal 60% more Physical Damage
-|level13_stat1_id                        = display_minion_monster_level
+|level13_stat1_id                        = base_display_minion_actor_level
 |level13_stat1_value                     = 55
 |level13_stat2_id                        = active_skill_minion_life_+%_final
 |level13_stat2_value                     = 24
@@ -188,7 +188,7 @@
 |level14                                 = True
 |level14_level_requirement               = 58
 |level14_stat_text                       = Minions have 26% more Maximum Life<br>Buff grants 11% more Physical Damage<br>Minions deal 65% more Physical Damage
-|level14_stat1_id                        = display_minion_monster_level
+|level14_stat1_id                        = base_display_minion_actor_level
 |level14_stat1_value                     = 58
 |level14_stat2_id                        = active_skill_minion_life_+%_final
 |level14_stat2_value                     = 26
@@ -199,7 +199,7 @@
 |level15                                 = True
 |level15_level_requirement               = 60
 |level15_stat_text                       = Minions have 28% more Maximum Life<br>Buff grants 11% more Physical Damage<br>Minions deal 70% more Physical Damage
-|level15_stat1_id                        = display_minion_monster_level
+|level15_stat1_id                        = base_display_minion_actor_level
 |level15_stat1_value                     = 60
 |level15_stat2_id                        = active_skill_minion_life_+%_final
 |level15_stat2_value                     = 28
@@ -210,7 +210,7 @@
 |level16                                 = True
 |level16_level_requirement               = 62
 |level16_stat_text                       = Minions have 30% more Maximum Life<br>Buff grants 12% more Physical Damage<br>Minions deal 75% more Physical Damage
-|level16_stat1_id                        = display_minion_monster_level
+|level16_stat1_id                        = base_display_minion_actor_level
 |level16_stat1_value                     = 62
 |level16_stat2_id                        = active_skill_minion_life_+%_final
 |level16_stat2_value                     = 30
@@ -221,7 +221,7 @@
 |level17                                 = True
 |level17_level_requirement               = 64
 |level17_stat_text                       = Minions have 32% more Maximum Life<br>Buff grants 12% more Physical Damage<br>Minions deal 80% more Physical Damage
-|level17_stat1_id                        = display_minion_monster_level
+|level17_stat1_id                        = base_display_minion_actor_level
 |level17_stat1_value                     = 64
 |level17_stat2_id                        = active_skill_minion_life_+%_final
 |level17_stat2_value                     = 32
@@ -232,7 +232,7 @@
 |level18                                 = True
 |level18_level_requirement               = 66
 |level18_stat_text                       = Minions have 34% more Maximum Life<br>Buff grants 12% more Physical Damage<br>Minions deal 85% more Physical Damage
-|level18_stat1_id                        = display_minion_monster_level
+|level18_stat1_id                        = base_display_minion_actor_level
 |level18_stat1_value                     = 66
 |level18_stat2_id                        = active_skill_minion_life_+%_final
 |level18_stat2_value                     = 34
@@ -243,7 +243,7 @@
 |level19                                 = True
 |level19_level_requirement               = 68
 |level19_stat_text                       = Minions have 36% more Maximum Life<br>Buff grants 12% more Physical Damage<br>Minions deal 90% more Physical Damage
-|level19_stat1_id                        = display_minion_monster_level
+|level19_stat1_id                        = base_display_minion_actor_level
 |level19_stat1_value                     = 68
 |level19_stat2_id                        = active_skill_minion_life_+%_final
 |level19_stat2_value                     = 36
@@ -254,7 +254,7 @@
 |level20                                 = True
 |level20_level_requirement               = 70
 |level20_stat_text                       = Minions have 38% more Maximum Life<br>Buff grants 12% more Physical Damage<br>Minions deal 95% more Physical Damage
-|level20_stat1_id                        = display_minion_monster_level
+|level20_stat1_id                        = base_display_minion_actor_level
 |level20_stat1_value                     = 70
 |level20_stat2_id                        = active_skill_minion_life_+%_final
 |level20_stat2_value                     = 38
@@ -265,7 +265,7 @@
 |level21                                 = True
 |level21_level_requirement               = 72
 |level21_stat_text                       = Minions have 40% more Maximum Life<br>Buff grants 13% more Physical Damage<br>Minions deal 100% more Physical Damage
-|level21_stat1_id                        = display_minion_monster_level
+|level21_stat1_id                        = base_display_minion_actor_level
 |level21_stat1_value                     = 72
 |level21_stat2_id                        = active_skill_minion_life_+%_final
 |level21_stat2_value                     = 40
@@ -276,7 +276,7 @@
 |level22                                 = True
 |level22_level_requirement               = 74
 |level22_stat_text                       = Minions have 42% more Maximum Life<br>Buff grants 13% more Physical Damage<br>Minions deal 105% more Physical Damage
-|level22_stat1_id                        = display_minion_monster_level
+|level22_stat1_id                        = base_display_minion_actor_level
 |level22_stat1_value                     = 74
 |level22_stat2_id                        = active_skill_minion_life_+%_final
 |level22_stat2_value                     = 42
@@ -287,7 +287,7 @@
 |level23                                 = True
 |level23_level_requirement               = 76
 |level23_stat_text                       = Minions have 44% more Maximum Life<br>Buff grants 13% more Physical Damage<br>Minions deal 110% more Physical Damage
-|level23_stat1_id                        = display_minion_monster_level
+|level23_stat1_id                        = base_display_minion_actor_level
 |level23_stat1_value                     = 76
 |level23_stat2_id                        = active_skill_minion_life_+%_final
 |level23_stat2_value                     = 44
@@ -298,7 +298,7 @@
 |level24                                 = True
 |level24_level_requirement               = 78
 |level24_stat_text                       = Minions have 46% more Maximum Life<br>Buff grants 13% more Physical Damage<br>Minions deal 115% more Physical Damage
-|level24_stat1_id                        = display_minion_monster_level
+|level24_stat1_id                        = base_display_minion_actor_level
 |level24_stat1_value                     = 78
 |level24_stat2_id                        = active_skill_minion_life_+%_final
 |level24_stat2_value                     = 46
@@ -309,7 +309,7 @@
 |level25                                 = True
 |level25_level_requirement               = 80
 |level25_stat_text                       = Minions have 48% more Maximum Life<br>Buff grants 13% more Physical Damage<br>Minions deal 120% more Physical Damage
-|level25_stat1_id                        = display_minion_monster_level
+|level25_stat1_id                        = base_display_minion_actor_level
 |level25_stat1_value                     = 80
 |level25_stat2_id                        = active_skill_minion_life_+%_final
 |level25_stat2_value                     = 48
@@ -320,7 +320,7 @@
 |level26                                 = True
 |level26_level_requirement               = 82
 |level26_stat_text                       = Minions have 50% more Maximum Life<br>Buff grants 14% more Physical Damage<br>Minions deal 125% more Physical Damage
-|level26_stat1_id                        = display_minion_monster_level
+|level26_stat1_id                        = base_display_minion_actor_level
 |level26_stat1_value                     = 82
 |level26_stat2_id                        = active_skill_minion_life_+%_final
 |level26_stat2_value                     = 50
@@ -331,7 +331,7 @@
 |level27                                 = True
 |level27_level_requirement               = 84
 |level27_stat_text                       = Minions have 52% more Maximum Life<br>Buff grants 14% more Physical Damage<br>Minions deal 130% more Physical Damage
-|level27_stat1_id                        = display_minion_monster_level
+|level27_stat1_id                        = base_display_minion_actor_level
 |level27_stat1_value                     = 84
 |level27_stat2_id                        = active_skill_minion_life_+%_final
 |level27_stat2_value                     = 52
@@ -342,7 +342,7 @@
 |level28                                 = True
 |level28_level_requirement               = 86
 |level28_stat_text                       = Minions have 54% more Maximum Life<br>Buff grants 14% more Physical Damage<br>Minions deal 135% more Physical Damage
-|level28_stat1_id                        = display_minion_monster_level
+|level28_stat1_id                        = base_display_minion_actor_level
 |level28_stat1_value                     = 86
 |level28_stat2_id                        = active_skill_minion_life_+%_final
 |level28_stat2_value                     = 54
@@ -353,7 +353,7 @@
 |level29                                 = True
 |level29_level_requirement               = 88
 |level29_stat_text                       = Minions have 56% more Maximum Life<br>Buff grants 14% more Physical Damage<br>Minions deal 140% more Physical Damage
-|level29_stat1_id                        = display_minion_monster_level
+|level29_stat1_id                        = base_display_minion_actor_level
 |level29_stat1_value                     = 88
 |level29_stat2_id                        = active_skill_minion_life_+%_final
 |level29_stat2_value                     = 56
@@ -364,7 +364,7 @@
 |level30                                 = True
 |level30_level_requirement               = 90
 |level30_stat_text                       = Minions have 58% more Maximum Life<br>Buff grants 14% more Physical Damage<br>Minions deal 145% more Physical Damage
-|level30_stat1_id                        = display_minion_monster_level
+|level30_stat1_id                        = base_display_minion_actor_level
 |level30_stat1_value                     = 90
 |level30_stat2_id                        = active_skill_minion_life_+%_final
 |level30_stat2_value                     = 58
@@ -375,7 +375,7 @@
 |level31                                 = True
 |level31_level_requirement               = 91
 |level31_stat_text                       = Minions have 60% more Maximum Life<br>Buff grants 14% more Physical Damage<br>Minions deal 147% more Physical Damage
-|level31_stat1_id                        = display_minion_monster_level
+|level31_stat1_id                        = base_display_minion_actor_level
 |level31_stat1_value                     = 91
 |level31_stat2_id                        = active_skill_minion_life_+%_final
 |level31_stat2_value                     = 60
@@ -386,7 +386,7 @@
 |level32                                 = True
 |level32_level_requirement               = 92
 |level32_stat_text                       = Minions have 62% more Maximum Life<br>Buff grants 15% more Physical Damage<br>Minions deal 150% more Physical Damage
-|level32_stat1_id                        = display_minion_monster_level
+|level32_stat1_id                        = base_display_minion_actor_level
 |level32_stat1_value                     = 92
 |level32_stat2_id                        = active_skill_minion_life_+%_final
 |level32_stat2_value                     = 62
@@ -397,7 +397,7 @@
 |level33                                 = True
 |level33_level_requirement               = 93
 |level33_stat_text                       = Minions have 64% more Maximum Life<br>Buff grants 15% more Physical Damage<br>Minions deal 152% more Physical Damage
-|level33_stat1_id                        = display_minion_monster_level
+|level33_stat1_id                        = base_display_minion_actor_level
 |level33_stat1_value                     = 93
 |level33_stat2_id                        = active_skill_minion_life_+%_final
 |level33_stat2_value                     = 64
@@ -408,7 +408,7 @@
 |level34                                 = True
 |level34_level_requirement               = 94
 |level34_stat_text                       = Minions have 66% more Maximum Life<br>Buff grants 15% more Physical Damage<br>Minions deal 155% more Physical Damage
-|level34_stat1_id                        = display_minion_monster_level
+|level34_stat1_id                        = base_display_minion_actor_level
 |level34_stat1_value                     = 94
 |level34_stat2_id                        = active_skill_minion_life_+%_final
 |level34_stat2_value                     = 66
@@ -419,7 +419,7 @@
 |level35                                 = True
 |level35_level_requirement               = 95
 |level35_stat_text                       = Minions have 68% more Maximum Life<br>Buff grants 15% more Physical Damage<br>Minions deal 157% more Physical Damage
-|level35_stat1_id                        = display_minion_monster_level
+|level35_stat1_id                        = base_display_minion_actor_level
 |level35_stat1_value                     = 95
 |level35_stat2_id                        = active_skill_minion_life_+%_final
 |level35_stat2_value                     = 68
@@ -430,7 +430,7 @@
 |level36                                 = True
 |level36_level_requirement               = 96
 |level36_stat_text                       = Minions have 70% more Maximum Life<br>Buff grants 15% more Physical Damage<br>Minions deal 160% more Physical Damage
-|level36_stat1_id                        = display_minion_monster_level
+|level36_stat1_id                        = base_display_minion_actor_level
 |level36_stat1_value                     = 96
 |level36_stat2_id                        = active_skill_minion_life_+%_final
 |level36_stat2_value                     = 70
@@ -441,7 +441,7 @@
 |level37                                 = True
 |level37_level_requirement               = 97
 |level37_stat_text                       = Minions have 72% more Maximum Life<br>Buff grants 15% more Physical Damage<br>Minions deal 162% more Physical Damage
-|level37_stat1_id                        = display_minion_monster_level
+|level37_stat1_id                        = base_display_minion_actor_level
 |level37_stat1_value                     = 97
 |level37_stat2_id                        = active_skill_minion_life_+%_final
 |level37_stat2_value                     = 72
@@ -452,7 +452,7 @@
 |level38                                 = True
 |level38_level_requirement               = 98
 |level38_stat_text                       = Minions have 74% more Maximum Life<br>Buff grants 15% more Physical Damage<br>Minions deal 165% more Physical Damage
-|level38_stat1_id                        = display_minion_monster_level
+|level38_stat1_id                        = base_display_minion_actor_level
 |level38_stat1_value                     = 98
 |level38_stat2_id                        = active_skill_minion_life_+%_final
 |level38_stat2_value                     = 74
@@ -463,7 +463,7 @@
 |level39                                 = True
 |level39_level_requirement               = 99
 |level39_stat_text                       = Minions have 76% more Maximum Life<br>Buff grants 15% more Physical Damage<br>Minions deal 167% more Physical Damage
-|level39_stat1_id                        = display_minion_monster_level
+|level39_stat1_id                        = base_display_minion_actor_level
 |level39_stat1_value                     = 99
 |level39_stat2_id                        = active_skill_minion_life_+%_final
 |level39_stat2_value                     = 76
@@ -474,7 +474,7 @@
 |level40                                 = True
 |level40_level_requirement               = 100
 |level40_stat_text                       = Minions have 78% more Maximum Life<br>Buff grants 15% more Physical Damage<br>Minions deal 170% more Physical Damage
-|level40_stat1_id                        = display_minion_monster_level
+|level40_stat1_id                        = base_display_minion_actor_level
 |level40_stat1_value                     = 100
 |level40_stat2_id                        = active_skill_minion_life_+%_final
 |level40_stat2_value                     = 78
