--- Writhing Invitation (quest item)
+++ Export
@@ -11,8 +11,8 @@
 |tags                                    = default
 |metadata_id                             = Metadata/Items/MapFragments/Primordial/QuestTangleKey
 |inventory_icon                          = Writhing Invitation (quest item)
-|release_version = 3.17.0
-|drop_text = Drops from the T11+ map boss with relevant quest state of [[The Eater of Worlds (quest)|The Eater of Worlds quest]].
+|drop_text                               = Drops from the T11+ map boss with relevant quest state of [[The Eater of Worlds (quest)|The Eater of Worlds quest]].
+|release_version                         = 3.17.0
 }}
 
 '''Writhing Invitation''' is a [[quest item]] and part of {{ql|The Eater of Worlds (quest)|The Eater of Worlds}} quest.