--- Crimson Jewel
+++ Export
@@ -11,21 +11,17 @@
 |help_text                               = Place into an allocated Jewel Socket on the Passive Skill Tree. Right click to remove from the Socket.
 |alternate_art_inventory_icons           = bloodgrip
 |release_version                         = 2.0.0
-|recipe1_description                     = {{Ml|PercentIncreasedLifeJewel}} {{c|magic|Crimson Jewel}}
-|recipe1_part1_item_name                 = The Endurance
-|recipe1_part1_amount                    = 4
 |quest_reward1_type                      = quest
 |quest_reward1_quest                     = Death to Purity
 |quest_reward1_quest_id                  = a5q4
 |quest_reward1_act                       = 5
-|quest_reward1_item_level                = 42
-|quest_reward1_rarity_id                 = rare
 |quest_reward2_type                      = quest
 |quest_reward2_quest                     = The Wings of Vastiri
 |quest_reward2_quest_id                  = a8q4
 |quest_reward2_act                       = 8
-|quest_reward2_item_level                = 57
-|quest_reward2_rarity_id                 = rare
+|recipe1_description                     = {{Ml|PercentIncreasedLifeJewel}} {{c|magic|Crimson Jewel}}
+|recipe1_part1_item_name                 = The Endurance
+|recipe1_part1_amount                    = 4
 }}
 
 '''Crimson Jewel''' is a [[jewel]] base type.
