--- Ancestral Tattoo of Bloodlines
+++ Export
@@ -10,11 +10,15 @@
 |stack_size                              = 10
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
-|tattoo_target                           = Small Attribute Passive Skill
-|tattoo_stat_text                        = 2% increased Recovery Rate of [[Life]], [[Mana]] and [[Energy Shield]] per Tribe for which you have an allocated Tattoo
+|tattoo_target                           = Small Attribute
+|tattoo_tribe                            = 10
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Attribute}} {{c|default|Passive Skill}}
 |tattoo_limit                            = 1 Ancestral Tattoo
-|description                             = Replaces a Small Attribute Passive Skill<br>2% increased Recovery Rate of [[Life]], [[Mana]] and [[Energy Shield]] per Tribe for which you have an allocated Tattoo<br>Limit 1 Ancestral Tattoo
-|inventory_icon                          = 
+|implicit2_text                          = {{c|default|Limit}} {{c|value|1 Ancestral Tattoo}}
+|tattoo_stat_text                        = 2% increased Recovery Rate of Life, Mana and Energy Shield
+per Tribe for which you have an allocated Tattoo
+|description                             = Grants "2% increased Recovery Rate of Life, [[Mana]] and Energy Shield
+per Tribe for which you have an allocated Tattoo"
 |drop_text                               = Can be obtained as a reward from [[Hinekora]] for completing a tournament in the [[Trial of the Ancestors]].
 |is_drop_restricted                      = true
 |release_version                         = 3.22.1