--- A Familiar Call
+++ Export
@@ -12,9 +12,9 @@
 |description                             = {{c|magic|Jewellery of Farrul}}<br>{{c|default|Item Level:}} {{c|normal|100}}<br>{{c|default|Shaper + Hunter Item}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
+|card_background                         = 0,4
 |drop_monsters                           = Metadata/Monsters/AtlasBosses/ChimeraBoss
 |release_version                         = 3.11.0
-|card_background                         = 0,4
 }}
 
 '''A Familiar Call''' is a [[divination card]]. A set of three can be exchanged for a random Shaper and Hunter [[influenced]] [[magic]] [[jewellery]] item with the {{ml|GrantsCatAspectCrafted}}<!-- of Farrul --> suffix and [[item level]] 100.
