--- Sanctum Necropolis Research
+++ Export
@@ -1,20 +1,20 @@
 {{Base item
 |rarity_id                               = normal
 |name                                    = Sanctum Necropolis Research
-|class_id                                = MiscMapItem
+|class_id                                = ItemisedSanctum
 |size_x                                  = 1
 |size_y                                  = 1
-|implicit1_text = {{c|stat|Area Level: }}{{c|normal|(68-83)}}<br>{{c|stat|Resolve: }}{{c|normal|#}}{{c|stat|/}}{{c|normal|300}}<br>{{c|stat|Inspiration: }}{{c|normal|#}}<br>{{c|stat|Aureus: }}{{c|normal|#}}<br>{{c|stat|Minor Boons: }}{{c|craftingdefences|<Minor Boons>}}<br>{{c|stat|Minor Afflictions: }}{{c|prophecy|<Minor Afflictions>}}
-|explicit1_text = <Modifiers from Relics>
-|is_unmodifiable = true
 |flavour_text                            = The Necropolis holds the darkest secrets of our religion,<br>lies that were never meant to be unearthed.
-|tags                                    = not_for_sale
+|drop_level                              = 68
+|drop_rarities_ids                       = normal
+|tags                                    = not_for_sale, default
 |metadata_id                             = Metadata/Items/Sanctum/SanctumFloor4
 |help_text                               = Right click this item to view details of the Sanctum in progress. Take this item to the Relic Altar in the Forbidden Sanctum to enter.
-|is_drop_restricted                               = true
-|drop_level                               = 68
 |drop_text                               = Obtained by completing the Sanctum Cathedral floor in [[the Forbidden Sanctum]].
+|is_drop_restricted                      = true
+|is_unmodifiable                         = true
 |release_version                         = 3.22.0
+|implicit1_text                          = {{c|stat|Area Level: }}{{c|normal|(68-83)}}<br>{{c|stat|Resolve: }}{{c|normal|#}}{{c|stat|/}}{{c|normal|300}}<br>{{c|stat|Inspiration: }}{{c|normal|#}}<br>{{c|stat|Aureus: }}{{c|normal|#}}<br>{{c|stat|Minor Boons: }}{{c|craftingdefences|<Minor Boons>}}<br>{{c|stat|Minor Afflictions: }}{{c|prophecy|<Minor Afflictions>}}
 }}
 
 '''Sanctum Necropolis Research''' is a [[sanctum research]] item. It can be inserted into the Relic Altar to access the Sanctum Necropolis floor of [[the Forbidden Sanctum]].
