--- Vaal Scouting Report
+++ Export
@@ -8,7 +8,7 @@
 |tags                                    = drops_in_maps_only, currency, default
 |metadata_id                             = Metadata/Items/Currency/ScoutingReports/AtlasScoutingReportCorrupted
 |stack_size                              = 20
-|description                             = Reroll all of Kirac's Atlas Missions, corrupting all non-Unique Maps.
+|description                             = Reroll all of Kirac's Atlas Missions, corrupting all non-Unique Maps
 |help_text                               = Right click this item while viewing Kirac's Atlas Missions to use it.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |inventory_icon                          = Scouting Report
