--- Wiki
+++ Export
@@ -28,9 +28,9 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = base_skill_show_average_damage_instead_of_dps
 |static_stat1_value                      = 1
-|static_stat2_id                         = is_area_damage
+|static_stat2_id                         = cannot_cancel_skill_before_contact_point
 |static_stat2_value                      = 1
-|static_stat3_id                         = cannot_cancel_skill_before_contact_point
+|static_stat3_id                         = is_area_damage
 |static_stat3_value                      = 1
 |static_stat4_id                         = skill_can_add_multiple_charges_per_action
 |static_stat4_value                      = 1
