--- Haunted Cephalopod Tricorne
+++ Export
@@ -0,0 +1,11 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Haunted Cephalopod Tricorne
+|class_id                                = Microtransaction
+|size_x                                  = 2
+|size_y                                  = 2
+|tags                                    = not_for_sale, default
+|metadata_id                             = Metadata/Items/MicrotransactionItemEffects/MicrotransactionHauntedCephalopodTricorne
+|stack_size                              = 1
+|description                             = Changes the appearance of any equipped helmet to the Haunted Cephalopod Tricorne. The sight of a tormented spirit sends the octopus on your head into a flying rage.
+}}