--- High-Impact Mine Support
+++ Export
@@ -7,7 +7,7 @@
 |drop_level                              = 31
 |tags                                    = support_gem, gem, default
 |metadata_id                             = Metadata/Items/Gems/SupportGemHighImpactMineSupport
-|help_text                               = This is a Support Gem. It does not grant a bonus to your character, but to skills in sockets connected to it. Place into an item socket connected to a socket containing the Active Skill Gem you wish to augment. Right click to remove from a socket.
+|help_text                               = This is a Support Gem. It does not grant a bonus to your character, but to skills in sockets connected to it. Place into an item socket connected to a socket containing the Skill Gem you wish to augment. Right click to remove from a socket.
 |dexterity_percent                       = 40
 |intelligence_percent                    = 60
 |gem_tags                                = Mine, Support, AoE, Aura
@@ -30,14 +30,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = support_remote_mine_2_mine_override_pvp_scaling_time_ms
-|static_stat1_value                      = 250
-|static_stat2_id                         = support_makes_skill_mine_pvp_damage_+%_final
-|static_stat2_value                      = -60
-|static_stat3_id                         = base_skill_is_mined
-|static_stat3_value                      = 1
-|static_stat4_id                         = base_skill_show_average_damage_instead_of_dps
-|static_stat4_value                      = 1
+|static_stat1_id                         = base_skill_is_mined
+|static_stat1_value                      = 1
+|static_stat2_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat2_value                      = 1
+|static_stat3_id                         = support_makes_skill_mine_pvp_damage_+%_final
+|static_stat3_value                      = -60
+|static_stat4_id                         = support_remote_mine_2_mine_override_pvp_scaling_time_ms
+|static_stat4_value                      = 250
 |static_stat5_id                         = disable_skill_if_melee_attack
 |static_stat5_value                      = 1
 |static_stat6_id                         = is_remote_mine
