--- Vivid Crystallised Lifeforce
+++ Export
@@ -28,8 +28,8 @@
 Metadata/Monsters/LeagueHarvest/Green/HarvestVultureParasiteT3Parasite,
 Metadata/Monsters/LeagueHarvest/Green/HarvestCatBoss
 |is_drop_restricted                      = true
+|is_sellable                             = false
 |release_version                         = 3.19.0
-|is_sellable                             = false
 }}
 
 '''Vivid Crystallised Lifeforce''' is one of the three Crystallised [[Lifeforce]]s that are dropped from [[harvest]] monsters, once all monsters of that plot has been killed.