--- The Dungeon Master
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = jewellery_divination, unique_divination, unique_jewellery_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardTheDungeonMaster
 |stack_size                              = 7
-|description                             = {{c|unique|Belt}}<br>{{c|default|Double-Influenced Item}}
+|description                             = {{c|unique|[[Belt]]}}<br>{{c|default|Double-Influenced Item}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_areas                              = MapWorldsCage, MapWorldsCells, MapWorldsDungeon, MapWorldsPen
