--- Hunter's Resolve
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = weapon_divination, unique_divination, unique_weapon_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardHuntersResolve
 |stack_size                              = 8
-|description                             = {{c|unique|Bow}}
+|description                             = {{c|unique|[[Bow]]}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_areas                              = 2_6_10, <!--2_6_12, 2_8_1,--> MapAtlasSprings, MapWorldsRamparts, MapWorldsForkingRiver, 2_6_12, 2_8_1
