--- Wiki
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = jewellery_divination, unique_divination, corrupted_divination, unique_corrupted_divination, unique_jewellery_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardTheGulf
 |stack_size                              = 5
-|description                             = {{c|unique|{{il|html=|page=Thread of Hope}}}}<br>{{c|corrupted|[[Two-Implicit]]}}<br>{{c|corrupted|[[Corrupted]]}}
+|description                             = {{c|unique|{{il|html=|Thread of Hope}}}}<br>{{c|corrupted|[[Two-Implicit]]}}<br>{{c|corrupted|[[Corrupted]]}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_text                               = Drops from Uber Elder only. It is unknown if this can drop from the Feared with a witnessed Uber Elder.
