--- Wiki
+++ Export
@@ -537,7 +537,6 @@
 |level40                                 = True
 |level40_level_requirement               = 100
 |level40_cost_amounts                    = 26
-|level40_damage_effectiveness            = 100
 |level40_stat_text                       = Deals 2801 to 4202 Fire Damage<br>Explosion deals base Fire Damage equal to 9.4% of the corpse's Maximum Life<br>+8 to radius
 |level40_stat1_id                        = spell_minimum_base_fire_damage
 |level40_stat1_value                     = 2801
