--- Hydrosphere
+++ Export
@@ -55,7 +55,7 @@
 |static_stat10_value                     = -10
 |static_stat11_id                        = water_sphere_does_weird_conversion_stuff
 |static_stat11_value                     = 1
-|stat_text                               = Deals (37-510) to (56-765) Physical Damage<br>Base duration is 8 seconds<br>Base secondary duration is 4 seconds<br>Pulses every 0.4 seconds while Frozen, Shocked, Brittle or Sapped<br>Can only be Hit once every 1 second<br>Drenched Enemies have Cold and Lightning Exposure, applying -10% to Resistances<br>100% of Physical Damage Converted to Cold Damage while Sphere has no Ailments<br>100% of Physical Damage Converted to Lightning Damage while sphere is Shocked or Sapped<br>100% of Physical Damage Converted to Cold Damage while Sphere is Frozen or Brittle
+|stat_text                               = Deals (37-510) to (56-765) Physical Damage<br>Base duration is 8 seconds<br>Base secondary duration is 4 seconds<br>Pulses every 0.4 seconds while Frozen, Shocked, Brittle or Sapped<br>100% of Physical Damage Converted to Cold Damage<br>Can only be Hit once every 1 second<br>Drenched Enemies have Cold and Lightning Exposure, applying -10% to Resistances<br>100% of Physical Damage Converted to Cold Damage while Sphere has no Ailments<br>100% of Physical Damage Converted to Lightning Damage while sphere is Shocked or Sapped<br>100% of Physical Damage Converted to Cold Damage while Sphere is Frozen or Brittle
 |level1                                  = True
 |level1_level_requirement                = 34
 |level1_cost_amounts                     = 12