--- Tattoo of the Ramako Makanga
+++ Export
@@ -10,7 +10,7 @@
 |stack_size                              = 10
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
-|tattoo_target                           = Small Dexterity
-|tattoo_stat_text                        = [[Skill|Skills]] fire an additional Projectile
-|description                             = Replaces a Small Dexterity Passive Skill<br>Grants "[[Skill|Skills]] fire an additional Projectile"
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Dexterity}} {{c|default|Passive Skill}}
+|implicit2_text                          = {{c|default|Requires}} {{c|value|7}} {{c|default|adjacent Passive Skills Allocated}}
+|description                             = Grants "[[Skill|Skills]] fire an additional Projectile"
 }}