--- The Journalist
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = armour_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardTheJournalist
 |stack_size                              = 10
-|description                             = {{c|rare|Helmet}}<br>{{c|default|Double-Veiled Item}}
+|description                             = {{c|rare|[[Helmet]]}}<br>{{c|default|Double-Veiled Item}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_areas                              = MapWorldsHauntedMansion, MapWorldsLookout, MapWorldsCrimsonTemple, MapWorldsCitySquare, MapWorldsDefiledCathedral, MapWorldsSilo, MapWorldsAlleyways
