--- Tattoo of the Hinekora Storyteller
+++ Export
@@ -10,7 +10,6 @@
 |stack_size                              = 10
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
-|tattoo_target                           = Small Intelligence
-|tattoo_stat_text                        = 3% increased [[Skill]] Effect Duration
-|description                             = Replaces a Small Intelligence Passive Skill<br>Grants "3% increased [[Skill]] Effect Duration"
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Intelligence}} {{c|default|Passive Skill}}
+|description                             = Grants "3% increased [[Skill]] Effect Duration"
 }}