--- Wiki
+++ Export
@@ -36,9 +36,9 @@
 |static_duration                         = 1
 |static_stat1_id                         = display_minion_monster_type
 |static_stat1_value                      = 19
-|static_stat2_id                         = max_number_of_vaal_absolution_sentinels
+|static_stat2_id                         = is_area_damage
 |static_stat2_value                      = 1
-|static_stat3_id                         = is_area_damage
+|static_stat3_id                         = max_number_of_vaal_absolution_sentinels
 |static_stat3_value                      = 1
 |static_stat4_id                         = base_skill_effect_duration
 |static_stat4_value                      = 1000
