--- Divine Justice
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = armour_divination, unique_divination, unique_armour_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardDivineJustice
 |stack_size                              = 1
-|description                             = {{c|unique|Helmet}}<br>{{c|enchanted|Eternal Labyrinth Enchantment}}
+|description                             = {{c|unique|[[Helmet]]}}<br>{{c|enchanted|Eternal Labyrinth Enchantment}}
 |stack_size_currency_tab                 = 5000
 |drop_text                               = Can drop from an [[Strongbox#Loot|Armourer's Strongbox]].
 |release_version                         = 3.9.0
