--- Ancestral Guild Stash
+++ Export
@@ -0,0 +1,14 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Ancestral Guild Stash
+|class_id                                = HideoutDoodad
+|size_x                                  = 2
+|size_y                                  = 2
+|tags                                    = default
+|metadata_id                             = Metadata/Items/Hideout/HideoutAncestralGuildStash
+|stack_size                              = 20
+|description                             = Creates an object in your hideout
+|help_text                               = Shift click to unstack.
+|is_master_doodad                        = False
+|variation_count                         = 0
+}}