--- Poisonous Concoction
+++ Export
@@ -34,9 +34,9 @@
 |static_attack_speed_multiplier          = 115
 |static_stat1_id                         = base_is_projectile
 |static_stat1_value                      = 1
-|static_stat2_id                         = is_area_damage
+|static_stat2_id                         = console_skill_dont_chase
 |static_stat2_value                      = 1
-|static_stat3_id                         = console_skill_dont_chase
+|static_stat3_id                         = is_area_damage
 |static_stat3_value                      = 1
 |static_stat4_id                         = projectile_behaviour_only_explode
 |static_stat4_value                      = 1
