--- Flame Surge
+++ Export
@@ -43,7 +43,7 @@
 |static_stat4_value                      = 2000
 |static_stat5_id                         = flame_surge_ignite_damage_as_burning_ground_damage_%
 |static_stat5_value                      = 25
-|stat_text                               = Deals (26-832) to (40-1247) Fire Damage<br>Base duration is 4 seconds<br>(50-88)% more Damage with Hits against Burning enemies<br>Cannot Ignite<br>+(0-9) to Area of Effect length<br>Hits on Ignited Enemies create Burning Ground, no more than once every 2 seconds<br>Burning Ground deals 25% of Fire Damage per second of Ignite affecting Enemy
+|stat_text                               = Base duration is 4 seconds<br>Deals (26-832) to (40-1247) Fire Damage<br>(50-88)% more Damage with Hits against Burning enemies<br>Cannot Ignite<br>+(0-9) to Area of Effect length<br>Hits on Ignited Enemies create Burning Ground, no more than once every 2 seconds<br>Burning Ground deals 25% of Fire Damage per second of Ignite affecting Enemy
 |level1                                  = True
 |level1_level_requirement                = 12
 |level1_cost_amounts                     = 6
