--- Wiki
+++ Export
@@ -38,9 +38,9 @@
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
 |static_duration                         = 1
-|static_stat1_id                         = skill_can_fire_arrows
+|static_stat1_id                         = base_is_projectile
 |static_stat1_value                      = 1
-|static_stat2_id                         = base_is_projectile
+|static_stat2_id                         = skill_can_fire_arrows
 |static_stat2_value                      = 1
 |static_stat3_id                         = base_skill_effect_duration
 |static_stat3_value                      = 1000
