--- Shaper's Orb (Tier 2)
+++ Export
@@ -0,0 +1,10 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Shaper's Orb (Tier 2)
+|class_id                                = QuestItem
+|size_x                                  = 1
+|size_y                                  = 1
+|drop_level                              = 69
+|tags                                    = default
+|metadata_id                             = Metadata/Items/QuestItems/MapUpgrades/MapUpgradeTier2_1
+}}