--- Wiki
+++ Export
@@ -14,7 +14,6 @@
 |help_text                               = Shift click to unstack.
 |drop_areas                              = EndGame_Labyrinth_trials_arrows
 |release_version                         = 3.11.0
-|card_background                         = 2
 }}
 
 '''The White Knight''' is a [[divination card]]. A set of eight can be exchanged for a [[Crusader item|Crusader-influenced]] [[normal]] {{il|Astral Plate}} with six linked [[sockets]] and [[item level]] 100.
