--- Shattering Steel
+++ Export
@@ -62,6 +62,8 @@
 |level1                                  = True
 |level1_level_requirement                = 12
 |level1_cost_amounts                     = 7
+|level1_damage_effectiveness             = 100
+|level1_damage_multiplier                = 100
 |level1_stat_text                        = 5 to 7 Added Attack Physical Damage<br>Projectiles deal up to 50% more Damage with Hits per Steel Shard consumed at the start of their movement, lowering this bonus as they travel farther
 |level1_stat1_id                         = attack_minimum_added_physical_damage
 |level1_stat1_value                      = 5
