--- Explosive Trap
+++ Export
@@ -45,7 +45,7 @@
 |static_stat5_value                      = 1
 |static_stat6_id                         = skill_physical_damage_%_to_convert_to_fire
 |static_stat6_value                      = 50
-|stat_text                               = Deals (4-437) to (6-655) Physical Damage<br>Trap lasts 4 seconds<br>+(0-6) to radius<br>50% of Physical Damage Converted to Fire Damage<br>Causes (3-7) smaller explosions
+|stat_text                               = Deals (4-437) to (6-655) Physical Damage<br>Trap lasts 4 seconds<br>+(0-0.6) metres to radius<br>50% of Physical Damage Converted to Fire Damage<br>Causes (3-7) smaller explosions
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_cost_amounts                     = 6
@@ -69,7 +69,7 @@
 |level3                                  = True
 |level3_level_requirement                = 4
 |level3_cost_amounts                     = 7
-|level3_stat_text                        = Deals 7 to 10 Physical Damage<br>+1 to radius<br>Causes 3 smaller explosions
+|level3_stat_text                        = Deals 7 to 10 Physical Damage<br>+0.1 metres to radius<br>Causes 3 smaller explosions
 |level3_stat1_id                         = spell_minimum_base_physical_damage
 |level3_stat1_value                      = 7
 |level3_stat2_id                         = spell_maximum_base_physical_damage
@@ -81,7 +81,7 @@
 |level4                                  = True
 |level4_level_requirement                = 7
 |level4_cost_amounts                     = 8
-|level4_stat_text                        = Deals 10 to 15 Physical Damage<br>+1 to radius<br>Causes 3 smaller explosions
+|level4_stat_text                        = Deals 10 to 15 Physical Damage<br>+0.1 metres to radius<br>Causes 3 smaller explosions
 |level4_stat1_id                         = spell_minimum_base_physical_damage
 |level4_stat1_value                      = 10
 |level4_stat2_id                         = spell_maximum_base_physical_damage
@@ -93,7 +93,7 @@
 |level5                                  = True
 |level5_level_requirement                = 11
 |level5_cost_amounts                     = 9
-|level5_stat_text                        = Deals 14 to 22 Physical Damage<br>+1 to radius<br>Causes 4 smaller explosions
+|level5_stat_text                        = Deals 14 to 22 Physical Damage<br>+0.1 metres to radius<br>Causes 4 smaller explosions
 |level5_stat1_id                         = spell_minimum_base_physical_damage
 |level5_stat1_value                      = 14
 |level5_stat2_id                         = spell_maximum_base_physical_damage
@@ -105,7 +105,7 @@
 |level6                                  = True
 |level6_level_requirement                = 16
 |level6_cost_amounts                     = 10
-|level6_stat_text                        = Deals 21 to 32 Physical Damage<br>+2 to radius<br>Causes 4 smaller explosions
+|level6_stat_text                        = Deals 21 to 32 Physical Damage<br>+0.2 metres to radius<br>Causes 4 smaller explosions
 |level6_stat1_id                         = spell_minimum_base_physical_damage
 |level6_stat1_value                      = 21
 |level6_stat2_id                         = spell_maximum_base_physical_damage
@@ -117,7 +117,7 @@
 |level7                                  = True
 |level7_level_requirement                = 20
 |level7_cost_amounts                     = 11
-|level7_stat_text                        = Deals 28 to 43 Physical Damage<br>+2 to radius<br>Causes 4 smaller explosions
+|level7_stat_text                        = Deals 28 to 43 Physical Damage<br>+0.2 metres to radius<br>Causes 4 smaller explosions
 |level7_stat1_id                         = spell_minimum_base_physical_damage
 |level7_stat1_value                      = 28
 |level7_stat2_id                         = spell_maximum_base_physical_damage
@@ -129,7 +129,7 @@
 |level8                                  = True
 |level8_level_requirement                = 24
 |level8_cost_amounts                     = 12
-|level8_stat_text                        = Deals 37 to 55 Physical Damage<br>+2 to radius<br>Causes 4 smaller explosions
+|level8_stat_text                        = Deals 37 to 55 Physical Damage<br>+0.2 metres to radius<br>Causes 4 smaller explosions
 |level8_stat1_id                         = spell_minimum_base_physical_damage
 |level8_stat1_value                      = 37
 |level8_stat2_id                         = spell_maximum_base_physical_damage
@@ -141,7 +141,7 @@
 |level9                                  = True
 |level9_level_requirement                = 28
 |level9_cost_amounts                     = 13
-|level9_stat_text                        = Deals 48 to 71 Physical Damage<br>+3 to radius<br>Causes 5 smaller explosions
+|level9_stat_text                        = Deals 48 to 71 Physical Damage<br>+0.3 metres to radius<br>Causes 5 smaller explosions
 |level9_stat1_id                         = spell_minimum_base_physical_damage
 |level9_stat1_value                      = 48
 |level9_stat2_id                         = spell_maximum_base_physical_damage
@@ -153,7 +153,7 @@
 |level10                                 = True
 |level10_level_requirement               = 32
 |level10_cost_amounts                    = 14
-|level10_stat_text                       = Deals 61 to 91 Physical Damage<br>+3 to radius<br>Causes 5 smaller explosions
+|level10_stat_text                       = Deals 61 to 91 Physical Damage<br>+0.3 metres to radius<br>Causes 5 smaller explosions
 |level10_stat1_id                        = spell_minimum_base_physical_damage
 |level10_stat1_value                     = 61
 |level10_stat2_id                        = spell_maximum_base_physical_damage
@@ -165,7 +165,7 @@
 |level11                                 = True
 |level11_level_requirement               = 36
 |level11_cost_amounts                    = 14
-|level11_stat_text                       = Deals 76 to 114 Physical Damage<br>+3 to radius<br>Causes 5 smaller explosions
+|level11_stat_text                       = Deals 76 to 114 Physical Damage<br>+0.3 metres to radius<br>Causes 5 smaller explosions
 |level11_stat1_id                        = spell_minimum_base_physical_damage
 |level11_stat1_value                     = 76
 |level11_stat2_id                        = spell_maximum_base_physical_damage
@@ -177,7 +177,7 @@
 |level12                                 = True
 |level12_level_requirement               = 40
 |level12_cost_amounts                    = 15
-|level12_stat_text                       = Deals 95 to 143 Physical Damage<br>+4 to radius<br>Causes 5 smaller explosions
+|level12_stat_text                       = Deals 95 to 143 Physical Damage<br>+0.4 metres to radius<br>Causes 5 smaller explosions
 |level12_stat1_id                        = spell_minimum_base_physical_damage
 |level12_stat1_value                     = 95
 |level12_stat2_id                        = spell_maximum_base_physical_damage
@@ -189,7 +189,7 @@
 |level13                                 = True
 |level13_level_requirement               = 44
 |level13_cost_amounts                    = 16
-|level13_stat_text                       = Deals 119 to 178 Physical Damage<br>+4 to radius<br>Causes 6 smaller explosions
+|level13_stat_text                       = Deals 119 to 178 Physical Damage<br>+0.4 metres to radius<br>Causes 6 smaller explosions
 |level13_stat1_id                        = spell_minimum_base_physical_damage
 |level13_stat1_value                     = 119
 |level13_stat2_id                        = spell_maximum_base_physical_damage
@@ -201,7 +201,7 @@
 |level14                                 = True
 |level14_level_requirement               = 48
 |level14_cost_amounts                    = 16
-|level14_stat_text                       = Deals 146 to 220 Physical Damage<br>+4 to radius<br>Causes 6 smaller explosions
+|level14_stat_text                       = Deals 146 to 220 Physical Damage<br>+0.4 metres to radius<br>Causes 6 smaller explosions
 |level14_stat1_id                        = spell_minimum_base_physical_damage
 |level14_stat1_value                     = 146
 |level14_stat2_id                        = spell_maximum_base_physical_damage
@@ -213,7 +213,7 @@
 |level15                                 = True
 |level15_level_requirement               = 52
 |level15_cost_amounts                    = 17
-|level15_stat_text                       = Deals 180 to 270 Physical Damage<br>+5 to radius<br>Causes 6 smaller explosions
+|level15_stat_text                       = Deals 180 to 270 Physical Damage<br>+0.5 metres to radius<br>Causes 6 smaller explosions
 |level15_stat1_id                        = spell_minimum_base_physical_damage
 |level15_stat1_value                     = 180
 |level15_stat2_id                        = spell_maximum_base_physical_damage
@@ -225,7 +225,7 @@
 |level16                                 = True
 |level16_level_requirement               = 56
 |level16_cost_amounts                    = 18
-|level16_stat_text                       = Deals 221 to 331 Physical Damage<br>+5 to radius<br>Causes 6 smaller explosions
+|level16_stat_text                       = Deals 221 to 331 Physical Damage<br>+0.5 metres to radius<br>Causes 6 smaller explosions
 |level16_stat1_id                        = spell_minimum_base_physical_damage
 |level16_stat1_value                     = 221
 |level16_stat2_id                        = spell_maximum_base_physical_damage
@@ -237,7 +237,7 @@
 |level17                                 = True
 |level17_level_requirement               = 60
 |level17_cost_amounts                    = 18
-|level17_stat_text                       = Deals 269 to 404 Physical Damage<br>+5 to radius<br>Causes 7 smaller explosions
+|level17_stat_text                       = Deals 269 to 404 Physical Damage<br>+0.5 metres to radius<br>Causes 7 smaller explosions
 |level17_stat1_id                        = spell_minimum_base_physical_damage
 |level17_stat1_value                     = 269
 |level17_stat2_id                        = spell_maximum_base_physical_damage
@@ -249,7 +249,7 @@
 |level18                                 = True
 |level18_level_requirement               = 64
 |level18_cost_amounts                    = 19
-|level18_stat_text                       = Deals 327 to 491 Physical Damage<br>+6 to radius<br>Causes 7 smaller explosions
+|level18_stat_text                       = Deals 327 to 491 Physical Damage<br>+0.6 metres to radius<br>Causes 7 smaller explosions
 |level18_stat1_id                        = spell_minimum_base_physical_damage
 |level18_stat1_value                     = 327
 |level18_stat2_id                        = spell_maximum_base_physical_damage
@@ -261,7 +261,7 @@
 |level19                                 = True
 |level19_level_requirement               = 67
 |level19_cost_amounts                    = 19
-|level19_stat_text                       = Deals 378 to 567 Physical Damage<br>+6 to radius<br>Causes 7 smaller explosions
+|level19_stat_text                       = Deals 378 to 567 Physical Damage<br>+0.6 metres to radius<br>Causes 7 smaller explosions
 |level19_stat1_id                        = spell_minimum_base_physical_damage
 |level19_stat1_value                     = 378
 |level19_stat2_id                        = spell_maximum_base_physical_damage
@@ -273,7 +273,7 @@
 |level20                                 = True
 |level20_level_requirement               = 70
 |level20_cost_amounts                    = 20
-|level20_stat_text                       = Deals 437 to 655 Physical Damage<br>+6 to radius<br>Causes 7 smaller explosions
+|level20_stat_text                       = Deals 437 to 655 Physical Damage<br>+0.6 metres to radius<br>Causes 7 smaller explosions
 |level20_stat1_id                        = spell_minimum_base_physical_damage
 |level20_stat1_value                     = 437
 |level20_stat2_id                        = spell_maximum_base_physical_damage
@@ -285,7 +285,7 @@
 |level21                                 = True
 |level21_level_requirement               = 72
 |level21_cost_amounts                    = 20
-|level21_stat_text                       = Deals 480 to 720 Physical Damage<br>+7 to radius<br>Causes 8 smaller explosions
+|level21_stat_text                       = Deals 480 to 720 Physical Damage<br>+0.7 metres to radius<br>Causes 8 smaller explosions
 |level21_stat1_id                        = spell_minimum_base_physical_damage
 |level21_stat1_value                     = 480
 |level21_stat2_id                        = spell_maximum_base_physical_damage
@@ -297,7 +297,7 @@
 |level22                                 = True
 |level22_level_requirement               = 74
 |level22_cost_amounts                    = 21
-|level22_stat_text                       = Deals 527 to 791 Physical Damage<br>+7 to radius<br>Causes 8 smaller explosions
+|level22_stat_text                       = Deals 527 to 791 Physical Damage<br>+0.7 metres to radius<br>Causes 8 smaller explosions
 |level22_stat1_id                        = spell_minimum_base_physical_damage
 |level22_stat1_value                     = 527
 |level22_stat2_id                        = spell_maximum_base_physical_damage
@@ -309,7 +309,7 @@
 |level23                                 = True
 |level23_level_requirement               = 76
 |level23_cost_amounts                    = 21
-|level23_stat_text                       = Deals 579 to 869 Physical Damage<br>+7 to radius<br>Causes 8 smaller explosions
+|level23_stat_text                       = Deals 579 to 869 Physical Damage<br>+0.7 metres to radius<br>Causes 8 smaller explosions
 |level23_stat1_id                        = spell_minimum_base_physical_damage
 |level23_stat1_value                     = 579
 |level23_stat2_id                        = spell_maximum_base_physical_damage
@@ -321,7 +321,7 @@
 |level24                                 = True
 |level24_level_requirement               = 78
 |level24_cost_amounts                    = 21
-|level24_stat_text                       = Deals 635 to 953 Physical Damage<br>+8 to radius<br>Causes 8 smaller explosions
+|level24_stat_text                       = Deals 635 to 953 Physical Damage<br>+0.8 metres to radius<br>Causes 8 smaller explosions
 |level24_stat1_id                        = spell_minimum_base_physical_damage
 |level24_stat1_value                     = 635
 |level24_stat2_id                        = spell_maximum_base_physical_damage
@@ -333,7 +333,7 @@
 |level25                                 = True
 |level25_level_requirement               = 80
 |level25_cost_amounts                    = 22
-|level25_stat_text                       = Deals 697 to 1046 Physical Damage<br>+8 to radius<br>Causes 8 smaller explosions
+|level25_stat_text                       = Deals 697 to 1046 Physical Damage<br>+0.8 metres to radius<br>Causes 8 smaller explosions
 |level25_stat1_id                        = spell_minimum_base_physical_damage
 |level25_stat1_value                     = 697
 |level25_stat2_id                        = spell_maximum_base_physical_damage
@@ -345,7 +345,7 @@
 |level26                                 = True
 |level26_level_requirement               = 82
 |level26_cost_amounts                    = 23
-|level26_stat_text                       = Deals 764 to 1146 Physical Damage<br>+8 to radius<br>Causes 9 smaller explosions
+|level26_stat_text                       = Deals 764 to 1146 Physical Damage<br>+0.8 metres to radius<br>Causes 9 smaller explosions
 |level26_stat1_id                        = spell_minimum_base_physical_damage
 |level26_stat1_value                     = 764
 |level26_stat2_id                        = spell_maximum_base_physical_damage
@@ -357,7 +357,7 @@
 |level27                                 = True
 |level27_level_requirement               = 84
 |level27_cost_amounts                    = 23
-|level27_stat_text                       = Deals 837 to 1256 Physical Damage<br>+9 to radius<br>Causes 9 smaller explosions
+|level27_stat_text                       = Deals 837 to 1256 Physical Damage<br>+0.9 metres to radius<br>Causes 9 smaller explosions
 |level27_stat1_id                        = spell_minimum_base_physical_damage
 |level27_stat1_value                     = 837
 |level27_stat2_id                        = spell_maximum_base_physical_damage
@@ -369,7 +369,7 @@
 |level28                                 = True
 |level28_level_requirement               = 86
 |level28_cost_amounts                    = 23
-|level28_stat_text                       = Deals 917 to 1376 Physical Damage<br>+9 to radius<br>Causes 9 smaller explosions
+|level28_stat_text                       = Deals 917 to 1376 Physical Damage<br>+0.9 metres to radius<br>Causes 9 smaller explosions
 |level28_stat1_id                        = spell_minimum_base_physical_damage
 |level28_stat1_value                     = 917
 |level28_stat2_id                        = spell_maximum_base_physical_damage
@@ -381,7 +381,7 @@
 |level29                                 = True
 |level29_level_requirement               = 88
 |level29_cost_amounts                    = 24
-|level29_stat_text                       = Deals 1004 to 1506 Physical Damage<br>+9 to radius<br>Causes 9 smaller explosions
+|level29_stat_text                       = Deals 1004 to 1506 Physical Damage<br>+0.9 metres to radius<br>Causes 9 smaller explosions
 |level29_stat1_id                        = spell_minimum_base_physical_damage
 |level29_stat1_value                     = 1004
 |level29_stat2_id                        = spell_maximum_base_physical_damage
@@ -393,7 +393,7 @@
 |level30                                 = True
 |level30_level_requirement               = 90
 |level30_cost_amounts                    = 24
-|level30_stat_text                       = Deals 1099 to 1649 Physical Damage<br>+10 to radius<br>Causes 9 smaller explosions
+|level30_stat_text                       = Deals 1099 to 1649 Physical Damage<br>+1 metre to radius<br>Causes 9 smaller explosions
 |level30_stat1_id                        = spell_minimum_base_physical_damage
 |level30_stat1_value                     = 1099
 |level30_stat2_id                        = spell_maximum_base_physical_damage
@@ -405,7 +405,7 @@
 |level31                                 = True
 |level31_level_requirement               = 91
 |level31_cost_amounts                    = 25
-|level31_stat_text                       = Deals 1150 to 1724 Physical Damage<br>+10 to radius<br>Causes 9 smaller explosions
+|level31_stat_text                       = Deals 1150 to 1724 Physical Damage<br>+1 metre to radius<br>Causes 9 smaller explosions
 |level31_stat1_id                        = spell_minimum_base_physical_damage
 |level31_stat1_value                     = 1150
 |level31_stat2_id                        = spell_maximum_base_physical_damage
@@ -417,7 +417,7 @@
 |level32                                 = True
 |level32_level_requirement               = 92
 |level32_cost_amounts                    = 25
-|level32_stat_text                       = Deals 1202 to 1804 Physical Damage<br>+10 to radius<br>Causes 10 smaller explosions
+|level32_stat_text                       = Deals 1202 to 1804 Physical Damage<br>+1 metre to radius<br>Causes 10 smaller explosions
 |level32_stat1_id                        = spell_minimum_base_physical_damage
 |level32_stat1_value                     = 1202
 |level32_stat2_id                        = spell_maximum_base_physical_damage
@@ -429,7 +429,7 @@
 |level33                                 = True
 |level33_level_requirement               = 93
 |level33_cost_amounts                    = 25
-|level33_stat_text                       = Deals 1257 to 1886 Physical Damage<br>+10 to radius<br>Causes 10 smaller explosions
+|level33_stat_text                       = Deals 1257 to 1886 Physical Damage<br>+1 metre to radius<br>Causes 10 smaller explosions
 |level33_stat1_id                        = spell_minimum_base_physical_damage
 |level33_stat1_value                     = 1257
 |level33_stat2_id                        = spell_maximum_base_physical_damage
@@ -441,7 +441,7 @@
 |level34                                 = True
 |level34_level_requirement               = 94
 |level34_cost_amounts                    = 26
-|level34_stat_text                       = Deals 1315 to 1972 Physical Damage<br>+10 to radius<br>Causes 10 smaller explosions
+|level34_stat_text                       = Deals 1315 to 1972 Physical Damage<br>+1 metre to radius<br>Causes 10 smaller explosions
 |level34_stat1_id                        = spell_minimum_base_physical_damage
 |level34_stat1_value                     = 1315
 |level34_stat2_id                        = spell_maximum_base_physical_damage
@@ -453,7 +453,7 @@
 |level35                                 = True
 |level35_level_requirement               = 95
 |level35_cost_amounts                    = 27
-|level35_stat_text                       = Deals 1375 to 2062 Physical Damage<br>+11 to radius<br>Causes 10 smaller explosions
+|level35_stat_text                       = Deals 1375 to 2062 Physical Damage<br>+1.1 metres to radius<br>Causes 10 smaller explosions
 |level35_stat1_id                        = spell_minimum_base_physical_damage
 |level35_stat1_value                     = 1375
 |level35_stat2_id                        = spell_maximum_base_physical_damage
@@ -465,7 +465,7 @@
 |level36                                 = True
 |level36_level_requirement               = 96
 |level36_cost_amounts                    = 27
-|level36_stat_text                       = Deals 1437 to 2156 Physical Damage<br>+11 to radius<br>Causes 10 smaller explosions
+|level36_stat_text                       = Deals 1437 to 2156 Physical Damage<br>+1.1 metres to radius<br>Causes 10 smaller explosions
 |level36_stat1_id                        = spell_minimum_base_physical_damage
 |level36_stat1_value                     = 1437
 |level36_stat2_id                        = spell_maximum_base_physical_damage
@@ -477,7 +477,7 @@
 |level37                                 = True
 |level37_level_requirement               = 97
 |level37_cost_amounts                    = 27
-|level37_stat_text                       = Deals 1503 to 2254 Physical Damage<br>+11 to radius<br>Causes 10 smaller explosions
+|level37_stat_text                       = Deals 1503 to 2254 Physical Damage<br>+1.1 metres to radius<br>Causes 10 smaller explosions
 |level37_stat1_id                        = spell_minimum_base_physical_damage
 |level37_stat1_value                     = 1503
 |level37_stat2_id                        = spell_maximum_base_physical_damage
@@ -489,7 +489,7 @@
 |level38                                 = True
 |level38_level_requirement               = 98
 |level38_cost_amounts                    = 28
-|level38_stat_text                       = Deals 1571 to 2356 Physical Damage<br>+11 to radius<br>Causes 10 smaller explosions
+|level38_stat_text                       = Deals 1571 to 2356 Physical Damage<br>+1.1 metres to radius<br>Causes 10 smaller explosions
 |level38_stat1_id                        = spell_minimum_base_physical_damage
 |level38_stat1_value                     = 1571
 |level38_stat2_id                        = spell_maximum_base_physical_damage
@@ -501,7 +501,7 @@
 |level39                                 = True
 |level39_level_requirement               = 99
 |level39_cost_amounts                    = 28
-|level39_stat_text                       = Deals 1642 to 2462 Physical Damage<br>+11 to radius<br>Causes 10 smaller explosions
+|level39_stat_text                       = Deals 1642 to 2462 Physical Damage<br>+1.1 metres to radius<br>Causes 10 smaller explosions
 |level39_stat1_id                        = spell_minimum_base_physical_damage
 |level39_stat1_value                     = 1642
 |level39_stat2_id                        = spell_maximum_base_physical_damage
@@ -513,7 +513,7 @@
 |level40                                 = True
 |level40_level_requirement               = 100
 |level40_cost_amounts                    = 29
-|level40_stat_text                       = Deals 1716 to 2574 Physical Damage<br>+11 to radius<br>Causes 10 smaller explosions
+|level40_stat_text                       = Deals 1716 to 2574 Physical Damage<br>+1.1 metres to radius<br>Causes 10 smaller explosions
 |level40_stat1_id                        = spell_minimum_base_physical_damage
 |level40_stat1_value                     = 1716
 |level40_stat2_id                        = spell_maximum_base_physical_damage