--- Shield Crush
+++ Export
@@ -41,7 +41,7 @@
 |static_stat2_value                      = 1
 |static_stat3_id                         = is_area_damage
 |static_stat3_value                      = 1
-|stat_text                               = (4-172) to (6-258) Base Off Hand Physical Damage<br>(3-5) to (4-7) Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+(0-6) to radius
+|stat_text                               = (4-172) to (6-258) Base Off Hand Physical Damage<br>(3-5) to (4-7) Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+(0-0.6) metres to radius
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_stat_text                        = 4 to 6 Base Off Hand Physical Damage<br>3 to 4 Added Physical Damage per 15 Armour or Evasion Rating on Shield
@@ -66,7 +66,7 @@
 |level2_stat4_value                      = 4
 |level3                                  = True
 |level3_level_requirement                = 4
-|level3_stat_text                        = 5 to 8 Base Off Hand Physical Damage<br>3 to 4 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+1 to radius
+|level3_stat_text                        = 5 to 8 Base Off Hand Physical Damage<br>3 to 4 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.1 metres to radius
 |level3_stat1_id                         = off_hand_local_minimum_added_physical_damage
 |level3_stat1_value                      = 5
 |level3_stat2_id                         = off_hand_local_maximum_added_physical_damage
@@ -79,7 +79,7 @@
 |level3_stat5_value                      = 1
 |level4                                  = True
 |level4_level_requirement                = 7
-|level4_stat_text                        = 7 to 10 Base Off Hand Physical Damage<br>4 to 5 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+1 to radius
+|level4_stat_text                        = 7 to 10 Base Off Hand Physical Damage<br>4 to 5 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.1 metres to radius
 |level4_stat1_id                         = off_hand_local_minimum_added_physical_damage
 |level4_stat1_value                      = 7
 |level4_stat2_id                         = off_hand_local_maximum_added_physical_damage
@@ -92,7 +92,7 @@
 |level4_stat5_value                      = 1
 |level5                                  = True
 |level5_level_requirement                = 11
-|level5_stat_text                        = 9 to 14 Base Off Hand Physical Damage<br>4 to 5 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+1 to radius
+|level5_stat_text                        = 9 to 14 Base Off Hand Physical Damage<br>4 to 5 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.1 metres to radius
 |level5_stat1_id                         = off_hand_local_minimum_added_physical_damage
 |level5_stat1_value                      = 9
 |level5_stat2_id                         = off_hand_local_maximum_added_physical_damage
@@ -105,7 +105,7 @@
 |level5_stat5_value                      = 1
 |level6                                  = True
 |level6_level_requirement                = 16
-|level6_stat_text                        = 13 to 20 Base Off Hand Physical Damage<br>4 to 5 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+2 to radius
+|level6_stat_text                        = 13 to 20 Base Off Hand Physical Damage<br>4 to 5 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.2 metres to radius
 |level6_stat1_id                         = off_hand_local_minimum_added_physical_damage
 |level6_stat1_value                      = 13
 |level6_stat2_id                         = off_hand_local_maximum_added_physical_damage
@@ -118,7 +118,7 @@
 |level6_stat5_value                      = 2
 |level7                                  = True
 |level7_level_requirement                = 20
-|level7_stat_text                        = 17 to 26 Base Off Hand Physical Damage<br>4 to 5 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+2 to radius
+|level7_stat_text                        = 17 to 26 Base Off Hand Physical Damage<br>4 to 5 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.2 metres to radius
 |level7_stat1_id                         = off_hand_local_minimum_added_physical_damage
 |level7_stat1_value                      = 17
 |level7_stat2_id                         = off_hand_local_maximum_added_physical_damage
@@ -131,7 +131,7 @@
 |level7_stat5_value                      = 2
 |level8                                  = True
 |level8_level_requirement                = 24
-|level8_stat_text                        = 22 to 32 Base Off Hand Physical Damage<br>4 to 5 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+2 to radius
+|level8_stat_text                        = 22 to 32 Base Off Hand Physical Damage<br>4 to 5 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.2 metres to radius
 |level8_stat1_id                         = off_hand_local_minimum_added_physical_damage
 |level8_stat1_value                      = 22
 |level8_stat2_id                         = off_hand_local_maximum_added_physical_damage
@@ -144,7 +144,7 @@
 |level8_stat5_value                      = 2
 |level9                                  = True
 |level9_level_requirement                = 28
-|level9_stat_text                        = 27 to 40 Base Off Hand Physical Damage<br>4 to 5 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+3 to radius
+|level9_stat_text                        = 27 to 40 Base Off Hand Physical Damage<br>4 to 5 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.3 metres to radius
 |level9_stat1_id                         = off_hand_local_minimum_added_physical_damage
 |level9_stat1_value                      = 27
 |level9_stat2_id                         = off_hand_local_maximum_added_physical_damage
@@ -157,7 +157,7 @@
 |level9_stat5_value                      = 3
 |level10                                 = True
 |level10_level_requirement               = 32
-|level10_stat_text                       = 33 to 50 Base Off Hand Physical Damage<br>4 to 5 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+3 to radius
+|level10_stat_text                       = 33 to 50 Base Off Hand Physical Damage<br>4 to 5 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.3 metres to radius
 |level10_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level10_stat1_value                     = 33
 |level10_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -170,7 +170,7 @@
 |level10_stat5_value                     = 3
 |level11                                 = True
 |level11_level_requirement               = 36
-|level11_stat_text                       = 40 to 60 Base Off Hand Physical Damage<br>4 to 5 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+3 to radius
+|level11_stat_text                       = 40 to 60 Base Off Hand Physical Damage<br>4 to 5 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.3 metres to radius
 |level11_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level11_stat1_value                     = 40
 |level11_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -183,7 +183,7 @@
 |level11_stat5_value                     = 3
 |level12                                 = True
 |level12_level_requirement               = 40
-|level12_stat_text                       = 49 to 73 Base Off Hand Physical Damage<br>4 to 6 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+4 to radius
+|level12_stat_text                       = 49 to 73 Base Off Hand Physical Damage<br>4 to 6 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.4 metres to radius
 |level12_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level12_stat1_value                     = 49
 |level12_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -196,7 +196,7 @@
 |level12_stat5_value                     = 4
 |level13                                 = True
 |level13_level_requirement               = 44
-|level13_stat_text                       = 58 to 88 Base Off Hand Physical Damage<br>4 to 6 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+4 to radius
+|level13_stat_text                       = 58 to 88 Base Off Hand Physical Damage<br>4 to 6 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.4 metres to radius
 |level13_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level13_stat1_value                     = 58
 |level13_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -209,7 +209,7 @@
 |level13_stat5_value                     = 4
 |level14                                 = True
 |level14_level_requirement               = 48
-|level14_stat_text                       = 70 to 105 Base Off Hand Physical Damage<br>4 to 6 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+4 to radius
+|level14_stat_text                       = 70 to 105 Base Off Hand Physical Damage<br>4 to 6 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.4 metres to radius
 |level14_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level14_stat1_value                     = 70
 |level14_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -222,7 +222,7 @@
 |level14_stat5_value                     = 4
 |level15                                 = True
 |level15_level_requirement               = 52
-|level15_stat_text                       = 83 to 124 Base Off Hand Physical Damage<br>4 to 6 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+5 to radius
+|level15_stat_text                       = 83 to 124 Base Off Hand Physical Damage<br>4 to 6 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.5 metres to radius
 |level15_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level15_stat1_value                     = 83
 |level15_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -235,7 +235,7 @@
 |level15_stat5_value                     = 5
 |level16                                 = True
 |level16_level_requirement               = 56
-|level16_stat_text                       = 98 to 147 Base Off Hand Physical Damage<br>4 to 6 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+5 to radius
+|level16_stat_text                       = 98 to 147 Base Off Hand Physical Damage<br>4 to 6 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.5 metres to radius
 |level16_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level16_stat1_value                     = 98
 |level16_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -248,7 +248,7 @@
 |level16_stat5_value                     = 5
 |level17                                 = True
 |level17_level_requirement               = 60
-|level17_stat_text                       = 116 to 174 Base Off Hand Physical Damage<br>4 to 6 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+5 to radius
+|level17_stat_text                       = 116 to 174 Base Off Hand Physical Damage<br>4 to 6 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.5 metres to radius
 |level17_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level17_stat1_value                     = 116
 |level17_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -261,7 +261,7 @@
 |level17_stat5_value                     = 5
 |level18                                 = True
 |level18_level_requirement               = 64
-|level18_stat_text                       = 136 to 204 Base Off Hand Physical Damage<br>4 to 6 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+6 to radius
+|level18_stat_text                       = 136 to 204 Base Off Hand Physical Damage<br>4 to 6 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.6 metres to radius
 |level18_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level18_stat1_value                     = 136
 |level18_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -274,7 +274,7 @@
 |level18_stat5_value                     = 6
 |level19                                 = True
 |level19_level_requirement               = 67
-|level19_stat_text                       = 153 to 230 Base Off Hand Physical Damage<br>5 to 7 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+6 to radius
+|level19_stat_text                       = 153 to 230 Base Off Hand Physical Damage<br>5 to 7 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.6 metres to radius
 |level19_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level19_stat1_value                     = 153
 |level19_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -287,7 +287,7 @@
 |level19_stat5_value                     = 6
 |level20                                 = True
 |level20_level_requirement               = 70
-|level20_stat_text                       = 172 to 258 Base Off Hand Physical Damage<br>5 to 7 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+6 to radius
+|level20_stat_text                       = 172 to 258 Base Off Hand Physical Damage<br>5 to 7 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.6 metres to radius
 |level20_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level20_stat1_value                     = 172
 |level20_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -300,7 +300,7 @@
 |level20_stat5_value                     = 6
 |level21                                 = True
 |level21_level_requirement               = 72
-|level21_stat_text                       = 186 to 279 Base Off Hand Physical Damage<br>5 to 7 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+7 to radius
+|level21_stat_text                       = 186 to 279 Base Off Hand Physical Damage<br>5 to 7 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.7 metres to radius
 |level21_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level21_stat1_value                     = 186
 |level21_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -313,7 +313,7 @@
 |level21_stat5_value                     = 7
 |level22                                 = True
 |level22_level_requirement               = 74
-|level22_stat_text                       = 201 to 302 Base Off Hand Physical Damage<br>5 to 7 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+7 to radius
+|level22_stat_text                       = 201 to 302 Base Off Hand Physical Damage<br>5 to 7 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.7 metres to radius
 |level22_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level22_stat1_value                     = 201
 |level22_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -326,7 +326,7 @@
 |level22_stat5_value                     = 7
 |level23                                 = True
 |level23_level_requirement               = 76
-|level23_stat_text                       = 217 to 326 Base Off Hand Physical Damage<br>5 to 7 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+7 to radius
+|level23_stat_text                       = 217 to 326 Base Off Hand Physical Damage<br>5 to 7 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.7 metres to radius
 |level23_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level23_stat1_value                     = 217
 |level23_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -339,7 +339,7 @@
 |level23_stat5_value                     = 7
 |level24                                 = True
 |level24_level_requirement               = 78
-|level24_stat_text                       = 234 to 351 Base Off Hand Physical Damage<br>6 to 8 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+8 to radius
+|level24_stat_text                       = 234 to 351 Base Off Hand Physical Damage<br>6 to 8 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.8 metres to radius
 |level24_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level24_stat1_value                     = 234
 |level24_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -352,7 +352,7 @@
 |level24_stat5_value                     = 8
 |level25                                 = True
 |level25_level_requirement               = 80
-|level25_stat_text                       = 253 to 379 Base Off Hand Physical Damage<br>6 to 8 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+8 to radius
+|level25_stat_text                       = 253 to 379 Base Off Hand Physical Damage<br>6 to 8 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.8 metres to radius
 |level25_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level25_stat1_value                     = 253
 |level25_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -365,7 +365,7 @@
 |level25_stat5_value                     = 8
 |level26                                 = True
 |level26_level_requirement               = 82
-|level26_stat_text                       = 272 to 408 Base Off Hand Physical Damage<br>6 to 8 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+8 to radius
+|level26_stat_text                       = 272 to 408 Base Off Hand Physical Damage<br>6 to 8 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.8 metres to radius
 |level26_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level26_stat1_value                     = 272
 |level26_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -378,7 +378,7 @@
 |level26_stat5_value                     = 8
 |level27                                 = True
 |level27_level_requirement               = 84
-|level27_stat_text                       = 293 to 440 Base Off Hand Physical Damage<br>6 to 8 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+9 to radius
+|level27_stat_text                       = 293 to 440 Base Off Hand Physical Damage<br>6 to 8 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.9 metres to radius
 |level27_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level27_stat1_value                     = 293
 |level27_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -391,7 +391,7 @@
 |level27_stat5_value                     = 9
 |level28                                 = True
 |level28_level_requirement               = 86
-|level28_stat_text                       = 316 to 474 Base Off Hand Physical Damage<br>6 to 8 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+9 to radius
+|level28_stat_text                       = 316 to 474 Base Off Hand Physical Damage<br>6 to 8 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.9 metres to radius
 |level28_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level28_stat1_value                     = 316
 |level28_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -404,7 +404,7 @@
 |level28_stat5_value                     = 9
 |level29                                 = True
 |level29_level_requirement               = 88
-|level29_stat_text                       = 340 to 510 Base Off Hand Physical Damage<br>6 to 8 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+9 to radius
+|level29_stat_text                       = 340 to 510 Base Off Hand Physical Damage<br>6 to 8 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+0.9 metres to radius
 |level29_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level29_stat1_value                     = 340
 |level29_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -417,7 +417,7 @@
 |level29_stat5_value                     = 9
 |level30                                 = True
 |level30_level_requirement               = 90
-|level30_stat_text                       = 366 to 549 Base Off Hand Physical Damage<br>6 to 8 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+10 to radius
+|level30_stat_text                       = 366 to 549 Base Off Hand Physical Damage<br>6 to 8 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+1 metre to radius
 |level30_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level30_stat1_value                     = 366
 |level30_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -430,7 +430,7 @@
 |level30_stat5_value                     = 10
 |level31                                 = True
 |level31_level_requirement               = 91
-|level31_stat_text                       = 379 to 569 Base Off Hand Physical Damage<br>6 to 8 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+10 to radius
+|level31_stat_text                       = 379 to 569 Base Off Hand Physical Damage<br>6 to 8 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+1 metre to radius
 |level31_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level31_stat1_value                     = 379
 |level31_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -443,7 +443,7 @@
 |level31_stat5_value                     = 10
 |level32                                 = True
 |level32_level_requirement               = 92
-|level32_stat_text                       = 393 to 590 Base Off Hand Physical Damage<br>6 to 8 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+10 to radius
+|level32_stat_text                       = 393 to 590 Base Off Hand Physical Damage<br>6 to 8 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+1 metre to radius
 |level32_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level32_stat1_value                     = 393
 |level32_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -456,7 +456,7 @@
 |level32_stat5_value                     = 10
 |level33                                 = True
 |level33_level_requirement               = 93
-|level33_stat_text                       = 408 to 612 Base Off Hand Physical Damage<br>6 to 8 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+10 to radius
+|level33_stat_text                       = 408 to 612 Base Off Hand Physical Damage<br>6 to 8 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+1 metre to radius
 |level33_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level33_stat1_value                     = 408
 |level33_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -469,7 +469,7 @@
 |level33_stat5_value                     = 10
 |level34                                 = True
 |level34_level_requirement               = 94
-|level34_stat_text                       = 423 to 634 Base Off Hand Physical Damage<br>7 to 9 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+10 to radius
+|level34_stat_text                       = 423 to 634 Base Off Hand Physical Damage<br>7 to 9 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+1 metre to radius
 |level34_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level34_stat1_value                     = 423
 |level34_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -482,7 +482,7 @@
 |level34_stat5_value                     = 10
 |level35                                 = True
 |level35_level_requirement               = 95
-|level35_stat_text                       = 438 to 657 Base Off Hand Physical Damage<br>7 to 9 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+10 to radius
+|level35_stat_text                       = 438 to 657 Base Off Hand Physical Damage<br>7 to 9 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+1 metre to radius
 |level35_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level35_stat1_value                     = 438
 |level35_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -495,7 +495,7 @@
 |level35_stat5_value                     = 10
 |level36                                 = True
 |level36_level_requirement               = 96
-|level36_stat_text                       = 454 to 682 Base Off Hand Physical Damage<br>7 to 9 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+11 to radius
+|level36_stat_text                       = 454 to 682 Base Off Hand Physical Damage<br>7 to 9 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+1.1 metres to radius
 |level36_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level36_stat1_value                     = 454
 |level36_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -508,7 +508,7 @@
 |level36_stat5_value                     = 11
 |level37                                 = True
 |level37_level_requirement               = 97
-|level37_stat_text                       = 471 to 706 Base Off Hand Physical Damage<br>7 to 9 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+11 to radius
+|level37_stat_text                       = 471 to 706 Base Off Hand Physical Damage<br>7 to 9 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+1.1 metres to radius
 |level37_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level37_stat1_value                     = 471
 |level37_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -521,7 +521,7 @@
 |level37_stat5_value                     = 11
 |level38                                 = True
 |level38_level_requirement               = 98
-|level38_stat_text                       = 488 to 732 Base Off Hand Physical Damage<br>7 to 9 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+11 to radius
+|level38_stat_text                       = 488 to 732 Base Off Hand Physical Damage<br>7 to 9 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+1.1 metres to radius
 |level38_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level38_stat1_value                     = 488
 |level38_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -534,7 +534,7 @@
 |level38_stat5_value                     = 11
 |level39                                 = True
 |level39_level_requirement               = 99
-|level39_stat_text                       = 506 to 759 Base Off Hand Physical Damage<br>7 to 9 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+11 to radius
+|level39_stat_text                       = 506 to 759 Base Off Hand Physical Damage<br>7 to 9 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+1.1 metres to radius
 |level39_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level39_stat1_value                     = 506
 |level39_stat2_id                        = off_hand_local_maximum_added_physical_damage
@@ -547,7 +547,7 @@
 |level39_stat5_value                     = 11
 |level40                                 = True
 |level40_level_requirement               = 100
-|level40_stat_text                       = 524 to 786 Base Off Hand Physical Damage<br>7 to 9 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+11 to radius
+|level40_stat_text                       = 524 to 786 Base Off Hand Physical Damage<br>7 to 9 Added Physical Damage per 15 Armour or Evasion Rating on Shield<br>+1.1 metres to radius
 |level40_stat1_id                        = off_hand_local_minimum_added_physical_damage
 |level40_stat1_value                     = 524
 |level40_stat2_id                        = off_hand_local_maximum_added_physical_damage