--- Wiki
+++ Export
@@ -37,7 +37,7 @@
 |static_stat1_value                      = 1
 |static_stat2_id                         = support_spell_boost_area_of_effect_+%_final_per_charge
 |static_stat2_value                      = -15
-|stat_text                               = Supported Skills have (30-49)% increased Area of Effect<br>Supported Spells lose 1 Intensity every 0.25 seconds while moving, or immediately if you teleport<br>Supported Spells gain Intensity when you Cast them, to a maximum of 3<br>Supported Skills deal (10–14)% more Area Damage with Hits per Intensity <br> Supported Skills have 15% less Area of Effect per Intensity
+|stat_text                               = Supported Skills have (30-49)% increased Area of Effect<br>Supported Spells lose 1 Intensity every 0.25 seconds while moving, or immediately if you teleport
 |level1                                  = True
 |level1_level_requirement                = 31
 |level1_stat_text                        = Supported Skills have 30% increased Area of Effect
