--- Wiki
+++ Export
@@ -26,14 +26,14 @@
 |static_vaal_soul_gain_prevention_time   = 8
 |static_attack_speed_multiplier          = 100
 |static_duration                         = 6
-|static_stat1_id                         = total_projectile_spread_angle_override
-|static_stat1_value                      = 180
-|static_stat2_id                         = show_number_of_projectiles
+|static_stat1_id                         = cannot_cancel_skill_before_contact_point
+|static_stat1_value                      = 1
+|static_stat2_id                         = console_skill_dont_chase
 |static_stat2_value                      = 1
-|static_stat3_id                         = cannot_cancel_skill_before_contact_point
+|static_stat3_id                         = show_number_of_projectiles
 |static_stat3_value                      = 1
-|static_stat4_id                         = console_skill_dont_chase
-|static_stat4_value                      = 1
+|static_stat4_id                         = total_projectile_spread_angle_override
+|static_stat4_value                      = 180
 |static_stat5_id                         = skill_physical_damage_%_to_convert_to_lightning
 |static_stat5_value                      = 50
 |static_stat6_id                         = base_skill_effect_duration
