--- Wiki
+++ Export
@@ -33,12 +33,12 @@
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
 |static_duration                         = 1.4
-|static_stat1_id                         = skill_override_pvp_scaling_time_ms
-|static_stat1_value                      = 450
-|static_stat2_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat1_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat1_value                      = 1
+|static_stat2_id                         = is_area_damage
 |static_stat2_value                      = 1
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
+|static_stat3_id                         = skill_override_pvp_scaling_time_ms
+|static_stat3_value                      = 450
 |static_stat4_id                         = base_skill_effect_duration
 |static_stat4_value                      = 1400
 |static_stat5_id                         = fire_storm_fireball_delay_ms
