--- Splitting Steel
+++ Export
@@ -137,6 +137,8 @@
 |level7                                  = True
 |level7_level_requirement                = 20
 |level7_cost_amounts                     = 6
+|level7_damage_effectiveness             = 100
+|level7_damage_multiplier                = 100
 |level7_stat_text                        = 7 to 10 Added Attack Physical Damage<br>+1 to radius<br>Projectiles Split towards 3 targets
 |level7_stat1_id                         = attack_minimum_added_physical_damage
 |level7_stat1_value                      = 7