--- Wiki
+++ Export
@@ -40,8 +40,6 @@
 |level1                                  = True
 |level1_level_requirement                = 28
 |level1_cost_amounts                     = 8
-|level1_damage_effectiveness             = 100
-|level1_damage_multiplier                = 100
 |level2                                  = True
 |level2_level_requirement                = 31
 |level2_cost_amounts                     = 8
