--- Frost Shield
+++ Export
@@ -49,7 +49,7 @@
 |static_stat4_value                      = 4
 |static_stat5_id                         = frost_globe_stage_gain_interval_ms
 |static_stat5_value                      = 300
-|stat_text                               = Base duration is 10 seconds<br>20% of Damage dealt to you or Allies in Area with Hits by Enemies in Area is taken from Frost Shield's Life before your Life or Energy Shield<br>60% of Damage dealt to you or Allies in Area with Hits by Enemies outside Area is taken from Frost Shield's Life before your Life or Energy Shield<br>You gain +(0.1-0.38)% to Spell Critical Strike Chance per Stage while in Area<br>Frost Shield has (190-910) maximum Life per Stage<br>You lose (150-730) Energy Shield per second<br>Maximum 4 Stages<br>Gains 1 Stage every 0.3 seconds while you are losing Energy Shield
+|stat_text                               = Base duration is 10 seconds<br>20% of Damage dealt to you or Allies in Area with Hits by Enemies in Area is taken from Frost Shield's Life before your Life or Energy Shield<br>60% of Damage dealt to you or Allies in Area with Hits by Enemies outside Area is taken from Frost Shield's Life before your Life or Energy Shield<br>You gain +(0.1-0.38)% to Spell Critical Strike Chance per Stage while in Area<br>Frost Shield has (190-910) maximum Life per Stage<br>Maximum 4 Stages<br>Gains 1 Stage every 0.3 seconds while you are losing Energy Shield<br>You lose (150-730) Energy Shield per second
 |level1                                  = True
 |level1_level_requirement                = 34
 |level1_cost_amounts                     = 30
