--- Viper Strike
+++ Export
@@ -84,6 +84,7 @@
 |level3_stat2_value                      = 10
 |level4                                  = True
 |level4_level_requirement                = 7
+|level4_damage_effectiveness             = 100
 |level4_damage_multiplier                = 100.1
 |level4_stat_text                        = 8 to 12 Added Chaos Damage
 |level4_stat1_id                         = attack_minimum_added_chaos_damage