--- Impending Doom Support
+++ Export
@@ -7,9 +7,8 @@
 |drop_level                              = 31
 |tags                                    = support_gem, gem, default
 |metadata_id                             = Metadata/Items/Gems/SupportGemImpendingDoom
-|help_text                               = This is a Support Gem. It does not grant a bonus to your character, but to skills in sockets connected to it. Place into an item socket connected to a socket containing the Active Skill Gem you wish to augment. Right click to remove from a socket.
+|help_text                               = This is a Support Gem. It does not grant a bonus to your character, but to skills in sockets connected to it. Place into an item socket connected to a socket containing the Skill Gem you wish to augment. Right click to remove from a socket.
 |intelligence_percent                    = 100
-|static_cost_multiplier                  = 130
 |gem_tags                                = Support, Hex, Spell, AoE, Chaos, Trigger
 |skill_id                                = ViciousHexSupport
 |support_gem_letter                      = X
@@ -17,6 +16,10 @@
 |quality_type1_weight                    = 50
 |quality_type2_weight                    = 100
 |quality_type3_weight                    = 20
+|static_cost_multiplier                  = 130
+|static_damage_effectiveness             = 100
+|static_damage_multiplier                = 100
+|static_attack_speed_multiplier          = 100
 |level1                                  = True
 |level1_level_requirement                = 31
 |level2                                  = True
@@ -100,6 +103,9 @@
 |gem_description                         = Deals chaos damage in an area.
 |active_skill_name                       = Doom Blast
 |cast_time                               = 1.0
+|static_stored_uses                      = 3
+|static_cooldown                         = 0.15
+|static_critical_strike_chance           = 5
 |quality_type1_stat1_id                  = base_skill_area_of_effect_+%
 |quality_type1_stat1_value               = 500
 |quality_type1_stat_text                 = 0.5% increased Area of Effect
@@ -110,7 +116,7 @@
 |quality_type3_stat1_value               = 50
 |quality_type3_stat2_id                  = base_cooldown_speed_+%
 |quality_type3_stat2_value               = -2000
-|quality_type3_stat_text                 = +.05 Cooldown Use<br>2% reduced Cooldown Recovery Rate
+|quality_type3_stat_text                 = +0.05 Cooldown Use<br>2% reduced Cooldown Recovery Rate
 |stat_text                               = Trigger Doom Blast when a Hex from Supported Curse Skills ends<br>Deals (102-1654) to (154-2481) Chaos Damage<br>Trigger This Skill when a Hex from Supported Skills ends
 |static_stat1_id                         = trigger_vicious_hex_explosion_when_curse_ends
 |static_stat1_value                      = 1
