--- Wiki
+++ Export
@@ -34,10 +34,10 @@
 |quality_type3_stat2_value               = 50
 |static_cost_types                       = Mana
 |static_attack_speed_multiplier          = 110
-|static_stat1_id                         = spectral_throw_forget_hit_list_time_override
-|static_stat1_value                      = 225
-|static_stat2_id                         = base_is_projectile
-|static_stat2_value                      = 1
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = spectral_throw_forget_hit_list_time_override
+|static_stat2_value                      = 225
 |stat_text                               = 
 |level1                                  = True
 |level1_level_requirement                = 1
