--- Tattoo of the Arohongui Warrior
+++ Export
@@ -10,7 +10,6 @@
 |stack_size                              = 10
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
-|tattoo_target                           = Small Dexterity
-|tattoo_stat_text                        = Adds 3 to 5 [[Cold Damage]]
-|description                             = Replaces a Small Dexterity Passive Skill<br>Grants "Adds 3 to 5 [[Cold Damage]]"
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Dexterity}} {{c|default|Passive Skill}}
+|description                             = Grants "Adds 3 to 5 [[Cold Damage]]"
 }}