--- Tattoo of the Tasalio Bladedancer
+++ Export
@@ -11,8 +11,9 @@
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |tattoo_target                           = Small Strength
-|tattoo_stat_text                        = [[Attack|Attacks]] have 5% chance to Maim on [[Hit]]
-|description                             = Replaces a Small Strength Passive Skill<br>Grants "[[Attack|Attacks]] have 5% chance to Maim on [[Hit]]"
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Strength}} {{c|default|Passive Skill}}
+|tattoo_stat_text                        = Attacks have 5% chance to Maim on Hit
+|description                             = Grants "[[Attack|Attacks]] have 5% chance to Maim on [[Hit]]"
 |inventory_icon                          = Tattoo strength 1
 |drop_text                               = Can be obtained as a reward for defeating the [[Tasalio]] tribe in the [[Trial of the Ancestors]].
 |is_drop_restricted                      = true
