--- Veiled Chaos Orb
+++ Export
@@ -8,7 +8,7 @@
 |tags                                    = currency, default
 |metadata_id                             = Metadata/Items/Currency/CurrencyRerollRareVeiled
 |stack_size                              = 20
-|description                             = Reforges a rare item with new random modifiers, including a Veiled modifier. Can only be used on items that can have a Veiled modifier.
+|description                             = Reforges a rare item with new random modifiers, including a Veiled modifier<br>Can only be used on items that can have a Veiled modifier
 |help_text                               = Right click this item then left click a rare item to apply it.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |drop_text                               = Veiled Chaos Orbs can also be obtained from [[Aisling]]'s room in Research [[Immortal Syndicate|Betrayal]] safehouses.
