--- Storm Burst
+++ Export
@@ -36,13 +36,13 @@
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
 |static_duration                         = 1.2
-|static_stat1_id                         = shock_art_variation
-|static_stat1_value                      = 2
-|static_stat2_id                         = visual_hit_effect_elemental_is_holy
-|static_stat2_value                      = 1
-|static_stat3_id                         = is_area_damage
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = shock_art_variation
+|static_stat2_value                      = 2
+|static_stat3_id                         = skill_can_add_multiple_charges_per_action
 |static_stat3_value                      = 1
-|static_stat4_id                         = skill_can_add_multiple_charges_per_action
+|static_stat4_id                         = visual_hit_effect_elemental_is_holy
 |static_stat4_value                      = 1
 |static_stat5_id                         = skill_physical_damage_%_to_convert_to_lightning
 |static_stat5_value                      = 50
