--- Vaal Venom Gyre
+++ Export
@@ -26,17 +26,17 @@
 |static_vaal_soul_gain_prevention_time   = 8
 |static_attack_speed_multiplier          = 100
 |static_duration                         = 12
-|static_stat1_id                         = projectile_speed_variation_+%
-|static_stat1_value                      = 15
-|static_stat2_id                         = snapping_adder_maximum_projectiles_released
-|static_stat2_value                      = 30
-|static_stat3_id                         = base_is_projectile
-|static_stat3_value                      = 1
-|static_stat4_id                         = visual_hit_effect_chaos_is_green
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = cannot_cancel_skill_before_contact_point
+|static_stat2_value                      = 1
+|static_stat3_id                         = projectile_speed_variation_+%
+|static_stat3_value                      = 15
+|static_stat4_id                         = projectiles_are_not_fired
 |static_stat4_value                      = 1
-|static_stat5_id                         = projectiles_are_not_fired
-|static_stat5_value                      = 1
-|static_stat6_id                         = cannot_cancel_skill_before_contact_point
+|static_stat5_id                         = snapping_adder_maximum_projectiles_released
+|static_stat5_value                      = 30
+|static_stat6_id                         = visual_hit_effect_chaos_is_green
 |static_stat6_value                      = 1
 |static_stat7_id                         = base_skill_effect_duration
 |static_stat7_value                      = 12000
