--- Tattoo of the Valako Shieldbearer
+++ Export
@@ -11,10 +11,12 @@
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |tattoo_target                           = Small Strength
-|tattoo_stat_text                        = +1% Chance to [[Block]] [[Attack Damage]]
-|description                             = Replaces a Small Strength Passive Skill<br>Grants "+1% Chance to [[Block]] [[Attack Damage]]"
+|tattoo_tribe                            = 9
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Strength}} {{c|default|Passive Skill}}
+|tattoo_stat_text                        = +1% Chance to Block Attack Damage
+|description                             = Grants "+1% Chance to [[Block]] [[Attack Damage]]"
 |inventory_icon                          = Tattoo strength 1
-|drop_text                               =  Can be obtained as a reward for defeating the [[Valako]] tribe in the [[Trial of the Ancestors]].
+|drop_text                               = Can be obtained as a reward for defeating the [[Valako]] tribe in the [[Trial of the Ancestors]].
 |is_drop_restricted                      = true
 |release_version                         = 3.22.0
 }}