--- Tattoo of the Kitava Heart Eater
+++ Export
@@ -11,6 +11,7 @@
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |tattoo_target                           = Small Intelligence
-|tattoo_stat_text                        = Killing Blows have 4% chance to Consume corpses to recover 10% of Maximum [[Life]]
-|description                             = Replaces a Small Intelligence Passive Skill<br>Grants "Killing Blows have 4% chance to Consume corpses to recover 10% of Maximum [[Life]]"
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Intelligence}} {{c|default|Passive Skill}}
+|tattoo_stat_text                        = Killing Blows have 4% chance to Consume corpses to recover 10% of Maximum Life
+|description                             = Grants "Killing Blows have 4% chance to Consume corpses to recover 10% of Maximum [[Life]]"
 }}