--- Vaal Ice Shot
+++ Export
@@ -321,6 +321,7 @@
 |level31_stat2_value                     = 232
 |level32                                 = True
 |level32_level_requirement               = 92
+|level32_damage_effectiveness            = 100
 |level32_damage_multiplier               = 99.7
 |level32_stat_text                       = 160 to 240 Added Cold Damage
 |level32_stat1_id                        = attack_minimum_added_cold_damage
@@ -329,6 +330,7 @@
 |level32_stat2_value                     = 240
 |level33                                 = True
 |level33_level_requirement               = 93
+|level33_damage_effectiveness            = 100
 |level33_damage_multiplier               = 100.3
 |level33_stat_text                       = 165 to 248 Added Cold Damage
 |level33_stat1_id                        = attack_minimum_added_cold_damage