--- Wiki
+++ Export
@@ -187,7 +187,6 @@
 |quality_type3_stat1_id                  = dummy_stat_display_nothing
 |quality_type3_stat1_value               = 0
 |quality_type3_stat_text                 = 
-|static_cost_types                       = Mana
 |stat_text                               = Base duration is (8-9.9) seconds<br>Triggers a Supported Spell from Linked target's location when they Hit an Enemy<br>Link breaks if target leaves range or line of sight for 4 seconds<br>Maximum 1 Link from any source per target<br>You die if Linked target dies<br>Supported Spells deal (44-38)% less Damage<br>Supported Spells are Triggered by Intuitive Link
 |static_stat1_id                         = base_deal_no_damage
 |static_stat1_value                      = 1
