--- Wiki
+++ Export
@@ -7,5 +7,5 @@
 |tags                                    = not_for_sale, default
 |metadata_id                             = Metadata/Items/MicrotransactionItemEffects/MicrotransactionTheBlackStarRing
 |stack_size                              = 1
-|description                             = Change the appearance of your equipped ring to the Ring of the Black Star.
+|description                             = Change the appearance of your equipped ring to the Ring of the Black Star. Displays a vast amount of power when you gain levels.
 }}