--- Wiki
+++ Export
@@ -14,7 +14,6 @@
 |help_text                               = Shift click to unstack.
 |drop_monsters                           = Metadata/Monsters/AtlasBosses/ChimeraBoss
 |release_version                         = 3.11.0
-|card_background                         = 0,4
 }}
 
 '''A Familiar Call''' is a [[divination card]]. A set of three can be exchanged for a random Shaper and Hunter [[influenced]] [[magic]] [[jewellery]] item with the {{ml|GrantsCatAspectCrafted}}<!-- of Farrul --> suffix and [[item level]] 100.
