--- The Bear Woman
+++ Export
@@ -14,7 +14,6 @@
 |help_text                               = Shift click to unstack.
 |drop_monsters                           = Metadata/Monsters/AvariusCasticus/AvariusCasticusDivineMap, Metadata/Monsters/Beasts/MapBeastBossWhite, Metadata/Monsters/DespairBear/DespairBearMap
 |release_version                         = 3.13.0
-|card_background                         = 5
 }}
 
 '''The Bear Woman''' is a [[divination card]]. A set of six can be exchanged for a [[Warlord item|Warlord-influenced]] [[magic]] {{il|Ursine Pelt}} with the {{ml|IncreasedLife9}}<!-- Fecund --> prefix.
