--- Wiki
+++ Export
@@ -42,11 +42,11 @@
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
 |static_duration                         = 3.5
-|static_stat1_id                         = is_area_damage
+|static_stat1_id                         = base_skill_is_trapped
 |static_stat1_value                      = 1
-|static_stat2_id                         = base_skill_is_trapped
+|static_stat2_id                         = base_skill_show_average_damage_instead_of_dps
 |static_stat2_value                      = 1
-|static_stat3_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat3_id                         = is_area_damage
 |static_stat3_value                      = 1
 |static_stat4_id                         = base_trap_duration
 |static_stat4_value                      = 4000
