--- Tattoo of the Tukohama Makanga
+++ Export
@@ -11,9 +11,12 @@
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |tattoo_target                           = Small Strength
-|tattoo_stat_text                        = 16% increased [[Life]] Regeneration rate
-|description                             = Replaces a Small Strength Passive Skill<br>Grants "16% increased [[Life]] Regeneration rate"
+|tattoo_tribe                            = 2
 |implicit1_text                          = {{c|default|Requires}} {{c|value|7}} {{c|default|adjacent Passive Skills Allocated}}
+|tattoo_min_adjacent                     = 7
+|implicit2_text                          = {{c|default|Requires}} {{c|value|7}} {{c|default|adjacent Passive Skills Allocated}}
+|tattoo_stat_text                        = 16% increased Life Regeneration rate
+|description                             = Grants "16% increased [[Life]] Regeneration rate"
 |inventory_icon                          = Tattoo strength 2
 |drop_text                               = Can be obtained as a reward for defeating the [[Tukohama]] tribe in the [[Trial of the Ancestors]].
 |is_drop_restricted                      = true