--- The Trial
+++ Export
@@ -6,7 +6,6 @@
 |size_y                                  = 1
 |flavour_text                            = You cannot journey to new lands until you have the courage to leave the safety of home.
 |drop_level                              = 73
-|drop_level_maximum             = 77
 |tags                                    = maps_divination, corrupted_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardTheTrial
 |stack_size                              = 7
