--- Chronicle of Atzoatl
+++ Export
@@ -8,7 +8,7 @@
 |drop_level                              = 1
 |tags                                    = default
 |metadata_id                             = Metadata/Items/Incursion/ItemisedTemple
-|help_text                               = Right click this item to view the Temple of Atzoatl.
+|help_text                               = Right click this item to view the Temple of Atzoatl.
 Can be used in a personal Map Device to open portals to the Temple of Atzoatl in the present day.
 |frame_type                              = currency
 |drop_text                               = Acquired from Alva
