--- Arachnamagus Helmet
+++ Export
@@ -7,5 +7,5 @@
 |tags                                    = not_for_sale, default
 |metadata_id                             = Metadata/Items/MicrotransactionItemEffects/MicrotransactionArachnamagusHelmet
 |stack_size                              = 1
-|description                             = Changes the appearance of any equipped helmet to the Arachnamagus Helmet. Killing monsters has a chance for them to erupt with a swarm of spiders.
+|description                             = Changes the appearance of any equipped helmet to the Arachnamagus Helmet. A swarm of spiders has a chance to erupt from monsters you kill or chests you open.
 }}