--- The Celestial Stone
+++ Export
@@ -12,9 +12,9 @@
 |description                             = {{c|white|[[Opal Ring]]}}<br>{{c|default|Item Level:}} {{c|normal|100}}<br>{{c|default|Shaper Item}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
+|card_background                         = 0
 |drop_text                               = Can drop from [[Strongbox#Loot|Jeweller's Strongboxes]].
 |release_version                         = 3.3.0
-|card_background                         = 0
 }}
 
 '''The Celestial Stone''' is a [[divination card]]. A set of ten can be exchanged for a [[Shaper item|Shaper-influenced]] [[normal]] {{il|Opal Ring}} with [[item level]] 100.
