--- Anchor Drop Rare Finisher Effect
+++ Export
@@ -0,0 +1,11 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Anchor Drop Rare Finisher Effect
+|class_id                                = Microtransaction
+|size_x                                  = 2
+|size_y                                  = 2
+|tags                                    = not_for_sale, default
+|metadata_id                             = Metadata/Items/MicrotransactionSkillEffects/MicrotransactionAnchorDroponKillEffect
+|stack_size                              = 1
+|description                             = Adds the Anchor Drop Rare Finisher Effect to any weapon. It amplifies the deaths of Rare Monsters you kill.
+}}