--- Wiki
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = jewellery_divination, unique_divination, unique_jewellery_divination, unique_corrupted_divination, corrupted_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardBlindVenture
 |stack_size                              = 7
-|description                             = {{c|unique|Ring}}<br>{{c|corrupted|[[Corrupted]]}}
+|description                             = {{c|unique|{{il|html=|Ring}}}}<br>{{c|corrupted|[[Corrupted]]}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_areas                              = 1_5_7, 2_10_8, MapAtlasMuseum, MapWorldsRelicChambers, MapWorldsMuseum
