--- Sulphur Avatar Flask Effect
+++ Export
@@ -0,0 +1,11 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Sulphur Avatar Flask Effect
+|class_id                                = Microtransaction
+|size_x                                  = 1
+|size_y                                  = 2
+|tags                                    = not_for_sale, default
+|metadata_id                             = Metadata/Items/MicrotransactionItemEffects/MicrotransactionAvatarSulphurFlaskEffect
+|stack_size                              = 1
+|description                             = Its effect plays while the affected Sulphur Flask is being used.
+}}