--- Wiki
+++ Export
@@ -16,10 +16,10 @@
 |physical_damage_min                     = 25
 |physical_damage_max                     = 75
 |weapon_range                            = 120
+|release_version                         = 0.10.0
 |recipe1_part1_item_name                 = The Brawny Battle Mage
 |recipe1_part1_amount                    = 5
 |recipe1_description                     = ilvl 100 {{ml|LocalIncreasedPhysicalDamagePercent8}} {{c|magic|Tornado Wand}}
-|release_version                         = 0.10.0
 }}
 
 '''Tornado Wand''' is a [[wand]] base type.
