--- Wiki
+++ Export
@@ -8,13 +8,12 @@
 |tags                                    = default
 |metadata_id                             = Metadata/Items/Heist/QuestContracts/HeistContractQuestAdiyah1
 |inventory_icon                          = Generic NPC Quest Contract
+|release_version                         = 3.12.3
 |implicit1_text                          = {{c|default|Client:}} {{c|value|Adiyah}}
-|implicit2_text                          = {{c|default|Heist Target:}} {{c|value|Solerai's Spear}} 
+|implicit2_text                          = {{c|default|Heist Target:}} {{c|value|Solerai's Spear}}
 |implicit3_text                          = {{c|default|Area Level:}} {{c|value|(34-#)}}
 |implicit4_text                          = {{c|default|Requires}} {{c|value|Perception }} {{c|default|(Level}} {{c|value|2}}{{c|default|)}}
-|implicit5_text                          = {{c|default|Requires}} {{c|value|Adiyah, the Wayfinder}} 
-|flavour_text                            = When hope was but a memory, the twin sister Sekhemas Solerai and Lundara arose to unite the Maraketh.<br/>They wielded the essence of gold and silver. The Spear is one of the two weapons that forged us."
-|release_version                         = 3.12.3
+|implicit5_text                          = {{c|default|Requires}} {{c|value|Adiyah, the Wayfinder}}
 }}
 
 '''Contract: A Matter of Honour''' is a [[quest]] [[contract]]. It is the first step of [[Adiyah, the Wayfinder]]'s quest chain.
