--- Tattoo of the Ngamahu Woodcarver
+++ Export
@@ -10,7 +10,6 @@
 |stack_size                              = 10
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
-|tattoo_target                           = Small Strength
-|tattoo_stat_text                        = 6% increased [[Totem]] [[Life]]
-|description                             = Replaces a Small Strength Passive Skill<br>Grants "6% increased [[Totem]] [[Life]]"
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Strength}} {{c|default|Passive Skill}}
+|description                             = Grants "6% increased [[Totem]] [[Life]]"
 }}