--- Book of Skill (Victario's Secrets)
+++ Export
@@ -7,8 +7,8 @@
 |drop_level                              = 1
 |tags                                    = default
 |metadata_id                             = Metadata/Items/QuestItems/SkillBooks/Book-a3q11v0
-|description                             = Grants a Passive Skill Point and two Passive Respec Points
-|help_text                               = A Book of Skill. Right-click to gain a Passive Skill Point and two Passive Respec Points.
+|description                             = Grants a Passive Skill Point
+|help_text                               = A Book of Skill. Right-click to gain a Passive Skill Point.
 |inventory_icon                          = Book of Skill 2
 |release_version                         = 0.10.0
 |quest_reward1_type                      = quest