--- Blade Trap
+++ Export
@@ -30,18 +30,18 @@
 |quality_type3_stat1_value               = 2000
 |static_cost_types                       = Mana
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = trap_variation
-|static_stat1_value                      = 18
-|static_stat2_id                         = is_area_damage
+|static_stat1_id                         = attack_is_not_melee_override
+|static_stat1_value                      = 1
+|static_stat2_id                         = base_skill_is_trapped
 |static_stat2_value                      = 1
-|static_stat3_id                         = base_skill_is_trapped
+|static_stat3_id                         = base_skill_show_average_damage_instead_of_dps
 |static_stat3_value                      = 1
-|static_stat4_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat4_id                         = is_area_damage
 |static_stat4_value                      = 1
-|static_stat5_id                         = attack_is_not_melee_override
+|static_stat5_id                         = trap_use_alternate_death_animation
 |static_stat5_value                      = 1
-|static_stat6_id                         = trap_use_alternate_death_animation
-|static_stat6_value                      = 1
+|static_stat6_id                         = trap_variation
+|static_stat6_value                      = 18
 |static_stat7_id                         = base_trap_duration
 |static_stat7_value                      = 4000
 |static_stat8_id                         = base_weapon_trap_total_rotation_%
@@ -234,7 +234,6 @@
 |level24                                 = True
 |level24_level_requirement               = 78
 |level24_cost_amounts                    = 21
-|level24_damage_effectiveness            = 100
 |level24_damage_multiplier               = 100.2
 |level24_stat_text                       = +8 to radius
 |level24_stat1_id                        = active_skill_base_radius_+
