--- A Stone Perfected
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = jewellery_divination, unique_divination, corrupted_divination, unique_corrupted_divination, unique_jewellery_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardAStonePerfected
 |stack_size                              = 5
-|description                             = {{c|unique|Jewel}}<br>{{c|magic|Primordial}}<br>{{c|corrupted|[[1-2 Implicit]]}}<br>{{c|corrupted|[[Corrupted]]}}
+|description                             = {{c|unique|[[Jewel]]}}<br>{{c|magic|Primordial}}<br>{{c|corrupted|[[1-2 Implicit]]}}<br>{{c|corrupted|[[Corrupted]]}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_areas                              = MapWorldsGrotto, MapWorldsWharf, MapWorldsFloodedMine, MapWorldsWaterways, MapWorldsVault, MapWorldsCoves, MapWorldsPeninsula, MapWorldsGeode
