--- Gemini Claw
+++ Export
@@ -16,7 +16,7 @@
 |attack_speed                            = 1.5
 |physical_damage_min                     = 23
 |physical_damage_max                     = 68
-|weapon_range                            = 11
+|weapon_range                            = 1.1
 |release_version                         = 2.0.0
 |recipe1_part1_item_name                 = The Rabid Rhoa
 |recipe1_part1_amount                    = 4