--- Wiki
+++ Export
@@ -25,11 +25,11 @@
 |static_vaal_stored_uses                 = 3
 |static_vaal_soul_gain_prevention_time   = 2
 |static_attack_speed_multiplier          = 70
-|static_stat1_id                         = vaal_burning_arrow_explode_on_hit
+|static_stat1_id                         = cannot_cancel_skill_before_contact_point
 |static_stat1_value                      = 1
 |static_stat2_id                         = skill_can_fire_arrows
 |static_stat2_value                      = 1
-|static_stat3_id                         = cannot_cancel_skill_before_contact_point
+|static_stat3_id                         = vaal_burning_arrow_explode_on_hit
 |static_stat3_value                      = 1
 |static_stat4_id                         = base_chance_to_ignite_%
 |static_stat4_value                      = 50
