--- Wiki
+++ Export
@@ -234,9 +234,9 @@
 |static_stat1_value                      = 1
 |static_stat2_id                         = ring_of_ice_placement_distance
 |static_stat2_value                      = 28
-|static_stat3_id                         = is_area_damage
+|static_stat3_id                         = base_skill_show_average_damage_instead_of_dps
 |static_stat3_value                      = 1
-|static_stat4_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat4_id                         = is_area_damage
 |static_stat4_value                      = 1
 |static_stat5_id                         = skill_disabled_unless_cloned
 |static_stat5_value                      = 1
