--- Instilling Orb
+++ Export
@@ -8,7 +8,7 @@
 |tags                                    = currency, default
 |metadata_id                             = Metadata/Items/Currency/CurrencyInstillingOrb
 |stack_size                              = 10
-|description                             = Adds an enchantment to a utility flask that will cause it to be used when certain conditions are met. Replaces any existing enchantment.
+|description                             = Adds an enchantment to a utility flask that will cause<br>it to be used when certain conditions are met<br>Replaces any existing enchantment
 |help_text                               = Right click this item then left click a flask to apply it.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |release_version                         = 3.15.0
