--- Wiki
+++ Export
@@ -7,5 +7,5 @@
 |tags                                    = not_for_sale, default
 |metadata_id                             = Metadata/Items/MicrotransactionCharacterEffects/MicrotransactionIdentificationBookCharacterEffect
 |stack_size                              = 1
-|description                             = Attaches the Shavronne's Tome hip attachment to any equipped belt.
+|description                             = Attaches the Shavronne's Tome hip attachment to any equipped belt. Identifying items will cause exploding books to appear nearby.
 }}