--- Prophecy Wand
+++ Export
@@ -15,7 +15,7 @@
 |attack_speed                            = 1.2
 |physical_damage_min                     = 35
 |physical_damage_max                     = 64
-|weapon_range                            = 120
+|weapon_range                            = 12
 |recipe1_part1_item_name                 = The Archmage's Right Hand
 |recipe1_part1_amount                    = 7
 |recipe1_description                     = ilvl 100 {{ml|SpellDamageOnWeapon7}} {{c|magic|Prophecy Wand}}