--- Shockwave Support
+++ Export
@@ -8,7 +8,7 @@
 |drop_level                              = 18
 |tags                                    = support_gem, gem, default
 |metadata_id                             = Metadata/Items/Gems/SupportGemShockwave
-|help_text                               = This is a Support Gem. It does not grant a bonus to your character, but to skills in sockets connected to it. Place into an item socket connected to a socket containing the Active Skill Gem you wish to augment. Right click to remove from a socket.
+|help_text                               = This is a Support Gem. It does not grant a bonus to your character, but to skills in sockets connected to it. Place into an item socket connected to a socket containing the Skill Gem you wish to augment. Right click to remove from a socket.
 |strength_percent                        = 100
 |gem_tags                                = Support, Melee, Attack, AoE
 |skill_id                                = SupportBluntWeapon
@@ -17,6 +17,10 @@
 |quality_type1_weight                    = 50
 |quality_type2_weight                    = 100
 |quality_type3_weight                    = 100
+|static_cost_multiplier                  = 140
+|static_damage_effectiveness             = 100
+|static_damage_multiplier                = 100
+|static_attack_speed_multiplier          = 100
 |level1                                  = True
 |level1_level_requirement                = 18
 |level2                                  = True
@@ -101,6 +105,7 @@
 |active_skill_name                       = Shockwave
 |item_class_id_restriction               = One Hand Mace, Sceptre, Two Hand Mace, Staff, Warstaff
 |cast_time                               = 1.0
+|static_cooldown                         = 1
 |level1_stored_uses                      = 1
 |level1_damage_effectiveness             = 160
 |level1_damage_multiplier                = 160
@@ -223,13 +228,13 @@
 |level40_damage_multiplier               = 336.4
 |quality_type1_stat1_id                  = melee_damage_+%
 |quality_type1_stat1_value               = 500
-|quality_type1_stat_text                 = Supported Skills deal 0.5% increased Melee Damage
+|quality_type1_stat_text                 = Supported Skills deal 0.5% increased Melee Damage<br>0.5% increased Melee Damage
 |quality_type2_stat1_id                  = base_skill_area_of_effect_+%
 |quality_type2_stat1_value               = 500
-|quality_type2_stat_text                 = Supported Skills have 0.5% increased Area of Effect
-|quality_type3_stat1_id                  = dummy_stat_display_nothing
-|quality_type3_stat1_value               = 0
-|quality_type3_stat_text                 = 
+|quality_type2_stat_text                 = Supported Skills have 0.5% increased Area of Effect<br>0.5% increased Area of Effect
+|quality_type3_stat1_id                  = base_cooldown_speed_+%
+|quality_type3_stat1_value               = 500
+|quality_type3_stat_text                 = 0.5% increased Cooldown Recovery Rate
 |stat_text                               = Supported Skills will Trigger Shockwave on Melee Hit<br>Supported Skills can only be used with Maces, Sceptres or Staves<br>+(0-3) to radius<br>(0-38)% increased Cooldown Recovery Rate
 |static_stat1_id                         = support_blunt_chance_to_trigger_shockwave_on_hit_%
 |static_stat1_value                      = 100
