--- Vaal Axe
+++ Export
@@ -16,7 +16,7 @@
 |attack_speed                            = 1.15
 |physical_damage_min                     = 104
 |physical_damage_max                     = 174
-|weapon_range                            = 13
+|weapon_range                            = 1.3
 |recipe1_part1_item_name                 = The Undisputed
 |recipe1_part1_amount                    = 8
 |recipe1_description                     = ilvl 100 elder {{ml|LocalIncreasedPhysicalDamagePercent8}} {{c|magic|Vaal Axe}}