--- Predator Support
+++ Export
@@ -7,7 +7,7 @@
 |drop_level                              = 18
 |tags                                    = support_gem, gem, default
 |metadata_id                             = Metadata/Items/Gems/SupportGemDeathmark
-|help_text                               = This is a Support Gem. It does not grant a bonus to your character, but to skills in sockets connected to it. Place into an item socket connected to a socket containing the Active Skill Gem you wish to augment. Right click to remove from a socket.
+|help_text                               = This is a Support Gem. It does not grant a bonus to your character, but to skills in sockets connected to it. Place into an item socket connected to a socket containing the Skill Gem you wish to augment. Right click to remove from a socket.
 |intelligence_percent                    = 100
 |gem_tags                                = Support, Minion, Spell, Duration
 |skill_id                                = SupportMinionFocusFire
@@ -16,6 +16,10 @@
 |quality_type1_weight                    = 50
 |quality_type2_weight                    = 100
 |quality_type3_weight                    = 100
+|static_cost_multiplier                  = 130
+|static_damage_effectiveness             = 100
+|static_damage_multiplier                = 100
+|static_attack_speed_multiplier          = 100
 |level1                                  = True
 |level1_level_requirement                = 18
 |level2                                  = True
@@ -99,17 +103,20 @@
 |gem_description                         = Applies a debuff to a specific enemy which signals your minions to consider that enemy to be the Prey. All minions from skills supported by any Predator Support will target that enemy. Only one enemy can be the Prey at a time. This skill cannot be used by Totems, Traps, or Mines.
 |active_skill_name                       = Signal Prey
 |cast_time                               = 0.0
+|static_stored_uses                      = 1
+|static_cooldown                         = 0.8
+|static_duration                         = 8
 |quality_type1_stat1_id                  = minion_damage_+%
 |quality_type1_stat1_value               = 500
 |quality_type1_stat_text                 = Minions from Supported Skills deal 0.5% increased Damage
-|quality_type2_stat1_id                  = dummy_stat_display_nothing
-|quality_type2_stat1_value               = 0
-|quality_type2_stat_text                 = 
+|quality_type2_stat1_id                  = resist_all_%
+|quality_type2_stat1_value               = -200
+|quality_type2_stat_text                 = Debuff applies -0.2% to all Resistances
 |quality_type3_stat1_id                  = support_minion_focus_fire_critical_strike_chance_+%_vs_focused_target
 |quality_type3_stat1_value               = 1500
-|quality_type3_stat_text                 = Minions from Supported Skills have 1.5% increased Critical Strike Chance against the Prey<br>Minions from Supported Skills have +0.75% to Critical Strike Multiplier against the Prey
 |quality_type3_stat2_id                  = support_minion_focus_fire_critical_strike_multiplier_+_vs_focused_target
 |quality_type3_stat2_value               = 750
+|quality_type3_stat_text                 = Minions from Supported Skills have 1.5% increased Critical Strike Chance against the Prey<br>Minions from Supported Skills have +0.75% to Critical Strike Multiplier against the Prey
 |stat_text                               = Minions from Supported Skills deal (6-12)% more Damage<br>Minions from Supported Skills deal (15-24)% more Damage to the Prey with Hits and Ailments<br>Minions from Supported Skills always target the Prey if possible<br>Base duration is 8 seconds
 |static_stat1_id                         = support_minion_use_focussed_target
 |static_stat1_value                      = 1
