--- Rage Vortex
+++ Export
@@ -53,7 +53,7 @@
 |static_stat10_value                     = 250
 |static_stat11_id                        = rage_slash_maximum_vortices
 |static_stat11_value                     = 1
-|stat_text                               = Base duration is 3 seconds<br>Sacrifices 20% of Rage to create a Ragestorm, if you have at least 10 Rage<br>Ragestorm deals 5% more Damage per 1 Rage Sacrificed<br>Ragestorm has +1 to Radius per 2 Rage sacrificed<br>Ragestorm Hits 250% more Frequently<br>Maximum 1 Ragestorm
+|stat_text                               = Base duration is 3 seconds<br>Sacrifices 20% of Rage to create a Ragestorm, if you have at least 10 Rage<br>Ragestorm deals 5% more Damage per 1 Rage Sacrificed<br>Ragestorm has +0.1 metres to radius per 2 Rage sacrificed<br>Ragestorm Hits 250% more Frequently<br>Maximum 1 Ragestorm
 |level1                                  = True
 |level1_level_requirement                = 28
 |level1_cost_amounts                     = 25