--- Wiki
+++ Export
@@ -31,7 +31,7 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = support_debilitate_hit_damage_+%_final_per_poison_stack
 |static_stat1_value                      = 5
-|stat_text                               = Supported Skills deal 5% more Damage with Hits for each Poison on the Enemy, up to (5-8)<br>Supported Skills deal (10-19)% more Damage with Poison
+|stat_text                               = Supported Skills deal 5% more Damage with Hits for each Poison on the Enemy, up to 0<br>Supported Skills deal (10-19)% more Damage with Poison
 |level1                                  = True
 |level1_level_requirement                = 38
 |level1_stat_text                        = Supported Skills deal 10% more Damage with Poison
