--- Left to Fate
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = maps_divination, corrupted_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardLeftToFate
 |stack_size                              = 4
-|description                             = {{c|rare|Map}}<br>{{c|default|Map Tier:}} {{c|normal|16}}<br>{{c|corrupted|[[Unidentified Corrupted]]}}
+|description                             = {{c|rare|[[Map]]}}<br>{{c|default|Map Tier:}} {{c|normal|16}}<br>{{c|corrupted|[[Unidentified Corrupted]]}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_areas                              = MapWorldsForkingRiver, MapWorldsRamparts
