--- Wiki
+++ Export
@@ -47,7 +47,7 @@
 |static_stat4_value                      = 70
 |static_stat5_id                         = storm_blade_energy_shield_+%_final
 |static_stat5_value                      = -50
-|stat_text                               = Energy Blades have minimum Lightning Damage equal to 2% of Energy Shield, plus (2-12)<br>Energy Blades have maximum Lightning Damage equal to 40% of Energy Shield, plus (40-235)<br>Two Handed Energy Blades have 70% more Lightning Damage<br>Buff grants 50% less Maximum Energy Shield
+|stat_text                               = Energy Blades have minimum Lightning Damage equal to 0% of Energy Shield, plus (2-12)<br>Energy Blades have maximum Lightning Damage equal to 0% of Energy Shield, plus (40-235)<br>Energy Blades have minimum Lightning Damage equal to 2% of Energy Shield, plus 0<br>Energy Blades have maximum Lightning Damage equal to 40% of Energy Shield, plus 0<br>Two Handed Energy Blades have 70% more Lightning Damage<br>Buff grants 50% less Maximum Energy Shield
 |level1                                  = True
 |level1_level_requirement                = 24
 |level1_cost_amounts                     = 136
