--- Wiki
+++ Export
@@ -24,17 +24,17 @@
 |static_vaal_stored_uses                 = 2
 |static_vaal_soul_gain_prevention_time   = 9
 |static_attack_speed_multiplier          = 80
-|static_stat1_id                         = base_totem_range
-|static_stat1_value                      = 50
-|static_stat2_id                         = ancestor_totem_parent_activiation_range
-|static_stat2_value                      = 120
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
-|static_stat4_id                         = base_skill_is_totemified
+|static_stat1_id                         = ancestor_totem_parent_activiation_range
+|static_stat1_value                      = 120
+|static_stat2_id                         = base_skill_is_totemified
+|static_stat2_value                      = 1
+|static_stat3_id                         = base_totem_range
+|static_stat3_value                      = 50
+|static_stat4_id                         = cannot_cancel_skill_before_contact_point
 |static_stat4_value                      = 1
-|static_stat5_id                         = totem_ignores_vaal_skill_cost
+|static_stat5_id                         = is_area_damage
 |static_stat5_value                      = 1
-|static_stat6_id                         = cannot_cancel_skill_before_contact_point
+|static_stat6_id                         = totem_ignores_vaal_skill_cost
 |static_stat6_value                      = 1
 |static_stat7_id                         = number_of_additional_totems_allowed
 |static_stat7_value                      = 1
@@ -51,8 +51,6 @@
 |stat_text                               = +1 to maximum number of Summoned Totems<br>Totem lasts 6 seconds<br>Modifiers to Totem Duration also apply to this Skill's Soul Gain Prevention<br>Summons a Totem which uses this Skill<br>32% more Melee Damage while Totem is Active<br>50% increased Totem Placement speed
 |level1                                  = True
 |level1_level_requirement                = 28
-|level1_damage_effectiveness             = 100
-|level1_damage_multiplier                = 100
 |level2                                  = True
 |level2_level_requirement                = 31
 |level2_damage_effectiveness             = 103
