--- Decurve Bow
+++ Export
@@ -14,7 +14,7 @@
 |attack_speed                            = 1.25
 |physical_damage_min                     = 24
 |physical_damage_max                     = 96
-|weapon_range                            = 120
+|weapon_range                            = 12
 |quest_reward1_type                      = quest
 |quest_reward1_quest                     = The King's Feast
 |quest_reward1_quest_id                  = a5q5