--- Wiki
+++ Export
@@ -6,7 +6,6 @@
 |size_y                                  = 1
 |flavour_text                            = To her beauty you submit,<br>lest your neck the great Queen slit.
 |drop_level                              = 23
-|drop_level_maximum            = 72
 |tags                                    = divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardHerMask
 |stack_size                              = 4
