--- Lacerate
+++ Export
@@ -43,6 +43,8 @@
 |stat_text                               = (5-75) to (8-113) Added Attack Physical Damage<br>+(0-9) to radius<br>(50-88)% more Damage with Bleeding while in Blood Stance<br>25% chance to cause Bleeding while in Blood Stance<br>50% increased angle while in Sand Stance
 |level1                                  = True
 |level1_level_requirement                = 12
+|level1_damage_effectiveness             = 100
+|level1_damage_multiplier                = 100
 |level1_stat_text                        = 5 to 8 Added Attack Physical Damage<br>50% more Damage with Bleeding while in Blood Stance
 |level1_stat1_id                         = attack_minimum_added_physical_damage
 |level1_stat1_value                      = 5