--- The Mortinomicon Exitio Immortalis
+++ Export
@@ -0,0 +1,11 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = The Mortinomicon Exitio Immortalis
+|class_id                                = QuestItem
+|size_x                                  = 1
+|size_y                                  = 1
+|flavour_text                            = Hoc sciendo scribo ut verba mea perdant animam<br>meam eternam. Mihi credendum est quod novus<br>deus meus non sit tam divinus quam ipse petit...
+|drop_level                              = 68
+|tags                                    = not_for_sale, sanctum_key, default
+|metadata_id                             = Metadata/Items/QuestItems/Sanctum/SanctumKeyQuest
+}}