--- Animate Weapon
+++ Export
@@ -46,7 +46,7 @@
 |level1_level_requirement                = 4
 |level1_cost_amounts                     = 4
 |level1_stat_text                        = Can use Items requiring up to Level 9<br>Minions' Attacks deal 5 to 8 additional Physical Damage<br>Animated Lingering Blades deal 1 to 2 Added Attack Physical Damage<br>Maximum 5 Animated Weapons
-|level1_stat1_id                         = display_minion_monster_level
+|level1_stat1_id                         = base_display_minion_actor_level
 |level1_stat1_value                      = 4
 |level1_stat2_id                         = animate_item_maximum_level_requirement
 |level1_stat2_value                      = 9
@@ -64,7 +64,7 @@
 |level2_level_requirement                = 6
 |level2_cost_amounts                     = 5
 |level2_stat_text                        = Can use Items requiring up to Level 11<br>Minions' Attacks deal 6 to 11 additional Physical Damage<br>Animated Lingering Blades deal 1 to 2 Added Attack Physical Damage<br>Minions deal 2% more Damage<br>Maximum 5 Animated Weapons<br>Minions have 2% more Attack Speed
-|level2_stat1_id                         = display_minion_monster_level
+|level2_stat1_id                         = base_display_minion_actor_level
 |level2_stat1_value                      = 6
 |level2_stat2_id                         = animate_item_maximum_level_requirement
 |level2_stat2_value                      = 11
@@ -86,7 +86,7 @@
 |level3_level_requirement                = 9
 |level3_cost_amounts                     = 5
 |level3_stat_text                        = Can use Items requiring up to Level 14<br>Minions' Attacks deal 10 to 15 additional Physical Damage<br>Animated Lingering Blades deal 1 to 3 Added Attack Physical Damage<br>Minions deal 4% more Damage<br>Maximum 6 Animated Weapons<br>Minions have 4% more Attack Speed
-|level3_stat1_id                         = display_minion_monster_level
+|level3_stat1_id                         = base_display_minion_actor_level
 |level3_stat1_value                      = 9
 |level3_stat2_id                         = animate_item_maximum_level_requirement
 |level3_stat2_value                      = 14
@@ -108,7 +108,7 @@
 |level4_level_requirement                = 12
 |level4_cost_amounts                     = 6
 |level4_stat_text                        = Can use Items requiring up to Level 18<br>Minions' Attacks deal 13 to 18 additional Physical Damage<br>Animated Lingering Blades deal 1 to 4 Added Attack Physical Damage<br>Minions deal 6% more Damage<br>Maximum 6 Animated Weapons<br>Minions have 6% more Attack Speed
-|level4_stat1_id                         = display_minion_monster_level
+|level4_stat1_id                         = base_display_minion_actor_level
 |level4_stat1_value                      = 12
 |level4_stat2_id                         = animate_item_maximum_level_requirement
 |level4_stat2_value                      = 18
@@ -130,7 +130,7 @@
 |level5_level_requirement                = 16
 |level5_cost_amounts                     = 7
 |level5_stat_text                        = Can use Items requiring up to Level 22<br>Minions' Attacks deal 17 to 25 additional Physical Damage<br>Animated Lingering Blades deal 1 to 5 Added Attack Physical Damage<br>Minions deal 8% more Damage<br>Maximum 7 Animated Weapons<br>Minions have 8% more Attack Speed
-|level5_stat1_id                         = display_minion_monster_level
+|level5_stat1_id                         = base_display_minion_actor_level
 |level5_stat1_value                      = 16
 |level5_stat2_id                         = animate_item_maximum_level_requirement
 |level5_stat2_value                      = 22
@@ -152,7 +152,7 @@
 |level6_level_requirement                = 20
 |level6_cost_amounts                     = 7
 |level6_stat_text                        = Can use Items requiring up to Level 26<br>Minions' Attacks deal 21 to 32 additional Physical Damage<br>Animated Lingering Blades deal 1 to 7 Added Attack Physical Damage<br>Minions deal 10% more Damage<br>Maximum 7 Animated Weapons<br>Minions have 10% more Attack Speed
-|level6_stat1_id                         = display_minion_monster_level
+|level6_stat1_id                         = base_display_minion_actor_level
 |level6_stat1_value                      = 20
 |level6_stat2_id                         = animate_item_maximum_level_requirement
 |level6_stat2_value                      = 26
@@ -174,7 +174,7 @@
 |level7_level_requirement                = 24
 |level7_cost_amounts                     = 8
 |level7_stat_text                        = Can use Items requiring up to Level 31<br>Minions' Attacks deal 26 to 38 additional Physical Damage<br>Animated Lingering Blades deal 2 to 9 Added Attack Physical Damage<br>Minions deal 12% more Damage<br>Maximum 8 Animated Weapons<br>Minions have 12% more Attack Speed
-|level7_stat1_id                         = display_minion_monster_level
+|level7_stat1_id                         = base_display_minion_actor_level
 |level7_stat1_value                      = 24
 |level7_stat2_id                         = animate_item_maximum_level_requirement
 |level7_stat2_value                      = 31
@@ -196,7 +196,7 @@
 |level8_level_requirement                = 28
 |level8_cost_amounts                     = 8
 |level8_stat_text                        = Can use Items requiring up to Level 35<br>Minions' Attacks deal 32 to 47 additional Physical Damage<br>Animated Lingering Blades deal 2 to 11 Added Attack Physical Damage<br>Minions deal 14% more Damage<br>Maximum 8 Animated Weapons<br>Minions have 14% more Attack Speed
-|level8_stat1_id                         = display_minion_monster_level
+|level8_stat1_id                         = base_display_minion_actor_level
 |level8_stat1_value                      = 28
 |level8_stat2_id                         = animate_item_maximum_level_requirement
 |level8_stat2_value                      = 35
@@ -218,7 +218,7 @@
 |level9_level_requirement                = 32
 |level9_cost_amounts                     = 9
 |level9_stat_text                        = Can use Items requiring up to Level 40<br>Minions' Attacks deal 39 to 57 additional Physical Damage<br>Animated Lingering Blades deal 3 to 14 Added Attack Physical Damage<br>Minions deal 16% more Damage<br>Maximum 9 Animated Weapons<br>Minions have 16% more Attack Speed
-|level9_stat1_id                         = display_minion_monster_level
+|level9_stat1_id                         = base_display_minion_actor_level
 |level9_stat1_value                      = 32
 |level9_stat2_id                         = animate_item_maximum_level_requirement
 |level9_stat2_value                      = 40
@@ -240,7 +240,7 @@
 |level10_level_requirement               = 36
 |level10_cost_amounts                    = 10
 |level10_stat_text                       = Can use Items requiring up to Level 44<br>Minions' Attacks deal 48 to 72 additional Physical Damage<br>Animated Lingering Blades deal 4 to 17 Added Attack Physical Damage<br>Minions deal 18% more Damage<br>Maximum 9 Animated Weapons<br>Minions have 18% more Attack Speed
-|level10_stat1_id                        = display_minion_monster_level
+|level10_stat1_id                        = base_display_minion_actor_level
 |level10_stat1_value                     = 36
 |level10_stat2_id                        = animate_item_maximum_level_requirement
 |level10_stat2_value                     = 44
@@ -262,7 +262,7 @@
 |level11_level_requirement               = 40
 |level11_cost_amounts                    = 10
 |level11_stat_text                       = Can use Items requiring up to Level 49<br>Minions' Attacks deal 53 to 81 additional Physical Damage<br>Animated Lingering Blades deal 4 to 21 Added Attack Physical Damage<br>Minions deal 20% more Damage<br>Maximum 10 Animated Weapons<br>Minions have 20% more Attack Speed
-|level11_stat1_id                        = display_minion_monster_level
+|level11_stat1_id                        = base_display_minion_actor_level
 |level11_stat1_value                     = 40
 |level11_stat2_id                        = animate_item_maximum_level_requirement
 |level11_stat2_value                     = 49
@@ -284,7 +284,7 @@
 |level12_level_requirement               = 44
 |level12_cost_amounts                    = 11
 |level12_stat_text                       = Can use Items requiring up to Level 53<br>Minions' Attacks deal 61 to 91 additional Physical Damage<br>Animated Lingering Blades deal 5 to 25 Added Attack Physical Damage<br>Minions deal 22% more Damage<br>Maximum 10 Animated Weapons<br>Minions have 22% more Attack Speed
-|level12_stat1_id                        = display_minion_monster_level
+|level12_stat1_id                        = base_display_minion_actor_level
 |level12_stat1_value                     = 44
 |level12_stat2_id                        = animate_item_maximum_level_requirement
 |level12_stat2_value                     = 53
@@ -306,7 +306,7 @@
 |level13_level_requirement               = 48
 |level13_cost_amounts                    = 12
 |level13_stat_text                       = Can use Items requiring up to Level 58<br>Minions' Attacks deal 68 to 99 additional Physical Damage<br>Animated Lingering Blades deal 6 to 30 Added Attack Physical Damage<br>Minions deal 24% more Damage<br>Maximum 11 Animated Weapons<br>Minions have 24% more Attack Speed
-|level13_stat1_id                        = display_minion_monster_level
+|level13_stat1_id                        = base_display_minion_actor_level
 |level13_stat1_value                     = 48
 |level13_stat2_id                        = animate_item_maximum_level_requirement
 |level13_stat2_value                     = 58
@@ -328,7 +328,7 @@
 |level14_level_requirement               = 52
 |level14_cost_amounts                    = 12
 |level14_stat_text                       = Can use Items requiring up to Level 62<br>Minions' Attacks deal 73 to 112 additional Physical Damage<br>Animated Lingering Blades deal 8 to 36 Added Attack Physical Damage<br>Minions deal 26% more Damage<br>Maximum 11 Animated Weapons<br>Minions have 26% more Attack Speed
-|level14_stat1_id                        = display_minion_monster_level
+|level14_stat1_id                        = base_display_minion_actor_level
 |level14_stat1_value                     = 52
 |level14_stat2_id                        = animate_item_maximum_level_requirement
 |level14_stat2_value                     = 62
@@ -350,7 +350,7 @@
 |level15_level_requirement               = 55
 |level15_cost_amounts                    = 13
 |level15_stat_text                       = Can use Items requiring up to Level 66<br>Minions' Attacks deal 83 to 124 additional Physical Damage<br>Animated Lingering Blades deal 9 to 42 Added Attack Physical Damage<br>Minions deal 28% more Damage<br>Maximum 12 Animated Weapons<br>Minions have 28% more Attack Speed
-|level15_stat1_id                        = display_minion_monster_level
+|level15_stat1_id                        = base_display_minion_actor_level
 |level15_stat1_value                     = 55
 |level15_stat2_id                        = animate_item_maximum_level_requirement
 |level15_stat2_value                     = 66
@@ -372,7 +372,7 @@
 |level16_level_requirement               = 58
 |level16_cost_amounts                    = 13
 |level16_stat_text                       = Can use Items requiring up to Level 70<br>Minions' Attacks deal 89 to 136 additional Physical Damage<br>Animated Lingering Blades deal 10 to 48 Added Attack Physical Damage<br>Minions deal 30% more Damage<br>Maximum 12 Animated Weapons<br>Minions have 30% more Attack Speed
-|level16_stat1_id                        = display_minion_monster_level
+|level16_stat1_id                        = base_display_minion_actor_level
 |level16_stat1_value                     = 58
 |level16_stat2_id                        = animate_item_maximum_level_requirement
 |level16_stat2_value                     = 70
@@ -394,7 +394,7 @@
 |level17_level_requirement               = 61
 |level17_cost_amounts                    = 14
 |level17_stat_text                       = Can use Items requiring up to Level 74<br>Minions' Attacks deal 99 to 148 additional Physical Damage<br>Animated Lingering Blades deal 12 to 55 Added Attack Physical Damage<br>Minions deal 32% more Damage<br>Maximum 13 Animated Weapons<br>Minions have 32% more Attack Speed
-|level17_stat1_id                        = display_minion_monster_level
+|level17_stat1_id                        = base_display_minion_actor_level
 |level17_stat1_value                     = 61
 |level17_stat2_id                        = animate_item_maximum_level_requirement
 |level17_stat2_value                     = 74
@@ -416,7 +416,7 @@
 |level18_level_requirement               = 64
 |level18_cost_amounts                    = 15
 |level18_stat_text                       = Can use Items requiring up to Level 78<br>Minions' Attacks deal 108 to 159 additional Physical Damage<br>Animated Lingering Blades deal 13 to 62 Added Attack Physical Damage<br>Minions deal 34% more Damage<br>Maximum 13 Animated Weapons<br>Minions have 34% more Attack Speed
-|level18_stat1_id                        = display_minion_monster_level
+|level18_stat1_id                        = base_display_minion_actor_level
 |level18_stat1_value                     = 64
 |level18_stat2_id                        = animate_item_maximum_level_requirement
 |level18_stat2_value                     = 78
@@ -438,7 +438,7 @@
 |level19_level_requirement               = 67
 |level19_cost_amounts                    = 15
 |level19_stat_text                       = Can use Items requiring up to Level 82<br>Minions' Attacks deal 115 to 173 additional Physical Damage<br>Animated Lingering Blades deal 15 to 71 Added Attack Physical Damage<br>Minions deal 36% more Damage<br>Maximum 14 Animated Weapons<br>Minions have 36% more Attack Speed
-|level19_stat1_id                        = display_minion_monster_level
+|level19_stat1_id                        = base_display_minion_actor_level
 |level19_stat1_value                     = 67
 |level19_stat2_id                        = animate_item_maximum_level_requirement
 |level19_stat2_value                     = 82
@@ -460,7 +460,7 @@
 |level20_level_requirement               = 70
 |level20_cost_amounts                    = 16
 |level20_stat_text                       = Can use Items requiring up to Level 100<br>Minions' Attacks deal 126 to 184 additional Physical Damage<br>Animated Lingering Blades deal 17 to 80 Added Attack Physical Damage<br>Minions deal 38% more Damage<br>Maximum 14 Animated Weapons<br>Minions have 38% more Attack Speed
-|level20_stat1_id                        = display_minion_monster_level
+|level20_stat1_id                        = base_display_minion_actor_level
 |level20_stat1_value                     = 70
 |level20_stat2_id                        = animate_item_maximum_level_requirement
 |level20_stat2_value                     = 100
@@ -482,7 +482,7 @@
 |level21_level_requirement               = 72
 |level21_cost_amounts                    = 17
 |level21_stat_text                       = Can use Items requiring up to Level 100<br>Minions' Attacks deal 130 to 190 additional Physical Damage<br>Animated Lingering Blades deal 19 to 86 Added Attack Physical Damage<br>Minions deal 39% more Damage<br>Maximum 15 Animated Weapons<br>Minions have 39% more Attack Speed
-|level21_stat1_id                        = display_minion_monster_level
+|level21_stat1_id                        = base_display_minion_actor_level
 |level21_stat1_value                     = 72
 |level21_stat2_id                        = animate_item_maximum_level_requirement
 |level21_stat2_value                     = 100
@@ -504,7 +504,7 @@
 |level22_level_requirement               = 74
 |level22_cost_amounts                    = 17
 |level22_stat_text                       = Can use Items requiring up to Level 100<br>Minions' Attacks deal 134 to 196 additional Physical Damage<br>Animated Lingering Blades deal 20 to 92 Added Attack Physical Damage<br>Minions deal 40% more Damage<br>Maximum 15 Animated Weapons<br>Minions have 40% more Attack Speed
-|level22_stat1_id                        = display_minion_monster_level
+|level22_stat1_id                        = base_display_minion_actor_level
 |level22_stat1_value                     = 74
 |level22_stat2_id                        = animate_item_maximum_level_requirement
 |level22_stat2_value                     = 100
@@ -526,7 +526,7 @@
 |level23_level_requirement               = 76
 |level23_cost_amounts                    = 18
 |level23_stat_text                       = Can use Items requiring up to Level 100<br>Minions' Attacks deal 138 to 202 additional Physical Damage<br>Animated Lingering Blades deal 21 to 99 Added Attack Physical Damage<br>Minions deal 41% more Damage<br>Maximum 16 Animated Weapons<br>Minions have 41% more Attack Speed
-|level23_stat1_id                        = display_minion_monster_level
+|level23_stat1_id                        = base_display_minion_actor_level
 |level23_stat1_value                     = 76
 |level23_stat2_id                        = animate_item_maximum_level_requirement
 |level23_stat2_value                     = 100
@@ -548,7 +548,7 @@
 |level24_level_requirement               = 78
 |level24_cost_amounts                    = 19
 |level24_stat_text                       = Can use Items requiring up to Level 100<br>Minions' Attacks deal 142 to 208 additional Physical Damage<br>Animated Lingering Blades deal 22 to 106 Added Attack Physical Damage<br>Minions deal 42% more Damage<br>Maximum 16 Animated Weapons<br>Minions have 42% more Attack Speed
-|level24_stat1_id                        = display_minion_monster_level
+|level24_stat1_id                        = base_display_minion_actor_level
 |level24_stat1_value                     = 78
 |level24_stat2_id                        = animate_item_maximum_level_requirement
 |level24_stat2_value                     = 100
@@ -570,7 +570,7 @@
 |level25_level_requirement               = 80
 |level25_cost_amounts                    = 19
 |level25_stat_text                       = Can use Items requiring up to Level 100<br>Minions' Attacks deal 146 to 214 additional Physical Damage<br>Animated Lingering Blades deal 24 to 114 Added Attack Physical Damage<br>Minions deal 43% more Damage<br>Maximum 17 Animated Weapons<br>Minions have 43% more Attack Speed
-|level25_stat1_id                        = display_minion_monster_level
+|level25_stat1_id                        = base_display_minion_actor_level
 |level25_stat1_value                     = 80
 |level25_stat2_id                        = animate_item_maximum_level_requirement
 |level25_stat2_value                     = 100
@@ -592,7 +592,7 @@
 |level26_level_requirement               = 82
 |level26_cost_amounts                    = 20
 |level26_stat_text                       = Can use Items requiring up to Level 100<br>Minions' Attacks deal 150 to 220 additional Physical Damage<br>Animated Lingering Blades deal 26 to 122 Added Attack Physical Damage<br>Minions deal 44% more Damage<br>Maximum 17 Animated Weapons<br>Minions have 44% more Attack Speed
-|level26_stat1_id                        = display_minion_monster_level
+|level26_stat1_id                        = base_display_minion_actor_level
 |level26_stat1_value                     = 82
 |level26_stat2_id                        = animate_item_maximum_level_requirement
 |level26_stat2_value                     = 100
@@ -614,7 +614,7 @@
 |level27_level_requirement               = 84
 |level27_cost_amounts                    = 20
 |level27_stat_text                       = Can use Items requiring up to Level 100<br>Minions' Attacks deal 155 to 227 additional Physical Damage<br>Animated Lingering Blades deal 28 to 131 Added Attack Physical Damage<br>Minions deal 45% more Damage<br>Maximum 18 Animated Weapons<br>Minions have 45% more Attack Speed
-|level27_stat1_id                        = display_minion_monster_level
+|level27_stat1_id                        = base_display_minion_actor_level
 |level27_stat1_value                     = 84
 |level27_stat2_id                        = animate_item_maximum_level_requirement
 |level27_stat2_value                     = 100
@@ -636,7 +636,7 @@
 |level28_level_requirement               = 86
 |level28_cost_amounts                    = 21
 |level28_stat_text                       = Can use Items requiring up to Level 100<br>Minions' Attacks deal 160 to 234 additional Physical Damage<br>Animated Lingering Blades deal 29 to 140 Added Attack Physical Damage<br>Minions deal 46% more Damage<br>Maximum 18 Animated Weapons<br>Minions have 46% more Attack Speed
-|level28_stat1_id                        = display_minion_monster_level
+|level28_stat1_id                        = base_display_minion_actor_level
 |level28_stat1_value                     = 86
 |level28_stat2_id                        = animate_item_maximum_level_requirement
 |level28_stat2_value                     = 100
@@ -658,7 +658,7 @@
 |level29_level_requirement               = 88
 |level29_cost_amounts                    = 22
 |level29_stat_text                       = Can use Items requiring up to Level 100<br>Minions' Attacks deal 165 to 241 additional Physical Damage<br>Animated Lingering Blades deal 31 to 149 Added Attack Physical Damage<br>Minions deal 47% more Damage<br>Maximum 19 Animated Weapons<br>Minions have 47% more Attack Speed
-|level29_stat1_id                        = display_minion_monster_level
+|level29_stat1_id                        = base_display_minion_actor_level
 |level29_stat1_value                     = 88
 |level29_stat2_id                        = animate_item_maximum_level_requirement
 |level29_stat2_value                     = 100
@@ -680,7 +680,7 @@
 |level30_level_requirement               = 90
 |level30_cost_amounts                    = 22
 |level30_stat_text                       = Can use Items requiring up to Level 100<br>Minions' Attacks deal 170 to 248 additional Physical Damage<br>Animated Lingering Blades deal 33 to 159 Added Attack Physical Damage<br>Minions deal 48% more Damage<br>Maximum 19 Animated Weapons<br>Minions have 48% more Attack Speed
-|level30_stat1_id                        = display_minion_monster_level
+|level30_stat1_id                        = base_display_minion_actor_level
 |level30_stat1_value                     = 90
 |level30_stat2_id                        = animate_item_maximum_level_requirement
 |level30_stat2_value                     = 100
@@ -702,7 +702,7 @@
 |level31_level_requirement               = 91
 |level31_cost_amounts                    = 23
 |level31_stat_text                       = Can use Items requiring up to Level 100<br>Minions' Attacks deal 175 to 255 additional Physical Damage<br>Animated Lingering Blades deal 34 to 163 Added Attack Physical Damage<br>Minions deal 49% more Damage<br>Maximum 20 Animated Weapons<br>Minions have 49% more Attack Speed
-|level31_stat1_id                        = display_minion_monster_level
+|level31_stat1_id                        = base_display_minion_actor_level
 |level31_stat1_value                     = 91
 |level31_stat2_id                        = animate_item_maximum_level_requirement
 |level31_stat2_value                     = 100
@@ -724,7 +724,7 @@
 |level32_level_requirement               = 92
 |level32_cost_amounts                    = 23
 |level32_stat_text                       = Can use Items requiring up to Level 100<br>Minions' Attacks deal 180 to 263 additional Physical Damage<br>Animated Lingering Blades deal 34 to 168 Added Attack Physical Damage<br>Minions deal 50% more Damage<br>Maximum 20 Animated Weapons<br>Minions have 50% more Attack Speed
-|level32_stat1_id                        = display_minion_monster_level
+|level32_stat1_id                        = base_display_minion_actor_level
 |level32_stat1_value                     = 92
 |level32_stat2_id                        = animate_item_maximum_level_requirement
 |level32_stat2_value                     = 100
@@ -746,7 +746,7 @@
 |level33_level_requirement               = 93
 |level33_cost_amounts                    = 23
 |level33_stat_text                       = Can use Items requiring up to Level 100<br>Minions' Attacks deal 185 to 271 additional Physical Damage<br>Animated Lingering Blades deal 36 to 172 Added Attack Physical Damage<br>Minions deal 51% more Damage<br>Maximum 21 Animated Weapons<br>Minions have 51% more Attack Speed
-|level33_stat1_id                        = display_minion_monster_level
+|level33_stat1_id                        = base_display_minion_actor_level
 |level33_stat1_value                     = 93
 |level33_stat2_id                        = animate_item_maximum_level_requirement
 |level33_stat2_value                     = 100
@@ -768,7 +768,7 @@
 |level34_level_requirement               = 94
 |level34_cost_amounts                    = 24
 |level34_stat_text                       = Can use Items requiring up to Level 100<br>Minions' Attacks deal 191 to 279 additional Physical Damage<br>Animated Lingering Blades deal 36 to 176 Added Attack Physical Damage<br>Minions deal 52% more Damage<br>Maximum 21 Animated Weapons<br>Minions have 52% more Attack Speed
-|level34_stat1_id                        = display_minion_monster_level
+|level34_stat1_id                        = base_display_minion_actor_level
 |level34_stat1_value                     = 94
 |level34_stat2_id                        = animate_item_maximum_level_requirement
 |level34_stat2_value                     = 100
@@ -790,7 +790,7 @@
 |level35_level_requirement               = 95
 |level35_cost_amounts                    = 24
 |level35_stat_text                       = Can use Items requiring up to Level 100<br>Minions' Attacks deal 197 to 287 additional Physical Damage<br>Animated Lingering Blades deal 37 to 180 Added Attack Physical Damage<br>Minions deal 53% more Damage<br>Maximum 22 Animated Weapons<br>Minions have 53% more Attack Speed
-|level35_stat1_id                        = display_minion_monster_level
+|level35_stat1_id                        = base_display_minion_actor_level
 |level35_stat1_value                     = 95
 |level35_stat2_id                        = animate_item_maximum_level_requirement
 |level35_stat2_value                     = 100
@@ -812,7 +812,7 @@
 |level36_level_requirement               = 96
 |level36_cost_amounts                    = 24
 |level36_stat_text                       = Can use Items requiring up to Level 100<br>Minions' Attacks deal 203 to 296 additional Physical Damage<br>Animated Lingering Blades deal 38 to 185 Added Attack Physical Damage<br>Minions deal 54% more Damage<br>Maximum 22 Animated Weapons<br>Minions have 54% more Attack Speed
-|level36_stat1_id                        = display_minion_monster_level
+|level36_stat1_id                        = base_display_minion_actor_level
 |level36_stat1_value                     = 96
 |level36_stat2_id                        = animate_item_maximum_level_requirement
 |level36_stat2_value                     = 100
@@ -834,7 +834,7 @@
 |level37_level_requirement               = 97
 |level37_cost_amounts                    = 25
 |level37_stat_text                       = Can use Items requiring up to Level 100<br>Minions' Attacks deal 209 to 305 additional Physical Damage<br>Animated Lingering Blades deal 39 to 189 Added Attack Physical Damage<br>Minions deal 55% more Damage<br>Maximum 23 Animated Weapons<br>Minions have 55% more Attack Speed
-|level37_stat1_id                        = display_minion_monster_level
+|level37_stat1_id                        = base_display_minion_actor_level
 |level37_stat1_value                     = 97
 |level37_stat2_id                        = animate_item_maximum_level_requirement
 |level37_stat2_value                     = 100
@@ -856,7 +856,7 @@
 |level38_level_requirement               = 98
 |level38_cost_amounts                    = 25
 |level38_stat_text                       = Can use Items requiring up to Level 100<br>Minions' Attacks deal 215 to 314 additional Physical Damage<br>Animated Lingering Blades deal 39 to 194 Added Attack Physical Damage<br>Minions deal 56% more Damage<br>Maximum 23 Animated Weapons<br>Minions have 56% more Attack Speed
-|level38_stat1_id                        = display_minion_monster_level
+|level38_stat1_id                        = base_display_minion_actor_level
 |level38_stat1_value                     = 98
 |level38_stat2_id                        = animate_item_maximum_level_requirement
 |level38_stat2_value                     = 100
@@ -878,7 +878,7 @@
 |level39_level_requirement               = 99
 |level39_cost_amounts                    = 25
 |level39_stat_text                       = Can use Items requiring up to Level 100<br>Minions' Attacks deal 221 to 323 additional Physical Damage<br>Animated Lingering Blades deal 41 to 198 Added Attack Physical Damage<br>Minions deal 57% more Damage<br>Maximum 24 Animated Weapons<br>Minions have 57% more Attack Speed
-|level39_stat1_id                        = display_minion_monster_level
+|level39_stat1_id                        = base_display_minion_actor_level
 |level39_stat1_value                     = 99
 |level39_stat2_id                        = animate_item_maximum_level_requirement
 |level39_stat2_value                     = 100
@@ -900,7 +900,7 @@
 |level40_level_requirement               = 100
 |level40_cost_amounts                    = 25
 |level40_stat_text                       = Can use Items requiring up to Level 100<br>Minions' Attacks deal 228 to 333 additional Physical Damage<br>Animated Lingering Blades deal 41 to 203 Added Attack Physical Damage<br>Minions deal 58% more Damage<br>Maximum 24 Animated Weapons<br>Minions have 58% more Attack Speed
-|level40_stat1_id                        = display_minion_monster_level
+|level40_stat1_id                        = base_display_minion_actor_level
 |level40_stat1_value                     = 100
 |level40_stat2_id                        = animate_item_maximum_level_requirement
 |level40_stat2_value                     = 100
