--- Journey Tattoo of the Body
+++ Export
@@ -11,8 +11,10 @@
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |tattoo_target                           = +30 to Strength Notable
-|tattoo_stat_text                        = +25 to Maximum [[Life]] per Allocated Journey Tattoo of the Body
-|description                             = Replaces a +30 to Strength Notable Passive Skill<br>+25 to Maximum Life per Allocated Journey Tattoo of the Body
+|tattoo_tribe                            = 10
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|+30 to Strength Notable}} {{c|default|Passive Skill}}
+|tattoo_stat_text                        = +25 to maximum Life per Allocated Journey Tattoo of the Body
+|description                             = Grants "+25 to maximum [[Life]] per Allocated Journey Tattoo of the Body"
 |inventory_icon                          = Journey Tattoo (resource)
 |drop_text                               = Can be obtained as a reward from [[Hinekora]] for completing a tournament in the [[Trial of the Ancestors]].
 |is_drop_restricted                      = true