--- Astral Protection
+++ Export
@@ -9,12 +9,12 @@
 |tags                                    = armour_divination, shaper_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardAstralProtection
 |stack_size                              = 5
-|description                             = {{c|white|Shield}}<br>{{c|default|Shaper Item}}
+|description                             = {{c|white|[[Shield]]}}<br>{{c|default|Shaper Item}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
+|card_background                         = 0
 |drop_areas                              = MapWorldsMoonTemple
 |release_version                         = 3.19.0
-|card_background                         = 0
 }}
 
 '''Astral Protection''' is a [[divination card]]. A set of five can be exchanged for a random [[Shaper item|Shaper-influenced]] [[normal]] [[shield]].
