--- Tattoo of the Ramako Shaman
+++ Export
@@ -11,6 +11,7 @@
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |tattoo_target                           = Small Dexterity
-|tattoo_stat_text                        = +2% chance to Suppress [[Spell Damage]]
-|description                             = Replaces a Small Dexterity Passive Skill<br>Grants "+2% chance to Suppress [[Spell Damage]]"
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Dexterity}} {{c|default|Passive Skill}}
+|tattoo_stat_text                        = +2% chance to Suppress Spell Damage
+|description                             = Grants "+2% chance to Suppress [[Spell Damage]]"
 }}