--- Wiki
+++ Export
@@ -35,7 +35,7 @@
 |static_stat1_value                      = 3
 |static_stat2_id                         = support_ruthless_big_hit_stun_base_duration_override_ms
 |static_stat2_value                      = 800
-|stat_text                               = Every third Attack with Supported Melee Attacks deals a Ruthless Blow<br>Ruthless Blows with Supported Skills deal (70-98)% more Melee Damage<br>Ruthless Blows with Supported Skills deal (70-98)% more Damage with Ailments caused by Melee Hits<br>Ruthless Blows with Supported Skills have a base Stun Duration of 0.8 seconds
+|stat_text                               = Every third Attack with Supported Melee Attacks deals a Ruthless Blow<br>Ruthless Blows with Supported Skills deal (70-98)% more Melee Damage<br>Ruthless Blows with Supported Skills deal (70-98)% more Damage with Bleeding caused by Melee Hits<br>Ruthless Blows with Supported Skills have a base Stun Duration of 0.8 seconds
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_stat_text                        = Ruthless Blows with Supported Skills deal 70% more Melee Damage<br>Ruthless Blows with Supported Skills deal 70% more Damage with Bleeding caused by Melee Hits
