--- Duality
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = armour_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardDuality
 |stack_size                              = 2
-|description                             = {{c|white|Body Armour}}<br>{{c|default|Item Level:}} {{c|normal|100}}<br>{{c|default|Double-Influenced Item}}
+|description                             = {{c|white|[[Body Armour]]}}<br>{{c|default|Item Level:}} {{c|normal|100}}<br>{{c|default|Double-Influenced Item}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_text                               = Drops from Architects in the [[Temple of Atzoatl]] (completed temple, not in Incursions).
