--- Vaal Ice Shot
+++ Export
@@ -25,9 +25,9 @@
 |static_vaal_soul_gain_prevention_time   = 8
 |static_attack_speed_multiplier          = 100
 |static_duration                         = 6
-|static_stat1_id                         = skill_can_fire_arrows
+|static_stat1_id                         = cannot_cancel_skill_before_contact_point
 |static_stat1_value                      = 1
-|static_stat2_id                         = cannot_cancel_skill_before_contact_point
+|static_stat2_id                         = skill_can_fire_arrows
 |static_stat2_value                      = 1
 |static_stat3_id                         = base_skill_effect_duration
 |static_stat3_value                      = 6000
@@ -321,7 +321,6 @@
 |level31_stat2_value                     = 232
 |level32                                 = True
 |level32_level_requirement               = 92
-|level32_damage_effectiveness            = 100
 |level32_damage_multiplier               = 99.7
 |level32_stat_text                       = 160 to 240 Added Cold Damage
 |level32_stat1_id                        = attack_minimum_added_cold_damage
@@ -330,7 +329,6 @@
 |level32_stat2_value                     = 240
 |level33                                 = True
 |level33_level_requirement               = 93
-|level33_damage_effectiveness            = 100
 |level33_damage_multiplier               = 100.3
 |level33_stat_text                       = 165 to 248 Added Cold Damage
 |level33_stat1_id                        = attack_minimum_added_cold_damage
