--- The Mortinomicon Exitio Immortalis
+++ Export
@@ -2,13 +2,12 @@
 |rarity_id                               = normal
 |name                                    = The Mortinomicon Exitio Immortalis
 |class_id                                = QuestItem
-|size_x                                  = 2
-|size_y                                  = 2
+|size_x                                  = 1
+|size_y                                  = 1
 |flavour_text                            = Hoc sciendo scribo ut verba mea perdant animam<br>meam eternam. Mihi credendum est quod novus<br>deus meus non sit tam divinus quam ipse petit...
 |drop_level                              = 68
-|tags                                    = not_for_sale
+|tags                                    = not_for_sale, sanctum_key, default
 |metadata_id                             = Metadata/Items/QuestItems/Sanctum/SanctumKeyQuest
-|help_text = Take this forbidden tome to Divinia.
 }}
 
 This item can be used as a {{il|Forbidden Tome}}.
