--- Wiki
+++ Export
@@ -38,10 +38,10 @@
 |static_stat2_value                      = 1
 |static_stat3_id                         = cast_on_damage_taken_%
 |static_stat3_value                      = 100
-|stat_text                               = This Gem can only Support Skill Gems requiring Level (38-70) or lower<br>Supported Skills deal (65-27)% less Damage<br>You cannot Cast Supported Triggerable Spells directly<br>Trigger Supported Spells when you take a total of (528–3272) Damage
+|stat_text                               = This Gem can only Support Skill Gems requiring Level (38-70) or lower<br>Supported Skills deal (65-27)% less Damage<br>You cannot Cast Supported Triggerable Spells directly
 |level1                                  = True
 |level1_level_requirement                = 38
-|level1_stat_text                        = This Gem can only Support Skill Gems requiring Level 38 or lower<br>Supported Skills deal 65% less Damage<br>Trigger Supported Spells when you take a total of 528 Damage
+|level1_stat_text                        = This Gem can only Support Skill Gems requiring Level 38 or lower<br>Supported Skills deal 65% less Damage
 |level1_stat1_id                         = local_support_gem_max_skill_level_requirement_to_support
 |level1_stat1_value                      = 38
 |level1_stat2_id                         = cast_on_damage_taken_damage_+%_final
@@ -50,7 +50,7 @@
 |level1_stat3_value                      = 528
 |level2                                  = True
 |level2_level_requirement                = 40
-|level2_stat_text                        = This Gem can only Support Skill Gems requiring Level 40 or lower<br>Supported Skills deal 63% less Damage<br>Trigger Supported Spells when you take a total of 583 Damage
+|level2_stat_text                        = This Gem can only Support Skill Gems requiring Level 40 or lower<br>Supported Skills deal 63% less Damage
 |level2_stat1_id                         = local_support_gem_max_skill_level_requirement_to_support
 |level2_stat1_value                      = 40
 |level2_stat2_id                         = cast_on_damage_taken_damage_+%_final
@@ -59,7 +59,7 @@
 |level2_stat3_value                      = 583
 |level3                                  = True
 |level3_level_requirement                = 42
-|level3_stat_text                        = This Gem can only Support Skill Gems requiring Level 42 or lower<br>Supported Skills deal 61% less Damage<br>Trigger Supported Spells when you take a total of 661 Damage
+|level3_stat_text                        = This Gem can only Support Skill Gems requiring Level 42 or lower<br>Supported Skills deal 61% less Damage
 |level3_stat1_id                         = local_support_gem_max_skill_level_requirement_to_support
 |level3_stat1_value                      = 42
 |level3_stat2_id                         = cast_on_damage_taken_damage_+%_final
@@ -68,7 +68,7 @@
 |level3_stat3_value                      = 661
 |level4                                  = True
 |level4_level_requirement                = 44
-|level4_stat_text                        = This Gem can only Support Skill Gems requiring Level 44 or lower<br>Supported Skills deal 59% less Damage<br>Trigger Supported Spells when you take a total of 725 Damage
+|level4_stat_text                        = This Gem can only Support Skill Gems requiring Level 44 or lower<br>Supported Skills deal 59% less Damage
 |level4_stat1_id                         = local_support_gem_max_skill_level_requirement_to_support
 |level4_stat1_value                      = 44
 |level4_stat2_id                         = cast_on_damage_taken_damage_+%_final
@@ -77,7 +77,7 @@
 |level4_stat3_value                      = 725
 |level5                                  = True
 |level5_level_requirement                = 46
-|level5_stat_text                        = This Gem can only Support Skill Gems requiring Level 46 or lower<br>Supported Skills deal 57% less Damage<br>Trigger Supported Spells when you take a total of 812 Damage
+|level5_stat_text                        = This Gem can only Support Skill Gems requiring Level 46 or lower<br>Supported Skills deal 57% less Damage
 |level5_stat1_id                         = local_support_gem_max_skill_level_requirement_to_support
 |level5_stat1_value                      = 46
 |level5_stat2_id                         = cast_on_damage_taken_damage_+%_final
@@ -86,7 +86,7 @@
 |level5_stat3_value                      = 812
 |level6                                  = True
 |level6_level_requirement                = 48
-|level6_stat_text                        = This Gem can only Support Skill Gems requiring Level 48 or lower<br>Supported Skills deal 55% less Damage<br>Trigger Supported Spells when you take a total of 897 Damage
+|level6_stat_text                        = This Gem can only Support Skill Gems requiring Level 48 or lower<br>Supported Skills deal 55% less Damage
 |level6_stat1_id                         = local_support_gem_max_skill_level_requirement_to_support
 |level6_stat1_value                      = 48
 |level6_stat2_id                         = cast_on_damage_taken_damage_+%_final
@@ -95,7 +95,7 @@
 |level6_stat3_value                      = 897
 |level7                                  = True
 |level7_level_requirement                = 50
-|level7_stat_text                        = This Gem can only Support Skill Gems requiring Level 50 or lower<br>Supported Skills deal 53% less Damage<br>Trigger Supported Spells when you take a total of 1003 Damage
+|level7_stat_text                        = This Gem can only Support Skill Gems requiring Level 50 or lower<br>Supported Skills deal 53% less Damage
 |level7_stat1_id                         = local_support_gem_max_skill_level_requirement_to_support
 |level7_stat1_value                      = 50
 |level7_stat2_id                         = cast_on_damage_taken_damage_+%_final
@@ -104,7 +104,7 @@
 |level7_stat3_value                      = 1003
 |level8                                  = True
 |level8_level_requirement                = 52
-|level8_stat_text                        = This Gem can only Support Skill Gems requiring Level 52 or lower<br>Supported Skills deal 51% less Damage<br>Trigger Supported Spells when you take a total of 1107 Damage
+|level8_stat_text                        = This Gem can only Support Skill Gems requiring Level 52 or lower<br>Supported Skills deal 51% less Damage
 |level8_stat1_id                         = local_support_gem_max_skill_level_requirement_to_support
 |level8_stat1_value                      = 52
 |level8_stat2_id                         = cast_on_damage_taken_damage_+%_final
@@ -113,7 +113,7 @@
 |level8_stat3_value                      = 1107
 |level9                                  = True
 |level9_level_requirement                = 54
-|level9_stat_text                        = This Gem can only Support Skill Gems requiring Level 54 or lower<br>Supported Skills deal 49% less Damage<br>Trigger Supported Spells when you take a total of 1221 Damage
+|level9_stat_text                        = This Gem can only Support Skill Gems requiring Level 54 or lower<br>Supported Skills deal 49% less Damage
 |level9_stat1_id                         = local_support_gem_max_skill_level_requirement_to_support
 |level9_stat1_value                      = 54
 |level9_stat2_id                         = cast_on_damage_taken_damage_+%_final
@@ -122,7 +122,7 @@
 |level9_stat3_value                      = 1221
 |level10                                 = True
 |level10_level_requirement               = 56
-|level10_stat_text                       = This Gem can only Support Skill Gems requiring Level 56 or lower<br>Supported Skills deal 47% less Damage<br>Trigger Supported Spells when you take a total of 1354 Damage
+|level10_stat_text                       = This Gem can only Support Skill Gems requiring Level 56 or lower<br>Supported Skills deal 47% less Damage
 |level10_stat1_id                        = local_support_gem_max_skill_level_requirement_to_support
 |level10_stat1_value                     = 56
 |level10_stat2_id                        = cast_on_damage_taken_damage_+%_final
@@ -131,7 +131,7 @@
 |level10_stat3_value                     = 1354
 |level11                                 = True
 |level11_level_requirement               = 58
-|level11_stat_text                       = This Gem can only Support Skill Gems requiring Level 58 or lower<br>Supported Skills deal 45% less Damage<br>Trigger Supported Spells when you take a total of 1485 Damage
+|level11_stat_text                       = This Gem can only Support Skill Gems requiring Level 58 or lower<br>Supported Skills deal 45% less Damage
 |level11_stat1_id                        = local_support_gem_max_skill_level_requirement_to_support
 |level11_stat1_value                     = 58
 |level11_stat2_id                        = cast_on_damage_taken_damage_+%_final
@@ -140,7 +140,7 @@
 |level11_stat3_value                     = 1485
 |level12                                 = True
 |level12_level_requirement               = 60
-|level12_stat_text                       = This Gem can only Support Skill Gems requiring Level 60 or lower<br>Supported Skills deal 43% less Damage<br>Trigger Supported Spells when you take a total of 1635 Damage
+|level12_stat_text                       = This Gem can only Support Skill Gems requiring Level 60 or lower<br>Supported Skills deal 43% less Damage
 |level12_stat1_id                        = local_support_gem_max_skill_level_requirement_to_support
 |level12_stat1_value                     = 60
 |level12_stat2_id                        = cast_on_damage_taken_damage_+%_final
@@ -149,7 +149,7 @@
 |level12_stat3_value                     = 1635
 |level13                                 = True
 |level13_level_requirement               = 62
-|level13_stat_text                       = This Gem can only Support Skill Gems requiring Level 62 or lower<br>Supported Skills deal 41% less Damage<br>Trigger Supported Spells when you take a total of 1804 Damage
+|level13_stat_text                       = This Gem can only Support Skill Gems requiring Level 62 or lower<br>Supported Skills deal 41% less Damage
 |level13_stat1_id                        = local_support_gem_max_skill_level_requirement_to_support
 |level13_stat1_value                     = 62
 |level13_stat2_id                        = cast_on_damage_taken_damage_+%_final
@@ -158,7 +158,7 @@
 |level13_stat3_value                     = 1804
 |level14                                 = True
 |level14_level_requirement               = 64
-|level14_stat_text                       = This Gem can only Support Skill Gems requiring Level 64 or lower<br>Supported Skills deal 39% less Damage<br>Trigger Supported Spells when you take a total of 1980 Damage
+|level14_stat_text                       = This Gem can only Support Skill Gems requiring Level 64 or lower<br>Supported Skills deal 39% less Damage
 |level14_stat1_id                        = local_support_gem_max_skill_level_requirement_to_support
 |level14_stat1_value                     = 64
 |level14_stat2_id                        = cast_on_damage_taken_damage_+%_final
@@ -167,7 +167,7 @@
 |level14_stat3_value                     = 1980
 |level15                                 = True
 |level15_level_requirement               = 65
-|level15_stat_text                       = This Gem can only Support Skill Gems requiring Level 65 or lower<br>Supported Skills deal 37% less Damage<br>Trigger Supported Spells when you take a total of 2184 Damage
+|level15_stat_text                       = This Gem can only Support Skill Gems requiring Level 65 or lower<br>Supported Skills deal 37% less Damage
 |level15_stat1_id                        = local_support_gem_max_skill_level_requirement_to_support
 |level15_stat1_value                     = 65
 |level15_stat2_id                        = cast_on_damage_taken_damage_+%_final
@@ -176,7 +176,7 @@
 |level15_stat3_value                     = 2184
 |level16                                 = True
 |level16_level_requirement               = 66
-|level16_stat_text                       = This Gem can only Support Skill Gems requiring Level 66 or lower<br>Supported Skills deal 35% less Damage<br>Trigger Supported Spells when you take a total of 2394 Damage
+|level16_stat_text                       = This Gem can only Support Skill Gems requiring Level 66 or lower<br>Supported Skills deal 35% less Damage
 |level16_stat1_id                        = local_support_gem_max_skill_level_requirement_to_support
 |level16_stat1_value                     = 66
 |level16_stat2_id                        = cast_on_damage_taken_damage_+%_final
@@ -185,7 +185,7 @@
 |level16_stat3_value                     = 2394
 |level17                                 = True
 |level17_level_requirement               = 67
-|level17_stat_text                       = This Gem can only Support Skill Gems requiring Level 67 or lower<br>Supported Skills deal 33% less Damage<br>Trigger Supported Spells when you take a total of 2621 Damage
+|level17_stat_text                       = This Gem can only Support Skill Gems requiring Level 67 or lower<br>Supported Skills deal 33% less Damage
 |level17_stat1_id                        = local_support_gem_max_skill_level_requirement_to_support
 |level17_stat1_value                     = 67
 |level17_stat2_id                        = cast_on_damage_taken_damage_+%_final
@@ -194,7 +194,7 @@
 |level17_stat3_value                     = 2621
 |level18                                 = True
 |level18_level_requirement               = 68
-|level18_stat_text                       = This Gem can only Support Skill Gems requiring Level 68 or lower<br>Supported Skills deal 31% less Damage<br>Trigger Supported Spells when you take a total of 2874 Damage
+|level18_stat_text                       = This Gem can only Support Skill Gems requiring Level 68 or lower<br>Supported Skills deal 31% less Damage
 |level18_stat1_id                        = local_support_gem_max_skill_level_requirement_to_support
 |level18_stat1_value                     = 68
 |level18_stat2_id                        = cast_on_damage_taken_damage_+%_final
@@ -203,7 +203,7 @@
 |level18_stat3_value                     = 2874
 |level19                                 = True
 |level19_level_requirement               = 69
-|level19_stat_text                       = This Gem can only Support Skill Gems requiring Level 69 or lower<br>Supported Skills deal 29% less Damage<br>Trigger Supported Spells when you take a total of 3142 Damage
+|level19_stat_text                       = This Gem can only Support Skill Gems requiring Level 69 or lower<br>Supported Skills deal 29% less Damage
 |level19_stat1_id                        = local_support_gem_max_skill_level_requirement_to_support
 |level19_stat1_value                     = 69
 |level19_stat2_id                        = cast_on_damage_taken_damage_+%_final
@@ -212,7 +212,7 @@
 |level19_stat3_value                     = 3142
 |level20                                 = True
 |level20_level_requirement               = 70
-|level20_stat_text                       = This Gem can only Support Skill Gems requiring Level 70 or lower<br>Supported Skills deal 27% less Damage<br>Trigger Supported Spells when you take a total of 3272 Damage
+|level20_stat_text                       = This Gem can only Support Skill Gems requiring Level 70 or lower<br>Supported Skills deal 27% less Damage
 |level20_stat1_id                        = local_support_gem_max_skill_level_requirement_to_support
 |level20_stat1_value                     = 70
 |level20_stat2_id                        = cast_on_damage_taken_damage_+%_final
@@ -221,7 +221,7 @@
 |level20_stat3_value                     = 3272
 |level21                                 = True
 |level21_level_requirement               = 72
-|level21_stat_text                       = This Gem can only Support Skill Gems requiring Level 72 or lower<br>Supported Skills deal 25% less Damage<br>Trigger Supported Spells when you take a total of 3580 Damage
+|level21_stat_text                       = This Gem can only Support Skill Gems requiring Level 72 or lower<br>Supported Skills deal 25% less Damage
 |level21_stat1_id                        = local_support_gem_max_skill_level_requirement_to_support
 |level21_stat1_value                     = 72
 |level21_stat2_id                        = cast_on_damage_taken_damage_+%_final
@@ -230,7 +230,7 @@
 |level21_stat3_value                     = 3580
 |level22                                 = True
 |level22_level_requirement               = 74
-|level22_stat_text                       = This Gem can only Support Skill Gems requiring Level 74 or lower<br>Supported Skills deal 23% less Damage<br>Trigger Supported Spells when you take a total of 3950 Damage
+|level22_stat_text                       = This Gem can only Support Skill Gems requiring Level 74 or lower<br>Supported Skills deal 23% less Damage
 |level22_stat1_id                        = local_support_gem_max_skill_level_requirement_to_support
 |level22_stat1_value                     = 74
 |level22_stat2_id                        = cast_on_damage_taken_damage_+%_final
@@ -239,7 +239,7 @@
 |level22_stat3_value                     = 3950
 |level23                                 = True
 |level23_level_requirement               = 76
-|level23_stat_text                       = This Gem can only Support Skill Gems requiring Level 76 or lower<br>Supported Skills deal 21% less Damage<br>Trigger Supported Spells when you take a total of 4350 Damage
+|level23_stat_text                       = This Gem can only Support Skill Gems requiring Level 76 or lower<br>Supported Skills deal 21% less Damage
 |level23_stat1_id                        = local_support_gem_max_skill_level_requirement_to_support
 |level23_stat1_value                     = 76
 |level23_stat2_id                        = cast_on_damage_taken_damage_+%_final
@@ -248,7 +248,7 @@
 |level23_stat3_value                     = 4350
 |level24                                 = True
 |level24_level_requirement               = 78
-|level24_stat_text                       = This Gem can only Support Skill Gems requiring Level 78 or lower<br>Supported Skills deal 19% less Damage<br>Trigger Supported Spells when you take a total of 4780 Damage
+|level24_stat_text                       = This Gem can only Support Skill Gems requiring Level 78 or lower<br>Supported Skills deal 19% less Damage
 |level24_stat1_id                        = local_support_gem_max_skill_level_requirement_to_support
 |level24_stat1_value                     = 78
 |level24_stat2_id                        = cast_on_damage_taken_damage_+%_final
@@ -257,7 +257,7 @@
 |level24_stat3_value                     = 4780
 |level25                                 = True
 |level25_level_requirement               = 80
-|level25_stat_text                       = This Gem can only Support Skill Gems requiring Level 80 or lower<br>Supported Skills deal 17% less Damage<br>Trigger Supported Spells when you take a total of 5240 Damage
+|level25_stat_text                       = This Gem can only Support Skill Gems requiring Level 80 or lower<br>Supported Skills deal 17% less Damage
 |level25_stat1_id                        = local_support_gem_max_skill_level_requirement_to_support
 |level25_stat1_value                     = 80
 |level25_stat2_id                        = cast_on_damage_taken_damage_+%_final
@@ -266,7 +266,7 @@
 |level25_stat3_value                     = 5240
 |level26                                 = True
 |level26_level_requirement               = 82
-|level26_stat_text                       = This Gem can only Support Skill Gems requiring Level 82 or lower<br>Supported Skills deal 15% less Damage<br>Trigger Supported Spells when you take a total of 5730 Damage
+|level26_stat_text                       = This Gem can only Support Skill Gems requiring Level 82 or lower<br>Supported Skills deal 15% less Damage
 |level26_stat1_id                        = local_support_gem_max_skill_level_requirement_to_support
 |level26_stat1_value                     = 82
 |level26_stat2_id                        = cast_on_damage_taken_damage_+%_final
@@ -275,7 +275,7 @@
 |level26_stat3_value                     = 5730
 |level27                                 = True
 |level27_level_requirement               = 84
-|level27_stat_text                       = This Gem can only Support Skill Gems requiring Level 84 or lower<br>Supported Skills deal 13% less Damage<br>Trigger Supported Spells when you take a total of 6250 Damage
+|level27_stat_text                       = This Gem can only Support Skill Gems requiring Level 84 or lower<br>Supported Skills deal 13% less Damage
 |level27_stat1_id                        = local_support_gem_max_skill_level_requirement_to_support
 |level27_stat1_value                     = 84
 |level27_stat2_id                        = cast_on_damage_taken_damage_+%_final
@@ -284,7 +284,7 @@
 |level27_stat3_value                     = 6250
 |level28                                 = True
 |level28_level_requirement               = 86
-|level28_stat_text                       = This Gem can only Support Skill Gems requiring Level 86 or lower<br>Supported Skills deal 11% less Damage<br>Trigger Supported Spells when you take a total of 6800 Damage
+|level28_stat_text                       = This Gem can only Support Skill Gems requiring Level 86 or lower<br>Supported Skills deal 11% less Damage
 |level28_stat1_id                        = local_support_gem_max_skill_level_requirement_to_support
 |level28_stat1_value                     = 86
 |level28_stat2_id                        = cast_on_damage_taken_damage_+%_final
@@ -293,7 +293,7 @@
 |level28_stat3_value                     = 6800
 |level29                                 = True
 |level29_level_requirement               = 88
-|level29_stat_text                       = This Gem can only Support Skill Gems requiring Level 88 or lower<br>Supported Skills deal 9% less Damage<br>Trigger Supported Spells when you take a total of 7380 Damage
+|level29_stat_text                       = This Gem can only Support Skill Gems requiring Level 88 or lower<br>Supported Skills deal 9% less Damage
 |level29_stat1_id                        = local_support_gem_max_skill_level_requirement_to_support
 |level29_stat1_value                     = 88
 |level29_stat2_id                        = cast_on_damage_taken_damage_+%_final
@@ -302,7 +302,7 @@
 |level29_stat3_value                     = 7380
 |level30                                 = True
 |level30_level_requirement               = 90
-|level30_stat_text                       = This Gem can only Support Skill Gems requiring Level 90 or lower<br>Supported Skills deal 7% less Damage<br>Trigger Supported Spells when you take a total of 7990 Damage
+|level30_stat_text                       = This Gem can only Support Skill Gems requiring Level 90 or lower<br>Supported Skills deal 7% less Damage
 |level30_stat1_id                        = local_support_gem_max_skill_level_requirement_to_support
 |level30_stat1_value                     = 90
 |level30_stat2_id                        = cast_on_damage_taken_damage_+%_final
