--- Fireball
+++ Export
@@ -47,7 +47,7 @@
 |static_stat2_value                      = 25
 |static_stat3_id                         = quality_display_active_skill_ignite_damage_is_gem
 |static_stat3_value                      = 1
-|stat_text                               = Deals (9-1640) to (14-2460) Fire Damage<br>25% chance to Ignite enemies<br>Projectiles gain Radius as they travel farther, up to +(0-9) Radius
+|stat_text                               = Deals (9-1640) to (14-2460) Fire Damage<br>25% chance to Ignite enemies<br>Projectiles gain radius as they travel farther, up to a maximum of +(0-0.9) metres to radius
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_cost_amounts                     = 6
@@ -67,7 +67,7 @@
 |level3                                  = True
 |level3_level_requirement                = 4
 |level3_cost_amounts                     = 7
-|level3_stat_text                        = Deals 13 to 20 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +1 Radius
+|level3_stat_text                        = Deals 13 to 20 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +0.1 metres to radius
 |level3_stat1_id                         = spell_minimum_base_fire_damage
 |level3_stat1_value                      = 13
 |level3_stat2_id                         = spell_maximum_base_fire_damage
@@ -77,7 +77,7 @@
 |level4                                  = True
 |level4_level_requirement                = 7
 |level4_cost_amounts                     = 8
-|level4_stat_text                        = Deals 19 to 28 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +1 Radius
+|level4_stat_text                        = Deals 19 to 28 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +0.1 metres to radius
 |level4_stat1_id                         = spell_minimum_base_fire_damage
 |level4_stat1_value                      = 19
 |level4_stat2_id                         = spell_maximum_base_fire_damage
@@ -87,7 +87,7 @@
 |level5                                  = True
 |level5_level_requirement                = 11
 |level5_cost_amounts                     = 9
-|level5_stat_text                        = Deals 28 to 42 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +2 Radius
+|level5_stat_text                        = Deals 28 to 42 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +0.2 metres to radius
 |level5_stat1_id                         = spell_minimum_base_fire_damage
 |level5_stat1_value                      = 28
 |level5_stat2_id                         = spell_maximum_base_fire_damage
@@ -97,7 +97,7 @@
 |level6                                  = True
 |level6_level_requirement                = 16
 |level6_cost_amounts                     = 10
-|level6_stat_text                        = Deals 43 to 65 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +2 Radius
+|level6_stat_text                        = Deals 43 to 65 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +0.2 metres to radius
 |level6_stat1_id                         = spell_minimum_base_fire_damage
 |level6_stat1_value                      = 43
 |level6_stat2_id                         = spell_maximum_base_fire_damage
@@ -107,7 +107,7 @@
 |level7                                  = True
 |level7_level_requirement                = 20
 |level7_cost_amounts                     = 11
-|level7_stat_text                        = Deals 60 to 91 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +3 Radius
+|level7_stat_text                        = Deals 60 to 91 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +0.3 metres to radius
 |level7_stat1_id                         = spell_minimum_base_fire_damage
 |level7_stat1_value                      = 60
 |level7_stat2_id                         = spell_maximum_base_fire_damage
@@ -117,7 +117,7 @@
 |level8                                  = True
 |level8_level_requirement                = 24
 |level8_cost_amounts                     = 12
-|level8_stat_text                        = Deals 83 to 124 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +3 Radius
+|level8_stat_text                        = Deals 83 to 124 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +0.3 metres to radius
 |level8_stat1_id                         = spell_minimum_base_fire_damage
 |level8_stat1_value                      = 83
 |level8_stat2_id                         = spell_maximum_base_fire_damage
@@ -127,7 +127,7 @@
 |level9                                  = True
 |level9_level_requirement                = 28
 |level9_cost_amounts                     = 13
-|level9_stat_text                        = Deals 111 to 167 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +4 Radius
+|level9_stat_text                        = Deals 111 to 167 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +0.4 metres to radius
 |level9_stat1_id                         = spell_minimum_base_fire_damage
 |level9_stat1_value                      = 111
 |level9_stat2_id                         = spell_maximum_base_fire_damage
@@ -137,7 +137,7 @@
 |level10                                 = True
 |level10_level_requirement               = 32
 |level10_cost_amounts                    = 14
-|level10_stat_text                       = Deals 148 to 222 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +4 Radius
+|level10_stat_text                       = Deals 148 to 222 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +0.4 metres to radius
 |level10_stat1_id                        = spell_minimum_base_fire_damage
 |level10_stat1_value                     = 148
 |level10_stat2_id                        = spell_maximum_base_fire_damage
@@ -147,7 +147,7 @@
 |level11                                 = True
 |level11_level_requirement               = 36
 |level11_cost_amounts                    = 15
-|level11_stat_text                       = Deals 195 to 292 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +5 Radius
+|level11_stat_text                       = Deals 195 to 292 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +0.5 metres to radius
 |level11_stat1_id                        = spell_minimum_base_fire_damage
 |level11_stat1_value                     = 195
 |level11_stat2_id                        = spell_maximum_base_fire_damage
@@ -157,7 +157,7 @@
 |level12                                 = True
 |level12_level_requirement               = 40
 |level12_cost_amounts                    = 16
-|level12_stat_text                       = Deals 255 to 383 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +5 Radius
+|level12_stat_text                       = Deals 255 to 383 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +0.5 metres to radius
 |level12_stat1_id                        = spell_minimum_base_fire_damage
 |level12_stat1_value                     = 255
 |level12_stat2_id                        = spell_maximum_base_fire_damage
@@ -167,7 +167,7 @@
 |level13                                 = True
 |level13_level_requirement               = 44
 |level13_cost_amounts                    = 18
-|level13_stat_text                       = Deals 332 to 498 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +6 Radius
+|level13_stat_text                       = Deals 332 to 498 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +0.6 metres to radius
 |level13_stat1_id                        = spell_minimum_base_fire_damage
 |level13_stat1_value                     = 332
 |level13_stat2_id                        = spell_maximum_base_fire_damage
@@ -177,7 +177,7 @@
 |level14                                 = True
 |level14_level_requirement               = 48
 |level14_cost_amounts                    = 19
-|level14_stat_text                       = Deals 429 to 643 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +6 Radius
+|level14_stat_text                       = Deals 429 to 643 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +0.6 metres to radius
 |level14_stat1_id                        = spell_minimum_base_fire_damage
 |level14_stat1_value                     = 429
 |level14_stat2_id                        = spell_maximum_base_fire_damage
@@ -187,7 +187,7 @@
 |level15                                 = True
 |level15_level_requirement               = 52
 |level15_cost_amounts                    = 20
-|level15_stat_text                       = Deals 552 to 828 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +7 Radius
+|level15_stat_text                       = Deals 552 to 828 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +0.7 metres to radius
 |level15_stat1_id                        = spell_minimum_base_fire_damage
 |level15_stat1_value                     = 552
 |level15_stat2_id                        = spell_maximum_base_fire_damage
@@ -197,7 +197,7 @@
 |level16                                 = True
 |level16_level_requirement               = 56
 |level16_cost_amounts                    = 21
-|level16_stat_text                       = Deals 707 to 1061 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +7 Radius
+|level16_stat_text                       = Deals 707 to 1061 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +0.7 metres to radius
 |level16_stat1_id                        = spell_minimum_base_fire_damage
 |level16_stat1_value                     = 707
 |level16_stat2_id                        = spell_maximum_base_fire_damage
@@ -207,7 +207,7 @@
 |level17                                 = True
 |level17_level_requirement               = 60
 |level17_cost_amounts                    = 22
-|level17_stat_text                       = Deals 903 to 1354 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +8 Radius
+|level17_stat_text                       = Deals 903 to 1354 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +0.8 metres to radius
 |level17_stat1_id                        = spell_minimum_base_fire_damage
 |level17_stat1_value                     = 903
 |level17_stat2_id                        = spell_maximum_base_fire_damage
@@ -217,7 +217,7 @@
 |level18                                 = True
 |level18_level_requirement               = 64
 |level18_cost_amounts                    = 23
-|level18_stat_text                       = Deals 1149 to 1723 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +8 Radius
+|level18_stat_text                       = Deals 1149 to 1723 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +0.8 metres to radius
 |level18_stat1_id                        = spell_minimum_base_fire_damage
 |level18_stat1_value                     = 1149
 |level18_stat2_id                        = spell_maximum_base_fire_damage
@@ -227,7 +227,7 @@
 |level19                                 = True
 |level19_level_requirement               = 67
 |level19_cost_amounts                    = 24
-|level19_stat_text                       = Deals 1374 to 2061 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +9 Radius
+|level19_stat_text                       = Deals 1374 to 2061 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +0.9 metres to radius
 |level19_stat1_id                        = spell_minimum_base_fire_damage
 |level19_stat1_value                     = 1374
 |level19_stat2_id                        = spell_maximum_base_fire_damage
@@ -237,7 +237,7 @@
 |level20                                 = True
 |level20_level_requirement               = 70
 |level20_cost_amounts                    = 25
-|level20_stat_text                       = Deals 1640 to 2460 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +9 Radius
+|level20_stat_text                       = Deals 1640 to 2460 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +0.9 metres to radius
 |level20_stat1_id                        = spell_minimum_base_fire_damage
 |level20_stat1_value                     = 1640
 |level20_stat2_id                        = spell_maximum_base_fire_damage
@@ -247,7 +247,7 @@
 |level21                                 = True
 |level21_level_requirement               = 72
 |level21_cost_amounts                    = 25
-|level21_stat_text                       = Deals 1845 to 2767 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +10 Radius
+|level21_stat_text                       = Deals 1845 to 2767 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +1 metre to radius
 |level21_stat1_id                        = spell_minimum_base_fire_damage
 |level21_stat1_value                     = 1845
 |level21_stat2_id                        = spell_maximum_base_fire_damage
@@ -257,7 +257,7 @@
 |level22                                 = True
 |level22_level_requirement               = 74
 |level22_cost_amounts                    = 26
-|level22_stat_text                       = Deals 2073 to 3110 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +10 Radius
+|level22_stat_text                       = Deals 2073 to 3110 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +1 metre to radius
 |level22_stat1_id                        = spell_minimum_base_fire_damage
 |level22_stat1_value                     = 2073
 |level22_stat2_id                        = spell_maximum_base_fire_damage
@@ -267,7 +267,7 @@
 |level23                                 = True
 |level23_level_requirement               = 76
 |level23_cost_amounts                    = 26
-|level23_stat_text                       = Deals 2328 to 3493 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +10 Radius
+|level23_stat_text                       = Deals 2328 to 3493 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +1 metre to radius
 |level23_stat1_id                        = spell_minimum_base_fire_damage
 |level23_stat1_value                     = 2328
 |level23_stat2_id                        = spell_maximum_base_fire_damage
@@ -277,7 +277,7 @@
 |level24                                 = True
 |level24_level_requirement               = 78
 |level24_cost_amounts                    = 27
-|level24_stat_text                       = Deals 2614 to 3921 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +10 Radius
+|level24_stat_text                       = Deals 2614 to 3921 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +1 metre to radius
 |level24_stat1_id                        = spell_minimum_base_fire_damage
 |level24_stat1_value                     = 2614
 |level24_stat2_id                        = spell_maximum_base_fire_damage
@@ -287,7 +287,7 @@
 |level25                                 = True
 |level25_level_requirement               = 80
 |level25_cost_amounts                    = 27
-|level25_stat_text                       = Deals 2933 to 4400 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +10 Radius
+|level25_stat_text                       = Deals 2933 to 4400 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +1 metre to radius
 |level25_stat1_id                        = spell_minimum_base_fire_damage
 |level25_stat1_value                     = 2933
 |level25_stat2_id                        = spell_maximum_base_fire_damage
@@ -297,7 +297,7 @@
 |level26                                 = True
 |level26_level_requirement               = 82
 |level26_cost_amounts                    = 28
-|level26_stat_text                       = Deals 3289 to 4934 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +11 Radius
+|level26_stat_text                       = Deals 3289 to 4934 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +1.1 metres to radius
 |level26_stat1_id                        = spell_minimum_base_fire_damage
 |level26_stat1_value                     = 3289
 |level26_stat2_id                        = spell_maximum_base_fire_damage
@@ -307,7 +307,7 @@
 |level27                                 = True
 |level27_level_requirement               = 84
 |level27_cost_amounts                    = 28
-|level27_stat_text                       = Deals 3687 to 5531 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +11 Radius
+|level27_stat_text                       = Deals 3687 to 5531 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +1.1 metres to radius
 |level27_stat1_id                        = spell_minimum_base_fire_damage
 |level27_stat1_value                     = 3687
 |level27_stat2_id                        = spell_maximum_base_fire_damage
@@ -317,7 +317,7 @@
 |level28                                 = True
 |level28_level_requirement               = 86
 |level28_cost_amounts                    = 29
-|level28_stat_text                       = Deals 4131 to 6197 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +11 Radius
+|level28_stat_text                       = Deals 4131 to 6197 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +1.1 metres to radius
 |level28_stat1_id                        = spell_minimum_base_fire_damage
 |level28_stat1_value                     = 4131
 |level28_stat2_id                        = spell_maximum_base_fire_damage
@@ -327,7 +327,7 @@
 |level29                                 = True
 |level29_level_requirement               = 88
 |level29_cost_amounts                    = 30
-|level29_stat_text                       = Deals 4627 to 6940 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +11 Radius
+|level29_stat_text                       = Deals 4627 to 6940 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +1.1 metres to radius
 |level29_stat1_id                        = spell_minimum_base_fire_damage
 |level29_stat1_value                     = 4627
 |level29_stat2_id                        = spell_maximum_base_fire_damage
@@ -337,7 +337,7 @@
 |level30                                 = True
 |level30_level_requirement               = 90
 |level30_cost_amounts                    = 30
-|level30_stat_text                       = Deals 5180 to 7770 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +11 Radius
+|level30_stat_text                       = Deals 5180 to 7770 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +1.1 metres to radius
 |level30_stat1_id                        = spell_minimum_base_fire_damage
 |level30_stat1_value                     = 5180
 |level30_stat2_id                        = spell_maximum_base_fire_damage
@@ -347,7 +347,7 @@
 |level31                                 = True
 |level31_level_requirement               = 91
 |level31_cost_amounts                    = 30
-|level31_stat_text                       = Deals 5480 to 8219 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +12 Radius
+|level31_stat_text                       = Deals 5480 to 8219 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +1.2 metres to radius
 |level31_stat1_id                        = spell_minimum_base_fire_damage
 |level31_stat1_value                     = 5480
 |level31_stat2_id                        = spell_maximum_base_fire_damage
@@ -357,7 +357,7 @@
 |level32                                 = True
 |level32_level_requirement               = 92
 |level32_cost_amounts                    = 31
-|level32_stat_text                       = Deals 5796 to 8694 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +12 Radius
+|level32_stat_text                       = Deals 5796 to 8694 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +1.2 metres to radius
 |level32_stat1_id                        = spell_minimum_base_fire_damage
 |level32_stat1_value                     = 5796
 |level32_stat2_id                        = spell_maximum_base_fire_damage
@@ -367,7 +367,7 @@
 |level33                                 = True
 |level33_level_requirement               = 93
 |level33_cost_amounts                    = 31
-|level33_stat_text                       = Deals 6131 to 9196 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +12 Radius
+|level33_stat_text                       = Deals 6131 to 9196 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +1.2 metres to radius
 |level33_stat1_id                        = spell_minimum_base_fire_damage
 |level33_stat1_value                     = 6131
 |level33_stat2_id                        = spell_maximum_base_fire_damage
@@ -377,7 +377,7 @@
 |level34                                 = True
 |level34_level_requirement               = 94
 |level34_cost_amounts                    = 31
-|level34_stat_text                       = Deals 6484 to 9726 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +12 Radius
+|level34_stat_text                       = Deals 6484 to 9726 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +1.2 metres to radius
 |level34_stat1_id                        = spell_minimum_base_fire_damage
 |level34_stat1_value                     = 6484
 |level34_stat2_id                        = spell_maximum_base_fire_damage
@@ -387,7 +387,7 @@
 |level35                                 = True
 |level35_level_requirement               = 95
 |level35_cost_amounts                    = 31
-|level35_stat_text                       = Deals 6856 to 10285 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +12 Radius
+|level35_stat_text                       = Deals 6856 to 10285 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +1.2 metres to radius
 |level35_stat1_id                        = spell_minimum_base_fire_damage
 |level35_stat1_value                     = 6856
 |level35_stat2_id                        = spell_maximum_base_fire_damage
@@ -397,7 +397,7 @@
 |level36                                 = True
 |level36_level_requirement               = 96
 |level36_cost_amounts                    = 32
-|level36_stat_text                       = Deals 7250 to 10875 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +13 Radius
+|level36_stat_text                       = Deals 7250 to 10875 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +1.3 metres to radius
 |level36_stat1_id                        = spell_minimum_base_fire_damage
 |level36_stat1_value                     = 7250
 |level36_stat2_id                        = spell_maximum_base_fire_damage
@@ -407,7 +407,7 @@
 |level37                                 = True
 |level37_level_requirement               = 97
 |level37_cost_amounts                    = 32
-|level37_stat_text                       = Deals 7665 to 11498 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +13 Radius
+|level37_stat_text                       = Deals 7665 to 11498 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +1.3 metres to radius
 |level37_stat1_id                        = spell_minimum_base_fire_damage
 |level37_stat1_value                     = 7665
 |level37_stat2_id                        = spell_maximum_base_fire_damage
@@ -417,7 +417,7 @@
 |level38                                 = True
 |level38_level_requirement               = 98
 |level38_cost_amounts                    = 32
-|level38_stat_text                       = Deals 8104 to 12156 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +13 Radius
+|level38_stat_text                       = Deals 8104 to 12156 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +1.3 metres to radius
 |level38_stat1_id                        = spell_minimum_base_fire_damage
 |level38_stat1_value                     = 8104
 |level38_stat2_id                        = spell_maximum_base_fire_damage
@@ -427,7 +427,7 @@
 |level39                                 = True
 |level39_level_requirement               = 99
 |level39_cost_amounts                    = 33
-|level39_stat_text                       = Deals 8567 to 12850 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +13 Radius
+|level39_stat_text                       = Deals 8567 to 12850 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +1.3 metres to radius
 |level39_stat1_id                        = spell_minimum_base_fire_damage
 |level39_stat1_value                     = 8567
 |level39_stat2_id                        = spell_maximum_base_fire_damage
@@ -437,7 +437,7 @@
 |level40                                 = True
 |level40_level_requirement               = 100
 |level40_cost_amounts                    = 33
-|level40_stat_text                       = Deals 9055 to 13583 Fire Damage<br>Projectiles gain Radius as they travel farther, up to +13 Radius
+|level40_stat_text                       = Deals 9055 to 13583 Fire Damage<br>Projectiles gain radius as they travel farther, up to a maximum of +1.3 metres to radius
 |level40_stat1_id                        = spell_minimum_base_fire_damage
 |level40_stat1_value                     = 9055
 |level40_stat2_id                        = spell_maximum_base_fire_damage