--- The Endless Darkness
+++ Export
@@ -15,7 +15,6 @@
 |drop_text                               = Drops from destroyed Harbinger Portals in [[The Beachhead]], {{il|Infused Beachhead}}, or [[Delve]]. Also drops from Harbinger Portals in map affected by {{c|magic|{{il|Kirac's Memory}} of Phaaryl}}.
 |drop_monsters                           = Metadata/Monsters/HarbingerPortal/HarbingerPortalDelve, Metadata/Monsters/HarbingerPortal/HarbingerPortal, Metadata/Monsters/HarbingerPortal/HarbingerPortalUber
 |release_version                         = 3.4.0
-|card_background                         = 0,1
 }}
 
 '''The Endless Darkness''' is a [[divination card]]. A set of nine can be exchanged for {{il|Voidforge}}.
