--- The Easy Stroll
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = maps_divination, corrupted_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardTheEasyStroll
 |stack_size                              = 2
-|description                             = {{c|rare|Map}}<br>{{c|default|Map Tier:}} {{c|normal|15}}<br>{{c|default|Modifiers:}} {{c|normal|8}}<br>{{c|corrupted|[[Corrupted]]}}
+|description                             = {{c|rare|[[Map]]}}<br>{{c|default|Map Tier:}} {{c|normal|15}}<br>{{c|default|Modifiers:}} {{c|normal|8}}<br>{{c|corrupted|[[Corrupted]]}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_areas                              = MapWorldsGardens, MapWorldsCourtyard, MapWorldsTerrace, MapWorldsOrchard
