--- Loyalty Tattoo of Kiloava
+++ Export
@@ -11,7 +11,10 @@
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |tattoo_target                           = Small Intelligence
-|tattoo_stat_text                        = Trigger Summon Spirit of Kiloava [[Skill]] when you [[Block]] Damage from a Unique Enemy
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Intelligence}} {{c|default|Passive Skill}}
 |tattoo_limit                            = 1 Loyalty Tattoo
-|description                             = Replaces a Small Intelligence Passive Skill<br>Grants "Trigger Summon Spirit of Kiloava [[Skill]] when you [[Block]] Damage from a Unique Enemy"<br>Limit 1 Loyalty Tattoo
+|implicit2_text                          = {{c|default|Limit}} {{c|value|1 Loyalty Tattoo}}
+|tattoo_stat_text                        = Trigger Summon Spirit of Kiloava Skill when you Block Damage from a Unique Enemy
+|tattoo_skill_id                         = TriggerSummonedAncestorKiloava
+|description                             = Grants "Trigger [[Skill:TriggerSummonedAncestorKiloava|Summon Spirit of Kiloava]] [[Skill]] when you [[Block]] Damage from a Unique Enemy"
 }}