--- Fracturing Orb
+++ Export
@@ -8,7 +8,7 @@
 |tags                                    = currency, default
 |metadata_id                             = Metadata/Items/Currency/CurrencyFractureRare
 |stack_size                              = 20
-|description                             = Fracture a random modifier on a rare item with at least 4 modifiers, locking it in place.
+|description                             = Fracture a random modifier on a rare item with at least 4 modifiers, locking it in place
 |help_text                               = Right click this item then left click a rare item to apply it. Cannot be used on Influenced, Synthesised or Fractured items.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |drop_text                               = Drops from [[Harbinger]]s or Harbinger-related rewards.
