--- Wiki
+++ Export
@@ -36,11 +36,11 @@
 |static_stat1_value                      = 20
 |static_stat2_id                         = base_is_projectile
 |static_stat2_value                      = 1
-|static_stat3_id                         = is_area_damage
+|static_stat3_id                         = console_skill_dont_chase
 |static_stat3_value                      = 1
-|static_stat4_id                         = projectile_behaviour_only_explode
+|static_stat4_id                         = is_area_damage
 |static_stat4_value                      = 1
-|static_stat5_id                         = console_skill_dont_chase
+|static_stat5_id                         = projectile_behaviour_only_explode
 |static_stat5_value                      = 1
 |static_stat6_id                         = skill_base_chaos_damage_%_maximum_energy_shield
 |static_stat6_value                      = 5
