--- The Traitor
+++ Export
@@ -11,7 +11,7 @@
 |tags                                    = weapon_divination, unique_divination, unique_weapon_divination, unique_corrupted_divination, corrupted_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardTheTraitor
 |stack_size                              = 4
-|description                             = {{c|unique|Wand}}<br>{{c|corrupted|[[Corrupted]]}}
+|description                             = {{c|unique|[[Wand]]}}<br>{{c|corrupted|[[Corrupted]]}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_areas                              = 1_3_7, 2_8_1, 2_8_11, MapAtlasPromenade, MapWorldsRamparts, MapWorldsPromenade
