--- Vaal Double Strike
+++ Export
@@ -27,9 +27,9 @@
 |static_duration                         = 6
 |static_stat1_id                         = base_skill_number_of_additional_hits
 |static_stat1_value                      = 1
-|static_stat2_id                         = number_of_monsters_to_summon
+|static_stat2_id                         = cannot_cancel_skill_before_contact_point
 |static_stat2_value                      = 1
-|static_stat3_id                         = cannot_cancel_skill_before_contact_point
+|static_stat3_id                         = number_of_monsters_to_summon
 |static_stat3_value                      = 1
 |static_stat4_id                         = base_skill_effect_duration
 |static_stat4_value                      = 6000
