--- Intimidating Cry
+++ Export
@@ -14,7 +14,6 @@
 |active_skill_name                       = Intimidating Cry
 |skill_id                                = IntimidatingCry
 |cast_time                               = 0.8
-|radius                                  = 60
 |required_level                          = 10
 |quality_type1_weight                    = 50
 |quality_type1_stat_text                 = 1% increased Cooldown Recovery Rate
@@ -38,22 +37,26 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_deal_no_damage
+|static_stat1_id                         = base_deal_no_attack_damage
 |static_stat1_value                      = 1
-|static_stat2_id                         = cannot_cancel_skill_before_contact_point
+|static_stat2_id                         = base_deal_no_secondary_damage
 |static_stat2_value                      = 1
-|static_stat3_id                         = use_intimidating_cry_buff_visual_for_intimidate
+|static_stat3_id                         = base_deal_no_spell_damage
 |static_stat3_value                      = 1
-|static_stat4_id                         = enemies_taunted_by_your_warcies_are_intimidated
+|static_stat4_id                         = cannot_cancel_skill_before_contact_point
 |static_stat4_value                      = 1
-|static_stat5_id                         = warcry_count_power_from_enemies
+|static_stat5_id                         = use_intimidating_cry_buff_visual_for_intimidate
 |static_stat5_value                      = 1
-|static_stat6_id                         = intimidating_cry_enemy_phys_reduction_%_penalty_vs_hit_per_5_MP
-|static_stat6_value                      = 5
-|static_stat7_id                         = skill_empowers_next_x_melee_attacks
-|static_stat7_value                      = 2
-|static_stat8_id                         = intimidating_cry_empowerd_attacks_deal_double_damage_display
-|static_stat8_value                      = 1
+|static_stat6_id                         = enemies_taunted_by_your_warcies_are_intimidated
+|static_stat6_value                      = 1
+|static_stat7_id                         = warcry_count_power_from_enemies
+|static_stat7_value                      = 1
+|static_stat8_id                         = intimidating_cry_enemy_phys_reduction_%_penalty_vs_hit_per_5_MP
+|static_stat8_value                      = 5
+|static_stat9_id                         = skill_empowers_next_x_melee_attacks
+|static_stat9_value                      = 2
+|static_stat10_id                        = intimidating_cry_empowerd_attacks_deal_double_damage_display
+|static_stat10_value                     = 1
 |stat_text                               = Base duration is (4-4.9) seconds<br>(0-38)% increased Warcry Speed<br>Enemies Taunted by this Warcry are Intimidated<br>Counts total Power of Enemies in Range<br>Buff causes your Hits to Overwhelm 5% Physical Damage Reduction per 5 Power, up to 30%<br>Exerts the next 2 Melee Attacks you perform<br>Exerted Attacks deal Double Damage
 |level1                                  = True
 |level1_level_requirement                = 10
@@ -454,6 +457,7 @@
 |level20_strength_requirement            = 155
 |level20_experience                      = 342026975
 |level21_strength_requirement            = 159
+|radius                                  = 60
 |release_version                         = 3.11.0
 |quest_reward1_type                      = quest
 |quest_reward1_quest                     = The Caged Brute
