--- Wiki
+++ Export
@@ -39,9 +39,9 @@
 |static_duration                         = 3
 |static_stat1_id                         = blind_art_variation
 |static_stat1_value                      = 1
-|static_stat2_id                         = is_area_damage
+|static_stat2_id                         = console_skill_dont_chase
 |static_stat2_value                      = 1
-|static_stat3_id                         = console_skill_dont_chase
+|static_stat3_id                         = is_area_damage
 |static_stat3_value                      = 1
 |static_stat4_id                         = skill_can_add_multiple_charges_per_action
 |static_stat4_value                      = 1
