--- Something Dark
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardSomethingDark
 |stack_size                              = 5
-|description                             = {{c|white|Blueprint}}<br>{{c|default|Area Level:}} {{c|normal|83}}<br>{{c|default|Fully Revealed}}
+|description                             = {{c|white|[[Blueprint]]}}<br>{{c|default|Area Level:}} {{c|normal|83}}<br>{{c|default|Fully Revealed}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_text                               = Drops from [[Breachlord]]s in [[Breachstone]]s.
