--- Wiki
+++ Export
@@ -7,5 +7,5 @@
 |tags                                    = not_for_sale, default
 |metadata_id                             = Metadata/Items/MicrotransactionItemEffects/MicrotransactionSoulstunningWeaponEffect
 |stack_size                              = 1
-|description                             = Adds the Soulstunning Weapon Effect to any equipped weapon.
+|description                             = Adds the Soulstunning Weapon Effect to any equipped weapon. Enemies stunned from attacks will have an out-of-body experience.
 }}