--- Imprint
+++ Export
@@ -12,8 +12,6 @@
 |drop_text                               = Imprint cannot drop naturally. It is created using {{il|Eternal Orb}} or [[Craicic Chimeral]]'s [[beastcrafting]] recipe.
 |is_drop_restricted                      = true
 |release_version                         = 0.11.0
-|sell_price1_name                        = Orb of Regret
-|sell_price1_amount                      = 1
 }}
 
 '''Imprints''' are created when an {{il|Eternal Orb}} is used on an item.
