--- Eclipse Staff
+++ Export
@@ -16,7 +16,7 @@
 |attack_speed                            = 1.2
 |physical_damage_min                     = 70
 |physical_damage_max                     = 145
-|weapon_range                            = 13
+|weapon_range                            = 1.3
 |alternate_art_inventory_icons           = soulthirst
 |release_version                         = 2.0.0
 |recipe1_part1_item_name                 = The Golden Era