--- Contract: Enoch's Remains
+++ Export
@@ -8,7 +8,6 @@
 |tags                                    = default
 |metadata_id                             = Metadata/Items/Heist/QuestContracts/HeistContractQuestHuck2
 |inventory_icon                          = Generic NPC Quest Contract
-|flavour_text                            = "Enoch deserved better.<br />This is the least I can do."
 }}
 {{stub}}
 '''Contract: Enoch's Remains''' is a [[quest]] [[contract]]. The client is [[Huck]]. It is the second quest of his personal storyline. The area level of this contract is depended on the area level of the Smuggler's Caches.
