--- Trap Support
+++ Export
@@ -7,7 +7,7 @@
 |drop_level                              = 8
 |tags                                    = up_to_level_8_gem, support_gem, gem, default
 |metadata_id                             = Metadata/Items/Gems/SupportGemTrap
-|help_text                               = This is a Support Gem. It does not grant a bonus to your character, but to skills in sockets connected to it. Place into an item socket connected to a socket containing the Active Skill Gem you wish to augment. Right click to remove from a socket.
+|help_text                               = This is a Support Gem. It does not grant a bonus to your character, but to skills in sockets connected to it. Place into an item socket connected to a socket containing the Skill Gem you wish to augment. Right click to remove from a socket.
 |dexterity_percent                       = 60
 |intelligence_percent                    = 40
 |gem_tags                                = Support, Trap
@@ -30,14 +30,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = support_makes_skill_trap_pvp_damage_+%_final
-|static_stat1_value                      = -60
-|static_stat2_id                         = base_skill_is_trapped
+|static_stat1_id                         = base_skill_is_trapped
+|static_stat1_value                      = 1
+|static_stat2_id                         = base_skill_show_average_damage_instead_of_dps
 |static_stat2_value                      = 1
-|static_stat3_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat3_id                         = is_trap
 |static_stat3_value                      = 1
-|static_stat4_id                         = is_trap
-|static_stat4_value                      = 1
+|static_stat4_id                         = support_makes_skill_trap_pvp_damage_+%_final
+|static_stat4_value                      = -60
 |static_stat5_id                         = disable_skill_if_melee_attack
 |static_stat5_value                      = 1
 |static_stat6_id                         = base_trap_duration
