--- Tattoo of the Tasalio Warrior
+++ Export
@@ -11,6 +11,7 @@
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |tattoo_target                           = Small Dexterity
-|tattoo_stat_text                        = 5% chance to Blind Enemies on [[Hit]] with [[Attack|Attacks]]
-|description                             = Replaces a Small Dexterity Passive Skill<br>Grants "5% chance to Blind Enemies on [[Hit]] with [[Attack|Attacks]]"
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Dexterity}} {{c|default|Passive Skill}}
+|tattoo_stat_text                        = 5% chance to Blind Enemies on Hit with Attacks
+|description                             = Grants "5% chance to Blind Enemies on [[Hit]] with [[Attack|Attacks]]"
 }}