--- Wiki
+++ Export
@@ -31,10 +31,10 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = mirage_archer_projectile_additional_height_offset
-|static_stat1_value                      = -138
-|static_stat2_id                         = disable_skill_if_weapon_not_bow
-|static_stat2_value                      = 1
+|static_stat1_id                         = disable_skill_if_weapon_not_bow
+|static_stat1_value                      = 1
+|static_stat2_id                         = mirage_archer_projectile_additional_height_offset
+|static_stat2_value                      = -138
 |static_stat3_id                         = skill_can_own_mirage_archers
 |static_stat3_value                      = 1
 |static_stat4_id                         = summon_mirage_archer_on_hit
