--- Journeyman Cartographer's Seal
+++ Export
@@ -14,10 +14,6 @@
 |drop_enabled                            = false
 |release_version                         = 2.4.0
 |removal_version                         = 3.9.0
-| sell_price1_name = Prime Sextant
-| sell_price1_amount =3
-| sell_price2_name = Orb of Scouring
-| sell_price2_amount =1
 }}
 '''Journeyman Cartographer's Seal''' is a legacy [[currency]] item. It was used be uncompletes a white or yellow [[map]] (tier 1 to 10) on the [[Atlas of Worlds]]. 
 
