--- Contract: Karst's Revenge
+++ Export
@@ -8,7 +8,6 @@
 |tags                                    = default
 |metadata_id                             = Metadata/Items/Heist/QuestContracts/HeistContractQuestKarst2
 |inventory_icon                          = Generic NPC Quest Contract
-|flavour_text                            = "Traitorous scum took the life I could've had.<br />I ain't usually the forgiving type,<br />but I feel in this case that I owe him a little gift."
 }}
 {{stub}}
 '''Contract: Karst's Revenge''' is a [[quest]] [[contract]]. The client is [[Karst]]. It is the second quest of his personal storyline. The area level of this contract is depended on the area level of the Smuggler's Caches.
