--- Cartographer's Delight
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = maps_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardCartographersDelight
 |stack_size                              = 3
-|description                             = {{c|normal|Map}}<br>{{c|default|Map Tier:}} {{c|normal|5}}
+|description                             = {{c|normal|[[Map]]}}<br>{{c|default|Map Tier:}} {{c|normal|5}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_areas                              = 2_8_13, 2_9_5
