--- Grand Eldritch Ichor
+++ Export
@@ -8,7 +8,7 @@
 |tags                                    = great_tangle_currency, eldritch_ichor, currency, default
 |metadata_id                             = Metadata/Items/Currency/CurrencyEldritchIchor3
 |stack_size                              = 10
-|description                             = Adds a Grand Eater of Worlds implicit modifier to a Body Armour, Boots, Gloves or Helmet. This replaces any existing implicit modifiers other than Searing Exarch implicit modifiers.
+|description                             = Adds a Grand Eater of Worlds implicit modifier to<br>a Body Armour, Boots, Gloves or Helmet<br>This replaces any existing implicit modifiers other than Searing Exarch implicit modifiers
 |help_text                               = Right click this item then left click a normal, magic or rare item to apply it. Cannot be used on Shaper, Elder or Elderslayer influenced items.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |drop_text                               = Drops from the [[Infinite Hunger]].
