--- Contract: Isla's Designs
+++ Export
@@ -7,15 +7,14 @@
 |drop_level                              = 55
 |tags                                    = default
 |metadata_id                             = Metadata/Items/Heist/QuestContracts/HeistContractQuestIsla2
-|release_version = 3.12.0
+|inventory_icon                          = Generic NPC Quest Contract
 |is_drop_restricted                      = true
-|inventory_icon                          = Generic NPC Quest Contract
+|release_version                         = 3.12.0
 |implicit1_text                          = {{c|default|Client:}} {{c|value|Isla}}
 |implicit2_text                          = {{c|default|Heist Target:}} {{c|value|Baffling Blueprints}}
 |implicit3_text                          = {{c|default|Area Level:}} {{c|value|(55-?)}}
 |implicit4_text                          = {{c|default|Requires}} {{c|value|Trap Disarmament}} {{c|default|(Level}} {{c|value|1}}{{c|default|)}}
 |implicit5_text                          = {{c|default|Requires}} {{c|value|Isla, the Engineer}}
-|flavour_text                            = ...the blueprints for my Extreme Warmth Beam, my Storm Wooing Rod,<br/>the Mind Sublimator, and the Unbreakable...!
 }}
 
 '''Contract: Isla's Designs''' is a [[quest]] [[contract]]. It is the second quest of [[Isla]]. This was followed by [[Unique item|unique]] contract {{il|Contract: Breaking the Unbreakable}}.
