--- Wiki
+++ Export
@@ -15,7 +15,6 @@
 |drop_text                               = Can also drop from {{il|Maven's Invitation: The Feared}}.
 |drop_monsters                           = Metadata/Monsters/AtlasBosses/TheShaperBoss, Metadata/Monsters/AtlasBosses/TheShaperBossUberElder
 |release_version                         = 3.13.0
-|card_background                         = 0,1
 }}
 
 '''Dying Light''' is a [[divination card]]. A set of ten can be exchanged for a Shaper and Elder [[Influenced item|influenced]] [[rare]] {{il|Diamond Ring}} with [[item level]] 100.
