--- Cold Snap
+++ Export
@@ -46,7 +46,7 @@
 |static_stat4_value                      = 1
 |static_stat5_id                         = chance_to_gain_frenzy_charge_on_killing_enemy_affected_by_cold_snap_ground_%
 |static_stat5_value                      = 25
-|stat_text                               = Deals (34-1440) to (51-2160) Cold Damage<br>Deals (28.7-1895.4) Base Cold Damage per second<br>Base duration is 5 seconds<br>Modifiers to Spell Damage apply to this Skill's Damage Over Time effect<br>(0-19)% increased Effect of Chill<br>25% chance to gain a Frenzy Charge when an Enemy Dies while in this Skill's Area
+|stat_text                               = Deals (34-1440) to (51-2160) Cold Damage<br>Base duration is 5 seconds<br>Deals (28.7-1895.4) Base Cold Damage per second<br>Modifiers to Spell Damage apply to this Skill's Damage Over Time effect<br>(0-19)% increased Effect of Chill<br>25% chance to gain a Frenzy Charge when an Enemy Dies while in this Skill's Area
 |level1                                  = True
 |level1_level_requirement                = 16
 |level1_cost_amounts                     = 11
