--- Bound Fossil
+++ Export
@@ -11,8 +11,8 @@
 |description                             = More Minion, Aura or Curse modifiers
 |help_text                               = Place in a Resonator to influence item crafting.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
+|drop_text                               = Found in [[Petrified Forest]] and [[Abyssal Depths (Delve)|Abyssal Depths]] biomes in the [[Azurite Mine]].
 |release_version                         = 3.4.0
-|drop_text                               = Found in [[Petrified Forest]] and [[Abyssal Depths (Delve)|Abyssal Depths]] biomes in the [[Azurite Mine]].
 }}
 
 '''Bound Fossil''' is a [[fossil]] type. 