--- The Catch
+++ Export
@@ -11,7 +11,7 @@
 |tags                                    = weapon_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardTheCatch
 |stack_size                              = 18
-|description                             = {{c|white|{{il|html=|Fishing Rod}}}}<br>{{c|enchanted|Incubated}}<br>{{c|default|Item Level:}} {{c|normal|99}}
+|description                             = {{c|white|[[Fishing Rod]]}}<br>{{c|enchanted|Incubated}}<br>{{c|default|Item Level:}} {{c|normal|99}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_areas                              = MapWorldsStrand, MapWorldsShipyard, MapWorldsBeach, MapWorldsAtoll, MapWorldsTropicalIsland, MapWorldsShore, MapWorldsPeninsula, MapWorldsCoves, MapWorldsAtollUnique
