--- Wiki
+++ Export
@@ -36,14 +36,14 @@
 |static_cost_amounts                     = 6
 |static_cost_types                       = Mana
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = total_projectile_spread_angle_override
-|static_stat1_value                      = 70
-|static_stat2_id                         = active_skill_projectile_speed_+%_variation_final
-|static_stat2_value                      = 30
+|static_stat1_id                         = active_skill_projectile_speed_+%_variation_final
+|static_stat1_value                      = 30
+|static_stat2_id                         = console_skill_dont_chase
+|static_stat2_value                      = 1
 |static_stat3_id                         = show_number_of_projectiles
 |static_stat3_value                      = 1
-|static_stat4_id                         = console_skill_dont_chase
-|static_stat4_value                      = 1
+|static_stat4_id                         = total_projectile_spread_angle_override
+|static_stat4_value                      = 70
 |static_stat5_id                         = skill_physical_damage_%_to_convert_to_lightning
 |static_stat5_value                      = 50
 |static_stat6_id                         = number_of_additional_projectiles
