--- Croaking Utility Flask
+++ Export
@@ -7,5 +7,5 @@
 |tags                                    = not_for_sale, default
 |metadata_id                             = Metadata/Items/MicrotransactionItemEffects/MicrotransactionCroakingUtilityFlask
 |stack_size                              = 1
-|description                             = Activating your Utility Flask causes two unsuspecting toads to shoot our of your body.
+|description                             = Activating your Utility Flask causes two unsuspecting toads to shoot out of your body.
 }}