--- Tattoo of the Tawhoa Naturalist
+++ Export
@@ -11,10 +11,13 @@
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |tattoo_target                           = Small Dexterity
-|tattoo_stat_text                        = 8% increased [[Life]] Recovery from Flasks<br>8% increased [[Mana]] Recovery from Flasks
-|description                             = Replaces a Small Dexterity Passive Skill<br>Grants "8% increased [[Life]] Recovery from Flasks<br>8% increased [[Mana]] Recovery from Flasks"
+|tattoo_tribe                            = 8
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Dexterity}} {{c|default|Passive Skill}}
+|tattoo_stat_text                        = 8% increased Life Recovery from Flasks
+8% increased Mana Recovery from Flasks
+|description                             = Grants "8% increased [[Life]] Recovery from Flasks<br>8% increased [[Mana]] Recovery from Flasks"
 |inventory_icon                          = Tattoo dexterity 1
-|drop_text                               =  Can be obtained as a reward for defeating the [[Tawhoa]] tribe in the [[Trial of the Ancestors]].
+|drop_text                               = Can be obtained as a reward for defeating the [[Tawhoa]] tribe in the [[Trial of the Ancestors]].
 |is_drop_restricted                      = true
 |release_version                         = 3.22.0
 }}