--- Wiki
+++ Export
@@ -0,0 +1,11 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Midnight Shade Portrait Frame
+|class_id                                = Microtransaction
+|size_x                                  = 2
+|size_y                                  = 2
+|tags                                    = not_for_sale, default
+|metadata_id                             = Metadata/Items/MicrotransactionCharacterEffects/MicrotransactionMidnightShadePortraitFrame
+|stack_size                              = 1
+|description                             = Adds the Midnight Shade Portrait Frame to this character.
+}}