--- Smite
+++ Export
@@ -47,7 +47,7 @@
 |static_stat4_value                      = 4000
 |static_stat5_id                         = base_smite_number_of_targets
 |static_stat5_value                      = 1
-|stat_text                               = 50% of Physical Damage Converted to Lightning Damage<br>Base duration is 4 seconds<br>+(0-6) to radius<br>(30-20)% less Area Damage<br>Lightning strikes 1 target<br>Aura grants (1-12) to (4-220) added Lightning Damage<br>Aura grants (10-19)% chance to Shock
+|stat_text                               = 50% of Physical Damage Converted to Lightning Damage<br>Base duration is 4 seconds<br>+(0-0.6) metres to radius<br>(30-20)% less Area Damage<br>Lightning strikes 1 target<br>Aura grants (1-12) to (4-220) added Lightning Damage<br>Aura grants (10-19)% chance to Shock
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_damage_effectiveness             = 196
@@ -78,7 +78,7 @@
 |level3_level_requirement                = 4
 |level3_damage_effectiveness             = 204
 |level3_damage_multiplier                = 204.1
-|level3_stat_text                        = +1 to radius<br>29% less Area Damage<br>Aura grants 1 to 6 added Lightning Damage<br>Aura grants 11% chance to Shock
+|level3_stat_text                        = +0.1 metres to radius<br>29% less Area Damage<br>Aura grants 1 to 6 added Lightning Damage<br>Aura grants 11% chance to Shock
 |level3_stat1_id                         = active_skill_base_radius_+
 |level3_stat1_value                      = 1
 |level3_stat2_id                         = active_skill_area_damage_+%_final
@@ -93,7 +93,7 @@
 |level4_level_requirement                = 7
 |level4_damage_effectiveness             = 208
 |level4_damage_multiplier                = 208.2
-|level4_stat_text                        = +1 to radius<br>28% less Area Damage<br>Aura grants 1 to 8 added Lightning Damage<br>Aura grants 11% chance to Shock
+|level4_stat_text                        = +0.1 metres to radius<br>28% less Area Damage<br>Aura grants 1 to 8 added Lightning Damage<br>Aura grants 11% chance to Shock
 |level4_stat1_id                         = active_skill_base_radius_+
 |level4_stat1_value                      = 1
 |level4_stat2_id                         = active_skill_area_damage_+%_final
@@ -108,7 +108,7 @@
 |level5_level_requirement                = 11
 |level5_damage_effectiveness             = 212
 |level5_damage_multiplier                = 212.2
-|level5_stat_text                        = +1 to radius<br>28% less Area Damage<br>Aura grants 1 to 11 added Lightning Damage<br>Aura grants 12% chance to Shock
+|level5_stat_text                        = +0.1 metres to radius<br>28% less Area Damage<br>Aura grants 1 to 11 added Lightning Damage<br>Aura grants 12% chance to Shock
 |level5_stat1_id                         = active_skill_base_radius_+
 |level5_stat1_value                      = 1
 |level5_stat2_id                         = active_skill_area_damage_+%_final
@@ -123,7 +123,7 @@
 |level6_level_requirement                = 16
 |level6_damage_effectiveness             = 216
 |level6_damage_multiplier                = 216.3
-|level6_stat_text                        = +2 to radius<br>27% less Area Damage<br>Aura grants 1 to 16 added Lightning Damage<br>Aura grants 12% chance to Shock
+|level6_stat_text                        = +0.2 metres to radius<br>27% less Area Damage<br>Aura grants 1 to 16 added Lightning Damage<br>Aura grants 12% chance to Shock
 |level6_stat1_id                         = active_skill_base_radius_+
 |level6_stat1_value                      = 2
 |level6_stat2_id                         = active_skill_area_damage_+%_final
@@ -138,7 +138,7 @@
 |level7_level_requirement                = 20
 |level7_damage_effectiveness             = 220
 |level7_damage_multiplier                = 220.3
-|level7_stat_text                        = +2 to radius<br>27% less Area Damage<br>Aura grants 1 to 21 added Lightning Damage<br>Aura grants 13% chance to Shock
+|level7_stat_text                        = +0.2 metres to radius<br>27% less Area Damage<br>Aura grants 1 to 21 added Lightning Damage<br>Aura grants 13% chance to Shock
 |level7_stat1_id                         = active_skill_base_radius_+
 |level7_stat1_value                      = 2
 |level7_stat2_id                         = active_skill_area_damage_+%_final
@@ -153,7 +153,7 @@
 |level8_level_requirement                = 24
 |level8_damage_effectiveness             = 224
 |level8_damage_multiplier                = 224.4
-|level8_stat_text                        = +2 to radius<br>26% less Area Damage<br>Aura grants 1 to 26 added Lightning Damage<br>Aura grants 13% chance to Shock
+|level8_stat_text                        = +0.2 metres to radius<br>26% less Area Damage<br>Aura grants 1 to 26 added Lightning Damage<br>Aura grants 13% chance to Shock
 |level8_stat1_id                         = active_skill_base_radius_+
 |level8_stat1_value                      = 2
 |level8_stat2_id                         = active_skill_area_damage_+%_final
@@ -168,7 +168,7 @@
 |level9_level_requirement                = 28
 |level9_damage_effectiveness             = 228
 |level9_damage_multiplier                = 228.4
-|level9_stat_text                        = +3 to radius<br>26% less Area Damage<br>Aura grants 2 to 33 added Lightning Damage<br>Aura grants 14% chance to Shock
+|level9_stat_text                        = +0.3 metres to radius<br>26% less Area Damage<br>Aura grants 2 to 33 added Lightning Damage<br>Aura grants 14% chance to Shock
 |level9_stat1_id                         = active_skill_base_radius_+
 |level9_stat1_value                      = 3
 |level9_stat2_id                         = active_skill_area_damage_+%_final
@@ -183,7 +183,7 @@
 |level10_level_requirement               = 32
 |level10_damage_effectiveness            = 232
 |level10_damage_multiplier               = 232.5
-|level10_stat_text                       = +3 to radius<br>25% less Area Damage<br>Aura grants 2 to 40 added Lightning Damage<br>Aura grants 14% chance to Shock
+|level10_stat_text                       = +0.3 metres to radius<br>25% less Area Damage<br>Aura grants 2 to 40 added Lightning Damage<br>Aura grants 14% chance to Shock
 |level10_stat1_id                        = active_skill_base_radius_+
 |level10_stat1_value                     = 3
 |level10_stat2_id                        = active_skill_area_damage_+%_final
@@ -198,7 +198,7 @@
 |level11_level_requirement               = 36
 |level11_damage_effectiveness            = 237
 |level11_damage_multiplier               = 236.5
-|level11_stat_text                       = +3 to radius<br>25% less Area Damage<br>Aura grants 3 to 49 added Lightning Damage<br>Aura grants 15% chance to Shock
+|level11_stat_text                       = +0.3 metres to radius<br>25% less Area Damage<br>Aura grants 3 to 49 added Lightning Damage<br>Aura grants 15% chance to Shock
 |level11_stat1_id                        = active_skill_base_radius_+
 |level11_stat1_value                     = 3
 |level11_stat2_id                        = active_skill_area_damage_+%_final
@@ -213,7 +213,7 @@
 |level12_level_requirement               = 40
 |level12_damage_effectiveness            = 241
 |level12_damage_multiplier               = 240.6
-|level12_stat_text                       = +4 to radius<br>24% less Area Damage<br>Aura grants 3 to 60 added Lightning Damage<br>Aura grants 15% chance to Shock
+|level12_stat_text                       = +0.4 metres to radius<br>24% less Area Damage<br>Aura grants 3 to 60 added Lightning Damage<br>Aura grants 15% chance to Shock
 |level12_stat1_id                        = active_skill_base_radius_+
 |level12_stat1_value                     = 4
 |level12_stat2_id                        = active_skill_area_damage_+%_final
@@ -228,7 +228,7 @@
 |level13_level_requirement               = 44
 |level13_damage_effectiveness            = 245
 |level13_damage_multiplier               = 244.6
-|level13_stat_text                       = +4 to radius<br>24% less Area Damage<br>Aura grants 4 to 72 added Lightning Damage<br>Aura grants 16% chance to Shock
+|level13_stat_text                       = +0.4 metres to radius<br>24% less Area Damage<br>Aura grants 4 to 72 added Lightning Damage<br>Aura grants 16% chance to Shock
 |level13_stat1_id                        = active_skill_base_radius_+
 |level13_stat1_value                     = 4
 |level13_stat2_id                        = active_skill_area_damage_+%_final
@@ -243,7 +243,7 @@
 |level14_level_requirement               = 48
 |level14_damage_effectiveness            = 249
 |level14_damage_multiplier               = 248.7
-|level14_stat_text                       = +4 to radius<br>23% less Area Damage<br>Aura grants 5 to 87 added Lightning Damage<br>Aura grants 16% chance to Shock
+|level14_stat_text                       = +0.4 metres to radius<br>23% less Area Damage<br>Aura grants 5 to 87 added Lightning Damage<br>Aura grants 16% chance to Shock
 |level14_stat1_id                        = active_skill_base_radius_+
 |level14_stat1_value                     = 4
 |level14_stat2_id                        = active_skill_area_damage_+%_final
@@ -258,7 +258,7 @@
 |level15_level_requirement               = 52
 |level15_damage_effectiveness            = 253
 |level15_damage_multiplier               = 252.7
-|level15_stat_text                       = +5 to radius<br>23% less Area Damage<br>Aura grants 5 to 104 added Lightning Damage<br>Aura grants 17% chance to Shock
+|level15_stat_text                       = +0.5 metres to radius<br>23% less Area Damage<br>Aura grants 5 to 104 added Lightning Damage<br>Aura grants 17% chance to Shock
 |level15_stat1_id                        = active_skill_base_radius_+
 |level15_stat1_value                     = 5
 |level15_stat2_id                        = active_skill_area_damage_+%_final
@@ -273,7 +273,7 @@
 |level16_level_requirement               = 56
 |level16_damage_effectiveness            = 257
 |level16_damage_multiplier               = 256.8
-|level16_stat_text                       = +5 to radius<br>22% less Area Damage<br>Aura grants 6 to 123 added Lightning Damage<br>Aura grants 17% chance to Shock
+|level16_stat_text                       = +0.5 metres to radius<br>22% less Area Damage<br>Aura grants 6 to 123 added Lightning Damage<br>Aura grants 17% chance to Shock
 |level16_stat1_id                        = active_skill_base_radius_+
 |level16_stat1_value                     = 5
 |level16_stat2_id                        = active_skill_area_damage_+%_final
@@ -288,7 +288,7 @@
 |level17_level_requirement               = 60
 |level17_damage_effectiveness            = 261
 |level17_damage_multiplier               = 260.8
-|level17_stat_text                       = +5 to radius<br>22% less Area Damage<br>Aura grants 8 to 146 added Lightning Damage<br>Aura grants 18% chance to Shock
+|level17_stat_text                       = +0.5 metres to radius<br>22% less Area Damage<br>Aura grants 8 to 146 added Lightning Damage<br>Aura grants 18% chance to Shock
 |level17_stat1_id                        = active_skill_base_radius_+
 |level17_stat1_value                     = 5
 |level17_stat2_id                        = active_skill_area_damage_+%_final
@@ -303,7 +303,7 @@
 |level18_level_requirement               = 64
 |level18_damage_effectiveness            = 265
 |level18_damage_multiplier               = 264.9
-|level18_stat_text                       = +6 to radius<br>21% less Area Damage<br>Aura grants 9 to 173 added Lightning Damage<br>Aura grants 18% chance to Shock
+|level18_stat_text                       = +0.6 metres to radius<br>21% less Area Damage<br>Aura grants 9 to 173 added Lightning Damage<br>Aura grants 18% chance to Shock
 |level18_stat1_id                        = active_skill_base_radius_+
 |level18_stat1_value                     = 6
 |level18_stat2_id                        = active_skill_area_damage_+%_final
@@ -318,7 +318,7 @@
 |level19_level_requirement               = 67
 |level19_damage_effectiveness            = 269
 |level19_damage_multiplier               = 268.9
-|level19_stat_text                       = +6 to radius<br>21% less Area Damage<br>Aura grants 10 to 195 added Lightning Damage<br>Aura grants 19% chance to Shock
+|level19_stat_text                       = +0.6 metres to radius<br>21% less Area Damage<br>Aura grants 10 to 195 added Lightning Damage<br>Aura grants 19% chance to Shock
 |level19_stat1_id                        = active_skill_base_radius_+
 |level19_stat1_value                     = 6
 |level19_stat2_id                        = active_skill_area_damage_+%_final
@@ -333,7 +333,7 @@
 |level20_level_requirement               = 70
 |level20_damage_effectiveness            = 273
 |level20_damage_multiplier               = 273
-|level20_stat_text                       = +6 to radius<br>20% less Area Damage<br>Aura grants 12 to 220 added Lightning Damage<br>Aura grants 19% chance to Shock
+|level20_stat_text                       = +0.6 metres to radius<br>20% less Area Damage<br>Aura grants 12 to 220 added Lightning Damage<br>Aura grants 19% chance to Shock
 |level20_stat1_id                        = active_skill_base_radius_+
 |level20_stat1_value                     = 6
 |level20_stat2_id                        = active_skill_area_damage_+%_final
@@ -348,7 +348,7 @@
 |level21_level_requirement               = 72
 |level21_damage_effectiveness            = 277
 |level21_damage_multiplier               = 277.1
-|level21_stat_text                       = +7 to radius<br>20% less Area Damage<br>Aura grants 13 to 239 added Lightning Damage<br>Aura grants 20% chance to Shock
+|level21_stat_text                       = +0.7 metres to radius<br>20% less Area Damage<br>Aura grants 13 to 239 added Lightning Damage<br>Aura grants 20% chance to Shock
 |level21_stat1_id                        = active_skill_base_radius_+
 |level21_stat1_value                     = 7
 |level21_stat2_id                        = active_skill_area_damage_+%_final
@@ -363,7 +363,7 @@
 |level22_level_requirement               = 74
 |level22_damage_effectiveness            = 281
 |level22_damage_multiplier               = 281.1
-|level22_stat_text                       = +7 to radius<br>19% less Area Damage<br>Aura grants 14 to 258 added Lightning Damage<br>Aura grants 20% chance to Shock
+|level22_stat_text                       = +0.7 metres to radius<br>19% less Area Damage<br>Aura grants 14 to 258 added Lightning Damage<br>Aura grants 20% chance to Shock
 |level22_stat1_id                        = active_skill_base_radius_+
 |level22_stat1_value                     = 7
 |level22_stat2_id                        = active_skill_area_damage_+%_final
@@ -378,7 +378,7 @@
 |level23_level_requirement               = 76
 |level23_damage_effectiveness            = 285
 |level23_damage_multiplier               = 285.2
-|level23_stat_text                       = +7 to radius<br>19% less Area Damage<br>Aura grants 15 to 280 added Lightning Damage<br>Aura grants 21% chance to Shock
+|level23_stat_text                       = +0.7 metres to radius<br>19% less Area Damage<br>Aura grants 15 to 280 added Lightning Damage<br>Aura grants 21% chance to Shock
 |level23_stat1_id                        = active_skill_base_radius_+
 |level23_stat1_value                     = 7
 |level23_stat2_id                        = active_skill_area_damage_+%_final
@@ -393,7 +393,7 @@
 |level24_level_requirement               = 78
 |level24_damage_effectiveness            = 289
 |level24_damage_multiplier               = 289.2
-|level24_stat_text                       = +8 to radius<br>18% less Area Damage<br>Aura grants 16 to 302 added Lightning Damage<br>Aura grants 21% chance to Shock
+|level24_stat_text                       = +0.8 metres to radius<br>18% less Area Damage<br>Aura grants 16 to 302 added Lightning Damage<br>Aura grants 21% chance to Shock
 |level24_stat1_id                        = active_skill_base_radius_+
 |level24_stat1_value                     = 8
 |level24_stat2_id                        = active_skill_area_damage_+%_final
@@ -408,7 +408,7 @@
 |level25_level_requirement               = 80
 |level25_damage_effectiveness            = 293
 |level25_damage_multiplier               = 293.3
-|level25_stat_text                       = +8 to radius<br>18% less Area Damage<br>Aura grants 17 to 327 added Lightning Damage<br>Aura grants 22% chance to Shock
+|level25_stat_text                       = +0.8 metres to radius<br>18% less Area Damage<br>Aura grants 17 to 327 added Lightning Damage<br>Aura grants 22% chance to Shock
 |level25_stat1_id                        = active_skill_base_radius_+
 |level25_stat1_value                     = 8
 |level25_stat2_id                        = active_skill_area_damage_+%_final
@@ -423,7 +423,7 @@
 |level26_level_requirement               = 82
 |level26_damage_effectiveness            = 297
 |level26_damage_multiplier               = 297.3
-|level26_stat_text                       = +8 to radius<br>17% less Area Damage<br>Aura grants 19 to 353 added Lightning Damage<br>Aura grants 22% chance to Shock
+|level26_stat_text                       = +0.8 metres to radius<br>17% less Area Damage<br>Aura grants 19 to 353 added Lightning Damage<br>Aura grants 22% chance to Shock
 |level26_stat1_id                        = active_skill_base_radius_+
 |level26_stat1_value                     = 8
 |level26_stat2_id                        = active_skill_area_damage_+%_final
@@ -438,7 +438,7 @@
 |level27_level_requirement               = 84
 |level27_damage_effectiveness            = 301
 |level27_damage_multiplier               = 301.4
-|level27_stat_text                       = +8 to radius<br>17% less Area Damage<br>Aura grants 20 to 381 added Lightning Damage<br>Aura grants 23% chance to Shock
+|level27_stat_text                       = +0.8 metres to radius<br>17% less Area Damage<br>Aura grants 20 to 381 added Lightning Damage<br>Aura grants 23% chance to Shock
 |level27_stat1_id                        = active_skill_base_radius_+
 |level27_stat1_value                     = 8
 |level27_stat2_id                        = active_skill_area_damage_+%_final
@@ -453,7 +453,7 @@
 |level28_level_requirement               = 86
 |level28_damage_effectiveness            = 305
 |level28_damage_multiplier               = 305.4
-|level28_stat_text                       = +9 to radius<br>16% less Area Damage<br>Aura grants 22 to 411 added Lightning Damage<br>Aura grants 23% chance to Shock
+|level28_stat_text                       = +0.9 metres to radius<br>16% less Area Damage<br>Aura grants 22 to 411 added Lightning Damage<br>Aura grants 23% chance to Shock
 |level28_stat1_id                        = active_skill_base_radius_+
 |level28_stat1_value                     = 9
 |level28_stat2_id                        = active_skill_area_damage_+%_final
@@ -468,7 +468,7 @@
 |level29_level_requirement               = 88
 |level29_damage_effectiveness            = 309
 |level29_damage_multiplier               = 309.5
-|level29_stat_text                       = +9 to radius<br>16% less Area Damage<br>Aura grants 23 to 444 added Lightning Damage<br>Aura grants 24% chance to Shock
+|level29_stat_text                       = +0.9 metres to radius<br>16% less Area Damage<br>Aura grants 23 to 444 added Lightning Damage<br>Aura grants 24% chance to Shock
 |level29_stat1_id                        = active_skill_base_radius_+
 |level29_stat1_value                     = 9
 |level29_stat2_id                        = active_skill_area_damage_+%_final
@@ -483,7 +483,7 @@
 |level30_level_requirement               = 90
 |level30_damage_effectiveness            = 314
 |level30_damage_multiplier               = 313.5
-|level30_stat_text                       = +9 to radius<br>15% less Area Damage<br>Aura grants 25 to 479 added Lightning Damage<br>Aura grants 24% chance to Shock
+|level30_stat_text                       = +0.9 metres to radius<br>15% less Area Damage<br>Aura grants 25 to 479 added Lightning Damage<br>Aura grants 24% chance to Shock
 |level30_stat1_id                        = active_skill_base_radius_+
 |level30_stat1_value                     = 9
 |level30_stat2_id                        = active_skill_area_damage_+%_final
@@ -498,7 +498,7 @@
 |level31_level_requirement               = 91
 |level31_damage_effectiveness            = 316
 |level31_damage_multiplier               = 315.6
-|level31_stat_text                       = +10 to radius<br>15% less Area Damage<br>Aura grants 26 to 497 added Lightning Damage<br>Aura grants 24% chance to Shock
+|level31_stat_text                       = +1 metre to radius<br>15% less Area Damage<br>Aura grants 26 to 497 added Lightning Damage<br>Aura grants 24% chance to Shock
 |level31_stat1_id                        = active_skill_base_radius_+
 |level31_stat1_value                     = 10
 |level31_stat2_id                        = active_skill_area_damage_+%_final
@@ -513,7 +513,7 @@
 |level32_level_requirement               = 92
 |level32_damage_effectiveness            = 318
 |level32_damage_multiplier               = 317.6
-|level32_stat_text                       = +10 to radius<br>15% less Area Damage<br>Aura grants 27 to 516 added Lightning Damage<br>Aura grants 25% chance to Shock
+|level32_stat_text                       = +1 metre to radius<br>15% less Area Damage<br>Aura grants 27 to 516 added Lightning Damage<br>Aura grants 25% chance to Shock
 |level32_stat1_id                        = active_skill_base_radius_+
 |level32_stat1_value                     = 10
 |level32_stat2_id                        = active_skill_area_damage_+%_final
@@ -528,7 +528,7 @@
 |level33_level_requirement               = 93
 |level33_damage_effectiveness            = 320
 |level33_damage_multiplier               = 319.6
-|level33_stat_text                       = +10 to radius<br>15% less Area Damage<br>Aura grants 28 to 536 added Lightning Damage<br>Aura grants 25% chance to Shock
+|level33_stat_text                       = +1 metre to radius<br>15% less Area Damage<br>Aura grants 28 to 536 added Lightning Damage<br>Aura grants 25% chance to Shock
 |level33_stat1_id                        = active_skill_base_radius_+
 |level33_stat1_value                     = 10
 |level33_stat2_id                        = active_skill_area_damage_+%_final
@@ -543,7 +543,7 @@
 |level34_level_requirement               = 94
 |level34_damage_effectiveness            = 322
 |level34_damage_multiplier               = 321.6
-|level34_stat_text                       = +10 to radius<br>14% less Area Damage<br>Aura grants 29 to 556 added Lightning Damage<br>Aura grants 25% chance to Shock
+|level34_stat_text                       = +1 metre to radius<br>14% less Area Damage<br>Aura grants 29 to 556 added Lightning Damage<br>Aura grants 25% chance to Shock
 |level34_stat1_id                        = active_skill_base_radius_+
 |level34_stat1_value                     = 10
 |level34_stat2_id                        = active_skill_area_damage_+%_final
@@ -558,7 +558,7 @@
 |level35_level_requirement               = 95
 |level35_damage_effectiveness            = 324
 |level35_damage_multiplier               = 323.7
-|level35_stat_text                       = +10 to radius<br>14% less Area Damage<br>Aura grants 30 to 577 added Lightning Damage<br>Aura grants 25% chance to Shock
+|level35_stat_text                       = +1 metre to radius<br>14% less Area Damage<br>Aura grants 30 to 577 added Lightning Damage<br>Aura grants 25% chance to Shock
 |level35_stat1_id                        = active_skill_base_radius_+
 |level35_stat1_value                     = 10
 |level35_stat2_id                        = active_skill_area_damage_+%_final
@@ -573,7 +573,7 @@
 |level36_level_requirement               = 96
 |level36_damage_effectiveness            = 326
 |level36_damage_multiplier               = 325.7
-|level36_stat_text                       = +10 to radius<br>14% less Area Damage<br>Aura grants 32 to 599 added Lightning Damage<br>Aura grants 26% chance to Shock
+|level36_stat_text                       = +1 metre to radius<br>14% less Area Damage<br>Aura grants 32 to 599 added Lightning Damage<br>Aura grants 26% chance to Shock
 |level36_stat1_id                        = active_skill_base_radius_+
 |level36_stat1_value                     = 10
 |level36_stat2_id                        = active_skill_area_damage_+%_final
@@ -588,7 +588,7 @@
 |level37_level_requirement               = 97
 |level37_damage_effectiveness            = 328
 |level37_damage_multiplier               = 327.7
-|level37_stat_text                       = +11 to radius<br>14% less Area Damage<br>Aura grants 33 to 621 added Lightning Damage<br>Aura grants 26% chance to Shock
+|level37_stat_text                       = +1.1 metres to radius<br>14% less Area Damage<br>Aura grants 33 to 621 added Lightning Damage<br>Aura grants 26% chance to Shock
 |level37_stat1_id                        = active_skill_base_radius_+
 |level37_stat1_value                     = 11
 |level37_stat2_id                        = active_skill_area_damage_+%_final
@@ -603,7 +603,7 @@
 |level38_level_requirement               = 98
 |level38_damage_effectiveness            = 330
 |level38_damage_multiplier               = 329.7
-|level38_stat_text                       = +11 to radius<br>13% less Area Damage<br>Aura grants 34 to 645 added Lightning Damage<br>Aura grants 26% chance to Shock
+|level38_stat_text                       = +1.1 metres to radius<br>13% less Area Damage<br>Aura grants 34 to 645 added Lightning Damage<br>Aura grants 26% chance to Shock
 |level38_stat1_id                        = active_skill_base_radius_+
 |level38_stat1_value                     = 11
 |level38_stat2_id                        = active_skill_area_damage_+%_final
@@ -618,7 +618,7 @@
 |level39_level_requirement               = 99
 |level39_damage_effectiveness            = 332
 |level39_damage_multiplier               = 331.8
-|level39_stat_text                       = +11 to radius<br>13% less Area Damage<br>Aura grants 35 to 669 added Lightning Damage<br>Aura grants 26% chance to Shock
+|level39_stat_text                       = +1.1 metres to radius<br>13% less Area Damage<br>Aura grants 35 to 669 added Lightning Damage<br>Aura grants 26% chance to Shock
 |level39_stat1_id                        = active_skill_base_radius_+
 |level39_stat1_value                     = 11
 |level39_stat2_id                        = active_skill_area_damage_+%_final
@@ -633,7 +633,7 @@
 |level40_level_requirement               = 100
 |level40_damage_effectiveness            = 334
 |level40_damage_multiplier               = 333.8
-|level40_stat_text                       = +11 to radius<br>13% less Area Damage<br>Aura grants 37 to 694 added Lightning Damage<br>Aura grants 27% chance to Shock
+|level40_stat_text                       = +1.1 metres to radius<br>13% less Area Damage<br>Aura grants 37 to 694 added Lightning Damage<br>Aura grants 27% chance to Shock
 |level40_stat1_id                        = active_skill_base_radius_+
 |level40_stat1_value                     = 11
 |level40_stat2_id                        = active_skill_area_damage_+%_final