--- Sunder
+++ Export
@@ -39,9 +39,9 @@
 |static_attack_speed_multiplier          = 75
 |static_stat1_id                         = base_sunder_wave_delay_ms
 |static_stat1_value                      = 400
-|static_stat2_id                         = is_area_damage
+|static_stat2_id                         = console_skill_dont_chase
 |static_stat2_value                      = 1
-|static_stat3_id                         = console_skill_dont_chase
+|static_stat3_id                         = is_area_damage
 |static_stat3_value                      = 1
 |static_stat4_id                         = shockwave_slam_explosion_damage_+%_final
 |static_stat4_value                      = -40
