--- The Lord of Celebration
+++ Export
@@ -12,9 +12,9 @@
 |description                             = {{c|magic|Sceptre of Celebration}}<br>{{c|default|Shaper Item}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
+|card_background                         = 0
 |drop_areas                              = MapWorldsCrimsonTemple, MapWorldsResidence, MapWorldsIvoryTemple, MapWorldsCrimsonTownship, MapWorldsLaboratory, MapWorldsBasilica, MapWorldsPalace, MapWorldsConservatory, MapWorldsHauntedMansion, MapWorldsVault, MapWorldsVilla, MapWorldsChateau, MapWorldsTemple, MapWorldsRelicChambers, MapWorldsMoonTempleUnique, MapWorldsVaalPyramidUnique
 |release_version                         = 3.6.0
-|card_background                         = 0
 }}
 
 '''The Lord of Celebration''' is a [[divination card]]. A set of four can be exchanged for a random [[Shaper item|Shaper-influenced]] [[magic]] [[sceptre]] with the {{ml|LocalIncreasedAttackSpeed8}}<!-- of Celebration --> suffix.
