--- Eternal Sword
+++ Export
@@ -16,7 +16,7 @@
 |attack_speed                            = 1.5
 |physical_damage_min                     = 41
 |physical_damage_max                     = 68
-|weapon_range                            = 11
+|weapon_range                            = 1.1
 |recipe1_part1_item_name                 = The King's Blade
 |recipe1_part1_amount                    = 5
 |recipe1_description                     = ilvl 66 {{ml|LocalIncreasedPhysicalDamagePercent5}} {{c|magic|Eternal Sword}}