--- The Enforcer
+++ Export
@@ -12,8 +12,8 @@
 |description                             = {{c|magic|Spiked Gloves of the Conquest (Culling Strike)}}<br>{{c|default|Item Level:}} {{c|normal|86}}<br>{{c|default|Warlord Item}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
+|card_background                         = 5
 |release_version                         = 3.19.0
-|card_background                         = 5
 }}
 
 '''The Enforcer''' is a [[divination card]]. A set of four can be exchanged for a pair of [[Warlord item|Warlord-influenced]] [[magic]] {{il|Spiked Gloves}} with the {{ml|CullingStrikeInfluence1}} suffix and [[item level]] 86.
