--- Wiki
+++ Export
@@ -13,7 +13,7 @@
 |stack_size_currency_tab                 = 5000
 |essence_level                           = 8
 |essence_type                            = 6
-|essence_category                        = Essence of Delirium
+|essence_category                        = Essence of Delerium
 |essence_monster_modifier_ids            = EssenceDaemonModLifeAndDamageCorrupt, EssenceInsectSummonedOnDeath, DecayOnHitEssenceMonster1
 |is_essence                              = True
 |release_version                         = 2.4.0
