--- Silver Flask
+++ Export
@@ -12,7 +12,7 @@
 |required_level                          = 27
 |flask_duration                          = 6
 |buff_id                                 = flask_utility_haste
-|buff_stat_text                          = Onslaught
+|buff_stat_text                          = 
 |charges_max                             = 60
 |charges_per_use                         = 40
 |release_version                         = 2.2.0
