--- Tangled Fossil
+++ Export
@@ -8,7 +8,7 @@
 |tags                                    = currency, default
 |metadata_id                             = Metadata/Items/Currency/CurrencyDelveCraftingRandom
 |stack_size                              = 10
-|description                             = Makes a random modifier type much more likely and prevents <br>another random modifier type. Effects revealed once resonator is<br>fully socketed.
+|description                             = Makes a random modifier type much more likely and prevents <br>another random modifier type<br>Effects revealed once resonator is fully socketed
 |help_text                               = Place in a Resonator to influence item crafting.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |drop_text                               = Can be dropped from [[Haunted Tomb]] node, spawnable only in Fungal Caverns biome
