--- Righteous Fire
+++ Export
@@ -41,7 +41,7 @@
 |static_stat3_value                      = 5400
 |static_stat4_id                         = base_nonlethal_fire_damage_%_of_maximum_energy_shield_taken_per_minute
 |static_stat4_value                      = 4200
-|stat_text                               = Deals (39-2447.9) Base Fire Damage per second<br>+(0-5) to radius<br>Deals 35% of your Maximum Life as Base Fire Damage per second<br>Deals 35% of your Maximum Energy Shield as Base Fire Damage per second<br>You Burn for 90% of your Maximum Life per second as Fire Damage<br>You Burn for 70% of your Maximum Energy Shield per second as Fire Damage<br>Grants (20-39)% more Spell Damage
+|stat_text                               = Deals (39-2447.9) Base Fire Damage per second<br>+(0-0.5) metres to radius<br>Deals 35% of your Maximum Life as Base Fire Damage per second<br>Deals 35% of your Maximum Energy Shield as Base Fire Damage per second<br>You Burn for 90% of your Maximum Life per second as Fire Damage<br>You Burn for 70% of your Maximum Energy Shield per second as Fire Damage<br>Grants (20-39)% more Spell Damage
 |level1                                  = True
 |level1_level_requirement                = 16
 |level1_stat_text                        = Deals 39 Base Fire Damage per second<br>Grants 20% more Spell Damage
@@ -65,7 +65,7 @@
 |level3_stat2_value                      = 22
 |level4                                  = True
 |level4_level_requirement                = 28
-|level4_stat_text                        = Deals 111.7 Base Fire Damage per second<br>+1 to radius<br>Grants 23% more Spell Damage
+|level4_stat_text                        = Deals 111.7 Base Fire Damage per second<br>+0.1 metres to radius<br>Grants 23% more Spell Damage
 |level4_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level4_stat1_value                      = 6699
 |level4_stat2_id                         = active_skill_base_radius_+
@@ -74,7 +74,7 @@
 |level4_stat3_value                      = 23
 |level5                                  = True
 |level5_level_requirement                = 31
-|level5_stat_text                        = Deals 142.4 Base Fire Damage per second<br>+1 to radius<br>Grants 24% more Spell Damage
+|level5_stat_text                        = Deals 142.4 Base Fire Damage per second<br>+0.1 metres to radius<br>Grants 24% more Spell Damage
 |level5_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level5_stat1_value                      = 8547
 |level5_stat2_id                         = active_skill_base_radius_+
@@ -83,7 +83,7 @@
 |level5_stat3_value                      = 24
 |level6                                  = True
 |level6_level_requirement                = 34
-|level6_stat_text                        = Deals 180.8 Base Fire Damage per second<br>+1 to radius<br>Grants 25% more Spell Damage
+|level6_stat_text                        = Deals 180.8 Base Fire Damage per second<br>+0.1 metres to radius<br>Grants 25% more Spell Damage
 |level6_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level6_stat1_value                      = 10845
 |level6_stat2_id                         = active_skill_base_radius_+
@@ -92,7 +92,7 @@
 |level6_stat3_value                      = 25
 |level7                                  = True
 |level7_level_requirement                = 37
-|level7_stat_text                        = Deals 228.3 Base Fire Damage per second<br>+1 to radius<br>Grants 26% more Spell Damage
+|level7_stat_text                        = Deals 228.3 Base Fire Damage per second<br>+0.1 metres to radius<br>Grants 26% more Spell Damage
 |level7_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level7_stat1_value                      = 13696
 |level7_stat2_id                         = active_skill_base_radius_+
@@ -101,7 +101,7 @@
 |level7_stat3_value                      = 26
 |level8                                  = True
 |level8_level_requirement                = 40
-|level8_stat_text                        = Deals 287.1 Base Fire Damage per second<br>+2 to radius<br>Grants 27% more Spell Damage
+|level8_stat_text                        = Deals 287.1 Base Fire Damage per second<br>+0.2 metres to radius<br>Grants 27% more Spell Damage
 |level8_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level8_stat1_value                      = 17226
 |level8_stat2_id                         = active_skill_base_radius_+
@@ -110,7 +110,7 @@
 |level8_stat3_value                      = 27
 |level9                                  = True
 |level9_level_requirement                = 43
-|level9_stat_text                        = Deals 359.8 Base Fire Damage per second<br>+2 to radius<br>Grants 28% more Spell Damage
+|level9_stat_text                        = Deals 359.8 Base Fire Damage per second<br>+0.2 metres to radius<br>Grants 28% more Spell Damage
 |level9_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level9_stat1_value                      = 21587
 |level9_stat2_id                         = active_skill_base_radius_+
@@ -119,7 +119,7 @@
 |level9_stat3_value                      = 28
 |level10                                 = True
 |level10_level_requirement               = 46
-|level10_stat_text                       = Deals 449.4 Base Fire Damage per second<br>+2 to radius<br>Grants 29% more Spell Damage
+|level10_stat_text                       = Deals 449.4 Base Fire Damage per second<br>+0.2 metres to radius<br>Grants 29% more Spell Damage
 |level10_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level10_stat1_value                     = 26964
 |level10_stat2_id                        = active_skill_base_radius_+
@@ -128,7 +128,7 @@
 |level10_stat3_value                     = 29
 |level11                                 = True
 |level11_level_requirement               = 49
-|level11_stat_text                       = Deals 559.7 Base Fire Damage per second<br>+2 to radius<br>Grants 30% more Spell Damage
+|level11_stat_text                       = Deals 559.7 Base Fire Damage per second<br>+0.2 metres to radius<br>Grants 30% more Spell Damage
 |level11_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level11_stat1_value                     = 33584
 |level11_stat2_id                        = active_skill_base_radius_+
@@ -137,7 +137,7 @@
 |level11_stat3_value                     = 30
 |level12                                 = True
 |level12_level_requirement               = 52
-|level12_stat_text                       = Deals 695.3 Base Fire Damage per second<br>+3 to radius<br>Grants 31% more Spell Damage
+|level12_stat_text                       = Deals 695.3 Base Fire Damage per second<br>+0.3 metres to radius<br>Grants 31% more Spell Damage
 |level12_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level12_stat1_value                     = 41720
 |level12_stat2_id                        = active_skill_base_radius_+
@@ -146,7 +146,7 @@
 |level12_stat3_value                     = 31
 |level13                                 = True
 |level13_level_requirement               = 55
-|level13_stat_text                       = Deals 861.8 Base Fire Damage per second<br>+3 to radius<br>Grants 32% more Spell Damage
+|level13_stat_text                       = Deals 861.8 Base Fire Damage per second<br>+0.3 metres to radius<br>Grants 32% more Spell Damage
 |level13_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level13_stat1_value                     = 51705
 |level13_stat2_id                        = active_skill_base_radius_+
@@ -155,7 +155,7 @@
 |level13_stat3_value                     = 32
 |level14                                 = True
 |level14_level_requirement               = 58
-|level14_stat_text                       = Deals 1065.7 Base Fire Damage per second<br>+3 to radius<br>Grants 33% more Spell Damage
+|level14_stat_text                       = Deals 1065.7 Base Fire Damage per second<br>+0.3 metres to radius<br>Grants 33% more Spell Damage
 |level14_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level14_stat1_value                     = 63942
 |level14_stat2_id                        = active_skill_base_radius_+
@@ -164,7 +164,7 @@
 |level14_stat3_value                     = 33
 |level15                                 = True
 |level15_level_requirement               = 60
-|level15_stat_text                       = Deals 1226.5 Base Fire Damage per second<br>+3 to radius<br>Grants 34% more Spell Damage
+|level15_stat_text                       = Deals 1226.5 Base Fire Damage per second<br>+0.3 metres to radius<br>Grants 34% more Spell Damage
 |level15_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level15_stat1_value                     = 73588
 |level15_stat2_id                        = active_skill_base_radius_+
@@ -173,7 +173,7 @@
 |level15_stat3_value                     = 34
 |level16                                 = True
 |level16_level_requirement               = 62
-|level16_stat_text                       = Deals 1410.3 Base Fire Damage per second<br>+4 to radius<br>Grants 35% more Spell Damage
+|level16_stat_text                       = Deals 1410.3 Base Fire Damage per second<br>+0.4 metres to radius<br>Grants 35% more Spell Damage
 |level16_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level16_stat1_value                     = 84621
 |level16_stat2_id                        = active_skill_base_radius_+
@@ -182,7 +182,7 @@
 |level16_stat3_value                     = 35
 |level17                                 = True
 |level17_level_requirement               = 64
-|level17_stat_text                       = Deals 1620.5 Base Fire Damage per second<br>+4 to radius<br>Grants 36% more Spell Damage
+|level17_stat_text                       = Deals 1620.5 Base Fire Damage per second<br>+0.4 metres to radius<br>Grants 36% more Spell Damage
 |level17_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level17_stat1_value                     = 97232
 |level17_stat2_id                        = active_skill_base_radius_+
@@ -191,7 +191,7 @@
 |level17_stat3_value                     = 36
 |level18                                 = True
 |level18_level_requirement               = 66
-|level18_stat_text                       = Deals 1860.7 Base Fire Damage per second<br>+4 to radius<br>Grants 37% more Spell Damage
+|level18_stat_text                       = Deals 1860.7 Base Fire Damage per second<br>+0.4 metres to radius<br>Grants 37% more Spell Damage
 |level18_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level18_stat1_value                     = 111640
 |level18_stat2_id                        = active_skill_base_radius_+
@@ -200,7 +200,7 @@
 |level18_stat3_value                     = 37
 |level19                                 = True
 |level19_level_requirement               = 68
-|level19_stat_text                       = Deals 2134.9 Base Fire Damage per second<br>+4 to radius<br>Grants 38% more Spell Damage
+|level19_stat_text                       = Deals 2134.9 Base Fire Damage per second<br>+0.4 metres to radius<br>Grants 38% more Spell Damage
 |level19_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level19_stat1_value                     = 128094
 |level19_stat2_id                        = active_skill_base_radius_+
@@ -209,7 +209,7 @@
 |level19_stat3_value                     = 38
 |level20                                 = True
 |level20_level_requirement               = 70
-|level20_stat_text                       = Deals 2447.9 Base Fire Damage per second<br>+5 to radius<br>Grants 39% more Spell Damage
+|level20_stat_text                       = Deals 2447.9 Base Fire Damage per second<br>+0.5 metres to radius<br>Grants 39% more Spell Damage
 |level20_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level20_stat1_value                     = 146876
 |level20_stat2_id                        = active_skill_base_radius_+
@@ -218,7 +218,7 @@
 |level20_stat3_value                     = 39
 |level21                                 = True
 |level21_level_requirement               = 72
-|level21_stat_text                       = Deals 2805.1 Base Fire Damage per second<br>+5 to radius<br>Grants 40% more Spell Damage
+|level21_stat_text                       = Deals 2805.1 Base Fire Damage per second<br>+0.5 metres to radius<br>Grants 40% more Spell Damage
 |level21_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level21_stat1_value                     = 168306
 |level21_stat2_id                        = active_skill_base_radius_+
@@ -227,7 +227,7 @@
 |level21_stat3_value                     = 40
 |level22                                 = True
 |level22_level_requirement               = 74
-|level22_stat_text                       = Deals 3212.4 Base Fire Damage per second<br>+5 to radius<br>Grants 41% more Spell Damage
+|level22_stat_text                       = Deals 3212.4 Base Fire Damage per second<br>+0.5 metres to radius<br>Grants 41% more Spell Damage
 |level22_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level22_stat1_value                     = 192747
 |level22_stat2_id                        = active_skill_base_radius_+
@@ -236,7 +236,7 @@
 |level22_stat3_value                     = 41
 |level23                                 = True
 |level23_level_requirement               = 76
-|level23_stat_text                       = Deals 3676.9 Base Fire Damage per second<br>+5 to radius<br>Grants 42% more Spell Damage
+|level23_stat_text                       = Deals 3676.9 Base Fire Damage per second<br>+0.5 metres to radius<br>Grants 42% more Spell Damage
 |level23_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level23_stat1_value                     = 220612
 |level23_stat2_id                        = active_skill_base_radius_+
@@ -245,7 +245,7 @@
 |level23_stat3_value                     = 42
 |level24                                 = True
 |level24_level_requirement               = 78
-|level24_stat_text                       = Deals 4206.1 Base Fire Damage per second<br>+6 to radius<br>Grants 43% more Spell Damage
+|level24_stat_text                       = Deals 4206.1 Base Fire Damage per second<br>+0.6 metres to radius<br>Grants 43% more Spell Damage
 |level24_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level24_stat1_value                     = 252369
 |level24_stat2_id                        = active_skill_base_radius_+
@@ -254,7 +254,7 @@
 |level24_stat3_value                     = 43
 |level25                                 = True
 |level25_level_requirement               = 80
-|level25_stat_text                       = Deals 4809.1 Base Fire Damage per second<br>+6 to radius<br>Grants 44% more Spell Damage
+|level25_stat_text                       = Deals 4809.1 Base Fire Damage per second<br>+0.6 metres to radius<br>Grants 44% more Spell Damage
 |level25_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level25_stat1_value                     = 288546
 |level25_stat2_id                        = active_skill_base_radius_+
@@ -263,7 +263,7 @@
 |level25_stat3_value                     = 44
 |level26                                 = True
 |level26_level_requirement               = 82
-|level26_stat_text                       = Deals 5495.8 Base Fire Damage per second<br>+6 to radius<br>Grants 45% more Spell Damage
+|level26_stat_text                       = Deals 5495.8 Base Fire Damage per second<br>+0.6 metres to radius<br>Grants 45% more Spell Damage
 |level26_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level26_stat1_value                     = 329746
 |level26_stat2_id                        = active_skill_base_radius_+
@@ -272,7 +272,7 @@
 |level26_stat3_value                     = 45
 |level27                                 = True
 |level27_level_requirement               = 84
-|level27_stat_text                       = Deals 6277.5 Base Fire Damage per second<br>+6 to radius<br>Grants 46% more Spell Damage
+|level27_stat_text                       = Deals 6277.5 Base Fire Damage per second<br>+0.6 metres to radius<br>Grants 46% more Spell Damage
 |level27_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level27_stat1_value                     = 376650
 |level27_stat2_id                        = active_skill_base_radius_+
@@ -281,7 +281,7 @@
 |level27_stat3_value                     = 46
 |level28                                 = True
 |level28_level_requirement               = 86
-|level28_stat_text                       = Deals 7167.2 Base Fire Damage per second<br>+7 to radius<br>Grants 47% more Spell Damage
+|level28_stat_text                       = Deals 7167.2 Base Fire Damage per second<br>+0.7 metres to radius<br>Grants 47% more Spell Damage
 |level28_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level28_stat1_value                     = 430030
 |level28_stat2_id                        = active_skill_base_radius_+
@@ -290,7 +290,7 @@
 |level28_stat3_value                     = 47
 |level29                                 = True
 |level29_level_requirement               = 88
-|level29_stat_text                       = Deals 8179.4 Base Fire Damage per second<br>+7 to radius<br>Grants 48% more Spell Damage
+|level29_stat_text                       = Deals 8179.4 Base Fire Damage per second<br>+0.7 metres to radius<br>Grants 48% more Spell Damage
 |level29_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level29_stat1_value                     = 490761
 |level29_stat2_id                        = active_skill_base_radius_+
@@ -299,7 +299,7 @@
 |level29_stat3_value                     = 48
 |level30                                 = True
 |level30_level_requirement               = 90
-|level30_stat_text                       = Deals 9330.6 Base Fire Damage per second<br>+7 to radius<br>Grants 49% more Spell Damage
+|level30_stat_text                       = Deals 9330.6 Base Fire Damage per second<br>+0.7 metres to radius<br>Grants 49% more Spell Damage
 |level30_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level30_stat1_value                     = 559834
 |level30_stat2_id                        = active_skill_base_radius_+
@@ -308,7 +308,7 @@
 |level30_stat3_value                     = 49
 |level31                                 = True
 |level31_level_requirement               = 91
-|level31_stat_text                       = Deals 9964.1 Base Fire Damage per second<br>+7 to radius<br>Grants 49% more Spell Damage
+|level31_stat_text                       = Deals 9964.1 Base Fire Damage per second<br>+0.7 metres to radius<br>Grants 49% more Spell Damage
 |level31_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level31_stat1_value                     = 597844
 |level31_stat2_id                        = active_skill_base_radius_+
@@ -317,7 +317,7 @@
 |level31_stat3_value                     = 49
 |level32                                 = True
 |level32_level_requirement               = 92
-|level32_stat_text                       = Deals 10639.5 Base Fire Damage per second<br>+8 to radius<br>Grants 50% more Spell Damage
+|level32_stat_text                       = Deals 10639.5 Base Fire Damage per second<br>+0.8 metres to radius<br>Grants 50% more Spell Damage
 |level32_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level32_stat1_value                     = 638372
 |level32_stat2_id                        = active_skill_base_radius_+
@@ -326,7 +326,7 @@
 |level32_stat3_value                     = 50
 |level33                                 = True
 |level33_level_requirement               = 93
-|level33_stat_text                       = Deals 11359.7 Base Fire Damage per second<br>+8 to radius<br>Grants 50% more Spell Damage
+|level33_stat_text                       = Deals 11359.7 Base Fire Damage per second<br>+0.8 metres to radius<br>Grants 50% more Spell Damage
 |level33_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level33_stat1_value                     = 681582
 |level33_stat2_id                        = active_skill_base_radius_+
@@ -335,7 +335,7 @@
 |level33_stat3_value                     = 50
 |level34                                 = True
 |level34_level_requirement               = 94
-|level34_stat_text                       = Deals 12127.5 Base Fire Damage per second<br>+8 to radius<br>Grants 51% more Spell Damage
+|level34_stat_text                       = Deals 12127.5 Base Fire Damage per second<br>+0.8 metres to radius<br>Grants 51% more Spell Damage
 |level34_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level34_stat1_value                     = 727648
 |level34_stat2_id                        = active_skill_base_radius_+
@@ -344,7 +344,7 @@
 |level34_stat3_value                     = 51
 |level35                                 = True
 |level35_level_requirement               = 95
-|level35_stat_text                       = Deals 12945.9 Base Fire Damage per second<br>+8 to radius<br>Grants 51% more Spell Damage
+|level35_stat_text                       = Deals 12945.9 Base Fire Damage per second<br>+0.8 metres to radius<br>Grants 51% more Spell Damage
 |level35_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level35_stat1_value                     = 776755
 |level35_stat2_id                        = active_skill_base_radius_+
@@ -353,7 +353,7 @@
 |level35_stat3_value                     = 51
 |level36                                 = True
 |level36_level_requirement               = 96
-|level36_stat_text                       = Deals 13818.4 Base Fire Damage per second<br>+9 to radius<br>Grants 52% more Spell Damage
+|level36_stat_text                       = Deals 13818.4 Base Fire Damage per second<br>+0.9 metres to radius<br>Grants 52% more Spell Damage
 |level36_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level36_stat1_value                     = 829101
 |level36_stat2_id                        = active_skill_base_radius_+
@@ -362,7 +362,7 @@
 |level36_stat3_value                     = 52
 |level37                                 = True
 |level37_level_requirement               = 97
-|level37_stat_text                       = Deals 14748.2 Base Fire Damage per second<br>+9 to radius<br>Grants 52% more Spell Damage
+|level37_stat_text                       = Deals 14748.2 Base Fire Damage per second<br>+0.9 metres to radius<br>Grants 52% more Spell Damage
 |level37_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level37_stat1_value                     = 884895
 |level37_stat2_id                        = active_skill_base_radius_+
@@ -371,7 +371,7 @@
 |level37_stat3_value                     = 52
 |level38                                 = True
 |level38_level_requirement               = 98
-|level38_stat_text                       = Deals 15739.4 Base Fire Damage per second<br>+9 to radius<br>Grants 53% more Spell Damage
+|level38_stat_text                       = Deals 15739.4 Base Fire Damage per second<br>+0.9 metres to radius<br>Grants 53% more Spell Damage
 |level38_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level38_stat1_value                     = 944361
 |level38_stat2_id                        = active_skill_base_radius_+
@@ -380,7 +380,7 @@
 |level38_stat3_value                     = 53
 |level39                                 = True
 |level39_level_requirement               = 99
-|level39_stat_text                       = Deals 16795.6 Base Fire Damage per second<br>+9 to radius<br>Grants 53% more Spell Damage
+|level39_stat_text                       = Deals 16795.6 Base Fire Damage per second<br>+0.9 metres to radius<br>Grants 53% more Spell Damage
 |level39_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level39_stat1_value                     = 1007737
 |level39_stat2_id                        = active_skill_base_radius_+
@@ -389,7 +389,7 @@
 |level39_stat3_value                     = 53
 |level40                                 = True
 |level40_level_requirement               = 100
-|level40_stat_text                       = Deals 17921.2 Base Fire Damage per second<br>+10 to radius<br>Grants 54% more Spell Damage
+|level40_stat_text                       = Deals 17921.2 Base Fire Damage per second<br>+1 metre to radius<br>Grants 54% more Spell Damage
 |level40_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level40_stat1_value                     = 1075274
 |level40_stat2_id                        = active_skill_base_radius_+