--- Wiki
+++ Export
@@ -32,10 +32,10 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = emerge_speed_+%
-|static_stat1_value                      = 100
-|static_stat2_id                         = display_minion_monster_type
-|static_stat2_value                      = 3
+|static_stat1_id                         = display_minion_monster_type
+|static_stat1_value                      = 3
+|static_stat2_id                         = emerge_speed_+%
+|static_stat2_value                      = 100
 |stat_text                               = (0-76)% increased Minion Maximum Life<br>Can use Items requiring up to Level (33-100)<br>Minions' Attacks deal (32-126) to (47-184) additional Physical Damage<br>Minions deal (0-76)% increased Physical Damage with Melee Attacks
 |level1                                  = True
 |level1_level_requirement                = 28
