--- The Unexpected Prize
+++ Export
@@ -6,7 +6,7 @@
 |size_y                                  = 1
 |flavour_text                            = They came from far and wide under a common banner.<br>They sought companionship and competition,<br>but found a treasure none had foreseen.
 |drop_level                              = 68
-|tags                                    = jewellery_divination, unique_divination, unique_jewellery_divination, divination_card, default
+|tags                                    = jewellery_divination, unique_divination, unique_jewellery_divination, unique_corrupted_divination, unique_corrupted_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardTheUnexpectedPrize
 |stack_size                              = 2
 |description                             = {{c|unique|Attribute Transforming Jewel}}<br>{{c|corrupted|[[Corrupted]]}}
