--- Contract: The Vinderi Bomb
+++ Export
@@ -8,7 +8,6 @@
 |tags                                    = default
 |metadata_id                             = Metadata/Items/Heist/QuestContracts/HeistContractQuestVinderi2
 |inventory_icon                          = Generic NPC Quest Contract
-|flavour_text                            =  "I wish I could say this was the first time I've forgotten to restock on this thing.<br />I forget what it was, but I'm sure I'll remember by the time we need to get it.<br />Was it beetles? Bottles? I think it began with 'B'.<br />Or it had no 'B'."
 }}
 {{stub}}
 '''Contract: The Vinderi Bomb''' is a [[quest]] [[contract]]. The client is [[Vinderi]]. It is the second quest of his personal storyline. The area level of this contract is depended on the area level of the Smuggler's Caches.
