--- Upgrade Scroll S19
+++ Export
@@ -0,0 +1,12 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Upgrade Scroll S19
+|class_id                                = Microtransaction
+|size_x                                  = 1
+|size_y                                  = 1
+|tags                                    = default
+|metadata_id                             = Metadata/Items/MicrotransactionCurrency/MicrotransactionUpgradeScrollS19
+|stack_size                              = 1
+|description                             = Can be used with an Upgradeable Microtransaction
+|help_text                               = Combine this item with an Upgradeable Microtransaction to use it
+}}