--- Primeval Remnant
+++ Export
@@ -15,9 +15,9 @@
 |implicit1                               = CrucibleMapGrantsCombiningForgeImplicit
 |drop_text                               = Obtained by opening a {{il|Igneous Geode}}.
 |is_drop_restricted                      = true
+|is_in_game                              = false
 |is_unmodifiable                         = true
 |release_version                         = 3.21.0
-|is_in_game                              = false
 |removal_version                         = 3.21.2c
 |implicit1_text                          = Contains a Forge that can Combine Crucible Passive Skill Trees <!--CrucibleMapGrantsCombiningForgeImplicit-->
 |recipe1_description                     = {{c|rare|Primeval Remnant}}
