--- Melee Physical Damage Support
+++ Export
@@ -18,7 +18,7 @@
 |quality_type1_stat1_id                  = melee_physical_damage_+%
 |quality_type1_stat1_value               = 500
 |quality_type2_weight                    = 10
-|quality_type2_stat_text                 = Supported Skills have +0.1 to Melee Strike Range
+|quality_type2_stat_text                 = Supported Skills have +0.01 metres to Melee Strike Range
 |quality_type2_stat1_id                  = melee_range_+
 |quality_type2_stat1_value               = 100
 |quality_type3_weight                    = 50