--- Tattoo of the Tukohama Warmonger
+++ Export
@@ -10,7 +10,6 @@
 |stack_size                              = 10
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
-|tattoo_target                           = Small Strength
-|tattoo_stat_text                        = +1 to Melee Strike Range
-|description                             = Replaces a Small Strength Passive Skill<br>Grants "+1 to Melee Strike Range"
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Strength}} {{c|default|Passive Skill}}
+|description                             = Grants "+1 to Melee Strike Range"
 }}