--- Assassin's Favour
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = weapon_divination, unique_divination, unique_weapon_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardAssassinsFavour
 |stack_size                              = 9
-|description                             = {{c|unique|Dagger}}
+|description                             = {{c|unique|[[Dagger]]}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_areas                              = 1_3_5, 2_8_8, MapAtlasArcade, MapAtlasBazaar, MapAtlasQuay, MapWorldsAlleyways, MapWorldsBazaar, MapWorldsArcade, MapWorldsPort
