--- Wiki
+++ Export
@@ -0,0 +1,11 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Observer Diamond Flask Effect
+|class_id                                = Microtransaction
+|size_x                                  = 2
+|size_y                                  = 2
+|tags                                    = not_for_sale, default
+|metadata_id                             = Metadata/Items/MicrotransactionItemEffects/MicrotransactionObserverDiamondFlaskEffect
+|stack_size                              = 1
+|description                             = Activating your Diamond Flask summons a tentacled abomination that fires its eye lasers at enemies you critically strike while the flask is active.
+}}