--- Vaal Lightning Arrow
+++ Export
@@ -24,15 +24,15 @@
 |static_vaal_stored_uses                 = 2
 |static_vaal_soul_gain_prevention_time   = 8
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = projectile_random_angle_based_on_distance_to_target_location_%
-|static_stat1_value                      = 35
-|static_stat2_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat1_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat1_value                      = 1
+|static_stat2_id                         = cannot_cancel_skill_before_contact_point
 |static_stat2_value                      = 1
-|static_stat3_id                         = skill_can_fire_arrows
-|static_stat3_value                      = 1
-|static_stat4_id                         = cannot_cancel_skill_before_contact_point
+|static_stat3_id                         = projectile_random_angle_based_on_distance_to_target_location_%
+|static_stat3_value                      = 35
+|static_stat4_id                         = skill_can_add_multiple_charges_per_action
 |static_stat4_value                      = 1
-|static_stat5_id                         = skill_can_add_multiple_charges_per_action
+|static_stat5_id                         = skill_can_fire_arrows
 |static_stat5_value                      = 1
 |static_stat6_id                         = always_pierce
 |static_stat6_value                      = 1
