--- Ranger Bow
+++ Export
@@ -14,7 +14,7 @@
 |attack_speed                            = 1.3
 |physical_damage_min                     = 56
 |physical_damage_max                     = 117
-|weapon_range                            = 120
+|weapon_range                            = 12
 |quest_reward1_type                      = quest
 |quest_reward1_quest                     = The Storm Blade
 |quest_reward1_quest_id                  = a9q3