--- Tattoo of the Tawhoa Shaman
+++ Export
@@ -11,10 +11,11 @@
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |tattoo_target                           = Small Intelligence
-|tattoo_stat_text                        = 2% of Damage taken Recouped as [[Life]]
-|description                             = Replaces a Small Intelligence Passive Skill<br>Grants "2% of Damage taken Recouped as [[Life]]"
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Intelligence}} {{c|default|Passive Skill}}
+|tattoo_stat_text                        = 2% of Damage taken Recouped as Life
+|description                             = Grants "2% of Damage taken Recouped as [[Life]]"
 |inventory_icon                          = Tattoo intelligence 1
-|drop_text                               =  Can be obtained as a reward for defeating the [[Tawhoa]] tribe in the [[Trial of the Ancestors]].
+|drop_text                               = Can be obtained as a reward for defeating the [[Tawhoa]] tribe in the [[Trial of the Ancestors]].
 |is_drop_restricted                      = true
 |release_version                         = 3.22.0
 }}
