--- Wiki
+++ Export
@@ -37,9 +37,9 @@
 |static_cost_amounts                     = 10
 |static_cost_types                       = Mana
 |static_attack_speed_multiplier          = 80
-|static_stat1_id                         = is_area_damage
+|static_stat1_id                         = console_skill_dont_chase
 |static_stat1_value                      = 1
-|static_stat2_id                         = console_skill_dont_chase
+|static_stat2_id                         = is_area_damage
 |static_stat2_value                      = 1
 |static_stat3_id                         = skill_physical_damage_%_to_convert_to_fire
 |static_stat3_value                      = 60
