--- Wiki
+++ Export
@@ -6,7 +6,6 @@
 |size_y                                  = 1
 |flavour_text                            = When the outcome is the same, does it matter if it is fortune or trickery?
 |drop_level                              = 74
-|drop_level_maximum                      = 74
 |tags                                    = gives_other_cards_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardLuckyDeck
 |stack_size                              = 9
