--- Siege Ballista
+++ Export
@@ -37,12 +37,12 @@
 |quality_type4_stat1_value               = 1000
 |static_cost_types                       = Mana
 |static_attack_speed_multiplier          = 50
-|static_stat1_id                         = base_totem_range
-|static_stat1_value                      = 80
-|static_stat2_id                         = base_is_projectile
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = base_skill_is_totemified
 |static_stat2_value                      = 1
-|static_stat3_id                         = base_skill_is_totemified
-|static_stat3_value                      = 1
+|static_stat3_id                         = base_totem_range
+|static_stat3_value                      = 80
 |static_stat4_id                         = skill_can_fire_arrows
 |static_stat4_value                      = 1
 |static_stat5_id                         = base_totem_duration
