--- Frenzy
+++ Export
@@ -12,7 +12,6 @@
 |gem_tags                                = Attack, Projectile, Melee, Strike, Bow
 |gem_description                         = Performs an attack that gives the character a frenzy charge if it hits. Frenzy charges increase your attack speed.
 |active_skill_name                       = Frenzy
-|item_class_id_restriction               = One Hand Sword, Thrusting One Hand Sword, Two Hand Sword, One Hand Axe, Two Hand Axe, One Hand Mace, Sceptre, Two Hand Mace, Staff, Dagger, Claw, Unarmed, Warstaff, Rune Dagger
 |skill_id                                = Frenzy
 |cast_time                               = 1.0
 |required_level                          = 16