--- Sacred Crystallised Lifeforce
+++ Export
@@ -11,8 +11,8 @@
 |help_text                               = Can be used at the Horticrafting bench in your hideout.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |drop_monsters                           = Metadata/Monsters/LeagueHarvest/Oshabi/OshabiBoss
+|is_sellable                             = false
 |release_version                         = 3.19.0
-|is_sellable                             = false
 }}
 
 {{Item acquisition}}