--- The Astromancer
+++ Export
@@ -12,10 +12,10 @@
 |description                             = {{c|unique|{{il|html=|The Eternity Shroud}}}}<br>{{c|corrupted|[[Two-Implicit]]}}<br>{{c|corrupted|[[Corrupted]]}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
+|card_background                         = 0,1
 |drop_text                               = May also drop from Flesh Sculptor due to the "heist_science_miniboss_monster" tag. {{speculation}}
 |drop_monsters                           = Metadata/Monsters/LeagueHeist/Science/ThaumaturgistFemaleMiniBoss
 |release_version                         = 3.13.0
-|card_background                         = 0,1
 }}
 
 '''The Astromancer''' is a [[divination card]]. A set of seven can be exchanged for a [[corrupted]] {{il|The Eternity Shroud}} with two [[corruption implicit modifier]]s.
