--- Energy Blade (One Handed Sword)
+++ Export
@@ -17,7 +17,7 @@
 |attack_speed                            = 1.7
 |physical_damage_min                     = 0
 |physical_damage_max                     = 0
-|weapon_range                            = 11
+|weapon_range                            = 1.1
 |inventory_icon                          = Energy Blade (One Handed Sword)
 |drop_enabled                            = false
 |release_version                         = 3.16.0