--- Wiki
+++ Export
@@ -39,11 +39,11 @@
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
 |static_duration                         = 3
-|static_stat1_id                         = number_of_monsters_to_summon
+|static_stat1_id                         = base_is_projectile
 |static_stat1_value                      = 1
 |static_stat2_id                         = display_minion_monster_type
 |static_stat2_value                      = 4
-|static_stat3_id                         = base_is_projectile
+|static_stat3_id                         = number_of_monsters_to_summon
 |static_stat3_value                      = 1
 |static_stat4_id                         = base_skill_effect_duration
 |static_stat4_value                      = 3000
