--- Wiki
+++ Export
@@ -37,11 +37,11 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = firewall_attached_projectile_effect_mtx
+|static_stat1_id                         = console_skill_dont_chase
 |static_stat1_value                      = 1
-|static_stat2_id                         = is_area_damage
+|static_stat2_id                         = firewall_attached_projectile_effect_mtx
 |static_stat2_value                      = 1
-|static_stat3_id                         = console_skill_dont_chase
+|static_stat3_id                         = is_area_damage
 |static_stat3_value                      = 1
 |static_stat4_id                         = spell_damage_modifiers_apply_to_skill_dot
 |static_stat4_value                      = 1
