--- The Tower
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = weapon_divination, unique_divination, unique_weapon_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardTheTower
 |stack_size                              = 6
-|description                             = {{c|unique|Staff}}
+|description                             = {{c|unique|[[Staff]]}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_areas                              = 1_3_18_1, 1_3_18_2, 2_6_7_2, MapAtlasPalace, MapAtlasResidence, MapAtlasTower, MapWorldsTower, MapWorldsResidence
