--- The Surveyor
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = maps_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardTheSurveyor
 |stack_size                              = 4
-|description                             = {{c|white|Map}}<br>{{c|default|Map Tier:}} {{c|normal|14}}
+|description                             = {{c|white|[[Map]]}}<br>{{c|default|Map Tier:}} {{c|normal|14}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_areas                              = MapAtlasPhantasmagoria, MapAtlasMalformation, MapAtlasCore, MapWorldsMalformation, MapWorldsCore, MapWorldsPhantasmagoria, MapWorldsCarcass
