--- Imprint
+++ Export
@@ -7,13 +7,11 @@
 |tags                                    = currency, default
 |metadata_id                             = Metadata/Items/Currency/CurrencyImprint
 |stack_size                              = 1
-|description                             = Restores an imprinted item onto the original. Can't be applied to Fractured Items
+|description                             = Restores an imprinted item onto the original<br>Can't be applied to Fractured Items
 |help_text                               = Right click this item then left click on the imprinted original item to restore its modifiers. An imprint cannot be dropped or traded to other players.
 |drop_text                               = Imprint cannot drop naturally. It is created using {{il|Eternal Orb}} or [[Craicic Chimeral]]'s [[beastcrafting]] recipe.
 |is_drop_restricted                      = true
 |release_version                         = 0.11.0
-|sell_price1_name                        = Orb of Regret
-|sell_price1_amount                      = 1
 }}
 
 '''Imprints''' are created when an {{il|Eternal Orb}} is used on an item.
