--- Wiki
+++ Export
@@ -45,8 +45,6 @@
 |level1                                  = True
 |level1_level_requirement                = 12
 |level1_cost_amounts                     = 7
-|level1_damage_effectiveness             = 100
-|level1_damage_multiplier                = 100
 |level2                                  = True
 |level2_level_requirement                = 15
 |level2_cost_amounts                     = 7
