--- Loyalty Tattoo of Rakiata
+++ Export
@@ -11,9 +11,12 @@
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |tattoo_target                           = Small Dexterity
-|tattoo_stat_text                        = Trigger Summon Spirit of Rakiata [[Skill]] on [[Critical Strike]] against Marked Unique Enemy
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Dexterity}} {{c|default|Passive Skill}}
 |tattoo_limit                            = 1 Loyalty Tattoo
-|description                             = Replaces a Small Dexterity Passive Skill<br>Grants "Trigger Summon Spirit of Rakiata [[Skill]] on [[Critical Strike]] against Marked Unique Enemy"<br>Limit 1 Loyalty Tattoo
+|implicit2_text                          = {{c|default|Limit}} {{c|value|1 Loyalty Tattoo}}
+|tattoo_stat_text                        = Trigger Summon Spirit of Rakiata Skill on Critical Strike against Marked Unique Enemy
+|tattoo_skill_id                         = TriggerSummonedAncestorRakiata
+|description                             = Grants "Trigger [[Skill:TriggerSummonedAncestorRakiata|Summon Spirit of Rakiata]] [[Skill]] on [[Critical Strike]] against Marked Unique Enemy"
 |inventory_icon                          = Tattoo dexterity 2
 |drop_text                               = Can be obtained as a reward for defeating the [[Tasalio]] tribe in the [[Trial of the Ancestors]].
 |is_drop_restricted                      = true