--- Vaal Domination
+++ Export
@@ -13,7 +13,6 @@
 |gem_tags                                = Spell, Minion, Duration, Vaal, Attack, Melee, Strike
 |gem_description                         = This spell causes one of your existing Sentinels of Dominance to Ascend, becoming more powerful, gaining new attacks, and overriding its duration, while maintaining its monster modifiers and other bonuses, including from support gems. If you cannot have more Ascended Sentinels of Dominance, an existing one will be refreshed instead.
 |active_skill_name                       = Vaal Domination
-|item_class_id_restriction               = One Hand Sword, Thrusting One Hand Sword, Two Hand Sword, One Hand Axe, Two Hand Axe, One Hand Mace, Sceptre, Two Hand Mace, Staff, Dagger, Claw, Unarmed, Warstaff, Rune Dagger
 |skill_id                                = VaalDomination
 |cast_time                               = 0.5
 |required_level                          = 28