--- Vaal Earthquake
+++ Export
@@ -156,6 +156,7 @@
 |level16_stat1_value                     = 395
 |level17                                 = True
 |level17_level_requirement               = 64
+|level17_damage_effectiveness            = 100
 |level17_damage_multiplier               = 100.3
 |level17_stat_text                       = Aftershock deals 398% more Damage with Hits and Ailments
 |level17_stat1_id                        = quake_slam_fully_charged_explosion_damage_+%_final
