--- Absolution
+++ Export
@@ -36,9 +36,9 @@
 |static_duration                         = 1
 |static_stat1_id                         = display_minion_monster_type
 |static_stat1_value                      = 19
-|static_stat2_id                         = max_number_of_vaal_absolution_sentinels
+|static_stat2_id                         = is_area_damage
 |static_stat2_value                      = 1
-|static_stat3_id                         = is_area_damage
+|static_stat3_id                         = max_number_of_vaal_absolution_sentinels
 |static_stat3_value                      = 1
 |static_stat4_id                         = base_skill_effect_duration
 |static_stat4_value                      = 1000
@@ -55,7 +55,7 @@
 |level1_level_requirement                = 12
 |level1_cost_amounts                     = 9
 |level1_stat_text                        = Deals 21 to 31 Physical Damage
-|level1_stat1_id                         = display_minion_monster_level
+|level1_stat1_id                         = base_display_minion_actor_level
 |level1_stat1_value                      = 12
 |level1_stat2_id                         = spell_minimum_base_physical_damage
 |level1_stat2_value                      = 21
@@ -65,7 +65,7 @@
 |level2_level_requirement                = 15
 |level2_cost_amounts                     = 10
 |level2_stat_text                        = Deals 26 to 40 Physical Damage
-|level2_stat1_id                         = display_minion_monster_level
+|level2_stat1_id                         = base_display_minion_actor_level
 |level2_stat1_value                      = 15
 |level2_stat2_id                         = spell_minimum_base_physical_damage
 |level2_stat2_value                      = 26
@@ -75,7 +75,7 @@
 |level3_level_requirement                = 19
 |level3_cost_amounts                     = 12
 |level3_stat_text                        = Deals 36 to 55 Physical Damage<br>+1 to radius
-|level3_stat1_id                         = display_minion_monster_level
+|level3_stat1_id                         = base_display_minion_actor_level
 |level3_stat1_value                      = 19
 |level3_stat2_id                         = spell_minimum_base_physical_damage
 |level3_stat2_value                      = 36
@@ -87,7 +87,7 @@
 |level4_level_requirement                = 23
 |level4_cost_amounts                     = 13
 |level4_stat_text                        = Deals 49 to 74 Physical Damage<br>+1 to radius
-|level4_stat1_id                         = display_minion_monster_level
+|level4_stat1_id                         = base_display_minion_actor_level
 |level4_stat1_value                      = 23
 |level4_stat2_id                         = spell_minimum_base_physical_damage
 |level4_stat2_value                      = 49
@@ -99,7 +99,7 @@
 |level5_level_requirement                = 27
 |level5_cost_amounts                     = 14
 |level5_stat_text                        = Deals 65 to 98 Physical Damage<br>+1 to radius
-|level5_stat1_id                         = display_minion_monster_level
+|level5_stat1_id                         = base_display_minion_actor_level
 |level5_stat1_value                      = 27
 |level5_stat2_id                         = spell_minimum_base_physical_damage
 |level5_stat2_value                      = 65
@@ -111,7 +111,7 @@
 |level6_level_requirement                = 31
 |level6_cost_amounts                     = 15
 |level6_stat_text                        = Deals 86 to 129 Physical Damage<br>+2 to radius
-|level6_stat1_id                         = display_minion_monster_level
+|level6_stat1_id                         = base_display_minion_actor_level
 |level6_stat1_value                      = 31
 |level6_stat2_id                         = spell_minimum_base_physical_damage
 |level6_stat2_value                      = 86
@@ -123,7 +123,7 @@
 |level7_level_requirement                = 35
 |level7_cost_amounts                     = 16
 |level7_stat_text                        = Deals 112 to 167 Physical Damage<br>+2 to radius
-|level7_stat1_id                         = display_minion_monster_level
+|level7_stat1_id                         = base_display_minion_actor_level
 |level7_stat1_value                      = 35
 |level7_stat2_id                         = spell_minimum_base_physical_damage
 |level7_stat2_value                      = 112
@@ -135,7 +135,7 @@
 |level8_level_requirement                = 38
 |level8_cost_amounts                     = 17
 |level8_stat_text                        = Deals 135 to 203 Physical Damage<br>+2 to radius
-|level8_stat1_id                         = display_minion_monster_level
+|level8_stat1_id                         = base_display_minion_actor_level
 |level8_stat1_value                      = 38
 |level8_stat2_id                         = spell_minimum_base_physical_damage
 |level8_stat2_value                      = 135
@@ -147,7 +147,7 @@
 |level9_level_requirement                = 41
 |level9_cost_amounts                     = 18
 |level9_stat_text                        = Deals 163 to 245 Physical Damage<br>+3 to radius
-|level9_stat1_id                         = display_minion_monster_level
+|level9_stat1_id                         = base_display_minion_actor_level
 |level9_stat1_value                      = 41
 |level9_stat2_id                         = spell_minimum_base_physical_damage
 |level9_stat2_value                      = 163
@@ -159,7 +159,7 @@
 |level10_level_requirement               = 44
 |level10_cost_amounts                    = 19
 |level10_stat_text                       = Deals 196 to 294 Physical Damage<br>+3 to radius
-|level10_stat1_id                        = display_minion_monster_level
+|level10_stat1_id                        = base_display_minion_actor_level
 |level10_stat1_value                     = 44
 |level10_stat2_id                        = spell_minimum_base_physical_damage
 |level10_stat2_value                     = 196
@@ -171,7 +171,7 @@
 |level11_level_requirement               = 47
 |level11_cost_amounts                    = 20
 |level11_stat_text                       = Deals 235 to 352 Physical Damage<br>+3 to radius
-|level11_stat1_id                        = display_minion_monster_level
+|level11_stat1_id                        = base_display_minion_actor_level
 |level11_stat1_value                     = 47
 |level11_stat2_id                        = spell_minimum_base_physical_damage
 |level11_stat2_value                     = 235
@@ -183,7 +183,7 @@
 |level12_level_requirement               = 50
 |level12_cost_amounts                    = 20
 |level12_stat_text                       = Deals 281 to 421 Physical Damage<br>+4 to radius
-|level12_stat1_id                        = display_minion_monster_level
+|level12_stat1_id                        = base_display_minion_actor_level
 |level12_stat1_value                     = 50
 |level12_stat2_id                        = spell_minimum_base_physical_damage
 |level12_stat2_value                     = 281
@@ -195,7 +195,7 @@
 |level13_level_requirement               = 53
 |level13_cost_amounts                    = 21
 |level13_stat_text                       = Deals 335 to 502 Physical Damage<br>+4 to radius
-|level13_stat1_id                        = display_minion_monster_level
+|level13_stat1_id                        = base_display_minion_actor_level
 |level13_stat1_value                     = 53
 |level13_stat2_id                        = spell_minimum_base_physical_damage
 |level13_stat2_value                     = 335
@@ -207,7 +207,7 @@
 |level14_level_requirement               = 56
 |level14_cost_amounts                    = 22
 |level14_stat_text                       = Deals 398 to 597 Physical Damage<br>+4 to radius
-|level14_stat1_id                        = display_minion_monster_level
+|level14_stat1_id                        = base_display_minion_actor_level
 |level14_stat1_value                     = 56
 |level14_stat2_id                        = spell_minimum_base_physical_damage
 |level14_stat2_value                     = 398
@@ -219,7 +219,7 @@
 |level15_level_requirement               = 59
 |level15_cost_amounts                    = 23
 |level15_stat_text                       = Deals 473 to 709 Physical Damage<br>+5 to radius
-|level15_stat1_id                        = display_minion_monster_level
+|level15_stat1_id                        = base_display_minion_actor_level
 |level15_stat1_value                     = 59
 |level15_stat2_id                        = spell_minimum_base_physical_damage
 |level15_stat2_value                     = 473
@@ -231,7 +231,7 @@
 |level16_level_requirement               = 62
 |level16_cost_amounts                    = 24
 |level16_stat_text                       = Deals 560 to 840 Physical Damage<br>+5 to radius
-|level16_stat1_id                        = display_minion_monster_level
+|level16_stat1_id                        = base_display_minion_actor_level
 |level16_stat1_value                     = 62
 |level16_stat2_id                        = spell_minimum_base_physical_damage
 |level16_stat2_value                     = 560
@@ -243,7 +243,7 @@
 |level17_level_requirement               = 64
 |level17_cost_amounts                    = 25
 |level17_stat_text                       = Deals 627 to 940 Physical Damage<br>+5 to radius
-|level17_stat1_id                        = display_minion_monster_level
+|level17_stat1_id                        = base_display_minion_actor_level
 |level17_stat1_value                     = 64
 |level17_stat2_id                        = spell_minimum_base_physical_damage
 |level17_stat2_value                     = 627
@@ -255,7 +255,7 @@
 |level18_level_requirement               = 66
 |level18_cost_amounts                    = 25
 |level18_stat_text                       = Deals 700 to 1050 Physical Damage<br>+6 to radius
-|level18_stat1_id                        = display_minion_monster_level
+|level18_stat1_id                        = base_display_minion_actor_level
 |level18_stat1_value                     = 66
 |level18_stat2_id                        = spell_minimum_base_physical_damage
 |level18_stat2_value                     = 700
@@ -267,7 +267,7 @@
 |level19_level_requirement               = 68
 |level19_cost_amounts                    = 26
 |level19_stat_text                       = Deals 782 to 1173 Physical Damage<br>+6 to radius
-|level19_stat1_id                        = display_minion_monster_level
+|level19_stat1_id                        = base_display_minion_actor_level
 |level19_stat1_value                     = 68
 |level19_stat2_id                        = spell_minimum_base_physical_damage
 |level19_stat2_value                     = 782
@@ -279,7 +279,7 @@
 |level20_level_requirement               = 70
 |level20_cost_amounts                    = 26
 |level20_stat_text                       = Deals 873 to 1310 Physical Damage<br>+6 to radius
-|level20_stat1_id                        = display_minion_monster_level
+|level20_stat1_id                        = base_display_minion_actor_level
 |level20_stat1_value                     = 70
 |level20_stat2_id                        = spell_minimum_base_physical_damage
 |level20_stat2_value                     = 873
@@ -291,7 +291,7 @@
 |level21_level_requirement               = 72
 |level21_cost_amounts                    = 27
 |level21_stat_text                       = Deals 974 to 1461 Physical Damage<br>+7 to radius
-|level21_stat1_id                        = display_minion_monster_level
+|level21_stat1_id                        = base_display_minion_actor_level
 |level21_stat1_value                     = 72
 |level21_stat2_id                        = spell_minimum_base_physical_damage
 |level21_stat2_value                     = 974
@@ -303,7 +303,7 @@
 |level22_level_requirement               = 74
 |level22_cost_amounts                    = 27
 |level22_stat_text                       = Deals 1086 to 1629 Physical Damage<br>+7 to radius
-|level22_stat1_id                        = display_minion_monster_level
+|level22_stat1_id                        = base_display_minion_actor_level
 |level22_stat1_value                     = 74
 |level22_stat2_id                        = spell_minimum_base_physical_damage
 |level22_stat2_value                     = 1086
@@ -315,7 +315,7 @@
 |level23_level_requirement               = 76
 |level23_cost_amounts                    = 28
 |level23_stat_text                       = Deals 1210 to 1815 Physical Damage<br>+7 to radius
-|level23_stat1_id                        = display_minion_monster_level
+|level23_stat1_id                        = base_display_minion_actor_level
 |level23_stat1_value                     = 76
 |level23_stat2_id                        = spell_minimum_base_physical_damage
 |level23_stat2_value                     = 1210
@@ -327,7 +327,7 @@
 |level24_level_requirement               = 78
 |level24_cost_amounts                    = 29
 |level24_stat_text                       = Deals 1347 to 2021 Physical Damage<br>+8 to radius
-|level24_stat1_id                        = display_minion_monster_level
+|level24_stat1_id                        = base_display_minion_actor_level
 |level24_stat1_value                     = 78
 |level24_stat2_id                        = spell_minimum_base_physical_damage
 |level24_stat2_value                     = 1347
@@ -339,7 +339,7 @@
 |level25_level_requirement               = 80
 |level25_cost_amounts                    = 29
 |level25_stat_text                       = Deals 1500 to 2250 Physical Damage<br>+8 to radius
-|level25_stat1_id                        = display_minion_monster_level
+|level25_stat1_id                        = base_display_minion_actor_level
 |level25_stat1_value                     = 80
 |level25_stat2_id                        = spell_minimum_base_physical_damage
 |level25_stat2_value                     = 1500
@@ -351,7 +351,7 @@
 |level26_level_requirement               = 82
 |level26_cost_amounts                    = 30
 |level26_stat_text                       = Deals 1669 to 2503 Physical Damage<br>+8 to radius
-|level26_stat1_id                        = display_minion_monster_level
+|level26_stat1_id                        = base_display_minion_actor_level
 |level26_stat1_value                     = 82
 |level26_stat2_id                        = spell_minimum_base_physical_damage
 |level26_stat2_value                     = 1669
@@ -363,7 +363,7 @@
 |level27_level_requirement               = 84
 |level27_cost_amounts                    = 30
 |level27_stat_text                       = Deals 1855 to 2783 Physical Damage<br>+9 to radius
-|level27_stat1_id                        = display_minion_monster_level
+|level27_stat1_id                        = base_display_minion_actor_level
 |level27_stat1_value                     = 84
 |level27_stat2_id                        = spell_minimum_base_physical_damage
 |level27_stat2_value                     = 1855
@@ -375,7 +375,7 @@
 |level28_level_requirement               = 86
 |level28_cost_amounts                    = 31
 |level28_stat_text                       = Deals 2062 to 3093 Physical Damage<br>+9 to radius
-|level28_stat1_id                        = display_minion_monster_level
+|level28_stat1_id                        = base_display_minion_actor_level
 |level28_stat1_value                     = 86
 |level28_stat2_id                        = spell_minimum_base_physical_damage
 |level28_stat2_value                     = 2062
@@ -387,7 +387,7 @@
 |level29_level_requirement               = 88
 |level29_cost_amounts                    = 32
 |level29_stat_text                       = Deals 2291 to 3437 Physical Damage<br>+9 to radius
-|level29_stat1_id                        = display_minion_monster_level
+|level29_stat1_id                        = base_display_minion_actor_level
 |level29_stat1_value                     = 88
 |level29_stat2_id                        = spell_minimum_base_physical_damage
 |level29_stat2_value                     = 2291
@@ -399,7 +399,7 @@
 |level30_level_requirement               = 90
 |level30_cost_amounts                    = 32
 |level30_stat_text                       = Deals 2544 to 3817 Physical Damage<br>+10 to radius
-|level30_stat1_id                        = display_minion_monster_level
+|level30_stat1_id                        = base_display_minion_actor_level
 |level30_stat1_value                     = 90
 |level30_stat2_id                        = spell_minimum_base_physical_damage
 |level30_stat2_value                     = 2544
@@ -411,7 +411,7 @@
 |level31_level_requirement               = 91
 |level31_cost_amounts                    = 32
 |level31_stat_text                       = Deals 2681 to 4021 Physical Damage<br>+10 to radius
-|level31_stat1_id                        = display_minion_monster_level
+|level31_stat1_id                        = base_display_minion_actor_level
 |level31_stat1_value                     = 91
 |level31_stat2_id                        = spell_minimum_base_physical_damage
 |level31_stat2_value                     = 2681
@@ -423,7 +423,7 @@
 |level32_level_requirement               = 92
 |level32_cost_amounts                    = 33
 |level32_stat_text                       = Deals 2824 to 4237 Physical Damage<br>+10 to radius
-|level32_stat1_id                        = display_minion_monster_level
+|level32_stat1_id                        = base_display_minion_actor_level
 |level32_stat1_value                     = 92
 |level32_stat2_id                        = spell_minimum_base_physical_damage
 |level32_stat2_value                     = 2824
@@ -435,7 +435,7 @@
 |level33_level_requirement               = 93
 |level33_cost_amounts                    = 33
 |level33_stat_text                       = Deals 2975 to 4463 Physical Damage<br>+10 to radius
-|level33_stat1_id                        = display_minion_monster_level
+|level33_stat1_id                        = base_display_minion_actor_level
 |level33_stat1_value                     = 93
 |level33_stat2_id                        = spell_minimum_base_physical_damage
 |level33_stat2_value                     = 2975
@@ -447,7 +447,7 @@
 |level34_level_requirement               = 94
 |level34_cost_amounts                    = 33
 |level34_stat_text                       = Deals 3134 to 4701 Physical Damage<br>+10 to radius
-|level34_stat1_id                        = display_minion_monster_level
+|level34_stat1_id                        = base_display_minion_actor_level
 |level34_stat1_value                     = 94
 |level34_stat2_id                        = spell_minimum_base_physical_damage
 |level34_stat2_value                     = 3134
@@ -459,7 +459,7 @@
 |level35_level_requirement               = 95
 |level35_cost_amounts                    = 34
 |level35_stat_text                       = Deals 3301 to 4952 Physical Damage<br>+10 to radius
-|level35_stat1_id                        = display_minion_monster_level
+|level35_stat1_id                        = base_display_minion_actor_level
 |level35_stat1_value                     = 95
 |level35_stat2_id                        = spell_minimum_base_physical_damage
 |level35_stat2_value                     = 3301
@@ -471,7 +471,7 @@
 |level36_level_requirement               = 96
 |level36_cost_amounts                    = 34
 |level36_stat_text                       = Deals 3477 to 5215 Physical Damage<br>+11 to radius
-|level36_stat1_id                        = display_minion_monster_level
+|level36_stat1_id                        = base_display_minion_actor_level
 |level36_stat1_value                     = 96
 |level36_stat2_id                        = spell_minimum_base_physical_damage
 |level36_stat2_value                     = 3477
@@ -483,7 +483,7 @@
 |level37_level_requirement               = 97
 |level37_cost_amounts                    = 34
 |level37_stat_text                       = Deals 3661 to 5492 Physical Damage<br>+11 to radius
-|level37_stat1_id                        = display_minion_monster_level
+|level37_stat1_id                        = base_display_minion_actor_level
 |level37_stat1_value                     = 97
 |level37_stat2_id                        = spell_minimum_base_physical_damage
 |level37_stat2_value                     = 3661
@@ -495,7 +495,7 @@
 |level38_level_requirement               = 98
 |level38_cost_amounts                    = 34
 |level38_stat_text                       = Deals 3855 to 5782 Physical Damage<br>+11 to radius
-|level38_stat1_id                        = display_minion_monster_level
+|level38_stat1_id                        = base_display_minion_actor_level
 |level38_stat1_value                     = 98
 |level38_stat2_id                        = spell_minimum_base_physical_damage
 |level38_stat2_value                     = 3855
@@ -507,7 +507,7 @@
 |level39_level_requirement               = 99
 |level39_cost_amounts                    = 35
 |level39_stat_text                       = Deals 4059 to 6088 Physical Damage<br>+11 to radius
-|level39_stat1_id                        = display_minion_monster_level
+|level39_stat1_id                        = base_display_minion_actor_level
 |level39_stat1_value                     = 99
 |level39_stat2_id                        = spell_minimum_base_physical_damage
 |level39_stat2_value                     = 4059
@@ -519,7 +519,7 @@
 |level40_level_requirement               = 100
 |level40_cost_amounts                    = 35
 |level40_stat_text                       = Deals 4273 to 6410 Physical Damage<br>+11 to radius
-|level40_stat1_id                        = display_minion_monster_level
+|level40_stat1_id                        = base_display_minion_actor_level
 |level40_stat1_value                     = 100
 |level40_stat2_id                        = spell_minimum_base_physical_damage
 |level40_stat2_value                     = 4273
