--- Vaal Ice Nova
+++ Export
@@ -26,9 +26,9 @@
 |static_damage_effectiveness             = 120
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = is_area_damage
+|static_stat1_id                         = cannot_cancel_skill_before_contact_point
 |static_stat1_value                      = 1
-|static_stat2_id                         = cannot_cancel_skill_before_contact_point
+|static_stat2_id                         = is_area_damage
 |static_stat2_value                      = 1
 |static_stat3_id                         = ice_nova_number_of_repeats
 |static_stat3_value                      = 5
