--- Journey Tattoo of Makanui
+++ Export
@@ -11,10 +11,12 @@
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |tattoo_target                           = Keystone
-|tattoo_stat_text                        = <random [[Keystone]]>
+|tattoo_tribe                            = 10
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Keystone}} {{c|default|Passive Skill}}
 |tattoo_limit                            = 1 Keystone Tattoo
-|description                             = Replaces a Keystone Passive Skill<br>Grants a random [[Keystone]]<br>Limit 1 Keystone Tattoo
-|inventory_icon                          = 
+|implicit2_text                          = {{c|default|Limit}} {{c|value|1 Keystone Tattoo}}
+|tattoo_stat_text                        = Grants a random Keystone
+|description                             = Grants a random Keystone
 |drop_text                               = Can be obtained as a reward from [[Hinekora]] for completing a tournament in the [[Trial of the Ancestors]].
 |is_drop_restricted                      = true
 |release_version                         = 3.22.1