--- Wiki
+++ Export
@@ -14,7 +14,6 @@
 |help_text                               = Shift click to unstack.
 |drop_monsters                           = Metadata/Monsters/BreachBosses/BreachBossChaosMap
 |release_version                         = 3.3.0
-|card_background                         = 1
 }}
 
 '''The Undisputed''' is a [[divination card]]. A set of eight can be exchanged for an [[Elder item|Elder-influenced]] [[magic]] {{il|Vaal Axe}} with the {{ml|LocalIncreasedPhysicalDamagePercent8}}<!-- Merciless --> prefix and [[item level]] 100.
