--- Vaal Molten Shell
+++ Export
@@ -48,7 +48,7 @@
 |static_stat8_value                      = 10000
 |static_stat9_id                         = display_vaal_molten_shell_alternate_description
 |static_stat9_value                      = 1
-|stat_text                               = Base duration is 9 seconds<br>Modifiers to Skill Effect Duration also apply to this Skill's Soul Gain Prevention<br>Buff grants +(50-858) to Armour<br>Buff can take Damage equal to 20% of your Armour, up to a maximum of 10000<br>(35-39)% of Damage from Hits is taken from the Buff before your Life or Energy Shield<br>Buff can take Damage equal to 20% of your Armour, up to a maximum of 10000<br>Reflects (300-9000)% of Damage taken from Buff as Fire Damage each second,<br>or when Buff expires or is depleted
+|stat_text                               = Base duration is 9 seconds<br>Modifiers to Skill Effect Duration also apply to this Skill's Soul Gain Prevention<br>Buff grants +(50-858) to Armour<br>(35-39)% of Damage from Hits is taken from the Buff before your Life or Energy Shield<br>Buff can take Damage equal to 20% of your Armour, up to a maximum of 10000<br>Reflects (300-9000)% of Damage taken from Buff as Fire Damage each second,<br>or when Buff expires or is depleted
 |level1                                  = True
 |level1_level_requirement                = 4
 |level1_stat_text                        = Buff grants +50 to Armour<br>35% of Damage from Hits is taken from the Buff before your Life or Energy Shield<br>Buff can take Damage equal to 20% of your Armour, up to a maximum of 10000<br>Reflects 300% of Damage taken from Buff as Fire Damage each second,<br>or when Buff expires or is depleted
