--- Wiki
+++ Export
@@ -14,7 +14,6 @@
 |help_text                               = Shift click to unstack.
 |drop_text                               = Can drop from [[Strongbox#Loot|Jeweller's Strongboxes]].
 |release_version                         = 3.3.0
-|card_background                         = 0
 }}
 
 '''The Celestial Stone''' is a [[divination card]]. A set of ten can be exchanged for a [[Shaper item|Shaper-influenced]] [[normal]] {{il|Opal Ring}} with [[item level]] 100.
