--- Wiki
+++ Export
@@ -38,12 +38,12 @@
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
 |static_duration                         = 0.5
-|static_stat1_id                         = shock_art_variation
-|static_stat1_value                      = 2
-|static_stat2_id                         = ignite_art_variation
-|static_stat2_value                      = 7
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
+|static_stat1_id                         = ignite_art_variation
+|static_stat1_value                      = 7
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = shock_art_variation
+|static_stat3_value                      = 2
 |static_stat4_id                         = visual_hit_effect_elemental_is_holy
 |static_stat4_value                      = 1
 |static_stat5_id                         = skill_physical_damage_%_to_convert_to_lightning
