--- Siege Axe
+++ Export
@@ -15,7 +15,7 @@
 |attack_speed                            = 1.5
 |physical_damage_min                     = 38
 |physical_damage_max                     = 70
-|weapon_range                            = 11
+|weapon_range                            = 1.1
 |release_version                         = 0.10.0
 }}
 '''Siege Axe''' is a [[one-handed axe]] base type. The weapon shares the same 2D art with levelling bases {{il|Boarding Axe}} and {{il|Tomahawk}}.