--- Vaal Ground Slam
+++ Export
@@ -26,9 +26,9 @@
 |static_vaal_stored_uses                 = 4
 |static_vaal_soul_gain_prevention_time   = 1
 |static_attack_speed_multiplier          = 90
-|static_stat1_id                         = is_area_damage
+|static_stat1_id                         = cannot_cancel_skill_before_contact_point
 |static_stat1_value                      = 1
-|static_stat2_id                         = cannot_cancel_skill_before_contact_point
+|static_stat2_id                         = is_area_damage
 |static_stat2_value                      = 1
 |static_stat3_id                         = base_stun_threshold_reduction_+%
 |static_stat3_value                      = 25
