--- Wiki
+++ Export
@@ -7,7 +7,7 @@
 |tags                                    = not_for_sale, default
 |metadata_id                             = Metadata/Items/Pets/FaithfulHoundPet
 |stack_size                              = 1
-|description                             = Summons your pet
+|description                             = Summons your pet. You can command it to sit, roll over or play dead, and even pet it!
 |cosmetic_type                           = Pet
 |release_version                         = 3.21.2b
 }}
