--- Polaric Invitation (quest item)
+++ Export
@@ -11,8 +11,8 @@
 |tags                                    = default
 |metadata_id                             = Metadata/Items/MapFragments/Primordial/QuestCleansingFireKey
 |inventory_icon                          = Polaric Invitation (quest item)
-|release_version = 3.17.0
-|is_drop_restricted = true
+|is_drop_restricted                      = true
+|release_version                         = 3.17.0
 }}
 
 '''Polaric Invitation''' is a [[quest item]] and part of {{ql|The Searing Exarch (quest)|The Searing Exarch}} quest.