--- Tattoo of the Ngamahu Shaman
+++ Export
@@ -11,8 +11,9 @@
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |tattoo_target                           = Small Strength
-|tattoo_stat_text                        = 15% reduced [[Ignite]] Duration on you
-|description                             = Replaces a Small Strength Passive Skill<br>Grants "15% reduced [[Ignite]] Duration on you"
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Strength}} {{c|default|Passive Skill}}
+|tattoo_stat_text                        = 15% reduced Ignite Duration on you
+|description                             = Grants "15% reduced [[Ignite]] Duration on you"
 |inventory_icon                          = Tattoo strength 1
 |drop_text                               = Can be obtained as a reward for defeating the [[Ngamahu]] tribe in the [[Trial of the Ancestors]].
 |is_drop_restricted                      = true
