--- Volcanic Fissure
+++ Export
@@ -29,9 +29,9 @@
 |static_attack_speed_multiplier          = 80
 |static_stat1_id                         = is_area_damage
 |static_stat1_value                      = 1
-|static_stat2_id                         = show_number_of_projectiles
+|static_stat2_id                         = projectiles_not_offset
 |static_stat2_value                      = 1
-|static_stat3_id                         = projectiles_not_offset
+|static_stat3_id                         = show_number_of_projectiles
 |static_stat3_value                      = 1
 |static_stat4_id                         = number_of_additional_projectiles
 |static_stat4_value                      = 4
