--- His Judgement
+++ Export
@@ -0,0 +1,15 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = His Judgement
+|class_id                                = DivinationCard
+|size_x                                  = 1
+|size_y                                  = 1
+|flavour_text                            = Disabled
+|drop_level                              = 68
+|tags                                    = armour_divination, divination_card, default
+|metadata_id                             = Metadata/Items/DivinationCards/DivinationCardHisJudgement
+|stack_size                              = 5
+|description                             = Disabled
+|stack_size_currency_tab                 = 5000
+|help_text                               = Shift click to unstack.
+}}