--- Wiki
+++ Export
@@ -147,7 +147,6 @@
 |quality_type3_stat1_id                  = bane_enemies_explode_on_death_%_chance
 |quality_type3_stat1_value               = 500
 |quality_type3_stat_text                 = Enemies have 0.5% chance to Explode on Death, dealing a quarter of their maximum Life as Chaos Damage
-|static_cost_types                       = Mana
 |stat_text                               = Only applies Hexes from Curse Skill Gems requiring Level (24-70) or lower<br>Deals (45.6-895.6) Base Chaos Damage per second<br>Base duration is 2 seconds<br>Modifiers to Spell Damage apply to this Skill's Damage Over Time effect<br>+(0-3) to radius<br>(28-42)% more Damage per Curse applied<br>50% increased Debuff Duration per Curse applied<br>Applied Curses have 25% less Effect<br>This Gem can only Support Skill Gems requiring Level (24-70) or lower<br>You cannot Cast Supported Curse Spells<br>25% less Effect of Supported Curses
 |static_stat1_id                         = base_skill_effect_duration
 |static_stat1_value                      = 2000
