--- Wiki
+++ Export
@@ -281,8 +281,6 @@
 |level21                                 = True
 |level21_level_requirement               = 72
 |level21_cost_amounts                    = 10
-|level21_damage_effectiveness            = 100
-|level21_damage_multiplier               = 100
 |level21_stat_text                       = 10 to 192 Added Lightning Damage<br>+6 to radius
 |level21_stat1_id                        = attack_minimum_added_lightning_damage
 |level21_stat1_value                     = 10
