--- Wiki
+++ Export
@@ -6,7 +6,6 @@
 |size_y                                  = 1
 |flavour_text                            = Be like water, friend.
 |drop_level                              = 58
-|drop_level_maximum                      = 80
 |tags                                    = maps_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardRainTempter
 |stack_size                              = 9
