--- Vaal Immortal Call
+++ Export
@@ -28,9 +28,9 @@
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
 |static_duration                         = 0.4
-|static_stat1_id                         = immortal_call_prevent_all_damage
+|static_stat1_id                         = base_deal_no_damage
 |static_stat1_value                      = 1
-|static_stat2_id                         = base_deal_no_damage
+|static_stat2_id                         = immortal_call_prevent_all_damage
 |static_stat2_value                      = 1
 |static_stat3_id                         = base_skill_effect_duration
 |static_stat3_value                      = 400
