--- Boneshatter
+++ Export
@@ -39,7 +39,7 @@
 |static_stat2_value                      = 6000
 |static_stat3_id                         = trauma_strike_shockwave_area_of_effect_+%_per_100ms_stun_duration_up_to_400%
 |static_stat3_value                      = 15
-|stat_text                               = (11-158) to (17-238) Added Attack Physical Damage<br>+2 to Melee Strike Range<br>Gain 1 Trauma the first time this Attack Hits an Enemy<br>Take (6-194) Physical Damage per Trauma when you gain Trauma<br>Trauma lasts 6 seconds<br>(2-4)% more Damage per Trauma<br>Pulse has 15% increased Area of Effect per 0.1s of Stun Duration, up to 400%
+|stat_text                               = (11-158) to (17-238) Added Attack Physical Damage<br>+0.2 metres to Melee Strike Range<br>Gain 1 Trauma the first time this Attack Hits an Enemy<br>Take (6-194) Physical Damage per Trauma when you gain Trauma<br>Trauma lasts 6 seconds<br>(2-4)% more Damage per Trauma<br>Pulse has 15% increased Area of Effect per 0.1s of Stun Duration, up to 400%
 |level1                                  = True
 |level1_level_requirement                = 28
 |level1_damage_effectiveness             = 200