--- Bodyswap
+++ Export
@@ -47,7 +47,7 @@
 |level1                                  = True
 |level1_level_requirement                = 10
 |level1_cost_amounts                     = 8
-|level1_stat_text                        = Deals 3 to 5 Fire Damage<br>Explosion deals base Fire Damage equal to 6% of the corpse's Maximum Life
+|level1_stat_text                        = This Spell deals 3 to 5, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 6% of the corpse's Maximum Life
 |level1_stat1_id                         = spell_minimum_base_fire_damage
 |level1_stat1_value                      = 3
 |level1_stat2_id                         = spell_maximum_base_fire_damage
@@ -57,7 +57,7 @@
 |level2                                  = True
 |level2_level_requirement                = 13
 |level2_cost_amounts                     = 9
-|level2_stat_text                        = Deals 5 to 7 Fire Damage<br>Explosion deals base Fire Damage equal to 6.1% of the corpse's Maximum Life
+|level2_stat_text                        = This Spell deals 5 to 7, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 6.1% of the corpse's Maximum Life
 |level2_stat1_id                         = spell_minimum_base_fire_damage
 |level2_stat1_value                      = 5
 |level2_stat2_id                         = spell_maximum_base_fire_damage
@@ -67,7 +67,7 @@
 |level3                                  = True
 |level3_level_requirement                = 17
 |level3_cost_amounts                     = 10
-|level3_stat_text                        = Deals 6 to 10 Fire Damage<br>Explosion deals base Fire Damage equal to 6.2% of the corpse's Maximum Life
+|level3_stat_text                        = This Spell deals 6 to 10, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 6.2% of the corpse's Maximum Life
 |level3_stat1_id                         = spell_minimum_base_fire_damage
 |level3_stat1_value                      = 6
 |level3_stat2_id                         = spell_maximum_base_fire_damage
@@ -77,7 +77,7 @@
 |level4                                  = True
 |level4_level_requirement                = 21
 |level4_cost_amounts                     = 11
-|level4_stat_text                        = Deals 9 to 13 Fire Damage<br>Explosion deals base Fire Damage equal to 6.3% of the corpse's Maximum Life
+|level4_stat_text                        = This Spell deals 9 to 13, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 6.3% of the corpse's Maximum Life
 |level4_stat1_id                         = spell_minimum_base_fire_damage
 |level4_stat1_value                      = 9
 |level4_stat2_id                         = spell_maximum_base_fire_damage
@@ -87,7 +87,7 @@
 |level5                                  = True
 |level5_level_requirement                = 25
 |level5_cost_amounts                     = 12
-|level5_stat_text                        = Deals 12 to 18 Fire Damage<br>Explosion deals base Fire Damage equal to 6.4% of the corpse's Maximum Life<br>+1 to radius
+|level5_stat_text                        = This Spell deals 12 to 18, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 6.4% of the corpse's Maximum Life<br>+1 to radius
 |level5_stat1_id                         = spell_minimum_base_fire_damage
 |level5_stat1_value                      = 12
 |level5_stat2_id                         = spell_maximum_base_fire_damage
@@ -99,7 +99,7 @@
 |level6                                  = True
 |level6_level_requirement                = 29
 |level6_cost_amounts                     = 13
-|level6_stat_text                        = Deals 16 to 24 Fire Damage<br>Explosion deals base Fire Damage equal to 6.5% of the corpse's Maximum Life<br>+1 to radius
+|level6_stat_text                        = This Spell deals 16 to 24, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 6.5% of the corpse's Maximum Life<br>+1 to radius
 |level6_stat1_id                         = spell_minimum_base_fire_damage
 |level6_stat1_value                      = 16
 |level6_stat2_id                         = spell_maximum_base_fire_damage
@@ -111,7 +111,7 @@
 |level7                                  = True
 |level7_level_requirement                = 33
 |level7_cost_amounts                     = 14
-|level7_stat_text                        = Deals 21 to 32 Fire Damage<br>Explosion deals base Fire Damage equal to 6.6% of the corpse's Maximum Life<br>+1 to radius
+|level7_stat_text                        = This Spell deals 21 to 32, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 6.6% of the corpse's Maximum Life<br>+1 to radius
 |level7_stat1_id                         = spell_minimum_base_fire_damage
 |level7_stat1_value                      = 21
 |level7_stat2_id                         = spell_maximum_base_fire_damage
@@ -123,7 +123,7 @@
 |level8                                  = True
 |level8_level_requirement                = 36
 |level8_cost_amounts                     = 15
-|level8_stat_text                        = Deals 26 to 39 Fire Damage<br>Explosion deals base Fire Damage equal to 6.7% of the corpse's Maximum Life<br>+1 to radius
+|level8_stat_text                        = This Spell deals 26 to 39, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 6.7% of the corpse's Maximum Life<br>+1 to radius
 |level8_stat1_id                         = spell_minimum_base_fire_damage
 |level8_stat1_value                      = 26
 |level8_stat2_id                         = spell_maximum_base_fire_damage
@@ -135,7 +135,7 @@
 |level9                                  = True
 |level9_level_requirement                = 39
 |level9_cost_amounts                     = 16
-|level9_stat_text                        = Deals 32 to 47 Fire Damage<br>Explosion deals base Fire Damage equal to 6.8% of the corpse's Maximum Life<br>+1 to radius
+|level9_stat_text                        = This Spell deals 32 to 47, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 6.8% of the corpse's Maximum Life<br>+1 to radius
 |level9_stat1_id                         = spell_minimum_base_fire_damage
 |level9_stat1_value                      = 32
 |level9_stat2_id                         = spell_maximum_base_fire_damage
@@ -147,7 +147,7 @@
 |level10                                 = True
 |level10_level_requirement               = 42
 |level10_cost_amounts                    = 16
-|level10_stat_text                       = Deals 38 to 58 Fire Damage<br>Explosion deals base Fire Damage equal to 6.9% of the corpse's Maximum Life<br>+1 to radius
+|level10_stat_text                       = This Spell deals 38 to 58, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 6.9% of the corpse's Maximum Life<br>+1 to radius
 |level10_stat1_id                        = spell_minimum_base_fire_damage
 |level10_stat1_value                     = 38
 |level10_stat2_id                        = spell_maximum_base_fire_damage
@@ -159,7 +159,7 @@
 |level11                                 = True
 |level11_level_requirement               = 45
 |level11_cost_amounts                    = 17
-|level11_stat_text                       = Deals 47 to 70 Fire Damage<br>Explosion deals base Fire Damage equal to 7% of the corpse's Maximum Life<br>+1 to radius
+|level11_stat_text                       = This Spell deals 47 to 70, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 7% of the corpse's Maximum Life<br>+1 to radius
 |level11_stat1_id                        = spell_minimum_base_fire_damage
 |level11_stat1_value                     = 47
 |level11_stat2_id                        = spell_maximum_base_fire_damage
@@ -171,7 +171,7 @@
 |level12                                 = True
 |level12_level_requirement               = 48
 |level12_cost_amounts                    = 18
-|level12_stat_text                       = Deals 56 to 84 Fire Damage<br>Explosion deals base Fire Damage equal to 7.1% of the corpse's Maximum Life<br>+1 to radius
+|level12_stat_text                       = This Spell deals 56 to 84, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 7.1% of the corpse's Maximum Life<br>+1 to radius
 |level12_stat1_id                        = spell_minimum_base_fire_damage
 |level12_stat1_value                     = 56
 |level12_stat2_id                        = spell_maximum_base_fire_damage
@@ -183,7 +183,7 @@
 |level13                                 = True
 |level13_level_requirement               = 51
 |level13_cost_amounts                    = 19
-|level13_stat_text                       = Deals 68 to 102 Fire Damage<br>Explosion deals base Fire Damage equal to 7.2% of the corpse's Maximum Life<br>+2 to radius
+|level13_stat_text                       = This Spell deals 68 to 102, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 7.2% of the corpse's Maximum Life<br>+2 to radius
 |level13_stat1_id                        = spell_minimum_base_fire_damage
 |level13_stat1_value                     = 68
 |level13_stat2_id                        = spell_maximum_base_fire_damage
@@ -195,7 +195,7 @@
 |level14                                 = True
 |level14_level_requirement               = 54
 |level14_cost_amounts                    = 19
-|level14_stat_text                       = Deals 81 to 122 Fire Damage<br>Explosion deals base Fire Damage equal to 7.3% of the corpse's Maximum Life<br>+2 to radius
+|level14_stat_text                       = This Spell deals 81 to 122, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 7.3% of the corpse's Maximum Life<br>+2 to radius
 |level14_stat1_id                        = spell_minimum_base_fire_damage
 |level14_stat1_value                     = 81
 |level14_stat2_id                        = spell_maximum_base_fire_damage
@@ -207,7 +207,7 @@
 |level15                                 = True
 |level15_level_requirement               = 57
 |level15_cost_amounts                    = 20
-|level15_stat_text                       = Deals 98 to 147 Fire Damage<br>Explosion deals base Fire Damage equal to 7.4% of the corpse's Maximum Life<br>+2 to radius
+|level15_stat_text                       = This Spell deals 98 to 147, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 7.4% of the corpse's Maximum Life<br>+2 to radius
 |level15_stat1_id                        = spell_minimum_base_fire_damage
 |level15_stat1_value                     = 98
 |level15_stat2_id                        = spell_maximum_base_fire_damage
@@ -219,7 +219,7 @@
 |level16                                 = True
 |level16_level_requirement               = 60
 |level16_cost_amounts                    = 21
-|level16_stat_text                       = Deals 117 to 175 Fire Damage<br>Explosion deals base Fire Damage equal to 7.5% of the corpse's Maximum Life<br>+2 to radius
+|level16_stat_text                       = This Spell deals 117 to 175, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 7.5% of the corpse's Maximum Life<br>+2 to radius
 |level16_stat1_id                        = spell_minimum_base_fire_damage
 |level16_stat1_value                     = 117
 |level16_stat2_id                        = spell_maximum_base_fire_damage
@@ -231,7 +231,7 @@
 |level17                                 = True
 |level17_level_requirement               = 63
 |level17_cost_amounts                    = 21
-|level17_stat_text                       = Deals 140 to 210 Fire Damage<br>Explosion deals base Fire Damage equal to 7.6% of the corpse's Maximum Life<br>+2 to radius
+|level17_stat_text                       = This Spell deals 140 to 210, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 7.6% of the corpse's Maximum Life<br>+2 to radius
 |level17_stat1_id                        = spell_minimum_base_fire_damage
 |level17_stat1_value                     = 140
 |level17_stat2_id                        = spell_maximum_base_fire_damage
@@ -243,7 +243,7 @@
 |level18                                 = True
 |level18_level_requirement               = 66
 |level18_cost_amounts                    = 22
-|level18_stat_text                       = Deals 167 to 250 Fire Damage<br>Explosion deals base Fire Damage equal to 7.7% of the corpse's Maximum Life<br>+2 to radius
+|level18_stat_text                       = This Spell deals 167 to 250, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 7.7% of the corpse's Maximum Life<br>+2 to radius
 |level18_stat1_id                        = spell_minimum_base_fire_damage
 |level18_stat1_value                     = 167
 |level18_stat2_id                        = spell_maximum_base_fire_damage
@@ -255,7 +255,7 @@
 |level19                                 = True
 |level19_level_requirement               = 68
 |level19_cost_amounts                    = 22
-|level19_stat_text                       = Deals 187 to 281 Fire Damage<br>Explosion deals base Fire Damage equal to 7.8% of the corpse's Maximum Life<br>+2 to radius
+|level19_stat_text                       = This Spell deals 187 to 281, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 7.8% of the corpse's Maximum Life<br>+2 to radius
 |level19_stat1_id                        = spell_minimum_base_fire_damage
 |level19_stat1_value                     = 187
 |level19_stat2_id                        = spell_maximum_base_fire_damage
@@ -267,7 +267,7 @@
 |level20                                 = True
 |level20_level_requirement               = 70
 |level20_cost_amounts                    = 23
-|level20_stat_text                       = Deals 210 to 315 Fire Damage<br>Explosion deals base Fire Damage equal to 7.9% of the corpse's Maximum Life<br>+2 to radius
+|level20_stat_text                       = This Spell deals 210 to 315, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 7.9% of the corpse's Maximum Life<br>+2 to radius
 |level20_stat1_id                        = spell_minimum_base_fire_damage
 |level20_stat1_value                     = 210
 |level20_stat2_id                        = spell_maximum_base_fire_damage
@@ -279,7 +279,7 @@
 |level21                                 = True
 |level21_level_requirement               = 72
 |level21_cost_amounts                    = 24
-|level21_stat_text                       = Deals 236 to 354 Fire Damage<br>Explosion deals base Fire Damage equal to 8% of the corpse's Maximum Life<br>+3 to radius
+|level21_stat_text                       = This Spell deals 236 to 354, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 8% of the corpse's Maximum Life<br>+3 to radius
 |level21_stat1_id                        = spell_minimum_base_fire_damage
 |level21_stat1_value                     = 236
 |level21_stat2_id                        = spell_maximum_base_fire_damage
@@ -291,7 +291,7 @@
 |level22                                 = True
 |level22_level_requirement               = 74
 |level22_cost_amounts                    = 24
-|level22_stat_text                       = Deals 265 to 397 Fire Damage<br>Explosion deals base Fire Damage equal to 8.1% of the corpse's Maximum Life<br>+3 to radius
+|level22_stat_text                       = This Spell deals 265 to 397, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 8.1% of the corpse's Maximum Life<br>+3 to radius
 |level22_stat1_id                        = spell_minimum_base_fire_damage
 |level22_stat1_value                     = 265
 |level22_stat2_id                        = spell_maximum_base_fire_damage
@@ -303,7 +303,7 @@
 |level23                                 = True
 |level23_level_requirement               = 76
 |level23_cost_amounts                    = 25
-|level23_stat_text                       = Deals 297 to 445 Fire Damage<br>Explosion deals base Fire Damage equal to 8.2% of the corpse's Maximum Life<br>+3 to radius
+|level23_stat_text                       = This Spell deals 297 to 445, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 8.2% of the corpse's Maximum Life<br>+3 to radius
 |level23_stat1_id                        = spell_minimum_base_fire_damage
 |level23_stat1_value                     = 297
 |level23_stat2_id                        = spell_maximum_base_fire_damage
@@ -315,7 +315,7 @@
 |level24                                 = True
 |level24_level_requirement               = 78
 |level24_cost_amounts                    = 25
-|level24_stat_text                       = Deals 332 to 498 Fire Damage<br>Explosion deals base Fire Damage equal to 8.3% of the corpse's Maximum Life<br>+3 to radius
+|level24_stat_text                       = This Spell deals 332 to 498, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 8.3% of the corpse's Maximum Life<br>+3 to radius
 |level24_stat1_id                        = spell_minimum_base_fire_damage
 |level24_stat1_value                     = 332
 |level24_stat2_id                        = spell_maximum_base_fire_damage
@@ -327,7 +327,7 @@
 |level25                                 = True
 |level25_level_requirement               = 80
 |level25_cost_amounts                    = 26
-|level25_stat_text                       = Deals 372 to 558 Fire Damage<br>Explosion deals base Fire Damage equal to 8.4% of the corpse's Maximum Life<br>+3 to radius
+|level25_stat_text                       = This Spell deals 372 to 558, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 8.4% of the corpse's Maximum Life<br>+3 to radius
 |level25_stat1_id                        = spell_minimum_base_fire_damage
 |level25_stat1_value                     = 372
 |level25_stat2_id                        = spell_maximum_base_fire_damage
@@ -339,7 +339,7 @@
 |level26                                 = True
 |level26_level_requirement               = 82
 |level26_cost_amounts                    = 26
-|level26_stat_text                       = Deals 416 to 625 Fire Damage<br>Explosion deals base Fire Damage equal to 8.5% of the corpse's Maximum Life<br>+3 to radius
+|level26_stat_text                       = This Spell deals 416 to 625, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 8.5% of the corpse's Maximum Life<br>+3 to radius
 |level26_stat1_id                        = spell_minimum_base_fire_damage
 |level26_stat1_value                     = 416
 |level26_stat2_id                        = spell_maximum_base_fire_damage
@@ -351,7 +351,7 @@
 |level27                                 = True
 |level27_level_requirement               = 84
 |level27_cost_amounts                    = 27
-|level27_stat_text                       = Deals 466 to 699 Fire Damage<br>Explosion deals base Fire Damage equal to 8.6% of the corpse's Maximum Life<br>+3 to radius
+|level27_stat_text                       = This Spell deals 466 to 699, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 8.6% of the corpse's Maximum Life<br>+3 to radius
 |level27_stat1_id                        = spell_minimum_base_fire_damage
 |level27_stat1_value                     = 466
 |level27_stat2_id                        = spell_maximum_base_fire_damage
@@ -363,7 +363,7 @@
 |level28                                 = True
 |level28_level_requirement               = 86
 |level28_cost_amounts                    = 27
-|level28_stat_text                       = Deals 521 to 781 Fire Damage<br>Explosion deals base Fire Damage equal to 8.7% of the corpse's Maximum Life<br>+3 to radius
+|level28_stat_text                       = This Spell deals 521 to 781, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 8.7% of the corpse's Maximum Life<br>+3 to radius
 |level28_stat1_id                        = spell_minimum_base_fire_damage
 |level28_stat1_value                     = 521
 |level28_stat2_id                        = spell_maximum_base_fire_damage
@@ -375,7 +375,7 @@
 |level29                                 = True
 |level29_level_requirement               = 88
 |level29_cost_amounts                    = 28
-|level29_stat_text                       = Deals 582 to 873 Fire Damage<br>Explosion deals base Fire Damage equal to 8.8% of the corpse's Maximum Life<br>+4 to radius
+|level29_stat_text                       = This Spell deals 582 to 873, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 8.8% of the corpse's Maximum Life<br>+4 to radius
 |level29_stat1_id                        = spell_minimum_base_fire_damage
 |level29_stat1_value                     = 582
 |level29_stat2_id                        = spell_maximum_base_fire_damage
@@ -387,7 +387,7 @@
 |level30                                 = True
 |level30_level_requirement               = 90
 |level30_cost_amounts                    = 28
-|level30_stat_text                       = Deals 650 to 975 Fire Damage<br>Explosion deals base Fire Damage equal to 8.9% of the corpse's Maximum Life<br>+4 to radius
+|level30_stat_text                       = This Spell deals 650 to 975, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 8.9% of the corpse's Maximum Life<br>+4 to radius
 |level30_stat1_id                        = spell_minimum_base_fire_damage
 |level30_stat1_value                     = 650
 |level30_stat2_id                        = spell_maximum_base_fire_damage
@@ -399,7 +399,7 @@
 |level31                                 = True
 |level31_level_requirement               = 91
 |level31_cost_amounts                    = 28
-|level31_stat_text                       = Deals 687 to 1031 Fire Damage<br>Explosion deals base Fire Damage equal to 8.9% of the corpse's Maximum Life<br>+4 to radius
+|level31_stat_text                       = This Spell deals 687 to 1031, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 8.9% of the corpse's Maximum Life<br>+4 to radius
 |level31_stat1_id                        = spell_minimum_base_fire_damage
 |level31_stat1_value                     = 687
 |level31_stat2_id                        = spell_maximum_base_fire_damage
@@ -411,7 +411,7 @@
 |level32                                 = True
 |level32_level_requirement               = 92
 |level32_cost_amounts                    = 29
-|level32_stat_text                       = Deals 726 to 1089 Fire Damage<br>Explosion deals base Fire Damage equal to 9% of the corpse's Maximum Life<br>+4 to radius
+|level32_stat_text                       = This Spell deals 726 to 1089, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 9% of the corpse's Maximum Life<br>+4 to radius
 |level32_stat1_id                        = spell_minimum_base_fire_damage
 |level32_stat1_value                     = 726
 |level32_stat2_id                        = spell_maximum_base_fire_damage
@@ -423,7 +423,7 @@
 |level33                                 = True
 |level33_level_requirement               = 93
 |level33_cost_amounts                    = 29
-|level33_stat_text                       = Deals 767 to 1151 Fire Damage<br>Explosion deals base Fire Damage equal to 9% of the corpse's Maximum Life<br>+4 to radius
+|level33_stat_text                       = This Spell deals 767 to 1151, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 9% of the corpse's Maximum Life<br>+4 to radius
 |level33_stat1_id                        = spell_minimum_base_fire_damage
 |level33_stat1_value                     = 767
 |level33_stat2_id                        = spell_maximum_base_fire_damage
@@ -435,7 +435,7 @@
 |level34                                 = True
 |level34_level_requirement               = 94
 |level34_cost_amounts                    = 29
-|level34_stat_text                       = Deals 810 to 1216 Fire Damage<br>Explosion deals base Fire Damage equal to 9.1% of the corpse's Maximum Life<br>+4 to radius
+|level34_stat_text                       = This Spell deals 810 to 1216, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 9.1% of the corpse's Maximum Life<br>+4 to radius
 |level34_stat1_id                        = spell_minimum_base_fire_damage
 |level34_stat1_value                     = 810
 |level34_stat2_id                        = spell_maximum_base_fire_damage
@@ -447,7 +447,7 @@
 |level35                                 = True
 |level35_level_requirement               = 95
 |level35_cost_amounts                    = 29
-|level35_stat_text                       = Deals 856 to 1284 Fire Damage<br>Explosion deals base Fire Damage equal to 9.1% of the corpse's Maximum Life<br>+4 to radius
+|level35_stat_text                       = This Spell deals 856 to 1284, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 9.1% of the corpse's Maximum Life<br>+4 to radius
 |level35_stat1_id                        = spell_minimum_base_fire_damage
 |level35_stat1_value                     = 856
 |level35_stat2_id                        = spell_maximum_base_fire_damage
@@ -459,7 +459,7 @@
 |level36                                 = True
 |level36_level_requirement               = 96
 |level36_cost_amounts                    = 30
-|level36_stat_text                       = Deals 904 to 1356 Fire Damage<br>Explosion deals base Fire Damage equal to 9.2% of the corpse's Maximum Life<br>+4 to radius
+|level36_stat_text                       = This Spell deals 904 to 1356, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 9.2% of the corpse's Maximum Life<br>+4 to radius
 |level36_stat1_id                        = spell_minimum_base_fire_damage
 |level36_stat1_value                     = 904
 |level36_stat2_id                        = spell_maximum_base_fire_damage
@@ -471,7 +471,7 @@
 |level37                                 = True
 |level37_level_requirement               = 97
 |level37_cost_amounts                    = 30
-|level37_stat_text                       = Deals 955 to 1433 Fire Damage<br>Explosion deals base Fire Damage equal to 9.2% of the corpse's Maximum Life<br>+4 to radius
+|level37_stat_text                       = This Spell deals 955 to 1433, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 9.2% of the corpse's Maximum Life<br>+4 to radius
 |level37_stat1_id                        = spell_minimum_base_fire_damage
 |level37_stat1_value                     = 955
 |level37_stat2_id                        = spell_maximum_base_fire_damage
@@ -483,7 +483,7 @@
 |level38                                 = True
 |level38_level_requirement               = 98
 |level38_cost_amounts                    = 30
-|level38_stat_text                       = Deals 1009 to 1513 Fire Damage<br>Explosion deals base Fire Damage equal to 9.3% of the corpse's Maximum Life<br>+4 to radius
+|level38_stat_text                       = This Spell deals 1009 to 1513, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 9.3% of the corpse's Maximum Life<br>+4 to radius
 |level38_stat1_id                        = spell_minimum_base_fire_damage
 |level38_stat1_value                     = 1009
 |level38_stat2_id                        = spell_maximum_base_fire_damage
@@ -495,7 +495,7 @@
 |level39                                 = True
 |level39_level_requirement               = 99
 |level39_cost_amounts                    = 30
-|level39_stat_text                       = Deals 1065 to 1598 Fire Damage<br>Explosion deals base Fire Damage equal to 9.3% of the corpse's Maximum Life<br>+4 to radius
+|level39_stat_text                       = This Spell deals 1065 to 1598, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 9.3% of the corpse's Maximum Life<br>+4 to radius
 |level39_stat1_id                        = spell_minimum_base_fire_damage
 |level39_stat1_value                     = 1065
 |level39_stat2_id                        = spell_maximum_base_fire_damage
@@ -507,7 +507,7 @@
 |level40                                 = True
 |level40_level_requirement               = 100
 |level40_cost_amounts                    = 31
-|level40_stat_text                       = Deals 1125 to 1687 Fire Damage<br>Explosion deals base Fire Damage equal to 9.4% of the corpse's Maximum Life<br>+4 to radius
+|level40_stat_text                       = This Spell deals 1125 to 1687, plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to 9.4% of the corpse's Maximum Life<br>+4 to radius
 |level40_stat1_id                        = spell_minimum_base_fire_damage
 |level40_stat1_value                     = 1125
 |level40_stat2_id                        = spell_maximum_base_fire_damage
