--- Orb of Dominance
+++ Export
@@ -8,7 +8,7 @@
 |tags                                    = currency, default
 |metadata_id                             = Metadata/Items/Currency/CurrencyUpgradeInfluenceMod
 |stack_size                              = 10
-|description                             = Removes one Influenced Modifier from an item with at least two Influenced Modifiers and upgrades another Influenced Modifier. Upgrading a modifier of the highest tier transforms the modifier into an Elevated Modifier. Attempting to upgrade an Elevated Modifier rerolls its values. Can be used on Body Armours, Boots, Gloves and Helmets.
+|description                             = Removes one Influenced Modifier from an item with at least two Influenced<br>Modifiers and upgrades another Influenced Modifier<br>Upgrading a modifier of the highest tier transforms the<br>modifier into an Elevated Modifier<br>Attempting to upgrade an Elevated Modifier rerolls its values<br>Can be used on Body Armours, Boots, Gloves and Helmets
 |help_text                               = Right click this item then left click an item with at least two Influenced Modifiers to apply it.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |name_list                               = Maven's Orb
