--- Immortal Body Gloves
+++ Export
@@ -0,0 +1,11 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Immortal Body Gloves
+|class_id                                = Microtransaction
+|size_x                                  = 2
+|size_y                                  = 2
+|tags                                    = not_for_sale, default
+|metadata_id                             = Metadata/Items/MicrotransactionItemEffects/MicrotransactionImmortalBodyGloves
+|stack_size                              = 1
+|description                             = Changes the appearance of any equipped gloves to the Immortal Body Gloves. Its vibrancy changes based on your current percentage of life.
+}}