--- Conversion Trap
+++ Export
@@ -34,9 +34,9 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_skill_is_trapped
+|static_stat1_id                         = base_deal_no_damage
 |static_stat1_value                      = 1
-|static_stat2_id                         = base_deal_no_damage
+|static_stat2_id                         = base_skill_is_trapped
 |static_stat2_value                      = 1
 |static_stat3_id                         = traps_do_not_explode_on_timeout
 |static_stat3_value                      = 1
