--- Honoured Tattoo of the Wise
+++ Export
@@ -11,9 +11,11 @@
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |tattoo_target                           = +30 to Intelligence Notable
-|tattoo_stat_text                        = +1 to [[Level]] of all [[Intelligence]] [[Skill]] Gems
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|+30 to Intelligence Notable}} {{c|default|Passive Skill}}
 |tattoo_limit                            = 1 Attribute Notable Tattoo
-|description                             = Replaces a +30 to Intelligence Notable Passive Skill<br>Grants "+1 to [[Level]] of all [[Intelligence]] [[Skill]] Gems"<br>Limit 1 Attribute Notable Tattoo
+|implicit2_text                          = {{c|default|Limit}} {{c|value|1 Attribute Notable Tattoo}}
+|tattoo_stat_text                        = +1 to Level of all Intelligence Skill Gems
+|description                             = Grants "+1 to [[Level]] of all [[Intelligence]] [[Skill]] Gems"
 |inventory_icon                          = Honoured tattoo intelligence
 |drop_text                               = Can be obtained as a reward from [[Hinekora]] for completing a tournament in the [[Trial of the Ancestors]].
 |is_drop_restricted                      = true