--- The Emptiness
+++ Export
@@ -12,7 +12,7 @@
 |description                             = {{c|gem|{{il|html=|Vaal Breach}}}}<br>{{c|default|Quality:}} {{c|augmented|+6%}}<br>{{c|corrupted|[[Corrupted]]}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
-|drop_text                              = Appears to drop in maps (unique maps?) with the [[Mysterious Barrels]] [[sextant]] modifier.
+|drop_text                               = Appears to drop in maps (unique maps?) with the [[Mysterious Barrels]] [[sextant]] modifier.
 |release_version                         = 3.15.0
 }}
 
