--- The World Eater
+++ Export
@@ -14,9 +14,9 @@
 |description                             = {{c|unique|{{il|html=|Starforge}}}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
+|card_background                         = 0
 |drop_monsters                           = Metadata/Monsters/LeagueHellscape/FleshFaction/HellscapeFleshBoss
 |release_version                         = 3.2.0
-|card_background                         = 0
 }}
 
 '''The World Eater''' is a [[divination card]]. A set of eight can be exchanged for {{il|Starforge}}.
