--- Tattoo of the Rongokurai Makanga
+++ Export
@@ -11,9 +11,11 @@
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |tattoo_target                           = Small Strength
-|tattoo_stat_text                        = 8% chance to Defend with 200% of [[Armour]]
-|description                             = Replaces a Small Strength Passive Skill<br>Grants "8% chance to Defend with 200% of [[Armour]]"
 |implicit1_text                          = {{c|default|Requires}} {{c|value|Maximum 1}} {{c|default|adjacent Passive Skills Allocated}}
+|tattoo_max_adjacent                     = 1
+|implicit2_text                          = {{c|default|Requires}} {{c|value|Maximum 1}} {{c|default|adjacent Passive Skill Allocated}}
+|tattoo_stat_text                        = 8% chance to Defend with 200% of Armour
+|description                             = Grants "8% chance to Defend with 200% of [[Armour]]"
 |inventory_icon                          = Tattoo strength 2
 |drop_text                               = Can be obtained as a reward for defeating the [[Rongokurai]] tribe in the [[Trial of the Ancestors]].
 |is_drop_restricted                      = true