--- Wiki
+++ Export
@@ -34,9 +34,9 @@
 |static_cooldown                         = 2
 |static_attack_speed_multiplier          = 120
 |static_duration                         = 3
-|static_stat1_id                         = ignores_proximity_shield
+|static_stat1_id                         = base_skill_show_average_damage_instead_of_dps
 |static_stat1_value                      = 1
-|static_stat2_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat2_id                         = ignores_proximity_shield
 |static_stat2_value                      = 1
 |static_stat3_id                         = melee_defer_damage_prediction
 |static_stat3_value                      = 1
