--- Wiki
+++ Export
@@ -42,10 +42,10 @@
 |static_stat1_value                      = 1
 |static_stat2_id                         = skill_can_add_multiple_charges_per_action
 |static_stat2_value                      = 1
-|static_stat3_id                         = lightning_tendrils_channelled_larger_pulse_interval
-|static_stat3_value                      = 3
-|static_stat4_id                         = lightning_tendrils_channelled_larger_pulse_always_crit
-|static_stat4_value                      = 1
+|static_stat3_id                         = lightning_tendrils_channelled_larger_pulse_always_crit
+|static_stat3_value                      = 1
+|static_stat4_id                         = lightning_tendrils_channelled_larger_pulse_interval
+|static_stat4_value                      = 3
 |stat_text                               = Deals (1-75) to (6-1420) Lightning Damage<br>+(0-7) to radius<br>Releases a Stronger Pulse every 3 Pulses
 |level1                                  = True
 |level1_level_requirement                = 1
