--- Void of the Elements
+++ Export
@@ -14,7 +14,6 @@
 |help_text                               = Shift click to unstack.
 |drop_monsters                           = Metadata/Monsters/AtlasBosses/TheElder/TheElderUber, Metadata/Monsters/AtlasBosses/TheElder/TheElder
 |release_version                         = 3.8.0
-|card_background                         = 1
 }}
 
 '''Void of the Elements''' is a [[divination card]]. A set of three can be exchanged for an [[Elder item|Elder-influenced]] [[magic]] {{il|Opal Ring}} with the {{ml|WeaponElementalDamage5}}<!-- Overpowering --> prefix and [[item level]] 100.
