--- Void Sphere
+++ Export
@@ -41,11 +41,11 @@
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
 |static_duration                         = 5
-|static_stat1_id                         = is_area_damage
+|static_stat1_id                         = damage_cannot_be_reflected_or_leech_if_used_by_other_object
 |static_stat1_value                      = 1
-|static_stat2_id                         = skill_can_add_multiple_charges_per_action
+|static_stat2_id                         = is_area_damage
 |static_stat2_value                      = 1
-|static_stat3_id                         = damage_cannot_be_reflected_or_leech_if_used_by_other_object
+|static_stat3_id                         = skill_can_add_multiple_charges_per_action
 |static_stat3_value                      = 1
 |static_stat4_id                         = base_skill_effect_duration
 |static_stat4_value                      = 5000
