--- Vigilant Strike
+++ Export
@@ -42,7 +42,7 @@
 |static_stat1_value                      = 2
 |static_stat2_id                         = gain_fortify_on_melee_hit_ms
 |static_stat2_value                      = 8000
-|stat_text                               = +2 to Melee Strike Range<br>Melee Hits Fortify for 8 seconds
+|stat_text                               = +0.2 metres to Melee Strike Range<br>Melee Hits Fortify for 8 seconds
 |level1                                  = True
 |level1_level_requirement                = 4
 |level1_damage_effectiveness             = 230