--- Contract: The Nameless Play
+++ Export
@@ -8,13 +8,12 @@
 |tags                                    = default
 |metadata_id                             = Metadata/Items/Heist/QuestContracts/HeistContractQuestGianna1
 |inventory_icon                          = Generic NPC Quest Contract
+|release_version                         = 3.12.0
 |implicit1_text                          = {{c|default|Client:}} {{c|value|Gianna}}
 |implicit2_text                          = {{c|default|Heist Target:}} {{c|value|"The Nameless Play"}}
 |implicit3_text                          = {{c|default|Area Level:}} {{c|value|(68-?)}}
 |implicit4_text                          = {{c|default|Requires}} {{c|value|Deception}} {{c|default|(Level}} {{c|value|2}}{{c|default|)}}
 |implicit5_text                          = {{c|default|Requires}} {{c|value|Gianna, the Master of Disguise}}
-|flavour_text                            = "I heard the script will send any who attempt to perform it mad,<br/>but that other than that, it is very good."
-|release_version                         = 3.12.0
 }}
 
 '''Contract: The Nameless Play''' is a [[quest]] [[contract]]. This contract was given by [[Gianna, the Master of Disguise]].
