--- Tattoo of the Valako Stormrider
+++ Export
@@ -11,6 +11,7 @@
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |tattoo_target                           = Small Intelligence
-|tattoo_stat_text                        = +8% to [[Lightning Resistance]]
-|description                             = Replaces a Small Intelligence Passive Skill<br>Grants "+8% to [[Lightning Resistance]]"
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Intelligence}} {{c|default|Passive Skill}}
+|tattoo_stat_text                        = +8% to Lightning Resistance
+|description                             = Grants "+8% to [[Lightning Resistance]]"
 }}