--- Wiki
+++ Export
@@ -151,7 +151,6 @@
 |quality_type4_stat1_id                  = dummy_stat_display_nothing
 |quality_type4_stat1_value               = 0
 |quality_type4_stat_text                 = 
-|static_cost_types                       = Mana
 |stat_text                               = Your Skills can have an additional Brand Attached to an Enemy<br>Activates every 1 second while Attached<br>Increases and Reductions to Cast Speed also apply to this Skill's Activation frequency<br>Can be Detached for a total Duration of 5 seconds<br>Can be Attached for a total Duration of 3 seconds<br>Supported Skills have 40% reduced Chaining range<br>Supported Skills have limited Projectile Range<br>Supported Skills have 40% less Area of Effect<br>Supported Skills deal (69-63)% less Damage<br>Supported Skills Deal (40-59)% more Damage with Hits against Branded Enemy<br>Supported Skills are Triggered by Arcanist Brand, from the Brand's location
 |static_stat1_id                         = base_deal_no_damage
 |static_stat1_value                      = 1
