--- Tattoo of the Tukohama Shaman
+++ Export
@@ -6,14 +6,16 @@
 |size_y                                  = 1
 |drop_level                              = 1
 |tags                                    = currency, default
-|metadata_id                             = Metadata/Items/Currency/TattoooftheTukohamaShaman
+|metadata_id                             = Metadata/Items/Currency/AncestralTattooTukohama1
 |stack_size                              = 10
-|description                             = Replaces a Small Strength Passive Skill<br>Grants "Regenerate 15 Life per second"
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
+|tattoo_target                           = Small Strength
+|tattoo_stat_text                        = Regenerate 15 [[Life]] per second
+|description                             = Replaces a Small Strength Passive Skill<br>Grants "Regenerate 15 [[Life]] per second"
+|drop_text                               = Can be obtained as a reward for defeating the [[Tukohama]] tribe in the [[Trial of the Ancestors]].
 |is_drop_restricted                      = true
 |release_version                         = 3.22.0
-|drop_text = Can be obtained as a reward for defeating the [[Tukohama]] tribe in the [[Trial of the Ancestors]].
 }}
 
 '''Tattoo of the Tukohama Shaman''' is a [[currency]] item. It replaces the stat of a single passive node with a new stat.
