--- Processional Relic
+++ Export
@@ -4,10 +4,10 @@
 |class_id                                = Relic
 |size_x                                  = 1
 |size_y                                  = 3
-|drop_level                              = 60
+|drop_level                              = 1
 |drop_rarities_ids                       = magic, unique
 |tags                                    = medium_sanctum_relic, default
-|metadata_id                             = Metadata/Items/Relics/Relic1x3
+|metadata_id                             = Metadata/Items/Relics/SanctumRelic1x3
 |help_text                               = Place this item on the Relic Altar at the start of each Sanctum run
 |drop_text                               = Can be found in the [[Forbidden Sanctum]] as rewards from minibosses or bosses, or purchased from [[Sister Divinia]] for Aureus.
 |is_drop_restricted                      = true
