--- Awakened Sextant
+++ Export
@@ -8,7 +8,7 @@
 |tags                                    = high_level_map, sextant, currency, default
 |metadata_id                             = Metadata/Items/Currency/CurrencyAddAtlasModHigh
 |stack_size                              = 10
-|description                             = Adds or rerolls a modifier on a Voidstone.
+|description                             = Adds or rerolls a modifier on a Voidstone
 |help_text                               = Right click this item then left click a Voidstone to apply it.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |release_version                         = 2.4.0
