--- Omen of Connections
+++ Export
@@ -0,0 +1,13 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Omen of Connections
+|class_id                                = StackableCurrency
+|size_x                                  = 1
+|size_y                                  = 1
+|drop_level                              = 1
+|tags                                    = currency, default
+|metadata_id                             = Metadata/Items/Currency/AncestralOmenOnFusingMakeFullLinks
+|stack_size                              = 1
+|description                             = Will be Consumed when you use an Orb of Fusing<br>on an item to ensure maximum Links
+|help_text                               = Each player can only consume one Omen in each combat area.
+}}