--- The Realm
+++ Export
@@ -12,6 +12,7 @@
 |description                             = {{c|gem|[[Portal]]}}<br>{{c|default|Quality:}} {{c|augmented|+1-20%}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
+|drop_areas                              = BreachBossFire, BreachBossFire5, BreachBossCold, BreachBossCold5, BreachBossLightning, BreachBossLightning5, BreachBossPhysical, BreachBossPhysical5, BreachBossChaos, BreachBossChaos5
 |drop_text                               = Can drop from [[Breach monster]]s or [[Breachstone]]s, or from [[Kirac]]'s [[Divination Card]] master mission maps.
 |drop_monsters                           = Metadata/Monsters/Hellion/HellionCommanderBreach,
 Metadata/Monsters/Hellion/HellionBreachHowl,
@@ -32,7 +33,6 @@
 Metadata/Monsters/SandLeaper/SandLeaperCommanderBreach,
 Metadata/Monsters/Miner/MinerBreachCold,
 Metadata/Monsters/Miner/MinerBreach2Cold
-|drop_areas                              = BreachBossFire, BreachBossFire5, BreachBossCold, BreachBossCold5, BreachBossLightning, BreachBossLightning5, BreachBossPhysical, BreachBossPhysical5, BreachBossChaos, BreachBossChaos5
 |release_version                         = 3.1.0
 }}
 
