--- Rain of Arrows
+++ Export
@@ -36,11 +36,11 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = base_is_projectile
 |static_stat1_value                      = 1
-|static_stat2_id                         = is_area_damage
+|static_stat2_id                         = cannot_pierce
 |static_stat2_value                      = 1
-|static_stat3_id                         = skill_can_fire_arrows
+|static_stat3_id                         = is_area_damage
 |static_stat3_value                      = 1
-|static_stat4_id                         = cannot_pierce
+|static_stat4_id                         = skill_can_fire_arrows
 |static_stat4_value                      = 1
 |stat_text                               = Fires (17-21) additional Arrows
 |level1                                  = True
