--- The Body
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = armour_divination, unique_divination, unique_armour_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardTheBody
 |stack_size                              = 4
-|description                             = {{c|unique|Body Armour}}
+|description                             = {{c|unique|[[Body Armour]]}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_text                               = Can drop from an [[Strongbox#Loot|Armourer's Strongbox]].
