--- Contract: Trial Run
+++ Export
@@ -14,7 +14,6 @@
 |implicit3_text                          = {{c|default|Area Level:}} {{c|value|45}}
 |implicit4_text                          = {{c|default|Requires}} {{c|value|Lockpicking }} {{c|default|(Level}} {{c|value|1}}{{c|default|)}}
 |implicit5_text                          = {{c|default|Requires}} {{c|value|Karst, the Lockpick}}
-|flavour_text                            = "The Faridun cannot earn burial in the sky.<br />They have other ways of keeping the dead."
 }}
 
 '''Contract: Trial Run''' is a [[quest]] [[contract]].
