--- Wiki
+++ Export
@@ -8,10 +8,10 @@
 |drop_level                              = 1
 |tags                                    = default
 |metadata_id                             = Metadata/Items/MapFragments/CurrencyOfferingToTheGoddess
-|help_text                               = Travel to the Aspirants' Plaza and spend this item to open the Eternal Labyrinth.
+|help_text                               = Can be used in a personal Map Device.
+|drop_text                               = This item can be obtained by completing Labyrinth trials in maps.
+|is_drop_restricted                      = true
 |release_version                         = 2.3.0
-|is_drop_restricted                      = true
-|drop_text                               = This item can be obtained by completing Labyrinth trials in maps.
 |recipe1_part1_item_name                 = Boon of Justice
 |recipe1_part1_amount                    = 6
 |recipe1_description                     = {{c|normal|Offering to the Goddess}}
