--- Wiki
+++ Export
@@ -8,14 +8,8 @@
 |tags                                    = default
 |metadata_id                             = Metadata/Items/Heist/QuestContracts/HeistContractQuestKurai3
 |inventory_icon                          = Generic NPC Quest Contract
-|explicit1_text                          = {{c|default|Client:}} {{c|value|Kurai}}
-|explicit2_text                          = {{c|default|Heist Target:}} {{c|value|Rescue Hana the Cat}}
-|explicit3_text                          = {{c|default|Area Level:}} {{c|value|(75-83)}}
-|explicit4_text                          = {{c|default|Requires}} {{c|value|Lockpicking}} {{c|default|(Level}} {{c|value|4}}{{c|default|)}}
-|explicit5_text                          = {{c|default|Requires}} {{c|value|Kurai, the Administrator}}
-|flavour_text                            = "I won't be bullied.<br/>We must get Hana back."
+|is_drop_restricted                      = true
 |release_version                         = 3.12.3
-|is_drop_restricted = true
 }}
 
 '''Contract: The Rescue''' is a quest contract. It is the third quest of [[Kurai]]'s personal storyline. It requires to hire Kurai herself as rogue.
