--- Eternal Orb
+++ Export
@@ -8,7 +8,7 @@
 |tags                                    = currency, default
 |metadata_id                             = Metadata/Items/Currency/CurrencyImprintOrb
 |stack_size                              = 10
-|description                             = Creates an imprint of an item for later restoration. The imprint can't be applied to Fractured Items
+|description                             = Creates an imprint of an item for later restoration<br>The imprint can't be applied to Fractured Items
 |help_text                               = Right click this item then left click on an item to create an imprint. This imprint can later be used to restore this specific item to that imprinted state.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |drop_enabled                            = false
