--- Wiki
+++ Export
@@ -188,7 +188,6 @@
 |quality_type3_stat1_id                  = warcry_grant_damage_+%_to_exerted_attacks
 |quality_type3_stat1_value               = 500
 |quality_type3_stat_text                 = Exerted Attacks deal 0.5% increased Damage
-|static_cost_types                       = Mana
 |stat_text                               = Base duration is (5-5.9) seconds<br>(0-38)% increased Warcry Speed<br>Counts total Power of Enemies in Range<br>Buff grants 10% increased Critical Strike Chance per 5 power, up to a maximum of 30%<br>Buff makes Increases and Reductions to Spell Damage also apply to Attacks at 25% of their value per 5 power, up to a maximum of 150%<br>Exerts the next 5 Melee Attacks you perform<br>Exerted Attacks Trigger a Supported Spell with their first Melee Hit<br>Supported Skills deal (44-38)% less Damage<br>Supported Spells are Triggered by Attacks Exerted by Battlemage's Cry
 |static_stat1_id                         = base_deal_no_damage
 |static_stat1_value                      = 1
