--- Mitts
+++ Export
@@ -10,7 +10,7 @@
 |tags                                    = armour_divination, unique_divination, unique_armour_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardMitts
 |stack_size                              = 5
-|description                             = {{c|unique|Gloves}}
+|description                             = {{c|unique|[[Gloves]]}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_areas                              = 1_4_7, 2_8_12_1, 2_8_12_2, MapAtlasCore, MapAtlasTempleUnique, MapWorldsIceberg, MapWorldsSummit, MapWorldsFrozenCabins, MapWorldsForbiddenWoods, MapWorldsColdRiver_, MapWorldsGorge, 2_9_2
