--- Wiki
+++ Export
@@ -14,7 +14,6 @@
 |help_text                               = Shift click to unstack.
 |drop_areas                              = MapWorldsMoonTemple
 |release_version                         = 3.19.0
-|card_background                         = 0
 }}
 
 '''Astral Protection''' is a [[divination card]]. A set of five can be exchanged for a random [[Shaper item|Shaper-influenced]] [[normal]] [[shield]].
