--- Wiki
+++ Export
@@ -24,13 +24,13 @@
 |static_vaal_stored_uses                 = 1
 |static_vaal_soul_gain_prevention_time   = 6
 |static_attack_speed_multiplier          = 80
-|static_stat1_id                         = is_area_damage
+|static_stat1_id                         = cannot_cancel_skill_before_contact_point
 |static_stat1_value                      = 1
-|static_stat2_id                         = show_number_of_projectiles
+|static_stat2_id                         = is_area_damage
 |static_stat2_value                      = 1
 |static_stat3_id                         = projectiles_not_offset
 |static_stat3_value                      = 1
-|static_stat4_id                         = cannot_cancel_skill_before_contact_point
+|static_stat4_id                         = show_number_of_projectiles
 |static_stat4_value                      = 1
 |static_stat5_id                         = active_skill_area_of_effect_+%_final
 |static_stat5_value                      = 50
