--- Junk Collector's Sack
+++ Export
@@ -0,0 +1,11 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Junk Collector's Sack
+|class_id                                = Microtransaction
+|size_x                                  = 2
+|size_y                                  = 2
+|tags                                    = not_for_sale, default
+|metadata_id                             = Metadata/Items/MicrotransactionItemEffects/MicrotransactionJunkCollectorsSack
+|stack_size                              = 1
+|description                             = Adds the Junk Collector's back attachment to any equipped body armour.
+}}