--- Wiki
+++ Export
@@ -200,9 +200,9 @@
 |static_stat5_value                      = 2
 |static_stat6_id                         = maximum_number_of_spiritual_cry_warriors
 |static_stat6_value                      = 5
-|static_stat7_id                         = force_lite_skill_effects
+|static_stat7_id                         = base_damage_not_from_skill_user
 |static_stat7_value                      = 1
-|static_stat8_id                         = base_damage_not_from_skill_user
+|static_stat8_id                         = force_lite_skill_effects
 |static_stat8_value                      = 1
 |static_stat9_id                         = no_spirit_strikes
 |static_stat9_value                      = 1
