--- A Mother's Parting Gift
+++ Export
@@ -6,7 +6,7 @@
 |size_y                                  = 1
 |flavour_text                            = Nature was her domain,<br>Love was her song,<br>Family was her devotion,<br>Knowledge was her gift.
 |drop_level                              = 23
-|tags                                    = jewellery_divination, unique_divination, unique_jewellery_divination, divination_card, default
+|tags                                    = jewellery_divination, unique_divination, unique_jewellery_divination, corrupted_divination, unique_corrupted_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardAMothersPartingGift
 |stack_size                              = 6
 |description                             = {{c|unique|{{il|html=|Fertile Mind}}}}<br>{{c|corrupted|[[Corrupted]]}}
