--- Tattoo of the Tukohama Warcaller
+++ Export
@@ -11,8 +11,10 @@
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |tattoo_target                           = Small Strength
+|tattoo_tribe                            = 2
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Strength}} {{c|default|Passive Skill}}
 |tattoo_stat_text                        = 10% increased Warcry Cooldown Recovery Rate
-|description                             = Replaces a Small Strength Passive Skill<br>Grants "10% increased Warcry Cooldown Recovery Rate"
+|description                             = Grants "10% increased Warcry Cooldown Recovery Rate"
 |inventory_icon                          = Tattoo strength 1
 |drop_text                               = Can be obtained as a reward for defeating the [[Tukohama]] tribe in the [[Trial of the Ancestors]].
 |is_drop_restricted                      = true