--- Wrath
+++ Export
@@ -35,7 +35,7 @@
 |static_mana_reservation_percent         = 50
 |static_stat1_id                         = base_deal_no_damage
 |static_stat1_value                      = 1
-|stat_text                               = +(0-19) to radius<br>You and nearby allies deal (2-16) to (37-248) additional Lightning Damage with Attacks<br>You and nearby allies deal (15-21)% more Spell Lightning Damage
+|stat_text                               = +(0-1.9) metres to radius<br>You and nearby allies deal (2-16) to (37-248) additional Lightning Damage with Attacks<br>You and nearby allies deal (15-21)% more Spell Lightning Damage
 |level1                                  = True
 |level1_level_requirement                = 24
 |level1_stat_text                        = You and nearby allies deal 2 to 37 additional Lightning Damage with Attacks<br>You and nearby allies deal 15% more Spell Lightning Damage
@@ -47,7 +47,7 @@
 |level1_stat3_value                      = 15
 |level2                                  = True
 |level2_level_requirement                = 27
-|level2_stat_text                        = +1 to radius<br>You and nearby allies deal 3 to 43 additional Lightning Damage with Attacks<br>You and nearby allies deal 15% more Spell Lightning Damage
+|level2_stat_text                        = +0.1 metres to radius<br>You and nearby allies deal 3 to 43 additional Lightning Damage with Attacks<br>You and nearby allies deal 15% more Spell Lightning Damage
 |level2_stat1_id                         = active_skill_base_radius_+
 |level2_stat1_value                      = 1
 |level2_stat2_id                         = attack_minimum_added_lightning_damage
@@ -58,7 +58,7 @@
 |level2_stat4_value                      = 15
 |level3                                  = True
 |level3_level_requirement                = 30
-|level3_stat_text                        = +2 to radius<br>You and nearby allies deal 3 to 50 additional Lightning Damage with Attacks<br>You and nearby allies deal 15% more Spell Lightning Damage
+|level3_stat_text                        = +0.2 metres to radius<br>You and nearby allies deal 3 to 50 additional Lightning Damage with Attacks<br>You and nearby allies deal 15% more Spell Lightning Damage
 |level3_stat1_id                         = active_skill_base_radius_+
 |level3_stat1_value                      = 2
 |level3_stat2_id                         = attack_minimum_added_lightning_damage
@@ -69,7 +69,7 @@
 |level3_stat4_value                      = 15
 |level4                                  = True
 |level4_level_requirement                = 33
-|level4_stat_text                        = +3 to radius<br>You and nearby allies deal 4 to 57 additional Lightning Damage with Attacks<br>You and nearby allies deal 16% more Spell Lightning Damage
+|level4_stat_text                        = +0.3 metres to radius<br>You and nearby allies deal 4 to 57 additional Lightning Damage with Attacks<br>You and nearby allies deal 16% more Spell Lightning Damage
 |level4_stat1_id                         = active_skill_base_radius_+
 |level4_stat1_value                      = 3
 |level4_stat2_id                         = attack_minimum_added_lightning_damage
@@ -80,7 +80,7 @@
 |level4_stat4_value                      = 16
 |level5                                  = True
 |level5_level_requirement                = 36
-|level5_stat_text                        = +4 to radius<br>You and nearby allies deal 4 to 66 additional Lightning Damage with Attacks<br>You and nearby allies deal 16% more Spell Lightning Damage
+|level5_stat_text                        = +0.4 metres to radius<br>You and nearby allies deal 4 to 66 additional Lightning Damage with Attacks<br>You and nearby allies deal 16% more Spell Lightning Damage
 |level5_stat1_id                         = active_skill_base_radius_+
 |level5_stat1_value                      = 4
 |level5_stat2_id                         = attack_minimum_added_lightning_damage
@@ -91,7 +91,7 @@
 |level5_stat4_value                      = 16
 |level6                                  = True
 |level6_level_requirement                = 39
-|level6_stat_text                        = +5 to radius<br>You and nearby allies deal 5 to 75 additional Lightning Damage with Attacks<br>You and nearby allies deal 16% more Spell Lightning Damage
+|level6_stat_text                        = +0.5 metres to radius<br>You and nearby allies deal 5 to 75 additional Lightning Damage with Attacks<br>You and nearby allies deal 16% more Spell Lightning Damage
 |level6_stat1_id                         = active_skill_base_radius_+
 |level6_stat1_value                      = 5
 |level6_stat2_id                         = attack_minimum_added_lightning_damage
@@ -102,7 +102,7 @@
 |level6_stat4_value                      = 16
 |level7                                  = True
 |level7_level_requirement                = 42
-|level7_stat_text                        = +6 to radius<br>You and nearby allies deal 5 to 85 additional Lightning Damage with Attacks<br>You and nearby allies deal 17% more Spell Lightning Damage
+|level7_stat_text                        = +0.6 metres to radius<br>You and nearby allies deal 5 to 85 additional Lightning Damage with Attacks<br>You and nearby allies deal 17% more Spell Lightning Damage
 |level7_stat1_id                         = active_skill_base_radius_+
 |level7_stat1_value                      = 6
 |level7_stat2_id                         = attack_minimum_added_lightning_damage
@@ -113,7 +113,7 @@
 |level7_stat4_value                      = 17
 |level8                                  = True
 |level8_level_requirement                = 45
-|level8_stat_text                        = +7 to radius<br>You and nearby allies deal 6 to 97 additional Lightning Damage with Attacks<br>You and nearby allies deal 17% more Spell Lightning Damage
+|level8_stat_text                        = +0.7 metres to radius<br>You and nearby allies deal 6 to 97 additional Lightning Damage with Attacks<br>You and nearby allies deal 17% more Spell Lightning Damage
 |level8_stat1_id                         = active_skill_base_radius_+
 |level8_stat1_value                      = 7
 |level8_stat2_id                         = attack_minimum_added_lightning_damage
@@ -124,7 +124,7 @@
 |level8_stat4_value                      = 17
 |level9                                  = True
 |level9_level_requirement                = 48
-|level9_stat_text                        = +8 to radius<br>You and nearby allies deal 7 to 109 additional Lightning Damage with Attacks<br>You and nearby allies deal 17% more Spell Lightning Damage
+|level9_stat_text                        = +0.8 metres to radius<br>You and nearby allies deal 7 to 109 additional Lightning Damage with Attacks<br>You and nearby allies deal 17% more Spell Lightning Damage
 |level9_stat1_id                         = active_skill_base_radius_+
 |level9_stat1_value                      = 8
 |level9_stat2_id                         = attack_minimum_added_lightning_damage
@@ -135,7 +135,7 @@
 |level9_stat4_value                      = 17
 |level10                                 = True
 |level10_level_requirement               = 50
-|level10_stat_text                       = +9 to radius<br>You and nearby allies deal 7 to 118 additional Lightning Damage with Attacks<br>You and nearby allies deal 18% more Spell Lightning Damage
+|level10_stat_text                       = +0.9 metres to radius<br>You and nearby allies deal 7 to 118 additional Lightning Damage with Attacks<br>You and nearby allies deal 18% more Spell Lightning Damage
 |level10_stat1_id                        = active_skill_base_radius_+
 |level10_stat1_value                     = 9
 |level10_stat2_id                        = attack_minimum_added_lightning_damage
@@ -146,7 +146,7 @@
 |level10_stat4_value                     = 18
 |level11                                 = True
 |level11_level_requirement               = 52
-|level11_stat_text                       = +10 to radius<br>You and nearby allies deal 8 to 128 additional Lightning Damage with Attacks<br>You and nearby allies deal 18% more Spell Lightning Damage
+|level11_stat_text                       = +1 metre to radius<br>You and nearby allies deal 8 to 128 additional Lightning Damage with Attacks<br>You and nearby allies deal 18% more Spell Lightning Damage
 |level11_stat1_id                        = active_skill_base_radius_+
 |level11_stat1_value                     = 10
 |level11_stat2_id                        = attack_minimum_added_lightning_damage
@@ -157,7 +157,7 @@
 |level11_stat4_value                     = 18
 |level12                                 = True
 |level12_level_requirement               = 54
-|level12_stat_text                       = +11 to radius<br>You and nearby allies deal 9 to 138 additional Lightning Damage with Attacks<br>You and nearby allies deal 18% more Spell Lightning Damage
+|level12_stat_text                       = +1.1 metres to radius<br>You and nearby allies deal 9 to 138 additional Lightning Damage with Attacks<br>You and nearby allies deal 18% more Spell Lightning Damage
 |level12_stat1_id                        = active_skill_base_radius_+
 |level12_stat1_value                     = 11
 |level12_stat2_id                        = attack_minimum_added_lightning_damage
@@ -168,7 +168,7 @@
 |level12_stat4_value                     = 18
 |level13                                 = True
 |level13_level_requirement               = 56
-|level13_stat_text                       = +12 to radius<br>You and nearby allies deal 9 to 149 additional Lightning Damage with Attacks<br>You and nearby allies deal 19% more Spell Lightning Damage
+|level13_stat_text                       = +1.2 metres to radius<br>You and nearby allies deal 9 to 149 additional Lightning Damage with Attacks<br>You and nearby allies deal 19% more Spell Lightning Damage
 |level13_stat1_id                        = active_skill_base_radius_+
 |level13_stat1_value                     = 12
 |level13_stat2_id                        = attack_minimum_added_lightning_damage
@@ -179,7 +179,7 @@
 |level13_stat4_value                     = 19
 |level14                                 = True
 |level14_level_requirement               = 58
-|level14_stat_text                       = +13 to radius<br>You and nearby allies deal 10 to 161 additional Lightning Damage with Attacks<br>You and nearby allies deal 19% more Spell Lightning Damage
+|level14_stat_text                       = +1.3 metres to radius<br>You and nearby allies deal 10 to 161 additional Lightning Damage with Attacks<br>You and nearby allies deal 19% more Spell Lightning Damage
 |level14_stat1_id                        = active_skill_base_radius_+
 |level14_stat1_value                     = 13
 |level14_stat2_id                        = attack_minimum_added_lightning_damage
@@ -190,7 +190,7 @@
 |level14_stat4_value                     = 19
 |level15                                 = True
 |level15_level_requirement               = 60
-|level15_stat_text                       = +14 to radius<br>You and nearby allies deal 11 to 173 additional Lightning Damage with Attacks<br>You and nearby allies deal 19% more Spell Lightning Damage
+|level15_stat_text                       = +1.4 metres to radius<br>You and nearby allies deal 11 to 173 additional Lightning Damage with Attacks<br>You and nearby allies deal 19% more Spell Lightning Damage
 |level15_stat1_id                        = active_skill_base_radius_+
 |level15_stat1_value                     = 14
 |level15_stat2_id                        = attack_minimum_added_lightning_damage
@@ -201,7 +201,7 @@
 |level15_stat4_value                     = 19
 |level16                                 = True
 |level16_level_requirement               = 62
-|level16_stat_text                       = +15 to radius<br>You and nearby allies deal 12 to 186 additional Lightning Damage with Attacks<br>You and nearby allies deal 20% more Spell Lightning Damage
+|level16_stat_text                       = +1.5 metres to radius<br>You and nearby allies deal 12 to 186 additional Lightning Damage with Attacks<br>You and nearby allies deal 20% more Spell Lightning Damage
 |level16_stat1_id                        = active_skill_base_radius_+
 |level16_stat1_value                     = 15
 |level16_stat2_id                        = attack_minimum_added_lightning_damage
@@ -212,7 +212,7 @@
 |level16_stat4_value                     = 20
 |level17                                 = True
 |level17_level_requirement               = 64
-|level17_stat_text                       = +16 to radius<br>You and nearby allies deal 13 to 200 additional Lightning Damage with Attacks<br>You and nearby allies deal 20% more Spell Lightning Damage
+|level17_stat_text                       = +1.6 metres to radius<br>You and nearby allies deal 13 to 200 additional Lightning Damage with Attacks<br>You and nearby allies deal 20% more Spell Lightning Damage
 |level17_stat1_id                        = active_skill_base_radius_+
 |level17_stat1_value                     = 16
 |level17_stat2_id                        = attack_minimum_added_lightning_damage
@@ -223,7 +223,7 @@
 |level17_stat4_value                     = 20
 |level18                                 = True
 |level18_level_requirement               = 66
-|level18_stat_text                       = +17 to radius<br>You and nearby allies deal 13 to 215 additional Lightning Damage with Attacks<br>You and nearby allies deal 20% more Spell Lightning Damage
+|level18_stat_text                       = +1.7 metres to radius<br>You and nearby allies deal 13 to 215 additional Lightning Damage with Attacks<br>You and nearby allies deal 20% more Spell Lightning Damage
 |level18_stat1_id                        = active_skill_base_radius_+
 |level18_stat1_value                     = 17
 |level18_stat2_id                        = attack_minimum_added_lightning_damage
@@ -234,7 +234,7 @@
 |level18_stat4_value                     = 20
 |level19                                 = True
 |level19_level_requirement               = 68
-|level19_stat_text                       = +18 to radius<br>You and nearby allies deal 14 to 231 additional Lightning Damage with Attacks<br>You and nearby allies deal 21% more Spell Lightning Damage
+|level19_stat_text                       = +1.8 metres to radius<br>You and nearby allies deal 14 to 231 additional Lightning Damage with Attacks<br>You and nearby allies deal 21% more Spell Lightning Damage
 |level19_stat1_id                        = active_skill_base_radius_+
 |level19_stat1_value                     = 18
 |level19_stat2_id                        = attack_minimum_added_lightning_damage
@@ -245,7 +245,7 @@
 |level19_stat4_value                     = 21
 |level20                                 = True
 |level20_level_requirement               = 70
-|level20_stat_text                       = +19 to radius<br>You and nearby allies deal 16 to 248 additional Lightning Damage with Attacks<br>You and nearby allies deal 21% more Spell Lightning Damage
+|level20_stat_text                       = +1.9 metres to radius<br>You and nearby allies deal 16 to 248 additional Lightning Damage with Attacks<br>You and nearby allies deal 21% more Spell Lightning Damage
 |level20_stat1_id                        = active_skill_base_radius_+
 |level20_stat1_value                     = 19
 |level20_stat2_id                        = attack_minimum_added_lightning_damage
@@ -256,7 +256,7 @@
 |level20_stat4_value                     = 21
 |level21                                 = True
 |level21_level_requirement               = 72
-|level21_stat_text                       = +20 to radius<br>You and nearby allies deal 17 to 267 additional Lightning Damage with Attacks<br>You and nearby allies deal 21% more Spell Lightning Damage
+|level21_stat_text                       = +2 metres to radius<br>You and nearby allies deal 17 to 267 additional Lightning Damage with Attacks<br>You and nearby allies deal 21% more Spell Lightning Damage
 |level21_stat1_id                        = active_skill_base_radius_+
 |level21_stat1_value                     = 20
 |level21_stat2_id                        = attack_minimum_added_lightning_damage
@@ -267,7 +267,7 @@
 |level21_stat4_value                     = 21
 |level22                                 = True
 |level22_level_requirement               = 74
-|level22_stat_text                       = +21 to radius<br>You and nearby allies deal 18 to 286 additional Lightning Damage with Attacks<br>You and nearby allies deal 22% more Spell Lightning Damage
+|level22_stat_text                       = +2.1 metres to radius<br>You and nearby allies deal 18 to 286 additional Lightning Damage with Attacks<br>You and nearby allies deal 22% more Spell Lightning Damage
 |level22_stat1_id                        = active_skill_base_radius_+
 |level22_stat1_value                     = 21
 |level22_stat2_id                        = attack_minimum_added_lightning_damage
@@ -278,7 +278,7 @@
 |level22_stat4_value                     = 22
 |level23                                 = True
 |level23_level_requirement               = 76
-|level23_stat_text                       = +22 to radius<br>You and nearby allies deal 19 to 306 additional Lightning Damage with Attacks<br>You and nearby allies deal 22% more Spell Lightning Damage
+|level23_stat_text                       = +2.2 metres to radius<br>You and nearby allies deal 19 to 306 additional Lightning Damage with Attacks<br>You and nearby allies deal 22% more Spell Lightning Damage
 |level23_stat1_id                        = active_skill_base_radius_+
 |level23_stat1_value                     = 22
 |level23_stat2_id                        = attack_minimum_added_lightning_damage
@@ -289,7 +289,7 @@
 |level23_stat4_value                     = 22
 |level24                                 = True
 |level24_level_requirement               = 78
-|level24_stat_text                       = +23 to radius<br>You and nearby allies deal 20 to 328 additional Lightning Damage with Attacks<br>You and nearby allies deal 22% more Spell Lightning Damage
+|level24_stat_text                       = +2.3 metres to radius<br>You and nearby allies deal 20 to 328 additional Lightning Damage with Attacks<br>You and nearby allies deal 22% more Spell Lightning Damage
 |level24_stat1_id                        = active_skill_base_radius_+
 |level24_stat1_value                     = 23
 |level24_stat2_id                        = attack_minimum_added_lightning_damage
@@ -300,7 +300,7 @@
 |level24_stat4_value                     = 22
 |level25                                 = True
 |level25_level_requirement               = 80
-|level25_stat_text                       = +24 to radius<br>You and nearby allies deal 22 to 351 additional Lightning Damage with Attacks<br>You and nearby allies deal 23% more Spell Lightning Damage
+|level25_stat_text                       = +2.4 metres to radius<br>You and nearby allies deal 22 to 351 additional Lightning Damage with Attacks<br>You and nearby allies deal 23% more Spell Lightning Damage
 |level25_stat1_id                        = active_skill_base_radius_+
 |level25_stat1_value                     = 24
 |level25_stat2_id                        = attack_minimum_added_lightning_damage
@@ -311,7 +311,7 @@
 |level25_stat4_value                     = 23
 |level26                                 = True
 |level26_level_requirement               = 82
-|level26_stat_text                       = +25 to radius<br>You and nearby allies deal 23 to 375 additional Lightning Damage with Attacks<br>You and nearby allies deal 23% more Spell Lightning Damage
+|level26_stat_text                       = +2.5 metres to radius<br>You and nearby allies deal 23 to 375 additional Lightning Damage with Attacks<br>You and nearby allies deal 23% more Spell Lightning Damage
 |level26_stat1_id                        = active_skill_base_radius_+
 |level26_stat1_value                     = 25
 |level26_stat2_id                        = attack_minimum_added_lightning_damage
@@ -322,7 +322,7 @@
 |level26_stat4_value                     = 23
 |level27                                 = True
 |level27_level_requirement               = 84
-|level27_stat_text                       = +26 to radius<br>You and nearby allies deal 25 to 401 additional Lightning Damage with Attacks<br>You and nearby allies deal 23% more Spell Lightning Damage
+|level27_stat_text                       = +2.6 metres to radius<br>You and nearby allies deal 25 to 401 additional Lightning Damage with Attacks<br>You and nearby allies deal 23% more Spell Lightning Damage
 |level27_stat1_id                        = active_skill_base_radius_+
 |level27_stat1_value                     = 26
 |level27_stat2_id                        = attack_minimum_added_lightning_damage
@@ -333,7 +333,7 @@
 |level27_stat4_value                     = 23
 |level28                                 = True
 |level28_level_requirement               = 86
-|level28_stat_text                       = +27 to radius<br>You and nearby allies deal 27 to 429 additional Lightning Damage with Attacks<br>You and nearby allies deal 24% more Spell Lightning Damage
+|level28_stat_text                       = +2.7 metres to radius<br>You and nearby allies deal 27 to 429 additional Lightning Damage with Attacks<br>You and nearby allies deal 24% more Spell Lightning Damage
 |level28_stat1_id                        = active_skill_base_radius_+
 |level28_stat1_value                     = 27
 |level28_stat2_id                        = attack_minimum_added_lightning_damage
@@ -344,7 +344,7 @@
 |level28_stat4_value                     = 24
 |level29                                 = True
 |level29_level_requirement               = 88
-|level29_stat_text                       = +28 to radius<br>You and nearby allies deal 29 to 458 additional Lightning Damage with Attacks<br>You and nearby allies deal 24% more Spell Lightning Damage
+|level29_stat_text                       = +2.8 metres to radius<br>You and nearby allies deal 29 to 458 additional Lightning Damage with Attacks<br>You and nearby allies deal 24% more Spell Lightning Damage
 |level29_stat1_id                        = active_skill_base_radius_+
 |level29_stat1_value                     = 28
 |level29_stat2_id                        = attack_minimum_added_lightning_damage
@@ -355,7 +355,7 @@
 |level29_stat4_value                     = 24
 |level30                                 = True
 |level30_level_requirement               = 90
-|level30_stat_text                       = +29 to radius<br>You and nearby allies deal 31 to 490 additional Lightning Damage with Attacks<br>You and nearby allies deal 24% more Spell Lightning Damage
+|level30_stat_text                       = +2.9 metres to radius<br>You and nearby allies deal 31 to 490 additional Lightning Damage with Attacks<br>You and nearby allies deal 24% more Spell Lightning Damage
 |level30_stat1_id                        = active_skill_base_radius_+
 |level30_stat1_value                     = 29
 |level30_stat2_id                        = attack_minimum_added_lightning_damage
@@ -366,7 +366,7 @@
 |level30_stat4_value                     = 24
 |level31                                 = True
 |level31_level_requirement               = 91
-|level31_stat_text                       = +29 to radius<br>You and nearby allies deal 32 to 506 additional Lightning Damage with Attacks<br>You and nearby allies deal 24% more Spell Lightning Damage
+|level31_stat_text                       = +2.9 metres to radius<br>You and nearby allies deal 32 to 506 additional Lightning Damage with Attacks<br>You and nearby allies deal 24% more Spell Lightning Damage
 |level31_stat1_id                        = active_skill_base_radius_+
 |level31_stat1_value                     = 29
 |level31_stat2_id                        = attack_minimum_added_lightning_damage
@@ -377,7 +377,7 @@
 |level31_stat4_value                     = 24
 |level32                                 = True
 |level32_level_requirement               = 92
-|level32_stat_text                       = +30 to radius<br>You and nearby allies deal 33 to 523 additional Lightning Damage with Attacks<br>You and nearby allies deal 25% more Spell Lightning Damage
+|level32_stat_text                       = +3 metres to radius<br>You and nearby allies deal 33 to 523 additional Lightning Damage with Attacks<br>You and nearby allies deal 25% more Spell Lightning Damage
 |level32_stat1_id                        = active_skill_base_radius_+
 |level32_stat1_value                     = 30
 |level32_stat2_id                        = attack_minimum_added_lightning_damage
@@ -388,7 +388,7 @@
 |level32_stat4_value                     = 25
 |level33                                 = True
 |level33_level_requirement               = 93
-|level33_stat_text                       = +30 to radius<br>You and nearby allies deal 34 to 540 additional Lightning Damage with Attacks<br>You and nearby allies deal 25% more Spell Lightning Damage
+|level33_stat_text                       = +3 metres to radius<br>You and nearby allies deal 34 to 540 additional Lightning Damage with Attacks<br>You and nearby allies deal 25% more Spell Lightning Damage
 |level33_stat1_id                        = active_skill_base_radius_+
 |level33_stat1_value                     = 30
 |level33_stat2_id                        = attack_minimum_added_lightning_damage
@@ -399,7 +399,7 @@
 |level33_stat4_value                     = 25
 |level34                                 = True
 |level34_level_requirement               = 94
-|level34_stat_text                       = +31 to radius<br>You and nearby allies deal 35 to 558 additional Lightning Damage with Attacks<br>You and nearby allies deal 25% more Spell Lightning Damage
+|level34_stat_text                       = +3.1 metres to radius<br>You and nearby allies deal 35 to 558 additional Lightning Damage with Attacks<br>You and nearby allies deal 25% more Spell Lightning Damage
 |level34_stat1_id                        = active_skill_base_radius_+
 |level34_stat1_value                     = 31
 |level34_stat2_id                        = attack_minimum_added_lightning_damage
@@ -410,7 +410,7 @@
 |level34_stat4_value                     = 25
 |level35                                 = True
 |level35_level_requirement               = 95
-|level35_stat_text                       = +31 to radius<br>You and nearby allies deal 36 to 576 additional Lightning Damage with Attacks<br>You and nearby allies deal 25% more Spell Lightning Damage
+|level35_stat_text                       = +3.1 metres to radius<br>You and nearby allies deal 36 to 576 additional Lightning Damage with Attacks<br>You and nearby allies deal 25% more Spell Lightning Damage
 |level35_stat1_id                        = active_skill_base_radius_+
 |level35_stat1_value                     = 31
 |level35_stat2_id                        = attack_minimum_added_lightning_damage
@@ -421,7 +421,7 @@
 |level35_stat4_value                     = 25
 |level36                                 = True
 |level36_level_requirement               = 96
-|level36_stat_text                       = +32 to radius<br>You and nearby allies deal 37 to 595 additional Lightning Damage with Attacks<br>You and nearby allies deal 25% more Spell Lightning Damage
+|level36_stat_text                       = +3.2 metres to radius<br>You and nearby allies deal 37 to 595 additional Lightning Damage with Attacks<br>You and nearby allies deal 25% more Spell Lightning Damage
 |level36_stat1_id                        = active_skill_base_radius_+
 |level36_stat1_value                     = 32
 |level36_stat2_id                        = attack_minimum_added_lightning_damage
@@ -432,7 +432,7 @@
 |level36_stat4_value                     = 25
 |level37                                 = True
 |level37_level_requirement               = 97
-|level37_stat_text                       = +32 to radius<br>You and nearby allies deal 38 to 614 additional Lightning Damage with Attacks<br>You and nearby allies deal 25% more Spell Lightning Damage
+|level37_stat_text                       = +3.2 metres to radius<br>You and nearby allies deal 38 to 614 additional Lightning Damage with Attacks<br>You and nearby allies deal 25% more Spell Lightning Damage
 |level37_stat1_id                        = active_skill_base_radius_+
 |level37_stat1_value                     = 32
 |level37_stat2_id                        = attack_minimum_added_lightning_damage
@@ -443,7 +443,7 @@
 |level37_stat4_value                     = 25
 |level38                                 = True
 |level38_level_requirement               = 98
-|level38_stat_text                       = +33 to radius<br>You and nearby allies deal 40 to 634 additional Lightning Damage with Attacks<br>You and nearby allies deal 26% more Spell Lightning Damage
+|level38_stat_text                       = +3.3 metres to radius<br>You and nearby allies deal 40 to 634 additional Lightning Damage with Attacks<br>You and nearby allies deal 26% more Spell Lightning Damage
 |level38_stat1_id                        = active_skill_base_radius_+
 |level38_stat1_value                     = 33
 |level38_stat2_id                        = attack_minimum_added_lightning_damage
@@ -454,7 +454,7 @@
 |level38_stat4_value                     = 26
 |level39                                 = True
 |level39_level_requirement               = 99
-|level39_stat_text                       = +33 to radius<br>You and nearby allies deal 41 to 655 additional Lightning Damage with Attacks<br>You and nearby allies deal 26% more Spell Lightning Damage
+|level39_stat_text                       = +3.3 metres to radius<br>You and nearby allies deal 41 to 655 additional Lightning Damage with Attacks<br>You and nearby allies deal 26% more Spell Lightning Damage
 |level39_stat1_id                        = active_skill_base_radius_+
 |level39_stat1_value                     = 33
 |level39_stat2_id                        = attack_minimum_added_lightning_damage
@@ -465,7 +465,7 @@
 |level39_stat4_value                     = 26
 |level40                                 = True
 |level40_level_requirement               = 100
-|level40_stat_text                       = +34 to radius<br>You and nearby allies deal 42 to 676 additional Lightning Damage with Attacks<br>You and nearby allies deal 26% more Spell Lightning Damage
+|level40_stat_text                       = +3.4 metres to radius<br>You and nearby allies deal 42 to 676 additional Lightning Damage with Attacks<br>You and nearby allies deal 26% more Spell Lightning Damage
 |level40_stat1_id                        = active_skill_base_radius_+
 |level40_stat1_value                     = 34
 |level40_stat2_id                        = attack_minimum_added_lightning_damage