--- Vaal Molten Strike
+++ Export
@@ -26,9 +26,9 @@
 |static_attack_speed_multiplier          = 70
 |static_stat1_id                         = active_skill_projectile_speed_+%_variation_final
 |static_stat1_value                      = 20
-|static_stat2_id                         = show_number_of_projectiles
+|static_stat2_id                         = cannot_cancel_skill_before_contact_point
 |static_stat2_value                      = 1
-|static_stat3_id                         = cannot_cancel_skill_before_contact_point
+|static_stat3_id                         = show_number_of_projectiles
 |static_stat3_value                      = 1
 |static_stat4_id                         = active_skill_base_radius_+
 |static_stat4_value                      = 4
