--- Wiki
+++ Export
@@ -8,14 +8,13 @@
 |tags                                    = default
 |metadata_id                             = Metadata/Items/Heist/QuestContracts/HeistContractQuestTibbs2
 |inventory_icon                          = Generic NPC Quest Contract
+|is_drop_restricted                      = true
 |release_version                         = 3.12.0
 |implicit1_text                          = {{c|default|Client:}} {{c|value|Tibbs}}
 |implicit2_text                          = {{c|default|Heist Target:}} {{c|value|Set Opal Up for Life}}
 |implicit3_text                          = {{c|default|Area Level:}} {{c|value|#}}
 |implicit4_text                          = {{c|default|Requires}} {{c|value|Brute Force}} {{c|default|(Level}} {{c|value|2}}{{c|default|)}}
 |implicit5_text                          = {{c|default|Requires}} {{c|value|Tibbs, the Giant}}
-|flavour_text                            = "Damn near everything I've earned.<br/>Worth every little bit."
-|is_drop_restricted = true
 }}
 
 '''Contract: Opal's Jewels''' is a [[quest]] [[contract]]. The client is [[Tibbs]]. The area level of this contract is depended on the area level of the Smuggler's Caches. This quest is a follow up of {{il|Contract: Finding Opal}}.
