--- Tattoo of the Hinekora Makanga
+++ Export
@@ -11,6 +11,9 @@
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |tattoo_target                           = Small Intelligence
-|tattoo_stat_text                        = 8% increased Effect of your [[Curse|Curses]]
-|description                             = Replaces a Small Intelligence Passive Skill<br>Grants "8% increased Effect of your [[Curse|Curses]]"
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Intelligence}} {{c|default|Passive Skill}}
+|tattoo_min_adjacent                     = 7
+|implicit2_text                          = {{c|default|Requires}} {{c|value|7}} {{c|default|adjacent Passive Skills Allocated}}
+|tattoo_stat_text                        = 8% increased Effect of your Curses
+|description                             = Grants "8% increased Effect of your [[Curse|Curses]]"
 }}