--- Contract: The Wedding Dress
+++ Export
@@ -8,7 +8,6 @@
 |tags                                    = default
 |metadata_id                             = Metadata/Items/Heist/QuestContracts/HeistContractQuestTullina1
 |inventory_icon                          = Generic NPC Quest Contract
-|flavour_text                            = "Let's just hope she wasn't wearing it when she wandered into the sea!"
 }}
 {{stub}}
 '''Contract: The Wedding Dress''' is a [[quest]] [[contract]]. The client is [[Tullina]]. It is the first quest of her personal storyline. The area level of this contract is depended on the area level of the Smuggler's Caches.
