--- Magebloom Map Device
+++ Export
@@ -7,7 +7,7 @@
 |tags                                    = default
 |metadata_id                             = Metadata/Items/Hideout/HideoutMagebloomMapDevice
 |stack_size                              = 20
-|description                             = Creates an object in your hideout
+|description                             = Creates an object in your hideout. This Map Device blossoms with greater intensity as you open higher tier Maps.
 |help_text                               = Shift click to unstack.
 |is_master_doodad                        = False
 |variation_count                         = 1
