--- Fishing Rod
+++ Export
@@ -15,7 +15,7 @@
 |attack_speed                            = 1.2
 |physical_damage_min                     = 8
 |physical_damage_max                     = 15
-|weapon_range                            = 13
+|weapon_range                            = 1.3
 |acquisition_tags                        = fishing
 |drop_text                               = Fishing Rod drops only for characters with a {{il|Fairgraves' Tricorne}} equipped.
 |recipe1_description                     = Item level 99 {{c|normal|Fishing Rod}} with {{c|enchanted|incubation effect}}