--- Tempered Foil
+++ Export
@@ -15,7 +15,7 @@
 |attack_speed                            = 1.4
 |physical_damage_min                     = 35
 |physical_damage_max                     = 65
-|weapon_range                            = 14
+|weapon_range                            = 1.4
 |quest_reward1_type                      = quest
 |quest_reward1_quest                     = The Storm Blade
 |quest_reward1_quest_id                  = a9q3