--- Wiki
+++ Export
@@ -25,11 +25,11 @@
 |static_vaal_soul_gain_prevention_time   = 9
 |static_attack_speed_multiplier          = 75
 |static_duration                         = 1
-|static_stat1_id                         = is_area_damage
+|static_stat1_id                         = cannot_cancel_skill_before_contact_point
 |static_stat1_value                      = 1
-|static_stat2_id                         = cannot_cancel_skill_before_contact_point
+|static_stat2_id                         = force_lite_skill_effects
 |static_stat2_value                      = 1
-|static_stat3_id                         = force_lite_skill_effects
+|static_stat3_id                         = is_area_damage
 |static_stat3_value                      = 1
 |static_stat4_id                         = is_player_earthquake
 |static_stat4_value                      = 1
@@ -156,7 +156,6 @@
 |level16_stat1_value                     = 395
 |level17                                 = True
 |level17_level_requirement               = 64
-|level17_damage_effectiveness            = 100
 |level17_damage_multiplier               = 100.3
 |level17_stat_text                       = Aftershock deals 398% more Damage with Hits and Ailments
 |level17_stat1_id                        = quake_slam_fully_charged_explosion_damage_+%_final
