--- Tattoo of the Arohongui Moonwarden
+++ Export
@@ -10,7 +10,6 @@
 |stack_size                              = 10
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
-|tattoo_target                           = Small Dexterity
-|tattoo_stat_text                        = +8% to [[Cold Resistance]]
-|description                             = Replaces a Small Dexterity Passive Skill<br>Grants "+8% to [[Cold Resistance]]"
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Dexterity}} {{c|default|Passive Skill}}
+|description                             = Grants "+8% to [[Cold Resistance]]"
 }}