--- Searing Bond
+++ Export
@@ -17,7 +17,7 @@
 |cast_time                               = 1.0
 |required_level                          = 12
 |quality_type1_weight                    = 50
-|quality_type1_stat_text                 = 1% increased totem life
+|quality_type1_stat_text                 = Supported Skills have 1% increased Totem Life
 |quality_type1_stat1_id                  = totem_life_+%
 |quality_type1_stat1_value               = 1000
 |quality_type2_weight                    = 100
@@ -36,10 +36,10 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_totem_range
-|static_stat1_value                      = 100
-|static_stat2_id                         = base_skill_is_totemified
-|static_stat2_value                      = 1
+|static_stat1_id                         = base_skill_is_totemified
+|static_stat1_value                      = 1
+|static_stat2_id                         = base_totem_range
+|static_stat2_value                      = 100
 |static_stat3_id                         = number_of_additional_totems_allowed
 |static_stat3_value                      = 1
 |static_stat4_id                         = base_totem_duration
