--- Fractured Fossil
+++ Export
@@ -8,7 +8,7 @@
 |tags                                    = currency, default
 |metadata_id                             = Metadata/Items/Currency/CurrencyDelveCraftingMirror
 |stack_size                              = 10
-|description                             = Creates a split copy. Cannot be used to split Influenced,<br>Enchanted, Fractured, or Synthesised items.
+|description                             = Creates a split copy<br>Cannot be used to split Influenced,<br>Enchanted, Fractured, or Synthesised items
 |help_text                               = Place in a Resonator to influence item crafting.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |drop_text                               = Can drop in the [[Azurite Mine]].
