--- Wiki
+++ Export
@@ -2,8 +2,8 @@
 |rarity_id                               = normal
 |name                                    = Scourgestruck Helmet
 |class_id                                = Microtransaction
-|size_x                                  = 1
-|size_y                                  = 1
+|size_x                                  = 2
+|size_y                                  = 2
 |tags                                    = not_for_sale, default
 |metadata_id                             = Metadata/Items/MicrotransactionItemEffects/MicrotransactionScourgestruckHelmet
 |stack_size                              = 1
