--- The Hale Heart
+++ Export
@@ -12,10 +12,10 @@
 |description                             = {{c|rare|Item}}<br>{{c|default|Item Level:}} {{c|normal|100}}<br>{{c|default|Elder Item}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
+|card_background                         = 1
 |drop_monsters                           = Metadata/Monsters/AtlasBosses/TheElder/ElderGuardian1, Metadata/Monsters/AtlasBosses/TheElder/ElderGuardian2, Metadata/Monsters/AtlasBosses/TheElder/ElderGuardian3, Metadata/Monsters/AtlasBosses/TheElder/ElderGuardian4_
 |is_drop_restricted                      = true
 |release_version                         = 3.3.0
-|card_background                         = 1
 }}
 
 '''The Hale Heart''' is a [[divination card]]. A set of four can be exchanged for a random [[Elder item|Elder-influenced]] [[rare]] item with [[item level]] 100.
