--- Herald of Thunder
+++ Export
@@ -40,9 +40,9 @@
 |static_stat1_value                      = 1
 |static_stat2_id                         = display_skill_deals_secondary_damage
 |static_stat2_value                      = 1
-|static_stat3_id                         = skill_can_add_multiple_charges_per_action
+|static_stat3_id                         = herald_of_thunder_pvp_scaling_time_uses_bolt_frequency
 |static_stat3_value                      = 1
-|static_stat4_id                         = herald_of_thunder_pvp_scaling_time_uses_bolt_frequency
+|static_stat4_id                         = skill_can_add_multiple_charges_per_action
 |static_stat4_value                      = 1
 |static_stat5_id                         = never_shock
 |static_stat5_value                      = 1
