--- Wiki
+++ Export
@@ -44,13 +44,13 @@
 |static_stat3_value                      = 10000
 |static_stat4_id                         = banner_additional_base_duration_per_stage_ms
 |static_stat4_value                      = 1000
-|static_stat5_id                         = attacks_impale_on_hit_%_chance
-|static_stat5_value                      = 20
-|static_stat6_id                         = banner_area_of_effect_+%_per_stage
-|static_stat6_value                      = 8
+|static_stat5_id                         = banner_area_of_effect_+%_per_stage
+|static_stat5_value                      = 8
+|static_stat6_id                         = attacks_impale_on_hit_%_chance
+|static_stat6_value                      = 20
 |static_stat7_id                         = banner_buff_effect_+%_per_stage
 |static_stat7_value                      = 1
-|stat_text                               = Nearby Enemies have (15-21)% less Accuracy Rating<br>Gain 1 Stage when you Impale an Enemy while carrying the Banner, up to 5 per second<br>Gain +10 to maximum Fortification for 0.05 seconds per Stage on Placing the Banner<br>Base Duration of 10 seconds after being Placed<br>+1 second to Base Placed Banner Duration per Stage<br>You and nearby Allies have 20% chance to Impale Enemies on Hit with Attacks<br>When placed, 8% increased Area of Effect per Stage<br>When placed, 1% increased Aura effect per Stage<br>You and nearby Allies have (0-19)% increased Impale Effect
+|stat_text                               = Nearby Enemies have (15-21)% less Accuracy Rating<br>Gain 1 Stage when you Impale an Enemy while carrying the Banner, up to 5 per second<br>Gain +10 to maximum Fortification for 0.05 seconds per Stage on Placing the Banner<br>Base Duration of 10 seconds after being Placed<br>+1 second to Base Placed Banner Duration per Stage<br>When placed, 8% increased Area of Effect per Stage<br>You and nearby Allies have 20% chance to Impale Enemies on Hit with Attacks<br>When placed, 1% increased Aura effect per Stage<br>You and nearby Allies have (0-19)% increased Impale Effect
 |level1                                  = True
 |level1_level_requirement                = 24
 |level1_stat_text                        = Nearby Enemies have 15% less Accuracy Rating
