--- Wiki
+++ Export
@@ -146,12 +146,11 @@
 |quality_type1_stat_text                 = 0.5% increased Attack Speed
 |quality_type2_stat1_id                  = dummy_stat_display_nothing
 |quality_type2_stat1_value               = 0
-|quality_type2_stat_text                 = 1% less Damage with Ailments per Snipe Stage<br />Supported Skills deal 0.5% more Damage with Hits per Snipe Stage
+|quality_type2_stat_text                 = 
 |quality_type3_stat1_id                  = dummy_stat_display_nothing
 |quality_type3_stat1_value               = 0
-|quality_type3_stat_text                 = Supported Skills have 1% increased Stun Duration on Enemies
+|quality_type3_stat_text                 = 
 |stat_text                               = Maximum 6 Stages<br>(50-80)% more Damage with Ailments per Snipe Stage<br>(70-108)% more Damage with Hits per Snipe Stage<br>(50-80)% more Damage with Ailments per Snipe Stage<br>Supported Skills deal 40% less Damage<br>Supported Skills deal (70-108)% more Damage with Hits per Snipe Stage
-|static_cost_types                       = Mana
 |static_stat1_id                         = override_turn_duration_ms
 |static_stat1_value                      = 100
 |static_stat2_id                         = skill_can_fire_arrows
