--- Tattoo of the Arohongui Warmonger
+++ Export
@@ -11,6 +11,7 @@
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |tattoo_target                           = Small Intelligence
-|tattoo_stat_text                        = Killing Blows have 10% chance to Shatter Enemies as though [[Freeze|Frozen]]
-|description                             = Replaces a Small Intelligence Passive Skill<br>Grants "Killing Blows have 10% chance to Shatter Enemies as though [[Freeze|Frozen]]"
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Intelligence}} {{c|default|Passive Skill}}
+|tattoo_stat_text                        = Killing Blows have 10% chance to Shatter Enemies as though Frozen
+|description                             = Grants "Killing Blows have 10% chance to Shatter Enemies as though [[Freeze|Frozen]]"
 }}