--- Burnished Foil
+++ Export
@@ -15,7 +15,7 @@
 |attack_speed                            = 1.4
 |physical_damage_min                     = 25
 |physical_damage_max                     = 46
-|weapon_range                            = 14
+|weapon_range                            = 1.4
 |quest_reward1_type                      = quest
 |quest_reward1_quest                     = The King's Feast
 |quest_reward1_quest_id                  = a5q5