--- Increased Duration Support
+++ Export
@@ -7,7 +7,7 @@
 |drop_level                              = 31
 |tags                                    = support_gem, gem, default
 |metadata_id                             = Metadata/Items/Gems/SupportGemIncreasedDuration
-|help_text                               = This is a Support Gem. It does not grant a bonus to your character, but to skills in sockets connected to it. Place into an item socket connected to a socket containing the Active Skill Gem you wish to augment. Right click to remove from a socket.
+|help_text                               = This is a Support Gem. It does not grant a bonus to your character, but to skills in sockets connected to it. Place into an item socket connected to a socket containing the Skill Gem you wish to augment. Right click to remove from a socket.
 |strength_percent                        = 100
 |gem_tags                                = Support, Duration
 |skill_id                                = SupportIncreasedDuration
@@ -26,7 +26,7 @@
 |quality_type2_stat3_id                  = skill_effect_duration_+%
 |quality_type2_stat3_value               = -500
 |quality_type3_weight                    = 50
-|quality_type3_stat_text                 = Supported Skills have 1% increased Duration of Ailments on Enemies
+|quality_type3_stat_text                 = 1% increased Duration of Ailments inflicted with Supported Skills
 |quality_type3_stat1_id                  = base_all_ailment_duration_+%
 |quality_type3_stat1_value               = 1000
 |static_cost_multiplier                  = 130
