--- Opal Sceptre
+++ Export
@@ -16,7 +16,7 @@
 |attack_speed                            = 1.25
 |physical_damage_min                     = 49
 |physical_damage_max                     = 73
-|weapon_range                            = 11
+|weapon_range                            = 1.1
 |release_version                         = 0.9.3
 |quest_reward1_type                      = quest
 |quest_reward1_quest                     = The Storm Blade