--- Orb of Conflict
+++ Export
@@ -8,7 +8,7 @@
 |tags                                    = currency, default
 |metadata_id                             = Metadata/Items/Currency/CurrencyConflictOrb
 |stack_size                              = 10
-|description                             = Unpredictably raise the strength of one Searing Exarch or Eater of Worlds modifier on an item and lower the strength of another. Lesser modifiers that have their strength lowered will be removed.
+|description                             = Unpredictably raise the strength of one Searing Exarch or Eater of Worlds modifier on an item and lower the strength of another<br>Lesser modifiers that have their strength lowered will be removed
 |help_text                               = Right click this item then left click a non-unique item to apply it. Can only be used on items that have both a Searing Exarch and an Eater of Worlds implicit modifier. The chance of raising or lowering a modifier depends on its relative strength.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |name_list                               = Orb of Conflict
