--- Wiki
+++ Export
@@ -33,9 +33,9 @@
 |static_damage_effectiveness             = 170
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = is_area_damage
+|static_stat1_id                         = base_skill_show_average_damage_instead_of_dps
 |static_stat1_value                      = 1
-|static_stat2_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat2_id                         = is_area_damage
 |static_stat2_value                      = 1
 |static_stat3_id                         = volatile_dead_base_number_of_corpses_to_consume
 |static_stat3_value                      = 3
