--- Master Cartographer's Seal
+++ Export
@@ -14,10 +14,6 @@
 |drop_enabled                            = false
 |release_version                         = 2.4.0
 |removal_version                         = 3.9.0
-| sell_price1_name = Awakened Sextant
-| sell_price1_amount =3
-| sell_price2_name = Orb of Scouring
-| sell_price2_amount =1
 }}
 '''Master Cartographer's Seal''' is a legacy [[currency]] item. It was used to be uncomplete any [[map]] on the [[Atlas of Worlds]]. 
 
