--- Dark Dreams
+++ Export
@@ -12,9 +12,9 @@
 |description                             = {{c|rare|{{il|html=|Bone Helmet}}}}<br>{{c|default|Elder Item}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
+|card_background                         = 1
 |drop_monsters                           = Metadata/Monsters/Exiles/DelveExileWitch1
 |release_version                         = 3.7.0
-|card_background                         = 1
 }}
 
 '''Dark Dreams''' is a [[divination card]]. A set of four can be exchanged for an [[Elder item|Elder-influenced]] [[rare]] {{il|Bone Helmet}}.<ref name="Announcement"/>
