--- Wiki
+++ Export
@@ -7,5 +7,5 @@
 |tags                                    = not_for_sale, default
 |metadata_id                             = Metadata/Items/MicrotransactionItemEffects/MicrotransactionUnderworldPortalEffect
 |stack_size                              = 1
-|description                             = This character's portals use the Underworld Portal Effect.
+|description                             = This character's portals use the Underworld Portal Effect. Its effect changes in Part 2 of the campaign and again in Maps, indicating what tier of map you're playing.
 }}