--- Searing Quicksilver Flask
+++ Export
@@ -7,5 +7,5 @@
 |tags                                    = not_for_sale, default
 |metadata_id                             = Metadata/Items/MicrotransactionItemEffects/MicrotransactionFlamingQuicksilverFlaskEffect
 |stack_size                              = 1
-|description                             = Applies a Searing effect to you when your Quicksilver Flask is in effect.
+|description                             = Activating your Quicksilver Flask causes an explosion and applies a Searing effect to you while the flask is in effect.
 }}