--- Cemetery Trapdoor Portal Effect
+++ Export
@@ -0,0 +1,11 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Cemetery Trapdoor Portal Effect
+|class_id                                = Microtransaction
+|size_x                                  = 2
+|size_y                                  = 2
+|tags                                    = not_for_sale, default
+|metadata_id                             = Metadata/Items/MicrotransactionItemEffects/MicrotransactionCemeteryTrapdoorPortalEffect
+|stack_size                              = 1
+|description                             = This character's portals use the Cemetery Trapdoor Spider Portal Effect. It features a spider that lies in wait for its next victim to approach.
+}}