--- Enkindling Orb
+++ Export
@@ -8,7 +8,7 @@
 |tags                                    = currency, default
 |metadata_id                             = Metadata/Items/Currency/CurrencyEnkindlingOrb
 |stack_size                              = 10
-|description                             = Adds an enchantment to a utility flask that will improve it but prevent it from gaining charges during its effect. Replaces any existing enchantment.
+|description                             = Adds an enchantment to a utility flask that will improve it<br>but prevent it from gaining charges during its effect<br>Replaces any existing enchantment
 |help_text                               = Right click this item then left click a flask to apply it.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |release_version                         = 3.15.0
