--- Medi-Drone
+++ Export
@@ -7,5 +7,5 @@
 |tags                                    = not_for_sale, default
 |metadata_id                             = Metadata/Items/Pets/GolemBallPet
 |stack_size                              = 1
-|description                             = Summons your pet
+|description                             = Summons your pet. It will use a healing beam effect on you whenever you're gaining life.
 }}