--- Quickstep
+++ Export
@@ -60,8 +60,8 @@
 |level10                                 = True
 |level10_level_requirement               = 1
 |level1_experience                       = 0
+|drop_enabled                            = false
 |drop_text                               = Guaranteed drop from [[Hillock]] in [[Ruthless mode]]; it is drop-disabled in normal game mode.
-|drop_enabled                            = false
 |release_version                         = 3.21.0
 }}
 '''Quickstep''' is a [[blink]] / [[movement]] / [[travel]] [[active skill|skill]] which causes the skill user to quickly move in the target direction. It works similarly to {{il|Dash}}, except it can't pass through terrain or monsters that you can't normally walk on or through.
