--- Wiki
+++ Export
@@ -13,7 +13,7 @@
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_areas                              = BreachBossChaos, BreachBossChaos5, BreachBossPhysical, BreachBossPhysical5, BreachBossLightning, BreachBossLightning5, BreachBossFire, BreachBossFire5, BreachBossCold, BreachBossCold5
-|drop_text = May be able to drop in other Breachstones.{{Confirm}}
+|drop_text                               = May be able to drop in other Breachstones.{{Confirm}}
 |release_version                         = 3.7.0
 }}
 
