--- Wiki
+++ Export
@@ -15,7 +15,6 @@
 |drop_text                               = Drops from the [[Guardians of the Void]].
 |drop_monsters                           = Metadata/Monsters/AtlasBosses/PhoenixBoss, Metadata/Monsters/AtlasBosses/HydraBoss, Metadata/Monsters/AtlasBosses/MinotaurBoss, Metadata/Monsters/AtlasBosses/ChimeraBoss
 |release_version                         = 3.3.0
-|card_background                         = 0
 }}
 
 '''Perfection''' is a [[divination card]]. A set of five can be exchanged for a random [[Shaper item|Shaper-influenced]] [[rare]] [[jewellery]] item with [[item level]] 100.
