--- Loyalty Tattoo of Utula
+++ Export
@@ -6,14 +6,13 @@
 |size_y                                  = 1
 |drop_level                              = 1
 |tags                                    = currency, default
-|metadata_id                             = Metadata/Items/Currency/LoyaltyTattooofUtula
+|metadata_id                             = Metadata/Items/Currency/AncestralTattooKitava6
 |stack_size                              = 10
-|description                             = Replaces a Small Strength Passive Skill<br>Grants "Trigger Level 20 Summon Spirit of Utula Skill on taking a Savage Hit from a Unique Enemy"<br>Limit 1 Loyalty Tattoo
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
+|drop_text                               = Can be obtained as a reward for defeating the [[Kitava]] tribe in the [[Trial of the Ancestors]].
 |is_drop_restricted                      = true
 |release_version                         = 3.22.0
-|drop_text = Can be obtained as a reward for defeating the [[Kitava]] tribe in the [[Trial of the Ancestors]].
 }}
 
 '''Loyalty Tattoo of Utula''' is a [[currency]] item. It replaces the stat of a single passive node with a new stat.
