--- Coronal Maul
+++ Export
@@ -15,7 +15,7 @@
 |attack_speed                            = 1.2
 |physical_damage_min                     = 91
 |physical_damage_max                     = 136
-|weapon_range                            = 13
+|weapon_range                            = 1.3
 |release_version                         = 2.0.0
 |recipe1_part1_item_name                 = The Warlord
 |recipe1_part1_amount                    = 6