--- Wiki
+++ Export
@@ -14,7 +14,6 @@
 |skill_id                                = SupportCastOnCritPlus
 |support_gem_letter                      = &
 |required_level                          = 72
-|static_cost_multiplier                  = 120
 |quality_type1_weight                    = 50
 |level1                                  = True
 |level1_level_requirement                = 72
@@ -62,9 +61,9 @@
 |stat_text                               = Supported Attacks will Trigger a Supported Spell when you Crit an Enemy<br>Supported Skills deal (9-7)% less Spell Damage<br>You cannot Cast Supported Triggerable Spells directly<br>Supported Spells have (10-22)% increased Cooldown Recovery Rate
 |static_stat1_id                         = cast_linked_spells_on_attack_crit_%
 |static_stat1_value                      = 100
-|static_stat2_id                         = triggered_skill_uses_main_hand_or_averaged_attack_time_for_pvp_scaling
+|static_stat2_id                         = cast_spell_on_linked_attack_crit
 |static_stat2_value                      = 1
-|static_stat3_id                         = cast_spell_on_linked_attack_crit
+|static_stat3_id                         = triggered_skill_uses_main_hand_or_averaged_attack_time_for_pvp_scaling
 |static_stat3_value                      = 1
 |static_stat4_id                         = spell_uncastable_if_triggerable
 |static_stat4_value                      = 1
