--- Wiki
+++ Export
@@ -31,10 +31,10 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = melee_splash
 |static_stat1_value                      = 1
-|static_stat2_id                         = support_melee_splash_damage_+%_final_for_splash
-|static_stat2_value                      = -40
-|static_stat3_id                         = quality_display_melee_splash_is_gem
-|static_stat3_value                      = 1
+|static_stat2_id                         = quality_display_melee_splash_is_gem
+|static_stat2_value                      = 1
+|static_stat3_id                         = support_melee_splash_damage_+%_final_for_splash
+|static_stat3_value                      = -40
 |stat_text                               = Supported Skills deal Splash Damage to surrounding targets<br>Supported Skills deal 40% less Damage to surrounding targets<br>Supported Skills have (0-57)% more Melee Splash Area of Effect
 |level1                                  = True
 |level1_level_requirement                = 8
