--- Lantern Bearer's Annihilating Light
+++ Export
@@ -7,5 +7,5 @@
 |tags                                    = not_for_sale, default
 |metadata_id                             = Metadata/Items/MicrotransactionItemEffects/MicrotransactionLanternBearersAnnihilatingLight
 |stack_size                              = 1
-|description                             = Lantern Bearer's Annihilating Light changes the appearance of your Annihilating Light. Black lightning, flames or ice afflicts enemies hit by elemental skils, which escalates based on the number of times hit.
+|description                             = Lantern Bearer's Annihilating Light changes the appearance of your Annihilating Light. Black lightning, flames or ice afflicts enemies hit by elemental skills, which escalates based on the number of times hit.
 }}