--- Crackling Lance
+++ Export
@@ -46,7 +46,7 @@
 |static_stat4_value                      = 3
 |static_stat5_id                         = disintegrate_damage_+%_final_per_intensity
 |static_stat5_value                      = 35
-|stat_text                               = Deals (28-458) to (84-1375) Lightning Damage<br>This Spell loses 1 Intensity every 0.25 seconds while moving, or immediately if you teleport<br>This Spell gains Intensity when you Cast it, to a maximum of 3<br>+3 to Beam Width Per Intensity<br>33% less branching angle per Intensity<br>35% more Damage with Hits and Ailments per Intensity
+|stat_text                               = Deals (28-458) to (84-1375) Lightning Damage<br>This Spell loses 1 Intensity every 0.25 seconds while moving, or immediately if you teleport<br>This Spell gains Intensity when you Cast it, to a maximum of 3<br>+0.3 metres to Beam Width Per Intensity<br>33% less branching angle per Intensity<br>35% more Damage with Hits and Ailments per Intensity
 |level1                                  = True
 |level1_level_requirement                = 28
 |level1_cost_amounts                     = 11