--- Sanctified Relic (strength)
+++ Export
@@ -0,0 +1,13 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Sanctified Relic
+|class_id                                = SanctumSpecialRelic
+|size_x                                  = 1
+|size_y                                  = 2
+|drop_level                              = 75
+|drop_rarities_ids                       = magic
+|tags                                    = str_special_relic, default
+|metadata_id                             = Metadata/Items/Relics/SanctumSpecialRelic1
+|help_text                               = Place this item on the Relic Altar to affect your character
+|inventory_icon                          = Sanctified Relic (strength)
+}}