--- Whispering Incubator
+++ Export
@@ -12,8 +12,8 @@
 |help_text                               = Right click this item then left click an item to apply it. The Incubated item drops after killing a specific number of monsters.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |drop_text                               = Can only be obtained from sources of Legion or Incubator rewards (e.g. [[Heist]]) and has a maximum drop level of 67. Cannot be found in maps.
+|drop_level_maximum                      = 67
 |release_version                         = 3.7.0
-|drop_level_maximum             = 67
 }}
 
 '''Whispering Incubator''' is an [[incubator]].
