--- Contract: The Admiral's Records
+++ Export
@@ -9,7 +9,6 @@
 |metadata_id                             = Metadata/Items/Heist/QuestContracts/HeistContractQuestFaustus1
 |inventory_icon                          = Generic NPC Quest Contract
 |release_version                         = 3.12.0
-|flavour_text                            = "Darnaw is stupid and confident.<br />I am certain he will have recorded his misdeeds."
 }}
 {{stub}}
 '''Contract: The Admiral's Records''' is a [[quest]] [[contract]]. The client is [[Faustus]]. It is the first quest of his personal storyline. The area level of this contract is depended on the area level of the Smuggler's Caches.
