--- Simulacrum
+++ Export
@@ -9,6 +9,7 @@
 |tags                                    = affliction_splinter, default
 |metadata_id                             = Metadata/Items/MapFragments/CurrencyAfflictionFragment
 |help_text                               = Can be used in a personal Map Device.
+|implicit1                               = AfflictionFragmentImplicit
 |is_drop_restricted                      = true
 |release_version                         = 3.10.0
 |recipe1_part1_item_name                 = Simulacrum Splinter
