--- Wintertide Brand
+++ Export
@@ -34,11 +34,11 @@
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
 |static_duration                         = 2
-|static_stat1_id                         = is_area_damage
+|static_stat1_id                         = console_skill_dont_chase
 |static_stat1_value                      = 1
-|static_stat2_id                         = skill_can_add_multiple_charges_per_action
+|static_stat2_id                         = is_area_damage
 |static_stat2_value                      = 1
-|static_stat3_id                         = console_skill_dont_chase
+|static_stat3_id                         = skill_can_add_multiple_charges_per_action
 |static_stat3_value                      = 1
 |static_stat4_id                         = base_number_of_sigils_allowed_per_target
 |static_stat4_value                      = 1
@@ -60,7 +60,7 @@
 |static_stat12_value                     = 20
 |static_stat13_id                        = winter_brand_max_number_of_stages
 |static_stat13_value                     = 20
-|stat_text                               = Your Skills can have an additional Brand Attached to an Enemy<br>Deals (13.7-261.2) Base Cold Damage per second<br>Activates every 0.25 seconds while Attached<br>Increases and Reductions to Cast Speed also apply to this Skill's Activation frequency<br>Can have an additional Brand Attached to an Enemy<br>Modifiers to Spell Damage apply to this Skill's Damage Over Time effect<br>Can be Detached for a total Duration of 6 seconds<br>Can be Attached for a total Duration of 2 seconds<br>Base Debuff Duration is 1 seconds<br>Debuff deals 20% more Damage per Stage<br>Maximum 20 Stages
+|stat_text                               = Your Skills can have an additional Brand Attached to an Enemy<br>Activates every 0.25 seconds while Attached<br>Increases and Reductions to Cast Speed also apply to this Skill's Activation frequency<br>Deals (13.7-261.2) Base Cold Damage per second<br>Can have an additional Brand Attached to an Enemy<br>Modifiers to Spell Damage apply to this Skill's Damage Over Time effect<br>Can be Detached for a total Duration of 6 seconds<br>Can be Attached for a total Duration of 2 seconds<br>Base Debuff Duration is 1 seconds<br>Debuff deals 20% more Damage per Stage<br>Maximum 20 Stages
 |level1                                  = True
 |level1_level_requirement                = 12
 |level1_cost_amounts                     = 6
