--- The Obscured
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = currency_league_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardTheObscured
 |stack_size                              = 7
-|description                             = {{c|normal|Breachstone}}
+|description                             = {{c|normal|[[Breachstone]]}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_text                               = The Obscured can drop from [[Abyss]] monsters, including from Abyss nodes in [[Delve]].
