--- Wiki
+++ Export
@@ -154,16 +154,13 @@
 |quality_type4_stat2_id                  = rune_paint_damage_+%_final_per_rune_level
 |quality_type4_stat2_value               = -3000
 |stat_text                               = Deals (36-444) to (107-1332) Lightning Damage<br>Base duration is 12 seconds<br>50% reduced Movement Speed<br>Runes have 30% more Area of Effect for each time they have been Improved<br>Runes deal 100% more Damage with Hits and Ailments for each time they have been Improved<br>A Rune is Improved for each (5-24) Mana Spent<br>Runes can be improved 3 times
-|static_cost_types                       = Mana
-|static_critical_strike_chance           = 6
-|static_damage_effectiveness             = 160
-|static_stat1_id                         = is_area_damage
+|static_stat1_id                         = base_skill_show_average_damage_instead_of_dps
 |static_stat1_value                      = 1
-|static_stat2_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat2_id                         = damage_cannot_be_reflected_or_leech_if_used_by_other_object
 |static_stat2_value                      = 1
-|static_stat3_id                         = skill_can_add_multiple_charges_per_action
+|static_stat3_id                         = is_area_damage
 |static_stat3_value                      = 1
-|static_stat4_id                         = damage_cannot_be_reflected_or_leech_if_used_by_other_object
+|static_stat4_id                         = skill_can_add_multiple_charges_per_action
 |static_stat4_value                      = 1
 |static_stat5_id                         = base_skill_effect_duration
 |static_stat5_value                      = 12000
@@ -171,10 +168,10 @@
 |static_stat6_value                      = -50
 |static_stat7_id                         = rune_paint_area_of_effect_+%_final_per_rune_level
 |static_stat7_value                      = 30
-|static_stat8_id                         = rune_paint_damage_+%_final_per_rune_level
-|static_stat8_value                      = 100
-|static_stat9_id                         = quality_display_rune_paint_is_gem
-|static_stat9_value                      = 1
+|static_stat8_id                         = quality_display_rune_paint_is_gem
+|static_stat8_value                      = 1
+|static_stat9_id                         = rune_paint_damage_+%_final_per_rune_level
+|static_stat9_value                      = 100
 |static_stat10_id                        = rune_paint_max_rune_level
 |static_stat10_value                     = 3
 |level1_stat_text                        = Deals 36 to 107 Lightning Damage<br>A Rune is Improved for each 5 Mana Spent
