--- Wiki
+++ Export
@@ -6,7 +6,6 @@
 |size_y                                  = 1
 |flavour_text                            = The others succumbed to madness and depravity, but Lori fought on. Alone in the darkness, she could still see the dull light of hope's distant dawn.
 |drop_level                              = 68
-|drop_level_maximum                      = 68
 |tags                                    = jewellery_divination, unique_divination, unique_jewellery_divination, unique_corrupted_divination, corrupted_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardHope
 |stack_size                              = 5
