--- Tattoo of the Hinekora Warmonger
+++ Export
@@ -11,8 +11,9 @@
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |tattoo_target                           = Small Intelligence
-|tattoo_stat_text                        = [[Minion|Minions]] have 6% increased maximum [[Life]]
-|description                             = Replaces a Small Intelligence Passive Skill<br>Grants "[[Minion|Minions]] have 6% increased maximum [[Life]]"
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Intelligence}} {{c|default|Passive Skill}}
+|tattoo_stat_text                        = Minions have 6% increased maximum Life
+|description                             = Grants "[[Minion|Minions]] have 6% increased maximum [[Life]]"
 |inventory_icon                          = Tattoo intelligence 1
 |drop_text                               = Can be obtained as a reward for defeating the [[Hinekora]] tribe in the [[Trial of the Ancestors]].
 |is_drop_restricted                      = true
