--- Wiki
+++ Export
@@ -9,7 +9,6 @@
 |metadata_id                             = Metadata/Items/Gems/SupportGemImpendingDoom
 |help_text                               = This is a Support Gem. It does not grant a bonus to your character, but to skills in sockets connected to it. Place into an item socket connected to a socket containing the Active Skill Gem you wish to augment. Right click to remove from a socket.
 |intelligence_percent                    = 100
-|static_cost_multiplier                  = 130
 |gem_tags                                = Support, Hex, Spell, AoE, Chaos, Trigger
 |skill_id                                = ViciousHexSupport
 |support_gem_letter                      = X
@@ -100,17 +99,15 @@
 |gem_description                         = Deals chaos damage in an area.
 |active_skill_name                       = Doom Blast
 |cast_time                               = 1.0
-|quality_type1_stat1_id                  = base_skill_area_of_effect_+%
-|quality_type1_stat1_value               = 500
-|quality_type1_stat_text                 = 0.5% increased Area of Effect
+|quality_type1_stat1_id                  = dummy_stat_display_nothing
+|quality_type1_stat1_value               = 0
+|quality_type1_stat_text                 = 
 |quality_type2_stat1_id                  = curse_skill_effect_duration_+%
 |quality_type2_stat1_value               = -2000
 |quality_type2_stat_text                 = Curse Skills have 2% reduced Skill Effect Duration
-|quality_type3_stat1_id                  = base_added_cooldown_count
-|quality_type3_stat1_value               = 50
-|quality_type3_stat2_id                  = base_cooldown_speed_+%
-|quality_type3_stat2_value               = -2000
-|quality_type3_stat_text                 = +.05 Cooldown Use<br>2% reduced Cooldown Recovery Rate
+|quality_type3_stat1_id                  = dummy_stat_display_nothing
+|quality_type3_stat1_value               = 0
+|quality_type3_stat_text                 = 
 |stat_text                               = Trigger Doom Blast when a Hex from Supported Curse Skills ends<br>Deals (102-1654) to (154-2481) Chaos Damage<br>Trigger This Skill when a Hex from Supported Skills ends
 |static_stat1_id                         = trigger_vicious_hex_explosion_when_curse_ends
 |static_stat1_value                      = 1
