--- Vaal Cold Snap
+++ Export
@@ -30,9 +30,9 @@
 |static_duration                         = 4
 |static_stat1_id                         = base_skill_show_average_damage_instead_of_dps
 |static_stat1_value                      = 1
-|static_stat2_id                         = is_area_damage
+|static_stat2_id                         = cannot_cancel_skill_before_contact_point
 |static_stat2_value                      = 1
-|static_stat3_id                         = cannot_cancel_skill_before_contact_point
+|static_stat3_id                         = is_area_damage
 |static_stat3_value                      = 1
 |static_stat4_id                         = base_skill_effect_duration
 |static_stat4_value                      = 4000
@@ -46,7 +46,7 @@
 |static_stat8_value                      = 100
 |static_stat9_id                         = vaal_cold_snap_gain_frenzy_charge_every_second_if_enemy_in_aura
 |static_stat9_value                      = 1
-|stat_text                               = Deals (46-1571) to (69-2356) Cold Damage<br>Deals (37.8-2454.5) Base Cold Damage per second<br>Base duration is 4 seconds<br>Modifiers to Spell Damage apply to this Skill's Damage Over Time effect<br>Modifiers to Skill Effect Duration also apply to this Skill's Soul Gain Prevention<br>Always Freezes enemies<br>(0-19)% increased Effect of Chill<br>Gain a Frenzy Charge when an Enemy Dies while in this Skill's Area
+|stat_text                               = Deals (46-1571) to (69-2356) Cold Damage<br>Base duration is 4 seconds<br>Deals (37.8-2454.5) Base Cold Damage per second<br>Modifiers to Spell Damage apply to this Skill's Damage Over Time effect<br>Modifiers to Skill Effect Duration also apply to this Skill's Soul Gain Prevention<br>Always Freezes enemies<br>(0-19)% increased Effect of Chill<br>Gain a Frenzy Charge when an Enemy Dies while in this Skill's Area
 |level1                                  = True
 |level1_level_requirement                = 16
 |level1_stat_text                        = Deals 46 to 69 Cold Damage<br>Deals 37.8 Base Cold Damage per second
