--- Wiki
+++ Export
@@ -41,9 +41,9 @@
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
 |static_duration                         = 2
-|static_stat1_id                         = is_area_damage
+|static_stat1_id                         = base_skill_show_average_damage_instead_of_dps
 |static_stat1_value                      = 1
-|static_stat2_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat2_id                         = is_area_damage
 |static_stat2_value                      = 1
 |static_stat3_id                         = frost_bomb_damage_+%_final_per_100ms_duration
 |static_stat3_value                      = 4
