--- Wiki
+++ Export
@@ -42,9 +42,9 @@
 |static_damage_effectiveness             = 90
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = is_area_damage
+|static_stat1_id                         = base_skill_show_average_damage_instead_of_dps
 |static_stat1_value                      = 1
-|static_stat2_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat2_id                         = is_area_damage
 |static_stat2_value                      = 1
 |static_stat3_id                         = base_chance_to_ignite_%
 |static_stat3_value                      = 50
