--- Wiki
+++ Export
@@ -17,12 +17,12 @@
 |physical_damage_min                     = 62
 |physical_damage_max                     = 115
 |weapon_range                            = 13
+|release_version                         = 0.10.0
 |quest_reward1_type                      = quest
 |quest_reward1_quest                     = The Storm Blade
 |quest_reward1_quest_id                  = a9q3
 |quest_reward1_act                       = 9
 |quest_reward1_class_ids                 = Duelist,Scion
-|release_version                         = 0.10.0
 }}
 
 '''Ezomyte Blade''' is a [[two-handed sword]] base type.
