--- Tattoo of the Valako Warrior
+++ Export
@@ -11,8 +11,9 @@
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |tattoo_target                           = Small Intelligence
-|tattoo_stat_text                        = Adds 1 to 7 [[Lightning Damage]]
-|description                             = Replaces a Small Intelligence Passive Skill<br>Grants "Adds 1 to 7 [[Lightning Damage]]"
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Intelligence}} {{c|default|Passive Skill}}
+|tattoo_stat_text                        = Adds 1 to 7 Lightning Damage
+|description                             = Grants "Adds 1 to 7 [[Lightning Damage]]"
 |inventory_icon                          = Tattoo intelligence 1
 |drop_text                               = Can be obtained as a reward for defeating the [[Valako]] tribe in the [[Trial of the Ancestors]].
 |is_drop_restricted                      = true