--- Tribute to the Goddess
+++ Export
@@ -8,7 +8,7 @@
 |drop_level                              = 1
 |tags                                    = default
 |metadata_id                             = Metadata/Items/MapFragments/CurrencyOfferingToTheGoddessTribute
-|help_text                               = Travel to the Aspirants' Plaza and spend this item to open the Eternal Labyrinth of Fortune.
+|help_text                               = Can be used in a personal Map Device.
 |drop_text                               = This item can be found in treasure chests at the end of the Eternal and Enriched Eternal Labyrinth.
 |is_drop_restricted                      = true
 |release_version                         = 3.11.0