--- Wiki
+++ Export
@@ -14,7 +14,6 @@
 |help_text                               = Shift click to unstack.
 |drop_areas                              = MapWorldsCrimsonTemple, MapWorldsResidence, MapWorldsIvoryTemple, MapWorldsCrimsonTownship, MapWorldsLaboratory, MapWorldsBasilica, MapWorldsPalace, MapWorldsConservatory, MapWorldsHauntedMansion, MapWorldsVault, MapWorldsVilla, MapWorldsChateau, MapWorldsTemple, MapWorldsRelicChambers, MapWorldsMoonTempleUnique, MapWorldsVaalPyramidUnique
 |release_version                         = 3.6.0
-|card_background                         = 0
 }}
 
 '''The Lord of Celebration''' is a [[divination card]]. A set of four can be exchanged for a random [[Shaper item|Shaper-influenced]] [[magic]] [[sceptre]] with the {{ml|LocalIncreasedAttackSpeed8}}<!-- of Celebration --> suffix.
