--- Silver Timekeeper's Map Device
+++ Export
@@ -0,0 +1,14 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Silver Timekeeper's Map Device
+|class_id                                = HideoutDoodad
+|size_x                                  = 2
+|size_y                                  = 2
+|tags                                    = default
+|metadata_id                             = Metadata/Items/Hideout/HideoutSilverTimekeepersMapDevice
+|stack_size                              = 20
+|description                             = Creates an object in your hideout. This map device tracks how long you've been in an area.
+|help_text                               = Shift click to unstack.
+|is_master_doodad                        = False
+|variation_count                         = 2
+}}