--- Eldritch Exalted Orb
+++ Export
@@ -8,7 +8,7 @@
 |tags                                    = all_eldritch_currency, currency, default
 |metadata_id                             = Metadata/Items/Currency/CurrencyEldritchAddModToRare
 |stack_size                              = 10
-|description                             = If The Searing Exarch is dominant, add a prefix modifier. If The Eater of Worlds is dominant, add a suffix modifier.
+|description                             = If The Searing Exarch is dominant, add a prefix modifier<br>If The Eater of Worlds is dominant, add a suffix modifier
 |help_text                               = Right click this item then left click a rare item with The Searing Exarch or The Eater of Worlds dominance to apply it. Rare items can have up to six random modifiers.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |drop_text                               = Can drop from monsters spawned by [[Searing Exarch]] or [[Eater of Worlds]] influence, or from the [[Eldritch Horrors]].
