--- Ash Winter Plate Helmet
+++ Export
@@ -0,0 +1,11 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Ash Winter Plate Helmet
+|class_id                                = Microtransaction
+|size_x                                  = 2
+|size_y                                  = 2
+|tags                                    = not_for_sale, default
+|metadata_id                             = Metadata/Items/MicrotransactionItemEffects/MicrotransactionAshWinterPlateHelmet
+|stack_size                              = 1
+|description                             = Changes the appearance of any equipped helmet to the Ash Winter Plate Helmet.
+}}