--- Omen of the Soul Devourer
+++ Export
@@ -8,7 +8,7 @@
 |tags                                    = currency, default
 |metadata_id                             = Metadata/Items/Currency/AncestralOmenOnLevelingUpGainSoulEater
 |stack_size                              = 1
-|description                             = Will be Consumed when you level up<br>to grant Soul Eater
+|description                             = Will be Consumed when you level up<br>to grant Soul Eater for 5 minutes
 |help_text                               = Each player can only consume one Omen in each combat area.
 |drop_text                               = Can be obtained as a reward from [[Hinekora]] for completing a tournament in the [[Trial of the Ancestors]].
 |is_drop_restricted                      = true