--- Contract: Findings for Fidium
+++ Export
@@ -8,13 +8,12 @@
 |tags                                    = default
 |metadata_id                             = Metadata/Items/Heist/QuestContracts/HeistContractQuestFaustus2
 |inventory_icon                          = Generic NPC Quest Contract
+|is_drop_restricted                      = true
 |release_version                         = 3.12.0
-|flavour_text                            = "Fidium's fealty to fairness<br/>makes him the perfect pawn in my plan.<br/>Our plan. Perfect pawn in OUR plan."
 |implicit1_text                          = {{c|default|Client:}} {{c|value|Faustus}}
 |implicit2_text                          = {{c|default|Heist Target:}} {{c|value|Plant Darnaw's Papers}}
 |implicit3_text                          = {{c|default|Area Level:}} {{c|value|#}}
 |implicit4_text                          = {{c|default|Requires}} {{c|value|Agility}} {{c|default|(Level}} {{c|value|2}}{{c|default|)}}
-|is_drop_restricted = true
 }}
 
 '''Contract: Findings for Fidium''' is a [[quest]] [[contract]]. The client is [[Faustus]]. It is the second quest of his personal storyline, which {{il|Contract: The Admiral's Records}} being the first. The area level of this contract is depended on the area level of the Smuggler's Caches.
