--- Sulphite Portal Effect
+++ Export
@@ -9,7 +9,7 @@
 |stack_size                              = 1
 |description                             = This character's portals use the Sulphite Portal Effect.
 |cosmetic_type                           = Portal
-|release_version = 3.7.4
+|release_version                         = 3.7.4
 }}
 
 '''Sulphite Portal Effect''' is a [[portal effect]]. it can be purchased from [https://www.pathofexile.com/shop/item/SulphitePortalEffect the shop] for 110 points.