--- Blade Trap
+++ Export
@@ -234,6 +234,7 @@
 |level24                                 = True
 |level24_level_requirement               = 78
 |level24_cost_amounts                    = 21
+|level24_damage_effectiveness            = 100
 |level24_damage_multiplier               = 100.2
 |level24_stat_text                       = +8 to radius
 |level24_stat1_id                        = active_skill_base_radius_+