--- Wiki
+++ Export
@@ -14,7 +14,6 @@
 |help_text                               = Shift click to unstack.
 |drop_areas                              = BreachBossChaos, BreachBossChaos5
 |release_version                         = 3.13.0
-|card_background                         = 1
 }}
 
 '''Keeper's Corruption''' is a [[divination card]]. A set of seven can be exchanged for an [[Elder item|Elder-influenced]] [[magic]] {{il|Bone Helmet}} with [[item level]] 89. The resulting item will also have any one of the [[prefix]]es named "The Elder's" that are eligible to appear on helmets and grants {{c|mod|X% increased Area Damage}} and {{c|mod|Socketed Gems are Supported by Level X Concentrated Effect}}. (See {{section link||List of possible prefixes}}.)
