--- Wiki
+++ Export
@@ -38,7 +38,7 @@
 |static_stat2_value                      = -50
 |static_stat3_id                         = buff_time_passed_+%_other_than_temporal_chains
 |static_stat3_value                      = -40
-|stat_text                               = Base duration is (5-8.8) seconds<br>+(0-10) to radius<br>50% less Effect of Curse against Players<br>Cursed Normal and Magic Enemies have (15-29)% less Action Speed<br>Other effects on Cursed enemies expire 25% slower<br>Cursed Rare and Unique Enemies have (9-18)% less Action Speed
+|stat_text                               = Base duration is (5-8.8) seconds<br>+(0-10) to radius<br>50% less Effect of Curse against Players<br>Cursed Normal and Magic Enemies have (15-29)% less Action Speed<br>Other effects on Cursed enemies expire 40% slower<br>Cursed Rare and Unique Enemies have (9-18)% less Action Speed
 |level1                                  = True
 |level1_level_requirement                = 24
 |level1_cost_amounts                     = 16
