--- Wiki
+++ Export
@@ -0,0 +1,11 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Rescue Hana
+|class_id                                = QuestItem
+|size_x                                  = 1
+|size_y                                  = 1
+|flavour_text                            = "I won't be bullied.<br>We must get Hana back."
+|drop_level                              = 1
+|tags                                    = default
+|metadata_id                             = Metadata/Items/Heist/QuestItems/HeistFinalObjectiveQuestKurai3
+}}