--- Viridian Jewel
+++ Export
@@ -14,14 +14,10 @@
 |quest_reward1_quest                     = Death to Purity
 |quest_reward1_quest_id                  = a5q4
 |quest_reward1_act                       = 5
-|quest_reward1_item_level                = 42
-|quest_reward1_rarity_id                 = rare
 |quest_reward2_type                      = quest
 |quest_reward2_quest                     = The Wings of Vastiri
 |quest_reward2_quest_id                  = a8q4
 |quest_reward2_act                       = 8
-|quest_reward2_item_level                = 57
-|quest_reward2_rarity_id                 = rare
 }}
 
 '''Viridian Jewel''' is a [[jewel]] base type.
