--- Nook's Crown
+++ Export
@@ -14,7 +14,6 @@
 |help_text                               = Shift click to unstack.
 |drop_text                               = [[Catarina, Master of Undeath]] in [[Mastermind's Lair]]
 |release_version                         = 3.8.0
-|card_background                         = 1
 }}
 
 '''Nook's Crown''' is a [[divination card]]. A set of four can be exchanged for an [[Elder item|Elder-influenced]] [[rare]] {{il|Bone Helmet}} with [[item level]] 100.
