--- Wild Strike
+++ Export
@@ -37,11 +37,11 @@
 |static_cost_amounts                     = 6
 |static_cost_types                       = Mana
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = fixed_projectile_spread
-|static_stat1_value                      = 70
-|static_stat2_id                         = show_number_of_projectiles
-|static_stat2_value                      = 1
-|static_stat3_id                         = console_skill_dont_chase
+|static_stat1_id                         = console_skill_dont_chase
+|static_stat1_value                      = 1
+|static_stat2_id                         = fixed_projectile_spread
+|static_stat2_value                      = 70
+|static_stat3_id                         = show_number_of_projectiles
 |static_stat3_value                      = 1
 |static_stat4_id                         = always_pierce
 |static_stat4_value                      = 1
