--- Wiki
+++ Export
@@ -40,9 +40,9 @@
 |static_attack_speed_multiplier          = 80
 |static_stat1_id                         = blood_spears_aoe_modifiers_apply_to_blood_spear_placement_range_at_%_value
 |static_stat1_value                      = 50
-|static_stat2_id                         = is_area_damage
+|static_stat2_id                         = console_skill_dont_chase
 |static_stat2_value                      = 1
-|static_stat3_id                         = console_skill_dont_chase
+|static_stat3_id                         = is_area_damage
 |static_stat3_value                      = 1
 |static_stat4_id                         = blood_spears_additional_number_of_spears_if_changed_stance_recently
 |static_stat4_value                      = 2
