--- Momentum Support
+++ Export
@@ -24,7 +24,7 @@
 |quality_type2_stat1_id                  = support_momentum_movement_speed_+%_per_stack_removed
 |quality_type2_stat1_value               = 100
 |quality_type3_weight                    = 50
-|quality_type3_stat_text                 = -0.005 seconds to time between gaining Momentum while Channelling a Supported Skill
+|quality_type3_stat_text                 = -0.01 seconds to time between gaining Momentum while Channelling a Supported Skill
 |quality_type3_stat1_id                  = support_momentum_stack_while_channelling_base_ms
 |quality_type3_stat1_value               = -5000
 |static_cost_multiplier                  = 110