--- Dialla's Subjugation
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = gems_divination, gems_quality_divination, corrupted_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardDiallasSubjugation
 |stack_size                              = 7
-|description                             = {{c|gem|Support Gem}}<br>{{c|default|Quality:}} {{c|augmented|+23%}}<br>{{c|corrupted|[[Corrupted]]}}
+|description                             = {{c|gem|[[Support Gem]]}}<br>{{c|default|Quality:}} {{c|augmented|+23%}}<br>{{c|corrupted|[[Corrupted]]}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_areas                              = MapAtlasIvoryTemple, MapAtlasTemple, MapWorldsTemple, MapWorldsMoonTempleUnique, MapWorldsColdRiver_, 2_8_12_1, 2_8_13
