--- Wiki
+++ Export
@@ -2,7 +2,7 @@
 {{Base item
 |rarity_id                               = normal
 |name                                    = Ivory Watchstone
-|class_id                                = AtlasRegionUpgradeItem
+|class_id                                = AtlasUpgradeItem
 |size_x                                  = 1
 |size_y                                  = 1
 |flavour_text                            = The memories and emotions of countless victims of the Elder,<br>locked in stone for eternity.
