--- Morphing Incubator
+++ Export
@@ -11,8 +11,8 @@
 |description                             = Adds an incubated Metamorph item to an equippable item<br>Item drops after killing {0} monsters
 |help_text                               = Right click this item then left click an item to apply it. The Incubated item drops after killing a specific number of monsters.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
+|drop_text                               = Drops from Legion encounters.
 |release_version                         = 3.14.0
-|drop_text                               = Drops from Legion encounters.
 }}
 
 '''Morphing Incubator''' is an [[incubator]].
