--- Negotiate with the Client
+++ Export
@@ -0,0 +1,11 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Negotiate with the Client
+|class_id                                = QuestItem
+|size_x                                  = 1
+|size_y                                  = 1
+|flavour_text                            = "An unusual request."
+|drop_level                              = 1
+|tags                                    = default
+|metadata_id                             = Metadata/Items/Heist/QuestItems/HeistFinalObjectiveQuestKurai2
+}}