--- Captured Soul (Shakari upgrade)
+++ Export
@@ -1,6 +1,6 @@
 {{Base item
 |rarity_id                               = normal
-|name                                    = Captured Soul of Terror of the Infinite Drifts
+|name                                    = Captured Soul of Avatar of Undoing
 |class_id                                = PantheonSoul
 |size_x                                  = 1
 |size_y                                  = 1
@@ -10,9 +10,9 @@
 |metadata_id                             = Metadata/Items/PantheonSouls/PantheonSoulShakariUpgrade1
 |help_text                               = A captured Soul. Give to Sin or Right-click to unlock a secondary Pantheon Power.
 |inventory_icon                          = Captured Soul
+|drop_text                               = This item is obtained by activating a [[map device]] with a {{il|Divine Vessel}} along with a {{il|Desert Spring Map}}, and then killing the map boss. The captured soul item will be in whichever map device was used to open the map.
+|is_drop_restricted                      = true
 |release_version                         = 3.1.0
-|is_drop_restricted = true
-|drop_text = This item is obtained by activating a [[map device]] with a {{il|Divine Vessel}} along with a {{il|Desert Spring Map}}, and then killing the map boss. The captured soul item will be in whichever map device was used to open the map.
 }}
 
 '''Captured Soul of Terror of the Infinite Drifts''' is a [[Pantheon soul]]. It can be given to [[Sin]] to upgrade the ''[[Soul of Shakari]]'' [[Pantheon|Pantheon power]].