--- Maven's Invitation: The Forgotten
+++ Export
@@ -10,7 +10,7 @@
 |tags                                    = maven_map, maven_void_map, default
 |metadata_id                             = Metadata/Items/MapFragments/Maven/MavenMapVoid3
 |help_text                               = Open portals to The Maven's Crucible by using this item in a personal Map Device.
-|implicit2                               = MavenInvitation
+|implicit1                               = MavenInvitation
 |inventory_icon                          = Maven's Invitation The Forgotten
 |drop_text                               = Can drop randomly from defeating map bosses. Has an increased chance to drop from a [[Synthesis boss]].
 |is_drop_restricted                      = true