--- Void Emperor Blastchain Mine Effect
+++ Export
@@ -7,7 +7,7 @@
 |tags                                    = not_for_sale, default
 |metadata_id                             = Metadata/Items/MicrotransactionSkillEffects/MicrotransactionCorruptedEmperorRemoteMineEffect
 |stack_size                              = 1
-|description                             = Your Blastchain Mine Support or High-Impact Mine Support becomes a Void Emperor Effect.
+|description                             = Your Blastchain Mine Support, High-Impact Mine Support or Locus Mine Support becomes a Void Emperor Effect.
 |cosmetic_type                           = Skill Gem Effect
 |release_version                         = 3.8.0e
 }}
