--- Wiki
+++ Export
@@ -8,5 +8,5 @@
 |metadata_id                             = Metadata/Items/MicrotransactionItemEffects/MicrotransactionGiftWrappingPaper
 |stack_size                              = 1
 |description                             = Gift wrap an item, which can then be dropped on the ground for anyone to collect.
-|help_text                               = Click this item, then left click the item you wish to gift wrap.
+|help_text                               =  Click this item, then left click the item you wish to gift wrap.
 }}