--- Misery in Darkness
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = armour_divination, unique_divination, unique_armour_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardMiseryInDarkness
 |stack_size                              = 6
-|description                             = {{c|unique|{{iil|Shroud of the Lightless}}}}
+|description                             = {{c|unique|{{il|html=|Shroud of the Lightless}}}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_monsters                           = Metadata/Monsters/ReligiousTemplar/AbyssLichBoss3
