--- Afterimage Flicker Strike Effect
+++ Export
@@ -0,0 +1,11 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Afterimage Flicker Strike Effect
+|class_id                                = Microtransaction
+|size_x                                  = 2
+|size_y                                  = 2
+|tags                                    = not_for_sale, default
+|metadata_id                             = Metadata/Items/MicrotransactionSkillEffects/MicrotransactionAfterimageFlickerStrikeEffect
+|stack_size                              = 1
+|description                             = Your Flicker Strike becomes an Afterimage Effect.
+}}