--- Destined to Crumble
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = armour_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardDestinedToCrumble
 |stack_size                              = 5
-|description                             = {{c|rare|Body Armour}}<br>{{c|default|Item Level:}} {{c|normal|100}}
+|description                             = {{c|rare|[[Body Armour]]}}<br>{{c|default|Item Level:}} {{c|normal|100}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_areas                              = MapAtlasAcademy, MapAtlasMuseum, MapAtlasScriptorium, MapWorldsAcademy, MapWorldsScriptorium, MapWorldsMuseum
