--- Locus Mine Support
+++ Export
@@ -61,7 +61,7 @@
 |static_stat13_value                     = 1
 |static_stat14_id                        = support_remote_mine_2_base_mine_detonation_time_ms
 |static_stat14_value                     = 250
-|stat_text                               = Supported Attack Skills cannot be used with Melee Weapons<br>Supported Skills always fire at least 1 Projectile<br>Supported Skills throw up to 2 additional Mines<br>Supported Skills fire 30% less Projectiles<br>Supported Skills gain a Base Reservation equal to Base Mana and Life Cost<br>Mine lasts 5 seconds<br>Mines from Supported Skills can't be Detonated while within a radius of 30 of you<br>Supported Skills deal (30-11)% less Damage<br>Mines using Supported Skills always target your location<br>Supported Skills throw Mines in an Arc<br>Supported Skills have a Base Mine Detonation Time of 0.25 seconds
+|stat_text                               = Supported Attack Skills cannot be used with Melee Weapons<br>Supported Skills always fire at least 1 Projectile<br>Supported Skills throw up to 2 additional Mines<br>Supported Skills fire 30% less Projectiles<br>Supported Skills gain a Base Reservation equal to Base Mana and Life Cost<br>Mine lasts 5 seconds<br>Mines from Supported Skills can't be Detonated while within 3 metres of you<br>Supported Skills deal (30-11)% less Damage<br>Mines using Supported Skills always target your location<br>Supported Skills throw Mines in an Arc<br>Supported Skills have a Base Mine Detonation Time of 0.25 seconds
 |level1                                  = True
 |level1_level_requirement                = 8
 |level1_stat_text                        = Supported Skills deal 30% less Damage