--- Wiki
+++ Export
@@ -2,13 +2,12 @@
 |rarity_id                               = normal
 |name                                    = Saqawal's Fate Rare Finisher Effect
 |class_id                                = Microtransaction
-|size_x                                  = 1
-|size_y                                  = 1
+|size_x                                  = 2
+|size_y                                  = 2
 |tags                                    = not_for_sale, default
 |metadata_id                             = Metadata/Items/MicrotransactionItemEffects/MicrotransactionSaqawalsFateRareFinisherEffect
 |stack_size                              = 1
-|description                             = Adds the Saqawal's Fate Rare Finisher Effect to any weapon. It amplifies the deaths of Rare Monsters your kill.
-
+|description                             = Adds the Saqawal's Fate Rare Finisher Effect to any weapon. It amplifies the deaths of Rare Monsters you kill.
 |release_version                         = 3.21.0
 |recipe1_part1_item_id                   = Metadata/Items/MicrotransactionCurrency/MysteryBoxCrucible
 |recipe1_part1_amount                    = 1
