--- Butcher Knife
+++ Export
@@ -16,7 +16,7 @@
 |attack_speed                            = 1.4
 |physical_damage_min                     = 7
 |physical_damage_max                     = 62
-|weapon_range                            = 10
+|weapon_range                            = 1
 |quest_reward1_type                      = quest
 |quest_reward1_quest                     = The King's Feast
 |quest_reward1_quest_id                  = a5q5