--- Wiki
+++ Export
@@ -34,9 +34,9 @@
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
 |static_mana_reservation_percent         = 25
-|static_stat1_id                         = blind_art_variation
+|static_stat1_id                         = base_deal_no_damage
 |static_stat1_value                      = 1
-|static_stat2_id                         = base_deal_no_damage
+|static_stat2_id                         = blind_art_variation
 |static_stat2_value                      = 1
 |stat_text                               = Nearby Enemies are Blinded while in Sand Stance<br>You take (11-15)% less Damage from Attacks from Enemies that aren't nearby while in Sand Stance<br>Nearby Enemies are Maimed while in Blood Stance<br>(0-50)% increased Cooldown Recovery Rate<br>Enemies Maimed by this Skill take (8-11)% increased Physical Damage
 |level1                                  = True
