--- Wiki
+++ Export
@@ -8,13 +8,12 @@
 |tags                                    = default
 |metadata_id                             = Metadata/Items/Heist/QuestContracts/HeistContractQuestAdiyah2
 |inventory_icon                          = Generic NPC Quest Contract
+|release_version                         = 3.12.3
 |implicit1_text                          = {{c|default|Client:}} {{c|value|Adiyah}}
-|implicit2_text                          = {{c|default|Heist Target:}} {{c|value|Nashta's War Journal}} 
+|implicit2_text                          = {{c|default|Heist Target:}} {{c|value|Nashta's War Journal}}
 |implicit3_text                          = {{c|default|Area Level:}} {{c|value|(70-#)}}
 |implicit4_text                          = {{c|default|Requires}} {{c|value|Perception }} {{c|default|(Level}} {{c|value|3}}{{c|default|)}}
-|implicit5_text                          = {{c|default|Requires}} {{c|value|Adiyah, the Wayfinder}} 
-|flavour_text                            = "Wraeclast is harsh, and the Plains of Vastiri are unforgiving. We Maraketh can<br/>ill afford an internal war. We must prove that Nashta is behind these attacks."
-|release_version                         = 3.12.3
+|implicit5_text                          = {{c|default|Requires}} {{c|value|Adiyah, the Wayfinder}}
 }}
 
 '''Contract: Flying False Colours''' is a [[quest]] [[contract]]. It is the second step of [[Adiyah, the Wayfinder]]'s quest chain. The quest follows {{il|Contract: A Matter of Honour}} and leads to {{il|Contract: Heart of Glory}}.
