--- Vaal Molten Strike
+++ Export
@@ -42,7 +42,7 @@
 |static_stat8_value                      = 1
 |static_stat9_id                         = molten_strike_projectiles_chain_when_impacting_ground
 |static_stat9_value                      = 1
-|stat_text                               = (2-75) to (3-113) Added Fire Damage<br>+4 to radius<br>Fires 8 additional Projectiles<br>Chains +8 Times<br>60% of Physical Damage Converted to Fire Damage<br>Can't be Evaded<br>Projectiles can Chain when impacting the ground<br>Projectiles do not change direction when Chaining
+|stat_text                               = (2-75) to (3-113) Added Fire Damage<br>+0.4 metres to radius<br>Fires 8 additional Projectiles<br>Chains +8 Times<br>60% of Physical Damage Converted to Fire Damage<br>Can't be Evaded<br>Projectiles can Chain when impacting the ground<br>Projectiles do not change direction when Chaining
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_damage_effectiveness             = 70