--- Wiki
+++ Export
@@ -16,7 +16,6 @@
 |help_text                               = Shift click to unstack.
 |drop_monsters                           = Metadata/Monsters/LeagueHellscape/FleshFaction/HellscapeFleshBoss
 |release_version                         = 3.2.0
-|card_background                         = 0
 }}
 
 '''The World Eater''' is a [[divination card]]. A set of eight can be exchanged for {{il|Starforge}}.
