--- Vaal Rain of Arrows
+++ Export
@@ -26,13 +26,13 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = base_is_projectile
 |static_stat1_value                      = 1
-|static_stat2_id                         = is_area_damage
+|static_stat2_id                         = cannot_cancel_skill_before_contact_point
 |static_stat2_value                      = 1
-|static_stat3_id                         = skill_can_fire_arrows
+|static_stat3_id                         = cannot_pierce
 |static_stat3_value                      = 1
-|static_stat4_id                         = cannot_pierce
+|static_stat4_id                         = is_area_damage
 |static_stat4_value                      = 1
-|static_stat5_id                         = cannot_cancel_skill_before_contact_point
+|static_stat5_id                         = skill_can_fire_arrows
 |static_stat5_value                      = 1
 |static_stat6_id                         = rain_of_arrows_additional_sequences
 |static_stat6_value                      = 4
@@ -178,7 +178,6 @@
 |level19_stat1_value                     = 33
 |level20                                 = True
 |level20_level_requirement               = 70
-|level20_damage_effectiveness            = 100
 |level20_damage_multiplier               = 100.1
 |level20_stat_text                       = Fires 33 additional Arrows
 |level20_stat1_id                        = base_number_of_additional_arrows
