--- Tattoo of the Ramako Fleetfoot
+++ Export
@@ -11,8 +11,9 @@
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |tattoo_target                           = Small Dexterity
-|tattoo_stat_text                        = 2% increased [[Movement Speed]]
-|description                             = Replaces a Small Dexterity Passive Skill<br>Grants "2% increased [[Movement Speed]]"
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Dexterity}} {{c|default|Passive Skill}}
+|tattoo_stat_text                        = 2% increased Movement Speed
+|description                             = Grants "2% increased [[Movement Speed]]"
 |inventory_icon                          = Tattoo dexterity 1
 |drop_text                               = Can be obtained as a reward for defeating the [[Ramako]] tribe in the [[Trial of the Ancestors]].
 |is_drop_restricted                      = true