--- Zealotry
+++ Export
@@ -37,7 +37,7 @@
 |static_stat1_value                      = 1
 |static_stat2_id                         = create_consecrated_ground_on_hit_%_vs_rare_or_unique_enemy
 |static_stat2_value                      = 10
-|stat_text                               = +(0-19) to radius<br>You and nearby Non-Minion Allies have a 10% chance to create Consecrated Ground on Hitting a Rare or Unique Enemy, lasting 8 seconds<br>You and nearby allies have (20-39)% increased Spell Critical Strike Chance<br>You and nearby allies deal (10-15)% more Spell Damage
+|stat_text                               = +(0-1.9) metres to radius<br>You and nearby Non-Minion Allies have a 10% chance to create Consecrated Ground on Hitting a Rare or Unique Enemy, lasting 8 seconds<br>You and nearby allies have (20-39)% increased Spell Critical Strike Chance<br>You and nearby allies deal (10-15)% more Spell Damage
 |level1                                  = True
 |level1_level_requirement                = 24
 |level1_stat_text                        = You and nearby allies have 20% increased Spell Critical Strike Chance<br>You and nearby allies deal 10% more Spell Damage
@@ -47,7 +47,7 @@
 |level1_stat2_value                      = 10
 |level2                                  = True
 |level2_level_requirement                = 27
-|level2_stat_text                        = +1 to radius<br>You and nearby allies have 21% increased Spell Critical Strike Chance<br>You and nearby allies deal 10% more Spell Damage
+|level2_stat_text                        = +0.1 metres to radius<br>You and nearby allies have 21% increased Spell Critical Strike Chance<br>You and nearby allies deal 10% more Spell Damage
 |level2_stat1_id                         = active_skill_base_radius_+
 |level2_stat1_value                      = 1
 |level2_stat2_id                         = spell_critical_strike_chance_+%
@@ -56,7 +56,7 @@
 |level2_stat3_value                      = 10
 |level3                                  = True
 |level3_level_requirement                = 30
-|level3_stat_text                        = +2 to radius<br>You and nearby allies have 22% increased Spell Critical Strike Chance<br>You and nearby allies deal 10% more Spell Damage
+|level3_stat_text                        = +0.2 metres to radius<br>You and nearby allies have 22% increased Spell Critical Strike Chance<br>You and nearby allies deal 10% more Spell Damage
 |level3_stat1_id                         = active_skill_base_radius_+
 |level3_stat1_value                      = 2
 |level3_stat2_id                         = spell_critical_strike_chance_+%
@@ -65,7 +65,7 @@
 |level3_stat3_value                      = 10
 |level4                                  = True
 |level4_level_requirement                = 33
-|level4_stat_text                        = +3 to radius<br>You and nearby allies have 23% increased Spell Critical Strike Chance<br>You and nearby allies deal 11% more Spell Damage
+|level4_stat_text                        = +0.3 metres to radius<br>You and nearby allies have 23% increased Spell Critical Strike Chance<br>You and nearby allies deal 11% more Spell Damage
 |level4_stat1_id                         = active_skill_base_radius_+
 |level4_stat1_value                      = 3
 |level4_stat2_id                         = spell_critical_strike_chance_+%
@@ -74,7 +74,7 @@
 |level4_stat3_value                      = 11
 |level5                                  = True
 |level5_level_requirement                = 36
-|level5_stat_text                        = +4 to radius<br>You and nearby allies have 24% increased Spell Critical Strike Chance<br>You and nearby allies deal 11% more Spell Damage
+|level5_stat_text                        = +0.4 metres to radius<br>You and nearby allies have 24% increased Spell Critical Strike Chance<br>You and nearby allies deal 11% more Spell Damage
 |level5_stat1_id                         = active_skill_base_radius_+
 |level5_stat1_value                      = 4
 |level5_stat2_id                         = spell_critical_strike_chance_+%
@@ -83,7 +83,7 @@
 |level5_stat3_value                      = 11
 |level6                                  = True
 |level6_level_requirement                = 39
-|level6_stat_text                        = +5 to radius<br>You and nearby allies have 25% increased Spell Critical Strike Chance<br>You and nearby allies deal 11% more Spell Damage
+|level6_stat_text                        = +0.5 metres to radius<br>You and nearby allies have 25% increased Spell Critical Strike Chance<br>You and nearby allies deal 11% more Spell Damage
 |level6_stat1_id                         = active_skill_base_radius_+
 |level6_stat1_value                      = 5
 |level6_stat2_id                         = spell_critical_strike_chance_+%
@@ -92,7 +92,7 @@
 |level6_stat3_value                      = 11
 |level7                                  = True
 |level7_level_requirement                = 42
-|level7_stat_text                        = +6 to radius<br>You and nearby allies have 26% increased Spell Critical Strike Chance<br>You and nearby allies deal 12% more Spell Damage
+|level7_stat_text                        = +0.6 metres to radius<br>You and nearby allies have 26% increased Spell Critical Strike Chance<br>You and nearby allies deal 12% more Spell Damage
 |level7_stat1_id                         = active_skill_base_radius_+
 |level7_stat1_value                      = 6
 |level7_stat2_id                         = spell_critical_strike_chance_+%
@@ -101,7 +101,7 @@
 |level7_stat3_value                      = 12
 |level8                                  = True
 |level8_level_requirement                = 45
-|level8_stat_text                        = +7 to radius<br>You and nearby allies have 27% increased Spell Critical Strike Chance<br>You and nearby allies deal 12% more Spell Damage
+|level8_stat_text                        = +0.7 metres to radius<br>You and nearby allies have 27% increased Spell Critical Strike Chance<br>You and nearby allies deal 12% more Spell Damage
 |level8_stat1_id                         = active_skill_base_radius_+
 |level8_stat1_value                      = 7
 |level8_stat2_id                         = spell_critical_strike_chance_+%
@@ -110,7 +110,7 @@
 |level8_stat3_value                      = 12
 |level9                                  = True
 |level9_level_requirement                = 48
-|level9_stat_text                        = +8 to radius<br>You and nearby allies have 28% increased Spell Critical Strike Chance<br>You and nearby allies deal 12% more Spell Damage
+|level9_stat_text                        = +0.8 metres to radius<br>You and nearby allies have 28% increased Spell Critical Strike Chance<br>You and nearby allies deal 12% more Spell Damage
 |level9_stat1_id                         = active_skill_base_radius_+
 |level9_stat1_value                      = 8
 |level9_stat2_id                         = spell_critical_strike_chance_+%
@@ -119,7 +119,7 @@
 |level9_stat3_value                      = 12
 |level10                                 = True
 |level10_level_requirement               = 50
-|level10_stat_text                       = +9 to radius<br>You and nearby allies have 29% increased Spell Critical Strike Chance<br>You and nearby allies deal 12% more Spell Damage
+|level10_stat_text                       = +0.9 metres to radius<br>You and nearby allies have 29% increased Spell Critical Strike Chance<br>You and nearby allies deal 12% more Spell Damage
 |level10_stat1_id                        = active_skill_base_radius_+
 |level10_stat1_value                     = 9
 |level10_stat2_id                        = spell_critical_strike_chance_+%
@@ -128,7 +128,7 @@
 |level10_stat3_value                     = 12
 |level11                                 = True
 |level11_level_requirement               = 52
-|level11_stat_text                       = +10 to radius<br>You and nearby allies have 30% increased Spell Critical Strike Chance<br>You and nearby allies deal 13% more Spell Damage
+|level11_stat_text                       = +1 metre to radius<br>You and nearby allies have 30% increased Spell Critical Strike Chance<br>You and nearby allies deal 13% more Spell Damage
 |level11_stat1_id                        = active_skill_base_radius_+
 |level11_stat1_value                     = 10
 |level11_stat2_id                        = spell_critical_strike_chance_+%
@@ -137,7 +137,7 @@
 |level11_stat3_value                     = 13
 |level12                                 = True
 |level12_level_requirement               = 54
-|level12_stat_text                       = +11 to radius<br>You and nearby allies have 31% increased Spell Critical Strike Chance<br>You and nearby allies deal 13% more Spell Damage
+|level12_stat_text                       = +1.1 metres to radius<br>You and nearby allies have 31% increased Spell Critical Strike Chance<br>You and nearby allies deal 13% more Spell Damage
 |level12_stat1_id                        = active_skill_base_radius_+
 |level12_stat1_value                     = 11
 |level12_stat2_id                        = spell_critical_strike_chance_+%
@@ -146,7 +146,7 @@
 |level12_stat3_value                     = 13
 |level13                                 = True
 |level13_level_requirement               = 56
-|level13_stat_text                       = +12 to radius<br>You and nearby allies have 32% increased Spell Critical Strike Chance<br>You and nearby allies deal 13% more Spell Damage
+|level13_stat_text                       = +1.2 metres to radius<br>You and nearby allies have 32% increased Spell Critical Strike Chance<br>You and nearby allies deal 13% more Spell Damage
 |level13_stat1_id                        = active_skill_base_radius_+
 |level13_stat1_value                     = 12
 |level13_stat2_id                        = spell_critical_strike_chance_+%
@@ -155,7 +155,7 @@
 |level13_stat3_value                     = 13
 |level14                                 = True
 |level14_level_requirement               = 58
-|level14_stat_text                       = +13 to radius<br>You and nearby allies have 33% increased Spell Critical Strike Chance<br>You and nearby allies deal 14% more Spell Damage
+|level14_stat_text                       = +1.3 metres to radius<br>You and nearby allies have 33% increased Spell Critical Strike Chance<br>You and nearby allies deal 14% more Spell Damage
 |level14_stat1_id                        = active_skill_base_radius_+
 |level14_stat1_value                     = 13
 |level14_stat2_id                        = spell_critical_strike_chance_+%
@@ -164,7 +164,7 @@
 |level14_stat3_value                     = 14
 |level15                                 = True
 |level15_level_requirement               = 60
-|level15_stat_text                       = +14 to radius<br>You and nearby allies have 34% increased Spell Critical Strike Chance<br>You and nearby allies deal 14% more Spell Damage
+|level15_stat_text                       = +1.4 metres to radius<br>You and nearby allies have 34% increased Spell Critical Strike Chance<br>You and nearby allies deal 14% more Spell Damage
 |level15_stat1_id                        = active_skill_base_radius_+
 |level15_stat1_value                     = 14
 |level15_stat2_id                        = spell_critical_strike_chance_+%
@@ -173,7 +173,7 @@
 |level15_stat3_value                     = 14
 |level16                                 = True
 |level16_level_requirement               = 62
-|level16_stat_text                       = +15 to radius<br>You and nearby allies have 35% increased Spell Critical Strike Chance<br>You and nearby allies deal 14% more Spell Damage
+|level16_stat_text                       = +1.5 metres to radius<br>You and nearby allies have 35% increased Spell Critical Strike Chance<br>You and nearby allies deal 14% more Spell Damage
 |level16_stat1_id                        = active_skill_base_radius_+
 |level16_stat1_value                     = 15
 |level16_stat2_id                        = spell_critical_strike_chance_+%
@@ -182,7 +182,7 @@
 |level16_stat3_value                     = 14
 |level17                                 = True
 |level17_level_requirement               = 64
-|level17_stat_text                       = +16 to radius<br>You and nearby allies have 36% increased Spell Critical Strike Chance<br>You and nearby allies deal 14% more Spell Damage
+|level17_stat_text                       = +1.6 metres to radius<br>You and nearby allies have 36% increased Spell Critical Strike Chance<br>You and nearby allies deal 14% more Spell Damage
 |level17_stat1_id                        = active_skill_base_radius_+
 |level17_stat1_value                     = 16
 |level17_stat2_id                        = spell_critical_strike_chance_+%
@@ -191,7 +191,7 @@
 |level17_stat3_value                     = 14
 |level18                                 = True
 |level18_level_requirement               = 66
-|level18_stat_text                       = +17 to radius<br>You and nearby allies have 37% increased Spell Critical Strike Chance<br>You and nearby allies deal 15% more Spell Damage
+|level18_stat_text                       = +1.7 metres to radius<br>You and nearby allies have 37% increased Spell Critical Strike Chance<br>You and nearby allies deal 15% more Spell Damage
 |level18_stat1_id                        = active_skill_base_radius_+
 |level18_stat1_value                     = 17
 |level18_stat2_id                        = spell_critical_strike_chance_+%
@@ -200,7 +200,7 @@
 |level18_stat3_value                     = 15
 |level19                                 = True
 |level19_level_requirement               = 68
-|level19_stat_text                       = +18 to radius<br>You and nearby allies have 38% increased Spell Critical Strike Chance<br>You and nearby allies deal 15% more Spell Damage
+|level19_stat_text                       = +1.8 metres to radius<br>You and nearby allies have 38% increased Spell Critical Strike Chance<br>You and nearby allies deal 15% more Spell Damage
 |level19_stat1_id                        = active_skill_base_radius_+
 |level19_stat1_value                     = 18
 |level19_stat2_id                        = spell_critical_strike_chance_+%
@@ -209,7 +209,7 @@
 |level19_stat3_value                     = 15
 |level20                                 = True
 |level20_level_requirement               = 70
-|level20_stat_text                       = +19 to radius<br>You and nearby allies have 39% increased Spell Critical Strike Chance<br>You and nearby allies deal 15% more Spell Damage
+|level20_stat_text                       = +1.9 metres to radius<br>You and nearby allies have 39% increased Spell Critical Strike Chance<br>You and nearby allies deal 15% more Spell Damage
 |level20_stat1_id                        = active_skill_base_radius_+
 |level20_stat1_value                     = 19
 |level20_stat2_id                        = spell_critical_strike_chance_+%
@@ -218,7 +218,7 @@
 |level20_stat3_value                     = 15
 |level21                                 = True
 |level21_level_requirement               = 72
-|level21_stat_text                       = +20 to radius<br>You and nearby allies have 40% increased Spell Critical Strike Chance<br>You and nearby allies deal 16% more Spell Damage
+|level21_stat_text                       = +2 metres to radius<br>You and nearby allies have 40% increased Spell Critical Strike Chance<br>You and nearby allies deal 16% more Spell Damage
 |level21_stat1_id                        = active_skill_base_radius_+
 |level21_stat1_value                     = 20
 |level21_stat2_id                        = spell_critical_strike_chance_+%
@@ -227,7 +227,7 @@
 |level21_stat3_value                     = 16
 |level22                                 = True
 |level22_level_requirement               = 74
-|level22_stat_text                       = +21 to radius<br>You and nearby allies have 41% increased Spell Critical Strike Chance<br>You and nearby allies deal 16% more Spell Damage
+|level22_stat_text                       = +2.1 metres to radius<br>You and nearby allies have 41% increased Spell Critical Strike Chance<br>You and nearby allies deal 16% more Spell Damage
 |level22_stat1_id                        = active_skill_base_radius_+
 |level22_stat1_value                     = 21
 |level22_stat2_id                        = spell_critical_strike_chance_+%
@@ -236,7 +236,7 @@
 |level22_stat3_value                     = 16
 |level23                                 = True
 |level23_level_requirement               = 76
-|level23_stat_text                       = +22 to radius<br>You and nearby allies have 42% increased Spell Critical Strike Chance<br>You and nearby allies deal 16% more Spell Damage
+|level23_stat_text                       = +2.2 metres to radius<br>You and nearby allies have 42% increased Spell Critical Strike Chance<br>You and nearby allies deal 16% more Spell Damage
 |level23_stat1_id                        = active_skill_base_radius_+
 |level23_stat1_value                     = 22
 |level23_stat2_id                        = spell_critical_strike_chance_+%
@@ -245,7 +245,7 @@
 |level23_stat3_value                     = 16
 |level24                                 = True
 |level24_level_requirement               = 78
-|level24_stat_text                       = +23 to radius<br>You and nearby allies have 43% increased Spell Critical Strike Chance<br>You and nearby allies deal 16% more Spell Damage
+|level24_stat_text                       = +2.3 metres to radius<br>You and nearby allies have 43% increased Spell Critical Strike Chance<br>You and nearby allies deal 16% more Spell Damage
 |level24_stat1_id                        = active_skill_base_radius_+
 |level24_stat1_value                     = 23
 |level24_stat2_id                        = spell_critical_strike_chance_+%
@@ -254,7 +254,7 @@
 |level24_stat3_value                     = 16
 |level25                                 = True
 |level25_level_requirement               = 80
-|level25_stat_text                       = +24 to radius<br>You and nearby allies have 44% increased Spell Critical Strike Chance<br>You and nearby allies deal 17% more Spell Damage
+|level25_stat_text                       = +2.4 metres to radius<br>You and nearby allies have 44% increased Spell Critical Strike Chance<br>You and nearby allies deal 17% more Spell Damage
 |level25_stat1_id                        = active_skill_base_radius_+
 |level25_stat1_value                     = 24
 |level25_stat2_id                        = spell_critical_strike_chance_+%
@@ -263,7 +263,7 @@
 |level25_stat3_value                     = 17
 |level26                                 = True
 |level26_level_requirement               = 82
-|level26_stat_text                       = +25 to radius<br>You and nearby allies have 45% increased Spell Critical Strike Chance<br>You and nearby allies deal 17% more Spell Damage
+|level26_stat_text                       = +2.5 metres to radius<br>You and nearby allies have 45% increased Spell Critical Strike Chance<br>You and nearby allies deal 17% more Spell Damage
 |level26_stat1_id                        = active_skill_base_radius_+
 |level26_stat1_value                     = 25
 |level26_stat2_id                        = spell_critical_strike_chance_+%
@@ -272,7 +272,7 @@
 |level26_stat3_value                     = 17
 |level27                                 = True
 |level27_level_requirement               = 84
-|level27_stat_text                       = +26 to radius<br>You and nearby allies have 46% increased Spell Critical Strike Chance<br>You and nearby allies deal 17% more Spell Damage
+|level27_stat_text                       = +2.6 metres to radius<br>You and nearby allies have 46% increased Spell Critical Strike Chance<br>You and nearby allies deal 17% more Spell Damage
 |level27_stat1_id                        = active_skill_base_radius_+
 |level27_stat1_value                     = 26
 |level27_stat2_id                        = spell_critical_strike_chance_+%
@@ -281,7 +281,7 @@
 |level27_stat3_value                     = 17
 |level28                                 = True
 |level28_level_requirement               = 86
-|level28_stat_text                       = +27 to radius<br>You and nearby allies have 47% increased Spell Critical Strike Chance<br>You and nearby allies deal 18% more Spell Damage
+|level28_stat_text                       = +2.7 metres to radius<br>You and nearby allies have 47% increased Spell Critical Strike Chance<br>You and nearby allies deal 18% more Spell Damage
 |level28_stat1_id                        = active_skill_base_radius_+
 |level28_stat1_value                     = 27
 |level28_stat2_id                        = spell_critical_strike_chance_+%
@@ -290,7 +290,7 @@
 |level28_stat3_value                     = 18
 |level29                                 = True
 |level29_level_requirement               = 88
-|level29_stat_text                       = +28 to radius<br>You and nearby allies have 48% increased Spell Critical Strike Chance<br>You and nearby allies deal 18% more Spell Damage
+|level29_stat_text                       = +2.8 metres to radius<br>You and nearby allies have 48% increased Spell Critical Strike Chance<br>You and nearby allies deal 18% more Spell Damage
 |level29_stat1_id                        = active_skill_base_radius_+
 |level29_stat1_value                     = 28
 |level29_stat2_id                        = spell_critical_strike_chance_+%
@@ -299,7 +299,7 @@
 |level29_stat3_value                     = 18
 |level30                                 = True
 |level30_level_requirement               = 90
-|level30_stat_text                       = +29 to radius<br>You and nearby allies have 49% increased Spell Critical Strike Chance<br>You and nearby allies deal 18% more Spell Damage
+|level30_stat_text                       = +2.9 metres to radius<br>You and nearby allies have 49% increased Spell Critical Strike Chance<br>You and nearby allies deal 18% more Spell Damage
 |level30_stat1_id                        = active_skill_base_radius_+
 |level30_stat1_value                     = 29
 |level30_stat2_id                        = spell_critical_strike_chance_+%
@@ -308,7 +308,7 @@
 |level30_stat3_value                     = 18
 |level31                                 = True
 |level31_level_requirement               = 91
-|level31_stat_text                       = +29 to radius<br>You and nearby allies have 49% increased Spell Critical Strike Chance<br>You and nearby allies deal 18% more Spell Damage
+|level31_stat_text                       = +2.9 metres to radius<br>You and nearby allies have 49% increased Spell Critical Strike Chance<br>You and nearby allies deal 18% more Spell Damage
 |level31_stat1_id                        = active_skill_base_radius_+
 |level31_stat1_value                     = 29
 |level31_stat2_id                        = spell_critical_strike_chance_+%
@@ -317,7 +317,7 @@
 |level31_stat3_value                     = 18
 |level32                                 = True
 |level32_level_requirement               = 92
-|level32_stat_text                       = +30 to radius<br>You and nearby allies have 50% increased Spell Critical Strike Chance<br>You and nearby allies deal 18% more Spell Damage
+|level32_stat_text                       = +3 metres to radius<br>You and nearby allies have 50% increased Spell Critical Strike Chance<br>You and nearby allies deal 18% more Spell Damage
 |level32_stat1_id                        = active_skill_base_radius_+
 |level32_stat1_value                     = 30
 |level32_stat2_id                        = spell_critical_strike_chance_+%
@@ -326,7 +326,7 @@
 |level32_stat3_value                     = 18
 |level33                                 = True
 |level33_level_requirement               = 93
-|level33_stat_text                       = +30 to radius<br>You and nearby allies have 50% increased Spell Critical Strike Chance<br>You and nearby allies deal 19% more Spell Damage
+|level33_stat_text                       = +3 metres to radius<br>You and nearby allies have 50% increased Spell Critical Strike Chance<br>You and nearby allies deal 19% more Spell Damage
 |level33_stat1_id                        = active_skill_base_radius_+
 |level33_stat1_value                     = 30
 |level33_stat2_id                        = spell_critical_strike_chance_+%
@@ -335,7 +335,7 @@
 |level33_stat3_value                     = 19
 |level34                                 = True
 |level34_level_requirement               = 94
-|level34_stat_text                       = +31 to radius<br>You and nearby allies have 51% increased Spell Critical Strike Chance<br>You and nearby allies deal 19% more Spell Damage
+|level34_stat_text                       = +3.1 metres to radius<br>You and nearby allies have 51% increased Spell Critical Strike Chance<br>You and nearby allies deal 19% more Spell Damage
 |level34_stat1_id                        = active_skill_base_radius_+
 |level34_stat1_value                     = 31
 |level34_stat2_id                        = spell_critical_strike_chance_+%
@@ -344,7 +344,7 @@
 |level34_stat3_value                     = 19
 |level35                                 = True
 |level35_level_requirement               = 95
-|level35_stat_text                       = +31 to radius<br>You and nearby allies have 51% increased Spell Critical Strike Chance<br>You and nearby allies deal 19% more Spell Damage
+|level35_stat_text                       = +3.1 metres to radius<br>You and nearby allies have 51% increased Spell Critical Strike Chance<br>You and nearby allies deal 19% more Spell Damage
 |level35_stat1_id                        = active_skill_base_radius_+
 |level35_stat1_value                     = 31
 |level35_stat2_id                        = spell_critical_strike_chance_+%
@@ -353,7 +353,7 @@
 |level35_stat3_value                     = 19
 |level36                                 = True
 |level36_level_requirement               = 96
-|level36_stat_text                       = +32 to radius<br>You and nearby allies have 52% increased Spell Critical Strike Chance<br>You and nearby allies deal 19% more Spell Damage
+|level36_stat_text                       = +3.2 metres to radius<br>You and nearby allies have 52% increased Spell Critical Strike Chance<br>You and nearby allies deal 19% more Spell Damage
 |level36_stat1_id                        = active_skill_base_radius_+
 |level36_stat1_value                     = 32
 |level36_stat2_id                        = spell_critical_strike_chance_+%
@@ -362,7 +362,7 @@
 |level36_stat3_value                     = 19
 |level37                                 = True
 |level37_level_requirement               = 97
-|level37_stat_text                       = +32 to radius<br>You and nearby allies have 52% increased Spell Critical Strike Chance<br>You and nearby allies deal 19% more Spell Damage
+|level37_stat_text                       = +3.2 metres to radius<br>You and nearby allies have 52% increased Spell Critical Strike Chance<br>You and nearby allies deal 19% more Spell Damage
 |level37_stat1_id                        = active_skill_base_radius_+
 |level37_stat1_value                     = 32
 |level37_stat2_id                        = spell_critical_strike_chance_+%
@@ -371,7 +371,7 @@
 |level37_stat3_value                     = 19
 |level38                                 = True
 |level38_level_requirement               = 98
-|level38_stat_text                       = +33 to radius<br>You and nearby allies have 53% increased Spell Critical Strike Chance<br>You and nearby allies deal 19% more Spell Damage
+|level38_stat_text                       = +3.3 metres to radius<br>You and nearby allies have 53% increased Spell Critical Strike Chance<br>You and nearby allies deal 19% more Spell Damage
 |level38_stat1_id                        = active_skill_base_radius_+
 |level38_stat1_value                     = 33
 |level38_stat2_id                        = spell_critical_strike_chance_+%
@@ -380,7 +380,7 @@
 |level38_stat3_value                     = 19
 |level39                                 = True
 |level39_level_requirement               = 99
-|level39_stat_text                       = +33 to radius<br>You and nearby allies have 53% increased Spell Critical Strike Chance<br>You and nearby allies deal 19% more Spell Damage
+|level39_stat_text                       = +3.3 metres to radius<br>You and nearby allies have 53% increased Spell Critical Strike Chance<br>You and nearby allies deal 19% more Spell Damage
 |level39_stat1_id                        = active_skill_base_radius_+
 |level39_stat1_value                     = 33
 |level39_stat2_id                        = spell_critical_strike_chance_+%
@@ -389,7 +389,7 @@
 |level39_stat3_value                     = 19
 |level40                                 = True
 |level40_level_requirement               = 100
-|level40_stat_text                       = +34 to radius<br>You and nearby allies have 54% increased Spell Critical Strike Chance<br>You and nearby allies deal 20% more Spell Damage
+|level40_stat_text                       = +3.4 metres to radius<br>You and nearby allies have 54% increased Spell Critical Strike Chance<br>You and nearby allies deal 20% more Spell Damage
 |level40_stat1_id                        = active_skill_base_radius_+
 |level40_stat1_value                     = 34
 |level40_stat2_id                        = spell_critical_strike_chance_+%