--- Wiki
+++ Export
@@ -7,7 +7,7 @@
 |tags                                    = not_for_sale, default
 |metadata_id                             = Metadata/Items/MicrotransactionItemEffects/MicrotransactionEngineersBronzeBodyArmour
 |stack_size                              = 1
-|description                             = Changes the appearance of any equipped gloves to the Engineer's Bronze Body Armour.
+|description                             = Changes the appearance of any equipped body armour to the Engineer's Bronze Body Armour.
 |cosmetic_type                           = Armour Skin
 |release_version                         = 3.21.0
 |recipe1_part1_item_id                   = Metadata/Items/MicrotransactionCurrency/MysteryBoxCrucible
