--- Firestorm
+++ Export
@@ -33,12 +33,12 @@
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
 |static_duration                         = 1.4
-|static_stat1_id                         = skill_override_pvp_scaling_time_ms
-|static_stat1_value                      = 450
-|static_stat2_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat1_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat1_value                      = 1
+|static_stat2_id                         = is_area_damage
 |static_stat2_value                      = 1
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
+|static_stat3_id                         = skill_override_pvp_scaling_time_ms
+|static_stat3_value                      = 450
 |static_stat4_id                         = base_skill_effect_duration
 |static_stat4_value                      = 1400
 |static_stat5_id                         = fire_storm_fireball_delay_ms
@@ -49,7 +49,7 @@
 |static_stat7_value                      = 325
 |static_stat8_id                         = firestorm_max_number_of_storms
 |static_stat8_value                      = 3
-|stat_text                               = Deals (23-361) to (35-541) Fire Damage<br>Base duration is 1.4 seconds<br>One impact every 0.15 seconds<br>First Impact has 100% more Area of Effect<br>First Impact deals 325% more Damage with Hits and Ailments<br>Maximum 3 Firestorms at a time
+|stat_text                               = Base duration is 1.4 seconds<br>Deals (23-361) to (35-541) Fire Damage<br>One impact every 0.15 seconds<br>First Impact has 100% more Area of Effect<br>First Impact deals 325% more Damage with Hits and Ailments<br>Maximum 3 Firestorms at a time
 |level1                                  = True
 |level1_level_requirement                = 28
 |level1_cost_amounts                     = 13
