--- Gouger
+++ Export
@@ -18,7 +18,7 @@
 |attack_speed                            = 1.5
 |physical_damage_min                     = 15
 |physical_damage_max                     = 51
-|weapon_range                            = 11
+|weapon_range                            = 1.1
 |quest_reward1_type                      = quest
 |quest_reward1_quest                     = The King's Feast
 |quest_reward1_quest_id                  = a5q5