--- Wiki
+++ Export
@@ -7,5 +7,5 @@
 |tags                                    = not_for_sale, default
 |metadata_id                             = Metadata/Items/MicrotransactionItemEffects/MicrotransactionReaperCloak
 |stack_size                              = 1
-|description                             = Adds The Soul Harvester's Cloak back attachment to any equipped body armour. Type /kills to see the number of monsters you have killed.
+|description                             = Adds The Soul Harvester's Cloak back attachment to any equipped body armour, its power is based on the number of enemies your character has killed. Type "/kills" to see your character's kill count.
 }}