--- Summon Reaper
+++ Export
@@ -35,16 +35,18 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = display_minion_monster_type
 |static_stat1_value                      = 18
-|static_stat2_id                         = base_number_of_reapers_allowed
+|static_stat2_id                         = infinite_minion_duration
 |static_stat2_value                      = 1
-|static_stat3_id                         = bleed_on_hit_with_attacks_%
-|static_stat3_value                      = 50
+|static_stat3_id                         = base_number_of_reapers_allowed
+|static_stat3_value                      = 1
+|static_stat4_id                         = bleed_on_hit_with_attacks_%
+|static_stat4_value                      = 50
 |stat_text                               = (20-39)% increased Minion Maximum Life<br>Maximum 1 Summoned Reaper<br>Minion's Attacks have 50% chance to inflict Bleeding<br>Minion deals (30-87)% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal (20-29)% less Damage<br>Reaper causes your Non-Reaper Minions to have (20-29)% less Maximum Life
 |level1                                  = True
 |level1_level_requirement                = 28
 |level1_cost_amounts                     = 15
 |level1_stat_text                        = 20% increased Minion Maximum Life<br>Minion deals 30% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 20% less Damage<br>Reaper causes your Non-Reaper Minions to have 20% less Maximum Life
-|level1_stat1_id                         = display_minion_monster_level
+|level1_stat1_id                         = base_display_minion_actor_level
 |level1_stat1_value                      = 28
 |level1_stat2_id                         = minion_maximum_life_+%
 |level1_stat2_value                      = 20
@@ -58,7 +60,7 @@
 |level2_level_requirement                = 31
 |level2_cost_amounts                     = 16
 |level2_stat_text                        = 21% increased Minion Maximum Life<br>Minion deals 33% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 20% less Damage<br>Reaper causes your Non-Reaper Minions to have 20% less Maximum Life
-|level2_stat1_id                         = display_minion_monster_level
+|level2_stat1_id                         = base_display_minion_actor_level
 |level2_stat1_value                      = 31
 |level2_stat2_id                         = minion_maximum_life_+%
 |level2_stat2_value                      = 21
@@ -72,7 +74,7 @@
 |level3_level_requirement                = 34
 |level3_cost_amounts                     = 17
 |level3_stat_text                        = 22% increased Minion Maximum Life<br>Minion deals 36% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 21% less Damage<br>Reaper causes your Non-Reaper Minions to have 21% less Maximum Life
-|level3_stat1_id                         = display_minion_monster_level
+|level3_stat1_id                         = base_display_minion_actor_level
 |level3_stat1_value                      = 34
 |level3_stat2_id                         = minion_maximum_life_+%
 |level3_stat2_value                      = 22
@@ -86,7 +88,7 @@
 |level4_level_requirement                = 37
 |level4_cost_amounts                     = 18
 |level4_stat_text                        = 23% increased Minion Maximum Life<br>Minion deals 39% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 21% less Damage<br>Reaper causes your Non-Reaper Minions to have 21% less Maximum Life
-|level4_stat1_id                         = display_minion_monster_level
+|level4_stat1_id                         = base_display_minion_actor_level
 |level4_stat1_value                      = 37
 |level4_stat2_id                         = minion_maximum_life_+%
 |level4_stat2_value                      = 23
@@ -100,7 +102,7 @@
 |level5_level_requirement                = 40
 |level5_cost_amounts                     = 19
 |level5_stat_text                        = 24% increased Minion Maximum Life<br>Minion deals 42% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 22% less Damage<br>Reaper causes your Non-Reaper Minions to have 22% less Maximum Life
-|level5_stat1_id                         = display_minion_monster_level
+|level5_stat1_id                         = base_display_minion_actor_level
 |level5_stat1_value                      = 40
 |level5_stat2_id                         = minion_maximum_life_+%
 |level5_stat2_value                      = 24
@@ -114,7 +116,7 @@
 |level6_level_requirement                = 42
 |level6_cost_amounts                     = 19
 |level6_stat_text                        = 25% increased Minion Maximum Life<br>Minion deals 45% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 22% less Damage<br>Reaper causes your Non-Reaper Minions to have 22% less Maximum Life
-|level6_stat1_id                         = display_minion_monster_level
+|level6_stat1_id                         = base_display_minion_actor_level
 |level6_stat1_value                      = 42
 |level6_stat2_id                         = minion_maximum_life_+%
 |level6_stat2_value                      = 25
@@ -128,7 +130,7 @@
 |level7_level_requirement                = 44
 |level7_cost_amounts                     = 20
 |level7_stat_text                        = 26% increased Minion Maximum Life<br>Minion deals 48% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 23% less Damage<br>Reaper causes your Non-Reaper Minions to have 23% less Maximum Life
-|level7_stat1_id                         = display_minion_monster_level
+|level7_stat1_id                         = base_display_minion_actor_level
 |level7_stat1_value                      = 44
 |level7_stat2_id                         = minion_maximum_life_+%
 |level7_stat2_value                      = 26
@@ -142,7 +144,7 @@
 |level8_level_requirement                = 46
 |level8_cost_amounts                     = 21
 |level8_stat_text                        = 27% increased Minion Maximum Life<br>Minion deals 51% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 23% less Damage<br>Reaper causes your Non-Reaper Minions to have 23% less Maximum Life
-|level8_stat1_id                         = display_minion_monster_level
+|level8_stat1_id                         = base_display_minion_actor_level
 |level8_stat1_value                      = 46
 |level8_stat2_id                         = minion_maximum_life_+%
 |level8_stat2_value                      = 27
@@ -156,7 +158,7 @@
 |level9_level_requirement                = 48
 |level9_cost_amounts                     = 21
 |level9_stat_text                        = 28% increased Minion Maximum Life<br>Minion deals 54% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 24% less Damage<br>Reaper causes your Non-Reaper Minions to have 24% less Maximum Life
-|level9_stat1_id                         = display_minion_monster_level
+|level9_stat1_id                         = base_display_minion_actor_level
 |level9_stat1_value                      = 48
 |level9_stat2_id                         = minion_maximum_life_+%
 |level9_stat2_value                      = 28
@@ -170,7 +172,7 @@
 |level10_level_requirement               = 50
 |level10_cost_amounts                    = 22
 |level10_stat_text                       = 29% increased Minion Maximum Life<br>Minion deals 57% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 24% less Damage<br>Reaper causes your Non-Reaper Minions to have 24% less Maximum Life
-|level10_stat1_id                        = display_minion_monster_level
+|level10_stat1_id                        = base_display_minion_actor_level
 |level10_stat1_value                     = 50
 |level10_stat2_id                        = minion_maximum_life_+%
 |level10_stat2_value                     = 29
@@ -184,7 +186,7 @@
 |level11_level_requirement               = 52
 |level11_cost_amounts                    = 22
 |level11_stat_text                       = 30% increased Minion Maximum Life<br>Minion deals 60% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 25% less Damage<br>Reaper causes your Non-Reaper Minions to have 25% less Maximum Life
-|level11_stat1_id                        = display_minion_monster_level
+|level11_stat1_id                        = base_display_minion_actor_level
 |level11_stat1_value                     = 52
 |level11_stat2_id                        = minion_maximum_life_+%
 |level11_stat2_value                     = 30
@@ -198,7 +200,7 @@
 |level12_level_requirement               = 54
 |level12_cost_amounts                    = 23
 |level12_stat_text                       = 31% increased Minion Maximum Life<br>Minion deals 63% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 25% less Damage<br>Reaper causes your Non-Reaper Minions to have 25% less Maximum Life
-|level12_stat1_id                        = display_minion_monster_level
+|level12_stat1_id                        = base_display_minion_actor_level
 |level12_stat1_value                     = 54
 |level12_stat2_id                        = minion_maximum_life_+%
 |level12_stat2_value                     = 31
@@ -212,7 +214,7 @@
 |level13_level_requirement               = 56
 |level13_cost_amounts                    = 24
 |level13_stat_text                       = 32% increased Minion Maximum Life<br>Minion deals 66% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 26% less Damage<br>Reaper causes your Non-Reaper Minions to have 26% less Maximum Life
-|level13_stat1_id                        = display_minion_monster_level
+|level13_stat1_id                        = base_display_minion_actor_level
 |level13_stat1_value                     = 56
 |level13_stat2_id                        = minion_maximum_life_+%
 |level13_stat2_value                     = 32
@@ -226,7 +228,7 @@
 |level14_level_requirement               = 58
 |level14_cost_amounts                    = 24
 |level14_stat_text                       = 33% increased Minion Maximum Life<br>Minion deals 69% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 26% less Damage<br>Reaper causes your Non-Reaper Minions to have 26% less Maximum Life
-|level14_stat1_id                        = display_minion_monster_level
+|level14_stat1_id                        = base_display_minion_actor_level
 |level14_stat1_value                     = 58
 |level14_stat2_id                        = minion_maximum_life_+%
 |level14_stat2_value                     = 33
@@ -240,7 +242,7 @@
 |level15_level_requirement               = 60
 |level15_cost_amounts                    = 25
 |level15_stat_text                       = 34% increased Minion Maximum Life<br>Minion deals 72% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 27% less Damage<br>Reaper causes your Non-Reaper Minions to have 27% less Maximum Life
-|level15_stat1_id                        = display_minion_monster_level
+|level15_stat1_id                        = base_display_minion_actor_level
 |level15_stat1_value                     = 60
 |level15_stat2_id                        = minion_maximum_life_+%
 |level15_stat2_value                     = 34
@@ -254,7 +256,7 @@
 |level16_level_requirement               = 62
 |level16_cost_amounts                    = 25
 |level16_stat_text                       = 35% increased Minion Maximum Life<br>Minion deals 75% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 27% less Damage<br>Reaper causes your Non-Reaper Minions to have 27% less Maximum Life
-|level16_stat1_id                        = display_minion_monster_level
+|level16_stat1_id                        = base_display_minion_actor_level
 |level16_stat1_value                     = 62
 |level16_stat2_id                        = minion_maximum_life_+%
 |level16_stat2_value                     = 35
@@ -268,7 +270,7 @@
 |level17_level_requirement               = 64
 |level17_cost_amounts                    = 26
 |level17_stat_text                       = 36% increased Minion Maximum Life<br>Minion deals 78% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 28% less Damage<br>Reaper causes your Non-Reaper Minions to have 28% less Maximum Life
-|level17_stat1_id                        = display_minion_monster_level
+|level17_stat1_id                        = base_display_minion_actor_level
 |level17_stat1_value                     = 64
 |level17_stat2_id                        = minion_maximum_life_+%
 |level17_stat2_value                     = 36
@@ -282,7 +284,7 @@
 |level18_level_requirement               = 66
 |level18_cost_amounts                    = 27
 |level18_stat_text                       = 37% increased Minion Maximum Life<br>Minion deals 81% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 28% less Damage<br>Reaper causes your Non-Reaper Minions to have 28% less Maximum Life
-|level18_stat1_id                        = display_minion_monster_level
+|level18_stat1_id                        = base_display_minion_actor_level
 |level18_stat1_value                     = 66
 |level18_stat2_id                        = minion_maximum_life_+%
 |level18_stat2_value                     = 37
@@ -296,7 +298,7 @@
 |level19_level_requirement               = 68
 |level19_cost_amounts                    = 27
 |level19_stat_text                       = 38% increased Minion Maximum Life<br>Minion deals 84% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 29% less Damage<br>Reaper causes your Non-Reaper Minions to have 29% less Maximum Life
-|level19_stat1_id                        = display_minion_monster_level
+|level19_stat1_id                        = base_display_minion_actor_level
 |level19_stat1_value                     = 68
 |level19_stat2_id                        = minion_maximum_life_+%
 |level19_stat2_value                     = 38
@@ -310,7 +312,7 @@
 |level20_level_requirement               = 70
 |level20_cost_amounts                    = 28
 |level20_stat_text                       = 39% increased Minion Maximum Life<br>Minion deals 87% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 29% less Damage<br>Reaper causes your Non-Reaper Minions to have 29% less Maximum Life
-|level20_stat1_id                        = display_minion_monster_level
+|level20_stat1_id                        = base_display_minion_actor_level
 |level20_stat1_value                     = 70
 |level20_stat2_id                        = minion_maximum_life_+%
 |level20_stat2_value                     = 39
@@ -324,7 +326,7 @@
 |level21_level_requirement               = 72
 |level21_cost_amounts                    = 29
 |level21_stat_text                       = 40% increased Minion Maximum Life<br>Minion deals 90% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 30% less Damage<br>Reaper causes your Non-Reaper Minions to have 30% less Maximum Life
-|level21_stat1_id                        = display_minion_monster_level
+|level21_stat1_id                        = base_display_minion_actor_level
 |level21_stat1_value                     = 72
 |level21_stat2_id                        = minion_maximum_life_+%
 |level21_stat2_value                     = 40
@@ -338,7 +340,7 @@
 |level22_level_requirement               = 74
 |level22_cost_amounts                    = 29
 |level22_stat_text                       = 41% increased Minion Maximum Life<br>Minion deals 93% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 30% less Damage<br>Reaper causes your Non-Reaper Minions to have 30% less Maximum Life
-|level22_stat1_id                        = display_minion_monster_level
+|level22_stat1_id                        = base_display_minion_actor_level
 |level22_stat1_value                     = 74
 |level22_stat2_id                        = minion_maximum_life_+%
 |level22_stat2_value                     = 41
@@ -352,7 +354,7 @@
 |level23_level_requirement               = 76
 |level23_cost_amounts                    = 30
 |level23_stat_text                       = 42% increased Minion Maximum Life<br>Minion deals 96% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 31% less Damage<br>Reaper causes your Non-Reaper Minions to have 31% less Maximum Life
-|level23_stat1_id                        = display_minion_monster_level
+|level23_stat1_id                        = base_display_minion_actor_level
 |level23_stat1_value                     = 76
 |level23_stat2_id                        = minion_maximum_life_+%
 |level23_stat2_value                     = 42
@@ -366,7 +368,7 @@
 |level24_level_requirement               = 78
 |level24_cost_amounts                    = 30
 |level24_stat_text                       = 43% increased Minion Maximum Life<br>Minion deals 99% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 31% less Damage<br>Reaper causes your Non-Reaper Minions to have 31% less Maximum Life
-|level24_stat1_id                        = display_minion_monster_level
+|level24_stat1_id                        = base_display_minion_actor_level
 |level24_stat1_value                     = 78
 |level24_stat2_id                        = minion_maximum_life_+%
 |level24_stat2_value                     = 43
@@ -380,7 +382,7 @@
 |level25_level_requirement               = 80
 |level25_cost_amounts                    = 31
 |level25_stat_text                       = 44% increased Minion Maximum Life<br>Minion deals 102% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 32% less Damage<br>Reaper causes your Non-Reaper Minions to have 32% less Maximum Life
-|level25_stat1_id                        = display_minion_monster_level
+|level25_stat1_id                        = base_display_minion_actor_level
 |level25_stat1_value                     = 80
 |level25_stat2_id                        = minion_maximum_life_+%
 |level25_stat2_value                     = 44
@@ -394,7 +396,7 @@
 |level26_level_requirement               = 82
 |level26_cost_amounts                    = 32
 |level26_stat_text                       = 45% increased Minion Maximum Life<br>Minion deals 105% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 32% less Damage<br>Reaper causes your Non-Reaper Minions to have 32% less Maximum Life
-|level26_stat1_id                        = display_minion_monster_level
+|level26_stat1_id                        = base_display_minion_actor_level
 |level26_stat1_value                     = 82
 |level26_stat2_id                        = minion_maximum_life_+%
 |level26_stat2_value                     = 45
@@ -408,7 +410,7 @@
 |level27_level_requirement               = 84
 |level27_cost_amounts                    = 32
 |level27_stat_text                       = 46% increased Minion Maximum Life<br>Minion deals 108% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 33% less Damage<br>Reaper causes your Non-Reaper Minions to have 33% less Maximum Life
-|level27_stat1_id                        = display_minion_monster_level
+|level27_stat1_id                        = base_display_minion_actor_level
 |level27_stat1_value                     = 84
 |level27_stat2_id                        = minion_maximum_life_+%
 |level27_stat2_value                     = 46
@@ -422,7 +424,7 @@
 |level28_level_requirement               = 86
 |level28_cost_amounts                    = 33
 |level28_stat_text                       = 47% increased Minion Maximum Life<br>Minion deals 111% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 33% less Damage<br>Reaper causes your Non-Reaper Minions to have 33% less Maximum Life
-|level28_stat1_id                        = display_minion_monster_level
+|level28_stat1_id                        = base_display_minion_actor_level
 |level28_stat1_value                     = 86
 |level28_stat2_id                        = minion_maximum_life_+%
 |level28_stat2_value                     = 47
@@ -436,7 +438,7 @@
 |level29_level_requirement               = 88
 |level29_cost_amounts                    = 33
 |level29_stat_text                       = 48% increased Minion Maximum Life<br>Minion deals 114% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 34% less Damage<br>Reaper causes your Non-Reaper Minions to have 34% less Maximum Life
-|level29_stat1_id                        = display_minion_monster_level
+|level29_stat1_id                        = base_display_minion_actor_level
 |level29_stat1_value                     = 88
 |level29_stat2_id                        = minion_maximum_life_+%
 |level29_stat2_value                     = 48
@@ -450,7 +452,7 @@
 |level30_level_requirement               = 90
 |level30_cost_amounts                    = 34
 |level30_stat_text                       = 49% increased Minion Maximum Life<br>Minion deals 117% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 34% less Damage<br>Reaper causes your Non-Reaper Minions to have 34% less Maximum Life
-|level30_stat1_id                        = display_minion_monster_level
+|level30_stat1_id                        = base_display_minion_actor_level
 |level30_stat1_value                     = 90
 |level30_stat2_id                        = minion_maximum_life_+%
 |level30_stat2_value                     = 49
@@ -464,7 +466,7 @@
 |level31_level_requirement               = 91
 |level31_cost_amounts                    = 34
 |level31_stat_text                       = 49% increased Minion Maximum Life<br>Minion deals 118% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 34% less Damage<br>Reaper causes your Non-Reaper Minions to have 34% less Maximum Life
-|level31_stat1_id                        = display_minion_monster_level
+|level31_stat1_id                        = base_display_minion_actor_level
 |level31_stat1_value                     = 91
 |level31_stat2_id                        = minion_maximum_life_+%
 |level31_stat2_value                     = 49
@@ -478,7 +480,7 @@
 |level32_level_requirement               = 92
 |level32_cost_amounts                    = 35
 |level32_stat_text                       = 50% increased Minion Maximum Life<br>Minion deals 120% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 35% less Damage<br>Reaper causes your Non-Reaper Minions to have 35% less Maximum Life
-|level32_stat1_id                        = display_minion_monster_level
+|level32_stat1_id                        = base_display_minion_actor_level
 |level32_stat1_value                     = 92
 |level32_stat2_id                        = minion_maximum_life_+%
 |level32_stat2_value                     = 50
@@ -492,7 +494,7 @@
 |level33_level_requirement               = 93
 |level33_cost_amounts                    = 35
 |level33_stat_text                       = 50% increased Minion Maximum Life<br>Minion deals 121% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 35% less Damage<br>Reaper causes your Non-Reaper Minions to have 35% less Maximum Life
-|level33_stat1_id                        = display_minion_monster_level
+|level33_stat1_id                        = base_display_minion_actor_level
 |level33_stat1_value                     = 93
 |level33_stat2_id                        = minion_maximum_life_+%
 |level33_stat2_value                     = 50
@@ -506,7 +508,7 @@
 |level34_level_requirement               = 94
 |level34_cost_amounts                    = 35
 |level34_stat_text                       = 51% increased Minion Maximum Life<br>Minion deals 123% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 35% less Damage<br>Reaper causes your Non-Reaper Minions to have 35% less Maximum Life
-|level34_stat1_id                        = display_minion_monster_level
+|level34_stat1_id                        = base_display_minion_actor_level
 |level34_stat1_value                     = 94
 |level34_stat2_id                        = minion_maximum_life_+%
 |level34_stat2_value                     = 51
@@ -520,7 +522,7 @@
 |level35_level_requirement               = 95
 |level35_cost_amounts                    = 36
 |level35_stat_text                       = 51% increased Minion Maximum Life<br>Minion deals 124% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 35% less Damage<br>Reaper causes your Non-Reaper Minions to have 35% less Maximum Life
-|level35_stat1_id                        = display_minion_monster_level
+|level35_stat1_id                        = base_display_minion_actor_level
 |level35_stat1_value                     = 95
 |level35_stat2_id                        = minion_maximum_life_+%
 |level35_stat2_value                     = 51
@@ -534,7 +536,7 @@
 |level36_level_requirement               = 96
 |level36_cost_amounts                    = 36
 |level36_stat_text                       = 52% increased Minion Maximum Life<br>Minion deals 126% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 36% less Damage<br>Reaper causes your Non-Reaper Minions to have 36% less Maximum Life
-|level36_stat1_id                        = display_minion_monster_level
+|level36_stat1_id                        = base_display_minion_actor_level
 |level36_stat1_value                     = 96
 |level36_stat2_id                        = minion_maximum_life_+%
 |level36_stat2_value                     = 52
@@ -548,7 +550,7 @@
 |level37_level_requirement               = 97
 |level37_cost_amounts                    = 36
 |level37_stat_text                       = 52% increased Minion Maximum Life<br>Minion deals 127% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 36% less Damage<br>Reaper causes your Non-Reaper Minions to have 36% less Maximum Life
-|level37_stat1_id                        = display_minion_monster_level
+|level37_stat1_id                        = base_display_minion_actor_level
 |level37_stat1_value                     = 97
 |level37_stat2_id                        = minion_maximum_life_+%
 |level37_stat2_value                     = 52
@@ -562,7 +564,7 @@
 |level38_level_requirement               = 98
 |level38_cost_amounts                    = 37
 |level38_stat_text                       = 53% increased Minion Maximum Life<br>Minion deals 129% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 36% less Damage<br>Reaper causes your Non-Reaper Minions to have 36% less Maximum Life
-|level38_stat1_id                        = display_minion_monster_level
+|level38_stat1_id                        = base_display_minion_actor_level
 |level38_stat1_value                     = 98
 |level38_stat2_id                        = minion_maximum_life_+%
 |level38_stat2_value                     = 53
@@ -576,7 +578,7 @@
 |level39_level_requirement               = 99
 |level39_cost_amounts                    = 37
 |level39_stat_text                       = 53% increased Minion Maximum Life<br>Minion deals 130% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 36% less Damage<br>Reaper causes your Non-Reaper Minions to have 36% less Maximum Life
-|level39_stat1_id                        = display_minion_monster_level
+|level39_stat1_id                        = base_display_minion_actor_level
 |level39_stat1_value                     = 99
 |level39_stat2_id                        = minion_maximum_life_+%
 |level39_stat2_value                     = 53
@@ -590,7 +592,7 @@
 |level40_level_requirement               = 100
 |level40_cost_amounts                    = 37
 |level40_stat_text                       = 54% increased Minion Maximum Life<br>Minion deals 132% more Damage with Bleeding<br>Reaper causes your Non-Reaper Minions to deal 37% less Damage<br>Reaper causes your Non-Reaper Minions to have 37% less Maximum Life
-|level40_stat1_id                        = display_minion_monster_level
+|level40_stat1_id                        = base_display_minion_actor_level
 |level40_stat1_value                     = 100
 |level40_stat2_id                        = minion_maximum_life_+%
 |level40_stat2_value                     = 54
