--- Wiki
+++ Export
@@ -40,14 +40,14 @@
 |static_damage_effectiveness             = 35
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_totem_range
-|static_stat1_value                      = 100
-|static_stat2_id                         = ignite_art_variation
-|static_stat2_value                      = 7
-|static_stat3_id                         = base_skill_is_totemified
-|static_stat3_value                      = 1
-|static_stat4_id                         = base_is_projectile
-|static_stat4_value                      = 1
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = base_skill_is_totemified
+|static_stat2_value                      = 1
+|static_stat3_id                         = base_totem_range
+|static_stat3_value                      = 100
+|static_stat4_id                         = ignite_art_variation
+|static_stat4_value                      = 7
 |static_stat5_id                         = visual_hit_effect_elemental_is_holy
 |static_stat5_value                      = 1
 |static_stat6_id                         = always_pierce
