--- Chain Hook
+++ Export
@@ -21,7 +21,7 @@
 |quality_type1_stat1_id                  = base_skill_area_of_effect_+%
 |quality_type1_stat1_value               = 500
 |quality_type2_weight                    = 100
-|quality_type2_stat_text                 = +0.2 to radius
+|quality_type2_stat_text                 = +0.02 metres to radius
 |quality_type2_stat1_id                  = active_skill_base_radius_+
 |quality_type2_stat1_value               = 200
 |quality_type3_weight                    = 50
@@ -37,7 +37,7 @@
 |static_stat2_value                      = 5
 |static_stat3_id                         = chain_strike_gain_x_rage_if_attack_hits
 |static_stat3_value                      = 1
-|stat_text                               = +1 Radius per 5 Rage<br>Gain 1 Rage if this Skill Hits any Enemies
+|stat_text                               = +0.1 metres to Radius per 0.5 Rage<br>Gain 1 Rage if this Skill Hits any Enemies
 |level1                                  = True
 |level1_level_requirement                = 12
 |level1_damage_effectiveness             = 115