--- Elemental Hit
+++ Export
@@ -12,7 +12,6 @@
 |gem_tags                                = Attack, Projectile, Melee, Strike, Fire, Cold, Lightning, AoE, Prismatic, Bow
 |gem_description                         = Each attack with this skill will choose an element at random, and will only be able to deal damage of that element. If the attack hits an enemy, it will also deal damage in an area around them, with the radius being larger if that enemy is suffering from an ailment of the chosen element. It will avoid choosing the same element twice in a row.
 |active_skill_name                       = Elemental Hit
-|item_class_id_restriction               = One Hand Sword, Thrusting One Hand Sword, Two Hand Sword, One Hand Axe, Two Hand Axe, One Hand Mace, Sceptre, Two Hand Mace, Staff, Dagger, Claw, Unarmed, Warstaff, Rune Dagger
 |skill_id                                = ElementalHit
 |cast_time                               = 1.0
 |required_level                          = 12