--- Loyalty Tattoo of Maata
+++ Export
@@ -11,11 +11,14 @@
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
 |tattoo_target                           = Small Intelligence
-|tattoo_stat_text                        = Trigger Summon Spirit of Maata [[Skill]] when you reach [[Low Life]] while
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|Small Intelligence}} {{c|default|Passive Skill}}
+|tattoo_limit                            = 1 Loyalty Tattoo
+|implicit2_text                          = {{c|default|Limit}} {{c|value|1 Loyalty Tattoo}}
+|tattoo_stat_text                        = Trigger Summon Spirit of Maata Skill when you reach Low Life while
 a Unique Enemy is in your Presence
-|tattoo_limit                            = 1 Loyalty Tattoo
-|description                             = Replaces a Small Intelligence Passive Skill<br>Grants "Trigger Summon Spirit of Maata [[Skill]] when you reach [[Low Life]] while
-a Unique Enemy is in your Presence"<br>Limit 1 Loyalty Tattoo
+|tattoo_skill_id                         = TriggerSummonedAncestorMaata
+|description                             = Grants "Trigger [[Skill:TriggerSummonedAncestorMaata|Summon Spirit of Maata]] [[Skill]] when you reach [[Low Life]] while
+a Unique Enemy is in your Presence"
 |drop_text                               = Can be obtained as a reward for defeating the [[Tawhoa]] tribe in the [[Trial of the Ancestors]].
 |is_drop_restricted                      = true
 |release_version                         = 3.22.0
