--- Ancestral Call Support
+++ Export
@@ -22,7 +22,7 @@
 |quality_type2_stat1_id                  = base_skill_area_of_effect_+%
 |quality_type2_stat1_value               = 500
 |quality_type3_weight                    = 20
-|quality_type3_stat_text                 = Supported Skills have +0.2 to Melee Strike Range
+|quality_type3_stat_text                 = Supported Skills have +0.02 metres to Melee Strike Range
 |quality_type3_stat1_id                  = melee_range_+
 |quality_type3_stat1_value               = 200
 |static_cost_multiplier                  = 130