--- Summon Skitterbots
+++ Export
@@ -34,14 +34,16 @@
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
 |static_mana_reservation_percent         = 35
-|static_stat1_id                         = display_skitterbot_limit_per_type
+|static_stat1_id                         = infinite_minion_duration
 |static_stat1_value                      = 1
-|static_stat2_id                         = display_skitterbot_chilling_aura
+|static_stat2_id                         = display_skitterbot_limit_per_type
 |static_stat2_value                      = 1
-|static_stat3_id                         = display_skitterbot_shocking_aura
+|static_stat3_id                         = display_skitterbot_chilling_aura
 |static_stat3_value                      = 1
-|static_stat4_id                         = skitterbots_trap_mine_damage_+%_final
-|static_stat4_value                      = 10
+|static_stat4_id                         = display_skitterbot_shocking_aura
+|static_stat4_value                      = 1
+|static_stat5_id                         = skitterbots_trap_mine_damage_+%_final
+|static_stat5_value                      = 10
 |stat_text                               = (0-76)% increased Minion Movement Speed<br>Minions have (0-19)% increased Chill Effect<br>Minions have (0-19)% increased Shock Effect<br>Maximum 1 Summoned Skitterbot of each type<br>Chilling Skitterbot's Aura Chills Enemies near it<br>Shocking Skitterbot's Aura Shocks Enemies near it<br>Skitterbots grant 10% more Trap and Mine Damage
 |level1                                  = True
 |level1_level_requirement                = 16
