--- Platinum Lex Ejoris Watchstone
+++ Export
@@ -1,7 +1,7 @@
 {{Base item
 |rarity_id                               = normal
 |name                                    = Platinum Lex Ejoris Watchstone
-|class_id                                = AtlasRegionUpgradeItem
+|class_id                                = AtlasUpgradeItem
 |size_x                                  = 1
 |size_y                                  = 1
 |flavour_text                            = Only the souls trapped within can bear witness to the Maven's dark proclivities, and they dare not speak up.
