--- Lacerate
+++ Export
@@ -32,9 +32,9 @@
 |static_cost_amounts                     = 8
 |static_cost_types                       = Mana
 |static_attack_speed_multiplier          = 60
-|static_stat1_id                         = is_area_damage
+|static_stat1_id                         = console_skill_dont_chase
 |static_stat1_value                      = 1
-|static_stat2_id                         = console_skill_dont_chase
+|static_stat2_id                         = is_area_damage
 |static_stat2_value                      = 1
 |static_stat3_id                         = chance_to_bleed_on_hit_%_chance_in_blood_stance
 |static_stat3_value                      = 25
@@ -43,8 +43,6 @@
 |stat_text                               = (5-75) to (8-113) Added Attack Physical Damage<br>+(0-9) to radius<br>(50-88)% more Damage with Bleeding while in Blood Stance<br>25% chance to cause Bleeding while in Blood Stance<br>50% increased angle while in Sand Stance
 |level1                                  = True
 |level1_level_requirement                = 12
-|level1_damage_effectiveness             = 100
-|level1_damage_multiplier                = 100
 |level1_stat_text                        = 5 to 8 Added Attack Physical Damage<br>50% more Damage with Bleeding while in Blood Stance
 |level1_stat1_id                         = attack_minimum_added_physical_damage
 |level1_stat1_value                      = 5
