--- Lost Worlds
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = maps_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardLostWorlds
 |stack_size                              = 8
-|description                             = {{c|white|Map}}<br>{{c|default|Map Tier:}} {{c|normal|15}}
+|description                             = {{c|white|[[Map]]}}<br>{{c|default|Map Tier:}} {{c|normal|15}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_text                               = Sold by [[Kirac]] for 1x {{il|Orb of Chance}}.
