--- Wiki
+++ Export
@@ -6,7 +6,6 @@
 |size_y                                  = 1
 |flavour_text                            = Sturdy and strong,<br>grown from the stream.<br>The Flora that live here,<br>A combatant's dream.
 |drop_level                              = 60
-|drop_level_maximum             = 75
 |tags                                    = weapon_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardTheFlorasGift
 |stack_size                              = 5
