--- Primal Crystallised Lifeforce
+++ Export
@@ -30,8 +30,8 @@
 Metadata/Monsters/LeagueHarvest/Blue/HarvestLanternMinerT3, 
 Metadata/Monsters/LeagueHarvest/Blue/HarvestBirdBoss
 |is_drop_restricted                      = true
+|is_sellable                             = false
 |release_version                         = 3.19.0
-|is_sellable                             = false
 }}
 
 '''Primal Crystallised Lifeforce''' is one of the three Crystallised [[Lifeforce]]s that are dropped from [[harvest]] monsters, once all monsters of that plot has been killed.