--- Infernal Cry
+++ Export
@@ -186,6 +186,8 @@
 |level40_cost_amounts                    = 23
 |level40_duration                        = 6.6
 |item_class_id_restriction               = One Hand Sword, Thrusting One Hand Sword, Two Hand Sword, One Hand Axe, Two Hand Axe, One Hand Mace, Sceptre, Two Hand Mace, Staff, Unarmed, Warstaff
+|level1_damage_effectiveness             = 100
+|level1_damage_multiplier                = 100
 |level2_damage_effectiveness             = 104
 |level2_damage_multiplier                = 103.75
 |level3_damage_effectiveness             = 108