--- Arcanist Brand
+++ Export
@@ -19,6 +19,11 @@
 |quality_type2_weight                    = 100
 |quality_type3_weight                    = 100
 |quality_type4_weight                    = 100
+|static_cost_types                       = Mana
+|static_damage_effectiveness             = 100
+|static_damage_multiplier                = 100
+|static_attack_speed_multiplier          = 100
+|static_duration                         = 3
 |level1                                  = True
 |level1_level_requirement                = 38
 |level1_cost_amounts                     = 18
@@ -139,19 +144,19 @@
 |level40                                 = True
 |level40_level_requirement               = 100
 |level40_cost_amounts                    = 43
+|static_cost_multiplier                  = 100
 |quality_type1_stat1_id                  = sigil_repeat_frequency_+%
 |quality_type1_stat1_value               = 500
 |quality_type1_stat_text                 = 0.5% increased Activation frequency
-|quality_type2_stat1_id                  = dummy_stat_display_nothing
-|quality_type2_stat1_value               = 0
-|quality_type2_stat_text                 = 
-|quality_type3_stat1_id                  = dummy_stat_display_nothing
-|quality_type3_stat1_value               = 0
-|quality_type3_stat_text                 = 
-|quality_type4_stat1_id                  = dummy_stat_display_nothing
-|quality_type4_stat1_value               = 0
-|quality_type4_stat_text                 = 
-|static_cost_types                       = Mana
+|quality_type2_stat1_id                  = base_skill_area_of_effect_+%
+|quality_type2_stat1_value               = 500
+|quality_type2_stat_text                 = Supported Skills have 0.5% increased Area of Effect
+|quality_type3_stat1_id                  = base_projectile_speed_+%
+|quality_type3_stat1_value               = 1000
+|quality_type3_stat_text                 = Supported Skills have 1% increased Projectile Speed
+|quality_type4_stat1_id                  = chaining_range_+%
+|quality_type4_stat1_value               = 1000
+|quality_type4_stat_text                 = Supported Skills have 1% increased Chaining range
 |stat_text                               = Your Skills can have an additional Brand Attached to an Enemy<br>Activates every 1 second while Attached<br>Increases and Reductions to Cast Speed also apply to this Skill's Activation frequency<br>Can be Detached for a total Duration of 5 seconds<br>Can be Attached for a total Duration of 3 seconds<br>Supported Skills have 40% reduced Chaining range<br>Supported Skills have limited Projectile Range<br>Supported Skills have 40% less Area of Effect<br>Supported Skills deal (69-63)% less Damage<br>Supported Skills Deal (40-59)% more Damage with Hits against Branded Enemy<br>Supported Skills are Triggered by Arcanist Brand, from the Brand's location
 |static_stat1_id                         = base_deal_no_damage
 |static_stat1_value                      = 1
