--- Summon Raging Spirit
+++ Export
@@ -10,7 +10,7 @@
 |help_text                               = Place into an item socket of the right colour to gain this skill. Right click to remove from a socket.
 |intelligence_percent                    = 100
 |gem_tags                                = Spell, Minion, Duration, Fire
-|gem_description                         = Summons a short-lived flaming skull that rushes at nearby enemies and attacks them rapidly. Enemies will not directly engage these spirits, and can pass through them.
+|gem_description                         = Summons a short-lived flaming skull that rushes at nearby enemies and attacks them rapidly, converting all its physical damage to fire. Enemies will not directly engage these spirits, and can pass through them.
 |active_skill_name                       = Summon Raging Spirit
 |skill_id                                = SummonRagingSpirit
 |cast_time                               = 0.5