--- Wiki
+++ Export
@@ -5,16 +5,11 @@
 |size_x                                  = 1
 |size_y                                  = 1
 |drop_level                              = 68
-|explicit1_text                          = {{c|default|Client:}} {{c|value|Kurai}}
-|explicit2_text                          = {{c|default|Heist Target:}} {{c|value|"O' Eternal" }}
-|explicit3_text                          = {{c|default|Area Level:}} {{c|value|(68-?)}}
-|explicit4_text                          = {{c|default|Requires}} {{c|value|Lockpicking}} {{c|default|(Level}} {{c|value|2}}{{c|default|)}}
 |tags                                    = default
 |metadata_id                             = Metadata/Items/Heist/QuestContracts/HeistContractQuestKurai1
 |inventory_icon                          = Generic NPC Quest Contract
-|flavour_text                            = "A symbolic piece of history,<br/>from a time when people of all backgrounds<br/>united against a common enemy."
+|drop_text                               = Drops from a [[Smuggler's Cache]] by random
 |release_version                         = 3.12.3
-| drop_text = Drops from a [[Smuggler's Cache]] by random
 }}
 
 '''Contract: Hyrri's Gift''' is a [[quest]] [[contract]]. It is the first step of the quest chain of [[Kurai, the Administrator]].
