--- Orb of Storms
+++ Export
@@ -36,9 +36,9 @@
 |static_damage_effectiveness             = 110
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = skill_can_add_multiple_charges_per_action
+|static_stat1_id                         = damage_cannot_be_reflected_or_leech_if_used_by_other_object
 |static_stat1_value                      = 1
-|static_stat2_id                         = damage_cannot_be_reflected_or_leech_if_used_by_other_object
+|static_stat2_id                         = skill_can_add_multiple_charges_per_action
 |static_stat2_value                      = 1
 |static_stat3_id                         = orb_of_storms_base_channelling_bolt_frequency_ms
 |static_stat3_value                      = 500
