--- Vaal Glacial Hammer
+++ Export
@@ -38,7 +38,7 @@
 |static_stat6_value                      = 2
 |static_stat7_id                         = cold_ailment_duration_+%
 |static_stat7_value                      = 35
-|stat_text                               = Base duration is (3.4-4.35) seconds<br>Modifiers to Skill Effect Duration also apply to this Skill's Soul Gain Prevention<br>25% chance to Freeze enemies<br>100% of Physical Damage Converted to Cold Damage<br>Freezes enemies as though dealing (200-390)% more Damage<br>Can't be Evaded<br>+2 to Melee Strike Range<br>35% increased Duration of Cold Ailments<br>(10-29)% increased Effect of Cold Ailments<br>Adds (2-75) to (3-113) Cold Damage against Chilled Enemies
+|stat_text                               = Base duration is (3.4-4.35) seconds<br>Modifiers to Skill Effect Duration also apply to this Skill's Soul Gain Prevention<br>25% chance to Freeze enemies<br>100% of Physical Damage Converted to Cold Damage<br>Freezes enemies as though dealing (200-390)% more Damage<br>Can't be Evaded<br>+0.2 metres to Melee Strike Range<br>35% increased Duration of Cold Ailments<br>(10-29)% increased Effect of Cold Ailments<br>Adds (2-75) to (3-113) Cold Damage against Chilled Enemies
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_damage_effectiveness             = 155