--- Vaal Blade Flurry
+++ Export
@@ -41,7 +41,7 @@
 |static_stat6_value                      = -50
 |static_stat7_id                         = vaal_charged_attack_radius_+_per_stage
 |static_stat7_value                      = 5
-|stat_text                               = (14-75) to (20-113) Added Attack Physical Damage<br>Blind enemies on hit<br>Can't be Evaded<br>(30-68)% increased Blind Effect<br>50% less Damage Taken while Channelling<br>Base Circle Radius is 10<br>+5 to Circle Radius per Stage<br>Channelling stops when Total Circle Radius reaches 90
+|stat_text                               = (14-75) to (20-113) Added Attack Physical Damage<br>Blind enemies on hit<br>Can't be Evaded<br>(30-68)% increased Blind Effect<br>50% less Damage Taken while Channelling<br>Base Circle Radius is 1 metre<br>+0.5 metres to Circle Radius per Stage<br>Channelling stops when Total Circle Radius reaches 9 metres
 |level1                                  = True
 |level1_level_requirement                = 28
 |level1_damage_effectiveness             = 224