--- The Fathomless Depths
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = armour_divination, unique_divination, unique_armour_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardTheFathomlessDepths
 |stack_size                              = 8
-|description                             = {{c|unique|{{iil|Lightpoacher}}}}
+|description                             = {{c|unique|{{il|html=|Lightpoacher}}}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_areas                              = MapWorldsFloodedMine, MapWorldsUndergroundSea, MapWorldsMineralPools
