--- The Opulent
+++ Export
@@ -10,7 +10,7 @@
 |tags                                    = jewellery_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardTheOpulecent
 |stack_size                              = 5
-|description                             = {{c|rare|{{il|html=|Ring}}}}<br>{{c|default|Item Level:}} {{c|normal|100}}
+|description                             = {{c|rare|[[Ring]]}}<br>{{c|default|Item Level:}} {{c|normal|100}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_areas                              = MapAtlasPalace, MapAtlasChateau, MapAtlasVault, MapAtlasVilla, MapWorldsChateau, MapWorldsRelicChambers, MapWorldsConservatory, MapWorldsVault, MapWorldsResidence, MapWorldsLaboratory, MapWorldsTemple, MapWorldsIvoryTemple, MapWorldsCrimsonTemple, MapWorldsVilla, MapWorldsVaalPyramidUnique, MapWorldsHauntedMansion, MapWorldsPalace, MapWorldsBasilica, MapWorldsCrimsonTownship, MapWorldsMoonTempleUnique
