--- Heavy Strike
+++ Export
@@ -39,7 +39,7 @@
 |static_stat2_value                      = 25
 |static_stat3_id                         = melee_range_+
 |static_stat3_value                      = 2
-|stat_text                               = Knocks Enemies Back on Hit<br>25% reduced Enemy Stun Threshold<br>+2 to Melee Strike Range<br>(20-39)% chance to deal Double Damage
+|stat_text                               = Knocks Enemies Back on Hit<br>25% reduced Enemy Stun Threshold<br>+0.2 metres to Melee Strike Range<br>(20-39)% chance to deal Double Damage
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_damage_effectiveness             = 195