--- Gavel
+++ Export
@@ -15,7 +15,7 @@
 |attack_speed                            = 1.15
 |physical_damage_min                     = 54
 |physical_damage_max                     = 101
-|weapon_range                            = 11
+|weapon_range                            = 1.1
 |quest_reward1_type                      = quest
 |quest_reward1_quest                     = The Storm Blade
 |quest_reward1_quest_id                  = a9q3