--- Whirling Blades
+++ Export
@@ -47,6 +47,8 @@
 |stat_text                               = When Dual Wielding, Deals 75% Damage from each Weapon combined<br>+0.6 seconds to Attack Time
 |level1                                  = True
 |level1_level_requirement                = 10
+|level1_damage_effectiveness             = 100
+|level1_damage_multiplier                = 100
 |level2                                  = True
 |level2_level_requirement                = 13
 |level2_damage_effectiveness             = 103
