--- Wiki
+++ Export
@@ -10,7 +10,6 @@
 |help_text                               = This is a Support Gem. It does not grant a bonus to your character, but to skills in sockets connected to it. Place into an item socket connected to a socket containing the Active Skill Gem you wish to augment. Right click to remove from a socket.
 |dexterity_percent                       = 40
 |intelligence_percent                    = 60
-|static_cost_multiplier                  = 120
 |gem_tags                                = Support, Channelling, Spell, Trigger
 |skill_id                                = SupportCastWhileChannelling
 |support_gem_letter                      = W
