--- Manabond
+++ Export
@@ -33,9 +33,9 @@
 |static_damage_effectiveness             = 160
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = is_area_damage
+|static_stat1_id                         = console_skill_dont_chase
 |static_stat1_value                      = 1
-|static_stat2_id                         = console_skill_dont_chase
+|static_stat2_id                         = is_area_damage
 |static_stat2_value                      = 1
 |stat_text                               = Deals (7-263) to (39-1488) Lightning Damage<br>Deals base Lightning Damage equal to (25-44)% of missing Unreserved Mana<br>+6 to length per 25% of Unreserved maximum Mana you have
 |level1                                  = True
