--- Tattoo of the Kitava Makanga
+++ Export
@@ -10,8 +10,7 @@
 |stack_size                              = 10
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
-|tattoo_target                           = Small Strength
-|tattoo_stat_text                        = 80% increased Maximum total [[Life]] Recovery per second from [[Leech]]
-|description                             = Replaces a Small Strength Passive Skill<br>Grants "80% increased Maximum total [[Life]] Recovery per second from [[Leech]]"
 |implicit1_text                          = {{c|default|Requires}} {{c|value|7}} {{c|default|adjacent Passive Skills Allocated}}
+|implicit2_text                          = {{c|default|Requires}} {{c|value|7}} {{c|default|adjacent Passive Skills Allocated}}
+|description                             = Grants "80% increased Maximum total [[Life]] Recovery per second from [[Leech]]"
 }}