--- Armageddon Brand
+++ Export
@@ -34,13 +34,13 @@
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
 |static_duration                         = 5
-|static_stat1_id                         = is_area_damage
+|static_stat1_id                         = base_skill_show_average_damage_instead_of_dps
 |static_stat1_value                      = 1
-|static_stat2_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat2_id                         = console_skill_dont_chase
 |static_stat2_value                      = 1
-|static_stat3_id                         = skill_can_add_multiple_charges_per_action
+|static_stat3_id                         = is_area_damage
 |static_stat3_value                      = 1
-|static_stat4_id                         = console_skill_dont_chase
+|static_stat4_id                         = skill_can_add_multiple_charges_per_action
 |static_stat4_value                      = 1
 |static_stat5_id                         = base_number_of_sigils_allowed_per_target
 |static_stat5_value                      = 1
@@ -52,7 +52,7 @@
 |static_stat8_value                      = 6000
 |static_stat9_id                         = base_skill_effect_duration
 |static_stat9_value                      = 5000
-|stat_text                               = Deals (69-924) to (104-1386) Fire Damage<br>Your Skills can have an additional Brand Attached to an Enemy<br>Activates every 1.5 seconds while Attached<br>Increases and Reductions to Cast Speed also apply to this Skill's Activation frequency<br>Can be Detached for a total Duration of 6 seconds<br>Can be Attached for a total Duration of 5 seconds<br>Deals (10-29)% more Damage with Hits against Branded Enemy
+|stat_text                               = Your Skills can have an additional Brand Attached to an Enemy<br>Activates every 1.5 seconds while Attached<br>Increases and Reductions to Cast Speed also apply to this Skill's Activation frequency<br>Deals (69-924) to (104-1386) Fire Damage<br>Can be Detached for a total Duration of 6 seconds<br>Can be Attached for a total Duration of 5 seconds<br>Deals (10-29)% more Damage with Hits against Branded Enemy
 |level1                                  = True
 |level1_level_requirement                = 28
 |level1_cost_amounts                     = 15
