--- Wiki
+++ Export
@@ -7,5 +7,5 @@
 |tags                                    = not_for_sale, default
 |metadata_id                             = Metadata/Items/MicrotransactionItemEffects/MicrotransactionResistanceCloak
 |stack_size                              = 1
-|description                             = Adds the Cloak of Elements back attachment to any equipped body armour.
+|description                             = Adds the Cloak of Elements back attachment to any equipped body armour, which indicates the strength of your Fire, Cold, and Lightning Resistances.
 }}