--- Vaal Cyclone
+++ Export
@@ -160,6 +160,7 @@
 |level13_stat2_value                     = 25
 |level14                                 = True
 |level14_level_requirement               = 58
+|level14_damage_effectiveness            = 100
 |level14_damage_multiplier               = 99.8
 |level14_stat_text                       = 18 to 27 Added Attack Physical Damage
 |level14_stat1_id                        = attack_minimum_added_physical_damage