--- Wiki
+++ Export
@@ -31,10 +31,10 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = support_rapid_activation_brand_skill_only_primary_duration_+%_final
-|static_stat1_value                      = -65
-|static_stat2_id                         = quality_display_swiftbrand_is_gem
-|static_stat2_value                      = 1
+|static_stat1_id                         = quality_display_swiftbrand_is_gem
+|static_stat1_value                      = 1
+|static_stat2_id                         = support_rapid_activation_brand_skill_only_primary_duration_+%_final
+|static_stat2_value                      = -65
 |static_stat3_id                         = support_rapid_activation_brand_skill_only_secondary_duration_+%_final
 |static_stat3_value                      = -65
 |stat_text                               = Supported Skills have 65% less Attached Duration<br>Supported Skills have 65% less Detached Duration<br>Supported Skills have (30-44)% more Activation frequency
