--- Wiki
+++ Export
@@ -37,15 +37,15 @@
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
 |static_duration                         = 4
-|static_stat1_id                         = is_area_damage
+|static_stat1_id                         = action_ignores_crit_tracking
 |static_stat1_value                      = 1
-|static_stat2_id                         = skill_can_add_multiple_charges_per_action
+|static_stat2_id                         = base_skill_show_average_damage_instead_of_dps
 |static_stat2_value                      = 1
-|static_stat3_id                         = action_ignores_crit_tracking
+|static_stat3_id                         = damage_originates_from_initiator_location
 |static_stat3_value                      = 1
-|static_stat4_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat4_id                         = is_area_damage
 |static_stat4_value                      = 1
-|static_stat5_id                         = damage_originates_from_initiator_location
+|static_stat5_id                         = skill_can_add_multiple_charges_per_action
 |static_stat5_value                      = 1
 |static_stat6_id                         = skill_cannot_gain_repeat_bonuses
 |static_stat6_value                      = 1
