--- Vampiric Soul Amulet
+++ Export
@@ -0,0 +1,11 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Vampiric Soul Amulet
+|class_id                                = Microtransaction
+|size_x                                  = 1
+|size_y                                  = 1
+|tags                                    = not_for_sale, default
+|metadata_id                             = Metadata/Items/MicrotransactionItemEffects/MicrotransactionVampiricSoulAmulet
+|stack_size                              = 1
+|description                             = Changes the appearance of your equipped Amulet to a Vampiric Soul version. Swarms circle you when you are leeching mana.
+}}