--- Nebula Character Effect
+++ Export
@@ -0,0 +1,11 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Nebula Character Effect
+|class_id                                = Microtransaction
+|size_x                                  = 2
+|size_y                                  = 2
+|tags                                    = not_for_sale, default
+|metadata_id                             = Metadata/Items/MicrotransactionCharacterEffects/MicrotransactionNebulaCharacterEffect
+|stack_size                              = 1
+|description                             = Attaches the Nebula Character Effect to your belt. It causes items you pick up to orbit you in an ethereal cloud of loot until you leave the area.
+}}