--- Wiki
+++ Export
@@ -8,12 +8,10 @@
 |drop_level                              = 84
 |tags                                    = default
 |metadata_id                             = Metadata/Items/QuestItems/MapUpgrades/MapUpgradeTierTo16
-|description                      = Upgrades any map on the Atlas to Tier 16
-|help_text                        = Right click this item then left click on a map on the Atlas to upgrade it to tier 16.
 |drop_enabled                            = False
+|is_in_game                              = false
 |release_version                         = 3.2.0
 |removal_version                         = 3.9.0
-|is_in_game	= false
 }}
 
 '''Elder's Orb''' is a non-tradable consumable item that can be used to upgrade the [[tier]] of any [[map]] to 16 in the [[Atlas of Worlds]].
