--- Vaal Flameblast
+++ Export
@@ -40,7 +40,7 @@
 |static_stat6_value                      = 110
 |static_stat7_id                         = vaal_flameblast_radius_+_per_stage
 |static_stat7_value                      = -2
-|stat_text                               = Deals (31-280) to (46-419) Fire Damage<br>50% chance to Ignite enemies<br>200% more Spell Damage for each stage<br>110% more Damage with Ailments for each stage<br>-2 to Radius for each Stage
+|stat_text                               = Deals (31-280) to (46-419) Fire Damage<br>50% chance to Ignite enemies<br>200% more Spell Damage for each stage<br>110% more Damage with Ailments for each stage<br>-0.2 metres to radius per Stage
 |level1                                  = True
 |level1_level_requirement                = 28
 |level1_stat_text                        = Deals 31 to 46 Fire Damage