--- The White Knight
+++ Export
@@ -12,9 +12,9 @@
 |description                             = {{c|white|Six-Link Astral Plate}}<br>{{c|default|Item Level:}} {{c|normal|100}}<br>{{c|default|Crusader Item}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
+|card_background                         = 2
 |drop_areas                              = EndGame_Labyrinth_trials_arrows
 |release_version                         = 3.11.0
-|card_background                         = 2
 }}
 
 '''The White Knight''' is a [[divination card]]. A set of eight can be exchanged for a [[Crusader item|Crusader-influenced]] [[normal]] {{il|Astral Plate}} with six linked [[sockets]] and [[item level]] 100.
