--- Fire Of Unknown Origin
+++ Export
@@ -4,9 +4,9 @@
 |class_id                                = DivinationCard
 |size_x                                  = 1
 |size_y                                  = 1
-|flavour_text                            = Faith sparks the flame,<br>a frenzy on the wind;<br>a fervour that drives us wildly<br>onward.
+|flavour_text                            = Faith sparks the flame,<br>a frenzy on the wind;<br>a fervour that drives us wildly onward.
 |drop_level                              = 81
-|tags                                    = currency_divination, currency_basic_divination, divination_card, default
+|tags                                    = jewellery_divination, unique_divination, unique_jewellery_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardFireOfUnknownOrigin
 |stack_size                              = 7
 |description                             = {{c|unique|{{il|html=|Nimis}}}}