--- Hinekora's Lock
+++ Export
@@ -7,14 +7,14 @@
 |size_y                                  = 1
 |drop_level                              = 1
 |tags                                    = currency, default
-|metadata_id                             = Metadata/Items/Currency/HinekorasLock
+|metadata_id                             = Metadata/Items/Currency/CurrencyHinekorasLock
 |stack_size                              = 10
-|description                             = Allows an item to foresee the result of the next [[Currency]] item used on it<br>Modifying the item in any way removes the ability to foresee
-|help_text                               = Right click this item then left click an item to apply it.
+|description                             = Allows an item to foresee the result of the next Currency item used on it<br>Modifying the item in any way removes the ability to foresee
+|help_text                               = Right click this item then left click an item to apply it.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
+|drop_text                               = Can be obtained as a reward from [[Hinekora]] for completing a tournament in the [[Trial of the Ancestors]].
 |is_drop_restricted                      = true
 |release_version                         = 3.22.0
-|drop_text = Can be obtained as a reward from [[Hinekora]] for completing a tournament in the [[Trial of the Ancestors]].
 }}
 
 '''Hinekora's Lock''' is an [[currency]] item exclusive to [[Ancestor league]]. Using it on an item allows you to foresee the result of the next [[Currency]] item you use on that item.
