--- Crystalline Geode
+++ Export
@@ -14,8 +14,8 @@
 |help_text                               = Right click on this item to open it.
 |drop_text                               = Obtained from vendoring items with a [[Crucible Passive Tree]] notable that causes it to sell for a Crystalline Geode, or from fully-charged [[Crucible]] unique monsters.
 |is_drop_restricted                      = true
+|is_in_game                              = false
 |release_version                         = 3.21.0
-|is_in_game                              = false
 |removal_version                         = 3.21.2c
 }}
 
