--- Ring of the Exorcist
+++ Export
@@ -7,5 +7,5 @@
 |tags                                    = not_for_sale, default
 |metadata_id                             = Metadata/Items/MicrotransactionItemEffects/MicrotransactionRingOfTheExorcist
 |stack_size                              = 1
-|description                             = Change the appearance of your equipped ring to the Ring of the Exorcist.
+|description                             = Changes the appearance of your equipped ring to the Ring of the Exorcist. Critical Hits from Spells will exorcise the ghosts of your enemies.
 }}