--- Frost Blades
+++ Export
@@ -40,12 +40,12 @@
 |static_stat3_value                      = 110
 |static_stat4_id                         = skill_physical_damage_%_to_convert_to_cold
 |static_stat4_value                      = 60
-|stat_text                               = (2-75) to (3-113) Added Cold Damage<br>Fires (4-7) additional Projectiles<br>60% of Physical Damage Converted to Cold Damage<br>+(18-21) to Melee Strike Range
+|stat_text                               = (2-75) to (3-113) Added Cold Damage<br>Fires (4-7) additional Projectiles<br>60% of Physical Damage Converted to Cold Damage<br>+(1.8-2.1) metres to Melee Strike Range
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_damage_effectiveness             = 110
 |level1_damage_multiplier                = 110
-|level1_stat_text                        = 2 to 3 Added Cold Damage<br>Fires 4 additional Projectiles<br>+18 to Melee Strike Range
+|level1_stat_text                        = 2 to 3 Added Cold Damage<br>Fires 4 additional Projectiles<br>+1.8 metres to Melee Strike Range
 |level1_stat1_id                         = attack_minimum_added_cold_damage
 |level1_stat1_value                      = 2
 |level1_stat2_id                         = attack_maximum_added_cold_damage
@@ -58,7 +58,7 @@
 |level2_level_requirement                = 2
 |level2_damage_effectiveness             = 116
 |level2_damage_multiplier                = 116
-|level2_stat_text                        = 2 to 3 Added Cold Damage<br>Fires 4 additional Projectiles<br>+18 to Melee Strike Range
+|level2_stat_text                        = 2 to 3 Added Cold Damage<br>Fires 4 additional Projectiles<br>+1.8 metres to Melee Strike Range
 |level2_stat1_id                         = attack_minimum_added_cold_damage
 |level2_stat1_value                      = 2
 |level2_stat2_id                         = attack_maximum_added_cold_damage
@@ -71,7 +71,7 @@
 |level3_level_requirement                = 4
 |level3_damage_effectiveness             = 122
 |level3_damage_multiplier                = 122
-|level3_stat_text                        = 3 to 4 Added Cold Damage<br>Fires 4 additional Projectiles<br>+18 to Melee Strike Range
+|level3_stat_text                        = 3 to 4 Added Cold Damage<br>Fires 4 additional Projectiles<br>+1.8 metres to Melee Strike Range
 |level3_stat1_id                         = attack_minimum_added_cold_damage
 |level3_stat1_value                      = 3
 |level3_stat2_id                         = attack_maximum_added_cold_damage
@@ -84,7 +84,7 @@
 |level4_level_requirement                = 7
 |level4_damage_effectiveness             = 128
 |level4_damage_multiplier                = 128
-|level4_stat_text                        = 4 to 6 Added Cold Damage<br>Fires 4 additional Projectiles<br>+18 to Melee Strike Range
+|level4_stat_text                        = 4 to 6 Added Cold Damage<br>Fires 4 additional Projectiles<br>+1.8 metres to Melee Strike Range
 |level4_stat1_id                         = attack_minimum_added_cold_damage
 |level4_stat1_value                      = 4
 |level4_stat2_id                         = attack_maximum_added_cold_damage
@@ -97,7 +97,7 @@
 |level5_level_requirement                = 11
 |level5_damage_effectiveness             = 134
 |level5_damage_multiplier                = 134
-|level5_stat_text                        = 5 to 8 Added Cold Damage<br>Fires 4 additional Projectiles<br>+18 to Melee Strike Range
+|level5_stat_text                        = 5 to 8 Added Cold Damage<br>Fires 4 additional Projectiles<br>+1.8 metres to Melee Strike Range
 |level5_stat1_id                         = attack_minimum_added_cold_damage
 |level5_stat1_value                      = 5
 |level5_stat2_id                         = attack_maximum_added_cold_damage
@@ -110,7 +110,7 @@
 |level6_level_requirement                = 16
 |level6_damage_effectiveness             = 140
 |level6_damage_multiplier                = 140
-|level6_stat_text                        = 7 to 11 Added Cold Damage<br>Fires 5 additional Projectiles<br>+19 to Melee Strike Range
+|level6_stat_text                        = 7 to 11 Added Cold Damage<br>Fires 5 additional Projectiles<br>+1.9 metres to Melee Strike Range
 |level6_stat1_id                         = attack_minimum_added_cold_damage
 |level6_stat1_value                      = 7
 |level6_stat2_id                         = attack_maximum_added_cold_damage
@@ -123,7 +123,7 @@
 |level7_level_requirement                = 20
 |level7_damage_effectiveness             = 146
 |level7_damage_multiplier                = 146
-|level7_stat_text                        = 9 to 13 Added Cold Damage<br>Fires 5 additional Projectiles<br>+19 to Melee Strike Range
+|level7_stat_text                        = 9 to 13 Added Cold Damage<br>Fires 5 additional Projectiles<br>+1.9 metres to Melee Strike Range
 |level7_stat1_id                         = attack_minimum_added_cold_damage
 |level7_stat1_value                      = 9
 |level7_stat2_id                         = attack_maximum_added_cold_damage
@@ -136,7 +136,7 @@
 |level8_level_requirement                = 24
 |level8_damage_effectiveness             = 152
 |level8_damage_multiplier                = 152
-|level8_stat_text                        = 11 to 17 Added Cold Damage<br>Fires 5 additional Projectiles<br>+19 to Melee Strike Range
+|level8_stat_text                        = 11 to 17 Added Cold Damage<br>Fires 5 additional Projectiles<br>+1.9 metres to Melee Strike Range
 |level8_stat1_id                         = attack_minimum_added_cold_damage
 |level8_stat1_value                      = 11
 |level8_stat2_id                         = attack_maximum_added_cold_damage
@@ -149,7 +149,7 @@
 |level9_level_requirement                = 28
 |level9_damage_effectiveness             = 158
 |level9_damage_multiplier                = 158
-|level9_stat_text                        = 14 to 20 Added Cold Damage<br>Fires 5 additional Projectiles<br>+19 to Melee Strike Range
+|level9_stat_text                        = 14 to 20 Added Cold Damage<br>Fires 5 additional Projectiles<br>+1.9 metres to Melee Strike Range
 |level9_stat1_id                         = attack_minimum_added_cold_damage
 |level9_stat1_value                      = 14
 |level9_stat2_id                         = attack_maximum_added_cold_damage
@@ -162,7 +162,7 @@
 |level10_level_requirement               = 32
 |level10_damage_effectiveness            = 164
 |level10_damage_multiplier               = 164
-|level10_stat_text                       = 16 to 25 Added Cold Damage<br>Fires 5 additional Projectiles<br>+19 to Melee Strike Range
+|level10_stat_text                       = 16 to 25 Added Cold Damage<br>Fires 5 additional Projectiles<br>+1.9 metres to Melee Strike Range
 |level10_stat1_id                        = attack_minimum_added_cold_damage
 |level10_stat1_value                     = 16
 |level10_stat2_id                        = attack_maximum_added_cold_damage
@@ -175,7 +175,7 @@
 |level11_level_requirement               = 36
 |level11_damage_effectiveness            = 171
 |level11_damage_multiplier               = 171
-|level11_stat_text                       = 20 to 30 Added Cold Damage<br>Fires 6 additional Projectiles<br>+20 to Melee Strike Range
+|level11_stat_text                       = 20 to 30 Added Cold Damage<br>Fires 6 additional Projectiles<br>+2 metres to Melee Strike Range
 |level11_stat1_id                        = attack_minimum_added_cold_damage
 |level11_stat1_value                     = 20
 |level11_stat2_id                        = attack_maximum_added_cold_damage
@@ -188,7 +188,7 @@
 |level12_level_requirement               = 40
 |level12_damage_effectiveness            = 177
 |level12_damage_multiplier               = 177
-|level12_stat_text                       = 24 to 35 Added Cold Damage<br>Fires 6 additional Projectiles<br>+20 to Melee Strike Range
+|level12_stat_text                       = 24 to 35 Added Cold Damage<br>Fires 6 additional Projectiles<br>+2 metres to Melee Strike Range
 |level12_stat1_id                        = attack_minimum_added_cold_damage
 |level12_stat1_value                     = 24
 |level12_stat2_id                        = attack_maximum_added_cold_damage
@@ -201,7 +201,7 @@
 |level13_level_requirement               = 44
 |level13_damage_effectiveness            = 183
 |level13_damage_multiplier               = 183
-|level13_stat_text                       = 28 to 42 Added Cold Damage<br>Fires 6 additional Projectiles<br>+20 to Melee Strike Range
+|level13_stat_text                       = 28 to 42 Added Cold Damage<br>Fires 6 additional Projectiles<br>+2 metres to Melee Strike Range
 |level13_stat1_id                        = attack_minimum_added_cold_damage
 |level13_stat1_value                     = 28
 |level13_stat2_id                        = attack_maximum_added_cold_damage
@@ -214,7 +214,7 @@
 |level14_level_requirement               = 48
 |level14_damage_effectiveness            = 189
 |level14_damage_multiplier               = 189
-|level14_stat_text                       = 33 to 49 Added Cold Damage<br>Fires 6 additional Projectiles<br>+20 to Melee Strike Range
+|level14_stat_text                       = 33 to 49 Added Cold Damage<br>Fires 6 additional Projectiles<br>+2 metres to Melee Strike Range
 |level14_stat1_id                        = attack_minimum_added_cold_damage
 |level14_stat1_value                     = 33
 |level14_stat2_id                        = attack_maximum_added_cold_damage
@@ -227,7 +227,7 @@
 |level15_level_requirement               = 52
 |level15_damage_effectiveness            = 195
 |level15_damage_multiplier               = 195
-|level15_stat_text                       = 39 to 58 Added Cold Damage<br>Fires 6 additional Projectiles<br>+20 to Melee Strike Range
+|level15_stat_text                       = 39 to 58 Added Cold Damage<br>Fires 6 additional Projectiles<br>+2 metres to Melee Strike Range
 |level15_stat1_id                        = attack_minimum_added_cold_damage
 |level15_stat1_value                     = 39
 |level15_stat2_id                        = attack_maximum_added_cold_damage
@@ -240,7 +240,7 @@
 |level16_level_requirement               = 56
 |level16_damage_effectiveness            = 201
 |level16_damage_multiplier               = 201
-|level16_stat_text                       = 45 to 68 Added Cold Damage<br>Fires 7 additional Projectiles<br>+21 to Melee Strike Range
+|level16_stat_text                       = 45 to 68 Added Cold Damage<br>Fires 7 additional Projectiles<br>+2.1 metres to Melee Strike Range
 |level16_stat1_id                        = attack_minimum_added_cold_damage
 |level16_stat1_value                     = 45
 |level16_stat2_id                        = attack_maximum_added_cold_damage
@@ -253,7 +253,7 @@
 |level17_level_requirement               = 60
 |level17_damage_effectiveness            = 207
 |level17_damage_multiplier               = 207
-|level17_stat_text                       = 52 to 79 Added Cold Damage<br>Fires 7 additional Projectiles<br>+21 to Melee Strike Range
+|level17_stat_text                       = 52 to 79 Added Cold Damage<br>Fires 7 additional Projectiles<br>+2.1 metres to Melee Strike Range
 |level17_stat1_id                        = attack_minimum_added_cold_damage
 |level17_stat1_value                     = 52
 |level17_stat2_id                        = attack_maximum_added_cold_damage
@@ -266,7 +266,7 @@
 |level18_level_requirement               = 64
 |level18_damage_effectiveness            = 213
 |level18_damage_multiplier               = 213
-|level18_stat_text                       = 61 to 91 Added Cold Damage<br>Fires 7 additional Projectiles<br>+21 to Melee Strike Range
+|level18_stat_text                       = 61 to 91 Added Cold Damage<br>Fires 7 additional Projectiles<br>+2.1 metres to Melee Strike Range
 |level18_stat1_id                        = attack_minimum_added_cold_damage
 |level18_stat1_value                     = 61
 |level18_stat2_id                        = attack_maximum_added_cold_damage
@@ -279,7 +279,7 @@
 |level19_level_requirement               = 67
 |level19_damage_effectiveness            = 219
 |level19_damage_multiplier               = 219
-|level19_stat_text                       = 68 to 102 Added Cold Damage<br>Fires 7 additional Projectiles<br>+21 to Melee Strike Range
+|level19_stat_text                       = 68 to 102 Added Cold Damage<br>Fires 7 additional Projectiles<br>+2.1 metres to Melee Strike Range
 |level19_stat1_id                        = attack_minimum_added_cold_damage
 |level19_stat1_value                     = 68
 |level19_stat2_id                        = attack_maximum_added_cold_damage
@@ -292,7 +292,7 @@
 |level20_level_requirement               = 70
 |level20_damage_effectiveness            = 225
 |level20_damage_multiplier               = 225
-|level20_stat_text                       = 75 to 113 Added Cold Damage<br>Fires 7 additional Projectiles<br>+21 to Melee Strike Range
+|level20_stat_text                       = 75 to 113 Added Cold Damage<br>Fires 7 additional Projectiles<br>+2.1 metres to Melee Strike Range
 |level20_stat1_id                        = attack_minimum_added_cold_damage
 |level20_stat1_value                     = 75
 |level20_stat2_id                        = attack_maximum_added_cold_damage
@@ -305,7 +305,7 @@
 |level21_level_requirement               = 72
 |level21_damage_effectiveness            = 231
 |level21_damage_multiplier               = 231
-|level21_stat_text                       = 81 to 122 Added Cold Damage<br>Fires 8 additional Projectiles<br>+22 to Melee Strike Range
+|level21_stat_text                       = 81 to 122 Added Cold Damage<br>Fires 8 additional Projectiles<br>+2.2 metres to Melee Strike Range
 |level21_stat1_id                        = attack_minimum_added_cold_damage
 |level21_stat1_value                     = 81
 |level21_stat2_id                        = attack_maximum_added_cold_damage
@@ -318,7 +318,7 @@
 |level22_level_requirement               = 74
 |level22_damage_effectiveness            = 237
 |level22_damage_multiplier               = 237
-|level22_stat_text                       = 87 to 130 Added Cold Damage<br>Fires 8 additional Projectiles<br>+22 to Melee Strike Range
+|level22_stat_text                       = 87 to 130 Added Cold Damage<br>Fires 8 additional Projectiles<br>+2.2 metres to Melee Strike Range
 |level22_stat1_id                        = attack_minimum_added_cold_damage
 |level22_stat1_value                     = 87
 |level22_stat2_id                        = attack_maximum_added_cold_damage
@@ -331,7 +331,7 @@
 |level23_level_requirement               = 76
 |level23_damage_effectiveness            = 243
 |level23_damage_multiplier               = 243
-|level23_stat_text                       = 93 to 140 Added Cold Damage<br>Fires 8 additional Projectiles<br>+22 to Melee Strike Range
+|level23_stat_text                       = 93 to 140 Added Cold Damage<br>Fires 8 additional Projectiles<br>+2.2 metres to Melee Strike Range
 |level23_stat1_id                        = attack_minimum_added_cold_damage
 |level23_stat1_value                     = 93
 |level23_stat2_id                        = attack_maximum_added_cold_damage
@@ -344,7 +344,7 @@
 |level24_level_requirement               = 78
 |level24_damage_effectiveness            = 249
 |level24_damage_multiplier               = 249
-|level24_stat_text                       = 100 to 150 Added Cold Damage<br>Fires 8 additional Projectiles<br>+22 to Melee Strike Range
+|level24_stat_text                       = 100 to 150 Added Cold Damage<br>Fires 8 additional Projectiles<br>+2.2 metres to Melee Strike Range
 |level24_stat1_id                        = attack_minimum_added_cold_damage
 |level24_stat1_value                     = 100
 |level24_stat2_id                        = attack_maximum_added_cold_damage
@@ -357,7 +357,7 @@
 |level25_level_requirement               = 80
 |level25_damage_effectiveness            = 255
 |level25_damage_multiplier               = 255
-|level25_stat_text                       = 107 to 160 Added Cold Damage<br>Fires 8 additional Projectiles<br>+22 to Melee Strike Range
+|level25_stat_text                       = 107 to 160 Added Cold Damage<br>Fires 8 additional Projectiles<br>+2.2 metres to Melee Strike Range
 |level25_stat1_id                        = attack_minimum_added_cold_damage
 |level25_stat1_value                     = 107
 |level25_stat2_id                        = attack_maximum_added_cold_damage
@@ -370,7 +370,7 @@
 |level26_level_requirement               = 82
 |level26_damage_effectiveness            = 261
 |level26_damage_multiplier               = 261
-|level26_stat_text                       = 114 to 172 Added Cold Damage<br>Fires 9 additional Projectiles<br>+23 to Melee Strike Range
+|level26_stat_text                       = 114 to 172 Added Cold Damage<br>Fires 9 additional Projectiles<br>+2.3 metres to Melee Strike Range
 |level26_stat1_id                        = attack_minimum_added_cold_damage
 |level26_stat1_value                     = 114
 |level26_stat2_id                        = attack_maximum_added_cold_damage
@@ -383,7 +383,7 @@
 |level27_level_requirement               = 84
 |level27_damage_effectiveness            = 267
 |level27_damage_multiplier               = 267
-|level27_stat_text                       = 123 to 184 Added Cold Damage<br>Fires 9 additional Projectiles<br>+23 to Melee Strike Range
+|level27_stat_text                       = 123 to 184 Added Cold Damage<br>Fires 9 additional Projectiles<br>+2.3 metres to Melee Strike Range
 |level27_stat1_id                        = attack_minimum_added_cold_damage
 |level27_stat1_value                     = 123
 |level27_stat2_id                        = attack_maximum_added_cold_damage
@@ -396,7 +396,7 @@
 |level28_level_requirement               = 86
 |level28_damage_effectiveness            = 273
 |level28_damage_multiplier               = 273
-|level28_stat_text                       = 131 to 197 Added Cold Damage<br>Fires 9 additional Projectiles<br>+23 to Melee Strike Range
+|level28_stat_text                       = 131 to 197 Added Cold Damage<br>Fires 9 additional Projectiles<br>+2.3 metres to Melee Strike Range
 |level28_stat1_id                        = attack_minimum_added_cold_damage
 |level28_stat1_value                     = 131
 |level28_stat2_id                        = attack_maximum_added_cold_damage
@@ -409,7 +409,7 @@
 |level29_level_requirement               = 88
 |level29_damage_effectiveness            = 279
 |level29_damage_multiplier               = 279
-|level29_stat_text                       = 140 to 210 Added Cold Damage<br>Fires 9 additional Projectiles<br>+23 to Melee Strike Range
+|level29_stat_text                       = 140 to 210 Added Cold Damage<br>Fires 9 additional Projectiles<br>+2.3 metres to Melee Strike Range
 |level29_stat1_id                        = attack_minimum_added_cold_damage
 |level29_stat1_value                     = 140
 |level29_stat2_id                        = attack_maximum_added_cold_damage
@@ -422,7 +422,7 @@
 |level30_level_requirement               = 90
 |level30_damage_effectiveness            = 286
 |level30_damage_multiplier               = 286
-|level30_stat_text                       = 150 to 224 Added Cold Damage<br>Fires 9 additional Projectiles<br>+23 to Melee Strike Range
+|level30_stat_text                       = 150 to 224 Added Cold Damage<br>Fires 9 additional Projectiles<br>+2.3 metres to Melee Strike Range
 |level30_stat1_id                        = attack_minimum_added_cold_damage
 |level30_stat1_value                     = 150
 |level30_stat2_id                        = attack_maximum_added_cold_damage
@@ -435,7 +435,7 @@
 |level31_level_requirement               = 91
 |level31_damage_effectiveness            = 289
 |level31_damage_multiplier               = 289
-|level31_stat_text                       = 155 to 232 Added Cold Damage<br>Fires 9 additional Projectiles<br>+23 to Melee Strike Range
+|level31_stat_text                       = 155 to 232 Added Cold Damage<br>Fires 9 additional Projectiles<br>+2.3 metres to Melee Strike Range
 |level31_stat1_id                        = attack_minimum_added_cold_damage
 |level31_stat1_value                     = 155
 |level31_stat2_id                        = attack_maximum_added_cold_damage
@@ -448,7 +448,7 @@
 |level32_level_requirement               = 92
 |level32_damage_effectiveness            = 292
 |level32_damage_multiplier               = 292
-|level32_stat_text                       = 160 to 240 Added Cold Damage<br>Fires 10 additional Projectiles<br>+24 to Melee Strike Range
+|level32_stat_text                       = 160 to 240 Added Cold Damage<br>Fires 10 additional Projectiles<br>+2.4 metres to Melee Strike Range
 |level32_stat1_id                        = attack_minimum_added_cold_damage
 |level32_stat1_value                     = 160
 |level32_stat2_id                        = attack_maximum_added_cold_damage
@@ -461,7 +461,7 @@
 |level33_level_requirement               = 93
 |level33_damage_effectiveness            = 295
 |level33_damage_multiplier               = 295
-|level33_stat_text                       = 165 to 248 Added Cold Damage<br>Fires 10 additional Projectiles<br>+24 to Melee Strike Range
+|level33_stat_text                       = 165 to 248 Added Cold Damage<br>Fires 10 additional Projectiles<br>+2.4 metres to Melee Strike Range
 |level33_stat1_id                        = attack_minimum_added_cold_damage
 |level33_stat1_value                     = 165
 |level33_stat2_id                        = attack_maximum_added_cold_damage
@@ -474,7 +474,7 @@
 |level34_level_requirement               = 94
 |level34_damage_effectiveness            = 298
 |level34_damage_multiplier               = 298
-|level34_stat_text                       = 171 to 256 Added Cold Damage<br>Fires 10 additional Projectiles<br>+24 to Melee Strike Range
+|level34_stat_text                       = 171 to 256 Added Cold Damage<br>Fires 10 additional Projectiles<br>+2.4 metres to Melee Strike Range
 |level34_stat1_id                        = attack_minimum_added_cold_damage
 |level34_stat1_value                     = 171
 |level34_stat2_id                        = attack_maximum_added_cold_damage
@@ -487,7 +487,7 @@
 |level35_level_requirement               = 95
 |level35_damage_effectiveness            = 301
 |level35_damage_multiplier               = 301
-|level35_stat_text                       = 176 to 265 Added Cold Damage<br>Fires 10 additional Projectiles<br>+24 to Melee Strike Range
+|level35_stat_text                       = 176 to 265 Added Cold Damage<br>Fires 10 additional Projectiles<br>+2.4 metres to Melee Strike Range
 |level35_stat1_id                        = attack_minimum_added_cold_damage
 |level35_stat1_value                     = 176
 |level35_stat2_id                        = attack_maximum_added_cold_damage
@@ -500,7 +500,7 @@
 |level36_level_requirement               = 96
 |level36_damage_effectiveness            = 304
 |level36_damage_multiplier               = 304
-|level36_stat_text                       = 182 to 273 Added Cold Damage<br>Fires 10 additional Projectiles<br>+24 to Melee Strike Range
+|level36_stat_text                       = 182 to 273 Added Cold Damage<br>Fires 10 additional Projectiles<br>+2.4 metres to Melee Strike Range
 |level36_stat1_id                        = attack_minimum_added_cold_damage
 |level36_stat1_value                     = 182
 |level36_stat2_id                        = attack_maximum_added_cold_damage
@@ -513,7 +513,7 @@
 |level37_level_requirement               = 97
 |level37_damage_effectiveness            = 307
 |level37_damage_multiplier               = 307
-|level37_stat_text                       = 188 to 282 Added Cold Damage<br>Fires 10 additional Projectiles<br>+24 to Melee Strike Range
+|level37_stat_text                       = 188 to 282 Added Cold Damage<br>Fires 10 additional Projectiles<br>+2.4 metres to Melee Strike Range
 |level37_stat1_id                        = attack_minimum_added_cold_damage
 |level37_stat1_value                     = 188
 |level37_stat2_id                        = attack_maximum_added_cold_damage
@@ -526,7 +526,7 @@
 |level38_level_requirement               = 98
 |level38_damage_effectiveness            = 310
 |level38_damage_multiplier               = 310
-|level38_stat_text                       = 194 to 292 Added Cold Damage<br>Fires 10 additional Projectiles<br>+24 to Melee Strike Range
+|level38_stat_text                       = 194 to 292 Added Cold Damage<br>Fires 10 additional Projectiles<br>+2.4 metres to Melee Strike Range
 |level38_stat1_id                        = attack_minimum_added_cold_damage
 |level38_stat1_value                     = 194
 |level38_stat2_id                        = attack_maximum_added_cold_damage
@@ -539,7 +539,7 @@
 |level39_level_requirement               = 99
 |level39_damage_effectiveness            = 313
 |level39_damage_multiplier               = 313
-|level39_stat_text                       = 201 to 301 Added Cold Damage<br>Fires 10 additional Projectiles<br>+24 to Melee Strike Range
+|level39_stat_text                       = 201 to 301 Added Cold Damage<br>Fires 10 additional Projectiles<br>+2.4 metres to Melee Strike Range
 |level39_stat1_id                        = attack_minimum_added_cold_damage
 |level39_stat1_value                     = 201
 |level39_stat2_id                        = attack_maximum_added_cold_damage
@@ -552,7 +552,7 @@
 |level40_level_requirement               = 100
 |level40_damage_effectiveness            = 316
 |level40_damage_multiplier               = 316
-|level40_stat_text                       = 207 to 311 Added Cold Damage<br>Fires 10 additional Projectiles<br>+24 to Melee Strike Range
+|level40_stat_text                       = 207 to 311 Added Cold Damage<br>Fires 10 additional Projectiles<br>+2.4 metres to Melee Strike Range
 |level40_stat1_id                        = attack_minimum_added_cold_damage
 |level40_stat1_value                     = 207
 |level40_stat2_id                        = attack_maximum_added_cold_damage