--- Wiki
+++ Export
@@ -27,10 +27,10 @@
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
 |static_duration                         = 15
-|static_stat1_id                         = display_minion_monster_type
-|static_stat1_value                      = 22
-|static_stat2_id                         = cannot_cancel_skill_before_contact_point
-|static_stat2_value                      = 1
+|static_stat1_id                         = cannot_cancel_skill_before_contact_point
+|static_stat1_value                      = 1
+|static_stat2_id                         = display_minion_monster_type
+|static_stat2_value                      = 22
 |static_stat3_id                         = base_skill_effect_duration
 |static_stat3_value                      = 15000
 |static_stat4_id                         = modifiers_to_skill_effect_duration_also_affect_soul_prevention_duration
