--- The Shepherd's Sandals
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = armour_divination, unique_divination, unique_armour_divination, unique_corrupted_divination, corrupted_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardTheShepherdsSandals
 |stack_size                              = 3
-|description                             = {{c|unique|Boots}}<br>{{c|default|Item Level:}} {{c|normal|100}}<br>{{c|corrupted|[[Two-Implicit]]}}<br>{{c|corrupted|[[Corrupted]]}}
+|description                             = {{c|unique|[[Boots]]}}<br>{{c|default|Item Level:}} {{c|normal|100}}<br>{{c|corrupted|[[Two-Implicit]]}}<br>{{c|corrupted|[[Corrupted]]}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_areas                              = MapWorldsGrotto, MapWorldsLair, MapWorldsBarrows, MapWorldsUndergroundRiverUnique
