--- Sundering Axe
+++ Export
@@ -16,7 +16,7 @@
 |attack_speed                            = 1.3
 |physical_damage_min                     = 74
 |physical_damage_max                     = 155
-|weapon_range                            = 13
+|weapon_range                            = 1.3
 |quest_reward1_type                      = quest
 |quest_reward1_quest                     = The Storm Blade
 |quest_reward1_quest_id                  = a9q3