--- Siphoning Trap
+++ Export
@@ -51,7 +51,7 @@
 |static_stat5_value                      = 1
 |static_stat6_id                         = ice_siphon_trap_max_beam_targets
 |static_stat6_value                      = 10
-|stat_text                               = Trap lasts 4 seconds<br>Deals (16.9-760.7) Base Cold Damage per second<br>Base duration is 3 seconds<br>Modifiers to Spell Damage apply to this Skill's Damage Over Time effect<br>Applies beams to the closest 10 Enemies<br>Regenerate (12.7-324.4) Life per second while any Enemy is affected<br>Regenerate (2.9-17.2) Mana per second while any Enemy is affected<br>Regenerate (1.3-32.4) Life per second for each affected Enemy<br>Regenerate (0.3-1.7) Mana per second for each affected Enemy
+|stat_text                               = Trap lasts 4 seconds<br>Base duration is 3 seconds<br>Deals (16.9-760.7) Base Cold Damage per second<br>Modifiers to Spell Damage apply to this Skill's Damage Over Time effect<br>Applies beams to the closest 10 Enemies<br>Regenerate (12.7-324.4) Life per second while any Enemy is affected<br>Regenerate (2.9-17.2) Mana per second while any Enemy is affected<br>Regenerate (1.3-32.4) Life per second for each affected Enemy<br>Regenerate (0.3-1.7) Mana per second for each affected Enemy
 |level1                                  = True
 |level1_level_requirement                = 10
 |level1_cost_amounts                     = 7
