--- Rebuking Blade
+++ Export
@@ -16,7 +16,7 @@
 |attack_speed                            = 1.3
 |physical_damage_min                     = 34
 |physical_damage_max                     = 63
-|weapon_range                            = 13
+|weapon_range                            = 1.3
 |acquisition_tags                        = experimented base type
 |drop_text                               = Experimented base types can be obtained from the [[Curio Display]] in the final reward rooms of {{il|Blueprint: Laboratory|Laboratory}} or {{il|Blueprint: Prohibited Library|Prohibited Library}} [[Grand Heist]]s.
 |is_drop_restricted                      = true