--- Wiki
+++ Export
@@ -0,0 +1,11 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Ferryman's Boots
+|class_id                                = Microtransaction
+|size_x                                  = 2
+|size_y                                  = 2
+|tags                                    = not_for_sale, default
+|metadata_id                             = Metadata/Items/MicrotransactionItemEffects/MicrotransactionFerrymansBoots
+|stack_size                              = 1
+|description                             = Changes the appearance of any equipped boots to the Ferryman's Boots. Moving leaves trapped souls in your wake, trying to break free.
+}}