--- Wiki
+++ Export
@@ -14,10 +14,6 @@
 |drop_enabled                            = false
 |release_version                         = 2.4.0
 |removal_version                         = 3.9.0
-| sell_price1_name = Simple Sextant
-| sell_price1_amount =3
-| sell_price2_name = Orb of Scouring
-| sell_price2_amount =1
 }}
 '''Apprentice Cartographer's Seal''' is a legacy [[currency]] item. It was used to be uncomplete a white [[map]] (tier 1 to 5) on the [[Atlas of Worlds]]. 
 
