--- Wiki
+++ Export
@@ -11,8 +11,8 @@
 |description                             = Upgrades The Fracturing Spinner to a more powerful version
 |help_text                               = Right click this item then left click an applicable Harbinger unique item to upgrade it.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
+|drop_text                               = This item can only be obtained by defeating the Harbinger Portal map boss in the {{il|Infused Beachhead}}.
 |drop_monsters                           = Metadata/Monsters/HarbingerPortal/HarbingerPortalUber
-|drop_text                               = This item can only be obtained by defeating the Harbinger Portal map boss in the {{il|Infused Beachhead}}.
 |is_drop_restricted                      = true
 |release_version                         = 3.11.0
 }}
