--- Emperor's Luck
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = currency_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardEmperorsLuck
 |stack_size                              = 5
-|description                             = {{c|currency|5x Currency}}
+|description                             = {{c|currency|5x [[Currency]]}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_areas                              = 1_3_15, 2_8_4, MapAtlasHighGardens, MapAtlasCourtyard, MapAtlasOrchard, MapWorldsCourtyard, MapWorldsOrchard, MapWorldsTerrace, MapWorldsGardens
