--- Wiki
+++ Export
@@ -7,14 +7,13 @@
 |drop_level                              = 46
 |tags                                    = default
 |metadata_id                             = Metadata/Items/Heist/QuestContracts/HeistContractQuestIsla1
-|release_version = 3.12.0
 |inventory_icon                          = Generic NPC Quest Contract
+|release_version                         = 3.12.0
 |implicit1_text                          = {{c|default|Client:}} {{c|value|Isla}}
 |implicit2_text                          = {{c|default|Heist Target:}} {{c|value|Pile of Inexplicable Junk}}
 |implicit3_text                          = {{c|default|Area Level:}} {{c|value|(46-?)}}
 |implicit4_text                          = {{c|default|Requires}} {{c|value|Trap Disarmament}} {{c|default|(Level}} {{c|value|1}}{{c|default|)}}
 |implicit5_text                          = {{c|default|Requires}} {{c|value|Isla, the Engineer}}
-|flavour_text                            = My inventions, my babies, deserve to be in better hands!
 }}
 
 '''Contract: Isla's Prototypes''' is a [[quest]] [[contract]]. It is the first quest of [[Isla]]. This was followed by another quest contract {{il|Contract: Isla's Designs}}.
