--- Boundless Realms
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = maps_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardBoundlessRealms
 |stack_size                              = 4
-|description                             = {{c|white|Map}}
+|description                             = {{c|white|[[Map]]}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_areas                              = 2_8_12_1, 2_8_12_2, MapWorldsTemple, MapWorldsMoonTempleUnique, MapWorldsColdRiver_, 2_8_13
