--- Wiki
+++ Export
@@ -43,7 +43,7 @@
 |static_stat2_value                      = 4
 |static_stat3_id                         = corpse_warp_area_of_effect_+%_final_when_consuming_corpse
 |static_stat3_value                      = 200
-|stat_text                               = This Spell deals (3-210) to (5-315), plus 4% of your maximum Life, as base Fire Damage<br>Explosion deals base Fire Damage equal to (6-7.9)% of the corpse's Maximum Life<br>+(0-2) to radius<br>200% more Area of Effect if Consuming a corpse
+|stat_text                               = This Spell deals 0 to 0, plus 4% of your maximum Life, as base Fire Damage<br>Deals (3-210) to (5-315) Fire Damage<br>Explosion deals base Fire Damage equal to (6-7.9)% of the corpse's Maximum Life<br>+(0-2) to radius<br>200% more Area of Effect if Consuming a corpse
 |level1                                  = True
 |level1_level_requirement                = 10
 |level1_cost_amounts                     = 8
