--- Ancestral Cry
+++ Export
@@ -14,7 +14,6 @@
 |active_skill_name                       = Ancestral Cry
 |skill_id                                = AncestralCry
 |cast_time                               = 0.8
-|radius                                  = 60
 |required_level                          = 16
 |quality_type1_weight                    = 50
 |quality_type1_stat_text                 = 1% increased Cooldown Recovery Rate
@@ -34,20 +33,24 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_deal_no_damage
+|static_stat1_id                         = base_deal_no_attack_damage
 |static_stat1_value                      = 1
-|static_stat2_id                         = cannot_cancel_skill_before_contact_point
+|static_stat2_id                         = base_deal_no_secondary_damage
 |static_stat2_value                      = 1
-|static_stat3_id                         = warcry_count_power_from_enemies
+|static_stat3_id                         = base_deal_no_spell_damage
 |static_stat3_value                      = 1
-|static_stat4_id                         = ancestral_cry_x_melee_range_per_5_monster_power
-|static_stat4_value                      = 2
-|static_stat5_id                         = skill_empowers_next_x_melee_attacks
-|static_stat5_value                      = 8
-|static_stat6_id                         = skill_empower_limitation_specifier_for_stat_description
-|static_stat6_value                      = 1
-|static_stat7_id                         = ancestral_cry_empowered_attacks_strike_X_additional_enemies
-|static_stat7_value                      = 2
+|static_stat4_id                         = cannot_cancel_skill_before_contact_point
+|static_stat4_value                      = 1
+|static_stat5_id                         = warcry_count_power_from_enemies
+|static_stat5_value                      = 1
+|static_stat6_id                         = ancestral_cry_x_melee_range_per_5_monster_power
+|static_stat6_value                      = 2
+|static_stat7_id                         = skill_empowers_next_x_melee_attacks
+|static_stat7_value                      = 8
+|static_stat8_id                         = skill_empower_limitation_specifier_for_stat_description
+|static_stat8_value                      = 1
+|static_stat9_id                         = ancestral_cry_empowered_attacks_strike_X_additional_enemies
+|static_stat9_value                      = 2
 |stat_text                               = Base duration is (4-4.9) seconds<br>(0-38)% increased Warcry Speed<br>Counts total Power of Enemies in Range<br>Buff grants +(28-257) to Armour per 5 Power, up to a maximum of +(170-1540)<br>Buff grants +2 Melee Strike Range per 5 Power, up to a maximum of +6<br>Exerts the next 8 Melee Strike Attacks you perform<br>Exerted Strikes target 2 additional Enemies
 |level1                                  = True
 |level1_level_requirement                = 16
@@ -608,6 +611,7 @@
 |level20_strength_requirement            = 155
 |level20_experience                      = 341913067
 |level21_strength_requirement            = 159
+|radius                                  = 60
 |release_version                         = 3.11.0
 |quest_reward1_type                      = quest
 |quest_reward1_quest                     = Intruders in Black
