--- The Fortunate
+++ Export
@@ -4,7 +4,7 @@
 |class_id                                = DivinationCard
 |size_x                                  = 1
 |size_y                                  = 1
-|flavour_text                            = The fortunate find<br>generosity in the divine.
+|flavour_text                            = The fortunate find <br>generosity in the divine.
 |drop_level                              = 68
 |tags                                    = currency_divination, currency_basic_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardTheFortunate
@@ -12,7 +12,7 @@
 |description                             = {{c|currency|2x {{il|html=|Divine Orb}}}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
-|drop_areas                            = MapWorldsTropicalIsland, MapWorldsMineralPools, MapWorldsPlateau, MapWorldsDunes, MapWorldsMesa, MapWorldsAtoll, MapWorldsEstuary, MapWorldsAridLake, MapWorldsLeyline, MapWorldsBeach, MapWorldsMarshes, MapWorldsPeninsula, MapWorldsSulphurVents, MapWorldsShore,  MapWorldsTerrace, MapWorldsCaldera, MapWorldsOrchard
+|drop_areas                              = MapWorldsTropicalIsland, MapWorldsMineralPools, MapWorldsPlateau, MapWorldsDunes, MapWorldsMesa, MapWorldsAtoll, MapWorldsEstuary, MapWorldsAridLake, MapWorldsLeyline, MapWorldsBeach, MapWorldsMarshes, MapWorldsPeninsula, MapWorldsSulphurVents, MapWorldsShore,  MapWorldsTerrace, MapWorldsCaldera, MapWorldsOrchard
 |release_version                         = 3.22.0
 }}
 
