--- Parasitic Passengers
+++ Export
@@ -9,7 +9,7 @@
 |tags                                    = currency_divination, currency_league_divination, divination_card, default
 |metadata_id                             = Metadata/Items/DivinationCards/DivinationCardParasiticPassengers
 |stack_size                              = 8
-|description                             = {{c|currency|10x Incubator}}<br>{{c|default|Item Level:}} {{c|normal|84}}
+|description                             = {{c|currency|10x [[Incubator]]}}<br>{{c|default|Item Level:}} {{c|normal|84}}
 |stack_size_currency_tab                 = 5000
 |help_text                               = Shift click to unstack.
 |drop_areas                              = MapWorldsInfestedValley, MapWorldsArachnidTomb, MapWorldsChannel  <!--MapWorldsPalace -->
