--- Honoured Tattoo of the Warlord
+++ Export
@@ -10,8 +10,7 @@
 |stack_size                              = 10
 |help_text                               = Right click this item then left click a valid Passive Skill. Maximum 50 Tattoos.<br>Shift click to unstack.
 |stack_size_currency_tab                 = 5000
-|tattoo_target                           = +30 to Strength Notable
-|tattoo_stat_text                        = +1 to [[Level]] of all [[Strength]] [[Skill]] Gems
-|tattoo_limit                            = 1 Attribute Notable Tattoo
-|description                             = Replaces a +30 to Strength Notable Passive Skill<br>Grants "+1 to [[Level]] of all [[Strength]] [[Skill]] Gems"<br>Limit 1 Attribute Notable Tattoo
+|implicit1_text                          = {{c|default|Replaces a}} {{c|value|+30 to Strength Notable}} {{c|default|Passive Skill}}
+|implicit2_text                          = {{c|default|Limit}} {{c|value|1 Attribute Notable Tattoo}}
+|description                             = Grants "+1 to [[Level]] of all [[Strength]] [[Skill]] Gems"
 }}