--- Wiki
+++ Export
@@ -13,7 +13,6 @@
 |implicit2_text                          = {{c|default|Heist Target:}} {{c|value|Blackguard Personnel Files}}
 |implicit3_text                          = {{c|default|Area Level:}} {{c|value|#}}
 |implicit4_text                          = {{c|default|Requires}} {{c|value|Lockpicking}} {{c|default|(Level}} {{c|value|1}}{{c|default|)}}
-|flavour_text                            = "Lost touch with a mate of mine when<br />he got recruited into some secret squad.<br />Just want to see how he's doing."
 }}
 
 '''Contract: Enoch's Whereabouts''' is a [[quest]] [[contract]]. The client is [[Huck]]. The area level of this contract is depended on the area level of the Smuggler's Caches.
