--- Map:Arcade Map (Ancestor)
+++ Export
@@ -0,0 +1,26 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Arcade Map
+|class_id                                = Map
+|size_x                                  = 1
+|size_y                                  = 1
+|drop_level                              = 74
+|drop_rarities_ids                       = normal, magic, rare, unique
+|tags                                    = not_for_sale, can_be_infected_map, urban, map, default
+|help_text                               = Travel to this Map by using it in a personal Map Device. Maps can only be used once.
+|map_tier                                = 8
+|map_guild_character                     = +
+|map_area_id                             = MapWorldsArcade
+|map_series                              = Ancestor
+|map_area_level                          = 75
+|unique_map_area_level                   = 75
+|inventory_icon                          = Arcade Map (Ancestor)
+|atlas_map_tier0                         = 8
+|atlas_map_tier1                         = 11
+|atlas_map_tier2                         = 15
+|atlas_map_tier3                         = 16
+|atlas_map_tier4                         = 16
+|atlas_region_minimum                    = 1
+|flavour_text                            = We create so that some part<br>of us may remain after death.
+|release_version                         = 3.22.0
+}}