--- Map:Lava Lake Map (Ancestor)
+++ Export
@@ -0,0 +1,26 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Lava Lake Map
+|class_id                                = Map
+|size_x                                  = 1
+|size_y                                  = 1
+|drop_level                              = 79
+|drop_rarities_ids                       = normal, magic, rare, unique
+|tags                                    = not_for_sale, can_be_infected_map, beach, forest, cannot_be_twinned, map, default
+|help_text                               = Travel to this Map by using it in a personal Map Device. Maps can only be used once.
+|map_tier                                = 13
+|map_guild_character                     = µ
+|map_area_id                             = MapWorldsLavaLake
+|map_series                              = Ancestor
+|map_area_level                          = 80
+|unique_map_area_level                   = 80
+|inventory_icon                          = Lava Lake Map (Ancestor)
+|atlas_map_tier0                         = 13
+|atlas_map_tier1                         = 16
+|atlas_map_tier2                         = 16
+|atlas_map_tier3                         = 16
+|atlas_map_tier4                         = 16
+|atlas_region_minimum                    = 1
+|flavour_text                            = At the heart of a peaceful, secluded valley,<br>an island of burning, consuming fury.
+|release_version                         = 3.22.0
+}}