--- Wiki
+++ Export
@@ -8,7 +8,7 @@
 |drop_rarities_ids                       = normal, magic, rare, unique
 |tags                                    = not_for_sale, can_be_infected_map, mountain, map, default
 |help_text                               = Travel to this Map by using it in a personal Map Device. Maps can only be used once.
-|map_tier                                = 4
+|map_tier                                = 5
 |map_guild_character                     = Í
 |map_area_id                             = MapWorldsAtoll
 |unique_map_guild_character              = Ç
