--- Map:Synthesised Map (Ancestor)
+++ Export
@@ -0,0 +1,20 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Synthesised Map
+|class_id                                = Map
+|size_x                                  = 1
+|size_y                                  = 1
+|drop_level                              = 76
+|drop_rarities_ids                       = normal, magic, rare, unique
+|tags                                    = not_for_sale, map, default
+|help_text                               = Travel to this Map by using it in a personal Map Device. Maps can only be used once.
+|map_tier                                = 10
+|map_guild_character                     = u
+|map_area_id                             = MapWorldsSynthesisedWorld
+|unique_map_area_id                      = Synthesis_MapGuardian1
+|unique_map_area_level                   = 77
+|map_series                              = Ancestor
+|map_area_level                          = 77
+|inventory_icon                          = Synthesised Map (Ancestor)
+|release_version                         = 3.22.0
+}}