--- Wiki
+++ Export
@@ -1,7 +1,6 @@
 {{Base item
 |rarity_id                               = normal
 |name                                    = Harbinger Map
-|name_list                               = Harbinger Map (Low Tier)
 |class_id                                = Map
 |size_x                                  = 1
 |size_y                                  = 1
@@ -17,5 +16,6 @@
 |map_area_level                          = 72
 |inventory_icon                          = The Beachhead (Low Tier)
 |release_version                         = 3.21.0
+|name_list                               = Harbinger Map (Low Tier)
 |drop_enabled                            = false
 }}