--- Wiki
+++ Export
@@ -24,6 +24,6 @@
 |atlas_region_minimum                    = 1
 |flavour_text                            = Voices. Visions. Violence.<br>They will haunt these grounds for all of eternity.
 |release_version                         = 3.21.0
-|is_drop_restricted = true
-|drop_text = Obtained by corrupting Tier 16 Maps.
+|drop_text                               = Obtained by corrupting Tier 16 Maps.
+|is_drop_restricted                      = true
 }}