--- Map:Maze Map (Ancestor)
+++ Export
@@ -0,0 +1,28 @@
+{{Base item
+|rarity_id                               = normal
+|name                                    = Maze Map
+|class_id                                = Map
+|size_x                                  = 1
+|size_y                                  = 1
+|drop_level                              = 80
+|drop_rarities_ids                       = normal, magic, rare, unique
+|tags                                    = not_for_sale, indoors_area, dungeon, temple, map, default
+|help_text                               = Travel to this Map by using it in a personal Map Device. Maps can only be used once.
+|map_tier                                = 14
+|map_guild_character                     = l
+|map_area_id                             = MapWorldsMaze
+|unique_map_guild_character              = ‹
+|unique_map_area_id                      = MapWorldsMazeUnique
+|unique_map_area_level                   = 81
+|map_series                              = Ancestor
+|map_area_level                          = 81
+|inventory_icon                          = Maze Map (Ancestor)
+|atlas_map_tier0                         = 14
+|atlas_map_tier1                         = 16
+|atlas_map_tier2                         = 16
+|atlas_map_tier3                         = 16
+|atlas_map_tier4                         = 16
+|atlas_region_minimum                    = 1
+|flavour_text                            = They live trapped forever in darkness<br>and confusion. We are no different.
+|release_version                         = 3.22.0
+}}