--- Wiki
+++ Export
@@ -1,7 +1,6 @@
 {{Base item
 |rarity_id                               = normal
 |name                                    = Harbinger Map
-|name_list                               = Harbinger Map (Max Tier)
 |class_id                                = Map
 |size_x                                  = 1
 |size_y                                  = 1
@@ -16,5 +15,6 @@
 |unique_map_area_level                   = 83
 |inventory_icon                          = Infused Beachhead
 |release_version                         = 3.21.0
+|name_list                               = Harbinger Map (Max Tier)
 |drop_enabled                            = false
 }}