--- Wiki
+++ Export
@@ -24,7 +24,7 @@
 |atlas_region_minimum                    = 1
 |flavour_text                            = One of my captors hid a beast within his heart.<br>Now his true self can be seen by all.
 |release_version                         = 3.21.0
-|is_drop_restricted = true
-|drop_text = Can be obtained randomly from killing a T14+ Final Map Boss, or from generic map rewards. Allocating [[Remnants of the Past]] will increase its drop rate.
-|drop_areas = MavenHub
+|drop_areas                              = MavenHub
+|drop_text                               = Can be obtained randomly from killing a T14+ Final Map Boss, or from generic map rewards. Allocating [[Remnants of the Past]] will increase its drop rate.
+|is_drop_restricted                      = true
 }}