--- Mastery Effect:Lightning4
+++ Export
@@ -2,5 +2,5 @@
 |id                                  = Lightning4
 |stat1_id                            = passive_mastery_shock_proliferation_radius
 |stat1_value                         = 10
-|stat_text                           = [[Shock|Shocks]] you inflict spread to other Enemies within a Radius of 10
+|stat_text                           = [[Shock|Shocks]] you inflict spread to other Enemies within 1 metre
 }}