--- Mastery Effect:Charges6
+++ Export
@@ -1,6 +1,6 @@
 {{Mastery effect
 |id                                  = Charges6
-|stat1_id                            =nearby_enemies_cannot_gain_charges
+|stat1_id                            = guardian_nearby_enemies_cannot_gain_charges
 |stat1_value                         = 1
-|stat_text                           = Nearby Enemies cannot gain Power, Frenzy or Endurance Charges
+|stat_text                           = Nearby Enemies cannot gain Power, [[Frenzy]] or [[Endurance Charge|Endurance Charges]]
 }}