--- Module:Minimap/minimap icons
+++ Export
@@ -1796,6 +1796,33 @@
 		id = "LakeReflectionDevice", 
 	},
 	{
-		id = "LakeChest", 
+		id = "Sanctum", 
+	},
+	{
+		id = "SanctumDeferral", 
+	},
+	{
+		id = "SanctumPact", 
+	},
+	{
+		id = "SanctumFountain", 
+	},
+	{
+		id = "SanctumRelicCorruptionDevice", 
+	},
+	{
+		id = "SanctumMap", 
+	},
+	{
+		id = "SanctumRelicAltar", 
+	},
+	{
+		id = "SanctumRelicLocker", 
+	},
+	{
+		id = "Crucible", 
+	},
+	{
+		id = "CrucibleForge", 
 	}}
 return data