--- Module:Minimap/minimap icons
+++ Export
@@ -1796,6 +1796,96 @@
 		id = "LakeReflectionDevice", 
 	},
 	{
-		id = "LakeChest", 
+		id = "Sanctum", 
+	},
+	{
+		id = "SanctumDeferral", 
+	},
+	{
+		id = "SanctumPact", 
+	},
+	{
+		id = "SanctumFountain", 
+	},
+	{
+		id = "SanctumRelicCorruptionDevice", 
+	},
+	{
+		id = "SanctumMap", 
+	},
+	{
+		id = "SanctumRelicAltar", 
+	},
+	{
+		id = "SanctumRelicLocker", 
+	},
+	{
+		id = "Crucible", 
+	},
+	{
+		id = "CrucibleForge", 
+	},
+	{
+		id = "AncestralChieftainKiloava", 
+	},
+	{
+		id = "AncestralChieftainAkoya", 
+	},
+	{
+		id = "AncestralChieftainMaata", 
+	},
+	{
+		id = "AncestralChieftainRakiata", 
+	},
+	{
+		id = "AncestralChieftainKahuturoa", 
+	},
+	{
+		id = "AncestralChieftainAhuana", 
+	},
+	{
+		id = "AncestralChieftainKaom", 
+	},
+	{
+		id = "AncestralChieftainNavali", 
+	},
+	{
+		id = "AncestralChieftainUtula", 
+	},
+	{
+		id = "AncestralChieftainTawhanuku", 
+	},
+	{
+		id = "AncestralChieftainIkiaho", 
+	},
+	{
+		id = "AncestralArenaEntrance", 
+	},
+	{
+		id = "AncestralEnemyChieftainTotem", 
+	},
+	{
+		id = "AncestralAllyChieftainTotem", 
+	},
+	{
+		id = "AncestralAllyChieftainTotemAttacked", 
+	},
+	{
+		id = "AncestralFieldItem", 
+	},
+	{
+		id = "AncestralEnemyTotem", 
+	},
+	{
+		id = "AncestralAllyTotem", 
+	},
+	{
+		id = "AncesltralAllyTotemAttacked", 
+	},
+	{
+		id = "AncestralEnemyChieftainTotemAttacked", 
+	},
+	{
+		id = "AncesltralEnemyTotemAttacked", 
 	}}
 return data