--- Modifier:MonsterEnemiesTaunted1
+++ Export
@@ -0,0 +1,11 @@
+{{Mod
+|id                               = MonsterEnemiesTaunted1
+|mod_groups                       = Nothing
+|domain                           = 3
+|generation_type                  = 3
+|required_level                   = 1
+|granted_buff_id                  = generic_buff_aura
+|granted_buff_value               = 20
+|mod_type                         = Nothing
+|stat_text                        = 
+}}