--- Modifier:MapCorruptedSideAreaMagicMonsters
+++ Export
@@ -0,0 +1,17 @@
+{{Mod
+|id                               = MapCorruptedSideAreaMagicMonsters
+|mod_groups                       = MapBloodlinesModOnMagics
+|domain                           = 5
+|generation_type                  = 2
+|required_level                   = 69
+|name                             = of Bloodlines
+|mod_type                         = MapCorruptedSideAreaMagicMonsters
+|stat_text                        = (20-30)% more Magic Monsters
+|stat1_id                         = map_number_of_magic_packs_+%
+|stat1_min                        = 20
+|stat1_max                        = 30
+|spawn_weight1_tag                = secret_area
+|spawn_weight1_value              = 1000
+|spawn_weight2_tag                = default
+|spawn_weight2_value              = 0
+}}