--- Modifier:AncestralItemMaximumLifeOnRespawn
+++ Export
@@ -0,0 +1,12 @@
+{{Mod
+|id                               = AncestralItemMaximumLifeOnRespawn
+|mod_groups                       = AncestralTrialItem
+|domain                           = 3
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = AncestralItemMaximumLifeOnRespawn
+|stat_text                        = 25% increased Maximum [[Life]] when Revived (Hidden)
+|stat1_id                         = ancestral_maximum_life_+%_on_respawn
+|stat1_min                        = 25
+|stat1_max                        = 25
+}}