--- Modifier:WeaponTreeSkillBurningArrowExplosiveArrow
+++ Export
@@ -0,0 +1,16 @@
+{{Mod
+|id                               = WeaponTreeSkillBurningArrowExplosiveArrow
+|mod_groups                       = WeaponTreeSkillBurningArrowExplosiveArrow
+|domain                           = 1
+|generation_type                  = 31
+|required_level                   = 1
+|mod_type                         = WeaponTreeSkillBurningArrowExplosiveArrow
+|stat_text                        = Killing Blows with [[Burning Arrow]] or [[Explosive Arrow]] Shatter Enemies as though [[Freeze|Frozen]]
+|stat1_id                         = burning_and_explosive_arrow_shatter_on_killing_blow
+|stat1_min                        = 1
+|stat1_max                        = 1
+|spawn_weight1_tag                = bow
+|spawn_weight1_value              = 1000
+|spawn_weight2_tag                = default
+|spawn_weight2_value              = 0
+}}