--- Modifier:StanceAncestralNgamahuDrunkRun
+++ Export
@@ -0,0 +1,13 @@
+{{Mod
+|id                               = StanceAncestralNgamahuDrunkRun
+|mod_groups                       = Stance
+|domain                           = 3
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = StanceVelocity
+|stat_text                        = 
+|stat1_id                         = stance_movement_speed_+%_final
+|stat1_min                        = 233
+|stat1_max                        = 233
+|tags                             = speed
+}}