--- Modifier:RandomSkillUnique~~1
+++ Export
@@ -0,0 +1,17 @@
+{{Mod
+|id                               = RandomSkillUnique__1
+|mod_groups                       = RandomSkill
+|domain                           = 1
+|generation_type                  = 3
+|required_level                   = 71
+|mod_type                         = RandomSkill
+|stat_text                        = +3 to [[Level]] of all (1-272) Gems
+|stat1_id                         = random_skill_gem_level_+_level
+|stat1_min                        = 3
+|stat1_max                        = 3
+|stat2_id                         = random_skill_gem_level_+_index
+|stat2_min                        = 1
+|stat2_max                        = 272
+|sell_price1_name                 = Alchemy Shard
+|sell_price1_amount               = 6
+}}