--- Modifier:MapCorruptedSideAreaMonsterElementalReflection
+++ Export
@@ -0,0 +1,24 @@
+{{Mod
+|id                               = MapCorruptedSideAreaMonsterElementalReflection
+|mod_groups                       = MapCorruptedSideAreaMonsterElementalReflection
+|domain                           = 5
+|generation_type                  = 1
+|required_level                   = 69
+|name                             = Mirrored
+|mod_type                         = MapCorruptedSideAreaMonsterElementalReflection
+|stat_text                        = 20% increased Quantity of Items found in this Area<br>Found Items have 3% chance to drop Corrupted in Area<br>Monsters reflect (20-25)% of [[Elemental Damage]]
+|stat1_id                         = map_item_drop_quantity_+%
+|stat1_min                        = 20
+|stat1_max                        = 20
+|stat2_id                         = map_items_drop_corrupted_%
+|stat2_min                        = 3
+|stat2_max                        = 3
+|stat3_id                         = map_monsters_reflect_%_elemental_damage
+|stat3_min                        = 20
+|stat3_max                        = 25
+|spawn_weight1_tag                = secret_area
+|spawn_weight1_value              = 600
+|spawn_weight2_tag                = default
+|spawn_weight2_value              = 0
+|tags                             = elemental_damage, damage, elemental
+}}