--- Modifier:AddedColdDamageUnique~~11
+++ Export
@@ -0,0 +1,18 @@
+{{Mod
+|id                               = AddedColdDamageUnique__11
+|mod_groups                       = ColdDamage
+|domain                           = 1
+|generation_type                  = 3
+|required_level                   = 71
+|mod_type                         = ColdDamage
+|stat_text                        = Adds (30-40) to (60-70) [[Cold Damage]] to [[Attack|Attacks]]
+|stat1_id                         = attack_minimum_added_cold_damage
+|stat1_min                        = 30
+|stat1_max                        = 40
+|stat2_id                         = attack_maximum_added_cold_damage
+|stat2_min                        = 60
+|stat2_max                        = 70
+|tags                             = elemental_damage, damage, elemental, cold, attack
+|sell_price1_name                 = Alteration Shard
+|sell_price1_amount               = 1
+}}