--- Modifier:MapAncestralLeague
+++ Export
@@ -0,0 +1,12 @@
+{{Mod
+|id                               = MapAncestralLeague
+|mod_groups                       = AncestralLeague
+|domain                           = 5
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = AncestralLeague
+|stat_text                        = 
+|stat1_id                         = map_ancestral_league
+|stat1_min                        = 1
+|stat1_max                        = 1
+}}