--- Modifier:SanctumRelicQuantity1
+++ Export
@@ -0,0 +1,19 @@
+{{Mod
+|id                               = SanctumRelicQuantity1
+|mod_groups                       = SanctumRelicQuantity
+|domain                           = 7
+|generation_type                  = 2
+|required_level                   = 1
+|name                             = of Discovery
+|mod_type                         = SanctumRelicQuantity
+|stat_text                        = (11-15)% increased Quantity of Relics Dropped by Monsters
+|stat1_id                         = map_sanctum_relic_drop_chance_+%
+|stat1_min                        = 11
+|stat1_max                        = 15
+|spawn_weight1_tag                = large_sanctum_relic
+|spawn_weight1_value              = 60
+|spawn_weight2_tag                = default
+|spawn_weight2_value              = 0
+|sell_price1_name                 = Alteration Shard
+|sell_price1_amount               = 1
+}}