--- Modifier:SanctumUniqueRelicNoSanctumDefences
+++ Export
@@ -0,0 +1,14 @@
+{{Mod
+|id                               = SanctumUniqueRelicNoSanctumDefences
+|mod_groups                       = SanctumUniqueRelic
+|domain                           = 7
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = SanctumUniqueRelicNoSanctumDefences
+|stat_text                        = No Resolve Mitigation, chance to Avoid Resolve loss or Resolve Aegis
+|stat1_id                         = sanctum_no_resolve_defences
+|stat1_min                        = 1
+|stat1_max                        = 1
+|sell_price1_name                 = Alteration Shard
+|sell_price1_amount               = 5
+}}