--- Modifier:SanctumUniqueRelicAdditionalSanctumKey2
+++ Export
@@ -0,0 +1,14 @@
+{{Mod
+|id                               = SanctumUniqueRelicAdditionalSanctumKey2
+|mod_groups                       = SanctumUniqueRelic
+|domain                           = 7
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = SanctumUniqueRelicAdditionalSanctumKey
+|stat_text                        = The Herald of the Scourge drops 2 additional Forbidden Tomes
+|stat1_id                         = map_lycia2_drop_additional_sanctum_key
+|stat1_min                        = 2
+|stat1_max                        = 2
+|sell_price1_name                 = Alteration Shard
+|sell_price1_amount               = 5
+}}