--- Modifier:MapCorruptedSideAreaSpawnIncursionArchitectsUnique1
+++ Export
@@ -0,0 +1,12 @@
+{{Mod
+|id                               = MapCorruptedSideAreaSpawnIncursionArchitectsUnique1
+|mod_groups                       = MapCorruptedSideAreaSpawnIncursionArchitectsUnique
+|domain                           = 5
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = MapCorruptedSideAreaSpawnIncursionArchitectsUnique
+|stat_text                        = Area contains memories of 20 Atzoatl Architects
+|stat1_id                         = map_spawn_X_additional_incursion_architects
+|stat1_min                        = 20
+|stat1_max                        = 20
+}}