--- Modifier:WeaponTreeSkillBlastRainArtilleryBallista
+++ Export
@@ -0,0 +1,19 @@
+{{Mod
+|id                               = WeaponTreeSkillBlastRainArtilleryBallista
+|mod_groups                       = WeaponTreeSkillBlastRainArtilleryBallista
+|domain                           = 1
+|generation_type                  = 31
+|required_level                   = 1
+|mod_type                         = WeaponTreeSkillBlastRainArtilleryBallista
+|stat_text                        = All Damage from [[Blast Rain]] and Artillery Ballista [[Hit|Hits]] can [[Poison]]<br>25% chance for [[Poison|Poisons]] inflicted with [[Blast Rain]] or Artillery Ballista to deal 100% more Damage
+|stat1_id                         = blast_rain_artillery_ballista_all_damage_can_poison
+|stat1_min                        = 1
+|stat1_max                        = 1
+|stat2_id                         = blast_rain_artillery_ballista_poison_damage_+100%_final_chance
+|stat2_min                        = 25
+|stat2_max                        = 25
+|spawn_weight1_tag                = bow
+|spawn_weight1_value              = 1000
+|spawn_weight2_tag                = default
+|spawn_weight2_value              = 0
+}}