--- Modifier:AncestralItemTotemCannotRecoverLifeWhileBeingChannelled
+++ Export
@@ -0,0 +1,12 @@
+{{Mod
+|id                               = AncestralItemTotemCannotRecoverLifeWhileBeingChannelled
+|mod_groups                       = AncestralTrialItem
+|domain                           = 3
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = AncestralItemTotemCannotRecoverLifeWhileBeingChannelled
+|stat_text                        = Your Ancestral [[Totem]] cannot recover [[Life]] while being Banished (Hidden)
+|stat1_id                         = ancestral_totem_cannot_recover_life_while_being_channelled
+|stat1_min                        = 1
+|stat1_max                        = 1
+}}