--- Modifier:AncestralItemChannellingInterruptDuration
+++ Export
@@ -0,0 +1,12 @@
+{{Mod
+|id                               = AncestralItemChannellingInterruptDuration
+|mod_groups                       = AncestralTrialItem
+|domain                           = 3
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = AncestralItemChannellingInterruptDuration
+|stat_text                        = 300% increased Interruption Duration against Enemies Banishing Ancestral [[Totem|Totems]] (Hidden)
+|stat1_id                         = ancestral_channelling_interrupt_duration_+%
+|stat1_min                        = 300
+|stat1_max                        = 300
+}}