--- Modifier:WeaponTreeSkillLightningArrowIceShot
+++ Export
@@ -0,0 +1,19 @@
+{{Mod
+|id                               = WeaponTreeSkillLightningArrowIceShot
+|mod_groups                       = WeaponTreeSkillLightningArrowIceShot
+|domain                           = 1
+|generation_type                  = 31
+|required_level                   = 1
+|mod_type                         = WeaponTreeSkillLightningArrowIceShot
+|stat_text                        = All Damage from [[Lightning Arrow]] and [[Ice Shot]] [[Hit|Hits]] can [[Ignite]]<br>25% chance for [[Ignite|Ignites]] inflicted with [[Lightning Arrow]] or [[Ice Shot]] to deal 100% more Damage
+|stat1_id                         = lightning_arrow_and_ice_shot_all_damage_can_ignite
+|stat1_min                        = 1
+|stat1_max                        = 1
+|stat2_id                         = lightning_arrow_and_ice_shot_ignite_damage_+100%_final_chance
+|stat2_min                        = 25
+|stat2_max                        = 25
+|spawn_weight1_tag                = bow
+|spawn_weight1_value              = 1000
+|spawn_weight2_tag                = default
+|spawn_weight2_value              = 0
+}}