--- Modifier:AncestralItemNearbyPlayersGainFlaskChargesAura
+++ Export
@@ -0,0 +1,12 @@
+{{Mod
+|id                               = AncestralItemNearbyPlayersGainFlaskChargesAura
+|mod_groups                       = AncestralTrialItem
+|domain                           = 3
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = AncestralItemNearbyPlayersGainFlaskChargesAura
+|stat_text                        = Nearby Ally Players gain 6 Flask [[Charge|Charges]] every 3 seconds (Hidden)
+|stat1_id                         = display_monster_has_flask_gain_aura
+|stat1_min                        = 6
+|stat1_max                        = 6
+}}