--- Modifier:SanctumResolveAegisFasterRecovery1
+++ Export
@@ -0,0 +1,19 @@
+{{Mod
+|id                               = SanctumResolveAegisFasterRecovery1
+|mod_groups                       = SanctumResolveAegisFasterRecovery
+|domain                           = 7
+|generation_type                  = 2
+|required_level                   = 1
+|name                             = of the Pixie
+|mod_type                         = SanctumResolveAegisFasterRecovery
+|stat_text                        = Resolve Aegis Recovers (11-15)% faster while not losing Resolve
+|stat1_id                         = map_sanctum_resolve_aegis_recovery_rate_+%
+|stat1_min                        = 11
+|stat1_max                        = 15
+|spawn_weight1_tag                = medium_sanctum_relic
+|spawn_weight1_value              = 60
+|spawn_weight2_tag                = default
+|spawn_weight2_value              = 0
+|sell_price1_name                 = Alteration Shard
+|sell_price1_amount               = 1
+}}