--- Modifier:LifeLeechInstantExertedAttacksUnique~~1
+++ Export
@@ -0,0 +1,14 @@
+{{Mod
+|id                               = LifeLeechInstantExertedAttacksUnique__1
+|mod_groups                       = LocalLifeLeechIsInstant
+|domain                           = 1
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = LifeLeechInstantExertedAttacks
+|stat_text                        = [[Life]] [[Leech]] from Exerted [[Attack|Attacks]] is instant
+|stat1_id                         = life_leech_is_instant_for_exerted_attacks
+|stat1_min                        = 1
+|stat1_max                        = 1
+|sell_price1_name                 = Alchemy Shard
+|sell_price1_amount               = 6
+}}