--- Modifier:AncestralTrialMonsterCurseEffectHinekora
+++ Export
@@ -0,0 +1,12 @@
+{{Mod
+|id                               = AncestralTrialMonsterCurseEffectHinekora
+|mod_groups                       = AncestralTrialMonster
+|domain                           = 3
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = AncestralTrialMonsterCurseEffect
+|stat_text                        = 
+|stat1_id                         = curse_effect_on_self_+%_final
+|stat1_min                        = -50
+|stat1_max                        = -50
+}}