--- Modifier:SanctumResolveReductionBasedOnArmour3
+++ Export
@@ -0,0 +1,19 @@
+{{Mod
+|id                               = SanctumResolveReductionBasedOnArmour3
+|mod_groups                       = SanctumResolveReductionBasedOnArmour
+|domain                           = 7
+|generation_type                  = 1
+|required_level                   = 84
+|name                             = Girded
+|mod_type                         = SanctumResolveReductionBasedOnArmour
+|stat_text                        = Resolve Mitigation from Enemy [[Hit|Hits]] is based on +(21-25)% of [[Armour]]
+|stat1_id                         = map_sanctum_resolve_damage_reduction_based_on_+%_of_armour
+|stat1_min                        = 21
+|stat1_max                        = 25
+|spawn_weight1_tag                = medium_sanctum_relic
+|spawn_weight1_value              = 30
+|spawn_weight2_tag                = default
+|spawn_weight2_value              = 0
+|sell_price1_name                 = Alteration Shard
+|sell_price1_amount               = 1
+}}