--- Modifier:AncestralTrialMonsterTurtleChannelSpeed
+++ Export
@@ -0,0 +1,12 @@
+{{Mod
+|id                               = AncestralTrialMonsterTurtleChannelSpeed
+|mod_groups                       = AncestralMonsterRespawnTimer
+|domain                           = 3
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = AncestralTrialMonsterForestTuataraCamo
+|stat_text                        = Banishing deals 50% increased Damage to Ancestral [[Totem|Totems]] (Hidden)
+|stat1_id                         = ancestral_channelling_damage_+%_against_totems
+|stat1_min                        = 50
+|stat1_max                        = 50
+}}