--- Modifier:MapCorruptedSideAreaPhysicalDamageAddedAsChaos
+++ Export
@@ -0,0 +1,24 @@
+{{Mod
+|id                               = MapCorruptedSideAreaPhysicalDamageAddedAsChaos
+|mod_groups                       = MapCorruptedSideAreaPhysicalDamageAddedAsChaos
+|domain                           = 5
+|generation_type                  = 1
+|required_level                   = 69
+|name                             = Profane
+|mod_type                         = MapCorruptedSideAreaPhysicalDamageAddedAsChaos
+|stat_text                        = 25% increased Quantity of Items found in this Area<br>Found Items have 3% chance to drop Corrupted in Area<br>Monsters deal (60-80)% extra [[Physical Damage]] as Chaos
+|stat1_id                         = map_item_drop_quantity_+%
+|stat1_min                        = 25
+|stat1_max                        = 25
+|stat2_id                         = map_items_drop_corrupted_%
+|stat2_min                        = 3
+|stat2_max                        = 3
+|stat3_id                         = map_monsters_%_physical_damage_to_add_as_chaos
+|stat3_min                        = 60
+|stat3_max                        = 80
+|spawn_weight1_tag                = secret_area
+|spawn_weight1_value              = 600
+|spawn_weight2_tag                = default
+|spawn_weight2_value              = 0
+|tags                             = physical, chaos
+}}