--- Modifier:AncestralItemBonusesOnLowLife
+++ Export
@@ -0,0 +1,15 @@
+{{Mod
+|id                               = AncestralItemBonusesOnLowLife
+|mod_groups                       = AncestralTrialItem
+|domain                           = 3
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = AncestralItemBonusesOnLowLife
+|stat_text                        = You have Onslaught while on [[Low Life]] (Hidden)<br>100% increased [[Armour]] and [[Evasion Rating]] when on [[Low Life]] (Hidden)
+|stat1_id                         = gain_onslaught_while_on_low_life
+|stat1_min                        = 1
+|stat1_max                        = 1
+|stat2_id                         = armour_and_evasion_on_low_life_+%
+|stat2_min                        = 100
+|stat2_max                        = 100
+}}