--- Modifier:AncestralItemShroudWalker
+++ Export
@@ -0,0 +1,12 @@
+{{Mod
+|id                               = AncestralItemShroudWalker
+|mod_groups                       = AncestralTrialItem
+|domain                           = 3
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = AncestralItemShroudWalker
+|stat_text                        = Shroud Walker (Hidden)
+|stat1_id                         = display_monster_has_shroud_walker_daemon
+|stat1_min                        = 1
+|stat1_max                        = 1
+}}