--- Modifier:AncestralItemLifeOnKill
+++ Export
@@ -0,0 +1,15 @@
+{{Mod
+|id                               = AncestralItemLifeOnKill
+|mod_groups                       = MaximumLifeOnKillPercent
+|domain                           = 3
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = MaximumLifeOnKillPercent
+|stat_text                        = Recover 100% of [[Life]] on [[Kill]] (Hidden)
+|stat1_id                         = recover_%_maximum_life_on_kill
+|stat1_min                        = 100
+|stat1_max                        = 100
+|tags                             = resource, life
+|sell_price1_name                 = Alchemy Shard
+|sell_price1_amount               = 2
+}}