--- Modifier:AncestralItemSlainEnemiesHaveExtraRespawnTimer
+++ Export
@@ -0,0 +1,12 @@
+{{Mod
+|id                               = AncestralItemSlainEnemiesHaveExtraRespawnTimer
+|mod_groups                       = AncestralTrialItem
+|domain                           = 3
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = AncestralItemSlainEnemiesHaveExtraRespawnTimer
+|stat_text                        = Slain Enemies have +10 seconds to their Time to Revive (Hidden)
+|stat1_id                         = ancestral_slain_enemies_respawn_timer_+
+|stat1_min                        = 10000
+|stat1_max                        = 10000
+}}