--- Modifier:AncestralItemFreezeEnemiesNearYouOrYourTotem
+++ Export
@@ -0,0 +1,15 @@
+{{Mod
+|id                               = AncestralItemFreezeEnemiesNearYouOrYourTotem
+|mod_groups                       = AncestralTrialItem
+|domain                           = 3
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = AncestralItemFreezeEnemiesNearYouOrYourTotem
+|stat_text                        = Periodically [[Freeze]] nearby Enemies (Hidden)<br>Your Ancestral [[Totem]] periodically [[Freeze|Freezes]] nearby Enemies (Hidden)
+|stat1_id                         = display_monster_has_periodic_freeze_daemon
+|stat1_min                        = 1
+|stat1_max                        = 1
+|stat2_id                         = ancestral_totem_periodically_freezes_nearby_enemies
+|stat2_min                        = 1
+|stat2_max                        = 1
+}}