--- Modifier:EverythingSomewhereRitualSingle
+++ Export
@@ -0,0 +1,12 @@
+{{Mod
+|id                               = EverythingSomewhereRitualSingle
+|mod_groups                       = EverythingEverywhere
+|domain                           = 5
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = EverythingEverywhereRitual
+|stat_text                        = Areas contain Ritual Altars
+|stat1_id                         = map_area_contains_rituals
+|stat1_min                        = 1
+|stat1_max                        = 1
+}}