--- Modifier:SanctumUniqueRelicDuplicateDeferrals1
+++ Export
@@ -0,0 +1,14 @@
+{{Mod
+|id                               = SanctumUniqueRelicDuplicateDeferrals1
+|mod_groups                       = SanctumUniqueRelic
+|domain                           = 7
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = SanctumUniqueRelicDuplicateDeferrals
+|stat_text                        = Duplicates up to 2 random Offer Rewards upon defeating the Herald of the Scourge
+|stat1_id                         = map_lycia2_duplicate_up_to_x_deferred_rewards
+|stat1_min                        = 2
+|stat1_max                        = 2
+|sell_price1_name                 = Alteration Shard
+|sell_price1_amount               = 5
+}}