--- Modifier:MapCorruptedSideAreaAtziriBossUnique1
+++ Export
@@ -0,0 +1,12 @@
+{{Mod
+|id                               = MapCorruptedSideAreaAtziriBossUnique1
+|mod_groups                       = MapCorruptedSideAreaAtziriBossUnique
+|domain                           = 5
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = MapCorruptedSideAreaAtziriBossUnique
+|stat_text                        = Unique Boss is replaced by Atziri, Queen of the Vaal
+|stat1_id                         = map_boss_replaced_with_atziri
+|stat1_min                        = 1
+|stat1_max                        = 1
+}}