--- Modifier:SanctumUniqueRelicOriginalSin
+++ Export
@@ -0,0 +1,14 @@
+{{Mod
+|id                               = SanctumUniqueRelicOriginalSin
+|mod_groups                       = SanctumUniqueRelic
+|domain                           = 7
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = SanctumUniqueRelicOriginalSin
+|stat_text                        = The Herald of the Scourge drops the Original Sin
+|stat1_id                         = map_lycia2_drop_additional_special_unique
+|stat1_min                        = 1
+|stat1_max                        = 1
+|sell_price1_name                 = Alteration Shard
+|sell_price1_amount               = 5
+}}