--- Modifier:SanctumResolveReductionBasedOnArmour1
+++ Export
@@ -0,0 +1,19 @@
+{{Mod
+|id                               = SanctumResolveReductionBasedOnArmour1
+|mod_groups                       = SanctumResolveReductionBasedOnArmour
+|domain                           = 7
+|generation_type                  = 1
+|required_level                   = 1
+|name                             = Reinforced
+|mod_type                         = SanctumResolveReductionBasedOnArmour
+|stat_text                        = Resolve Mitigation from Enemy [[Hit|Hits]] is based on +(11-15)% of [[Armour]]
+|stat1_id                         = map_sanctum_resolve_damage_reduction_based_on_+%_of_armour
+|stat1_min                        = 11
+|stat1_max                        = 15
+|spawn_weight1_tag                = medium_sanctum_relic
+|spawn_weight1_value              = 60
+|spawn_weight2_tag                = default
+|spawn_weight2_value              = 0
+|sell_price1_name                 = Alteration Shard
+|sell_price1_amount               = 1
+}}