--- Modifier:EverythingSomewhereExpeditionSingle
+++ Export
@@ -0,0 +1,12 @@
+{{Mod
+|id                               = EverythingSomewhereExpeditionSingle
+|mod_groups                       = EverythingEverywhere
+|domain                           = 5
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = EverythingEverywhereExpedition
+|stat_text                        = Area contains an Expedition Encounter
+|stat1_id                         = map_expedition_league
+|stat1_min                        = 1
+|stat1_max                        = 1
+}}