--- Modifier:AncestralItemLightningThornsDaemon
+++ Export
@@ -0,0 +1,14 @@
+{{Mod
+|id                               = AncestralItemLightningThornsDaemon
+|mod_groups                       = AncestralTrialItem
+|domain                           = 3
+|generation_type                  = 3
+|required_level                   = 1
+|granted_buff_id                  = ancestral_lightning_thorns_retaliation
+|granted_buff_value               = 0
+|mod_type                         = AncestralItemLightningThornsDaemon
+|stat_text                        = Lightning Thorns (Hidden)
+|stat1_id                         = display_monster_has_lightning_thorns_daemon
+|stat1_min                        = 1
+|stat1_max                        = 1
+}}