--- Modifier:AmcestralTrialMonsterLeechResistanceKitava
+++ Export
@@ -0,0 +1,12 @@
+{{Mod
+|id                               = AmcestralTrialMonsterLeechResistanceKitava
+|mod_groups                       = AncestralTrialMonster
+|domain                           = 3
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = AmcestralTrialMonsterLeechResistance
+|stat_text                        = 
+|stat1_id                         = resist_life_leech_%
+|stat1_min                        = 50
+|stat1_max                        = 50
+}}