--- Modifier:LowLifeInstantLifeRecoveryUnique~~1
+++ Export
@@ -0,0 +1,15 @@
+{{Mod
+|id                               = LowLifeInstantLifeRecoveryUnique__1
+|mod_groups                       = InstantFlaskRecovery
+|domain                           = 1
+|generation_type                  = 3
+|required_level                   = 38
+|mod_type                         = LowLifeInstantLifeRecovery
+|stat_text                        = [[Life]] Flasks used while on [[Low Life]] apply Recovery Instantly
+|stat1_id                         = life_flask_recovery_is_instant_while_on_low_life
+|stat1_min                        = 1
+|stat1_max                        = 1
+|tags                             = flask, resource, life
+|sell_price1_name                 = Alchemy Shard
+|sell_price1_amount               = 2
+}}