--- Modifier:AncestralItemFasterStartOfEnergyShieldRecharge
+++ Export
@@ -0,0 +1,15 @@
+{{Mod
+|id                               = AncestralItemFasterStartOfEnergyShieldRecharge
+|mod_groups                       = EnergyShieldDelay
+|domain                           = 3
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = EnergyShieldDelay
+|stat_text                        = 100% faster start of [[Energy Shield]] Recharge (Hidden)
+|stat1_id                         = energy_shield_delay_-%
+|stat1_min                        = 100
+|stat1_max                        = 100
+|tags                             = defences, energy_shield
+|sell_price1_name                 = Alchemy Shard
+|sell_price1_amount               = 2
+}}