--- Modifier:SanctumUniqueRelicLyciaDamageTaken2
+++ Export
@@ -0,0 +1,14 @@
+{{Mod
+|id                               = SanctumUniqueRelicLyciaDamageTaken2
+|mod_groups                       = SanctumUniqueRelic
+|domain                           = 7
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = SanctumUniqueRelicLyciaDamageTaken
+|stat_text                        = The Herald of the Scourge takes 75% less Damage
+|stat1_id                         = map_lycia2_damage_taken_+%_final
+|stat1_min                        = -75
+|stat1_max                        = -75
+|sell_price1_name                 = Alteration Shard
+|sell_price1_amount               = 5
+}}