--- Modifier:AncestralItemProximityShield
+++ Export
@@ -0,0 +1,12 @@
+{{Mod
+|id                               = AncestralItemProximityShield
+|mod_groups                       = ProximityShield
+|domain                           = 3
+|generation_type                  = 3
+|required_level                   = 1
+|mod_type                         = AncestralTrialProximityShield
+|stat_text                        = Proximity [[Shield]] (Hidden)
+|stat1_id                         = display_monster_has_proximity_shield_daemon
+|stat1_min                        = 1
+|stat1_max                        = 1
+}}