--- Monster:Metadata/Monsters/KitavaBoss/Kitava2
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/KitavaBoss/Kitava2
 |monster_type_id                  = Kitava
-|mod_ids                          = MonsterActBossAct5, MonsterImplicitCannotBeStunned1, MonsterCannotBeSlowed, StunRepeatImmunityActBoss, BossHealOnPlayerDeathMinor
+|mod_ids                          = MonsterActBossAct5, MonsterCannotBeSlowed, StunRepeatImmunityActBoss, BossHealOnPlayerDeathMinor, MonsterImplicitCannotBeStunned1
 |tags                             = bludgeoning_weapon, caster, demon, extra_extra_large_height, fire_affinity, flesh_armour, is_unarmed, physical_affinity, red_blood, very_slow_movement
 |skill_ids                        = KitavaSlamMiddle, KitavaSlamRight, KitavaSlamLeft, KitavaSlamMiddleRight, KitavaSlamMiddleLeft, KitavaSwipeRight, KitavaSwipeLeft, KitavaDevouringDarknessRight, KitavaDevouringDarknessLeft, KitavaXBlast, KitavaThrowDebrisLeft, KitavaThrowDebrisRight, KitavaSlamRightNoCone, KitavaSlamLeftNoCone
 |name                             = Kitava, the Insatiable