--- Monster:Metadata/Monsters/LeagueDelve/LavaFossilEncounter/DelveLavaGargoyleBoss
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/LeagueDelve/LavaFossilEncounter/DelveLavaGargoyleBoss
 |monster_type_id                  = Minotaur
-|mod_ids                          = CannotBeSlowedBelowValueBosses, MonsterImplicitCannotBeStunned1, MonsterMinotaurBoss, StunRepeatImmunityActBoss
+|mod_ids                          = CannotBeSlowedBelowValueBosses, MonsterMinotaurBoss, StunRepeatImmunityActBoss
 |tags                             = beast, caster, delve_monster, demon, fire_affinity, flesh_armour, gladiator, has_one_hand_mace, has_one_handed_melee, large_height, large_model, medium_movement, melee, not_dex, not_int, physical_affinity, red_blood, slashing_weapon
 |skill_ids                        = DelveLavaGargoyleBossMelee, MinotaurEnrage, EnduranceChargeSlamMagmaDelve, AbberathsFury
 |name                             = Gehennix
