--- Monster:Metadata/Monsters/KitavaBoss/Kitava1MapElder
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/KitavaBoss/Kitava1MapElder
 |monster_type_id                  = KitavaMap
-|mod_ids                          = MonsterMapBoss, MonsterImplicitCannotBeStunned1, MonsterCannotBeSlowed, StunRepeatImmunityMapBoss_
+|mod_ids                          = MonsterMapBoss, MonsterCannotBeSlowed, StunRepeatImmunityMapBoss_, MonsterImplicitCannotBeStunned1
 |tags                             = bludgeoning_weapon, demon, elder_map_boss, extra_extra_large_height, flesh_armour, is_unarmed, non_attacking, red_blood, very_slow_movement
 |name                             = Lord of the Grey
 |size                             = 20