--- Monster:Metadata/Monsters/LeagueSanctum/Guards/Guard1
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/LeagueSanctum/Guards/Guard1
 |monster_type_id                  = SanctumGuard1
 |mod_ids                          = MonsterTagSanctumGuard, MonsterSanctumGuardVolatileOnDeath, MonsterSanctumGuardSummonGargoyle
-|tags                             = bludgeoning_weapon, caster, fast_movement, fire_affinity, flesh_armour, has_two_hand_mace, has_two_handed_melee, human, humanoid, medium_height, melee, not_dex, not_str, physical_affinity, ranged, red_blood, sanctum_guard
+|tags                             = bludgeoning_weapon, caster, fast_movement, fire_affinity, flesh_armour, has_two_hand_mace, has_two_handed_melee, human, humanoid, medium_height, melee, not_dex, not_str, physical_affinity, ranged, red_blood, sanctum_guard, sanctum_monster
 |skill_ids                        = GSSanctumFrontalBeam, EAASanctumSpinningSlam, GASanctumSpinningSlam, GTSanctumSpikeWallLeft, GPSSanctumSpikeWall, GTSanctumSpikeWallRight, GPSSanctumSpikeWallInverse, GTSanctumIceSpearCircle, GPSSanctumIceSpike
 |name                             = Deathly Archmage
 |size                             = 3
