--- Monster:Metadata/Monsters/LeagueCrucible/Cold/Pyromaniac
+++ Export
@@ -0,0 +1,16 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueCrucible/Cold/Pyromaniac
+|monster_type_id                  = CruciblePyromaniac
+|tags                             = caster, demon, fire_affinity, flesh_armour, humanoid, is_unarmed, medium_height, medium_movement, not_int, not_str, ranged, slashing_weapon, undying
+|skill_ids                        = MMSPyromaniacIceMortar, CrucibleIceStormTrap
+|name                             = Forged Frostbearer
+|size                             = 3
+|minimum_attack_distance          = 6
+|maximum_attack_distance          = 7
+|model_size_multiplier            = 1.32
+|experience_multiplier            = 1.4
+|damage_multiplier                = 1.4
+|health_multiplier                = 1.68
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.5
+}}