--- Monster:Metadata/Monsters/CorpseWheel/CorpseWheelHellscape
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/CorpseWheel/CorpseWheelHellscape
 |monster_type_id                  = CorpseWheel
 |mod_ids                          = MonsterNecromancerRaisable
-|tags                             = bone_armour, bones, has_one_hand_sword, has_one_handed_melee, medium_movement, melee, no_strider_mod, not_dex, physical_affinity, skeleton, slashing_weapon
+|tags                             = bone_armour, bones, has_one_hand_sword, has_one_handed_melee, medium_movement, melee, no_strider_mod, not_dex, physical_affinity, skeleton, slashing_weapon, undead
 |skill_ids                        = CorpseWheelCharge, MeleeAtAnimationSpeed, EASWheelTaunt
 |name                             = Dreadwheel
 |size                             = 3
