--- Monster:Metadata/Monsters/LeagueAncestral/AncestralKaomMinion~
+++ Export
@@ -0,0 +1,16 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueAncestral/AncestralKaomMinion_
+|monster_type_id                  = AncestralKaomMinion
+|tags                             = human, humanoid, not_int, red_blood, slow_movement
+|skill_ids                        = MeleeAtAnimationSpeedUnique, MPSKaomFireFist, GAKaomSlam, GAKaomFistBlast, GAKaomFistBlastFinal, EAAKaomGroundSlam, AncestralKaomUnarmedCombo
+|name                             = Spirit of Kaom
+|size                             = 3
+|minimum_attack_distance          = 5
+|maximum_attack_distance          = 20
+|model_size_multiplier            = 1.0
+|experience_multiplier            = 1.0
+|damage_multiplier                = 1.3
+|health_multiplier                = 2.0
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.0
+}}