--- Monster:Metadata/Monsters/Miner/MinerLanternMapBoss
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/Miner/MinerLanternMapBoss
 |monster_type_id                  = MinerLanternMapBoss
 |mod_ids                          = LanternBossLifeRegenerationOnLowLife, IgniteArtVariation, MonsterMapBoss, StunRepeatImmunityMapBoss_
-|tags                             = caster, fast_movement, fire_affinity, has_one_hand_sword, has_one_handed_melee, large_height, melee, mud_blood, non_unique_map_boss, not_int, not_str, physical_affinity, ranged, stone_armour, undead
+|tags                             = caster, fast_movement, fire_affinity, has_one_hand_sword, has_one_handed_melee, large_height, melee, mud_blood, non_unique_map_boss, not_int, not_str, physical_affinity, ranged, stone_armour, undead, unholy_trio_boss_map
 |skill_ids                        = MinerThrowFire, LanternBossRighteousFire, MinerLanternHeal, MeleeAtAnimationSpeed, DeathExplodeMinerHammerBoss, MinerThrowFireMapBoss
 |name                             = Messenger of the Hollows
 |size                             = 3
