--- Monster:Metadata/Monsters/TestSpawn/CooldownTestSameSkillNoGroup
+++ Export
@@ -0,0 +1,16 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/TestSpawn/CooldownTestSameSkillNoGroup
+|monster_type_id                  = TestSpawn
+|tags                             = animal_claw_weapon, flesh_armour, has_dagger, has_one_handed_melee, melee, physical_affinity, red_blood, small_height, very_fast_movement
+|skill_ids                        = GACooldownTest1, GACooldownTest2
+|name                             = [DNT] Same active skill, no cooldown group
+|size                             = 3
+|minimum_attack_distance          = 3
+|maximum_attack_distance          = 10
+|model_size_multiplier            = 1.3
+|experience_multiplier            = 1.0
+|damage_multiplier                = 100.0
+|health_multiplier                = 100.0
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.005
+}}