--- Monster:Metadata/Monsters/LeagueCrucible/Cold/KaruiElder
+++ Export
@@ -0,0 +1,16 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueCrucible/Cold/KaruiElder
+|monster_type_id                  = CrucibleKaruiElder
+|tags                             = caster, fire_affinity, flesh_armour, has_dual_wield_melee, has_one_hand_mace, has_one_handed_melee, humanoid, large_height, ranged, slashing_weapon, slow_movement
+|skill_ids                        = MeleeAtAnimationSpeedCold, CrucibleColdAbyssSlam, SSMCrucibleSummonWaterTornado
+|name                             = Forged Warrior
+|size                             = 3
+|minimum_attack_distance          = 6
+|maximum_attack_distance          = 10
+|model_size_multiplier            = 1.15
+|experience_multiplier            = 1.8
+|damage_multiplier                = 1.8
+|health_multiplier                = 2.52
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.5
+}}