--- Monster:Metadata/Monsters/LeagueAncestral/AncestralJadeHulk
+++ Export
@@ -0,0 +1,17 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueAncestral/AncestralJadeHulk
+|monster_type_id                  = AncestralJadeHulk
+|mod_ids                          = AncestralTrialMonsterBlockReductionTukohama, AncestralTrialMonsterIntermediateFleeAndDamage
+|tags                             = beast, bludgeoning_weapon, has_one_hand_mace, has_one_handed_melee, light_armour, mammal_beast, medium_height, melee, not_dex, not_int, physical_affinity, red_blood, slow_movement
+|skill_ids                        = MeleeAtAnimationSpeed, DTTAncestralJadeHulkLeapSlam, GAAncestralJadeHulkLeapImpact, TCAncestralLeagueKaruiHulk, GAAncestralLeagueKaruiHulkCharge
+|name                             = Jade Hulk
+|size                             = 3
+|minimum_attack_distance          = 5
+|maximum_attack_distance          = 14
+|model_size_multiplier            = 1.0
+|experience_multiplier            = 1.0
+|damage_multiplier                = 1.5
+|health_multiplier                = 3.0
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.5
+}}