--- Monster:Metadata/Monsters/LeagueCrucible/Fire/CityStalker
+++ Export
@@ -0,0 +1,16 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueCrucible/Fire/CityStalker
+|monster_type_id                  = CrucibleCityStalker
+|tags                             = demon, humanoid, medium_movement, not_int, undying
+|skill_ids                        = MeleeAtAnimationSpeedFire
+|name                             = Forged Stalker
+|size                             = 3
+|minimum_attack_distance          = 6
+|maximum_attack_distance          = 8
+|model_size_multiplier            = 0.9
+|experience_multiplier            = 0.6
+|damage_multiplier                = 0.6
+|health_multiplier                = 0.6
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.245
+}}