--- Monster:Metadata/Monsters/Skeletons/SkeletonMeleeVaal1Royale
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/Skeletons/SkeletonMeleeVaal1Royale
 |monster_type_id                  = ConstructMeleeRoyale
 |mod_ids                          = MonsterCastsElementalHitText
-|tags                             = bone_armour, bones, construct, has_two_hand_sword, has_two_handed_melee, medium_height, melee, not_dex, not_int, physical_affinity, slashing_weapon, very_slow_movement
+|tags                             = bone_armour, bones, construct, has_two_hand_sword, has_two_handed_melee, medium_height, melee, not_dex, not_int, physical_affinity, slashing_weapon, undead, very_slow_movement
 |skill_ids                        = Melee, ElementalHitSkeletonKnight
 |name                             = Vaal Fallen
 |size                             = 3