--- Monster:Metadata/Monsters/LeagueAncestral/AncestralUtula
+++ Export
@@ -0,0 +1,17 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueAncestral/AncestralUtula
+|monster_type_id                  = AncestralUtula
+|mod_ids                          = AmcestralTrialMonsterLeechResistanceKitava, AncestralTrialChieftainBalance
+|tags                             = fast_movement, human, humanoid, not_int, red_blood
+|skill_ids                        = MeleeAtAnimationSpeedUnique, UtulaTableCharge, UtulaFireSwing, GAEchoingQuake, GAUtulaChargeEnd, MPSUtulaFireball, GAUtulaXSlamLacerate, CGEUtulaDegen
+|name                             = Utula
+|size                             = 3
+|minimum_attack_distance          = 5
+|maximum_attack_distance          = 19
+|model_size_multiplier            = 1.0
+|experience_multiplier            = 1.0
+|damage_multiplier                = 1.2
+|health_multiplier                = 1.2
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.5
+}}