--- Monster:Metadata/Monsters/LeagueHeist/Robot/RobotForgeHoundMiniboss
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/LeagueHeist/Robot/RobotForgeHoundMiniboss
 |monster_type_id                  = HeistRobotForgeHoundMiniboss
-|mod_ids                          = MonsterImplicitCannotBeStunned1, CannotBeTaunted, MonsterCannotBeKnockedBack
+|mod_ids                          = CannotBeTaunted, MonsterCannotBeKnockedBack, StunRepeatImmunityBoss__
 |tags                             = animal_claw_weapon, caster, construct, fire_affinity, is_unarmed, medium_movement, melee, metal_armour, not_dex, not_int, physical_affinity
 |skill_ids                        = Melee, DoLiterallyNothing, GAHeistRobotHoundStomp, MDHeistRobotHoundLaserTarget, GSRoboHoundBellyDamage
 |name                             = GR8B0-Y