--- Monster:Metadata/Monsters/HalfSkeleton/HalfSkeletonRangedBreachChaosSpectre
+++ Export
@@ -1,6 +1,6 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/HalfSkeleton/HalfSkeletonRangedBreachChaosSpectre
-|monster_type_id                  = BreachHalfSkeletonRangedChaosMap
+|monster_type_id                  = BreachHalfSkeletonRangedChaosMapSpectre
 |mod_ids                          = BreachReducedBeyondPortalChance
 |part1_mod_ids                    = MonsterPhysicalAddedAsChaosBreach
 |part2_mod_ids                    = MonsterPhysicalAddedAsChaosBreach
@@ -14,7 +14,7 @@
 |model_size_multiplier            = 1.0
 |experience_multiplier            = 1.0
 |damage_multiplier                = 1.5
-|health_multiplier                = 4.68
+|health_multiplier                = 3.6
 |critical_strike_chance           = 5.0
 |attack_speed                     = 1.5
 }}