--- Monster:Metadata/Monsters/LeagueAncestral/AncestralFishingTaiaha
+++ Export
@@ -0,0 +1,17 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueAncestral/AncestralFishingTaiaha
+|monster_type_id                  = AncestralFishingTaiaha
+|mod_ids                          = AncestralTrialMonsterStealsFlaskChargesTasalio, AncestralTrialMonsterChampionFleeAndDamage, MonsterIncreasedTrapsAllowed
+|tags                             = fast_movement, human, humanoid, karui, not_int, red_blood
+|skill_ids                        = MeleeAtAnimationSpeed, MonsterBearTrapAncestral, MonsterBearTrapAncestralMulti
+|name                             = Trawler
+|size                             = 3
+|minimum_attack_distance          = 5
+|maximum_attack_distance          = 13
+|model_size_multiplier            = 1.0
+|experience_multiplier            = 1.0
+|damage_multiplier                = 2.0
+|health_multiplier                = 5.5
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.5
+}}