--- Monster:Metadata/Monsters/Wb/WbFireLeader1
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/Wb/WbFireLeader1
 |monster_type_id                  = WbFireLeader1
-|mod_ids                          = MonsterElementalSkeletonFire, MonsterMapBoss, StunRepeatImmunityMapBoss_
+|mod_ids                          = MonsterElementalSkeletonFire, MonsterMapBoss, StunRepeatImmunityMapBoss_, RedbladeWarlord
 |tags                             = caster, cleaving_weapon, fire_affinity, has_dual_wield_melee, has_one_hand_sword, has_one_handed_melee, human, humanoid, medium_height, medium_movement, melee, non_unique_map_boss, physical_affinity, plate_armour, ranged, red_blood, wb_leader
 |skill_ids                        = Melee, WbSawbladeCyclone, WbLivingBombFire, WbMoltenStrike, WbMoltenStrikeLeader, WbFireBeaconLeader, WbFirestormLeader, WbFireLeader1Wait
 |name                             = Uruk Baleh