--- Monster:Metadata/Monsters/Spirit/TormentedShavronne
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/Spirit/TormentedShavronne
 |monster_type_id                  = TormentedSpiritShavronne
 |mod_ids                          = TormentSpiritDropsIncrease3, TormentSpiritPhasing, TormentAdditionalRareItemsSpirit, MonsterNoMapDrops
-|tags                             = ghost, ghost_armour, ghost_blood, is_unarmed, medium_height, medium_movement, non_attacking
+|tags                             = ghost, ghost_armour, ghost_blood, is_unarmed, medium_height, medium_movement, non_attacking, undead
 |skill_ids                        = Haunt, TormentSpiritPause, WalkEmergeTormentedSpirit
 |name                             = Echo of Shavronne
 |size                             = 3