--- Monster:Metadata/Monsters/AtlasBosses/TheElder/TheElderUnstable
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/AtlasBosses/TheElder/TheElderUnstable
 |monster_type_id                  = TheElderUnstable
-|mod_ids                          = MonsterMapBoss, CannotBeSlowedBelowValueBosses, ElderNoEnergyShieldRecharge, ElderEnergyShieldStartsAtZero, MonsterChaosTakenOnES, MonsterCannotBeSlowed, MonsterImplicitCannotBeStunned1, MonsterCannotBeKnockedBack, StunRepeatImmunityMapBossSuper, PinnacleAtlasBoss, MonsterDroppedItemsInfluenceElder
+|mod_ids                          = MonsterMapBoss, CannotBeSlowedBelowValueBosses, ElderNoEnergyShieldRecharge, ElderEnergyShieldStartsAtZero, MonsterChaosTakenOnES, MonsterCannotBeSlowed, MonsterCannotBeKnockedBack, StunRepeatImmunityMapBossSuper, PinnacleAtlasBoss, MonsterDroppedItemsInfluenceElderBoss
 |tags                             = elder_boss, eldritch, flesh_armour, human, humanoid, immobile, is_unarmed, large_height, non_attacking, red_blood
 |name                             = The Elder
 |size                             = 3