--- Monster:Metadata/Monsters/Skeletons/ConstructMeleeBossSideAreaInvasion
+++ Export
@@ -3,7 +3,7 @@
 |monster_type_id                  = ConstructMeleeArmouredBossSideArea
 |mod_ids                          = MonsterSideAreaBoss, MonsterAdditionalTotems2, MonsterInvasionMercilessDrops2, MonsterNoMapDrops, IsInvasionBoss, MonsterKillTrapsMinesTotemsOnDeath
 |endgame_mod_ids                  = MonsterInvasionBossMap
-|tags                             = bludgeoning_weapon, bone_armour, bones, caster, chaos_affinity, cold_affinity, construct, fire_affinity, has_two_hand_mace, has_two_handed_melee, invasion_boss, large_height, lightning_affinity, not_dex, not_int, physical_affinity, ranged, very_slow_movement
+|tags                             = bludgeoning_weapon, bone_armour, bones, caster, chaos_affinity, cold_affinity, fire_affinity, has_two_hand_mace, has_two_handed_melee, invasion_boss, large_height, lightning_affinity, not_dex, not_int, physical_affinity, ranged, skeleton, undead, very_slow_movement
 |skill_ids                        = MonsterArcticBreathSideArea, MonsterFreezingPulseSideArea, MonsterFlameRedSideArea, MonsterFireballSideArea, MonsterShockNovaSideArea, MonsterStormCallSideArea, MonsterShockwaveTotemSideArea, MonsterSpinestormSideArea, MonsterEtherealKnivesSideArea
 |name                             = Haviri, Vaal Metalsmith
 |size                             = 3