--- Monster:Metadata/Monsters/LeagueSanctum/Boss/Lycia2Boss
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/LeagueSanctum/Boss/Lycia2Boss
 |monster_type_id                  = SanctumLycia2Boss
 |mod_ids                          = MonsterMapBoss, CannotBeSlowedBelowValueBosses, StunRepeatImmunityMapBossSuper
-|tags                             = caster, demon, human, humanoid, not_dex, ranged, red_blood, very_fast_movement
+|tags                             = caster, demon, human, humanoid, not_dex, ranged, red_blood, sanctum_monster, very_fast_movement
 |skill_ids                        = MPSSanctumLycia2BasicProjectile, SanctumLycia2Melee, EDSSanctumLycia2DelayedBlast, SSMSanctumLycia2SummonVortex, EASSanctumLycia2MoveVortexes, DTTSanctumLycia2Dash, DTTSanctumLycia2DashSlam, GASanctumLycia2DashSlamImpact, GTSanctumLycia2TriggeredCircleWaves, GSSanctumLycia2CircleWave, GTSanctumLycia2TriggeredCircleWavesTriggerer, EASSanctumLyciaWindTunnel, GSSanctumLycia2WindTunnelImpact, GSSanctumLycia2WindTunnelImpactFinal
 |name                             = Lycia, Herald of the Scourge
 |size                             = 4
@@ -11,7 +11,7 @@
 |model_size_multiplier            = 1.25
 |experience_multiplier            = 3.0
 |damage_multiplier                = 2.25
-|health_multiplier                = 13.5
+|health_multiplier                = 27.0
 |critical_strike_chance           = 5.0
 |attack_speed                     = 1.5
 }}