--- Monster:Metadata/Monsters/Skeletons/SkeletonMeleeKnightElementalMaulIncursionDelve
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/Skeletons/SkeletonMeleeKnightElementalMaulIncursionDelve
 |monster_type_id                  = VaalConstructIncursionTempleMap
-|tags                             = bludgeoning_weapon, bone_armour, bones, construct, delve_monster, has_two_hand_mace, has_two_handed_melee, medium_height, melee, not_dex, not_int, physical_affinity, very_slow_movement
+|tags                             = bludgeoning_weapon, bone_armour, bones, construct, delve_monster, has_two_hand_mace, has_two_handed_melee, medium_height, melee, not_dex, not_int, physical_affinity, skeleton, undead, very_slow_movement
 |skill_ids                        = MeleeAtAnimationSpeed, ElementalHitSkeletonKnightIncursion
 |name                             = Vaal Fallen
 |size                             = 3