--- Monster:Metadata/Monsters/BreachBosses/BreachBossLightningWild
+++ Export
@@ -1,6 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/BreachBosses/BreachBossLightningWild
 |monster_type_id                  = BreachBossLightningWild
+|mod_ids                          = MonsterNoDrops
 |tags                             = caster, demon, has_wand, light_armour, lightning_affinity, medium_height, red_blood, very_fast_movement
 |skill_ids                        = BreachBossLightningWhip, BreachLightningOrbs, BreachRecall
 |name                             = Esh, Forked Thought