--- Monster:Metadata/Monsters/LeagueCrucible/CriticalSpeed/DemonRanged
+++ Export
@@ -0,0 +1,17 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueCrucible/CriticalSpeed/DemonRanged
+|monster_type_id                  = CrucibleDemonRanged
+|mod_ids                          = MonsterConvertToFireDamage2, MonsterRangeVarianceMedium
+|tags                             = demon, flesh_armour, is_unarmed, medium_height, medium_movement, not_int, not_str, physical_affinity, puncturing_weapon, ranged
+|skill_ids                        = EmptyActionFemaleDemonProjectiles, EmptyActionFemaleDemonProjectilesAhead, CrucibleDemonFemaleRangedGeometryProjectileLeft, CrucibleDemonFemaleRangedGeometryProjectileRight, CrucibleDemonFemaleRangedGeometryProjectileLeftAhead, CrucibleDemonFemaleRangedGeometryProjectileRightAhead
+|name                             = Forged Tentacle Demon
+|size                             = 3
+|minimum_attack_distance          = 6
+|maximum_attack_distance          = 6
+|model_size_multiplier            = 1.0
+|experience_multiplier            = 1.4
+|damage_multiplier                = 1.96
+|health_multiplier                = 1.68
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.995
+}}