--- Monster:Metadata/Monsters/Prophecy/VaalFallenMaul
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/Prophecy/VaalFallenMaul
 |monster_type_id                  = ConstructMeleeArmouredProphecy
 |mod_ids                          = MonsterCastsElementalHitText
-|tags                             = bludgeoning_weapon, bone_armour, bones, construct, has_two_hand_mace, has_two_handed_melee, medium_height, melee, not_dex, not_int, physical_affinity, very_slow_movement
+|tags                             = bludgeoning_weapon, bone_armour, bones, has_two_hand_mace, has_two_handed_melee, medium_height, melee, not_dex, not_int, physical_affinity, skeleton, undead, very_slow_movement
 |skill_ids                        = Melee, ElementalHitSkeletonKnight
 |name                             = Vaal Memory
 |size                             = 3