--- Monster:Metadata/Monsters/LeagueSanctum/Guards/Guard9
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/LeagueSanctum/Guards/Guard9
 |monster_type_id                  = SanctumGuard9
 |mod_ids                          = MonsterTagSanctumGuard, MonsterSanctumGuardVolatileOnDeath, MonsterSanctumGuardSummonGargoyle, MonsterSanctumGuardMirrorImage
-|tags                             = fast_movement, human, humanoid, not_str, ranged, red_blood, sanctum_guard
+|tags                             = fast_movement, human, humanoid, not_str, ranged, red_blood, sanctum_guard, sanctum_monster
 |skill_ids                        = SanctumCausticBomb, SanctumSmokeMine, MMSSanctumSingleTrapThrow, MMSSanctumMultiTrapThrow, GSSanctumThugTorchBomb
 |name                             = Masked Combatant
 |size                             = 3
