--- Monster:Metadata/Monsters/RaisedSkeletons/NecromancerRaisedSkeletonStandard
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/RaisedSkeletons/NecromancerRaisedSkeletonStandard
 |monster_type_id                  = NecromancerRaisedSkeleton
 |mod_ids                          = MonsterSpeedAndDamageFixupComplete, MonsterAttackBlock30Bypass15, CannotGainAfflictedMods, SummonSkeletonsSkeletonIsWarrior
-|tags                             = bone_armour, bones, demon, has_one_hand_sword, has_one_handed_melee, medium_height, melee, physical_affinity, slashing_weapon, very_slow_movement
+|tags                             = bone_armour, bones, has_one_hand_sword, has_one_handed_melee, medium_height, melee, physical_affinity, skeleton, slashing_weapon, undead, very_slow_movement
 |skill_ids                        = Melee
 |name                             = Summoned Skeleton
 |size                             = 3
