--- Monster:Metadata/Monsters/Undying/CityStalkerFemaleMapBoss
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/Undying/CityStalkerFemaleMapBoss
 |monster_type_id                  = CityStalkerMapBoss
 |mod_ids                          = MonsterIncreasedTrapsAllowed, MonsterMapBoss, MonsterImplicitTrapsInvulnerable4Sec, MonsterKillTrapsMinesTotemsOnDeath, StunRepeatImmunityMapBoss_
-|tags                             = caster, demon, fast_movement, fire_affinity, flesh_armour, has_one_hand_sword, has_one_handed_melee, humanoid, lightning_affinity, medium_height, melee, non_unique_map_boss, not_int, physical_affinity, red_blood, undying
+|tags                             = caster, demon, dominus_monster_map, fast_movement, fire_affinity, flesh_armour, has_one_hand_sword, has_one_handed_melee, humanoid, lightning_affinity, medium_height, melee, non_unique_map_boss, not_int, physical_affinity, red_blood, undying
 |skill_ids                        = MeleeAtAnimationSpeedUnique, MonsterLightningTrapMapBoss, MonsterFireTrapMapBoss, MonsterBearTrapMapBoss
 |name                             = Excellis Aurafix
 |size                             = 3
