--- Monster:Metadata/Monsters/KitavaBoss/KitavaFinalMapArchnemesis
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/KitavaBoss/KitavaFinalMapArchnemesis
 |monster_type_id                  = KitavaFinalMapStandalone
-|mod_ids                          = MonsterMapBoss, MonsterImplicitCannotBeStunned1, MonsterCannotBeSlowed, StunRepeatImmunityMapBoss_
+|mod_ids                          = MonsterMapBoss, MonsterCannotBeSlowed, StunRepeatImmunityMapBoss_, MonsterImplicitCannotBeStunned1
 |tags                             = bludgeoning_weapon, caster, demon, fire_affinity, flesh_armour, immobile, is_unarmed, physical_affinity, red_blood
 |skill_ids                        = KitavaXBlastFinalStandalone, KitavaSwipeRightFinalStandalone, KitavaSwipeLeftFinalStandalone, KitavaFinalDevouringDarknessRightArchnemesis, KitavaFinalDevouringDarknessLeftArchnemesis, KitavaUltimateDeathBreathRightStandalone, KitavaUltimateDeathBreathLeftStandalone, KitavaBeamOrbMiddleFinalPhaseStandalone, KitavaBeamOrbRightFinalPhaseStandalone, GTStandaloneKitavaXNuke, SOStandaloneKitavaXNuke, GSStandaloneKitavaXNuke
 |name                             = Kitava, The Destroyer
