--- Monster:Metadata/Monsters/SkeletonCannon/SkeletonCannonAbyssLichMinion
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/SkeletonCannon/SkeletonCannonAbyssLichMinion
 |monster_type_id                  = SkeletonCannonAbyss
 |mod_ids                          = SpiritChargeTier1, VariableEmergeSpeedAbyss, CannotBeUsedAsMinion
-|tags                             = abyssal, bludgeoning_weapon, bone_armour, extra_small_height, has_claw, has_one_handed_melee, medium_movement, not_int, physical_affinity, ranged, skeleton
+|tags                             = abyssal, bludgeoning_weapon, bone_armour, extra_small_height, has_claw, has_one_handed_melee, medium_movement, not_int, physical_affinity, ranged, skeleton, undead
 |skill_ids                        = SkeletonCannonProjectile
 |name                             = Risen Maw
 |size                             = 4
