--- Monster:Metadata/Monsters/Skeletons/ConstructMeleeBossSideArea
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/Skeletons/ConstructMeleeBossSideArea
 |monster_type_id                  = ConstructMeleeArmouredBossSideArea
 |mod_ids                          = MonsterSideAreaBoss, MonsterAdditionalTotems2, MonsterInvasionMercilessDrops2, MonsterNoMapDrops, MonsterKillTrapsMinesTotemsOnDeath
-|tags                             = bludgeoning_weapon, bone_armour, bones, caster, chaos_affinity, cold_affinity, construct, fire_affinity, has_two_hand_mace, has_two_handed_melee, large_height, lightning_affinity, not_dex, not_int, physical_affinity, ranged, vaalsidearea_boss, very_slow_movement
+|tags                             = bludgeoning_weapon, bone_armour, bones, caster, chaos_affinity, cold_affinity, fire_affinity, has_two_hand_mace, has_two_handed_melee, large_height, lightning_affinity, not_dex, not_int, physical_affinity, ranged, skeleton, undead, vaalsidearea_boss, very_slow_movement
 |skill_ids                        = MonsterArcticBreathSideArea, MonsterFreezingPulseSideArea, MonsterFlameRedSideArea, MonsterFireballSideArea, MonsterShockNovaSideArea, MonsterStormCallSideArea, MonsterShockwaveTotemSideArea, MonsterSpinestormSideArea, MonsterEtherealKnivesSideArea
 |name                             = Haviri, Vaal Metalsmith
 |size                             = 3