--- Monster:Metadata/Monsters/LeagueAncestral/AncestralForestKuniKuni
+++ Export
@@ -0,0 +1,17 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueAncestral/AncestralForestKuniKuni
+|monster_type_id                  = AncestralForestKuniKuni
+|mod_ids                          = AncestralTrailMonsterAilmentResistanceTawhoa, AncestralTrialMonsterBasicFleeAndDamage
+|tags                             = beast, mammal_beast, medium_movement, not_int, red_blood
+|skill_ids                        = MeleeAtAnimationSpeed, AncestralKuniCharge, AncestralKuniTableCharge
+|name                             = Enraged Kunekune
+|size                             = 3
+|minimum_attack_distance          = 5
+|maximum_attack_distance          = 12
+|model_size_multiplier            = 1.0
+|experience_multiplier            = 1.0
+|damage_multiplier                = 1.25
+|health_multiplier                = 1.1
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.5
+}}