--- Monster:Metadata/Monsters/LeagueSanctum/Guards/Guard6
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/LeagueSanctum/Guards/Guard6
 |monster_type_id                  = SanctumGuard6
 |mod_ids                          = MonsterTagSanctumGuard, MonsterSanctumGuardVolatileOnDeath, MonsterSanctumGuardSummonGargoyle
-|tags                             = bone_armour, caster, fast_movement, fire_affinity, has_bow, human, humanoid, large_height, not_int, puncturing_weapon, ranged, red_blood, sanctum_guard
+|tags                             = bone_armour, caster, fast_movement, fire_affinity, has_bow, human, humanoid, large_height, not_int, puncturing_weapon, ranged, red_blood, sanctum_guard, sanctum_monster
 |skill_ids                        = MeleeLioneyeSanctum, EmptyActionSanctumLioneyeAttackLeft, EmptyActionSanctumLioneyeAttackRight, EmptyActionSanctumLioneyeAttackBack, LioneyeTriggeredProjectileSanctum, GTSanctumLioneyeArrowstorm, SanctumLioneyeFireCircleBombard, EmptyActionSanctumLioneyeFlareStorm, SanctumLioneyeSpawnFlareObject
 |name                             = Templar Marksman
 |size                             = 3
