--- Monster:Metadata/Monsters/Atziri/AtziriSpear2
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/Atziri/AtziriSpear2
 |monster_type_id                  = AtziriMirroredUber
-|mod_ids                          = MonsterAtziriMapBoss, MonsterNoDropsOrExperience, MonsterImplicitCannotBeStunned1, MonsterMapBoss, StunRepeatImmunityMapBoss_
+|mod_ids                          = MonsterAtziriMapBoss, MonsterNoDropsOrExperience, MonsterMapBoss, StunRepeatImmunityMapBoss_
 |tags                             = caster, extra_large_height, flesh_armour, humanoid, immobile, is_unarmed, large_model, physical_affinity, puncturing_weapon, red_blood
 |skill_ids                        = AtziriSpearThrow
 |name                             = Atziri, Queen of the Vaal