--- Monster:Metadata/Monsters/Skeletons/SkeletonMeleeKnightElementalSwordHellscape
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/Skeletons/SkeletonMeleeKnightElementalSwordHellscape
 |monster_type_id                  = ConstructMeleeArmoured
-|tags                             = bone_armour, bones, construct, has_two_hand_sword, has_two_handed_melee, melee, not_dex, not_int, physical_affinity, slashing_weapon, very_slow_movement
+|tags                             = bone_armour, bones, has_two_hand_sword, has_two_handed_melee, melee, not_dex, not_int, physical_affinity, skeleton, slashing_weapon, undead, very_slow_movement
 |skill_ids                        = MeleeAtAnimationSpeed, ElementalHitSkeletonKnight
 |name                             = Vaal Fallen
 |size                             = 3