--- Monster:Metadata/Monsters/LeagueSanctum/Guards/Guard4
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/LeagueSanctum/Guards/Guard4
 |monster_type_id                  = SanctumGuard4
 |mod_ids                          = MonsterTagSanctumGuard, MonsterSanctumGuardVolatileOnDeath, MonsterSanctumGuardSummonGargoyle
-|tags                             = bludgeoning_weapon, caster, fire_affinity, has_staff, has_two_handed_melee, human, humanoid, light_armour, lightning_affinity, medium_movement, melee, not_dex, not_str, physical_affinity, ranged, red_blood, sanctum_guard
+|tags                             = bludgeoning_weapon, caster, fire_affinity, has_staff, has_two_handed_melee, human, humanoid, light_armour, lightning_affinity, medium_movement, melee, not_dex, not_str, physical_affinity, ranged, red_blood, sanctum_guard, sanctum_monster
 |skill_ids                        = SanctumLightningBolt, GTSanctumLightningBoltCone, EDSSanctumBoneBlast, GTSanctumBoneStormCall, SSMSanctumSummonReaperVolatiles, GTSanctumSummonReaperVolatiles, GPSSanctumGuardReaperProjectile
 |name                             = Undying Archmage
 |size                             = 3
