--- Monster:Metadata/Monsters/Skeletons/CowardsTrial/SkeletonCannonCowardsTrial
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/Skeletons/CowardsTrial/SkeletonCannonCowardsTrial
 |monster_type_id                  = BetrayalSkeletonCannon
-|tags                             = bludgeoning_weapon, bone_armour, caster, extra_small_height, has_claw, has_one_handed_melee, medium_movement, melee, not_int, physical_affinity, skeleton
+|tags                             = bludgeoning_weapon, bone_armour, caster, extra_small_height, has_claw, has_one_handed_melee, medium_movement, melee, not_int, physical_affinity, skeleton, undead
 |skill_ids                        = Melee, WalkEmergeBetrayal, EmptyActionBetrayalSkeletonCannonBombard, BetrayalSkeletonCannonBombard
 |name                             = Ravager Maw
 |size                             = 4