--- Monster:Metadata/Monsters/Spirit/TormentedLibrarian
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/Spirit/TormentedLibrarian
 |monster_type_id                  = TormentedSpirit
 |mod_ids                          = TormentSpiritDropsIncrease3, TormentSpiritPhasing
-|tags                             = ghost, ghost_armour, ghost_blood, is_unarmed, medium_height, medium_movement, non_attacking
+|tags                             = ghost, ghost_armour, ghost_blood, is_unarmed, medium_height, medium_movement, non_attacking, undead
 |skill_ids                        = Haunt, TormentSpiritPause, WalkEmergeTormentedSpirit
 |name                             = Tormented Seditionist
 |size                             = 3