--- Monster:Metadata/Monsters/BreachBosses/BreachBossPhysicalWild
+++ Export
@@ -1,6 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/BreachBosses/BreachBossPhysicalWild
 |monster_type_id                  = BreachBossPhysicalWild
+|mod_ids                          = MonsterNoDrops
 |tags                             = caster, cleaving_weapon, demon, has_two_hand_axe, has_two_handed_melee, light_armour, medium_height, melee, physical_affinity, ranged, red_blood, very_fast_movement
 |skill_ids                        = Melee, BreachBoneNovaWild, BreachBoneMortarBoss, BreachBossCleavePhysical
 |name                             = Uul-Netol, Unburdened Flesh
