--- Monster:Metadata/Monsters/LeagueBlight/Lightning/BlightLightningVolatileCorpseWheel
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/LeagueBlight/Lightning/BlightLightningVolatileCorpseWheel
 |monster_type_id                  = BlightLightningCorpseWheel
 |mod_ids                          = BlightLightningType, MonsterElementalSkeletonLightning, MonsterImmuneToShock
-|tags                             = blight_monster, bone_armour, bones, caster, extra_small_height, has_one_hand_sword, has_one_handed_melee, lightning_affinity, no_strider_mod, not_dex, skeleton, slashing_weapon, uses_suicide_explode, very_slow_movement
+|tags                             = blight_monster, bone_armour, bones, caster, extra_small_height, has_one_hand_sword, has_one_handed_melee, lightning_affinity, no_strider_mod, not_dex, skeleton, slashing_weapon, undead, uses_suicide_explode, very_slow_movement
 |skill_ids                        = BlightCorpseWheelSuicideExplosion
 |name                             = Blighted Dreadwheel
 |size                             = 1
