--- Monster:Metadata/Monsters/LeagueSanctum/Guards/Guard7
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/LeagueSanctum/Guards/Guard7
 |monster_type_id                  = SanctumGuard7
 |mod_ids                          = MonsterTagSanctumGuard, MonsterSanctumGuardVolatileOnDeath, MonsterSanctumGuardSummonGargoyle
-|tags                             = caster, human, humanoid, medium_movement, not_dex, not_int, ranged, red_blood, sanctum_guard
+|tags                             = caster, human, humanoid, medium_movement, not_dex, not_int, ranged, red_blood, sanctum_guard, sanctum_monster
 |skill_ids                        = MPSSanctumGuardFireProjectile, SanctumMiniFlamenado, GTSanctumFireTornados, GTSanctumFireMarkerBlast, GSSanctumFlameBlastImpact
 |name                             = Crimson Archmage
 |size                             = 3