--- Monster:Metadata/Monsters/LeagueAncestral/AncestralJadecaster~
+++ Export
@@ -0,0 +1,17 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueAncestral/AncestralJadecaster_
+|monster_type_id                  = AncestralJadecaster
+|mod_ids                          = AncestralTrialMonsterChampionFleeAndDamage, AncestralTrialMonsterBlockReductionTukohama
+|tags                             = human, humanoid, karui, medium_movement, red_blood
+|skill_ids                        = MeleeAtAnimationSpeed, MPSAncestralJadeProjectile, EASAncestralEncaseEnemy, GAAncestralJadePrison, SSMAncestralJadeVolatile, SSMAncestralJadeVolatileDefense
+|name                             = Jadecrafter
+|size                             = 3
+|minimum_attack_distance          = 5
+|maximum_attack_distance          = 14
+|model_size_multiplier            = 1.0
+|experience_multiplier            = 1.0
+|damage_multiplier                = 2.25
+|health_multiplier                = 3.85
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.5
+}}