--- Monster:Metadata/Monsters/Miner/MinerHammerMapBoss
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/Miner/MinerHammerMapBoss
 |monster_type_id                  = MinerMapBoss
 |mod_ids                          = MonsterMapBoss, StunRepeatImmunityMapBoss_
-|tags                             = bludgeoning_weapon, caster, has_one_hand_mace, has_one_handed_melee, medium_height, melee, mud_blood, non_unique_map_boss, physical_affinity, slow_movement, stone_armour, undead
+|tags                             = bludgeoning_weapon, caster, has_one_hand_mace, has_one_handed_melee, medium_height, melee, mud_blood, non_unique_map_boss, physical_affinity, slow_movement, stone_armour, undead, unholy_trio_boss_map
 |skill_ids                        = MeleeAtAnimationSpeed, MonsterGlacialCascadePhysical, MinerBeaconMapBoss
 |name                             = Lord of the Hollows
 |size                             = 3