--- Monster:Metadata/Monsters/Atziri/Atziri
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/Atziri/Atziri
 |monster_type_id                  = Atziri
-|mod_ids                          = MonsterAtziriMapBoss, AtziriReflectCurses, AtziriMinorDamageReflect, MonsterImplicitCannotBeStunned1, CannotBeSlowedBelowValueBosses, MonsterMapBoss, StunRepeatImmunityMapBoss_
+|mod_ids                          = MonsterAtziriMapBoss, AtziriReflectCurses, AtziriMinorDamageReflect, CannotBeSlowedBelowValueBosses, MonsterMapBoss, StunRepeatImmunityMapBoss_
 |tags                             = atziri_boss, caster, extra_large_height, fast_movement, fire_affinity, flesh_armour, has_one_hand_sword, has_one_handed_melee, humanoid, large_model, lightning_affinity, physical_affinity, puncturing_weapon, ranged, red_blood, unique_map_boss
 |skill_ids                        = AtziriMirrorImage, AtziriSummonDemons, AtziriStormCall, AtziriStormCallEmpowered, AtziriFlameblast, AtziriFlameblastEmpowered, AtziriSpearThrow, AtziriSpearThrowEmpowered
 |name                             = Atziri, Queen of the Vaal
