--- Monster:Metadata/Monsters/Daemon/TormentDaemonAnomaliesAndTentaclesFast
+++ Export
@@ -3,7 +3,7 @@
 |monster_type_id                  = Daemon
 |mod_ids                          = MonsterDamageVersusPlayerMinions25
 |tags                             = extra_extra_small_height, ghost_armour, is_daemon, is_unarmed, physical_affinity, very_slow_movement
-|skill_ids                        = TormentSummonAnomalyDaemonFast, TormentSummonTentacleSlammerDaemonFast
+|skill_ids                        = TormentSummonAnomalyDaemonFast, TormentSummonTentacleSlammerDaemonFast, TormentSummonAnomalyDaemonFastPlayer
 |name                             = Torment Daemon
 |size                             = 3
 |minimum_attack_distance          = 6