--- Monster:Metadata/Monsters/Skeletons/SkeletonMeleeKnightElementalSwordIncursionDelve
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/Skeletons/SkeletonMeleeKnightElementalSwordIncursionDelve
 |monster_type_id                  = VaalConstructIncursionTempleMap
-|tags                             = bone_armour, bones, construct, delve_monster, has_two_hand_sword, has_two_handed_melee, medium_height, melee, not_dex, not_int, physical_affinity, slashing_weapon, very_slow_movement
+|tags                             = bone_armour, bones, construct, delve_monster, has_two_hand_sword, has_two_handed_melee, medium_height, melee, not_dex, not_int, physical_affinity, skeleton, slashing_weapon, undead, very_slow_movement
 |skill_ids                        = MeleeAtAnimationSpeed, ElementalHitSkeletonKnightIncursion
 |name                             = Vaal Fallen
 |size                             = 3
