--- Monster:Metadata/Monsters/LeagueSanctum/Fodder/PaleElite2
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/LeagueSanctum/Fodder/PaleElite2
 |monster_type_id                  = SanctumPaleElite2
-|tags                             = bone_armour, demon, fast_movement, is_unarmed, not_int, physical_affinity, puncturing_weapon, red_blood
+|tags                             = bone_armour, demon, fast_movement, is_unarmed, not_int, physical_affinity, puncturing_weapon, red_blood, sanctum_monster
 |skill_ids                        = EAAHellscapeStabbyStab, DTTHellscapeStabbySkyStab, DTTHellscapeStabWeb, GASanctumStabbyCleave1, GASanctumStabbyCleave2, GASanctumStabbyCleave3, GASanctumStabbyCleaveStab, GASanctumStabWeb, GASanctumPaleEliteSkyStab
 |name                             = Unearthly Horror
 |size                             = 3
