--- Monster:Metadata/Monsters/LeagueSanctum/Fodder/GargoyleSmall
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/LeagueSanctum/Fodder/GargoyleSmall
 |monster_type_id                  = SanctumGargoyleSmall
-|tags                             = construct, has_one_hand_mace, has_one_handed_melee, medium_height, medium_movement, melee, mud_blood, not_dex, not_int, physical_affinity, slashing_weapon, stone_armour
+|tags                             = construct, has_one_hand_mace, has_one_handed_melee, medium_height, medium_movement, melee, mud_blood, not_dex, not_int, physical_affinity, sanctum_monster, slashing_weapon, stone_armour, unusable_corpse
 |skill_ids                        = DTTSanctumIzaroGargoyleLeapSlam, GASanctumIzaroGargoyleLeapImpact, MeleeAtAnimationSpeedSanctum
 |name                             = Aureate Gargoyle
 |size                             = 3
