--- Monster:Metadata/Monsters/Skeletons/SkeletonMeleeKnightElementalLargeSwordIncursionMapMemoryLine
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/Skeletons/SkeletonMeleeKnightElementalLargeSwordIncursionMapMemoryLine
 |monster_type_id                  = VaalConstructLargeIncursionMap
-|tags                             = bone_armour, bones, construct, has_one_hand_sword, has_one_handed_melee, incursion_monster, large_height, melee, no_spirit_walkers_archnemesis, not_dex, not_int, physical_affinity, slashing_weapon, slow_movement
+|tags                             = bone_armour, bones, construct, has_one_hand_sword, has_one_handed_melee, incursion_monster, large_height, melee, no_spirit_walkers_archnemesis, not_dex, not_int, physical_affinity, skeleton, slashing_weapon, slow_movement, undead
 |skill_ids                        = MeleeAtAnimationSpeed, ElementalHitSkeletonKnightIncursion
 |name                             = Colossal Vaal Fallen
 |size                             = 3