--- Monster:Metadata/Monsters/KitavaBoss/Kitava3
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/KitavaBoss/Kitava3
 |monster_type_id                  = Kitava
-|mod_ids                          = MonsterActBossAct5, MonsterImplicitCannotBeStunned1, MonsterCannotBeSlowed, StunRepeatImmunityActBoss, BossHealOnPlayerDeathMinor
+|mod_ids                          = MonsterActBossAct5, MonsterCannotBeSlowed, StunRepeatImmunityActBoss, BossHealOnPlayerDeathMinor, MonsterImplicitCannotBeStunned1
 |tags                             = bludgeoning_weapon, demon, extra_extra_large_height, flesh_armour, is_unarmed, melee, physical_affinity, red_blood, very_slow_movement
 |skill_ids                        = Melee
 |name                             = Kitava, the Insatiable