--- Monster:Metadata/Monsters/LeagueAncestral/AncestralIcecaster
+++ Export
@@ -0,0 +1,17 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueAncestral/AncestralIcecaster
+|monster_type_id                  = AncestralIcecaster
+|mod_ids                          = AncestralTrialMonsterTotemDestructionRangeIcecaster, AncestralTrialMonsterIntermediateFleeAndDamage
+|tags                             = human, humanoid, karui, medium_movement, not_str, red_blood
+|skill_ids                        = MeleeAtAnimationSpeed, MPSAncestralIceCasterProjectile, CGEAncestralChilledGround, EASAncestralLeagueKaruiIceCasterBlinkBack, AncestorLeagueIceCasterFrostblink
+|name                             = Sunset Sage
+|size                             = 3
+|minimum_attack_distance          = 5
+|maximum_attack_distance          = 14
+|model_size_multiplier            = 1.0
+|experience_multiplier            = 1.0
+|damage_multiplier                = 1.5
+|health_multiplier                = 1.5
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.5
+}}