--- Monster:Metadata/Monsters/LeagueBlight/Lightning/BlightLightningCageSkeleton
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/LeagueBlight/Lightning/BlightLightningCageSkeleton
 |monster_type_id                  = BlightLightningCageSkeleton
 |mod_ids                          = BlightLightningType, MonsterImmuneToShock
-|tags                             = blight_monster, bones, caster, flesh_armour, is_unarmed, large_height, lightning_affinity, not_dex, not_int, ranged, skeleton, slashing_weapon, very_slow_movement
+|tags                             = blight_monster, bones, caster, flesh_armour, is_unarmed, large_height, lightning_affinity, not_dex, not_int, ranged, skeleton, slashing_weapon, undead, very_slow_movement
 |skill_ids                        = GeometryAttackBlightCageSkeletonSpin, BlightCageSkeletonTriggeredSpark
 |name                             = Blighted Condemned
 |size                             = 1