--- Monster:Metadata/Monsters/LeagueCrucible/Lightning/HydraBoss
+++ Export
@@ -0,0 +1,17 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueCrucible/Lightning/HydraBoss
+|monster_type_id                  = CrucibleHydraBoss
+|mod_ids                          = MonsterCrucibleMapBoss, CannotBeSlowedBelowValueBosses, StunRepeatImmunityMapBossSuper
+|tags                             = cold_affinity, flesh_armour, has_bow, human, humanoid, large_height, not_int, not_str, physical_affinity, puncturing_weapon, ranged, standalone_map_boss, very_fast_movement
+|skill_ids                        = HydraDoomArrow, HydraBarrage, HydraForkArrow, HydraTeleportShotStandalone, GTStandaloneHydraTurretSpawner1, GTStandaloneHydraTurretSpawner2, SOStandaloneHydraTurret1, SOStandaloneHydraTurret2
+|name                             = Ankhona, Runner of the Hollow
+|size                             = 4
+|minimum_attack_distance          = 6
+|maximum_attack_distance          = 80
+|model_size_multiplier            = 1.69
+|experience_multiplier            = 2.5
+|damage_multiplier                = 2.5
+|health_multiplier                = 13.75
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.605
+}}