--- Monster:Metadata/Monster/CageSpider/CageSpider
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monster/CageSpider/CageSpider
 |monster_type_id                  = CageSpider
-|tags                             = flesh_armour, is_unarmed, large_height, melee, not_dex, not_int, physical_affinity, skeleton, slashing_weapon, slow_movement
+|tags                             = flesh_armour, is_unarmed, large_height, melee, not_dex, not_int, physical_affinity, skeleton, slashing_weapon, slow_movement, undead
 |skill_ids                        = Melee, CageSpiderCyclone
 |name                             = Rattling Condemned
 |size                             = 3