--- Monster:Metadata/Monsters/LeagueSanctum/Fodder/PaleElite1
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/LeagueSanctum/Fodder/PaleElite1
 |monster_type_id                  = SanctumPaleElite1
-|tags                             = animal_claw_weapon, bone_armour, caster, demon, is_unarmed, lightning_affinity, medium_movement, not_dex, not_str, red_blood
+|tags                             = animal_claw_weapon, bone_armour, caster, demon, is_unarmed, lightning_affinity, medium_movement, not_dex, not_str, red_blood, sanctum_monster
 |skill_ids                        = TBSanctumPaleLightningBoltSpammableLeft, TBSanctumPaleLightningBoltSpammableRight, GSSanctumPaleEliteBoltImpact, GSSanctumPaleEliteOmegaBeam, TeleportHellscapePaleElite, TeleportHellscapePaleEliteForwards, TeleportHellscapePaleEliteToAlly
 |name                             = Lurid Angel
 |size                             = 3
