--- Monster:Metadata/Monsters/KitavaBoss/KitavaFinal
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/KitavaBoss/KitavaFinal
 |monster_type_id                  = KitavaFinal
-|mod_ids                          = MonsterActBossAct10, MonsterActBossAct10BonusRarity_, MonsterImplicitCannotBeStunned1, MonsterCannotBeSlowed, StunRepeatImmunityActBoss, BossHealOnPlayerDeath
+|mod_ids                          = MonsterActBossAct10, MonsterActBossAct10BonusRarity_, MonsterCannotBeSlowed, StunRepeatImmunityActBoss, BossHealOnPlayerDeath, MonsterImplicitCannotBeStunned1
 |tags                             = bludgeoning_weapon, caster, demon, extra_extra_large_height, fire_affinity, flesh_armour, is_unarmed, physical_affinity, red_blood, very_slow_movement
 |skill_ids                        = KitavaXBlastFinal, KitavaSwipeRightFinal, KitavaSwipeLeftFinal, KitavaBeamSweepFrontCentre, KitavaXNuke, KitavaDeathBreathMiddle, KitavaDeathBreathRight, KitavaDeathBreathLeft, KitavaBeamSweepFrontRight, KitavaBeamSweepFrontLeft, KitavaBeamSweepMiddleCentre, KitavaBeamSweepMiddleRight, KitavaBeamSweepMiddleLeft, KitavaBeamSweepBackCentre, KitavaBeamSweepBackRight, KitavaBeamSweepBackLeft, KitavaBeamSweepDoubleCentre, KitavaBeamSweepDoubleRight, KitavaBeamSweepDoubleLeft, KitavaFinalDevouringDarknessRight, KitavaFinalDevouringDarknessLeft, KitavaFinalDevouringDarknessBoth, KitavaBeamOrbMiddle, KitavaBeamOrbRight, KitavaBeamOrbLeft, KitavaUltimateDeathBreathRight, KitavaUltimateDeathBreathLeft, KitavaBeamOrbMiddleFinalPhase, KitavaBeamOrbRightFinalPhase, KitavaBeamOrbLeftFinalPhase
 |name                             = Kitava, the Insatiable