--- Monster:Metadata/Monsters/LeagueAncestral/AncestralThunderbird
+++ Export
@@ -0,0 +1,17 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueAncestral/AncestralThunderbird
+|monster_type_id                  = AncestralThunderbird
+|mod_ids                          = AncestralTrailMonsterCriticalResistanceValako, AncestralTrialMonsterBasicFleeAndDamage
+|tags                             = avian_beast, beast, bludgeoning_weapon, fast_movement, has_one_hand_mace, has_one_handed_melee, light_armour, medium_height, melee, not_int, not_str, physical_affinity, red_blood
+|skill_ids                        = MeleeAtAnimationSpeed, LWAncestralThunderbird
+|name                             = Thunderbird
+|size                             = 3
+|minimum_attack_distance          = 5
+|maximum_attack_distance          = 5
+|model_size_multiplier            = 1.0
+|experience_multiplier            = 1.0
+|damage_multiplier                = 0.75
+|health_multiplier                = 0.25
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.5
+}}