--- Monster:Metadata/Monsters/BanditLeaderInt/BanditLeaderAlira
+++ Export
@@ -4,7 +4,7 @@
 |mod_ids                          = MonsterSpellBlock20Bypass20, StunRepeatImmunityActBoss, MonsterQuestBossAct2
 |tags                             = bandit, caster, fire_affinity, has_wand, human, humanoid, light_armour, medium_height, medium_movement, not_dex, not_str, physical_affinity, ranged, red_blood
 |skill_ids                        = MeleeAtAnimationSpeedUnique, MonsterDetonateDeadAreaAlira
-|name                             = Alira
+|name                             = Alira Darktongue
 |size                             = 2
 |minimum_attack_distance          = 6
 |maximum_attack_distance          = 35