--- Monster:Metadata/Monsters/Skeletons/SkeletonCasterKnightLightning
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/Skeletons/SkeletonCasterKnightLightning
 |monster_type_id                  = ConstructCasterLightning
-|tags                             = bone_armour, bones, caster, construct, is_unarmed, lightning_affinity, medium_height, not_dex, not_str, ranged, slashing_weapon, very_slow_movement
+|tags                             = bone_armour, bones, caster, construct, is_unarmed, lightning_affinity, medium_height, not_dex, not_str, ranged, slashing_weapon, undead, very_slow_movement
 |skill_ids                        = SkeletonProjectileLightning
 |name                             = Vaal Thunderguard
 |size                             = 3
