--- Monster:Metadata/Monsters/VultureParasite/VultureParasiteBoss
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/VultureParasite/VultureParasiteBoss
 |monster_type_id                  = VultureParasiteBoss
-|tags                             = avian_beast, beast, flesh_armour, flying, has_one_hand_sword, has_one_handed_melee, large_model, medium_height, medium_movement, melee, physical_affinity, puncturing_weapon, red_blood
+|tags                             = avian_beast, beast, bird_boss, flesh_armour, flying, has_one_hand_sword, has_one_handed_melee, large_model, medium_height, medium_movement, melee, physical_affinity, puncturing_weapon, red_blood
 |skill_ids                        = MeleeAtAnimationSpeed, VultureBossCleave, SummonParasitesOnDeath, VultureParasiteTakeOff
 |name                             = The Hundred Foot Shadow
 |size                             = 5