--- Monster:Metadata/Monsters/BreachBosses/BreachBossOverlordWild
+++ Export
@@ -1,6 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/BreachBosses/BreachBossOverlordWild
 |monster_type_id                  = BreachBossOverlordWild
+|mod_ids                          = MonsterNoDrops
 |tags                             = bludgeoning_weapon, caster, chaos_affinity, demon, has_staff, has_two_handed_melee, light_armour, medium_height, melee, physical_affinity, ranged, red_blood, very_fast_movement
 |skill_ids                        = MeleeAtAnimationSpeedUnique, EASBreachOverlordFireSkill, EASBreachOverlordColdSkill, EASBreachOverlordLightningSkill, EASBreachOverlordPhysicalSkill, EASBreachOverlordChaosSkill, ThrownWeaponBreachBoss, BreachBossFrostBlades, SSMBreachBossOverlordSummonCloneFire, SSMBreachBossOverlordSummonCloneCold, SSMBreachBossOverlordSummonCloneLightning, SSMBreachBossOverlordSummonClonePhysical, SSMBreachBossOverlordSummonCloneChaos
 |name                             = Xesht-Ula, the Open Hand
