--- Monster:Metadata/Monsters/Spirit/TormentedSpiritTest1
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/Spirit/TormentedSpiritTest1
 |monster_type_id                  = TormentedSpiritNew
 |mod_ids                          = TormentSpiritDropsIncrease, TormentSpiritPhasing, TormentAdditionalRareItemsSpirit
-|tags                             = fast_movement, ghost, ghost_armour, ghost_blood, is_unarmed, medium_height, non_attacking
+|tags                             = fast_movement, ghost, ghost_armour, ghost_blood, is_unarmed, medium_height, non_attacking, undead
 |skill_ids                        = Haunt, TormentSpiritPause
 |name                             = Tormented Thug
 |size                             = 3