--- Monster:Metadata/Monsters/BreachBosses/BreachBossColdWild
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/BreachBosses/BreachBossColdWild
 |monster_type_id                  = BreachBossColdWild
-|mod_ids                          = MonsterKillTrapsMinesTotemsOnDeath, MonsterAttackBlock30Bypass15
+|mod_ids                          = MonsterKillTrapsMinesTotemsOnDeath, MonsterAttackBlock30Bypass15, MonsterNoDrops
 |tags                             = caster, cold_affinity, demon, has_one_hand_sword, has_one_handed_melee, light_armour, medium_height, ranged, red_blood, slashing_weapon, very_fast_movement
 |skill_ids                        = BreachBossFrostBlades, SummonBreachIceTotem, BreachColdBarrageBossWild
 |name                             = Tul, Creeping Avalanche
