--- Monster:Metadata/Monsters/LeagueAncestral/AncestralMaata
+++ Export
@@ -0,0 +1,17 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueAncestral/AncestralMaata
+|monster_type_id                  = AncestralMaata
+|mod_ids                          = AncestralTrailMonsterAilmentResistanceTawhoa, AncestralTrialChieftainBalance
+|tags                             = human, humanoid, medium_movement, not_str, red_blood
+|skill_ids                        = MeleeAtAnimationSpeedUnique, DonutCircleMaata, SSMMaataVine, GSMaataVineCascade, GTMaataVineCascade
+|name                             = Maata
+|size                             = 3
+|minimum_attack_distance          = 5
+|maximum_attack_distance          = 20
+|model_size_multiplier            = 1.0
+|experience_multiplier            = 1.0
+|damage_multiplier                = 1.0
+|health_multiplier                = 0.9
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.5
+}}