--- Monster:Metadata/Monsters/MavenBoss/TheMavenMap
+++ Export
@@ -3,7 +3,7 @@
 |monster_type_id                  = TheMavenMap
 |mod_ids                          = MonsterMapBoss, CannotBeSlowedBelowValueBosses, StunRepeatImmunityMapBossSuper, MonsterNoDropsOrExperience, CannotGainAfflictedMods
 |tags                             = immobile, is_unarmed, light_armour, non_attacking, not_dex, red_blood
-|skill_ids                        = EASMavenBossNewMonsterWave, EASMavenBossHeal, MavenMapQuickGuardAlly, EGMavenMapSpaceVines, EASMavenMapBossDecayBuff, SSMMavenMapChillOrbs, SSMMavenMapDegenOrbs, SSMMavenMapVolatileOrbs, SSMMavenMapBeamOrbs, FaceLastTarget, EASMavenMapSpawnClone
+|skill_ids                        = EASMavenBossNewMonsterWave, EASMavenBossHeal, MavenMapQuickGuardAlly, EGMavenMapSpaceVines, EASMavenMapBossDecayBuff, SSMMavenMapChillOrbs, SSMMavenMapDegenOrbs, SSMMavenMapVolatileOrbs, SSMMavenMapBeamOrbs, FaceLastTarget, EASMavenMapSpawnClone, EASMavenSpawnKeystoneBoss
 |name                             = The Maven
 |size                             = 4
 |minimum_attack_distance          = 6
