--- Monster:Metadata/Monsters/LeagueSanctum/Boss/CrimsonPriestBossClone
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/LeagueSanctum/Boss/CrimsonPriestBossClone
 |monster_type_id                  = CrimsonPriestCloneSanctum
 |mod_ids                          = MonsterNoDropsOrExperience
-|tags                             = hard_armour, immobile, is_unarmed, non_attacking, red_blood, slashing_weapon, undead, zombie
+|tags                             = hard_armour, immobile, is_unarmed, non_attacking, red_blood, sanctum_monster, slashing_weapon, undead, zombie
 |skill_ids                        = FaceLastTarget
 |name                             = Crimson Priest
 |size                             = 4