--- Monster:Metadata/Monsters/Skeletons/SkeletonMeleeKnightLightningSwordMaps~
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/Skeletons/SkeletonMeleeKnightLightningSwordMaps_
 |monster_type_id                  = ConstructMeleeLightningMaps
 |mod_ids                          = MonsterPhysicalAddedAsLightningSkeletonMaps
-|tags                             = bone_armour, bones, cleaving_weapon, construct, has_two_hand_sword, has_two_handed_melee, medium_height, melee, not_dex, not_int, physical_affinity, very_slow_movement
+|tags                             = bone_armour, bones, cleaving_weapon, construct, has_two_hand_sword, has_two_handed_melee, medium_height, melee, not_dex, not_int, physical_affinity, skeleton, undead, very_slow_movement
 |skill_ids                        = MeleeAtAnimationSpeed
 |name                             = Vaal Bone Warden
 |size                             = 3