--- Monster:Metadata/Monsters/LeagueSanctum/Fodder/Harpy
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/LeagueSanctum/Fodder/Harpy
 |monster_type_id                  = SanctumHarpy
-|tags                             = avian_beast, beast, caster, cleaving_weapon, demon, fast_movement, fire_affinity, flesh_armour, humanoid, is_unarmed, melee, not_str, physical_affinity, red_blood
+|tags                             = avian_beast, beast, caster, cleaving_weapon, demon, fast_movement, fire_affinity, flesh_armour, humanoid, is_unarmed, melee, not_str, physical_affinity, red_blood, sanctum_monster
 |skill_ids                        = MeleeAtAnimationSpeedSanctum, MMSSanctumHarpy
 |name                             = Infernal Harpy
 |size                             = 3