--- Monster:Metadata/Monsters/TemplarJudge/AbsolutionTemplarJudge
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/TemplarJudge/AbsolutionTemplarJudge
 |monster_type_id                  = AbsolutionTemplarJudge
-|tags                             = bludgeoning_weapon, caster, fast_movement, flesh_armour, has_one_hand_mace, has_one_handed_melee, human, melee, not_dex, physical_affinity, red_blood, templar
+|tags                             = bludgeoning_weapon, caster, fast_movement, flesh_armour, has_one_hand_mace, has_one_handed_melee, human, melee, not_dex, physical_affinity, templar
 |skill_ids                        = Melee, AbsolutionMinion, AbsolutionMinionEmpowered
 |name                             = Sentinel of Absolution
 |size                             = 3
