--- Monster:Metadata/Monsters/Spirit/TormentedMiner4
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/Spirit/TormentedMiner4
 |monster_type_id                  = TormentedSpiritStrong
 |mod_ids                          = TormentSpiritPhasing, MonsterNoMapDrops
-|tags                             = ghost, ghost_armour, ghost_blood, is_unarmed, medium_height, medium_movement, non_attacking
+|tags                             = ghost, ghost_armour, ghost_blood, is_unarmed, medium_height, medium_movement, non_attacking, undead
 |skill_ids                        = Haunt, WalkEmergeTormentedSpirit
 |name                             = Tormented Miner
 |size                             = 3