--- Monster:Metadata/Monsters/LeagueAncestral/AncestralAhuana
+++ Export
@@ -0,0 +1,17 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueAncestral/AncestralAhuana
+|monster_type_id                  = AncestralAhuana
+|mod_ids                          = AncestralTrialMonsterAccuracyRamako, AncestralTrialChieftainBalance
+|tags                             = human, humanoid, medium_movement, not_int, not_str, red_blood
+|skill_ids                        = MPWAhuanaIceShot, MonsterSnipeAhuana, AhuanaRainOfArrows, GAAhuanaIceShotExplosion, GAAhuanaIceShotExplosionBig, GAAhuanaJumpShot, AhuanaEvadeLeft, AhuanaEvadeRight, GTAhuanaArrowRain, GAAhuanaArrowRain, MPWAhuanaFakeArrow
+|name                             = Ahuana
+|size                             = 3
+|minimum_attack_distance          = 5
+|maximum_attack_distance          = 70
+|model_size_multiplier            = 1.0
+|experience_multiplier            = 1.0
+|damage_multiplier                = 1.45
+|health_multiplier                = 0.9
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.5
+}}