--- Monster:Metadata/Monsters/Skeletons/ConstructBossInvasion
+++ Export
@@ -3,7 +3,7 @@
 |monster_type_id                  = ConstructMeleeBossInvasion
 |mod_ids                          = MonsterCastsElementalHitText, MonsterInvasionBoss, MonsterNoMapDrops, IsInvasionBoss
 |endgame_mod_ids                  = MonsterInvasionBossMap
-|tags                             = bone_armour, bones, construct, has_two_hand_sword, has_two_handed_melee, invasion_boss, large_height, melee, not_dex, not_int, physical_affinity, slashing_weapon, very_slow_movement
+|tags                             = bone_armour, bones, has_two_hand_sword, has_two_handed_melee, invasion_boss, large_height, melee, not_dex, not_int, physical_affinity, skeleton, slashing_weapon, undead, very_slow_movement
 |skill_ids                        = Melee, ElementalHitSkeletonKnightInvasion
 |name                             = The Spiritless
 |size                             = 3