--- Monster:Metadata/Monsters/AtlasBosses/TheShaperBossStandalone
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/AtlasBosses/TheShaperBossStandalone
 |monster_type_id                  = TheShaperBoss
-|mod_ids                          = MonsterMapBoss, CannotBeSlowedBelowValueBosses, StunRepeatImmunityMapBossSuper, PinnacleAtlasBoss, MonsterDroppedItemsInfluenceShaper
+|mod_ids                          = MonsterMapBoss, CannotBeSlowedBelowValueBosses, StunRepeatImmunityMapBossSuper, PinnacleAtlasBoss, MonsterDroppedItemsInfluenceShaperBoss
 |tags                             = caster, cold_affinity, human, humanoid, is_unarmed, large_height, light_armour, melee, physical_affinity, ranged, red_blood, standalone_map_boss, very_slow_movement
 |skill_ids                        = ShaperMelee, AtlasBossCelestialBeam, AtlasBossAcceleratingProjectiles, AtlasBossFlickerSlam, ShaperSummonFragment, SSMStandaloneShaperAnomaly
 |name                             = The Shaper
