--- Monster:Metadata/Monsters/LeagueAncestral/AncestralAkoyaMinion
+++ Export
@@ -0,0 +1,16 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueAncestral/AncestralAkoyaMinion
+|monster_type_id                  = AncestralAkoyaMinion
+|tags                             = human, humanoid, medium_movement, not_int, red_blood
+|skill_ids                        = MeleeAtAnimationSpeedUnique, GAAkoyaLacerate, GAAkoyaCleave, GAAkoyaLeapSlam, AkoyaLacerate, MPSAkoyaJadeSpray, DTTAkoyaLeapSlam, GSAkoyaRockExplosion
+|name                             = Spirit of Akoya
+|size                             = 3
+|minimum_attack_distance          = 5
+|maximum_attack_distance          = 20
+|model_size_multiplier            = 1.0
+|experience_multiplier            = 1.0
+|damage_multiplier                = 1.2
+|health_multiplier                = 1.1
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.0
+}}