--- Monster:Metadata/Monsters/Daemon/TormentDaemonStatue
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/Daemon/TormentDaemonStatue
 |monster_type_id                  = DaemonUseSkillOnce
 |tags                             = extra_extra_small_height, ghost_armour, is_daemon, is_unarmed, non_attacking, very_slow_movement
-|skill_ids                        = TormentEmbezzlerSummonStatue
+|skill_ids                        = TormentEmbezzlerSummonStatue, TormentEmbezzlerSummonStatuePlayer
 |name                             = Torment Daemon
 |minimum_attack_distance          = 6
 |maximum_attack_distance          = 6
