--- Monster:Metadata/Monsters/LeagueSanctum/Fodder/GargoyleMedium
+++ Export
@@ -2,12 +2,12 @@
 |metadata_id                      = Metadata/Monsters/LeagueSanctum/Fodder/GargoyleMedium
 |monster_type_id                  = SanctumGargoyleMedium
 |mod_ids                          = MonsterAttackBlock30Bypass20
-|tags                             = construct, mud_blood, not_dex, not_int, slow_movement, stone_construct
+|tags                             = construct, mud_blood, not_dex, not_int, sanctum_monster, slow_movement, stone_construct
 |skill_ids                        = MeleeAtAnimationSpeedUniqueSanctum, DTTSanctumMediumGargoyleLeapSlam, GASanctumMediumGargoyleLeapImpact, GASanctumMediumGargoyleLeapImpactCascade
 |name                             = Igneous Gargoyle
 |size                             = 3
 |minimum_attack_distance          = 6
-|maximum_attack_distance          = 18
+|maximum_attack_distance          = 14
 |model_size_multiplier            = 1.0
 |experience_multiplier            = 1.5
 |damage_multiplier                = 0.75
