--- Monster:Metadata/Monsters/LeagueSanctum/Guards/Guard10
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/LeagueSanctum/Guards/Guard10
 |monster_type_id                  = SanctumGuard10
 |mod_ids                          = MonsterTagSanctumGuard, MonsterSanctumGuardVolatileOnDeath, MonsterSanctumGuardSummonGargoyle
-|tags                             = fast_movement, human, humanoid, not_int, ranged, red_blood, sanctum_guard
+|tags                             = fast_movement, human, humanoid, not_int, ranged, red_blood, sanctum_guard, sanctum_monster
 |skill_ids                        = MeleeLightningBowSanctum, SanctumGuardConeArrowLightning, SanctumGuardDeceleratingProjectile, SanctumGuardDeceleratingProjectileImpact, SSMSanctumLightningVortex
 |name                             = Azure Assassin
 |size                             = 3
