--- Monster:Metadata/Monsters/LeagueDelve/VaalBloodEncounter/SkeletonMeleeKnightElementalSword
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/LeagueDelve/VaalBloodEncounter/SkeletonMeleeKnightElementalSword
 |monster_type_id                  = ConstructMeleeArmouredDelve
 |mod_ids                          = CannotBeUsedAsMinion
-|tags                             = bone_armour, bones, construct, delve_monster, has_two_hand_sword, has_two_handed_melee, medium_height, not_dex, not_int, physical_affinity, slashing_weapon, very_slow_movement
+|tags                             = bone_armour, bones, delve_monster, has_two_hand_sword, has_two_handed_melee, medium_height, not_dex, not_int, physical_affinity, skeleton, slashing_weapon, undead, very_slow_movement
 |skill_ids                        = ElementalHitSkeletonKnight
 |name                             = Colossal Blood Fallen
 |size                             = 3