--- Monster:Metadata/Monsters/LeagueCrucible/Chaos/SkeletonBeast
+++ Export
@@ -0,0 +1,17 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueCrucible/Chaos/SkeletonBeast
+|monster_type_id                  = CrucibleSkeletonBeast
+|mod_ids                          = MonsterNecromancerRaisable
+|tags                             = animal_claw_weapon, beast, bone_armour, bones, has_one_hand_sword, has_one_handed_melee, large_model, mammal_beast, melee, not_dex, not_int, physical_affinity, ursine_beast, very_slow_movement
+|skill_ids                        = MeleeAtAnimationSpeedChaos, EAACrucibleBeastSlam, GTCrucibleBeastCascadeLeft, GTCrucibleBeastCascadeRight, GACrucibleBeastCascadeImpact, GACrucibleBeastCascadeFinalImpact, GTCrucibleBeastCascadeFinalImpact
+|name                             = Forged Skeletal Bear
+|size                             = 4
+|minimum_attack_distance          = 6
+|maximum_attack_distance          = 15
+|model_size_multiplier            = 1.2
+|experience_multiplier            = 2.0
+|damage_multiplier                = 3.0
+|health_multiplier                = 3.6
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.095
+}}