--- Monster:Metadata/Monsters/Totems/AbyssCrystalPillarMap4
+++ Export
@@ -0,0 +1,17 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/Totems/AbyssCrystalPillarMap4
+|monster_type_id                  = MonsterTotemAbyss
+|mod_ids                          = VariableEmergeSpeedAbyss, MonsterImplicitCannotBeStunned1, ImmuneToKnockback
+|tags                             = abyssal, caster, construct, ghost_blood, hard_armour, immobile, is_unarmed, large_height, not_dex, not_int, physical_affinity, puncturing_weapon
+|skill_ids                        = AbyssPillarMortar
+|name                             = Stygian Spire
+|size                             = 2
+|minimum_attack_distance          = 6
+|maximum_attack_distance          = 6
+|model_size_multiplier            = 2.0
+|experience_multiplier            = 3.0
+|damage_multiplier                = 0.99
+|health_multiplier                = 18.75
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.5
+}}