--- Monster:Metadata/Monsters/Skeletons/SkeletonMeleeKnightFireSword
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/Skeletons/SkeletonMeleeKnightFireSword
 |monster_type_id                  = ConstructMeleeFire
 |mod_ids                          = MonsterElementalSkeletonFire
-|tags                             = bone_armour, bones, cleaving_weapon, construct, has_two_hand_sword, has_two_handed_melee, medium_height, melee, not_dex, not_int, physical_affinity, very_slow_movement
+|tags                             = bone_armour, bones, cleaving_weapon, construct, has_two_hand_sword, has_two_handed_melee, medium_height, melee, not_dex, not_int, physical_affinity, skeleton, undead, very_slow_movement
 |skill_ids                        = MeleeAtAnimationSpeed
 |name                             = Vaal Flame Guardian
 |size                             = 3