--- Monster:Metadata/Monsters/Skeletons/SkeletonMeleeKnightElementalLargeSword2Hellscape
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/Skeletons/SkeletonMeleeKnightElementalLargeSword2Hellscape
 |monster_type_id                  = ConstructLarge2
-|tags                             = bone_armour, bones, construct, has_one_hand_sword, has_one_handed_melee, not_dex, physical_affinity, slashing_weapon, slow_movement
+|tags                             = bone_armour, bones, has_one_hand_sword, has_one_handed_melee, not_dex, physical_affinity, skeleton, slashing_weapon, slow_movement, undead
 |skill_ids                        = ElementalStrikeVaalColossus
 |name                             = Vaal Gargantuan
 |size                             = 3