--- Monster:Metadata/Monsters/MinerLarge/MinerLargeMapBoss
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/MinerLarge/MinerLargeMapBoss
 |monster_type_id                  = MinerLargeMapBoss
 |mod_ids                          = MonsterMapBoss, StunRepeatImmunityMapBoss_
-|tags                             = bludgeoning_weapon, has_one_hand_mace, has_one_handed_melee, large_height, large_model, melee, mud_blood, non_unique_map_boss, not_dex, not_int, physical_affinity, ranged, stone_armour, undead, very_slow_movement
+|tags                             = bludgeoning_weapon, has_one_hand_mace, has_one_handed_melee, large_height, large_model, melee, mud_blood, non_unique_map_boss, not_dex, not_int, physical_affinity, ranged, stone_armour, undead, unholy_trio_boss_map, very_slow_movement
 |skill_ids                        = MeleeAtAnimationSpeed, GroundSlamMiner, MinerThrowSpawn, MinerThrowSpawnMapBoss
 |name                             = Champion of the Hollows
 |size                             = 3
