--- Monster:Metadata/Monsters/LeagueSanctum/Guards/Guard8~
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/LeagueSanctum/Guards/Guard8_
 |monster_type_id                  = SanctumGuard8
 |mod_ids                          = MonsterTagSanctumGuard, MonsterSanctumGuardVolatileOnDeath, MonsterSanctumGuardSummonGargoyle, MonsterAttackBlock30Bypass20
-|tags                             = fast_movement, human, humanoid, not_dex, not_int, ranged, red_blood, sanctum_guard
+|tags                             = fast_movement, human, humanoid, not_dex, not_int, ranged, red_blood, sanctum_guard, sanctum_monster
 |skill_ids                        = MeleeAtAnimationSpeedSanctumGuard, SanctumMilitaryShieldCharge, GASanctumGuardBounce, GASanctumFireThrust, EASSanctumGuardProjCleave, GASanctumFireGuardCleave, MPWSanctumFireGuardCleave
 |name                             = Infernal Sentinel
 |size                             = 3