--- Monster:Metadata/Monsters/KitavaBoss/Kitava1Map
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/KitavaBoss/Kitava1Map
 |monster_type_id                  = KitavaMap
-|mod_ids                          = MonsterMapBoss, MonsterImplicitCannotBeStunned1, MonsterCannotBeSlowed, StunRepeatImmunityMapBoss_
+|mod_ids                          = MonsterMapBoss, MonsterCannotBeSlowed, StunRepeatImmunityMapBoss_, MonsterImplicitCannotBeStunned1
 |tags                             = bludgeoning_weapon, demon, extra_extra_large_height, fire_affinity, flesh_armour, is_unarmed, kitava_map_boss, non_unique_map_boss, physical_affinity, red_blood, very_slow_movement
 |skill_ids                        = KitavaSlamMiddleMap, KitavaSlamRightMap, KitavaSlamLeftMap, KitavaSlamMiddleRightMap, KitavaSlamMiddleLeftMap, KitavaSwipeRight, KitavaSwipeLeft, KitavaThrowDebrisLeftMap, KitavaThrowDebrisRightMap, KitavaSlamRightNoConeMap, KitavaSlamLeftNoConeMap
 |name                             = Lord of the Grey