--- Monster:Metadata/Monsters/LeagueAncestral/AncestralKaruiShieldMace
+++ Export
@@ -0,0 +1,17 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueAncestral/AncestralKaruiShieldMace
+|monster_type_id                  = AncestralShieldWaller
+|mod_ids                          = MonsterAttackBlock30Bypass15, AncestralTrialMonsterOverwhelmRongokurai, CannotBeSlowedBelowValueBosses, AncestralTrialMonsterBasicFleeAndDamage
+|tags                             = human, humanoid, karui, medium_movement, not_dex, not_int, red_blood
+|skill_ids                        = MeleeAtAnimationSpeed, SSMAncestralWall, GTAncestralWallOffence, GTAncestralWallDefence, GTAncestralWallFlankLeft, GTAncestralWallFlankRight, EASAncestralWallFlank
+|name                             = Fieldmaster
+|size                             = 3
+|minimum_attack_distance          = 5
+|maximum_attack_distance          = 13
+|model_size_multiplier            = 1.0
+|experience_multiplier            = 1.0
+|damage_multiplier                = 1.0
+|health_multiplier                = 1.25
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.5
+}}