--- Monster:Metadata/Monsters/Skeletons/SkeletonMeleeKnightColdSword2
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/Skeletons/SkeletonMeleeKnightColdSword2
 |monster_type_id                  = ConstructMeleeCold
 |mod_ids                          = MonsterElementalSkeletonCold
-|tags                             = bone_armour, bones, construct, has_two_hand_sword, has_two_handed_melee, medium_height, melee, not_dex, not_int, physical_affinity, slashing_weapon, very_slow_movement
+|tags                             = bone_armour, bones, construct, has_two_hand_sword, has_two_handed_melee, medium_height, melee, not_dex, not_int, physical_affinity, skeleton, slashing_weapon, undead, very_slow_movement
 |skill_ids                        = MeleeAtAnimationSpeed
 |name                             = Vaal Frost Guardian
 |size                             = 3
