--- Monster:Metadata/Monsters/LeagueCrucible/Fire/DriftingEye
+++ Export
@@ -0,0 +1,16 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueCrucible/Fire/DriftingEye
+|monster_type_id                  = CrucibleDriftingEye
+|tags                             = animal_claw_weapon, beast, caster, fast_movement, flesh_armour, is_unarmed, lightning_affinity, medium_height, melee, not_str, physical_affinity
+|skill_ids                        = MeleeAtAnimationSpeedFire, EASCrucibleSquidNoodleBeams, TBCrucibleSquidNoodleBeam, GSCrucibleSquidNoodleBeamImpact
+|name                             = Forged Drifting Eye
+|size                             = 3
+|minimum_attack_distance          = 4
+|maximum_attack_distance          = 12
+|model_size_multiplier            = 1.2
+|experience_multiplier            = 1.2
+|damage_multiplier                = 1.56
+|health_multiplier                = 1.44
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.005
+}}