--- Monster:Metadata/Monsters/Wb/WbFireLeader3
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/Wb/WbFireLeader3
 |monster_type_id                  = WbFireLeader
-|mod_ids                          = MonsterMapBoss, StunRepeatImmunityMapBoss_
+|mod_ids                          = MonsterMapBoss, StunRepeatImmunityMapBoss_, RedbladeWarlord
 |tags                             = bludgeoning_weapon, caster, fire_affinity, has_staff, has_two_handed_melee, human, humanoid, medium_movement, medium_small_height, melee, non_unique_map_boss, physical_affinity, plate_armour, ranged, red_blood, summoner, wb_leader
 |skill_ids                        = Melee, WbFireMortar, WbFireMortarNova, WbVaalFlameCall, WbSummonFlameGolem, WbSummonFireElementals, WbFireLeader1Wait
 |name                             = Leli Goya, Daughter of Ash
