--- Monster:Metadata/Monsters/Skeletons/LabyrinthLieutenantSkeletonRangedMinion
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/Skeletons/LabyrinthLieutenantSkeletonRangedMinion
 |monster_type_id                  = LabyrinthLieutenantSkeletonRangedMinion
 |mod_ids                          = MonsterNecromancerRaisable, MonsterFiresLightningArrowsText
-|tags                             = bone_armour, bones, has_bow, lightning_affinity, medium_height, not_int, not_str, physical_affinity, puncturing_weapon, ranged, very_slow_movement
+|tags                             = bone_armour, bones, has_bow, lightning_affinity, medium_height, not_int, not_str, physical_affinity, puncturing_weapon, ranged, skeleton, undead, very_slow_movement
 |skill_ids                        = Melee, MonsterLightningArrow
 |name                             = Charged Engineer
 |size                             = 3