--- Monster:Metadata/Monsters/Skeletons/LabyrinthLieutenantSkeletonMeleeMinion
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/Skeletons/LabyrinthLieutenantSkeletonMeleeMinion
 |monster_type_id                  = LabyrinthLieutenantSkeletonMeleeMinion
 |mod_ids                          = MonsterElementalSkeletonCold, MonsterNecromancerRaisable
-|tags                             = bludgeoning_weapon, bone_armour, bones, has_one_hand_mace, has_one_handed_melee, medium_height, melee, physical_affinity, very_slow_movement
+|tags                             = bludgeoning_weapon, bone_armour, bones, has_one_hand_mace, has_one_handed_melee, medium_height, melee, physical_affinity, skeleton, undead, very_slow_movement
 |skill_ids                        = Melee
 |name                             = Frozen Builder
 |size                             = 3
