--- Monster:Metadata/Monsters/Skeletons/SkeletonCasterKnightFireMachinarium
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/Skeletons/SkeletonCasterKnightFireMachinarium
 |monster_type_id                  = ConstructCasterFire
-|tags                             = bone_armour, bones, caster, construct, fire_affinity, is_unarmed, medium_height, not_dex, ranged, slashing_weapon, very_slow_movement
+|tags                             = bone_armour, bones, caster, construct, fire_affinity, is_unarmed, medium_height, not_dex, ranged, slashing_weapon, undead, very_slow_movement
 |skill_ids                        = SkeletonProjectileFire
 |name                             = Vaal Arsonist
 |size                             = 3
