--- Monster:Metadata/Monsters/LeagueSanctum/Fodder/Jaguar
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/LeagueSanctum/Fodder/Jaguar
 |monster_type_id                  = JaguarDemonSanctum
 |mod_ids                          = MonsterPhysicalAddedAsColdSkeletonMaps
-|tags                             = beast, ghost, ghost_armour, ghost_blood, is_unarmed, medium_movement, melee, not_str, physical_affinity, undead
+|tags                             = beast, ghost, ghost_armour, ghost_blood, is_unarmed, medium_movement, melee, not_str, physical_affinity, sanctum_monster, undead
 |skill_ids                        = MeleeAtAnimationSpeedSanctum, JaguarDemonDashSanctum
 |name                             = Stalking Geist
 |size                             = 3
