--- Monster:Metadata/Monsters/SkeletonCannon/SkeletonCannon1
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/SkeletonCannon/SkeletonCannon1
 |monster_type_id                  = SkeletonCannon
 |mod_ids                          = MonsterNecromancerRaisable
-|tags                             = bludgeoning_weapon, bone_armour, caster, extra_small_height, is_unarmed, medium_movement, not_int, physical_affinity, ranged, skeleton
+|tags                             = bludgeoning_weapon, bone_armour, caster, extra_small_height, is_unarmed, medium_movement, not_int, physical_affinity, ranged, skeleton, undead
 |skill_ids                        = SkeletonCannonMortar, SkeletonCannonBoneMortar, SkeletonCannonBoneNova
 |name                             = Bone Husk
 |size                             = 4
