--- Monster:Metadata/Monsters/LeagueAncestral/AncestralUtulaMinion~
+++ Export
@@ -0,0 +1,16 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueAncestral/AncestralUtulaMinion_
+|monster_type_id                  = AncestralUtulaMinion
+|tags                             = fast_movement, human, humanoid, not_int, red_blood
+|skill_ids                        = MeleeAtAnimationSpeedUnique, UtulaTableCharge, UtulaFireSwing, GAEchoingQuake, GAUtulaChargeEnd, MPSUtulaFireball, GAUtulaXSlamLacerate, CGEUtulaDegen
+|name                             = Spirit of Utula
+|size                             = 3
+|minimum_attack_distance          = 5
+|maximum_attack_distance          = 19
+|model_size_multiplier            = 1.0
+|experience_multiplier            = 1.0
+|damage_multiplier                = 1.2
+|health_multiplier                = 1.2
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.0
+}}