--- Monster:Metadata/Monsters/Skeletons/ConstructRangedBossSideAreaInvasion
+++ Export
@@ -3,7 +3,7 @@
 |monster_type_id                  = ConstructRangedArmouredBossSideArea
 |mod_ids                          = MonsterSideAreaBoss, MonsterInvasionMercilessDrops1, MonsterNoMapDrops, IsInvasionBoss
 |endgame_mod_ids                  = MonsterInvasionBossMap
-|tags                             = bone_armour, bones, caster, construct, fire_affinity, has_one_hand_mace, has_one_handed_melee, invasion_boss, large_height, not_int, physical_affinity, puncturing_weapon, summoner, very_slow_movement
+|tags                             = bone_armour, bones, caster, fire_affinity, has_one_hand_mace, has_one_handed_melee, invasion_boss, large_height, not_int, physical_affinity, puncturing_weapon, skeleton, summoner, undead, very_slow_movement
 |skill_ids                        = DesecrateSideAreaBoss, RaiseZombieSideAreaBoss, FleshOfferingSideAreaBoss, MonsterPunishmentSideArea, MonsterDetonateDeadSideAreaBoss
 |name                             = Kutec, Vaal Fleshsmith
 |size                             = 3
