--- Monster:Metadata/Monsters/LeagueAncestral/AncestralTuatara~
+++ Export
@@ -0,0 +1,17 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueAncestral/AncestralTuatara_
+|monster_type_id                  = AncestralTuatara
+|mod_ids                          = AncestralTrialMonsterBasicFleeAndDamage
+|tags                             = beast, bludgeoning_weapon, has_one_hand_mace, has_one_handed_melee, light_armour, medium_height, medium_movement, melee, not_int, physical_affinity, red_blood, reptile_beast
+|skill_ids                        = MeleeAtAnimationSpeed
+|name                             = Tuatara
+|size                             = 3
+|minimum_attack_distance          = 5
+|maximum_attack_distance          = 14
+|model_size_multiplier            = 1.0
+|experience_multiplier            = 1.0
+|damage_multiplier                = 1.0
+|health_multiplier                = 1.0
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.5
+}}