--- Monster:Metadata/Monsters/LeagueSynthesis/SynthesisSoulstealerCorpseWheelSpectre~
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/LeagueSynthesis/SynthesisSoulstealerCorpseWheelSpectre_
 |monster_type_id                  = SynthesisCorpseWheel
 |mod_ids                          = MonsterTagSynthesised
-|tags                             = bludgeoning_weapon, bone_armour, is_unarmed, medium_movement, melee, no_strider_mod, not_dex, not_str, physical_affinity, skeleton
+|tags                             = bludgeoning_weapon, bone_armour, is_unarmed, medium_movement, melee, no_strider_mod, not_dex, not_str, physical_affinity, skeleton, undead
 |skill_ids                        = MeleeAtAnimationSpeed
 |name                             = Razor Halo
 |size                             = 3