--- Monster:Metadata/Monsters/LeagueSanctum/Fodder/Lithomancer
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/LeagueSanctum/Fodder/Lithomancer
 |monster_type_id                  = SanctumLithomancer
-|tags                             = caster, human, humanoid, is_unarmed, light_armour, medium_height, mud_blood, not_dex, not_str, physical_affinity, ranged, very_fast_movement
+|tags                             = caster, human, humanoid, is_unarmed, light_armour, medium_height, mud_blood, not_dex, not_str, physical_affinity, ranged, sanctum_monster, very_fast_movement
 |skill_ids                        = SanctumLithomancerProjectile, ProtoVaalMapBossVolatileSummonSanctumFodder
 |name                             = Decaying Lithomancer
 |size                             = 3
