--- Monster:Metadata/Monsters/Spirit/TormentedPoisoner
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/Spirit/TormentedPoisoner
 |monster_type_id                  = TormentedSpiritChaosResist
 |mod_ids                          = TormentSpiritDropsIncrease2, TormentSpiritPhasing
-|tags                             = ghost, ghost_armour, ghost_blood, is_unarmed, medium_height, medium_movement, non_attacking
+|tags                             = ghost, ghost_armour, ghost_blood, is_unarmed, medium_height, medium_movement, non_attacking, undead
 |skill_ids                        = Haunt, TormentSpiritPause, WalkEmergeTormentedSpirit
 |name                             = Tormented Alchemist
 |size                             = 3
