--- Monster:Metadata/Monsters/GargoyleBoss/GargoyleBoss
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/GargoyleBoss/GargoyleBoss
 |monster_type_id                  = Minotaur
-|mod_ids                          = CannotBeSlowedBelowValueBosses, MonsterImplicitCannotBeStunned1, MonsterMinotaurBoss, ImmuneToLavaDamage, LabyrinthDegenTrapImmunity1, LabyrinthSpikeTrapDamageTaken2, LabyrinthArrowTrapDamageTaken2_, LabyrinthGuillotineTrapDamageTaken2, StunRepeatImmunityActBoss
+|mod_ids                          = CannotBeSlowedBelowValueBosses, MonsterMinotaurBoss, ImmuneToLavaDamage, LabyrinthDegenTrapImmunity1, LabyrinthSpikeTrapDamageTaken2, LabyrinthArrowTrapDamageTaken2_, LabyrinthGuillotineTrapDamageTaken2, StunRepeatImmunityActBoss
 |tags                             = argus_labyrinth, beast, demon, extra_large_height, flesh_armour, gladiator, has_one_hand_mace, has_one_handed_melee, large_model, medium_movement, melee, not_dex, not_int, physical_affinity, red_blood, slashing_weapon
 |skill_ids                        = Melee, MinotaurSunder, MinotaurEnrage
 |name                             = Argus