--- Monster:Metadata/Monsters/LeagueAncestral/AncestralJadeCutter
+++ Export
@@ -0,0 +1,17 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueAncestral/AncestralJadeCutter
+|monster_type_id                  = AncestralJadeCutter
+|mod_ids                          = AncestralTrialMonsterBlockReductionTukohama, AncestralTrialMonsterIntermediateFleeAndDamage
+|tags                             = fast_movement, human, humanoid, karui, not_dex, not_int, red_blood
+|skill_ids                        = MeleeAtAnimationSpeed, EASAncestralDestroyTotemBuffs, GAAncestralLeagueJadeChop
+|name                             = Jade Shaman
+|size                             = 3
+|minimum_attack_distance          = 5
+|maximum_attack_distance          = 13
+|model_size_multiplier            = 1.0
+|experience_multiplier            = 1.0
+|damage_multiplier                = 2.0
+|health_multiplier                = 2.75
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.5
+}}