--- Monster:Metadata/Monsters/LeagueCrucible/Lightning/CrabCoconut
+++ Export
@@ -0,0 +1,16 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueCrucible/Lightning/CrabCoconut
+|monster_type_id                  = CrucibleCoconutCrab
+|tags                             = beast, crustacean_beast, not_dex, not_int, slow_movement
+|skill_ids                        = MeleeAtAnimationSpeedLightning
+|name                             = Forged Crab
+|size                             = 2
+|minimum_attack_distance          = 6
+|maximum_attack_distance          = 8
+|model_size_multiplier            = 1.0
+|experience_multiplier            = 0.6
+|damage_multiplier                = 0.9
+|health_multiplier                = 0.72
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.5
+}}