--- Monster:Metadata/Monsters/Skeletons/SkeletonCasterKnightColdMachinariumNormal
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/Skeletons/SkeletonCasterKnightColdMachinariumNormal
 |monster_type_id                  = ConstructCasterCold
-|tags                             = bone_armour, bones, caster, cold_affinity, construct, is_unarmed, medium_height, not_str, ranged, slashing_weapon, very_slow_movement
+|tags                             = bone_armour, bones, caster, cold_affinity, construct, is_unarmed, medium_height, not_str, ranged, slashing_weapon, undead, very_slow_movement
 |skill_ids                        = SkeletonProjectileCold
 |name                             = Vaal Frostguard
 |size                             = 3
