--- Monster:Metadata/Monsters/Skeletons/SkeletonMeleeKnightElementalLargeSwordBoss
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/Skeletons/SkeletonMeleeKnightElementalLargeSwordBoss
 |monster_type_id                  = ConstructLargeBossType
 |mod_ids                          = MonsterTwoAdditionalTotems, MonsterKillTrapsMinesTotemsOnDeath
-|tags                             = bone_armour, bones, construct, has_one_hand_sword, has_one_handed_melee, large_height, melee, not_dex, not_int, physical_affinity, ranged, slashing_weapon, slow_movement
+|tags                             = bone_armour, bones, has_one_hand_sword, has_one_handed_melee, large_height, melee, not_dex, not_int, physical_affinity, ranged, skeleton, slashing_weapon, slow_movement, undead
 |skill_ids                        = MeleeAtAnimationSpeedUnique, ThrownWeaponSkeletonColossalBoss, SummonSkeletonColossalAnimatedWeapons, SkeletalMeleeKnightWitherTotem
 |name                             = Blightblade
 |size                             = 3