--- Monster:Metadata/Monsters/Parasite/ParasiteEmergeOnDeathUnique
+++ Export
@@ -4,7 +4,7 @@
 |mod_ids                          = ParasitePoisonOnHit
 |tags                             = beast, caster, chaos_affinity, extra_extra_small_height, fast_movement, flesh_armour, has_one_hand_mace, has_one_handed_melee, insect, insect_blood, melee, physical_affinity, ranged
 |skill_ids                        = Melee, ParasitePoisonProjectile, SpellNovaParasiteUnique, ParasiteLeapSlam
-|name                             = The Encephelophage
+|name                             = The Encephalophage
 |size                             = 2
 |minimum_attack_distance          = 6
 |maximum_attack_distance          = 8