--- Monster:Metadata/Monsters/LeagueBlight/Generic/SkeletonCannonBoss
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/LeagueBlight/Generic/SkeletonCannonBoss
 |monster_type_id                  = BlightSkeletonCannonBoss
 |mod_ids                          = MonsterBlightMapBoss, MonsterCannotBeKnockedBack
-|tags                             = blight_boss, blight_monster, bludgeoning_weapon, bone_armour, caster, extra_small_height, is_unarmed, melee, not_int, physical_affinity, ranged, skeleton, very_slow_movement
+|tags                             = blight_boss, blight_monster, bludgeoning_weapon, bone_armour, caster, extra_small_height, is_unarmed, melee, not_int, physical_affinity, ranged, skeleton, undead, very_slow_movement
 |skill_ids                        = Melee, BlightSkeletonCannonMortar, BlightSkeletonCannonBoneMortar, SkeletonCannonBoneNova
 |name                             = Sketh, Spore Amalgamation
 |size                             = 1
