--- Monster:Metadata/Monster/CageSpider/CageSpider2Hellscape
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monster/CageSpider/CageSpider2Hellscape
 |monster_type_id                  = CageSpider2
-|tags                             = caster, flesh_armour, is_unarmed, melee, not_dex, not_int, physical_affinity, ranged, skeleton, slashing_weapon, slow_movement
+|tags                             = caster, flesh_armour, is_unarmed, melee, not_dex, not_int, physical_affinity, ranged, skeleton, slashing_weapon, slow_movement, undead
 |skill_ids                        = Melee, CageSpiderCycloneTriggerSandstorms, CageSpiderSandSpark
 |name                             = Sandworn Slaves
 |size                             = 3
