--- Monster:Metadata/Monsters/LeagueAncestral/AncestralKuniKuni
+++ Export
@@ -0,0 +1,17 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueAncestral/AncestralKuniKuni
+|monster_type_id                  = AncestralKuniKuni
+|mod_ids                          = AmcestralTrialMonsterLeechResistanceKitava, AncestralTrialMonsterBasicFleeAndDamage
+|tags                             = beast, bludgeoning_weapon, has_one_hand_mace, has_one_handed_melee, light_armour, mammal_beast, medium_height, medium_movement, melee, not_int, physical_affinity, red_blood
+|skill_ids                        = MeleeAtAnimationSpeed, GSKuniKuniCorpseChannel
+|name                             = Consuming Kunekune
+|size                             = 3
+|minimum_attack_distance          = 5
+|maximum_attack_distance          = 12
+|model_size_multiplier            = 1.0
+|experience_multiplier            = 1.0
+|damage_multiplier                = 1.15
+|health_multiplier                = 1.15
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.5
+}}