--- Monster:Metadata/Monsters/MassSkeleton/MassSkeletonMapBoss5
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/MassSkeleton/MassSkeletonMapBoss5
 |monster_type_id                  = MassSkeletonMapBoss5
-|tags                             = bones, fire_affinity, has_bow, large_model, medium_movement, medium_small_height, non_unique_map_boss, not_dex, not_int, physical_affinity, raises_dead, ranged, skeleton, slashing_weapon, stone_armour, undead
+|tags                             = bones, fire_affinity, has_bow, large_model, matryoshka_boss, medium_movement, medium_small_height, non_unique_map_boss, not_dex, not_int, physical_affinity, raises_dead, ranged, skeleton, slashing_weapon, stone_armour, undead
 |skill_ids                        = MeleeAtAnimationSpeedUnique, SkeletonMassBowProjectile, MassReviveUndeadMassSkeletonBoss, MassSkeletonTornadoShotMapBoss, MassSkeletonBossBlastRain
 |name                             = Unravelling Horror
 |size                             = 3
