--- Monster:Metadata/Monsters/Wb/WbFireLeader2
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/Wb/WbFireLeader2
 |monster_type_id                  = WbFireLeader
-|mod_ids                          = MonsterElementalSkeletonFire, MonsterIncreasedTrapsAllowed, MonsterImplicitTrapsInvulnerable3, MonsterKillTrapsMinesTotemsOnDeath, BaseTrapDuration1, MonsterMapBoss, StunRepeatImmunityMapBoss_
+|mod_ids                          = MonsterElementalSkeletonFire, MonsterIncreasedTrapsAllowed, MonsterImplicitTrapsInvulnerable3, MonsterKillTrapsMinesTotemsOnDeath, BaseTrapDuration1, MonsterMapBoss, StunRepeatImmunityMapBoss_, RedbladeWarlord
 |tags                             = caster, cleaving_weapon, fire_affinity, has_dual_wield_melee, has_one_hand_sword, has_one_handed_melee, human, humanoid, medium_height, medium_movement, melee, non_unique_map_boss, physical_affinity, plate_armour, red_blood, wb_leader
 |skill_ids                        = Melee, WbReaveFire, WbFlickerStrikeFire, WbFireEnrage, WbFireElementalCone, WbFireLeader1Wait, WbFireEnrageEnraged, WbFireUpheaval, WbFireNovaTrap
 |name                             = El'Abin, Bloodeater
