--- Monster:Metadata/Monsters/LeagueSanctum/Fodder/Revenant
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/LeagueSanctum/Fodder/Revenant
 |monster_type_id                  = SanctumRevenant
 |mod_ids                          = MonsterPhysicalAddedAsColdSkeletonMaps
-|tags                             = caster, fire_affinity, ghost, ghost_armour, ghost_blood, is_unarmed, melee, not_dex, not_str, physical_affinity, ranged, undead, very_slow_movement
+|tags                             = caster, fire_affinity, ghost, ghost_armour, ghost_blood, is_unarmed, melee, not_dex, not_str, physical_affinity, ranged, sanctum_monster, undead, very_slow_movement
 |skill_ids                        = GSSanctumSerpentWave, MPSPhantasmBasicSanctum, EAASanctumRevenantTeleport
 |name                             = Azure Wraith
 |size                             = 3