--- Monster:Metadata/Monsters/LeagueAncestral/AncestralKaruiSpear~
+++ Export
@@ -0,0 +1,17 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueAncestral/AncestralKaruiSpear_
+|monster_type_id                  = AncestralKaruiSpear
+|mod_ids                          = AmcestralTrialMonsterLeechResistanceKitava, AncestralTrialMonsterChampionFleeAndDamage
+|tags                             = bludgeoning_weapon, fast_movement, has_one_hand_mace, has_one_handed_melee, human, humanoid, karui, light_armour, medium_height, melee, not_int, physical_affinity, red_blood
+|skill_ids                        = MeleeAtAnimationSpeed, AncestralSpearTeleport
+|name                             = Bloodbound Warrior
+|size                             = 3
+|minimum_attack_distance          = 5
+|maximum_attack_distance          = 13
+|model_size_multiplier            = 1.0
+|experience_multiplier            = 1.0
+|damage_multiplier                = 2.0
+|health_multiplier                = 5.0
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.5
+}}