--- Monster:Metadata/Monsters/LeagueCrucible/Fire/MineBat
+++ Export
@@ -0,0 +1,16 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueCrucible/Fire/MineBat
+|monster_type_id                  = CrucibleMineBat
+|tags                             = beast, caster, flesh_armour, flying, is_unarmed, lightning_affinity, mammal_beast, medium_height, melee, not_int, not_str, physical_affinity, slow_movement
+|skill_ids                        = MeleeAtAnimationSpeedFire, GSCrucibleBatFlameBreath, EASCrucibleBatFlameBreath, DTTCrucibleBatDash
+|name                             = Forged Bat
+|size                             = 3
+|minimum_attack_distance          = 6
+|maximum_attack_distance          = 9
+|model_size_multiplier            = 0.9
+|experience_multiplier            = 0.8
+|damage_multiplier                = 1.26
+|health_multiplier                = 0.96
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.5
+}}