--- Monster:Metadata/Monsters/SkeletonCannon/SkeletonCannonAbyssDelve
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/SkeletonCannon/SkeletonCannonAbyssDelve
 |monster_type_id                  = SkeletonCannonAbyss
 |mod_ids                          = SpiritChargeTier2, VariableEmergeSpeedAbyss, CannotBeUsedAsMinion, BreachReducedBeyondPortalChance
-|tags                             = abyssal, bludgeoning_weapon, bone_armour, caster, delve_monster, extra_small_height, has_claw, has_one_handed_melee, medium_movement, not_int, physical_affinity, ranged, skeleton
+|tags                             = abyssal, bludgeoning_weapon, bone_armour, caster, delve_monster, extra_small_height, has_claw, has_one_handed_melee, medium_movement, not_int, physical_affinity, ranged, skeleton, undead
 |skill_ids                        = SummonSkeletonAbyssSkeletonCannonDelve, SkeletonCannonProjectile, AbyssSkeletonCannonProjectile, AbyssBoneHuskSpellNova
 |name                             = Pit Maw
 |size                             = 4