--- Monster:Metadata/Monsters/LeagueAncestral/AncestralSpearRope
+++ Export
@@ -0,0 +1,17 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueAncestral/AncestralSpearRope
+|monster_type_id                  = AncestralSpearRope
+|mod_ids                          = AncestralTrialMonsterStealsFlaskChargesTasalio, AncestralTrialMonsterBasicFleeAndDamage
+|tags                             = fast_movement, human, humanoid, karui, not_int, not_str, red_blood
+|skill_ids                        = MeleeAtAnimationSpeed, MPWAncestralSpearThrowRope, AncestralSpearEnsnare
+|name                             = Spearfisher
+|size                             = 3
+|minimum_attack_distance          = 5
+|maximum_attack_distance          = 16
+|model_size_multiplier            = 1.0
+|experience_multiplier            = 1.0
+|damage_multiplier                = 1.25
+|health_multiplier                = 1.0
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.5
+}}