--- Monster:Metadata/Monsters/LeagueSanctum/Fodder/GargoyleMedium
+++ Export
@@ -7,7 +7,7 @@
 |name                             = Igneous Gargoyle
 |size                             = 3
 |minimum_attack_distance          = 6
-|maximum_attack_distance          = 18
+|maximum_attack_distance          = 14
 |model_size_multiplier            = 1.0
 |experience_multiplier            = 1.5
 |damage_multiplier                = 0.75
