--- Monster:Metadata/Monsters/BreachBosses/BreachBossFireWild
+++ Export
@@ -1,6 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/BreachBosses/BreachBossFireWild
 |monster_type_id                  = BreachBossFireWild
+|mod_ids                          = MonsterNoDrops
 |part1_mod_ids                    = MonsterPhysicalAddedAsFireBreach
 |part2_mod_ids                    = MonsterPhysicalAddedAsFireBreach2
 |endgame_mod_ids                  = MonsterPhysicalAddedAsFireBreachMaps_