--- Monster:Metadata/Monsters/LeagueSanctum/Guards/Guard5
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/LeagueSanctum/Guards/Guard5
 |monster_type_id                  = SanctumGuard5
 |mod_ids                          = MonsterTagSanctumGuard, MonsterSanctumGuardVolatileOnDeath, MonsterSanctumGuardSummonGargoyle
-|tags                             = caster, fire_affinity, has_one_hand_sword, has_one_handed_melee, human, humanoid, light_armour, melee, not_int, not_str, physical_affinity, red_blood, sanctum_guard, slashing_weapon, very_fast_movement
+|tags                             = caster, fire_affinity, has_one_hand_sword, has_one_handed_melee, human, humanoid, light_armour, melee, not_int, not_str, physical_affinity, red_blood, sanctum_guard, sanctum_monster, slashing_weapon, very_fast_movement
 |skill_ids                        = GASanctumDanaiSweepCleave, EAASanctumDanaiSweep, SanctumDanaiTeleportStrike, SanctumDanaiFlickerAttack
 |name                             = Deadly Bladedancer
 |size                             = 3
