--- Monster:Metadata/Monsters/LeagueAncestral/AncestralIkiahoMinion
+++ Export
@@ -0,0 +1,16 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueAncestral/AncestralIkiahoMinion
+|monster_type_id                  = AncestralIkiahoMinion
+|tags                             = human, humanoid, medium_movement, not_dex, not_str, red_blood
+|skill_ids                        = AncestralIkiahoArcticBreath, AncestralIkiahoIceStorm, EASIkiahoIceStorm
+|name                             = Spirit of Ikiaho
+|size                             = 3
+|minimum_attack_distance          = 5
+|maximum_attack_distance          = 10
+|model_size_multiplier            = 1.0
+|experience_multiplier            = 1.0
+|damage_multiplier                = 1.2
+|health_multiplier                = 1.0
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.0
+}}