--- Monster:Metadata/Monsters/LeagueCrucible/Lightning/SeaWitch
+++ Export
@@ -0,0 +1,16 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueCrucible/Lightning/SeaWitch
+|monster_type_id                  = CrucibleSeaWitch
+|tags                             = caster, cold_affinity, demon, flesh_armour, has_dagger, has_one_handed_melee, medium_height, melee, not_dex, not_str, physical_affinity, slow_movement
+|skill_ids                        = MeleeAtAnimationSpeedLightning, GSCrucibleSeaWitchLightningWave
+|name                             = Forged Static Witch
+|size                             = 3
+|minimum_attack_distance          = 6
+|maximum_attack_distance          = 9
+|model_size_multiplier            = 1.15
+|experience_multiplier            = 1.0
+|damage_multiplier                = 1.27
+|health_multiplier                = 1.4
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.005
+}}