--- Monster:Metadata/Monsters/LeagueCrucible/Chaos/LargeSkeleton
+++ Export
@@ -0,0 +1,17 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueCrucible/Chaos/LargeSkeleton
+|monster_type_id                  = CrucibleLargeSkeleton
+|mod_ids                          = MonsterNecromancerRaisable
+|tags                             = bone_armour, bones, cleaving_weapon, has_one_hand_mace, has_one_handed_melee, melee, not_dex, not_int, physical_affinity, skeleton, slow_movement, undead
+|skill_ids                        = MeleeAtAnimationSpeedChaos, CrucibleSkeletonBonePrison
+|name                             = Forged Skeleton
+|size                             = 3
+|minimum_attack_distance          = 6
+|maximum_attack_distance          = 16
+|model_size_multiplier            = 1.0
+|experience_multiplier            = 1.4
+|damage_multiplier                = 1.4
+|health_multiplier                = 2.52
+|critical_strike_chance           = 5.0
+|attack_speed                     = 2.25
+}}