--- Monster:Metadata/Monsters/Atziri/AtziriSpear
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/Atziri/AtziriSpear
 |monster_type_id                  = AtziriMirrored
-|mod_ids                          = MonsterAtziriMapBoss, MonsterNoDropsOrExperience, MonsterImplicitCannotBeStunned1, MonsterMapBoss, StunRepeatImmunityMapBoss_
+|mod_ids                          = MonsterAtziriMapBoss, MonsterNoDropsOrExperience, MonsterMapBoss, StunRepeatImmunityMapBoss_
 |tags                             = extra_large_height, flesh_armour, humanoid, immobile, is_unarmed, large_model, physical_affinity, red_blood, slashing_weapon
 |skill_ids                        = AtziriSpearThrow
 |name                             = Atziri, Queen of the Vaal