--- Monster:Metadata/Monsters/KitavaBoss/Kitava2MapStandalone
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/KitavaBoss/Kitava2MapStandalone
 |monster_type_id                  = KitavaMapStandalone
-|mod_ids                          = MonsterMapBoss, MonsterImplicitCannotBeStunned1, MonsterCannotBeSlowed, StunRepeatImmunityMapBoss_
+|mod_ids                          = MonsterMapBoss, MonsterCannotBeSlowed, StunRepeatImmunityMapBoss_, MonsterImplicitCannotBeStunned1
 |tags                             = bludgeoning_weapon, caster, demon, flesh_armour, immobile, is_unarmed, kitava_height, physical_affinity, red_blood, standalone_map_boss
 |skill_ids                        = KitavaSlamMiddleMapStandalone, KitavaSlamRightMapStandalone, KitavaSlamLeftMapStandalone, KitavaSlamMiddleRightMapStandalone, KitavaSlamMiddleLeftMapStandalone, KitavaSwipeRightStandalone, KitavaSwipeLeftStandalone, KitavaDevouringDarknessRightMapStandalone, KitavaDevouringDarknessLeftMapStandalone, KitavaXBlastFinalStandalone, KitavaSlamRightNoConeMapStandalone, KitavaSlamLeftNoConeMapStandalone, FaceLastTarget
 |name                             = Lord of the Grey
