--- Monster:Metadata/Monsters/LeagueCrucible/Physical/Minions/PlagueInsect
+++ Export
@@ -0,0 +1,17 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueCrucible/Physical/Minions/PlagueInsect
+|monster_type_id                  = VolatileCore
+|mod_ids                          = MonsterNoDropsOrExperience, CannotGainAfflictedMods
+|tags                             = construct, flesh_armour, insect_blood, is_unarmed, melee, physical_affinity, uses_suicide_explode, very_slow_movement
+|skill_ids                        = CrucibleInsectBloodExplosion
+|name                             = Volatile Core
+|size                             = 3
+|minimum_attack_distance          = 6
+|maximum_attack_distance          = 6
+|model_size_multiplier            = 1.0
+|experience_multiplier            = 1.0
+|damage_multiplier                = 1.0
+|health_multiplier                = 1.0
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.5
+}}