--- Monster:Metadata/Monsters/Skeletons/SkeletonCasterLightningSparkAtziri
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/Skeletons/SkeletonCasterLightningSparkAtziri
 |monster_type_id                  = ConstructCasterLightningAtziri
 |mod_ids                          = MonsterCastsSparkText
-|tags                             = bone_armour, bones, caster, construct, is_unarmed, lightning_affinity, medium_height, not_dex, not_str, ranged, slashing_weapon, very_slow_movement
+|tags                             = bone_armour, bones, caster, construct, is_unarmed, lightning_affinity, medium_height, not_dex, not_str, ranged, slashing_weapon, undead, very_slow_movement
 |skill_ids                        = SkeletonProjectileLightning, MonsterSpark
 |name                             = Vaal Thunderguard
 |size                             = 3