--- Monster:Metadata/Monsters/LeagueCrucible/Chaos/SkeletonArcher
+++ Export
@@ -0,0 +1,17 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueCrucible/Chaos/SkeletonArcher
+|monster_type_id                  = CrucibleSkeletonArcher
+|mod_ids                          = MonsterNecromancerRaisable
+|tags                             = bone_armour, bones, cold_affinity, has_bow, medium_movement, not_int, not_str, physical_affinity, puncturing_weapon, ranged, skeleton, undead
+|skill_ids                        = MeleeAtAnimationSpeedChaos, CrucibleSkeletonCausticArrow
+|name                             = Forged Skeleton
+|size                             = 3
+|minimum_attack_distance          = 6
+|maximum_attack_distance          = 40
+|model_size_multiplier            = 0.8
+|experience_multiplier            = 0.8
+|damage_multiplier                = 1.24
+|health_multiplier                = 0.96
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.5
+}}