--- Monster:Metadata/Monsters/LeagueSanctum/Fodder/Reliquarian
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/LeagueSanctum/Fodder/Reliquarian
 |monster_type_id                  = SanctumReliquarian
-|tags                             = animal_claw_weapon, caster, demon, fast_movement, fire_affinity, flesh_armour, human, humanoid, is_unarmed, medium_height, melee, not_dex, not_str, physical_affinity, ranged, red_blood
+|tags                             = animal_claw_weapon, caster, demon, fast_movement, fire_affinity, flesh_armour, human, humanoid, is_unarmed, medium_height, melee, not_dex, not_str, physical_affinity, ranged, red_blood, sanctum_monster
 |skill_ids                        = ReliquaryMonsterFirestormSanctum, GTSanctumReliquarianFireball, GSSanctumHolyFireballImpact, MPSSanctumReliquarianFireball
 |name                             = Stitched Reliquarian
 |size                             = 3