--- Monster:Metadata/Monsters/LeagueLake/CorpseWheelBossLake
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/LeagueLake/CorpseWheelBossLake
 |monster_type_id                  = CorpseWheelBossLake
-|tags                             = bone_armour, bones, caster, has_one_hand_sword, has_one_handed_melee, medium_height, melee, no_strider_mod, not_dex, physical_affinity, skeleton, slashing_weapon, very_fast_movement
+|tags                             = bone_armour, bones, caster, has_one_hand_sword, has_one_handed_melee, medium_height, melee, no_strider_mod, not_dex, physical_affinity, skeleton, slashing_weapon, undead, very_fast_movement
 |skill_ids                        = CorpseWheelChargeBoss, MeleeAtAnimationSpeedUnique, CorpseWheelBoneNova, CorpseWheelBladeVortex
 |name                             = Effigy of Whirling Undeath
 |size                             = 3