--- Monster:Metadata/Monsters/LeagueAncestral/AncestralBuffHorn~~
+++ Export
@@ -0,0 +1,17 @@
+{{Monster
+|metadata_id                      = Metadata/Monsters/LeagueAncestral/AncestralBuffHorn__
+|monster_type_id                  = AncestralBuffHorn
+|mod_ids                          = AncestralTrialMonsterBlockReductionTukohama, AncestralTrialMonsterBasicFleeAndDamage
+|tags                             = human, humanoid, karui, not_int, red_blood, very_slow_movement
+|skill_ids                        = MeleeAtAnimationSpeed, EGAncestralJadeHornBlow
+|name                             = Warcaller
+|size                             = 3
+|minimum_attack_distance          = 5
+|maximum_attack_distance          = 27
+|model_size_multiplier            = 1.0
+|experience_multiplier            = 1.0
+|damage_multiplier                = 1.05
+|health_multiplier                = 1.35
+|critical_strike_chance           = 5.0
+|attack_speed                     = 1.5
+}}