--- Monster:Metadata/Monsters/LeagueSanctum/Guards/Guard3
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/LeagueSanctum/Guards/Guard3
 |monster_type_id                  = SanctumGuard3
 |mod_ids                          = MonsterTagSanctumGuard, MonsterSanctumGuardVolatileOnDeath, MonsterSanctumGuardSummonGargoyle
-|tags                             = caster, cold_affinity, fast_movement, has_bow, human, humanoid, light_armour, medium_height, not_int, not_str, puncturing_weapon, ranged, red_blood, sanctum_guard
+|tags                             = caster, cold_affinity, fast_movement, has_bow, human, humanoid, light_armour, medium_height, not_int, not_str, puncturing_weapon, ranged, red_blood, sanctum_guard, sanctum_monster
 |skill_ids                        = SanctumRinSnipe, EAASanctumRinSnipe, EDASanctumTriggeredRainOfArrows, GTSanctumTriggeredRainOfArrows
 |name                             = Entombing Warden
 |size                             = 3