--- Monster:Metadata/Monsters/AtlasBosses/MinotaurBossElder
+++ Export
@@ -1,7 +1,7 @@
 {{Monster
 |metadata_id                      = Metadata/Monsters/AtlasBosses/MinotaurBossElder
 |monster_type_id                  = AtlasBossMinotaur
-|mod_ids                          = MonsterMapBoss, CannotBeSlowedBelowValueBosses, MonsterImplicitCannotBeStunned1, StunRepeatImmunityMapBossSuper
+|mod_ids                          = MonsterMapBoss, CannotBeSlowedBelowValueBosses, StunRepeatImmunityMapBossSuper
 |tags                             = bludgeoning_weapon, elder_map_boss, fast_movement, flesh_armour, has_two_hand_mace, has_two_handed_melee, human, humanoid, large_height, non_attacking, not_dex, not_int, red_blood
 |name                             = Guardian of the Minotaur
 |size                             = 4
