--- Monster:Metadata/Monsters/ZombieDismember/Blackguard/ProphecyBlackguardZombieBoss
+++ Export
@@ -2,7 +2,7 @@
 |metadata_id                      = Metadata/Monsters/ZombieDismember/Blackguard/ProphecyBlackguardZombieBoss
 |monster_type_id                  = ProphecyBlackguardZombie
 |mod_ids                          = ProphecyAdditionalRareItemsBoss
-|tags                             = caster, cold_affinity, flesh_armour, is_unarmed, medium_height, melee, not_dex, not_int, physical_affinity, red_blood, very_slow_movement
+|tags                             = caster, cold_affinity, flesh_armour, is_unarmed, medium_height, melee, not_dex, not_int, physical_affinity, red_blood, undead, very_slow_movement, zombie
 |skill_ids                        = MeleeAtAnimationSpeed, ProphecySummonZombiesBoss, ProphecyColdSnap, ProphecyBlackguardGrab
 |name                             = Captain of the Damned
 |size                             = 3