--- Wiki
+++ Export
@@ -0,0 +1,13 @@
+{{Passive skill
+|id                                  = AscendancyElementalist8
+|int_id                              = 51391
+|name                                = Shaper of Desolation
+|reminder_text                       = (These are consecutive)
+(Each Conflux means all damage causes that Ailment)
+|ascendancy_class                    = Elementalist
+|is_notable                          = True
+|icon                                = ElementalEnlightenment (Elementalist)
+|stat1_id                            = elementalist_gain_shaper_of_desolation_every_10_seconds
+|stat1_value                         = 1
+|stat_text                           = Every 14 seconds:<br />Gain Chilling Conflux for 4 seconds<br />Gain Shocking Conflux for 4 seconds<br />Gain Igniting Conflux for 4 seconds<br />Gain Chilling, Shocking and Igniting Conflux for 2 seconds
+}}