--- Passive Skill:Atlas~heist~4342~~
+++ Export
@@ -6,5 +6,5 @@
 |icon                                = HeistNotable1 (AtlasTrees)
 |stat1_id                            = map_heist_contract_item_drop_chance_+%
 |stat1_value                         = 8
-|stat_text                           = 8% increased quantity of Heist Contracts found in your Maps
+|stat_text                           = 8% increased Heist Contracts found in your Maps
 }}