--- Passive Skill:AscendancySpecialEldritch1
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = AscendancySpecialEldritch1
 |int_id                              = 27602
 |name                                = Nine Lives
-|ascendancy_class                    = Witch
+|ascendancy_class                    = Necromancer
 |icon                                = 
 |stat1_id                            = damage_taken_goes_to_life_mana_es_over_4_seconds_%
 |stat1_value                         = 25
