--- Wiki
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = phase_acrobatics1184
 |int_id                              = 14914
 |name                                = Phase Acrobatics
-|flavour_text                        = With enough practice, you can feel the spells
+|flavour_text                        = With enough practice, you can feel the spells
 coming before they're even cast.
 |is_keystone                         = True
 |icon                                = KeystonePhaseAcrobatics
