--- Passive Skill:Lava~lash1
+++ Export
@@ -9,5 +9,5 @@
 |stat2_id                            = reduce_enemy_fire_resistance_with_weapons_%
 |stat2_value                         = 8
 |stat_text                           = 30% increased [[Fire Damage]] with [[Attack]] [[Skill|Skills]]<br>Damage with Weapons Penetrates 8% [[Fire Resistance]]
-|connections                         = mastery_fire122,mastery_fire122,weapon_fire_damage1713
+|connections                         = mastery_fire122,stun_threshold2871,mastery_fire122,stun_threshold2871,weapon_fire_damage1712
 }}