--- Passive Skill:Eldritch~battery1142
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = eldritch_battery1142
 |int_id                              = 56075
 |name                                = Eldritch Battery
-|flavour_text                        = What need have I for defence when my enemies
+|flavour_text                        = What need have I for defence when my enemies
 are reduced to ash and splinters?
 |is_keystone                         = True
 |icon                                = KeystoneEldritchBattery
