--- Passive Skill:AscendancySpecialEldritch5
+++ Export
@@ -8,5 +8,5 @@
 |stat1_value                         = 20
 |stat2_id                            = ranger_hidden_ascendancy_non_damaging_elemental_ailment_effect_+%_final
 |stat2_value                         = 100
-|stat_text                           = Non-Damaging Elemental Ailments you inflict spread to nearby enemies in a radius of 20<br>Non-Damaging Elemental Ailments you inflict have 100% more Effect
+|stat_text                           = Non-Damaging Elemental Ailments you inflict spread to other enemies within 2 metres<br>Non-Damaging Elemental Ailments you inflict have 100% more Effect
 }}