--- Passive Skill:Weapon~artistry620
+++ Export
@@ -10,6 +10,6 @@
 |stat2_value                         = 8
 |stat3_id                            = melee_range_+
 |stat3_value                         = 2
-|stat_text                           = +4% Chance to [[Block]] [[Attack Damage]] while [[Dual Wield|Dual Wielding]] or holding a [[Shield]]<br>8% increased [[Attack Speed]]<br>+2 to Melee Strike Range
+|stat_text                           = +4% Chance to [[Block]] [[Attack Damage]] while [[Dual Wield|Dual Wielding]] or holding a [[Shield]]<br>8% increased [[Attack Speed]]<br>+0.2 metres to Melee Strike Range
 |connections                         = one_handed_attack_speed1027,one_handed_damage1050
 }}