--- Passive Skill:AscendancySpecialEldritch6
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = AscendancySpecialEldritch6
 |int_id                              = 48999
 |name                                = Soul Drinker
-|ascendancy_class                    = Trickster
+|ascendancy_class                    = Shadow
 |is_notable                          = True
 |icon                                = SoulThiefTrickster (Trickster)
 |stat1_id                            = energy_shield_leech_does_not_stop_on_full_energy_shield
