--- Passive Skill:AscendancyChieftan13
+++ Export
@@ -9,5 +9,5 @@
 |stat2_id                            = fire_damage_+%
 |stat2_value                         = 10
 |stat_text                           = Regenerate 0.5% of [[Life]] per second<br>10% increased [[Fire Damage]]
-|connections                         = AscendancyChieftan9,AscendancyChieftan14,AscendancyChieftan9
+|connections                         = AscendancyChieftan8,AscendancyChieftainStart,AscendancyChieftan8
 }}