--- Wiki
+++ Export
@@ -0,0 +1,11 @@
+{{Passive skill
+|id                                  = mortal_conviction_keystone2348
+|int_id                              = 9403
+|name                                = Mortal Conviction
+|flavour_text                        = Lonely is the man who seeks power without cost.
+|is_keystone                         = True
+|icon                                = mortalconviction
+|stat1_id                            = old_grandmaster_free_aura_from_old_mortal_conviction
+|stat1_value                         = 1
+|stat_text                           = 
+}}