--- Passive Skill:AscendancyGuardian12
+++ Export
@@ -5,10 +5,8 @@
 |ascendancy_class                    = Guardian
 |is_notable                          = True
 |icon                                = UnwaveringFaith (Guardian)
-|stat1_id                            = auras_grant_additional_physical_damage_reduction_%_to_you_and_your_allies
-|stat1_value                         = 1
-|stat2_id                            = guardian_auras_grant_life_regeneration_per_minute_%
-|stat2_value                         = 12
-|stat_text                           = Auras from your [[Skill|Skills]] grant +1% [[Physical Damage]] Reduction to you and Allies<br>Auras from your [[Skill|Skills]] grant 0.2% of [[Life]] Regenerated per second to<br />you and Allies
+|stat1_id                            = auras_grant_life_mana_es_recovery_rate_+%_to_you_and_your_allies
+|stat1_value                         = 5
+|stat_text                           = Auras from your [[Skill|Skills]] grant 5% increased Recovery Rate of Life,<br />Mana and [[Energy Shield]] to you and Allies
 |connections                         = AscendancyGuardian9
 }}