--- Passive Skill:Atlas~keystone~smallnodes~1
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = atlas_keystone_smallnodes_1
 |int_id                              = 40658
 |name                                = Wandering Path
-|flavour_text                        = Embrace the journey, Nomad, for there is no destination.
+|flavour_text                        = Embrace the journey, Nomad, for there is no destination.
 This is the prime inescapable truth of eternity.
 |is_keystone                         = True
 |is_atlas_passive                    = True
