--- Passive Skill:Atlas~keystone~ritual~variable~shop~prices
+++ Export
@@ -2,8 +2,8 @@
 |id                                  = atlas_keystone_ritual_variable_shop_prices
 |int_id                              = 1733
 |name                                = Arbitrary Tenets
-|flavour_text                        = They are unforgivable sinners and fools...
-We pray each Sacrato. They pray each Kaso.
+|flavour_text                        = They are unforgivable sinners and fools...
+We pray each Sacrato. They pray each Kaso.
 They are absolutely mad!
 |is_keystone                         = True
 |is_atlas_passive                    = True
