--- Passive Skill:Affliction~notable~strike~leader
+++ Export
@@ -10,5 +10,5 @@
 |stat2_value                         = 30
 |stat3_id                            = melee_range_+_while_wielding_shield
 |stat3_value                         = 2
-|stat_text                           = +2% Chance to [[Block]] [[Attack Damage]]<br>[[Attack]] [[Skill|Skills]] deal 30% increased Damage while holding a [[Shield]]<br>+2 to Melee Strike Range while Holding a [[Shield]]
+|stat_text                           = +2% Chance to [[Block]] [[Attack Damage]]<br>[[Attack]] [[Skill|Skills]] deal 30% increased Damage while holding a [[Shield]]<br>+0.2 metres to Melee Strike Range while Holding a [[Shield]]
 }}