--- Wiki
+++ Export
@@ -0,0 +1,13 @@
+{{Passive skill
+|id                                  = atlas_elder_shaper_2_5
+|int_id                              = 60492
+|name                                = Gaze into the Abyss
+|is_notable                          = True
+|is_atlas_passive                    = True
+|icon                                = ElderShaperNotable2 (AtlasTrees)
+|stat1_id                            = map_elder_boss_watchers_eye_drop_additional_chance_%
+|stat1_value                         = 10
+|stat2_id                            = map_shaper_boss_drops_X_additional_shaper_items
+|stat2_value                         = 3
+|stat_text                           = The Elder has +10% chance to drop a Watcher's Eye<br>The Shaper drops 3 additional Shaper Items
+}}