--- Passive Skill:AscendancyChieftan11
+++ Export
@@ -5,12 +5,10 @@
 |ascendancy_class                    = Chieftain
 |is_notable                          = True
 |icon                                = TukomahaWarsHerald (Chieftain)
-|stat1_id                            = totem_damage_leeched_as_life_to_you_permyriad
-|stat1_value                         = 200
-|stat2_id                            = ancestor_totem_parent_activation_range_+%
-|stat2_value                         = 100
-|stat3_id                            = ancestor_totem_buff_effect_+%
-|stat3_value                         = 100
-|stat_text                           = 2% of Damage dealt by your [[Totem|Totems]] is Leeched to you as [[Life]]<br>Ancestor [[Totem|Totems]] have 100% increased Activation range<br>100% increased Effect of Buffs your Ancestor [[Totem|Totems]] grant while Active
-|connections                         = AscendancyChieftan6
+|stat1_id                            = chieftain_body_armour_supported_by_level_x_ancestral_call
+|stat1_value                         = 30
+|stat2_id                            = chieftain_body_armour_supported_by_level_x_fist_of_war
+|stat2_value                         = 20
+|stat_text                           = [[Skill|Skills]] from Equipped Body [[Armour]] are Supported by [[Level]] 30 Ancestral Call<br>[[Skill|Skills]] from Equipped Body [[Armour]] are Supported by [[Level]] 20 Fist of War
+|connections                         = AscendancyChieftan1
 }}