--- Passive Skill:Atlas~keystone~delirium~infinite~fog
+++ Export
@@ -0,0 +1,16 @@
+{{Passive skill
+|id                                  = atlas_keystone_delirium_infinite_fog
+|int_id                              = 44043
+|name                                = Unending Nightmare
+|flavour_text                        = What if it's all just in your head?
+|is_keystone                         = True
+|is_atlas_passive                    = True
+|icon                                = UnendingNightmare (AtlasTrees)
+|stat1_id                            = map_delirium_fog_never_dissipates
+|stat1_value                         = 1
+|stat2_id                            = map_delirium_no_splinters
+|stat2_value                         = 1
+|stat3_id                            = map_delirium_no_delirium_orbs
+|stat3_value                         = 1
+|stat_text                           = Delirium Fog in your Maps never dissipates<br>Simulacrum Splinters cannot be found in your Maps<br>Delirium Orbs cannot be found in your Maps
+}}