--- Passive Skill:Atlas~keystone~notables~1~
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = atlas_keystone_notables_1_
 |int_id                              = 38059
 |name                                = Grand Design
-|flavour_text                        = Ignore everything but your obsession, Nomad.
+|flavour_text                        = Ignore everything but your obsession, Nomad.
 A mortal life is too short a season to entertain distractions.
 |is_keystone                         = True
 |is_atlas_passive                    = True
