--- Wiki
+++ Export
@@ -0,0 +1,10 @@
+{{Passive skill
+|id                                  = affliction_notable_whispers_of_death
+|int_id                              = 57104
+|name                                = Evil Eye
+|is_notable                          = True
+|icon                                = CurseEffectNotable
+|stat1_id                            = enemies_damage_taken_+%_while_cursed
+|stat1_value                         = 6
+|stat_text                           = Enemies you [[Curse]] take 6% increased Damage
+}}