--- Passive Skill:Atlas~wormhole~1~2
+++ Export
@@ -7,5 +7,7 @@
 |icon                                = MasteryBlank
 |stat1_id                            = dummy_display_stat_wormhole_pair_1
 |stat1_value                         = 1
+|stat2_id                            = atlas_number_of_gateways_allocated
+|stat2_value                         = 1
 |stat_text                           = Connects to Mortal Gateway
 }}