--- Passive Skill:Atlas~keystone~legion~alert~system
+++ Export
@@ -0,0 +1,12 @@
+{{Passive skill
+|id                                  = atlas_keystone_legion_alert_system
+|int_id                              = 48004
+|name                                = Timeless Conflict
+|flavour_text                        = Haste is irrelevant in the face of eternity.
+|is_keystone                         = True
+|is_atlas_passive                    = True
+|icon                                = RuptureMeter (AtlasTrees)
+|stat1_id                            = map_legion_uses_alert_instead_of_timer
+|stat1_value                         = 1
+|stat_text                           = Legion Encounters in your Maps have no Timer<br />Breaking out Monsters and Chests that are in stasis progressively causes a Schism<br />Legion Encounters in your Maps begin once the Schism has occurred
+}}