--- Passive Skill:Staff~accu~notable2569
+++ Export
@@ -8,6 +8,6 @@
 |stat1_value                         = 2
 |stat2_id                            = melee_skills_area_of_effect_+%
 |stat2_value                         = 12
-|stat_text                           = +2 to Melee Strike Range<br>Melee [[Skill|Skills]] have 12% increased Area of Effect
+|stat_text                           = +0.2 metres to Melee Strike Range<br>Melee [[Skill|Skills]] have 12% increased Area of Effect
 |connections                         = one_handed_damage1235,two_handed_weapon_damage603,attack_speed1889,two_handed_weapon_damage603
 }}