--- Passive Skill:AscendancySpecialEldritch5
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = AscendancySpecialEldritch5
 |int_id                              = 18054
 |name                                = Fury of Nature
-|ascendancy_class                    = Raider
+|ascendancy_class                    = Ranger
 |icon                                = 
 |stat1_id                            = ascendancy_non_damaging_elemental_ailment_proliferation_radius
 |stat1_value                         = 20
