--- Passive Skill:Atlas~keystone~harvest~upgraded~plots
+++ Export
@@ -0,0 +1,14 @@
+{{Passive skill
+|id                                  = atlas_keystone_harvest_upgraded_plots
+|int_id                              = 51310
+|name                                = Crop Rotation
+|flavour_text                        = A good sword is worthless without a hundred plowshares behind it.
+|is_keystone                         = True
+|is_atlas_passive                    = True
+|icon                                = CropRotation (AtlasTrees)
+|stat1_id                            = map_harvest_seeds_are_tier_1
+|stat1_value                         = 1
+|stat2_id                            = map_harvest_seeds_of_other_colours_have_chance_to_upgrade_on_completing_plot
+|stat2_value                         = 1
+|stat_text                           = Harvest Crops in your Maps contain only Tier 1 Plants<br>Harvesting Crops in your Maps has a chance to upgrade the Tier of Plants of different colours
+}}