--- Passive Skill:Impaler~keystone2780
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = impaler_keystone2780
 |int_id                              = 57257
 |name                                = The Impaler
-|flavour_text                        = Punish your enemies with the excessive cruelty of a tyrant,
+|flavour_text                        = Punish your enemies with the excessive cruelty of a tyrant,
 then move on without a second thought.
 |is_keystone                         = True
 |icon                                = ImpaleKeystone
