--- Passive Skill:AscendancySpecialEldritch3
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = AscendancySpecialEldritch3
 |int_id                              = 52435
 |name                                = Indomitable Resolve
-|ascendancy_class                    = Berserker
+|ascendancy_class                    = Marauder
 |icon                                = 
 |stat1_id                            = marauder_hidden_ascendancy_damage_taken_+%_final
 |stat1_value                         = -25
