--- Wiki
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = ghost_dance_keystone2852_
 |int_id                              = 35255
 |name                                = Ghost Dance
-|flavour_text                        = Surround yourself with the dead,
+|flavour_text                        = Surround yourself with the dead,
 so that you may avoid joining them.
 |is_keystone                         = True
 |icon                                = AcrobaticWillpower (Trickster)
