--- Passive Skill:AscendancyJuggernaut13
+++ Export
@@ -5,7 +5,7 @@
 |buff_id                             = juggernaut_phys_damage_prevented_as_life_regen
 |ascendancy_class                    = Juggernaut
 |is_notable                          = True
-|icon                                = Untiring (Juggernaut)
+|icon                                = Unbreakable (Juggernaut)
 |stat1_id                            = gain_%_total_phys_damage_prevented_in_the_past_10_sec_as_life_regen_per_sec
 |stat1_value                         = 15
 |stat2_id                            = life_regeneration_rate_+%
