--- Passive Skill:Attack~ignite~chance2867
+++ Export
@@ -0,0 +1,10 @@
+{{Passive skill
+|id                                  = attack_ignite_chance2867
+|int_id                              = 64695
+|name                                = Attack Ignite Chance
+|icon                                = firedamagestr
+|stat1_id                            = attack_ignite_chance_%
+|stat1_value                         = 15
+|stat_text                           = [[Attack|Attacks]] have 15% chance to [[Ignite]]
+|connections                         = attack_ignite_notable2876,attack_ignite_chance2868,attack_ignite_notable2876
+}}