--- Passive Skill:Atlas~keystone~beyond~bosses~cannot~spawn
+++ Export
@@ -2,8 +2,8 @@
 |id                                  = atlas_keystone_beyond_bosses_cannot_spawn
 |int_id                              = 58043
 |name                                = Endless Tide
-|flavour_text                        = You fight on, muscles burning, limbs leaden,
-hoping for that singular moment of victory that
+|flavour_text                        = You fight on, muscles burning, limbs leaden,
+hoping for that singular moment of victory that
 might finally bring relief... but it never comes.
 |is_keystone                         = True
 |is_atlas_passive                    = True
