--- Passive Skill:Atlas~keystone~torment~spirits~possess~players
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = atlas_keystone_torment_spirits_possess_players
 |int_id                              = 62654
 |name                                = Speaker of the Dead
-|flavour_text                        = For nine hundred years, they have had no voice, no hope.
+|flavour_text                        = For nine hundred years, they have had no voice, no hope.
 Finally, they have found a kindred soul... in you.
 |is_keystone                         = True
 |is_atlas_passive                    = True
