--- Passive Skill:Strikes~range2732
+++ Export
@@ -7,6 +7,6 @@
 |stat1_value                         = 10
 |stat2_id                            = melee_range_+
 |stat2_value                         = 1
-|stat_text                           = 10% increased Melee Damage<br>+1 to Melee Strike Range
+|stat_text                           = 10% increased Melee Damage<br>+0.1 metres to Melee Strike Range
 |connections                         = strikes_range2733,strikes_damage2731_,strikes_range2733
 }}