--- Passive Skill:Verdant~precision~keystone2858
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = verdant_precision_keystone2858
 |int_id                              = 63620
 |name                                = Precise Technique
-|flavour_text                        = Fear not the warrior that has practiced ten thousand techniques once.
+|flavour_text                        = Fear not the warrior that has practiced ten thousand techniques once.
 Fear the warrior that has practiced one technique ten thousand times.
 |is_keystone                         = True
 |icon                                = PreciseTechnique
