--- Wiki
+++ Export
@@ -0,0 +1,15 @@
+{{Passive skill
+|id                                  = AscendancyGuardian10
+|int_id                              = 50356
+|name                                = Prayer of Glory
+|ascendancy_class                    = Guardian
+|is_notable                          = True
+|icon                                = PrayerOfGlory (Guardian)
+|stat1_id                            = warcries_are_instant
+|stat1_value                         = 1
+|stat2_id                            = guardian_warcry_grant_attack_cast_and_movement_speed_to_you_and_nearby_allies_+%
+|stat2_value                         = 15
+|stat3_id                            = warcries_cost_no_mana
+|stat3_value                         = 1
+|stat_text                           = Using Warcries is Instant<br>If you've Warcried Recently, you and nearby allies have 15% increased Attack, Cast and [[Movement Speed]]<br>Warcries have no Cost
+}}