--- Passive Skill:Atlas~torment~2~6
+++ Export
@@ -6,5 +6,5 @@
 |icon                                = TormentNode1 (AtlasTrees)
 |stat1_id                            = map_additional_tormented_spirit_chance_%
 |stat1_value                         = 10
-|stat_text                           = Your Maps have a 10% chance to contain an additional Tormented Spirit
+|stat_text                           = Your Maps have a 10% chance to be haunted by an additional Tormented Spirit
 }}