--- Wiki
+++ Export
@@ -2,10 +2,10 @@
 |id                                  = vaal_spell_keystone_2194
 |int_id                              = 27637
 |name                                = Vaal Dominion
-|flavour_text                        = Gain Soul Harvest for 4 seconds when you Hit an Enemy with a Non-Vaal Spell if you have not gained Soul Harvest in the past 0.5 seconds
-Reminder text:
-(Soul Harvest adds 1 Vaal Soul to a damaging Vaal Spell or Vaal Minion Spell every second
-Up to 4 Soul Harvests can be active at a time)
+|flavour_text                        = Gain Soul Harvest for 4 seconds when you Hit an Enemy with a Non-Vaal Spell if you have not gained Soul Harvest in the past 0.5 seconds
+Reminder text:
+(Soul Harvest adds 1 Vaal Soul to a damaging Vaal Spell or Vaal Minion Spell every second
+Up to 4 Soul Harvests can be active at a time)
 x% less Damage with Non-Vaal Skills
 |is_keystone                         = True
 |icon                                = newtitanicmight
