--- Passive Skill:Atlas~beyond~4~3~
+++ Export
@@ -0,0 +1,13 @@
+{{Passive skill
+|id                                  = atlas_beyond_4_3_
+|int_id                              = 25151
+|name                                = Voracious Throng
+|is_notable                          = True
+|is_atlas_passive                    = True
+|icon                                = BeyondBoss3 (AtlasTrees)
+|stat1_id                            = map_beyond_flesh_faction_chance_+%
+|stat1_value                         = 100
+|stat2_id                            = map_beyond_unique_item_quantity_+%_final_from_flesh_faction
+|stat2_value                         = 30
+|stat_text                           = Beyond Demons in your Maps have 100% increased chance to be followers of Ghorr<br>30% more Unique Items found from Beyond Demons in your Maps<br />that are followers of Ghorr
+}}