--- Passive Skill:Atlas~keystone~maven~influence~spawns~extra~map~bosses
+++ Export
@@ -0,0 +1,13 @@
+{{Passive skill
+|id                                  = atlas_keystone_maven_influence_spawns_extra_map_bosses
+|int_id                              = 34384
+|name                                = Destructive Play
+|flavour_text                        = She makes the rules up as she goes.
+It was never going to be fair.
+|is_keystone                         = True
+|is_atlas_passive                    = True
+|icon                                = DestructivePlay (AtlasTrees)
+|stat1_id                            = atlas_keystone_maven_spawns_extra_map_bosses_while_witnessing_chance_%
+|stat1_value                         = 100
+|stat_text                           = The Maven summons 1 to 3 additional Bosses when Witnessing Map Bosses<br />Modifiers to the Final Map Boss in each Map also apply to these summoned Bosses
+}}