--- Passive Skill:Strength778
+++ Export
@@ -6,5 +6,5 @@
 |stat1_id                            = base_strength
 |stat1_value                         = 10
 |stat_text                           = +10 to [[Strength]]
-|connections                         = strength820,agility863,melee_area_of_effect2358,aura_effect1555,strength794,agility863,melee_area_of_effect2358,savant954,strength777,strength817
+|connections                         = strength820,agility863,melee_area_of_effect2358,attack_ignite_chance2868,strength794,agility863,melee_area_of_effect2358,savant954,strength777,strength817
 }}