--- Passive Skill:AscendancySpecialEldritch4
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = AscendancySpecialEldritch4
 |int_id                              = 42469
 |name                                = Fatal Flourish
-|ascendancy_class                    = Champion
+|ascendancy_class                    = Duelist
 |icon                                = 
 |stat1_id                            = non_travel_attack_skill_repeat_count
 |stat1_value                         = 1
