--- Passive Skill:Affliction~notable~blast-freeze
+++ Export
@@ -8,5 +8,5 @@
 |stat1_value                         = 20
 |stat2_id                            = spread_freeze_to_nearby_enemies
 |stat2_value                         = 1
-|stat_text                           = 20% increased [[Cold Damage]]<br>[[Freeze|Freezes]] you inflict spread to other Enemies within a Radius of 12
+|stat_text                           = 20% increased [[Cold Damage]]<br>[[Freeze|Freezes]] you inflict spread to other Enemies within 1.2 metres
 }}