--- Wiki
+++ Export
@@ -10,5 +10,5 @@
 |stat2_value                         = 1
 |stat3_id                            = returning_projectiles_always_pierce
 |stat3_value                         = 1
-|stat_text                           = [[Return]]ing Projectiles have 150% increased Speed
+|stat_text                           = 15% increased Projectile Speed<br>[[Attack]] Projectiles Return to you<br>Returning Projectiles Pierce all Targets
 }}