--- Wiki
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = pain_attunement1178
 |int_id                              = 31703
 |name                                = Pain Attunement
-|flavour_text                        = Embrace the pain, drink it in.
+|flavour_text                        = Embrace the pain, drink it in.
 Your enemies will know your agony tenfold.
 |is_keystone                         = True
 |icon                                = KeystonePainAttunement
