--- Passive Skill:Atlas~beyond~4~1
+++ Export
@@ -0,0 +1,13 @@
+{{Passive skill
+|id                                  = atlas_beyond_4_1
+|int_id                              = 25892
+|name                                = Swarming Hive
+|is_notable                          = True
+|is_atlas_passive                    = True
+|icon                                = BeyondBoss1 (AtlasTrees)
+|stat1_id                            = map_beyond_demon_faction_chance_+%
+|stat1_value                         = 100
+|stat2_id                            = map_beyond_divination_card_quantity_+%_final_from_demon_faction
+|stat2_value                         = 30
+|stat_text                           = Beyond Demons in your Maps have 100% increased chance to be followers of K'tash<br>30% more Divination Cards found from Beyond Demons in your Maps<br />that are followers of K'Tash
+}}