--- Passive Skill:Strength818
+++ Export
@@ -6,5 +6,5 @@
 |stat1_id                            = base_strength
 |stat1_value                         = 10
 |stat_text                           = +10 to [[Strength]]
-|connections                         = strength820,knockback_distance148,strength792,stun_duration1599
+|connections                         = strength820,knockback_distance148,strength792,axe_sword_damage1603,stun_duration1599
 }}