--- Wiki
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = atlas_keystone_monsterlife_1
 |int_id                              = 2493
 |name                                = Wellspring of Creation
-|flavour_text                        = The waters of life roil and flow,
+|flavour_text                        = The waters of life roil and flow,
 birthing horrors of flesh and marrow.
 |is_keystone                         = True
 |is_atlas_passive                    = True
