--- Passive Skill:Atlas~beyond~4~7
+++ Export
@@ -0,0 +1,10 @@
+{{Passive skill
+|id                                  = atlas_beyond_4_7
+|int_id                              = 8020
+|name                                = Beyond Demon Item Quantity
+|is_atlas_passive                    = True
+|icon                                = BeyondNode1 (AtlasTrees)
+|stat1_id                            = map_beyond_monster_item_quantity_+%
+|stat1_value                         = 5
+|stat_text                           = 5% increased Quantity of Items dropped by Beyond Demons in your Maps
+}}