--- Passive Skill:Atlas~keystone~fortune~favours~doubled
+++ Export
@@ -0,0 +1,12 @@
+{{Passive skill
+|id                                  = atlas_keystone_fortune_favours_doubled
+|int_id                              = 38833
+|name                                = Bold Undertakings
+|flavour_text                        = Fortune will rain upon those who bring humour to Chaos.
+|is_keystone                         = True
+|is_atlas_passive                    = True
+|icon                                = BoldUndertaking (AtlasTrees)
+|stat1_id                            = atlas_keystone_fortune_favours_the_brave_doubled
+|stat1_value                         = 1
+|stat_text                           = Fortune Favours the Brave applies an additional random Option<br />100% more cost of the Fortune Favours the Brave Map Crafting Option
+}}