--- Passive Skill:Atlas~keystone~ritual~single~shop~window
+++ Export
@@ -2,8 +2,8 @@
 |id                                  = atlas_keystone_ritual_single_shop_window
 |int_id                              = 36064
 |name                                = Immutable Dogma
-|flavour_text                        = Do not doubt. Do not question.
-This is the way it has always been.
+|flavour_text                        = Do not doubt. Do not question.
+This is the way it has always been.
 This is the way it shall always be.
 |is_keystone                         = True
 |is_atlas_passive                    = True
