--- Passive Skill:Atlas~torment~3~4
+++ Export
@@ -0,0 +1,11 @@
+{{Passive skill
+|id                                  = atlas_torment_3_4
+|int_id                              = 9345
+|name                                = Paranormal Meeting
+|is_notable                          = True
+|is_atlas_passive                    = True
+|icon                                = TormentNotable3 (AtlasTrees)
+|stat1_id                            = map_spawn_extra_torment_spirits
+|stat1_value                         = 1
+|stat_text                           = Your Maps are haunted by an additional Tormented Spirit
+}}