--- Passive Skill:Atlas~vaal~2~4
+++ Export
@@ -0,0 +1,15 @@
+{{Passive skill
+|id                                  = atlas_vaal_2_4
+|int_id                              = 63874
+|name                                = Unimaginable Horrors
+|is_notable                          = True
+|is_atlas_passive                    = True
+|icon                                = VaalCluster (AtlasTrees)
+|stat1_id                            = map_vaal_side_area_boss_maximum_life_+%_final
+|stat1_value                         = 100
+|stat2_id                            = map_vaal_side_area_boss_damage_+%_final
+|stat2_value                         = 30
+|stat3_id                            = map_vaal_side_area_vaal_vessel_%_chance_to_duplicate_rewards
+|stat3_value                         = 20
+|stat_text                           = Unique Bosses in Vaal Side Areas in your Maps have 100% more [[Life]]<br>Unique Bosses in Vaal Side Areas in your Maps deal 30% more Damage<br>Vaal Side Areas in your Maps have 20% chance for Rewards from Vaal Vessels to be Duplicated
+}}