--- Passive Skill:Aura~effect1555
+++ Export
@@ -6,5 +6,5 @@
 |stat1_id                            = non_curse_aura_effect_+%
 |stat1_value                         = 6
 |stat_text                           = 6% increased effect of Non-Curse Auras from your [[Skill|Skills]]
-|connections                         = aura_area_of_effect1202,banners2750,banners2750,strength778
+|connections                         = aura_area_of_effect1202,banners2750,strength794,banners2750
 }}