--- Passive Skill:Atlas~torment~1~3
+++ Export
@@ -4,7 +4,7 @@
 |name                                = Seance
 |is_notable                          = True
 |is_atlas_passive                    = True
-|icon                                = TormentNotable1 (AtlasTrees)
+|icon                                = TormentNotable2 (AtlasTrees)
 |stat1_id                            = map_rare_monsters_possessed_with_touched_minions_chance_%
 |stat1_value                         = 100
 |stat2_id                            = map_rare_monsters_possessed_with_touched_minions_num
