--- Passive Skill:Wind~dancer~keystone2694
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = wind_dancer_keystone2694
 |int_id                              = 11239
 |name                                = Wind Dancer
-|flavour_text                        = Invite the enemy when you are strong, evade them when you are weak;
+|flavour_text                        = Invite the enemy when you are strong, evade them when you are weak;
 strike with the coming of the storm.
 |is_keystone                         = True
 |icon                                = WindDancer
