--- Passive Skill:Stun~threshold2871
+++ Export
@@ -0,0 +1,10 @@
+{{Passive skill
+|id                                  = stun_threshold2871
+|int_id                              = 27697
+|name                                = Stun Threshold Reduction
+|icon                                = stunstr
+|stat1_id                            = base_stun_threshold_reduction_+%
+|stat1_value                         = 10
+|stat_text                           = 10% reduced Enemy Stun Threshold
+|connections                         = lava_lash1,melee_ignite_notable2875
+}}