--- Passive Skill:Strength775
+++ Export
@@ -6,5 +6,5 @@
 |stat1_id                            = base_strength
 |stat1_value                         = 10
 |stat_text                           = +10 to [[Strength]]
-|connections                         = strength793,armour_endurance_charge2375_,strength781,blood_magic1143,armour_endurance_charge2375_,super_strength2438,strength793
+|connections                         = strength793,armour_endurance_charge2375_,fire_damage_with_attacks2863,strength781,blood_magic1143,armour_endurance_charge2375_,super_strength2438,strength793
 }}