--- Wiki
+++ Export
@@ -0,0 +1,15 @@
+{{Passive skill
+|id                                  = AscendancyDeadeye12
+|int_id                              = 21455
+|name                                = Powerful Precision
+|ascendancy_class                    = Deadeye
+|is_notable                          = True
+|icon                                = PowerfulPrecision (DeadEye)
+|stat1_id                            = piercing_projectiles_critical_strike_chance_+%
+|stat1_value                         = 100
+|stat2_id                            = projectile_base_number_of_targets_to_pierce
+|stat2_value                         = 3
+|stat3_id                            = projectiles_pierce_all_nearby_targets
+|stat3_value                         = 1
+|stat_text                           = Projectiles have 100% increased [[Critical Strike Chance]] against Targets they Pierce<br>Projectiles Pierce 3 additional Targets<br>Projectiles Pierce all nearby Targets
+}}