--- Passive Skill:Affliction~notable~fan~the~flames
+++ Export
@@ -6,5 +6,5 @@
 |icon                                = FireDamageOverTimeNotable
 |stat1_id                            = spread_ignite_to_nearby_enemies
 |stat1_value                         = 1
-|stat_text                           = [[Ignite|Ignites]] you inflict spread to other Enemies within a Radius of 15
+|stat_text                           = [[Ignite|Ignites]] you inflict spread to other Enemies within a Radius of 1.5 metres
 }}