--- Passive Skill:Atlas~keystone~rollable~vaal~side~areas
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = atlas_keystone_rollable_vaal_side_areas
 |int_id                              = 43160
 |name                                = Lucid Dreams
-|flavour_text                        = Shattered fragments of reality float through the void,
+|flavour_text                        = Shattered fragments of reality float through the void,
 waiting to be shaped by your mind's desire.
 |is_keystone                         = True
 |is_atlas_passive                    = True
