--- Passive Skill:Strength792
+++ Export
@@ -6,5 +6,5 @@
 |stat1_id                            = base_strength
 |stat1_value                         = 10
 |stat_text                           = +10 to [[Strength]]
-|connections                         = strength818,axe_sword_damage1603,life_leech142,strength785,strength829,strength818
+|connections                         = strength818,life_leech142,strength785,ignite_bleed_duration2872,strength829,strength818
 }}