--- Passive Skill:AscendancyGuardian2
+++ Export
@@ -5,10 +5,10 @@
 |ascendancy_class                    = Guardian
 |is_notable                          = True
 |icon                                = TimeOfNeed (Guardian)
-|stat1_id                            = curse_effect_on_self_+%
-|stat1_value                         = -80
-|stat2_id                            = guardian_gain_life_regeneration_per_minute_%_for_1_second_every_10_seconds
-|stat2_value                         = 1800
-|stat_text                           = 80% reduced Effect of [[Curse|Curses]] on you<br>Every 4 seconds, Regenerate 30% of [[Life]] over one second
+|stat1_id                            = guardian_gain_life_regeneration_per_minute_%_for_1_second_every_10_seconds
+|stat1_value                         = 6000
+|stat2_id                            = guardian_remove_curses_and_status_ailments_every_10_seconds
+|stat2_value                         = 1
+|stat_text                           = Every 4 seconds, Regenerate 100% of [[Life]] over one second<br>Every 4 seconds, remove [[Curse|Curses]] and Elemental Ailments from you
 |connections                         = AscendancyGuardian5
 }}