--- Wiki
+++ Export
@@ -0,0 +1,12 @@
+{{Passive skill
+|id                                  = atlas_keystone_shaper_elder_guardians
+|int_id                              = 38868
+|name                                = Inevitable Ambush
+|flavour_text                        = The hunter must become the hunted.
+|is_keystone                         = True
+|is_atlas_passive                    = True
+|icon                                = ElderShaperNotable3 (AtlasTrees)
+|stat1_id                            = map_shaper_elder_guardians_assisted_by_allies
+|stat1_value                         = 1
+|stat_text                           = Shaper and Elder Guardians are healed and joined by an ally on first reaching 33% Life<br />Allies have a chance to drop their Fragments
+}}