--- Passive Skill:Atlas~keystone~maven~influence~spawns~extra~map~bosses
+++ Export
@@ -9,5 +9,5 @@
 |icon                                = DestructivePlay (AtlasTrees)
 |stat1_id                            = atlas_keystone_maven_spawns_extra_map_bosses_while_witnessing_chance_%
 |stat1_value                         = 100
-|stat_text                           = The Maven casts Up the Stakes, summoning 1 to 3 additional Bosses when Witnessing Map Bosses<br />Modifiers to the Final Map Boss in each Map also apply to these summoned Bosses
+|stat_text                           = The Maven casts Up the Stakes, summoning 1 to 3 additional Bosses<br />when Witnessing Map Bosses<br />Modifiers to the Final Map Boss in each Map also apply to these summoned Bosses
 }}