--- Wiki
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = atlas_keystone_stronger_rares
 |int_id                              = 6956
 |name                                = Valley of Darkness
-|flavour_text                        = Darkening mists shroud the pale sky,
+|flavour_text                        = Darkening mists shroud the pale sky,
 blurring the barrier between nightmare and dream.
 |is_keystone                         = True
 |is_atlas_passive                    = True
