--- Passive Skill:Atlas~keystone~enhanced~kirac~crafts
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = atlas_keystone_enhanced_kirac_crafts
 |int_id                              = 52359
 |name                                = Overloaded Circuits
-|flavour_text                        = "Supplying the Map Device with more power has... unexpected effects.
+|flavour_text                        = "Supplying the Map Device with more power has... unexpected effects.
 Somehow, though, I doubt you'll be deterred."
 |is_keystone                         = True
 |is_atlas_passive                    = True
