--- Passive Skill:Atlas~keystone~random~master~missions
+++ Export
@@ -0,0 +1,16 @@
+{{Passive skill
+|id                                  = atlas_keystone_random_master_missions
+|int_id                              = 7235
+|name                                = All Hands
+|flavour_text                        = The enemy is at the gates. Every single soldier we have may still not be enough...
+|is_keystone                         = True
+|is_atlas_passive                    = True
+|icon                                = AllHands (AtlasTrees)
+|stat1_id                            = map_master_missions_cannot_be_gained_on_map_completion
+|stat1_value                         = 1
+|stat2_id                            = map_master_encounter_chance_%
+|stat2_value                         = 40
+|stat3_id                            = map_master_mission_on_map_completion_chance_applies_to_master_encounter_chance_instead_at_%_value
+|stat3_value                         = 150
+|stat_text                           = Your Maps do not grant Master Missions when Completed<br>Your Maps have +40% chance to contain a random Master encounter<br>Modifiers to chance to grant an additional Master Mission on Map Completion instead apply to chance for that Master to be randomly encountered in your Maps at 150% of their value
+}}