--- Passive Skill:Atlas~torment~2~5
+++ Export
@@ -4,7 +4,7 @@
 |name                                = Unrelenting Torment
 |is_notable                          = True
 |is_atlas_passive                    = True
-|icon                                = TormentNode1 (AtlasTrees)
+|icon                                = TormentNotable1 (AtlasTrees)
 |stat1_id                            = map_possessed_monster_item_quantity_+%_final
 |stat1_value                         = 30
 |stat2_id                            = map_boss_chance_to_be_surrounded_by_spirits_%
