--- Passive Skill:Atlas~keystone~altars~1
+++ Export
@@ -2,8 +2,8 @@
 |id                                  = atlas_keystone_altars_1
 |int_id                              = 33008
 |name                                = Eldritch Gaze
-|flavour_text                        = You were drunk on the power of the cosmos,
-until a hideous gaze settled upon you,
+|flavour_text                        = You were drunk on the power of the cosmos,
+until a hideous gaze settled upon you,
 and tendrils began to unfurl...
 |is_keystone                         = True
 |is_atlas_passive                    = True
