--- Passive Skill:Supreme~ego~keystone2696
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = supreme_ego_keystone2696
 |int_id                              = 49639
 |name                                = Supreme Ego
-|flavour_text                        = You are the heart of the empire, shining bright so that all are lifted up by your light.
+|flavour_text                        = You are the heart of the empire, shining bright so that all are lifted up by your light.
 Let none tell you otherwise.
 |is_keystone                         = True
 |icon                                = SupremeEgo
