--- Passive Skill:AscendancySpecialEldritch2
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = AscendancySpecialEldritch2
 |int_id                              = 57568
 |name                                = Searing Purity
-|ascendancy_class                    = Guardian
+|ascendancy_class                    = Templar
 |icon                                = 
 |stat1_id                            = chaos_hit_and_dot_damage_%_taken_as_lightning
 |stat1_value                         = 45
