--- Passive Skill:Atlas~keystone~enhanced~kirac~crafts
+++ Export
@@ -0,0 +1,13 @@
+{{Passive skill
+|id                                  = atlas_keystone_enhanced_kirac_crafts
+|int_id                              = 52359
+|name                                = Overloaded Circuits
+|flavour_text                        = "Supplying the Map Device with more power has... unexpected effects.
+Somehow, though, I doubt you'll be deterred."
+|is_keystone                         = True
+|is_atlas_passive                    = True
+|icon                                = OverloadedCircuits (AtlasTrees)
+|stat1_id                            = atlas_keystone_kirac_crafts_grant_notables
+|stat1_value                         = 1
+|stat_text                           = League Map Crafting Options also choose 3 random Notable Atlas Passive [[Skill|Skills]] associated with that League to treat as allocated<br />100% more cost of Map Crafting Options
+}}