--- Passive Skill:Atlas~keystone~ritual~variable~shop~prices
+++ Export
@@ -0,0 +1,14 @@
+{{Passive skill
+|id                                  = atlas_keystone_ritual_variable_shop_prices
+|int_id                              = 1733
+|name                                = Arbitrary Tenets
+|flavour_text                        = They are unforgivable sinners and fools...
+We pray each Sacrato. They pray each Kaso.
+They are absolutely mad!
+|is_keystone                         = True
+|is_atlas_passive                    = True
+|icon                                = ArbitraryTenets (AtlasTrees)
+|stat1_id                            = map_ritual_offered_rewards_have_additional_variable_cost
+|stat1_value                         = 1
+|stat_text                           = Favours at Ritual Altars in your Maps randomly cost between 90% less and 80% more
+}}