--- Passive Skill:AscendancyAscendant33~
+++ Export
@@ -8,14 +8,12 @@
 |icon                                = Chieftain (Ascendants)
 |stat1_id                            = apply_covered_in_ash_to_attacker_on_hit_%_vs_rare_or_unique_enemy
 |stat1_value                         = 20
-|stat2_id                            = totem_damage_leeched_as_life_to_you_permyriad
-|stat2_value                         = 100
-|stat3_id                            = life_regeneration_rate_per_minute_%
-|stat3_value                         = 120
+|stat2_id                            = unaffected_by_ignite
+|stat2_value                         = 1
+|stat3_id                            = totems_taunt_enemies_around_them_for_x_seconds_when_summoned
+|stat3_value                         = 2
 |stat4_id                            = strength_+%
 |stat4_value                         = 10
-|stat5_id                            = fire_damage_+%
-|stat5_value                         = 40
-|stat_text                           = 20% chance to Cover Rare or Unique Enemies in Ash for 10 Seconds on [[Hit]]<br>1% of Damage dealt by your [[Totem|Totems]] is Leeched to you as [[Life]]<br>Regenerate 2% of [[Life]] per second<br>10% increased [[Strength]]<br>40% increased [[Fire Damage]]
+|stat_text                           = 20% chance to Cover Rare or Unique Enemies in Ash for 10 Seconds on [[Hit]]<br>Unaffected by [[Ignite]]<br>[[Totem|Totems]] Taunt Enemies around them for 2 seconds when Summoned<br>10% increased [[Strength]]
 |connections                         = AscendancyAscendant25
 }}