--- Wiki
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = sacred_bastion_keystone2779
 |int_id                              = 24720
 |name                                = Imbalanced Guard
-|flavour_text                        = Layer only your most vital areas, child. We have less steel,
+|flavour_text                        = Layer only your most vital areas, child. We have less steel,
 but we are willing to bleed, and that is why we will be victorious.
 |is_keystone                         = True
 |icon                                = SacredBastionKeystone
