--- Passive Skill:AscendancySpecialEldritch7
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = AscendancySpecialEldritch7
 |int_id                              = 19355
 |name                                = Unleashed Potential
-|ascendancy_class                    = Ascendant
+|ascendancy_class                    = Scion
 |icon                                = 
 |stat1_id                            = max_endurance_charges
 |stat1_value                         = 1
