--- Wiki
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = vaal_pact1420
 |int_id                              = 10808
 |name                                = Vaal Pact
-|flavour_text                        = My ancestral pact was sealed. Forevermore, I would gain sustenance
+|flavour_text                        = My ancestral pact was sealed. Forevermore, I would gain sustenance
 only from the ravaged flesh of my enemies.
 |is_keystone                         = True
 |icon                                = vaalpact
