--- Passive Skill:AscendancySlayer12
+++ Export
@@ -13,6 +13,6 @@
 |stat3_value                         = 50
 |stat4_id                            = slayer_damage_+%_final_from_distance
 |stat4_value                         = 15
-|stat_text                           = 5% increased Area of Effect per Enemy killed recently, up to 50%<br>+4 to Melee Strike Range<br>50% increased Global [[Accuracy Rating]]<br>Deal up to 15% more Melee Damage to Enemies, based on proximity
+|stat_text                           = 5% increased Area of Effect per Enemy killed recently, up to 50%<br>+0.4 metres to Melee Strike Range<br>50% increased Global [[Accuracy Rating]]<br>Deal up to 15% more Melee Damage to Enemies, based on proximity
 |connections                         = AscendancySlayer6
 }}