--- Wiki
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = atlas_keystone_monsterdamage_1
 |int_id                              = 36386
 |name                                = Dance of Destruction
-|flavour_text                        = Atlassian aberrations engage in a deadly dance.
+|flavour_text                        = Atlassian aberrations engage in a deadly dance.
 Will you be joining them in the fray?
 |is_keystone                         = True
 |is_atlas_passive                    = True
