--- Wiki
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = number_of_totems_allowed1269
 |int_id                              = 41970
 |name                                = Ancestral Bond
-|flavour_text                        = A wooden construct, mute and blind.
+|flavour_text                        = A wooden construct, mute and blind.
 But fear the wrath of shackled mind.
 |is_keystone                         = True
 |icon                                = totemmax
