--- Passive Skill:Atlas~torment~3~2
+++ Export
@@ -0,0 +1,10 @@
+{{Passive skill
+|id                                  = atlas_torment_3_2
+|int_id                              = 2461
+|name                                = Tormented Spirit Duration
+|is_atlas_passive                    = True
+|icon                                = TormentNode1 (AtlasTrees)
+|stat1_id                            = map_tormented_spirits_duration_+%
+|stat1_value                         = 25
+|stat_text                           = Tormented Spirits in your Maps have 25% increased Duration
+}}