--- Wiki
+++ Export
@@ -2,7 +2,7 @@
 |id                                  = unwavering_stance1139
 |int_id                              = 40907
 |name                                = Unwavering Stance
-|flavour_text                        = Stand your ground, child, keep your senses.
+|flavour_text                        = Stand your ground, child, keep your senses.
 The pain is fleeting, but victory is forever.
 |is_keystone                         = True
 |icon                                = KeystoneUnwaveringStance
