--- Skill:ExileInfernalBlow
+++ Export
@@ -10,15 +10,16 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 0.5
 |static_stat1_id                         = active_skill_damage_+%_final
 |static_stat1_value                      = 25
-|static_stat2_id                         = corpse_explosion_monster_life_%
-|static_stat2_value                      = 25
-|static_stat3_id                         = base_skill_effect_duration
-|static_stat3_value                      = 500
+|static_stat2_id                         = base_skill_effect_duration
+|static_stat2_value                      = 500
+|static_stat3_id                         = corpse_explosion_monster_life_%
+|static_stat3_value                      = 25
 |static_stat4_id                         = skill_physical_damage_%_to_convert_to_fire
 |static_stat4_value                      = 50
-|stat_text                               = Deals 25% more Damage<br>Explosion deals base Fire Damage equal to 25% of the corpse's Maximum Life<br>Base duration is 0.5 seconds<br>50% of Physical Damage Converted to Fire Damage
+|stat_text                               = Deals 25% more Damage<br>Base duration is 0.5 seconds<br>Explosion deals base Fire Damage equal to 25% of the corpse's Maximum Life<br>50% of Physical Damage Converted to Fire Damage
 |level1                                  = True
 |level1_level_requirement                = 4
 |level1_cost_amounts                     = 9
