--- Skill:ReaperWraithChargeMarker
+++ Export
@@ -15,9 +15,9 @@
 |static_stat2_value                      = 1
 |static_stat3_id                         = skill_cannot_be_interrupted
 |static_stat3_value                      = 1
-|static_stat4_id                         = skill_cannot_be_stunned
+|static_stat4_id                         = skill_cannot_be_knocked_back
 |static_stat4_value                      = 1
-|static_stat5_id                         = skill_cannot_be_knocked_back
+|static_stat5_id                         = skill_cannot_be_stunned
 |static_stat5_value                      = 1
 |stat_text                               = 
 |level1                                  = True
