--- Skill:TotemMapBossFrostBolt
+++ Export
@@ -8,15 +8,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_cast_speed_+%
-|static_stat1_value                      = 75
-|static_stat2_id                         = active_skill_cast_speed_+%_final
-|static_stat2_value                      = -33
-|static_stat3_id                         = always_pierce
-|static_stat3_value                      = 1
-|static_stat4_id                         = base_is_projectile
-|static_stat4_value                      = 1
-|stat_text                               = Deals (18-2538) to (27-3807) Cold Damage<br>Skills Repeat an additional (1-2) Times<br>(30-0)% increased Projectile Speed<br>75% increased Cast Speed<br>33% less Cast Speed<br>Projectiles Pierce all Targets
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = always_pierce
+|static_stat2_value                      = 1
+|static_stat3_id                         = active_skill_cast_speed_+%_final
+|static_stat3_value                      = -33
+|static_stat4_id                         = base_cast_speed_+%
+|static_stat4_value                      = 75
+|stat_text                               = Deals (18-2538) to (27-3807) Cold Damage<br>Skills Repeat an additional (1-2) Times<br>Projectiles Pierce all Targets<br>33% less Cast Speed<br>75% increased Cast Speed<br>(30-0)% increased Projectile Speed
 |level1                                  = True
 |level1_level_requirement                = 3
 |level1_stat_text                        = Deals 18 to 27 Cold Damage<br>Skills Repeat an additional Time<br>30% increased Projectile Speed
