--- Skill:GoatmanMoltenShellInvasion
+++ Export
@@ -9,16 +9,16 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 10
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = 50
-|static_stat2_id                         = base_skill_effect_duration
-|static_stat2_value                      = 10000
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
-|stat_text                               = Deals (15-5246) to (23-7869) Fire Damage<br>Shields break after (4-2648) total Damage is prevented<br>(162-4334) additional Armour<br>Base duration is 10 seconds
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = base_skill_effect_duration
+|static_stat3_value                      = 10000
+|stat_text                               = Base duration is 10 seconds<br>Deals (15-5246) to (23-7869) Fire Damage<br>Shields break after (4-2648) total Damage is prevented<br>(162-4334) additional Armour
 |level1                                  = True
 |level1_level_requirement                = 3
-|level1_critical_strike_chance           = 0
 |level1_stat_text                        = Deals 15 to 23 Fire Damage<br>Shields break after 4 total Damage is prevented<br>162 additional Armour
 |level1_stat1_id                         = spell_minimum_base_fire_damage
 |level1_stat1_value                      = 15
@@ -30,7 +30,6 @@
 |level1_stat4_value                      = 162
 |level2                                  = True
 |level2_level_requirement                = 5
-|level2_critical_strike_chance           = 0
 |level2_stat_text                        = Deals 20 to 30 Fire Damage<br>Shields break after 5 total Damage is prevented<br>314 additional Armour
 |level2_stat1_id                         = spell_minimum_base_fire_damage
 |level2_stat1_value                      = 20
@@ -42,7 +41,6 @@
 |level2_stat4_value                      = 314
 |level3                                  = True
 |level3_level_requirement                = 8
-|level3_critical_strike_chance           = 0
 |level3_stat_text                        = Deals 27 to 41 Fire Damage<br>Shields break after 8 total Damage is prevented<br>484 additional Armour
 |level3_stat1_id                         = spell_minimum_base_fire_damage
 |level3_stat1_value                      = 27
@@ -54,7 +52,6 @@
 |level3_stat4_value                      = 484
 |level4                                  = True
 |level4_level_requirement                = 12
-|level4_critical_strike_chance           = 0
 |level4_stat_text                        = Deals 41 to 61 Fire Damage<br>Shields break after 11 total Damage is prevented<br>686 additional Armour
 |level4_stat1_id                         = spell_minimum_base_fire_damage
 |level4_stat1_value                      = 41
@@ -66,7 +63,6 @@
 |level4_stat4_value                      = 686
 |level5                                  = True
 |level5_level_requirement                = 15
-|level5_critical_strike_chance           = 0
 |level5_stat_text                        = Deals 54 to 81 Fire Damage<br>Shields break after 15 total Damage is prevented<br>847 additional Armour
 |level5_stat1_id                         = spell_minimum_base_fire_damage
 |level5_stat1_value                      = 54
@@ -78,7 +74,6 @@
 |level5_stat4_value                      = 847
 |level6                                  = True
 |level6_level_requirement                = 19
-|level6_critical_strike_chance           = 0
 |level6_stat_text                        = Deals 76 to 114 Fire Damage<br>Shields break after 22 total Damage is prevented<br>1021 additional Armour
 |level6_stat1_id                         = spell_minimum_base_fire_damage
 |level6_stat1_value                      = 76
@@ -90,7 +85,6 @@
 |level6_stat4_value                      = 1021
 |level7                                  = True
 |level7_level_requirement                = 22
-|level7_critical_strike_chance           = 0
 |level7_stat_text                        = Deals 97 to 146 Fire Damage<br>Shields break after 29 total Damage is prevented<br>1195 additional Armour
 |level7_stat1_id                         = spell_minimum_base_fire_damage
 |level7_stat1_value                      = 97
@@ -102,7 +96,6 @@
 |level7_stat4_value                      = 1195
 |level8                                  = True
 |level8_level_requirement                = 26
-|level8_critical_strike_chance           = 0
 |level8_stat_text                        = Deals 133 to 200 Fire Damage<br>Shields break after 41 total Damage is prevented<br>1369 additional Armour
 |level8_stat1_id                         = spell_minimum_base_fire_damage
 |level8_stat1_value                      = 133
@@ -114,7 +107,6 @@
 |level8_stat4_value                      = 1369
 |level9                                  = True
 |level9_level_requirement                = 30
-|level9_critical_strike_chance           = 0
 |level9_stat_text                        = Deals 180 to 270 Fire Damage<br>Shields break after 57 total Damage is prevented<br>1594 additional Armour
 |level9_stat1_id                         = spell_minimum_base_fire_damage
 |level9_stat1_value                      = 180
