--- Skill:ElderGuardianChaosDodgeRight
+++ Export
@@ -9,10 +9,10 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_dodge_distance
-|static_stat1_value                      = 40
-|static_stat2_id                         = monster_dodge_direction
-|static_stat2_value                      = 1
+|static_stat1_id                         = monster_dodge_direction
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_dodge_distance
+|static_stat2_value                      = 40
 |static_stat3_id                         = cyclone_movement_speed_+%_final
 |static_stat3_value                      = 100
 |stat_text                               = 100% more Movement Speed
