--- Skill:CelestialOrbOfStormsShaperInfluence
+++ Export
@@ -9,19 +9,20 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage
-|static_stat1_value                      = 3
-|static_stat2_id                         = spell_maximum_base_lightning_damage
-|static_stat2_value                      = 9
-|static_stat3_id                         = orb_of_storms_base_bolt_frequency_ms
-|static_stat3_value                      = 1000
-|static_stat4_id                         = projectile_number_to_split
-|static_stat4_value                      = 2
-|static_stat5_id                         = base_skill_effect_duration
-|static_stat5_value                      = 6000
-|static_stat6_id                         = storm_cloud_destroy_when_caster_dies
-|static_stat6_value                      = 1
-|stat_text                               = Deals 3 to 9 Lightning Damage<br>Strikes every 1 second, and when you use a Lightning Skill near the Orb<br>Beams Split towards 2 additional targets<br>Base duration is 6 seconds
+|static_duration                         = 6
+|static_stat1_id                         = storm_cloud_destroy_when_caster_dies
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_lightning_damage
+|static_stat2_value                      = 3
+|static_stat3_id                         = spell_maximum_base_lightning_damage
+|static_stat3_value                      = 9
+|static_stat4_id                         = base_skill_effect_duration
+|static_stat4_value                      = 6000
+|static_stat5_id                         = orb_of_storms_base_bolt_frequency_ms
+|static_stat5_value                      = 1000
+|static_stat6_id                         = projectile_number_to_split
+|static_stat6_value                      = 2
+|stat_text                               = Deals 3 to 9 Lightning Damage<br>Base duration is 6 seconds<br>Strikes every 1 second, and when you use a Lightning Skill near the Orb<br>Beams Split towards 2 additional targets
 |level1                                  = True
 |level1_level_requirement                = 1
 }}