--- Skill:DarkExileReave
+++ Export
@@ -10,12 +10,12 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = -9
-|static_stat2_id                         = attack_speed_+%
-|static_stat2_value                      = 55
-|static_stat3_id                         = reave_area_of_effect_+%_final_per_stage
-|static_stat3_value                      = 80
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = attack_speed_+%
+|static_stat3_value                      = 55
+|static_stat4_id                         = reave_area_of_effect_+%_final_per_stage
+|static_stat4_value                      = 80
 |stat_text                               = 55% increased Attack Speed<br>80% more Area of Effect for each stage
 |level1                                  = True
 |level1_level_requirement                = 73
