--- Skill:AbyssTeleportStrike
+++ Export
@@ -9,11 +9,11 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = main_hand_base_maximum_attack_distance
 |static_stat1_value                      = 20
-|static_stat2_id                         = active_skill_attack_speed_+%_final
-|static_stat2_value                      = -84
-|static_stat3_id                         = global_cannot_crit
-|static_stat3_value                      = 1
-|stat_text                               = 84% less Attack Speed<br>Never deal Critical Strikes
+|static_stat2_id                         = global_cannot_crit
+|static_stat2_value                      = 1
+|static_stat3_id                         = active_skill_attack_speed_+%_final
+|static_stat3_value                      = -84
+|stat_text                               = Never deal Critical Strikes<br>84% less Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 1
 }}