--- Skill:HydraDoomArrow
+++ Export
@@ -11,14 +11,14 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = 45
-|static_stat2_id                         = doom_arrow_number_of_arrows
-|static_stat2_value                      = 10
-|static_stat3_id                         = active_skill_damage_+%_final
-|static_stat3_value                      = 120
-|static_stat4_id                         = skill_physical_damage_%_to_convert_to_cold
-|static_stat4_value                      = 50
-|static_stat5_id                         = base_is_projectile
-|static_stat5_value                      = 1
+|static_stat2_id                         = base_is_projectile
+|static_stat2_value                      = 1
+|static_stat3_id                         = doom_arrow_number_of_arrows
+|static_stat3_value                      = 10
+|static_stat4_id                         = active_skill_damage_+%_final
+|static_stat4_value                      = 120
+|static_stat5_id                         = skill_physical_damage_%_to_convert_to_cold
+|static_stat5_value                      = 50
 |stat_text                               = Deals 120% more Damage<br>50% of Physical Damage Converted to Cold Damage
 |level1                                  = True
 |level1_level_requirement                = 83
