--- Skill:GravitySphereTriggered
+++ Export
@@ -0,0 +1,26 @@
+{{Skill
+|gem_description                         = Creates a Gravity Sphere which Hinders enemies in an area around it, with the debuff being stronger on enemies closer to the sphere. The Gravity Sphere will consume the corpses of any enemies which die in its area. Can only have one Gravity Sphere at a time.
+|active_skill_name                       = Gravity Sphere
+|skill_id                                = GravitySphereTriggered
+|cast_time                               = 0.6
+|required_level                          = 70
+|static_cost_amounts                     = 24
+|static_cost_types                       = Mana
+|static_stored_uses                      = 1
+|static_cooldown                         = 1.5
+|static_damage_effectiveness             = 100
+|static_damage_multiplier                = 100
+|static_attack_speed_multiplier          = 100
+|static_duration                         = 2
+|static_stat1_id                         = base_skill_effect_duration
+|static_stat1_value                      = 2000
+|static_stat2_id                         = base_blackhole_tick_rate_ms
+|static_stat2_value                      = 400
+|static_stat3_id                         = blackhole_hinder_%
+|static_stat3_value                      = -39
+|static_stat4_id                         = chance_to_trigger_level_20_gravity_sphere_on_cast_with_storm_burst_or_divine_ire_%
+|static_stat4_value                      = 100
+|stat_text                               = Base duration is 2 seconds<br>Pulses every 0.4 seconds<br>Enemies in range are Hindered, with up to 39% reduced Movement Speed, based on distance from the Void Sphere<br>This Skill is Triggered when you Cast Storm Burst or Divine Ire
+|level1                                  = True
+|level1_level_requirement                = 70
+}}