--- Skill:GSBlindingGaze
+++ Export
@@ -12,10 +12,10 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = 25
-|static_stat2_id                         = spell_maximum_action_distance_+%
-|static_stat2_value                      = -80
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = spell_maximum_action_distance_+%
+|static_stat3_value                      = -80
 |stat_text                               = Deals (14-1506) to (26-2797) Lightning Damage<br>(0-40)% more Cast Speed
 |level1                                  = True
 |level1_level_requirement                = 3
