--- Skill:InvasionLeaguestoneSummonMonsters
+++ Export
@@ -7,10 +7,10 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = summoned_monster_rarity
-|static_stat1_value                      = 1
-|static_stat2_id                         = summon_monster_radius
-|static_stat2_value                      = 62
+|static_stat1_id                         = summon_monster_radius
+|static_stat1_value                      = 62
+|static_stat2_id                         = summoned_monster_rarity
+|static_stat2_value                      = 1
 |stat_text                               = 
 |level1                                  = True
 |level1_level_requirement                = 1
