--- Skill:GSExpeditionZombieOnDeath
+++ Export
@@ -8,10 +8,10 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = skill_physical_damage_%_to_convert_to_cold
-|static_stat1_value                      = 50
-|static_stat2_id                         = is_area_damage
-|static_stat2_value                      = 1
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = skill_physical_damage_%_to_convert_to_cold
+|static_stat2_value                      = 50
 |stat_text                               = Deals (16-1281) to (24-1922) Physical Damage<br>Deals (15-0)% more Damage<br>50% of Physical Damage Converted to Cold Damage
 |level1                                  = True
 |level1_level_requirement                = 0
