--- Skill:ExpeditionWhirlingBlades
+++ Export
@@ -1,5 +1,5 @@
 {{Skill
-|gem_description                         = Dive through enemies, dealing weapon damage. Only works with daggers, claws and one handed swords. Cannot be supported by Multistrike.
+|gem_description                         = Dive through enemies, dealing weapon damage. If dual wielding attacks with both weapons, dealing the damage of both in one hit. Only works with Daggers, Claws, and One-Handed Swords. Cannot be supported by Multistrike.
 |active_skill_name                       = Whirling Blades
 |item_class_id_restriction               = Dagger, One Hand Sword, Thrusting One Hand Sword, Claw, Rune Dagger
 |skill_id                                = ExpeditionWhirlingBlades
@@ -10,16 +10,16 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 110
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = skill_sound_variation
-|static_stat1_value                      = 3
-|static_stat2_id                         = monster_flurry
+|static_stat1_id                         = cast_time_overrides_attack_duration
+|static_stat1_value                      = 1
+|static_stat2_id                         = ignores_proximity_shield
 |static_stat2_value                      = 1
-|static_stat3_id                         = bleed_on_melee_attack_chance_%
-|static_stat3_value                      = 20
-|static_stat4_id                         = cast_time_overrides_attack_duration
-|static_stat4_value                      = 1
-|static_stat5_id                         = ignores_proximity_shield
-|static_stat5_value                      = 1
+|static_stat3_id                         = monster_flurry
+|static_stat3_value                      = 1
+|static_stat4_id                         = skill_sound_variation
+|static_stat4_value                      = 3
+|static_stat5_id                         = bleed_on_melee_attack_chance_%
+|static_stat5_value                      = 20
 |stat_text                               = Melee Attacks have 20% chance to cause Bleeding
 |level1                                  = True
 |level1_level_requirement                = 1
