--- Skill:HandfingerHammerThrowMultipleProjectiles
+++ Export
@@ -9,12 +9,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = projectile_spread_radius
-|static_stat1_value                      = 100
-|static_stat2_id                         = number_of_additional_projectiles
-|static_stat2_value                      = 2
-|static_stat3_id                         = base_is_projectile
-|static_stat3_value                      = 1
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = projectile_spread_radius
+|static_stat2_value                      = 100
+|static_stat3_id                         = number_of_additional_projectiles
+|static_stat3_value                      = 2
 |stat_text                               = Fires 2 additional Projectiles
 |level1                                  = True
 |level1_level_requirement                = 2
