--- Skill:MonsterFirestormLegends
+++ Export
@@ -8,21 +8,22 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_fire_damage
-|static_stat1_value                      = 719
-|static_stat2_id                         = spell_maximum_base_fire_damage
-|static_stat2_value                      = 1078
-|static_stat3_id                         = firestorm_base_area_of_effect_+%
-|static_stat3_value                      = 56
-|static_stat4_id                         = base_chance_to_ignite_%
-|static_stat4_value                      = 25
-|static_stat5_id                         = fire_storm_fireball_delay_ms
-|static_stat5_value                      = 120
-|static_stat6_id                         = base_skill_effect_duration
-|static_stat6_value                      = 1000
-|static_stat7_id                         = is_area_damage
-|static_stat7_value                      = 1
-|stat_text                               = Deals 719 to 1078 Fire Damage<br>25% chance to Ignite enemies<br>One impact every 0.12 seconds<br>Base duration is 1 seconds
+|static_duration                         = 1
+|static_stat1_id                         = firestorm_base_area_of_effect_+%
+|static_stat1_value                      = 56
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = spell_minimum_base_fire_damage
+|static_stat3_value                      = 719
+|static_stat4_id                         = spell_maximum_base_fire_damage
+|static_stat4_value                      = 1078
+|static_stat5_id                         = base_skill_effect_duration
+|static_stat5_value                      = 1000
+|static_stat6_id                         = base_chance_to_ignite_%
+|static_stat6_value                      = 25
+|static_stat7_id                         = fire_storm_fireball_delay_ms
+|static_stat7_value                      = 120
+|stat_text                               = Deals 719 to 1078 Fire Damage<br>Base duration is 1 seconds<br>25% chance to Ignite enemies<br>One impact every 0.12 seconds
 |level1                                  = True
 |level1_level_requirement                = 74
 }}