--- Skill:MapKaruiElderDropArrowBlink
+++ Export
@@ -10,21 +10,22 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_fire_damage_to_deal_per_minute
-|static_stat1_value                      = 648
-|static_stat2_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat2_value                      = -35
+|static_duration                         = 2.5
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = -35
+|static_stat2_id                         = base_is_projectile
+|static_stat2_value                      = 1
 |static_stat3_id                         = ground_fire_art_variation
 |static_stat3_value                      = 2
-|static_stat4_id                         = base_arrow_speed_+%
-|static_stat4_value                      = 100
-|static_stat5_id                         = base_skill_effect_duration
-|static_stat5_value                      = 2500
-|static_stat6_id                         = base_is_projectile
-|static_stat6_value                      = 1
-|static_stat7_id                         = is_area_damage
-|static_stat7_value                      = 1
-|stat_text                               = Deals 10.8 Base Fire Damage per second<br>100% increased Arrow Speed<br>Base duration is 2.5 seconds
+|static_stat4_id                         = is_area_damage
+|static_stat4_value                      = 1
+|static_stat5_id                         = base_fire_damage_to_deal_per_minute
+|static_stat5_value                      = 648
+|static_stat6_id                         = base_skill_effect_duration
+|static_stat6_value                      = 2500
+|static_stat7_id                         = base_arrow_speed_+%
+|static_stat7_value                      = 100
+|stat_text                               = Deals 10.8 Base Fire Damage per second<br>Base duration is 2.5 seconds<br>100% increased Arrow Speed
 |level1                                  = True
 |level1_level_requirement                = 1
 }}