--- Skill:TouchOfFire
+++ Export
@@ -1,6 +1,6 @@
 {{Skill
-|gem_description                         = [DNT] 
-|active_skill_name                       = [DNT] Touch of Fire
+|gem_description                         = Become touched by the flames of the Cleansing Fire.
+|active_skill_name                       = Approaching Flames
 |skill_id                                = TouchOfFire
 |cast_time                               = 0.0
 |required_level                          = 70
@@ -11,7 +11,7 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = display_touch_of_fire
 |static_stat1_value                      = 1
-|stat_text                               = [DNT] Grants Flame-Touched
+|stat_text                               = Become Flame-Touched
 |level1                                  = True
 |level1_level_requirement                = 70
 }}