--- Skill:AtlasOrionBeamSweepNoFinalExplosionLongUber
+++ Export
@@ -10,19 +10,19 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 911
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 1367
-|static_stat3_id                         = skill_physical_damage_%_to_convert_to_chaos
-|static_stat3_value                      = 25
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_physical_damage
+|static_stat2_value                      = 911
+|static_stat3_id                         = spell_maximum_base_physical_damage
+|static_stat3_value                      = 1367
 |static_stat4_id                         = skill_physical_damage_%_to_convert_to_lightning
 |static_stat4_value                      = 25
-|static_stat5_id                         = skill_physical_damage_%_to_convert_to_fire
+|static_stat5_id                         = skill_physical_damage_%_to_convert_to_chaos
 |static_stat5_value                      = 25
-|static_stat6_id                         = is_area_damage
-|static_stat6_value                      = 1
-|stat_text                               = Deals 911 to 1367 Physical Damage<br>25% of Physical Damage Converted to Chaos Damage<br>25% of Physical Damage Converted to Lightning Damage<br>25% of Physical Damage Converted to Fire Damage
+|static_stat6_id                         = skill_physical_damage_%_to_convert_to_fire
+|static_stat6_value                      = 25
+|stat_text                               = Deals 911 to 1367 Physical Damage<br>25% of Physical Damage Converted to Lightning Damage<br>25% of Physical Damage Converted to Chaos Damage<br>25% of Physical Damage Converted to Fire Damage
 |level1                                  = True
 |level1_level_requirement                = 84
 }}