--- Skill:FigureheadGhostProjectile
+++ Export
@@ -7,20 +7,20 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
+|static_stat1_id                         = base_is_projectile
 |static_stat1_value                      = 1
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 5
+|static_stat2_id                         = monster_projectile_variation
+|static_stat2_value                      = 19
 |static_stat3_id                         = projectile_spread_radius
 |static_stat3_value                      = 15
 |static_stat4_id                         = spell_maximum_action_distance_+%
 |static_stat4_value                      = -30
-|static_stat5_id                         = monster_projectile_variation
-|static_stat5_value                      = 19
-|static_stat6_id                         = active_skill_cast_speed_+%_final
-|static_stat6_value                      = -20
-|static_stat7_id                         = base_is_projectile
-|static_stat7_value                      = 1
+|static_stat5_id                         = spell_minimum_base_physical_damage
+|static_stat5_value                      = 1
+|static_stat6_id                         = spell_maximum_base_physical_damage
+|static_stat6_value                      = 5
+|static_stat7_id                         = active_skill_cast_speed_+%_final
+|static_stat7_value                      = -20
 |stat_text                               = Deals 1 to 5 Physical Damage<br>20% less Cast Speed
 |level1                                  = True
 |level1_level_requirement                = 3
