--- Skill:ExileDarkPact
+++ Export
@@ -8,15 +8,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_chaos_damage
-|static_stat1_value                      = 5
-|static_stat2_id                         = spell_maximum_base_chaos_damage
-|static_stat2_value                      = 7
-|static_stat3_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat3_value                      = 50
-|static_stat4_id                         = skeletal_chains_no_minions_targets_self
-|static_stat4_value                      = 1
-|static_stat5_id                         = is_area_damage
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = 50
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = spell_minimum_base_chaos_damage
+|static_stat3_value                      = 5
+|static_stat4_id                         = spell_maximum_base_chaos_damage
+|static_stat4_value                      = 7
+|static_stat5_id                         = skeletal_chains_no_minions_targets_self
 |static_stat5_value                      = 1
 |stat_text                               = Deals 5 to 7 Chaos Damage<br>Uses your Life if no Skeletons in range
 |level1                                  = True
