--- Skill:BrutusCageSummonMaps
+++ Export
@@ -14,6 +14,7 @@
 |stat_text                               = Deals (3-161) to (4-241) Physical Damage<br>Base duration is (5.5-14.5) seconds
 |level1                                  = True
 |level1_level_requirement                = 9
+|level1_duration                         = 5.5
 |level1_stat_text                        = Deals 3 to 4 Physical Damage<br>Base duration is 5.5 seconds
 |level1_stat1_id                         = secondary_minimum_base_physical_damage
 |level1_stat1_value                      = 3
@@ -23,6 +24,7 @@
 |level1_stat3_value                      = 5500
 |level2                                  = True
 |level2_level_requirement                = 66
+|level2_duration                         = 14.5
 |level2_stat_text                        = Deals 161 to 241 Physical Damage<br>Base duration is 14.5 seconds
 |level2_stat1_id                         = secondary_minimum_base_physical_damage
 |level2_stat1_value                      = 161
