--- Skill:DanaiLightningBeamTriggered
+++ Export
@@ -8,21 +8,21 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 16
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 24
-|static_stat3_id                         = active_skill_ignite_damage_+%_final
-|static_stat3_value                      = 400
-|static_stat4_id                         = skill_physical_damage_%_to_convert_to_fire
-|static_stat4_value                      = 50
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_physical_damage
+|static_stat2_value                      = 16
+|static_stat3_id                         = spell_maximum_base_physical_damage
+|static_stat3_value                      = 24
+|static_stat4_id                         = always_ignite
+|static_stat4_value                      = 1
 |static_stat5_id                         = skill_physical_damage_%_to_convert_to_lightning
 |static_stat5_value                      = 50
-|static_stat6_id                         = always_ignite
-|static_stat6_value                      = 1
-|static_stat7_id                         = is_area_damage
-|static_stat7_value                      = 1
-|stat_text                               = Deals 16 to 24 Physical Damage<br>400% more Damage with Ignite<br>50% of Physical Damage Converted to Fire Damage<br>50% of Physical Damage Converted to Lightning Damage<br>Hits always Ignite
+|static_stat6_id                         = skill_physical_damage_%_to_convert_to_fire
+|static_stat6_value                      = 50
+|static_stat7_id                         = active_skill_ignite_damage_+%_final
+|static_stat7_value                      = 400
+|stat_text                               = Deals 16 to 24 Physical Damage<br>Hits always Ignite<br>50% of Physical Damage Converted to Lightning Damage<br>50% of Physical Damage Converted to Fire Damage<br>400% more Damage with Ignite
 |level1                                  = True
 |level1_level_requirement                = 0
 }}