--- Skill:SynthesisGuardian2FrontalBlast
+++ Export
@@ -7,17 +7,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_ignite_damage_+%_final
-|static_stat1_value                      = 100
-|static_stat2_id                         = skill_physical_damage_%_to_convert_to_fire
-|static_stat2_value                      = 100
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = always_ignite
+|static_stat2_value                      = 1
 |static_stat3_id                         = active_skill_damage_+%_final
 |static_stat3_value                      = 800
-|static_stat4_id                         = always_ignite
-|static_stat4_value                      = 1
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
-|stat_text                               = 100% more Damage with Ignite<br>100% of Physical Damage Converted to Fire Damage<br>Deals 800% more Damage<br>Hits always Ignite
+|static_stat4_id                         = skill_physical_damage_%_to_convert_to_fire
+|static_stat4_value                      = 100
+|static_stat5_id                         = active_skill_ignite_damage_+%_final
+|static_stat5_value                      = 100
+|stat_text                               = Hits always Ignite<br>Deals 800% more Damage<br>100% of Physical Damage Converted to Fire Damage<br>100% more Damage with Ignite
 |level1                                  = True
 |level1_level_requirement                = 0
 }}