--- Skill:TemplarCultistShieldCharge
+++ Export
@@ -10,21 +10,21 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = shield_charge_extra_distance
-|static_stat1_value                      = 15
-|static_stat2_id                         = maim_on_hit_%
-|static_stat2_value                      = 50
+|static_stat1_id                         = ignores_proximity_shield
+|static_stat1_value                      = 1
+|static_stat2_id                         = shield_charge_extra_distance
+|static_stat2_value                      = 15
 |static_stat3_id                         = attack_speed_+%
 |static_stat3_value                      = 15
-|static_stat4_id                         = shield_charge_damage_+%_maximum
-|static_stat4_value                      = 20
-|static_stat5_id                         = base_movement_velocity_+%
-|static_stat5_value                      = 50
-|static_stat6_id                         = shield_charge_scaling_stun_threshold_reduction_+%_at_maximum_range
+|static_stat4_id                         = shield_charge_scaling_stun_threshold_reduction_+%_at_maximum_range
+|static_stat4_value                      = 50
+|static_stat5_id                         = shield_charge_damage_+%_maximum
+|static_stat5_value                      = 20
+|static_stat6_id                         = base_movement_velocity_+%
 |static_stat6_value                      = 50
-|static_stat7_id                         = ignores_proximity_shield
-|static_stat7_value                      = 1
-|stat_text                               = 50% chance to Maim on Hit<br>15% increased Attack Speed<br>20% more Damage with Hits at Maximum Charge Distance<br>50% increased Movement Speed<br>50% increased Stun Threshold reduction on enemies at Maximum charge distance
+|static_stat7_id                         = maim_on_hit_%
+|static_stat7_value                      = 50
+|stat_text                               = 15% increased Attack Speed<br>50% increased Stun Threshold reduction on enemies at Maximum charge distance<br>20% more Damage with Hits at Maximum Charge Distance<br>50% increased Movement Speed<br>50% chance to Maim on Hit
 |level1                                  = True
 |level1_level_requirement                = 1
 }}