--- Skill:ShakariBurrowOnceStuck
+++ Export
@@ -9,15 +9,15 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = emerge_speed_+%
 |static_stat1_value                      = 100
-|static_stat2_id                         = newshocknova_first_ring_damage_+%_final
-|static_stat2_value                      = 500
-|static_stat3_id                         = active_skill_attack_speed_+%_final
-|static_stat3_value                      = -14
-|static_stat4_id                         = active_skill_damage_+%_final
-|static_stat4_value                      = -80
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
-|stat_text                               = Ring deals 500% more Damage<br>14% less Attack Speed<br>Deals 80% less Damage
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = active_skill_damage_+%_final
+|static_stat3_value                      = -80
+|static_stat4_id                         = active_skill_attack_speed_+%_final
+|static_stat4_value                      = -14
+|static_stat5_id                         = newshocknova_first_ring_damage_+%_final
+|static_stat5_value                      = 500
+|stat_text                               = Deals 80% less Damage<br>14% less Attack Speed<br>Ring deals 500% more Damage
 |level1                                  = True
 |level1_level_requirement                = 1
 }}