--- Skill:IceNovaTriggered
+++ Export
@@ -30,7 +30,7 @@
 |static_stat8_value                      = 118
 |static_stat9_id                         = maximum_added_cold_damage_vs_chilled_enemies
 |static_stat9_value                      = 172
-|stat_text                               = Deals 769 to 1130 Cold Damage<br>+4 to radius<br>20% less Area of Effect when Cast on Frostbolt<br>This Skill is Triggered at the Final Burst Location of Glacial Cascade<br>Can expand from up to 2 Frostbolt Projectiles<br>Adds 118 to 172 Cold Damage against Chilled Enemies
+|stat_text                               = Deals 769 to 1130 Cold Damage<br>+0.4 metres to radius<br>20% less Area of Effect when Cast on Frostbolt<br>This Skill is Triggered at the Final Burst Location of Glacial Cascade<br>Can expand from up to 2 Frostbolt Projectiles<br>Adds 118 to 172 Cold Damage against Chilled Enemies
 |level1                                  = True
 |level1_level_requirement                = 70
 }}