--- Skill:DarkExileShockedGroundRainOfArrows
+++ Export
@@ -12,14 +12,14 @@
 |static_stat1_value                      = -20
 |static_stat2_id                         = animation_effect_variation
 |static_stat2_value                      = 1
-|static_stat3_id                         = rain_of_arrows_place_shocking_ground_duration_ms
-|static_stat3_value                      = 3000
-|static_stat4_id                         = active_skill_damage_+%_final
-|static_stat4_value                      = 15
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
-|static_stat6_id                         = base_is_projectile
-|static_stat6_value                      = 1
+|static_stat3_id                         = base_is_projectile
+|static_stat3_value                      = 1
+|static_stat4_id                         = is_area_damage
+|static_stat4_value                      = 1
+|static_stat5_id                         = rain_of_arrows_place_shocking_ground_duration_ms
+|static_stat5_value                      = 3000
+|static_stat6_id                         = active_skill_damage_+%_final
+|static_stat6_value                      = 15
 |stat_text                               = Deals 15% more Damage
 |level1                                  = True
 |level1_level_requirement                = 73
