--- Skill:LivingBomb
+++ Export
@@ -9,6 +9,7 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 2
 |static_stat1_id                         = spell_minimum_base_physical_damage
 |static_stat1_value                      = 3
 |static_stat2_id                         = spell_maximum_base_physical_damage
