--- Skill:LightningTendrils
+++ Export
@@ -6,7 +6,7 @@
 |required_level                          = 1
 |static_cost_types                       = Mana
 |static_critical_strike_chance           = 6
-|static_damage_effectiveness             = 35
+|static_damage_effectiveness             = 130
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = base_skill_number_of_additional_hits
