--- Skill:ExileFlameDash
+++ Export
@@ -10,19 +10,20 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_fire_damage
-|static_stat1_value                      = 3
-|static_stat2_id                         = spell_maximum_base_fire_damage
-|static_stat2_value                      = 5
-|static_stat3_id                         = base_cast_speed_+%
-|static_stat3_value                      = 70
-|static_stat4_id                         = base_skill_effect_duration
-|static_stat4_value                      = 4000
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
-|static_stat6_id                         = base_fire_damage_to_deal_per_minute
-|static_stat6_value                      = 369
-|stat_text                               = Deals 3 to 5 Fire Damage<br>70% increased Cast Speed<br>Base duration is 4 seconds<br>Deals 6.2 Base Fire Damage per second
+|static_duration                         = 4
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_fire_damage
+|static_stat2_value                      = 3
+|static_stat3_id                         = spell_maximum_base_fire_damage
+|static_stat3_value                      = 5
+|static_stat4_id                         = base_fire_damage_to_deal_per_minute
+|static_stat4_value                      = 369
+|static_stat5_id                         = base_skill_effect_duration
+|static_stat5_value                      = 4000
+|static_stat6_id                         = base_cast_speed_+%
+|static_stat6_value                      = 70
+|stat_text                               = Deals 3 to 5 Fire Damage<br>Deals 6.2 Base Fire Damage per second<br>Base duration is 4 seconds<br>70% increased Cast Speed
 |level1                                  = True
 |level1_level_requirement                = 1
 }}