--- Skill:StrDexMissionThrownWeaponVaal
+++ Export
@@ -10,13 +10,13 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_projectile_speed_+%
-|static_stat1_value                      = -20
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
 |static_stat2_id                         = active_skill_damage_+%_final
 |static_stat2_value                      = -40
-|static_stat3_id                         = base_is_projectile
-|static_stat3_value                      = 1
-|stat_text                               = 20% reduced Projectile Speed<br>Deals 40% less Damage
+|static_stat3_id                         = base_projectile_speed_+%
+|static_stat3_value                      = -20
+|stat_text                               = Deals 40% less Damage<br>20% reduced Projectile Speed
 |level1                                  = True
 |level1_level_requirement                = 1
 }}