--- Skill:SummonArbalists
+++ Export
@@ -11,11 +11,11 @@
 |static_stat1_value                      = 2
 |static_stat2_id                         = cast_on_gain_skill
 |static_stat2_value                      = 1
-|static_stat3_id                         = minions_cannot_taunt_enemies
+|static_stat3_id                         = no_cost
 |static_stat3_value                      = 1
-|static_stat4_id                         = no_cost
+|static_stat4_id                         = minions_cannot_taunt_enemies
 |static_stat4_value                      = 1
-|stat_text                               = +2 to number of Summoned Arbalists<br>This Spell is Triggered when Gained<br>Minions cannot Taunt Enemies<br>This Skill Costs nothing
+|stat_text                               = +2 to number of Summoned Arbalists<br>This Spell is Triggered when Gained<br>This Skill Costs nothing<br>Minions cannot Taunt Enemies
 |level1                                  = True
 |level1_level_requirement                = 70
 }}