--- Skill:BetrayalVaganWhirlingCharge
+++ Export
@@ -13,10 +13,10 @@
 |static_stat1_value                      = -15
 |static_stat2_id                         = combo_attack_first_hit_damage_+%_final
 |static_stat2_value                      = 100
-|static_stat3_id                         = active_skill_attack_speed_+%_final
-|static_stat3_value                      = 50
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = active_skill_attack_speed_+%_final
+|static_stat4_value                      = 50
 |stat_text                               = 50% more Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 1
