--- Skill:AxisCasterBossLightningOrbTriggered
+++ Export
@@ -9,14 +9,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage
-|static_stat1_value                      = 84
-|static_stat2_id                         = spell_maximum_base_lightning_damage
-|static_stat2_value                      = 127
-|static_stat3_id                         = base_skill_effect_duration
-|static_stat3_value                      = 6500
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
+|static_duration                         = 6.5
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_lightning_damage
+|static_stat2_value                      = 84
+|static_stat3_id                         = spell_maximum_base_lightning_damage
+|static_stat3_value                      = 127
+|static_stat4_id                         = base_skill_effect_duration
+|static_stat4_value                      = 6500
 |stat_text                               = Deals 84 to 127 Lightning Damage<br>Base duration is 6.5 seconds
 |level1                                  = True
 |level1_level_requirement                = 68
