--- Skill:RunicCircleSearingBond
+++ Export
@@ -9,21 +9,21 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_fire_damage_to_deal_per_minute
-|static_stat1_value                      = 92
-|static_stat2_id                         = number_of_totems_summoned_in_formation
-|static_stat2_value                      = 5
-|static_stat3_id                         = base_active_skill_totem_level
-|static_stat3_value                      = 74
-|static_stat4_id                         = base_totem_range
-|static_stat4_value                      = 100
+|static_stat1_id                         = base_active_skill_totem_level
+|static_stat1_value                      = 74
+|static_stat2_id                         = base_skill_is_totemified
+|static_stat2_value                      = 1
+|static_stat3_id                         = base_totem_range
+|static_stat3_value                      = 100
+|static_stat4_id                         = number_of_totems_summoned_in_formation
+|static_stat4_value                      = 5
 |static_stat5_id                         = base_totem_duration
 |static_stat5_value                      = 20000
-|static_stat6_id                         = is_totem
-|static_stat6_value                      = 1
-|static_stat7_id                         = base_skill_is_totemified
+|static_stat6_id                         = base_fire_damage_to_deal_per_minute
+|static_stat6_value                      = 92
+|static_stat7_id                         = is_totem
 |static_stat7_value                      = 1
-|stat_text                               = Deals 1.5 Base Fire Damage per second<br>Totem lasts 20 seconds<br>Summons a Totem which uses this Skill
+|stat_text                               = Totem lasts 20 seconds<br>Deals 1.5 Base Fire Damage per second<br>Summons a Totem which uses this Skill
 |level1                                  = True
 |level1_level_requirement                = 4
 }}