--- Skill:SkeletonTemporalChains
+++ Export
@@ -9,17 +9,18 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 4
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = 9
-|static_stat2_id                         = temporal_chains_action_speed_+%_vs_rare_or_unique_final
-|static_stat2_value                      = -10
-|static_stat3_id                         = buff_time_passed_+%_other_than_temporal_chains
-|static_stat3_value                      = -40
-|static_stat4_id                         = temporal_chains_action_speed_+%_final
-|static_stat4_value                      = -20
-|static_stat5_id                         = base_skill_effect_duration
-|static_stat5_value                      = 4000
-|stat_text                               = Cursed Rare and Unique Enemies have 10% less Action Speed<br>Other effects on Cursed enemies expire 40% slower<br>Cursed Normal and Magic Enemies have 20% less Action Speed<br>Base duration is 4 seconds
+|static_stat2_id                         = base_skill_effect_duration
+|static_stat2_value                      = 4000
+|static_stat3_id                         = temporal_chains_action_speed_+%_final
+|static_stat3_value                      = -20
+|static_stat4_id                         = buff_time_passed_+%_other_than_temporal_chains
+|static_stat4_value                      = -40
+|static_stat5_id                         = temporal_chains_action_speed_+%_vs_rare_or_unique_final
+|static_stat5_value                      = -10
+|stat_text                               = Base duration is 4 seconds<br>Cursed Normal and Magic Enemies have 20% less Action Speed<br>Other effects on Cursed enemies expire 40% slower<br>Cursed Rare and Unique Enemies have 10% less Action Speed
 |level1                                  = True
 |level1_level_requirement                = 0
 }}