--- Skill:DawnFireBeamDivine
+++ Export
@@ -9,15 +9,16 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 1.5
 |static_stat1_id                         = base_fire_damage_to_deal_per_minute
 |static_stat1_value                      = 345
-|static_stat2_id                         = base_fire_damage_resistance_%
-|static_stat2_value                      = -25
+|static_stat2_id                         = base_skill_effect_duration
+|static_stat2_value                      = 1500
 |static_stat3_id                         = fire_beam_additional_stack_damage_+%_final
 |static_stat3_value                      = -40
-|static_stat4_id                         = base_skill_effect_duration
-|static_stat4_value                      = 1500
-|stat_text                               = Deals 5.8 Base Fire Damage per second<br>Fire Exposure applies -25% to Fire Resistance<br>Additional Debuff stages add 60% of Damage<br>Base duration is 1.5 seconds
+|static_stat4_id                         = base_fire_damage_resistance_%
+|static_stat4_value                      = -25
+|stat_text                               = Deals 5.8 Base Fire Damage per second<br>Base duration is 1.5 seconds<br>Additional Debuff stages add 60% of Damage<br>Fire Exposure applies -25% to Fire Resistance
 |level1                                  = True
 |level1_level_requirement                = 1
 }}