--- Skill:ProtoVaalMapBossBoulderHit
+++ Export
@@ -7,21 +7,19 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = geometry_attack_%_target_es_to_deal_as_additional_damage
-|static_stat1_value                      = 8
-|static_stat2_id                         = geometry_attack_%_target_life_to_deal_as_additional_damage
-|static_stat2_value                      = 13
-|static_stat3_id                         = reduce_enemy_dodge_%
-|static_stat3_value                      = 100
-|static_stat4_id                         = global_reduce_enemy_block_%
-|static_stat4_value                      = 100
+|static_stat1_id                         = cannot_be_blocked_or_dodged_or_suppressed
+|static_stat1_value                      = 1
+|static_stat2_id                         = geometry_attack_%_target_es_to_deal_as_additional_damage
+|static_stat2_value                      = 8
+|static_stat3_id                         = geometry_attack_%_target_life_to_deal_as_additional_damage
+|static_stat3_value                      = 13
+|static_stat4_id                         = is_area_damage
+|static_stat4_value                      = 1
 |static_stat5_id                         = global_knockback
 |static_stat5_value                      = 1
 |static_stat6_id                         = global_always_hit
 |static_stat6_value                      = 1
-|static_stat7_id                         = is_area_damage
-|static_stat7_value                      = 1
-|stat_text                               = Enemies have 100% reduced chance to Dodge Hits from this Skill<br>Enemy Block Chance reduced by 100% against this Skill<br>Knocks Enemies Back on Hit<br>Can't be Evaded
+|stat_text                               = Knocks Enemies Back on Hit<br>Can't be Evaded
 |level1                                  = True
 |level1_level_requirement                = 1
 }}