--- Skill:GSHarvestIguanaSpikeRain
+++ Export
@@ -8,15 +8,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_cold_damage
-|static_stat1_value                      = 4
-|static_stat2_id                         = spell_maximum_base_cold_damage
-|static_stat2_value                      = 6
-|static_stat3_id                         = spell_maximum_action_distance_+%
-|static_stat3_value                      = -20
-|static_stat4_id                         = cannot_stun
-|static_stat4_value                      = 1
-|static_stat5_id                         = is_area_damage
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_maximum_action_distance_+%
+|static_stat2_value                      = -20
+|static_stat3_id                         = spell_minimum_base_cold_damage
+|static_stat3_value                      = 4
+|static_stat4_id                         = spell_maximum_base_cold_damage
+|static_stat4_value                      = 6
+|static_stat5_id                         = cannot_stun
 |static_stat5_value                      = 1
 |stat_text                               = Deals 4 to 6 Cold Damage<br>Your Hits cannot Stun Enemies
 |level1                                  = True
