--- Skill:GSHarvestSandLeaperSnipe
+++ Export
@@ -10,14 +10,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage
-|static_stat1_value                      = 12
-|static_stat2_id                         = spell_maximum_base_lightning_damage
-|static_stat2_value                      = 22
-|static_stat3_id                         = spell_maximum_action_distance_+%
-|static_stat3_value                      = -40
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_maximum_action_distance_+%
+|static_stat2_value                      = -40
+|static_stat3_id                         = spell_minimum_base_lightning_damage
+|static_stat3_value                      = 12
+|static_stat4_id                         = spell_maximum_base_lightning_damage
+|static_stat4_value                      = 22
 |stat_text                               = Deals 12 to 22 Lightning Damage
 |level1                                  = True
 |level1_level_requirement                = 0
