--- Skill:ParasitePoisonProjectileSingle
+++ Export
@@ -7,11 +7,11 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 70
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_projectile_variation
-|static_stat1_value                      = 27
-|static_stat2_id                         = global_poison_on_hit
-|static_stat2_value                      = 1
-|static_stat3_id                         = base_is_projectile
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_projectile_variation
+|static_stat2_value                      = 27
+|static_stat3_id                         = global_poison_on_hit
 |static_stat3_value                      = 1
 |stat_text                               = Poisons Enemies on Hit
 |level1                                  = True
