--- Skill:AtlasBossAcceleratingProjectilesFragment
+++ Export
@@ -8,13 +8,13 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = always_pierce
+|static_stat1_id                         = base_is_projectile
 |static_stat1_value                      = 1
-|static_stat2_id                         = base_is_projectile
+|static_stat2_id                         = projectile_uses_contact_position
 |static_stat2_value                      = 1
-|static_stat3_id                         = projectile_uses_contact_position
+|static_stat3_id                         = always_pierce
 |static_stat3_value                      = 1
-|stat_text                               = Deals (5639-5984) to (8459-8976) Cold Damage<br>Penetrates (25-40)% Cold Resistance<br>Fires (0-2) additional Projectiles<br>Projectiles Pierce all Targets
+|stat_text                               = Deals (5639-5984) to (8459-8976) Cold Damage<br>Projectiles Pierce all Targets<br>Fires (0-2) additional Projectiles<br>Penetrates (25-40)% Cold Resistance
 |level1                                  = True
 |level1_level_requirement                = 84
 |level1_stat_text                        = Deals 5639 to 8459 Cold Damage<br>Penetrates 25% Cold Resistance
@@ -26,13 +26,13 @@
 |level1_stat3_value                      = 25
 |level2                                  = True
 |level2_level_requirement                = 85
-|level2_stat_text                        = Deals 5984 to 8976 Cold Damage<br>Penetrates 40% Cold Resistance<br>Fires 2 additional Projectiles
+|level2_stat_text                        = Deals 5984 to 8976 Cold Damage<br>Fires 2 additional Projectiles<br>Penetrates 40% Cold Resistance
 |level2_stat1_id                         = spell_minimum_base_cold_damage
 |level2_stat1_value                      = 5984
 |level2_stat2_id                         = spell_maximum_base_cold_damage
 |level2_stat2_value                      = 8976
-|level2_stat3_id                         = base_reduce_enemy_cold_resistance_%
-|level2_stat3_value                      = 40
-|level2_stat4_id                         = number_of_additional_projectiles
-|level2_stat4_value                      = 2
+|level2_stat3_id                         = number_of_additional_projectiles
+|level2_stat3_value                      = 2
+|level2_stat4_id                         = base_reduce_enemy_cold_resistance_%
+|level2_stat4_value                      = 40
 }}