--- Skill:GladiatorBossPunctureTrap
+++ Export
@@ -28,6 +28,8 @@
 |stat_text                               = Trap lasts 30 seconds<br>Base duration is 5.3 seconds<br>Causes Bleeding
 |level1                                  = True
 |level1_level_requirement                = 38
+|level1_damage_effectiveness             = 100
+|level1_damage_multiplier                = 100
 |level2                                  = True
 |level2_level_requirement                = 56
 |level2_damage_effectiveness             = 140
