--- Skill:SupportAttackAndCastSpeedFinalEssence1
+++ Export
@@ -1,6 +1,7 @@
 {{Skill
 |skill_id                                = SupportAttackAndCastSpeedFinalEssence1
 |required_level                          = 0
+|static_cost_multiplier                  = 100
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
