--- Skill:IvoryTempleMilitryBoss2Melee
+++ Export
@@ -7,12 +7,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = skill_physical_damage_%_to_convert_to_lightning
-|static_stat1_value                      = 50
-|static_stat2_id                         = skill_can_fire_arrows
+|static_stat1_id                         = skill_can_fire_arrows
+|static_stat1_value                      = 1
+|static_stat2_id                         = skill_can_fire_wand_projectiles
 |static_stat2_value                      = 1
-|static_stat3_id                         = skill_can_fire_wand_projectiles
-|static_stat3_value                      = 1
+|static_stat3_id                         = skill_physical_damage_%_to_convert_to_lightning
+|static_stat3_value                      = 50
 |stat_text                               = 50% of Physical Damage Converted to Lightning Damage
 |level1                                  = True
 |level1_level_requirement                = 1
