--- Skill:MonsterReaveInvasion
+++ Export
@@ -10,17 +10,17 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = 59
-|static_stat2_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat2_value                      = -25
-|static_stat3_id                         = active_skill_damage_+%_final
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
 |static_stat3_value                      = -25
-|static_stat4_id                         = physical_damage_+%
-|static_stat4_value                      = 12
-|static_stat5_id                         = reave_area_of_effect_+%_final_per_stage
-|static_stat5_value                      = 80
-|static_stat6_id                         = is_area_damage
-|static_stat6_value                      = 1
-|stat_text                               = Skills Repeat an additional (2-8) Times<br>(100-280)% increased Attack Speed<br>Deals 25% less Damage<br>12% increased Physical Damage<br>80% more Area of Effect for each stage
+|static_stat4_id                         = active_skill_damage_+%_final
+|static_stat4_value                      = -25
+|static_stat5_id                         = physical_damage_+%
+|static_stat5_value                      = 12
+|static_stat6_id                         = reave_area_of_effect_+%_final_per_stage
+|static_stat6_value                      = 80
+|stat_text                               = Skills Repeat an additional (2-8) Times<br>Deals 25% less Damage<br>(100-280)% increased Attack Speed<br>12% increased Physical Damage<br>80% more Area of Effect for each stage
 |level1                                  = True
 |level1_level_requirement                = 3
 |level1_stat_text                        = Skills Repeat an additional 2 Times<br>100% increased Attack Speed
