--- Skill:WbFlammability
+++ Export
@@ -9,13 +9,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 5
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = 9
-|static_stat2_id                         = chance_to_be_ignited_%
-|static_stat2_value                      = 25
-|static_stat3_id                         = base_skill_effect_duration
-|static_stat3_value                      = 5000
-|stat_text                               = Cursed enemies have -(15-18)% to Fire Resistance<br>Hits against Cursed Enemies have +25% chance to Ignite<br>Base duration is 5 seconds
+|static_stat2_id                         = base_skill_effect_duration
+|static_stat2_value                      = 5000
+|static_stat3_id                         = chance_to_be_ignited_%
+|static_stat3_value                      = 25
+|stat_text                               = Base duration is 5 seconds<br>Cursed enemies have -(15-18)% to Fire Resistance<br>Hits against Cursed Enemies have +25% chance to Ignite
 |level1                                  = True
 |level1_level_requirement                = 10
 |level1_stat_text                        = Cursed enemies have -15% to Fire Resistance
