--- Skill:ArchnemesisBubbleShieldPlayer
+++ Export
@@ -7,16 +7,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_skill_effect_duration
-|static_stat1_value                      = 2000
-|static_stat2_id                         = quick_guard_damage_absorbed_%
-|static_stat2_value                      = 90
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
-|stat_text                               = Deals (16-11501) to (23-17252) Fire Damage<br>0% of Damage from Hits is taken from the Buff before Life or Energy Shield<br>Buff can take (50-6000) Damage<br>Base duration is 2 seconds<br>90% of Damage from Hits is taken from the Buff before Life or Energy Shield<br>Buff can take 0 Damage
+|static_duration                         = 2
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = base_skill_effect_duration
+|static_stat2_value                      = 2000
+|static_stat3_id                         = quick_guard_damage_absorbed_%
+|static_stat3_value                      = 90
+|stat_text                               = Base duration is 2 seconds<br>Deals (16-11501) to (23-17252) Fire Damage<br>90% of Damage from Hits is taken from the Buff before Life or Energy Shield<br>Buff can take (50-6000) Damage
 |level1                                  = True
 |level1_level_requirement                = 1
-|level1_stat_text                        = Deals 16 to 23 Fire Damage<br>0% of Damage from Hits is taken from the Buff before Life or Energy Shield<br>Buff can take 50 Damage
+|level1_stat_text                        = Deals 16 to 23 Fire Damage<br>90% of Damage from Hits is taken from the Buff before Life or Energy Shield<br>Buff can take 50 Damage
 |level1_stat1_id                         = spell_minimum_base_fire_damage
 |level1_stat1_value                      = 16
 |level1_stat2_id                         = spell_maximum_base_fire_damage
@@ -25,7 +26,7 @@
 |level1_stat3_value                      = 50
 |level2                                  = True
 |level2_level_requirement                = 45
-|level2_stat_text                        = Deals 833 to 1249 Fire Damage<br>0% of Damage from Hits is taken from the Buff before Life or Energy Shield<br>Buff can take 1800 Damage
+|level2_stat_text                        = Deals 833 to 1249 Fire Damage<br>90% of Damage from Hits is taken from the Buff before Life or Energy Shield<br>Buff can take 1800 Damage
 |level2_stat1_id                         = spell_minimum_base_fire_damage
 |level2_stat1_value                      = 833
 |level2_stat2_id                         = spell_maximum_base_fire_damage
@@ -34,7 +35,7 @@
 |level2_stat3_value                      = 1800
 |level3                                  = True
 |level3_level_requirement                = 46
-|level3_stat_text                        = Deals 894 to 1342 Fire Damage<br>0% of Damage from Hits is taken from the Buff before Life or Energy Shield<br>Buff can take 1900 Damage
+|level3_stat_text                        = Deals 894 to 1342 Fire Damage<br>90% of Damage from Hits is taken from the Buff before Life or Energy Shield<br>Buff can take 1900 Damage
 |level3_stat1_id                         = spell_minimum_base_fire_damage
 |level3_stat1_value                      = 894
 |level3_stat2_id                         = spell_maximum_base_fire_damage
@@ -43,7 +44,7 @@
 |level3_stat3_value                      = 1900
 |level4                                  = True
 |level4_level_requirement                = 67
-|level4_stat_text                        = Deals 3790 to 5684 Fire Damage<br>0% of Damage from Hits is taken from the Buff before Life or Energy Shield<br>Buff can take 4000 Damage
+|level4_stat_text                        = Deals 3790 to 5684 Fire Damage<br>90% of Damage from Hits is taken from the Buff before Life or Energy Shield<br>Buff can take 4000 Damage
 |level4_stat1_id                         = spell_minimum_base_fire_damage
 |level4_stat1_value                      = 3790
 |level4_stat2_id                         = spell_maximum_base_fire_damage
@@ -52,7 +53,7 @@
 |level4_stat3_value                      = 4000
 |level5                                  = True
 |level5_level_requirement                = 68
-|level5_stat_text                        = Deals 4050 to 6075 Fire Damage<br>0% of Damage from Hits is taken from the Buff before Life or Energy Shield<br>Buff can take 4100 Damage
+|level5_stat_text                        = Deals 4050 to 6075 Fire Damage<br>90% of Damage from Hits is taken from the Buff before Life or Energy Shield<br>Buff can take 4100 Damage
 |level5_stat1_id                         = spell_minimum_base_fire_damage
 |level5_stat1_value                      = 4050
 |level5_stat2_id                         = spell_maximum_base_fire_damage
@@ -61,7 +62,7 @@
 |level5_stat3_value                      = 4100
 |level6                                  = True
 |level6_level_requirement                = 84
-|level6_stat_text                        = Deals 11501 to 17252 Fire Damage<br>0% of Damage from Hits is taken from the Buff before Life or Energy Shield<br>Buff can take 6000 Damage
+|level6_stat_text                        = Deals 11501 to 17252 Fire Damage<br>90% of Damage from Hits is taken from the Buff before Life or Energy Shield<br>Buff can take 6000 Damage
 |level6_stat1_id                         = spell_minimum_base_fire_damage
 |level6_stat1_value                      = 11501
 |level6_stat2_id                         = spell_maximum_base_fire_damage
