--- Skill:ProphecyShrapnelShot
+++ Export
@@ -6,15 +6,15 @@
 |cast_time                               = 1.0
 |required_level                          = 1
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = skill_physical_damage_%_to_convert_to_lightning
-|static_stat1_value                      = 40
-|static_stat2_id                         = base_arrows_always_pierce
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = skill_can_fire_arrows
 |static_stat2_value                      = 1
-|static_stat3_id                         = base_is_projectile
+|static_stat3_id                         = base_arrows_always_pierce
 |static_stat3_value                      = 1
-|static_stat4_id                         = skill_can_fire_arrows
-|static_stat4_value                      = 1
-|stat_text                               = Fires (2-4) additional Projectiles<br>40% of Physical Damage Converted to Lightning Damage<br>Arrows Pierce all Targets
+|static_stat4_id                         = skill_physical_damage_%_to_convert_to_lightning
+|static_stat4_value                      = 40
+|stat_text                               = Arrows Pierce all Targets<br>40% of Physical Damage Converted to Lightning Damage<br>Fires (2-4) additional Projectiles
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_damage_effectiveness             = 65
