--- Skill:TalismanPoisonProjectiles
+++ Export
@@ -9,17 +9,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_reverse_point_blank_damage_-%_at_minimum_range
-|static_stat1_value                      = 30
-|static_stat2_id                         = monster_projectile_variation
-|static_stat2_value                      = 5
-|static_stat3_id                         = number_of_additional_projectiles
-|static_stat3_value                      = 4
-|static_stat4_id                         = base_projectile_speed_+%
-|static_stat4_value                      = -35
+|static_stat1_id                         = monster_projectile_variation
+|static_stat1_value                      = 5
+|static_stat2_id                         = monster_reverse_point_blank_damage_-%_at_minimum_range
+|static_stat2_value                      = 30
+|static_stat3_id                         = base_projectile_speed_+%
+|static_stat3_value                      = -35
+|static_stat4_id                         = number_of_additional_projectiles
+|static_stat4_value                      = 4
 |static_stat5_id                         = global_poison_on_hit
 |static_stat5_value                      = 1
-|stat_text                               = Deals (3-624) to (5-936) Chaos Damage<br>(0-40)% increased Cast Speed<br>Fires 4 additional Projectiles<br>35% reduced Projectile Speed<br>Poisons Enemies on Hit
+|stat_text                               = Deals (3-624) to (5-936) Chaos Damage<br>(0-40)% increased Cast Speed<br>35% reduced Projectile Speed<br>Fires 4 additional Projectiles<br>Poisons Enemies on Hit
 |level1                                  = True
 |level1_level_requirement                = 3
 |level1_stat_text                        = Deals 3 to 5 Chaos Damage
