--- Skill:PerandusSummonChaosGolem
+++ Export
@@ -11,11 +11,11 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = alternate_minion
 |static_stat1_value                      = 1
-|static_stat2_id                         = chaos_golem_grants_additional_physical_damage_reduction_%
-|static_stat2_value                      = 5
-|static_stat3_id                         = base_number_of_golems_allowed
-|static_stat3_value                      = 3
-|stat_text                               = (0-15)% increased Character Size<br>(0-82)% increased Minion Maximum Life<br>Golems grant 5% additional Physical Damage Reduction<br>Maximum 3 Summoned Golems
+|static_stat2_id                         = base_number_of_golems_allowed
+|static_stat2_value                      = 3
+|static_stat3_id                         = chaos_golem_grants_additional_physical_damage_reduction_%
+|static_stat3_value                      = 5
+|stat_text                               = (0-15)% increased Character Size<br>Maximum 3 Summoned Golems<br>(0-82)% increased Minion Maximum Life<br>Golems grant 5% additional Physical Damage Reduction
 |level1                                  = True
 |level1_level_requirement                = 1
 |level2                                  = True
