--- Skill:MeleeDemonFemale
+++ Export
@@ -13,7 +13,7 @@
 |static_stat2_value                      = 1
 |static_stat3_id                         = melee_range_+
 |static_stat3_value                      = 10
-|stat_text                               = +10 to Melee Strike Range
+|stat_text                               = +1 metre to Melee Strike Range
 |level1                                  = True
 |level1_level_requirement                = 1
 }}