--- Skill:SpectralSkullShieldCharge
+++ Export
@@ -10,17 +10,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_damage_+%_final
-|static_stat1_value                      = -50
-|static_stat2_id                         = shield_charge_damage_+%_maximum
-|static_stat2_value                      = 300
-|static_stat3_id                         = base_movement_velocity_+%
-|static_stat3_value                      = 500
-|static_stat4_id                         = shield_charge_stun_duration_+%_maximum
-|static_stat4_value                      = 200
-|static_stat5_id                         = ignores_proximity_shield
-|static_stat5_value                      = 1
-|stat_text                               = Deals 50% less Damage<br>300% more Damage with Hits at Maximum Charge Distance<br>500% increased Movement Speed<br>200% increased Stun Duration on enemies at Maximum charge distance
+|static_stat1_id                         = ignores_proximity_shield
+|static_stat1_value                      = 1
+|static_stat2_id                         = active_skill_damage_+%_final
+|static_stat2_value                      = -50
+|static_stat3_id                         = shield_charge_stun_duration_+%_maximum
+|static_stat3_value                      = 200
+|static_stat4_id                         = shield_charge_damage_+%_maximum
+|static_stat4_value                      = 300
+|static_stat5_id                         = base_movement_velocity_+%
+|static_stat5_value                      = 500
+|stat_text                               = Deals 50% less Damage<br>200% increased Stun Duration on enemies at Maximum charge distance<br>300% more Damage with Hits at Maximum Charge Distance<br>500% increased Movement Speed
 |level1                                  = True
 |level1_level_requirement                = 1
 }}