--- Skill:BlightSkeletonRainOfArrows
+++ Export
@@ -8,16 +8,16 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_number_of_additional_arrows
-|static_stat1_value                      = 10
-|static_stat2_id                         = base_is_projectile
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = cannot_pierce
 |static_stat2_value                      = 1
 |static_stat3_id                         = is_area_damage
 |static_stat3_value                      = 1
 |static_stat4_id                         = skill_can_fire_arrows
 |static_stat4_value                      = 1
-|static_stat5_id                         = cannot_pierce
-|static_stat5_value                      = 1
+|static_stat5_id                         = base_number_of_additional_arrows
+|static_stat5_value                      = 10
 |stat_text                               = Fires 10 additional Arrows
 |level1                                  = True
 |level1_level_requirement                = 1
