--- Skill:ElderProjectileUnstableSpellcast
+++ Export
@@ -8,25 +8,25 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_cold_damage
-|static_stat1_value                      = 5006
-|static_stat2_id                         = spell_maximum_base_cold_damage
-|static_stat2_value                      = 7509
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_projectile_variation
+|static_stat2_value                      = 46
 |static_stat3_id                         = skill_visual_scale_+%
 |static_stat3_value                      = -30
-|static_stat4_id                         = monster_projectile_variation
-|static_stat4_value                      = 46
-|static_stat5_id                         = number_of_additional_projectiles
-|static_stat5_value                      = 10
-|static_stat6_id                         = base_projectile_speed_+%
-|static_stat6_value                      = -25
+|static_stat4_id                         = spell_minimum_base_cold_damage
+|static_stat4_value                      = 5006
+|static_stat5_id                         = spell_maximum_base_cold_damage
+|static_stat5_value                      = 7509
+|static_stat6_id                         = always_pierce
+|static_stat6_value                      = 1
 |static_stat7_id                         = projectiles_nova
 |static_stat7_value                      = 1
-|static_stat8_id                         = always_pierce
-|static_stat8_value                      = 1
-|static_stat9_id                         = base_is_projectile
-|static_stat9_value                      = 1
-|stat_text                               = Deals 5006 to 7509 Cold Damage<br>Fires 10 additional Projectiles<br>25% reduced Projectile Speed<br>Fires Projectiles in a circle<br>Projectiles Pierce all Targets
+|static_stat8_id                         = base_projectile_speed_+%
+|static_stat8_value                      = -25
+|static_stat9_id                         = number_of_additional_projectiles
+|static_stat9_value                      = 10
+|stat_text                               = Deals 5006 to 7509 Cold Damage<br>Projectiles Pierce all Targets<br>Fires Projectiles in a circle<br>25% reduced Projectile Speed<br>Fires 10 additional Projectiles
 |level1                                  = True
 |level1_level_requirement                = 82
 }}