--- Skill:LegionEternalEmpireMeleeCleave
+++ Export
@@ -10,11 +10,11 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 130
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_attack_speed_+%_final
-|static_stat1_value                      = -44
-|static_stat2_id                         = global_maim_on_hit
-|static_stat2_value                      = 1
-|static_stat3_id                         = is_area_damage
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = active_skill_attack_speed_+%_final
+|static_stat2_value                      = -44
+|static_stat3_id                         = global_maim_on_hit
 |static_stat3_value                      = 1
 |stat_text                               = 44% less Attack Speed<br>Maim on Hit
 |level1                                  = True
