--- Skill:AfflictionDemonMeleeRight
+++ Export
@@ -9,17 +9,17 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = base_knockback_speed_+%
 |static_stat1_value                      = 150
-|static_stat2_id                         = active_skill_damage_+%_final
-|static_stat2_value                      = -20
-|static_stat3_id                         = skill_physical_damage_%_to_convert_to_chaos
-|static_stat3_value                      = 25
+|static_stat2_id                         = determine_knockback_direction_from_melee_pattern
+|static_stat2_value                      = 1
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
 |static_stat4_id                         = global_knockback
 |static_stat4_value                      = 1
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
-|static_stat6_id                         = determine_knockback_direction_from_melee_pattern
-|static_stat6_value                      = 1
-|stat_text                               = Deals 20% less Damage<br>25% of Physical Damage Converted to Chaos Damage<br>Knocks Enemies Back on Hit
+|static_stat5_id                         = active_skill_damage_+%_final
+|static_stat5_value                      = -20
+|static_stat6_id                         = skill_physical_damage_%_to_convert_to_chaos
+|static_stat6_value                      = 25
+|stat_text                               = Knocks Enemies Back on Hit<br>Deals 20% less Damage<br>25% of Physical Damage Converted to Chaos Damage
 |level1                                  = True
 |level1_level_requirement                = 1
 }}