--- Skill:GladiatorBossCyclone
+++ Export
@@ -17,21 +17,33 @@
 |level1                                  = True
 |level1_level_requirement                = 4
 |level1_cost_amounts                     = 24
+|level1_damage_effectiveness             = 100
+|level1_damage_multiplier                = 100
 |level2                                  = True
 |level2_level_requirement                = 9
 |level2_cost_amounts                     = 25
+|level2_damage_effectiveness             = 100
+|level2_damage_multiplier                = 100
 |level3                                  = True
 |level3_level_requirement                = 13
 |level3_cost_amounts                     = 26
+|level3_damage_effectiveness             = 100
+|level3_damage_multiplier                = 100
 |level4                                  = True
 |level4_level_requirement                = 18
 |level4_cost_amounts                     = 27
+|level4_damage_effectiveness             = 100
+|level4_damage_multiplier                = 100
 |level5                                  = True
 |level5_level_requirement                = 24
 |level5_cost_amounts                     = 28
+|level5_damage_effectiveness             = 100
+|level5_damage_multiplier                = 100
 |level6                                  = True
 |level6_level_requirement                = 27
 |level6_cost_amounts                     = 28
+|level6_damage_effectiveness             = 100
+|level6_damage_multiplier                = 100
 |level6_stat_text                        = 2% increased Attack Speed<br>2% increased Physical Damage
 |level6_stat1_id                         = attack_speed_+%
 |level6_stat1_value                      = 2
@@ -40,6 +52,8 @@
 |level7                                  = True
 |level7_level_requirement                = 30
 |level7_cost_amounts                     = 28
+|level7_damage_effectiveness             = 100
+|level7_damage_multiplier                = 100
 |level7_stat_text                        = 4% increased Attack Speed<br>5% increased Physical Damage
 |level7_stat1_id                         = attack_speed_+%
 |level7_stat1_value                      = 4
@@ -48,6 +62,8 @@
 |level8                                  = True
 |level8_level_requirement                = 33
 |level8_cost_amounts                     = 29
+|level8_damage_effectiveness             = 100
+|level8_damage_multiplier                = 100
 |level8_stat_text                        = 6% increased Attack Speed<br>7% increased Physical Damage
 |level8_stat1_id                         = attack_speed_+%
 |level8_stat1_value                      = 6
@@ -56,6 +72,8 @@
 |level9                                  = True
 |level9_level_requirement                = 35
 |level9_cost_amounts                     = 29
+|level9_damage_effectiveness             = 100
+|level9_damage_multiplier                = 100
 |level9_stat_text                        = 8% increased Attack Speed<br>9% increased Physical Damage
 |level9_stat1_id                         = attack_speed_+%
 |level9_stat1_value                      = 8
@@ -64,6 +82,8 @@
 |level10                                 = True
 |level10_level_requirement               = 38
 |level10_cost_amounts                    = 29
+|level10_damage_effectiveness            = 100
+|level10_damage_multiplier               = 100
 |level10_stat_text                       = 10% increased Attack Speed<br>12% increased Physical Damage
 |level10_stat1_id                        = attack_speed_+%
 |level10_stat1_value                     = 10
@@ -72,6 +92,8 @@
 |level11                                 = True
 |level11_level_requirement               = 40
 |level11_cost_amounts                    = 30
+|level11_damage_effectiveness            = 100
+|level11_damage_multiplier               = 100
 |level11_stat_text                       = 12% increased Attack Speed<br>14% increased Physical Damage
 |level11_stat1_id                        = attack_speed_+%
 |level11_stat1_value                     = 12
@@ -80,6 +102,8 @@
 |level12                                 = True
 |level12_level_requirement               = 43
 |level12_cost_amounts                    = 30
+|level12_damage_effectiveness            = 100
+|level12_damage_multiplier               = 100
 |level12_stat_text                       = 14% increased Attack Speed<br>16% increased Physical Damage
 |level12_stat1_id                        = attack_speed_+%
 |level12_stat1_value                     = 14
@@ -88,6 +112,8 @@
 |level13                                 = True
 |level13_level_requirement               = 46
 |level13_cost_amounts                    = 30
+|level13_damage_effectiveness            = 100
+|level13_damage_multiplier               = 100
 |level13_stat_text                       = 16% increased Attack Speed<br>19% increased Physical Damage
 |level13_stat1_id                        = attack_speed_+%
 |level13_stat1_value                     = 16
@@ -96,6 +122,8 @@
 |level14                                 = True
 |level14_level_requirement               = 48
 |level14_cost_amounts                    = 30
+|level14_damage_effectiveness            = 100
+|level14_damage_multiplier               = 100
 |level14_stat_text                       = 18% increased Attack Speed<br>21% increased Physical Damage
 |level14_stat1_id                        = attack_speed_+%
 |level14_stat1_value                     = 18
@@ -104,6 +132,8 @@
 |level15                                 = True
 |level15_level_requirement               = 50
 |level15_cost_amounts                    = 30
+|level15_damage_effectiveness            = 100
+|level15_damage_multiplier               = 100
 |level15_stat_text                       = 20% increased Attack Speed<br>24% increased Physical Damage
 |level15_stat1_id                        = attack_speed_+%
 |level15_stat1_value                     = 20
@@ -112,6 +142,8 @@
 |level16                                 = True
 |level16_level_requirement               = 52
 |level16_cost_amounts                    = 30
+|level16_damage_effectiveness            = 100
+|level16_damage_multiplier               = 100
 |level16_stat_text                       = 22% increased Attack Speed<br>26% increased Physical Damage
 |level16_stat1_id                        = attack_speed_+%
 |level16_stat1_value                     = 22
@@ -120,6 +152,8 @@
 |level17                                 = True
 |level17_level_requirement               = 54
 |level17_cost_amounts                    = 31
+|level17_damage_effectiveness            = 100
+|level17_damage_multiplier               = 100
 |level17_stat_text                       = 24% increased Attack Speed<br>28% increased Physical Damage
 |level17_stat1_id                        = attack_speed_+%
 |level17_stat1_value                     = 24
