--- Skill:HiddenBlade
+++ Export
@@ -5,17 +5,16 @@
 |skill_id                                = HiddenBlade
 |cast_time                               = 1.0
 |required_level                          = 70
-|static_stored_uses                      = 1
 |static_damage_effectiveness             = 250
 |static_damage_multiplier                = 250
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = skill_has_trigger_from_unique_item
+|static_stat1_id                         = base_is_projectile
 |static_stat1_value                      = 1
 |static_stat2_id                         = projectiles_nova
 |static_stat2_value                      = 1
-|static_stat3_id                         = base_is_projectile
+|static_stat3_id                         = skill_has_trigger_from_unique_item
 |static_stat3_value                      = 1
-|stat_text                               = This Skill is Triggered by a Unique Item<br>Fires Projectiles in a circle
+|stat_text                               = Fires Projectiles in a circle<br>This Skill is Triggered by a Unique Item
 |level1                                  = True
 |level1_level_requirement                = 70
 }}