--- Skill:GladiatorBossFlameblastMetamorph
+++ Export
@@ -10,13 +10,13 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = 20
-|static_stat2_id                         = flameblast_ailment_damage_+%_final_per_stack
-|static_stat2_value                      = 90
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
 |static_stat3_id                         = charged_blast_spell_damage_+%_final_per_stack
 |static_stat3_value                      = 90
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
-|stat_text                               = Deals (6-332) to (9-498) Fire Damage<br>90% more Damage with Ailments for each stage<br>90% more Spell Damage for each stage
+|static_stat4_id                         = flameblast_ailment_damage_+%_final_per_stack
+|static_stat4_value                      = 90
+|stat_text                               = Deals (6-332) to (9-498) Fire Damage<br>90% more Spell Damage for each stage<br>90% more Damage with Ailments for each stage
 |level1                                  = True
 |level1_level_requirement                = 3
 |level1_stat_text                        = Deals 6 to 9 Fire Damage
