--- Skill:GSAncestralMultiHookStun
+++ Export
@@ -0,0 +1,23 @@
+{{Skill
+|gem_description                         = 
+|active_skill_name                       = 
+|skill_id                                = GSAncestralMultiHookStun
+|cast_time                               = 1.0
+|required_level                          = 0
+|static_damage_effectiveness             = 100
+|static_damage_multiplier                = 100
+|static_attack_speed_multiplier          = 100
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_physical_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = spell_maximum_base_physical_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = base_stun_duration_+%
+|static_stat4_value                      = 200
+|static_stat5_id                         = always_stun
+|static_stat5_value                      = 1
+|stat_text                               = Deals 1 to 1 Physical Damage<br>200% increased Stun Duration on enemies<br>Stuns Enemies
+|level1                                  = True
+|level1_level_requirement                = 0
+}}