--- Skill:HeistTurretDamageMortarCultist
+++ Export
@@ -8,20 +8,20 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 7
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 10
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = projectile_uses_contact_position
+|static_stat2_value                      = 1
 |static_stat3_id                         = spell_maximum_action_distance_+%
 |static_stat3_value                      = -33
-|static_stat4_id                         = skill_physical_damage_%_to_convert_to_lightning
-|static_stat4_value                      = 50
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
-|static_stat6_id                         = use_scaled_contact_offset
-|static_stat6_value                      = 1
-|static_stat7_id                         = projectile_uses_contact_position
-|static_stat7_value                      = 1
+|static_stat4_id                         = use_scaled_contact_offset
+|static_stat4_value                      = 1
+|static_stat5_id                         = spell_minimum_base_physical_damage
+|static_stat5_value                      = 7
+|static_stat6_id                         = spell_maximum_base_physical_damage
+|static_stat6_value                      = 10
+|static_stat7_id                         = skill_physical_damage_%_to_convert_to_lightning
+|static_stat7_value                      = 50
 |stat_text                               = Deals 7 to 10 Physical Damage<br>50% of Physical Damage Converted to Lightning Damage
 |level1                                  = True
 |level1_level_requirement                = 0
