--- Skill:UnholyTrioTriggerCascade2
+++ Export
@@ -9,16 +9,16 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = trigger_cascade_angle_between_arms
-|static_stat1_value                      = 90
-|static_stat2_id                         = trigger_cascade_number_of_arms
-|static_stat2_value                      = 3
-|static_stat3_id                         = trigger_cascade_ms_between_spikes
-|static_stat3_value                      = 250
-|static_stat4_id                         = trigger_cascade_angle
-|static_stat4_value                      = 5
-|static_stat5_id                         = trigger_cascade_B
-|static_stat5_value                      = 100
+|static_stat1_id                         = trigger_cascade_B
+|static_stat1_value                      = 100
+|static_stat2_id                         = trigger_cascade_angle
+|static_stat2_value                      = 5
+|static_stat3_id                         = trigger_cascade_angle_between_arms
+|static_stat3_value                      = 90
+|static_stat4_id                         = trigger_cascade_ms_between_spikes
+|static_stat4_value                      = 250
+|static_stat5_id                         = trigger_cascade_number_of_arms
+|static_stat5_value                      = 3
 |static_stat6_id                         = trigger_cascade_number_of_spikes
 |static_stat6_value                      = 6
 |stat_text                               = 
