--- Skill:HeistRobotTotemFireSnap
+++ Export
@@ -8,17 +8,18 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_fire_damage
-|static_stat1_value                      = 6
-|static_stat2_id                         = spell_maximum_base_fire_damage
-|static_stat2_value                      = 9
+|static_duration                         = 6
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = base_fire_damage_to_deal_per_minute
+|static_stat2_value                      = 648
 |static_stat3_id                         = base_skill_effect_duration
 |static_stat3_value                      = 6000
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
-|static_stat5_id                         = base_fire_damage_to_deal_per_minute
-|static_stat5_value                      = 648
-|stat_text                               = Deals 6 to 9 Fire Damage<br>Base duration is 6 seconds<br>Deals 10.8 Base Fire Damage per second
+|static_stat4_id                         = spell_minimum_base_fire_damage
+|static_stat4_value                      = 6
+|static_stat5_id                         = spell_maximum_base_fire_damage
+|static_stat5_value                      = 9
+|stat_text                               = Deals 10.8 Base Fire Damage per second<br>Base duration is 6 seconds<br>Deals 6 to 9 Fire Damage
 |level1                                  = True
 |level1_level_requirement                = 1
 }}