--- Skill:KitavaFinalDevouringDarknessLeftMap
+++ Export
@@ -9,17 +9,18 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_physical_damage_to_deal_per_minute
-|static_stat1_value                      = 327
+|static_duration                         = 12
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
 |static_stat2_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
 |static_stat2_value                      = -70
-|static_stat3_id                         = base_secondary_skill_effect_duration
-|static_stat3_value                      = 4000
+|static_stat3_id                         = base_physical_damage_to_deal_per_minute
+|static_stat3_value                      = 327
 |static_stat4_id                         = base_skill_effect_duration
 |static_stat4_value                      = 12000
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
-|stat_text                               = Deals 5.5 Base Physical Damage per second<br>Base secondary duration is 4 seconds<br>Base duration is 12 seconds
+|static_stat5_id                         = base_secondary_skill_effect_duration
+|static_stat5_value                      = 4000
+|stat_text                               = Deals 5.5 Base Physical Damage per second<br>Base duration is 12 seconds<br>Base secondary duration is 4 seconds
 |level1                                  = True
 |level1_level_requirement                = 1
 }}