--- Skill:BetrayalHakuMortar
+++ Export
@@ -11,18 +11,18 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = -30
-|static_stat2_id                         = projectile_spread_radius_per_additional_projectile
-|static_stat2_value                      = 4
-|static_stat3_id                         = projectile_spread_radius
-|static_stat3_value                      = 15
-|static_stat4_id                         = number_of_additional_projectiles
-|static_stat4_value                      = 4
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
-|static_stat6_id                         = base_is_projectile
+|static_stat2_id                         = base_is_projectile
+|static_stat2_value                      = 1
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = projectile_spread_radius
+|static_stat4_value                      = 15
+|static_stat5_id                         = projectile_spread_radius_per_additional_projectile
+|static_stat5_value                      = 4
+|static_stat6_id                         = projectile_uses_contact_position
 |static_stat6_value                      = 1
-|static_stat7_id                         = projectile_uses_contact_position
-|static_stat7_value                      = 1
+|static_stat7_id                         = number_of_additional_projectiles
+|static_stat7_value                      = 4
 |stat_text                               = Deals (5-356) to (8-534) Fire Damage<br>Deals (30-0)% less Damage<br>Fires 4 additional Projectiles
 |level1                                  = True
 |level1_level_requirement                = 1
