--- Skill:MaligaroTelekineticBladesMap
+++ Export
@@ -9,13 +9,13 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 5
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 7
-|static_stat3_id                         = always_pierce
-|static_stat3_value                      = 1
-|static_stat4_id                         = projectile_uses_contact_position
+|static_stat1_id                         = projectile_uses_contact_position
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_physical_damage
+|static_stat2_value                      = 5
+|static_stat3_id                         = spell_maximum_base_physical_damage
+|static_stat3_value                      = 7
+|static_stat4_id                         = always_pierce
 |static_stat4_value                      = 1
 |stat_text                               = Deals 5 to 7 Physical Damage<br>Projectiles Pierce all Targets
 |level1                                  = True
