--- Skill:SpiritOfferingTriggeredUnique
+++ Export
@@ -9,23 +9,24 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spirit_offering_critical_strike_multiplier_+
-|static_stat1_value                      = 39
-|static_stat2_id                         = spirit_offering_critical_strike_chance_+%
-|static_stat2_value                      = 148
-|static_stat3_id                         = offering_skill_effect_duration_per_corpse
-|static_stat3_value                      = 1000
-|static_stat4_id                         = base_skill_effect_duration
-|static_stat4_value                      = 5000
-|static_stat5_id                         = triggered_by_item_buff
+|static_duration                         = 5
+|static_stat1_id                         = base_deal_no_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = offerings_also_buff_you
+|static_stat2_value                      = 1
+|static_stat3_id                         = base_skill_effect_duration
+|static_stat3_value                      = 5000
+|static_stat4_id                         = offering_skill_effect_duration_per_corpse
+|static_stat4_value                      = 1000
+|static_stat5_id                         = spell_uncastable_if_triggerable
 |static_stat5_value                      = 1
-|static_stat6_id                         = spell_uncastable_if_triggerable
-|static_stat6_value                      = 1
-|static_stat7_id                         = offerings_also_buff_you
-|static_stat7_value                      = 1
-|static_stat8_id                         = base_deal_no_damage
+|static_stat6_id                         = spirit_offering_critical_strike_chance_+%
+|static_stat6_value                      = 148
+|static_stat7_id                         = spirit_offering_critical_strike_multiplier_+
+|static_stat7_value                      = 39
+|static_stat8_id                         = triggered_by_item_buff
 |static_stat8_value                      = 1
-|stat_text                               = Grants Minions +39% to Critical Strike Multiplier<br>Grants Minions 148% increased Critical Strike Chance<br>Additional 1 seconds Base Duration per extra corpse Consumed<br>Base duration is 5 seconds<br>This Skill is Triggered by the Item granting it<br>You cannot Cast this Spell directly<br>Your Offering Skills also affect you
+|stat_text                               = Your Offering Skills also affect you<br>Base duration is 5 seconds<br>Additional 1 seconds Base Duration per extra corpse Consumed<br>You cannot Cast this Spell directly<br>Grants Minions 148% increased Critical Strike Chance<br>Grants Minions +39% to Critical Strike Multiplier<br>This Skill is Triggered by the Item granting it
 |level1                                  = True
 |level1_level_requirement                = 70
 }}