--- Skill:BansheeBubbleBurstTriggered
+++ Export
@@ -8,19 +8,19 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 12
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 19
-|static_stat3_id                         = cast_on_trigger_cascade_event_%
-|static_stat3_value                      = 100
-|static_stat4_id                         = cluster_burst_spawn_amount
-|static_stat4_value                      = 2
+|static_stat1_id                         = cast_on_trigger_cascade_event_%
+|static_stat1_value                      = 100
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = spell_minimum_base_physical_damage
+|static_stat3_value                      = 12
+|static_stat4_id                         = spell_maximum_base_physical_damage
+|static_stat4_value                      = 19
 |static_stat5_id                         = spell_uncastable_if_triggerable
 |static_stat5_value                      = 1
-|static_stat6_id                         = is_area_damage
-|static_stat6_value                      = 1
-|stat_text                               = Deals 12 to 19 Physical Damage<br>Creates 2 explosions<br>You cannot Cast this Spell directly
+|static_stat6_id                         = cluster_burst_spawn_amount
+|static_stat6_value                      = 2
+|stat_text                               = Deals 12 to 19 Physical Damage<br>You cannot Cast this Spell directly<br>Creates 2 explosions
 |level1                                  = True
 |level1_level_requirement                = 1
 }}