--- Skill:ProphecySkeletonSpark
+++ Export
@@ -9,13 +9,16 @@
 |static_damage_effectiveness             = 70
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = number_of_additional_projectiles
-|static_stat1_value                      = 2
+|static_duration                         = 3.5
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
 |static_stat2_id                         = base_skill_effect_duration
 |static_stat2_value                      = 3500
-|static_stat3_id                         = base_is_projectile
-|static_stat3_value                      = 1
-|stat_text                               = Deals (2-12) to (6-36) Lightning Damage<br>Fires 2 additional Projectiles<br>Base duration is 3.5 seconds
+|static_stat3_id                         = base_projectile_speed_+%
+|static_stat3_value                      = 25
+|static_stat4_id                         = number_of_additional_projectiles
+|static_stat4_value                      = 2
+|stat_text                               = Deals (2-12) to (6-36) Lightning Damage<br>Base duration is 3.5 seconds<br>25% increased Projectile Speed<br>Fires 2 additional Projectiles
 |level1                                  = True
 |level1_level_requirement                = 4
 |level1_cost_amounts                     = 50
