--- Skill:BloodwingThrownShieldProjectile
+++ Export
@@ -10,11 +10,11 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = thrown_shield_num_additional_projectiles
-|static_stat1_value                      = 5
-|static_stat2_id                         = always_pierce
-|static_stat2_value                      = 1
-|static_stat3_id                         = base_is_projectile
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = thrown_shield_num_additional_projectiles
+|static_stat2_value                      = 5
+|static_stat3_id                         = always_pierce
 |static_stat3_value                      = 1
 |stat_text                               = Projectiles Pierce all Targets
 |level1                                  = True
