--- Skill:VaalSaucerTranslateRotate7
+++ Export
@@ -9,6 +9,7 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 2.5
 |static_stat1_id                         = translate_rotate_rotation_angle_rate
 |static_stat1_value                      = 15
 |static_stat2_id                         = base_skill_effect_duration
