--- Skill:GASanctumSentinelMaceFootstepCascade
+++ Export
@@ -0,0 +1,21 @@
+{{Skill
+|gem_description                         = 
+|active_skill_name                       = 
+|skill_id                                = GASanctumSentinelMaceFootstepCascade
+|cast_time                               = 1.0
+|required_level                          = 0
+|static_damage_effectiveness             = 100
+|static_damage_multiplier                = 75
+|static_attack_speed_multiplier          = 100
+|static_stat1_id                         = base_sanctum_damage
+|static_stat1_value                      = 30
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = global_knockback
+|static_stat3_value                      = 1
+|static_stat4_id                         = active_skill_area_of_effect_+%_final
+|static_stat4_value                      = -10
+|stat_text                               = Knocks Enemies Back on Hit<br>10% less Area of Effect
+|level1                                  = True
+|level1_level_requirement                = 0
+}}