--- Skill:BeyondDemonBlindNova
+++ Export
@@ -13,7 +13,7 @@
 |static_stat1_value                      = 477
 |static_stat2_id                         = global_chance_to_blind_on_hit_%
 |static_stat2_value                      = 100
-|stat_text                               = 100% chance to Blind enemies on hit
+|stat_text                               = Blind enemies on hit
 |level1                                  = True
 |level1_level_requirement                = 1
 }}