--- Skill:VaalArcBossSideArea
+++ Export
@@ -12,7 +12,7 @@
 |static_stat1_value                      = -50
 |static_stat2_id                         = base_chance_to_shock_%
 |static_stat2_value                      = 10
-|stat_text                               = Deals (1-46) to (3-138) Lightning Damage<br>Chains +(10-20) Times<br>50% reduced Skill Range<br>10% chance to Shock enemies
+|stat_text                               = Deals (1-46) to (3-138) Lightning Damage<br>50% reduced Skill Range<br>10% chance to Shock enemies<br>Chains +(10-20) Times
 |level1                                  = True
 |level1_level_requirement                = 4
 |level1_cost_amounts                     = 200
