--- Skill:SupportCastOnCritTriggeredPlus
+++ Export
@@ -6,18 +6,19 @@
 |quality_type1_stat_text                 = Supported Skills have 1% increased Spell Critical Strike Chance
 |quality_type1_stat1_id                  = spell_critical_strike_chance_+%
 |quality_type1_stat1_value               = 1000
+|static_cost_multiplier                  = 120
 |static_stored_uses                      = 1
 |static_cooldown                         = 0.15
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_uncastable_if_triggerable
+|static_stat1_id                         = cast_spell_on_linked_attack_crit
 |static_stat1_value                      = 1
 |static_stat2_id                         = triggered_skill_uses_main_hand_or_averaged_attack_time_for_pvp_scaling
 |static_stat2_value                      = 1
-|static_stat3_id                         = cast_spell_on_linked_attack_crit
+|static_stat3_id                         = spell_uncastable_if_triggerable
 |static_stat3_value                      = 1
-|stat_text                               = Supported Skills deal (9-2)% less Spell Damage<br>Supported Spells have (10-52)% increased Cooldown Recovery Rate<br>You cannot Cast Supported Triggerable Spells directly
+|stat_text                               = Supported Skills deal (9-2)% less Spell Damage<br>You cannot Cast Supported Triggerable Spells directly<br>Supported Spells have (10-52)% increased Cooldown Recovery Rate
 |level1                                  = True
 |level1_level_requirement                = 72
 |level1_stat_text                        = Supported Skills deal 9% less Spell Damage<br>Supported Spells have 10% increased Cooldown Recovery Rate
