--- Skill:ShrineArcPlayer
+++ Export
@@ -8,11 +8,11 @@
 |static_damage_effectiveness             = 20
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = number_of_chains
-|static_stat1_value                      = 2
-|static_stat2_id                         = base_chance_to_shock_%
-|static_stat2_value                      = 10
-|stat_text                               = Deals (1-83) to (7-1574) Lightning Damage<br>Chains +2 Times<br>10% chance to Shock enemies
+|static_stat1_id                         = base_chance_to_shock_%
+|static_stat1_value                      = 10
+|static_stat2_id                         = number_of_chains
+|static_stat2_value                      = 2
+|stat_text                               = Deals (1-83) to (7-1574) Lightning Damage<br>10% chance to Shock enemies<br>Chains +2 Times
 |level1                                  = True
 |level1_level_requirement                = 4
 |level1_stat_text                        = Deals 1 to 7 Lightning Damage
