--- Skill:DarkwoodBoneGolemLeapSlam
+++ Export
@@ -12,9 +12,9 @@
 |static_attack_speed_multiplier          = 56
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = 30
-|static_stat2_id                         = is_area_damage
+|static_stat2_id                         = disable_attack_repeats
 |static_stat2_value                      = 1
-|static_stat3_id                         = disable_attack_repeats
+|static_stat3_id                         = is_area_damage
 |static_stat3_value                      = 1
 |stat_text                               = 
 |level1                                  = True
