--- Skill:AtlasExileOrionCarpetProjectile
+++ Export
@@ -8,13 +8,13 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_projectile_variation
-|static_stat1_value                      = 129
-|static_stat2_id                         = always_pierce
+|static_stat1_id                         = base_deal_no_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = base_is_projectile
 |static_stat2_value                      = 1
-|static_stat3_id                         = base_is_projectile
-|static_stat3_value                      = 1
-|static_stat4_id                         = base_deal_no_damage
+|static_stat3_id                         = monster_projectile_variation
+|static_stat3_value                      = 129
+|static_stat4_id                         = always_pierce
 |static_stat4_value                      = 1
 |stat_text                               = Deals (6-2762) to (9-4143) Physical Damage<br>Projectiles Pierce all Targets
 |level1                                  = True
