--- Skill:FireMonsterWhirlingBlades
+++ Export
@@ -1,5 +1,5 @@
 {{Skill
-|gem_description                         = Dive through enemies, dealing weapon damage. Only works with daggers, claws and one handed swords. Cannot be supported by Multistrike.
+|gem_description                         = Dive through enemies, dealing weapon damage. If dual wielding attacks with both weapons, dealing the damage of both in one hit. Only works with Daggers, Claws, and One-Handed Swords. Cannot be supported by Multistrike.
 |active_skill_name                       = Whirling Blades
 |item_class_id_restriction               = Dagger, One Hand Sword, Thrusting One Hand Sword, Claw, Rune Dagger
 |skill_id                                = FireMonsterWhirlingBlades
@@ -8,19 +8,20 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = whirling_blades_base_ground_fire_damage_to_deal_per_minute
-|static_stat1_value                      = 95
-|static_stat2_id                         = monster_flurry
+|static_duration                         = 6
+|static_stat1_id                         = cast_time_overrides_attack_duration
+|static_stat1_value                      = 1
+|static_stat2_id                         = ignores_proximity_shield
 |static_stat2_value                      = 1
-|static_stat3_id                         = attack_speed_+%
-|static_stat3_value                      = -50
-|static_stat4_id                         = base_skill_effect_duration
-|static_stat4_value                      = 6000
-|static_stat5_id                         = cast_time_overrides_attack_duration
-|static_stat5_value                      = 1
-|static_stat6_id                         = ignores_proximity_shield
-|static_stat6_value                      = 1
-|stat_text                               = 50% reduced Attack Speed<br>Base duration is 6 seconds
+|static_stat3_id                         = monster_flurry
+|static_stat3_value                      = 1
+|static_stat4_id                         = whirling_blades_base_ground_fire_damage_to_deal_per_minute
+|static_stat4_value                      = 95
+|static_stat5_id                         = base_skill_effect_duration
+|static_stat5_value                      = 6000
+|static_stat6_id                         = attack_speed_+%
+|static_stat6_value                      = -50
+|stat_text                               = Base duration is 6 seconds<br>50% reduced Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 3
 }}