--- Skill:BurrowingViperHide
+++ Export
@@ -7,12 +7,13 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_skill_effect_duration
-|static_stat1_value                      = 500
-|static_stat2_id                         = skill_cannot_be_interrupted
+|static_duration                         = 0.5
+|static_stat1_id                         = skill_cannot_be_interrupted
+|static_stat1_value                      = 1
+|static_stat2_id                         = skill_cannot_be_stunned
 |static_stat2_value                      = 1
-|static_stat3_id                         = skill_cannot_be_stunned
-|static_stat3_value                      = 1
+|static_stat3_id                         = base_skill_effect_duration
+|static_stat3_value                      = 500
 |stat_text                               = Base duration is 0.5 seconds
 |level1                                  = True
 |level1_level_requirement                = 1
