--- Skill:EnduranceChargeSlamBestiaryCrabBoss
+++ Export
@@ -10,12 +10,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 250
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = skill_physical_damage_%_to_convert_to_cold
-|static_stat1_value                      = 25
-|static_stat2_id                         = active_skill_attack_speed_+%_final
-|static_stat2_value                      = -42
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = skill_physical_damage_%_to_convert_to_cold
+|static_stat2_value                      = 25
+|static_stat3_id                         = active_skill_attack_speed_+%_final
+|static_stat3_value                      = -42
 |stat_text                               = 25% of Physical Damage Converted to Cold Damage<br>42% less Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 1
