--- Skill:DelveAzuriteShardExplode
+++ Export
@@ -7,18 +7,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_projectile_variation
-|static_stat1_value                      = 62
-|static_stat2_id                         = skill_physical_damage_%_to_convert_to_cold
-|static_stat2_value                      = 75
-|static_stat3_id                         = never_freeze
-|static_stat3_value                      = 1
-|static_stat4_id                         = base_is_projectile
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_projectile_variation
+|static_stat2_value                      = 62
+|static_stat3_id                         = skill_physical_damage_%_to_convert_to_cold
+|static_stat3_value                      = 75
+|static_stat4_id                         = never_freeze
 |static_stat4_value                      = 1
 |stat_text                               = Deals (3-66) to (5-99) Physical Damage<br>Deals (80-1)% less Damage<br>75% of Physical Damage Converted to Cold Damage<br>Cannot Freeze
 |level1                                  = True
 |level1_level_requirement                = 1
-|level1_critical_strike_chance           = 0
 |level1_stat_text                        = Deals 3 to 5 Physical Damage<br>Deals 80% less Damage
 |level1_stat1_id                         = spell_minimum_base_physical_damage
 |level1_stat1_value                      = 3
