--- Skill:WbFireNova
+++ Export
@@ -9,21 +9,21 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_fire_damage
-|static_stat1_value                      = 156
-|static_stat2_id                         = spell_maximum_base_fire_damage
-|static_stat2_value                      = 234
-|static_stat3_id                         = base_cast_speed_+%
-|static_stat3_value                      = 100
-|static_stat4_id                         = fire_nova_damage_+%_per_repeat_final
-|static_stat4_value                      = 30
-|static_stat5_id                         = skill_repeat_count
-|static_stat5_value                      = 5
-|static_stat6_id                         = cannot_stun
-|static_stat6_value                      = 1
-|static_stat7_id                         = is_area_damage
-|static_stat7_value                      = 1
-|stat_text                               = Deals 156 to 234 Fire Damage<br>100% increased Cast Speed<br>30% more Damage with Hits and Ailments per Repeat<br>Skills Repeat an additional 5 Times<br>Your Hits cannot Stun Enemies
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = cannot_stun
+|static_stat2_value                      = 1
+|static_stat3_id                         = skill_repeat_count
+|static_stat3_value                      = 5
+|static_stat4_id                         = spell_minimum_base_fire_damage
+|static_stat4_value                      = 156
+|static_stat5_id                         = spell_maximum_base_fire_damage
+|static_stat5_value                      = 234
+|static_stat6_id                         = base_cast_speed_+%
+|static_stat6_value                      = 100
+|static_stat7_id                         = fire_nova_damage_+%_per_repeat_final
+|static_stat7_value                      = 30
+|stat_text                               = Your Hits cannot Stun Enemies<br>Skills Repeat an additional 5 Times<br>Deals 156 to 234 Fire Damage<br>100% increased Cast Speed<br>30% more Damage with Hits and Ailments per Repeat
 |level1                                  = True
 |level1_level_requirement                = 60
 }}