--- Skill:IncursionSummonDemonDaemon
+++ Export
@@ -7,17 +7,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = summon_specific_monsters_radius_+%
-|static_stat1_value                      = 50
-|static_stat2_id                         = alternate_minion
-|static_stat2_value                      = 353
-|static_stat3_id                         = summoned_monsters_are_minions
+|static_stat1_id                         = alternate_minion
+|static_stat1_value                      = 353
+|static_stat2_id                         = cannot_be_taunted
+|static_stat2_value                      = 1
+|static_stat3_id                         = skill_cannot_be_interrupted
 |static_stat3_value                      = 1
-|static_stat4_id                         = summoned_monsters_no_drops_or_experience
-|static_stat4_value                      = 1
-|static_stat5_id                         = skill_cannot_be_interrupted
+|static_stat4_id                         = summon_specific_monsters_radius_+%
+|static_stat4_value                      = 50
+|static_stat5_id                         = summoned_monsters_are_minions
 |static_stat5_value                      = 1
-|static_stat6_id                         = cannot_be_taunted
+|static_stat6_id                         = summoned_monsters_no_drops_or_experience
 |static_stat6_value                      = 1
 |stat_text                               = 
 |level1                                  = True
