--- Skill:AbyssVendigoComboAttack
+++ Export
@@ -9,10 +9,10 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 70
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = maligaro_combo_attack_third_hit_damage_+%_final
-|static_stat1_value                      = 30
-|static_stat2_id                         = is_area_damage
-|static_stat2_value                      = 1
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = maligaro_combo_attack_third_hit_damage_+%_final
+|static_stat2_value                      = 30
 |stat_text                               = 
 |level1                                  = True
 |level1_level_requirement                = 1
