--- Skill:ProphecyBloodGroundSlam
+++ Export
@@ -10,13 +10,13 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_attack_speed_+%_final
-|static_stat1_value                      = -55
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
 |static_stat2_id                         = active_skill_damage_+%_final
 |static_stat2_value                      = -30
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
-|stat_text                               = 55% less Attack Speed<br>Deals 30% less Damage
+|static_stat3_id                         = active_skill_attack_speed_+%_final
+|static_stat3_value                      = -55
+|stat_text                               = Deals 30% less Damage<br>55% less Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 3
 }}