--- Skill:AfflictionDemonWeaponThrowRight
+++ Export
@@ -7,19 +7,19 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 40
-|static_stat1_id                         = number_of_projectiles_override
+|static_stat1_id                         = base_is_projectile
 |static_stat1_value                      = 1
 |static_stat2_id                         = monster_projectile_variation
 |static_stat2_value                      = 104
-|static_stat3_id                         = always_pierce
+|static_stat3_id                         = number_of_projectiles_override
 |static_stat3_value                      = 1
-|static_stat4_id                         = base_is_projectile
+|static_stat4_id                         = projectile_uses_contact_direction
 |static_stat4_value                      = 1
 |static_stat5_id                         = projectile_uses_contact_position
 |static_stat5_value                      = 1
 |static_stat6_id                         = use_scaled_contact_offset
 |static_stat6_value                      = 1
-|static_stat7_id                         = projectile_uses_contact_direction
+|static_stat7_id                         = always_pierce
 |static_stat7_value                      = 1
 |stat_text                               = Projectiles Pierce all Targets
 |level1                                  = True
