--- Skill:BestiaryLynxCleave
+++ Export
@@ -8,17 +8,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat1_value                      = -20
-|static_stat2_id                         = physical_damage_+%
-|static_stat2_value                      = 35
-|static_stat3_id                         = cleave_damage_+%_final_while_dual_wielding
-|static_stat3_value                      = -40
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
-|static_stat5_id                         = skill_double_hits_when_dual_wielding
-|static_stat5_value                      = 1
-|stat_text                               = 35% increased Physical Damage<br>When Dual Wielding, Deals 60% Damage from each Weapon combined
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat2_value                      = -20
+|static_stat3_id                         = skill_double_hits_when_dual_wielding
+|static_stat3_value                      = 1
+|static_stat4_id                         = active_skill_merged_damage_+%_final_while_dual_wielding
+|static_stat4_value                      = -40
+|static_stat5_id                         = physical_damage_+%
+|static_stat5_value                      = 35
+|stat_text                               = When Dual Wielding, Deals 60% Damage from each Weapon combined<br>35% increased Physical Damage
 |level1                                  = True
 |level1_level_requirement                = 1
 }}