--- Skill:MerveilDaughterLightningNova
+++ Export
@@ -8,20 +8,20 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_projectile_variation
-|static_stat1_value                      = 4
-|static_stat2_id                         = base_is_projectile
-|static_stat2_value                      = 1
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_projectile_variation
+|static_stat2_value                      = 4
 |stat_text                               = Deals (10-125) to (31-374) Lightning Damage<br>Deals (0-50)% less Damage<br>(0-250)% increased Cast Speed
 |level1                                  = True
 |level1_level_requirement                = 15
 |level1_stat_text                        = Deals 10 to 31 Lightning Damage
-|level1_stat1_id                         = spell_minimum_base_lightning_damage
-|level1_stat1_value                      = 10
-|level1_stat2_id                         = spell_maximum_base_lightning_damage
-|level1_stat2_value                      = 31
-|level1_stat3_id                         = spell_maximum_action_distance_+%
-|level1_stat3_value                      = -35
+|level1_stat1_id                         = spell_maximum_action_distance_+%
+|level1_stat1_value                      = -35
+|level1_stat2_id                         = spell_minimum_base_lightning_damage
+|level1_stat2_value                      = 10
+|level1_stat3_id                         = spell_maximum_base_lightning_damage
+|level1_stat3_value                      = 31
 |level2                                  = True
 |level2_level_requirement                = 66
 |level2_stat_text                        = Deals 125 to 374 Lightning Damage<br>Deals 50% less Damage<br>250% increased Cast Speed
