--- Skill:BlightTowerIcePrison~buff3
+++ Export
@@ -2,15 +2,18 @@
 |gem_description                         = 
 |active_skill_name                       = 
 |skill_id                                = BlightTowerIcePrison_buff3
-|cast_time                               = 0.7
+|cast_time                               = 0.35
 |required_level                          = 1
 |static_stored_uses                      = 3
 |static_cooldown                         = 3.5
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_skill_effect_duration
-|static_stat1_value                      = 3000
+|static_duration                         = 3
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = 25
+|static_stat2_id                         = base_skill_effect_duration
+|static_stat2_value                      = 3000
 |stat_text                               = Base duration is 3 seconds
 |level1                                  = True
 |level1_level_requirement                = 1
