--- Skill:JaguarDemonDashSanctum
+++ Export
@@ -15,7 +15,7 @@
 |static_stat2_value                      = 25
 |static_stat3_id                         = melee_range_+
 |static_stat3_value                      = 50
-|stat_text                               = +50 to Melee Strike Range
+|stat_text                               = +5 metres to Melee Strike Range
 |level1                                  = True
 |level1_level_requirement                = 0
 }}