--- Skill:BetrayalCatarinaSoulMortar2
+++ Export
@@ -10,27 +10,27 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 5
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 7
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
 |static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
 |static_stat3_value                      = -40
 |static_stat4_id                         = projectile_minimum_range
 |static_stat4_value                      = 15
 |static_stat5_id                         = projectile_spread_radius
 |static_stat5_value                      = 25
-|static_stat6_id                         = skill_physical_damage_%_to_convert_to_fire
-|static_stat6_value                      = 50
-|static_stat7_id                         = number_of_additional_projectiles
-|static_stat7_value                      = 1
-|static_stat8_id                         = is_area_damage
-|static_stat8_value                      = 1
-|static_stat9_id                         = base_is_projectile
+|static_stat6_id                         = projectile_uses_contact_position
+|static_stat6_value                      = 1
+|static_stat7_id                         = spell_minimum_base_physical_damage
+|static_stat7_value                      = 5
+|static_stat8_id                         = spell_maximum_base_physical_damage
+|static_stat8_value                      = 7
+|static_stat9_id                         = number_of_additional_projectiles
 |static_stat9_value                      = 1
-|static_stat10_id                        = projectile_uses_contact_position
-|static_stat10_value                     = 1
-|stat_text                               = Deals 5 to 7 Physical Damage<br>50% of Physical Damage Converted to Fire Damage<br>Fires an additional Projectile
+|static_stat10_id                        = skill_physical_damage_%_to_convert_to_fire
+|static_stat10_value                     = 50
+|stat_text                               = Deals 5 to 7 Physical Damage<br>Fires an additional Projectile<br>50% of Physical Damage Converted to Fire Damage
 |level1                                  = True
 |level1_level_requirement                = 1
 }}