--- Skill:MeleeAtAnimationSpeedDeferDamagePrediction
+++ Export
@@ -7,17 +7,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = skill_can_fire_arrows
+|static_stat1_id                         = action_attack_or_cast_time_uses_animation_length
 |static_stat1_value                      = 1
-|static_stat2_id                         = skill_can_fire_wand_projectiles
+|static_stat2_id                         = melee_defer_damage_prediction
 |static_stat2_value                      = 1
-|static_stat3_id                         = action_attack_or_cast_time_uses_animation_length
+|static_stat3_id                         = projectile_uses_contact_position
 |static_stat3_value                      = 1
-|static_stat4_id                         = projectile_uses_contact_position
+|static_stat4_id                         = skill_can_fire_arrows
 |static_stat4_value                      = 1
-|static_stat5_id                         = use_scaled_contact_offset
+|static_stat5_id                         = skill_can_fire_wand_projectiles
 |static_stat5_value                      = 1
-|static_stat6_id                         = melee_defer_damage_prediction
+|static_stat6_id                         = use_scaled_contact_offset
 |static_stat6_value                      = 1
 |stat_text                               = 
 |level1                                  = True
