--- Skill:ProphecyWaterVortex
+++ Export
@@ -7,16 +7,16 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_reverse_point_blank_damage_-%_at_minimum_range
-|static_stat1_value                      = 35
-|static_stat2_id                         = base_projectile_speed_+%
-|static_stat2_value                      = 25
-|static_stat3_id                         = number_of_additional_projectiles
-|static_stat3_value                      = 2
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
-|static_stat5_id                         = base_is_projectile
-|static_stat5_value                      = 1
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = monster_reverse_point_blank_damage_-%_at_minimum_range
+|static_stat3_value                      = 35
+|static_stat4_id                         = base_projectile_speed_+%
+|static_stat4_value                      = 25
+|static_stat5_id                         = number_of_additional_projectiles
+|static_stat5_value                      = 2
 |stat_text                               = Deals (2-360) to (3-540) Cold Damage<br>(-25-5)% increased Cast Speed<br>25% increased Projectile Speed<br>Fires 2 additional Projectiles
 |level1                                  = True
 |level1_level_requirement                = 3
