--- Skill:BlightTowerArc~buff1
+++ Export
@@ -8,17 +8,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage
-|static_stat1_value                      = 49
-|static_stat2_id                         = spell_maximum_base_lightning_damage
-|static_stat2_value                      = 146
-|static_stat3_id                         = number_of_chains
-|static_stat3_value                      = 8
+|static_stat1_id                         = disable_visual_hit_effect
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_lightning_damage
+|static_stat2_value                      = 49
+|static_stat3_id                         = spell_maximum_base_lightning_damage
+|static_stat3_value                      = 146
 |static_stat4_id                         = base_chance_to_shock_%
 |static_stat4_value                      = 10
-|static_stat5_id                         = disable_visual_hit_effect
-|static_stat5_value                      = 1
-|stat_text                               = Deals 49 to 146 Lightning Damage<br>Chains +8 Times<br>10% chance to Shock enemies
+|static_stat5_id                         = number_of_chains
+|static_stat5_value                      = 8
+|stat_text                               = Deals 49 to 146 Lightning Damage<br>10% chance to Shock enemies<br>Chains +8 Times
 |level1                                  = True
 |level1_level_requirement                = 1
 }}