--- Skill:ZanaRestoreLife
+++ Export
@@ -9,6 +9,7 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 3
 |static_stat1_id                         = restore_skill_es_regeneration_rate_per_minute_%
 |static_stat1_value                      = 600
 |static_stat2_id                         = restore_skill_life_regeneration_rate_per_minute_%
