--- Skill:TriggerCascadeGreustArcher
+++ Export
@@ -7,18 +7,18 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = trigger_cascade_B
-|static_stat1_value                      = 3
-|static_stat2_id                         = trigger_cascade_target_variation
-|static_stat2_value                      = 1
+|static_stat1_id                         = trigger_cascade_A
+|static_stat1_value                      = 6
+|static_stat2_id                         = trigger_cascade_B
+|static_stat2_value                      = 3
 |static_stat3_id                         = trigger_cascade_behaviour_variation
 |static_stat3_value                      = 1
 |static_stat4_id                         = trigger_cascade_ms_between_spikes
 |static_stat4_value                      = 100
-|static_stat5_id                         = trigger_cascade_A
-|static_stat5_value                      = 6
-|static_stat6_id                         = trigger_cascade_number_of_spikes
-|static_stat6_value                      = 10
+|static_stat5_id                         = trigger_cascade_number_of_spikes
+|static_stat5_value                      = 10
+|static_stat6_id                         = trigger_cascade_target_variation
+|static_stat6_value                      = 1
 |stat_text                               = 
 |level1                                  = True
 |level1_level_requirement                = 1
