--- Skill:MachinariumFireSummon
+++ Export
@@ -10,22 +10,22 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_fire_damage
-|static_stat1_value                      = 594
-|static_stat2_id                         = spell_maximum_base_fire_damage
-|static_stat2_value                      = 891
-|static_stat3_id                         = alternate_minion
-|static_stat3_value                      = 3
+|static_stat1_id                         = alternate_minion
+|static_stat1_value                      = 3
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = number_of_inca_minions_allowed
+|static_stat3_value                      = 20
 |static_stat4_id                         = projectile_distance_override
 |static_stat4_value                      = 5
-|static_stat5_id                         = number_of_inca_minions_allowed
-|static_stat5_value                      = 20
-|static_stat6_id                         = is_area_damage
+|static_stat5_id                         = projectile_uses_contact_direction
+|static_stat5_value                      = 1
+|static_stat6_id                         = projectile_uses_contact_position
 |static_stat6_value                      = 1
-|static_stat7_id                         = projectile_uses_contact_position
-|static_stat7_value                      = 1
-|static_stat8_id                         = projectile_uses_contact_direction
-|static_stat8_value                      = 1
+|static_stat7_id                         = spell_minimum_base_fire_damage
+|static_stat7_value                      = 594
+|static_stat8_id                         = spell_maximum_base_fire_damage
+|static_stat8_value                      = 891
 |stat_text                               = Deals 594 to 891 Fire Damage
 |level1                                  = True
 |level1_level_requirement                = 80
