--- Skill:WbNewShockNova3
+++ Export
@@ -13,12 +13,12 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = 100
-|static_stat2_id                         = base_chance_to_shock_%
-|static_stat2_value                      = 15
-|static_stat3_id                         = newshocknova_first_ring_damage_+%_final
-|static_stat3_value                      = -50
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = base_chance_to_shock_%
+|static_stat3_value                      = 15
+|static_stat4_id                         = newshocknova_first_ring_damage_+%_final
+|static_stat4_value                      = -50
 |stat_text                               = Deals (2-966) to (6-2898) Lightning Damage<br>15% chance to Shock enemies<br>Ring deals 50% less Damage
 |level1                                  = True
 |level1_level_requirement                = 1
