--- Skill:DoomBossDrillCharge
+++ Export
@@ -10,14 +10,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 75
-|static_stat1_id                         = base_movement_velocity_+%
-|static_stat1_value                      = 70
-|static_stat2_id                         = ignores_proximity_shield
+|static_stat1_id                         = ignores_proximity_shield
+|static_stat1_value                      = 1
+|static_stat2_id                         = shield_charge_end_slide_wait_for_animation
 |static_stat2_value                      = 1
 |static_stat3_id                         = shield_charge_remove_stun_and_knockback
 |static_stat3_value                      = 1
-|static_stat4_id                         = shield_charge_end_slide_wait_for_animation
-|static_stat4_value                      = 1
+|static_stat4_id                         = base_movement_velocity_+%
+|static_stat4_value                      = 70
 |stat_text                               = 70% increased Movement Speed
 |level1                                  = True
 |level1_level_requirement                = 1
