--- Skill:BanditExplosiveArrowChampion
+++ Export
@@ -22,7 +22,7 @@
 |static_stat5_value                      = 13
 |static_stat6_id                         = fuse_arrow_explosion_radius_+_per_fuse_arrow_orb
 |static_stat6_value                      = 2
-|stat_text                               = Base duration is 1 seconds<br>Explosion deals 8 to 13 Base Fire Damage per Fuse Charge<br>+2 to Explosion Radius per Explosive Arrow on Target
+|stat_text                               = Base duration is 1 seconds<br>Explosion deals 8 to 13 Base Fire Damage per Fuse Charge<br>+0.2 metres to Explosion Radius per Explosive Arrow on Target
 |level1                                  = True
 |level1_level_requirement                = 4
 }}