--- Skill:FrogBossDexMissionChaosStorm
+++ Export
@@ -10,19 +10,20 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_chaos_damage
-|static_stat1_value                      = 1
-|static_stat2_id                         = spell_maximum_base_chaos_damage
+|static_duration                         = 5
+|static_stat1_id                         = firestorm_base_area_of_effect_+%
+|static_stat1_value                      = 95
+|static_stat2_id                         = is_area_damage
 |static_stat2_value                      = 1
-|static_stat3_id                         = firestorm_base_area_of_effect_+%
-|static_stat3_value                      = 95
-|static_stat4_id                         = fire_storm_fireball_delay_ms
-|static_stat4_value                      = 125
+|static_stat3_id                         = spell_minimum_base_chaos_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = spell_maximum_base_chaos_damage
+|static_stat4_value                      = 1
 |static_stat5_id                         = base_skill_effect_duration
 |static_stat5_value                      = 5000
-|static_stat6_id                         = is_area_damage
-|static_stat6_value                      = 1
-|stat_text                               = Deals 1 to 1 Chaos Damage<br>One impact every 0.12 seconds<br>Base duration is 5 seconds
+|static_stat6_id                         = fire_storm_fireball_delay_ms
+|static_stat6_value                      = 125
+|stat_text                               = Deals 1 to 1 Chaos Damage<br>Base duration is 5 seconds<br>One impact every 0.12 seconds
 |level1                                  = True
 |level1_level_requirement                = 3
 }}