--- Skill:NinjaCopMeleeThrust
+++ Export
@@ -7,13 +7,13 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 150
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = global_chance_to_blind_on_hit_%
-|static_stat1_value                      = 100
-|static_stat2_id                         = global_always_hit
-|static_stat2_value                      = 1
-|static_stat3_id                         = is_area_damage
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = global_chance_to_blind_on_hit_%
+|static_stat2_value                      = 100
+|static_stat3_id                         = global_always_hit
 |static_stat3_value                      = 1
-|stat_text                               = Deals (-50-1250)% more Damage<br>100% chance to Blind enemies on hit<br>Can't be Evaded
+|stat_text                               = Deals (-50-1250)% more Damage<br>Blind enemies on hit<br>Can't be Evaded
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_stat_text                        = Deals 50% less Damage
