--- Skill:MonsterLesserMultiFireballSpectre
+++ Export
@@ -8,11 +8,11 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_maximum_action_distance_+%
-|static_stat1_value                      = -50
-|static_stat2_id                         = number_of_additional_projectiles
-|static_stat2_value                      = 1
-|static_stat3_id                         = base_is_projectile
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_maximum_action_distance_+%
+|static_stat2_value                      = -50
+|static_stat3_id                         = number_of_additional_projectiles
 |static_stat3_value                      = 1
 |stat_text                               = Deals (4-584) to (5-876) Fire Damage<br>Fires an additional Projectile
 |level1                                  = True
