--- Skill:WbFrostBlades
+++ Export
@@ -16,9 +16,11 @@
 |static_stat4_value                      = 40
 |static_stat5_id                         = melee_weapon_range_+
 |static_stat5_value                      = 14
-|stat_text                               = Fires 2 additional Projectiles<br>40% of Physical Damage Converted to Cold Damage<br>+14 to Melee Strike Range
+|stat_text                               = Fires 2 additional Projectiles<br>40% of Physical Damage Converted to Cold Damage<br>+1.4 metres to Melee Strike Range
 |level1                                  = True
 |level1_level_requirement                = 1
+|level1_damage_effectiveness             = 100
+|level1_damage_multiplier                = 100
 |level2                                  = True
 |level2_level_requirement                = 4
 |level2_damage_effectiveness             = 101