--- Skill:AtlasExileOrionSpammableBeamCold3
+++ Export
@@ -8,15 +8,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = chill_effect_+%
+|static_stat1_id                         = active_skill_chill_effect_+%_final
 |static_stat1_value                      = 200
-|static_stat2_id                         = skill_physical_damage_%_to_convert_to_cold
-|static_stat2_value                      = 75
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
 |static_stat3_id                         = always_freeze
 |static_stat3_value                      = 1
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
-|stat_text                               = Deals (8-3591) to (12-5386) Physical Damage<br>200% increased Effect of Chill<br>75% of Physical Damage Converted to Cold Damage<br>Always Freezes enemies
+|static_stat4_id                         = skill_physical_damage_%_to_convert_to_cold
+|static_stat4_value                      = 75
+|stat_text                               = Deals (8-3591) to (12-5386) Physical Damage<br>Always Freezes enemies<br>75% of Physical Damage Converted to Cold Damage
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_stat_text                        = Deals 8 to 12 Physical Damage
