--- Skill:ExileLightningResistAura
+++ Export
@@ -11,23 +11,23 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = base_deal_no_damage
 |static_stat1_value                      = 1
-|stat_text                               = +(20-40)% to Lightning Resistance<br>+(1-6)% to maximum Lightning Resistance<br>+(20-45) to radius
+|stat_text                               = +(1-6)% to maximum Lightning Resistance<br>+(20-40)% to Lightning Resistance<br>+(20-45) to radius
 |level1                                  = True
 |level1_level_requirement                = 1
-|level1_stat_text                        = +20% to Lightning Resistance<br>+1% to maximum Lightning Resistance<br>+20 to radius
-|level1_stat1_id                         = base_lightning_damage_resistance_%
-|level1_stat1_value                      = 20
-|level1_stat2_id                         = base_maximum_lightning_damage_resistance_%
-|level1_stat2_value                      = 1
+|level1_stat_text                        = +1% to maximum Lightning Resistance<br>+20% to Lightning Resistance<br>+20 to radius
+|level1_stat1_id                         = base_maximum_lightning_damage_resistance_%
+|level1_stat1_value                      = 1
+|level1_stat2_id                         = base_lightning_damage_resistance_%
+|level1_stat2_value                      = 20
 |level1_stat3_id                         = active_skill_base_radius_+
 |level1_stat3_value                      = 20
 |level2                                  = True
 |level2_level_requirement                = 75
-|level2_stat_text                        = +40% to Lightning Resistance<br>+6% to maximum Lightning Resistance<br>+45 to radius
-|level2_stat1_id                         = base_lightning_damage_resistance_%
-|level2_stat1_value                      = 40
-|level2_stat2_id                         = base_maximum_lightning_damage_resistance_%
-|level2_stat2_value                      = 6
+|level2_stat_text                        = +6% to maximum Lightning Resistance<br>+40% to Lightning Resistance<br>+45 to radius
+|level2_stat1_id                         = base_maximum_lightning_damage_resistance_%
+|level2_stat1_value                      = 6
+|level2_stat2_id                         = base_lightning_damage_resistance_%
+|level2_stat2_value                      = 40
 |level2_stat3_id                         = active_skill_base_radius_+
 |level2_stat3_value                      = 45
 }}