--- Skill:DjinnDemonTornadoBurst
+++ Export
@@ -7,14 +7,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_cold_damage
-|static_stat1_value                      = 14
-|static_stat2_id                         = spell_maximum_base_cold_damage
-|static_stat2_value                      = 23
-|static_stat3_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat3_value                      = -33
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = -33
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = spell_minimum_base_cold_damage
+|static_stat3_value                      = 14
+|static_stat4_id                         = spell_maximum_base_cold_damage
+|static_stat4_value                      = 23
 |stat_text                               = Deals 14 to 23 Cold Damage
 |level1                                  = True
 |level1_level_requirement                = 0
