--- Skill:HarbingerProjectileBuffUber
+++ Export
@@ -9,11 +9,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = number_of_additional_projectiles
-|static_stat1_value                      = 2
-|static_stat2_id                         = base_skill_effect_duration
-|static_stat2_value                      = 4000
-|stat_text                               = Fires 2 additional Projectiles<br>Base duration is 4 seconds
+|static_duration                         = 4
+|static_stat1_id                         = base_skill_effect_duration
+|static_stat1_value                      = 4000
+|static_stat2_id                         = number_of_additional_projectiles
+|static_stat2_value                      = 2
+|stat_text                               = Base duration is 4 seconds<br>Fires 2 additional Projectiles
 |level1                                  = True
 |level1_level_requirement                = 1
 }}