--- Skill:ArchnemesisManaDonut
+++ Export
@@ -7,10 +7,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = mana_degeneration_per_minute_%
-|static_stat1_value                      = 600
-|static_stat2_id                         = base_lightning_damage_to_deal_per_minute
-|static_stat2_value                      = 1036
+|static_stat1_id                         = base_lightning_damage_to_deal_per_minute
+|static_stat1_value                      = 1036
+|static_stat2_id                         = base_sanctum_damage_to_deal_per_minute
+|static_stat2_value                      = 600
+|static_stat3_id                         = mana_degeneration_per_minute_%
+|static_stat3_value                      = 600
 |stat_text                               = Lose 10% of Mana per Second
 |level1                                  = True
 |level1_level_requirement                = 1
