--- Skill:DelveModSocketedSpellDamageFinal2h
+++ Export
@@ -2,6 +2,7 @@
 |skill_id                                = DelveModSocketedSpellDamageFinal2h
 |support_gem_letter                      = d
 |required_level                          = 0
+|static_cost_multiplier                  = 100
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
