--- Skill:GAConsumeBossTentacleSlam
+++ Export
@@ -6,11 +6,11 @@
 |required_level                          = 1
 |static_damage_effectiveness             = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = skill_physical_damage_%_to_convert_to_lightning
-|static_stat1_value                      = 75
-|static_stat2_id                         = is_area_damage
-|static_stat2_value                      = 1
-|stat_text                               = Enemy Block Chance reduced by (0-1000)% against this Skill<br>Enemies have (0-1000)% reduced chance to Dodge Hits from this Skill<br>75% of Physical Damage Converted to Lightning Damage
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = skill_physical_damage_%_to_convert_to_lightning
+|static_stat2_value                      = 75
+|stat_text                               = 75% of Physical Damage Converted to Lightning Damage<br>Enemy Block Chance reduced by (0-1000)% against this Skill<br>Enemies have (0-1000)% reduced chance to Dodge Hits from this Skill
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_damage_multiplier                = 310
