--- Skill:AtlasBasiliskScorpionSting
+++ Export
@@ -9,11 +9,11 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 200
 |static_attack_speed_multiplier          = 50
-|static_stat1_id                         = melee_range_+
-|static_stat1_value                      = 15
-|static_stat2_id                         = global_poison_on_hit
-|static_stat2_value                      = 1
-|static_stat3_id                         = is_area_damage
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = melee_range_+
+|static_stat2_value                      = 15
+|static_stat3_id                         = global_poison_on_hit
 |static_stat3_value                      = 1
 |stat_text                               = +15 to Melee Strike Range<br>Poisons Enemies on Hit
 |level1                                  = True
