--- Skill:GemlingCascade2
+++ Export
@@ -24,7 +24,7 @@
 |static_stat6_value                      = 100
 |static_stat7_id                         = upheaval_number_of_spikes
 |static_stat7_value                      = 6
-|stat_text                               = Deals 7 to 11 Physical Damage<br>+3 to radius<br>100% of Physical Damage Converted to Cold Damage<br>Causes 6 Bursts
+|stat_text                               = Deals 7 to 11 Physical Damage<br>+0.3 metres to radius<br>100% of Physical Damage Converted to Cold Damage<br>Causes 6 Bursts
 |level1                                  = True
 |level1_level_requirement                = 1
 }}