--- Skill:BrutusLightningMultiSlam
+++ Export
@@ -13,12 +13,12 @@
 |static_stat1_value                      = 100
 |static_stat2_id                         = ground_slam_cone_angle_override
 |static_stat2_value                      = 30
-|static_stat3_id                         = physical_damage_%_to_add_as_lightning
-|static_stat3_value                      = 100
-|static_stat4_id                         = active_skill_attack_speed_+%_final
-|static_stat4_value                      = -75
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = physical_damage_%_to_add_as_lightning
+|static_stat4_value                      = 100
+|static_stat5_id                         = active_skill_attack_speed_+%_final
+|static_stat5_value                      = -75
 |stat_text                               = Gain 100% of Physical Damage as Extra Lightning Damage<br>75% less Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 1
