--- Skill:MeleeEinhar
+++ Export
@@ -9,17 +9,17 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = arrow_projectile_variation
 |static_stat1_value                      = 3
-|static_stat2_id                         = active_skill_attack_speed_+%_final
-|static_stat2_value                      = 25
-|static_stat3_id                         = global_always_hit
+|static_stat2_id                         = skill_can_fire_arrows
+|static_stat2_value                      = 1
+|static_stat3_id                         = skill_can_fire_wand_projectiles
 |static_stat3_value                      = 1
 |static_stat4_id                         = always_pierce
 |static_stat4_value                      = 1
-|static_stat5_id                         = skill_can_fire_arrows
-|static_stat5_value                      = 1
-|static_stat6_id                         = skill_can_fire_wand_projectiles
+|static_stat5_id                         = active_skill_attack_speed_+%_final
+|static_stat5_value                      = 25
+|static_stat6_id                         = global_always_hit
 |static_stat6_value                      = 1
-|stat_text                               = 25% more Attack Speed<br>Can't be Evaded<br>Projectiles Pierce all Targets
+|stat_text                               = Projectiles Pierce all Targets<br>25% more Attack Speed<br>Can't be Evaded
 |level1                                  = True
 |level1_level_requirement                = 1
 }}