--- Skill:DancingDervishCycloneChannelled
+++ Export
@@ -22,7 +22,7 @@
 |static_stat6_value                      = 1
 |static_stat7_id                         = cyclone_stage_decay_time_ms
 |static_stat7_value                      = 330
-|stat_text                               = 8% increased Area of Effect per 1 additional Melee Strike Range<br>Gains 1 Stage every 0.17 seconds while Channelling<br>Maximum 6 Stages<br>+1 to radius per Stage<br>Lose 1 Stage every 0.33 seconds while not Channelling
+|stat_text                               = 8% increased Area of Effect per 0.1 metre additional Melee Strike Range<br>Gains 1 Stage every 0.17 seconds while Channelling<br>Maximum 6 Stages<br>+0.1 metres to radius per Stage<br>Lose 1 Stage every 0.33 seconds while not Channelling
 |level1                                  = True
 |level1_level_requirement                = 0
 }}