--- Skill:ExileShadowProjectileFireNova
+++ Export
@@ -9,19 +9,19 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_maximum_action_distance_+%
-|static_stat1_value                      = -60
-|static_stat2_id                         = base_cast_speed_+%
-|static_stat2_value                      = 175
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_maximum_action_distance_+%
+|static_stat2_value                      = -60
 |static_stat3_id                         = skill_repeat_count
 |static_stat3_value                      = 3
-|static_stat4_id                         = number_of_additional_projectiles
-|static_stat4_value                      = 19
-|static_stat5_id                         = projectiles_nova
-|static_stat5_value                      = 1
-|static_stat6_id                         = base_is_projectile
-|static_stat6_value                      = 1
-|stat_text                               = Deals (4-840) to (6-1309) Fire Damage<br>175% increased Cast Speed<br>Skills Repeat an additional 3 Times<br>Fires 19 additional Projectiles<br>Fires Projectiles in a circle
+|static_stat4_id                         = projectiles_nova
+|static_stat4_value                      = 1
+|static_stat5_id                         = base_cast_speed_+%
+|static_stat5_value                      = 175
+|static_stat6_id                         = number_of_additional_projectiles
+|static_stat6_value                      = 19
+|stat_text                               = Skills Repeat an additional 3 Times<br>Deals (4-840) to (6-1309) Fire Damage<br>Fires Projectiles in a circle<br>175% increased Cast Speed<br>Fires 19 additional Projectiles
 |level1                                  = True
 |level1_level_requirement                = 4
 |level1_cooldown                         = 6
