--- Skill:BetrayalSiegeCannon
+++ Export
@@ -11,12 +11,12 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = betrayal_siege_cannon_beam_damage_+%_final
 |static_stat1_value                      = -90
-|static_stat2_id                         = skill_physical_damage_%_to_convert_to_chaos
-|static_stat2_value                      = 20
-|static_stat3_id                         = is_area_damage
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = visual_hit_effect_chaos_is_green
 |static_stat3_value                      = 1
-|static_stat4_id                         = visual_hit_effect_chaos_is_green
-|static_stat4_value                      = 1
+|static_stat4_id                         = skill_physical_damage_%_to_convert_to_chaos
+|static_stat4_value                      = 20
 |stat_text                               = Deals (5-13) to (7-19) Physical Damage<br>Deals (20-0)% less Damage<br>20% of Physical Damage Converted to Chaos Damage
 |level1                                  = True
 |level1_level_requirement                = 1
