--- Skill:ExileHaste
+++ Export
@@ -12,13 +12,13 @@
 |static_mana_reservation_percent         = 1
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = 40
-|static_stat2_id                         = base_movement_velocity_+%
-|static_stat2_value                      = 8
-|static_stat3_id                         = cast_speed_+%_granted_from_skill
+|static_stat2_id                         = cast_speed_+%_granted_from_skill
+|static_stat2_value                      = 15
+|static_stat3_id                         = attack_speed_+%_granted_from_skill
 |static_stat3_value                      = 15
-|static_stat4_id                         = attack_speed_+%_granted_from_skill
-|static_stat4_value                      = 15
-|stat_text                               = 8% increased Movement Speed<br>15% increased Cast Speed<br>15% increased Attack Speed
+|static_stat4_id                         = base_movement_velocity_+%
+|static_stat4_value                      = 8
+|stat_text                               = Grants 15% increased Cast Speed<br>Grants 15% increased Attack Speed<br>8% increased Movement Speed
 |level1                                  = True
 |level1_level_requirement                = 1
 }}