--- Skill:SupportCastWhileChannellingTriggeredPlus
+++ Export
@@ -6,14 +6,15 @@
 |quality_type1_stat_text                 = Supported Triggered Spells deal 0.5% increased Damage
 |quality_type1_stat1_id                  = triggered_skill_damage_+%
 |quality_type1_stat1_value               = 500
+|static_cost_multiplier                  = 120
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = support_cast_while_channelling_triggered_skill_damage_+%_final
-|static_stat1_value                      = -30
-|static_stat2_id                         = spell_uncastable_if_triggerable
-|static_stat2_value                      = 1
-|static_stat3_id                         = cast_spell_while_linked_skill_channelling
+|static_stat1_id                         = cast_spell_while_linked_skill_channelling
+|static_stat1_value                      = 1
+|static_stat2_id                         = support_cast_while_channelling_triggered_skill_damage_+%_final
+|static_stat2_value                      = -30
+|static_stat3_id                         = spell_uncastable_if_triggerable
 |static_stat3_value                      = 1
 |stat_text                               = Supported Triggered Spells deal 30% less Damage<br>You cannot Cast Supported Triggerable Spells directly
 |level1                                  = True
