--- Skill:ProphecyFrostBolt
+++ Export
@@ -8,17 +8,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_cold_damage
-|static_stat1_value                      = 14
-|static_stat2_id                         = spell_maximum_base_cold_damage
-|static_stat2_value                      = 22
-|static_stat3_id                         = active_skill_cast_speed_+%_final
-|static_stat3_value                      = -33
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_cold_damage
+|static_stat2_value                      = 14
+|static_stat3_id                         = spell_maximum_base_cold_damage
+|static_stat3_value                      = 22
 |static_stat4_id                         = always_pierce
 |static_stat4_value                      = 1
-|static_stat5_id                         = base_is_projectile
-|static_stat5_value                      = 1
-|stat_text                               = Deals 14 to 22 Cold Damage<br>33% less Cast Speed<br>Projectiles Pierce all Targets
+|static_stat5_id                         = active_skill_cast_speed_+%_final
+|static_stat5_value                      = -33
+|stat_text                               = Deals 14 to 22 Cold Damage<br>Projectiles Pierce all Targets<br>33% less Cast Speed
 |level1                                  = True
 |level1_level_requirement                = 3
 }}