--- Skill:WbSummonFlameGolem
+++ Export
@@ -13,11 +13,11 @@
 |static_stat1_value                      = 1
 |static_stat2_id                         = display_minion_monster_type
 |static_stat2_value                      = 7
-|static_stat3_id                         = base_number_of_golems_allowed
-|static_stat3_value                      = 1
-|static_stat4_id                         = base_actor_scale_+%
-|static_stat4_value                      = 50
-|stat_text                               = Maximum 1 Summoned Golem<br>50% increased Character Size
+|static_stat3_id                         = base_actor_scale_+%
+|static_stat3_value                      = 50
+|static_stat4_id                         = base_number_of_golems_allowed
+|static_stat4_value                      = 1
+|stat_text                               = 50% increased Character Size<br>Maximum 1 Summoned Golem
 |level1                                  = True
 |level1_level_requirement                = 3
 |level1_stat1_id                         = display_minion_monster_level
