--- Skill:MPSProjectileRetaliationProjectile
+++ Export
@@ -10,23 +10,23 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 34
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 51
-|static_stat3_id                         = monster_projectile_variation
-|static_stat3_value                      = 187
-|static_stat4_id                         = monster_reverse_point_blank_damage_-%_at_minimum_range
-|static_stat4_value                      = 80
-|static_stat5_id                         = number_of_additional_projectiles
-|static_stat5_value                      = 5
-|static_stat6_id                         = projectiles_nova
+|static_stat1_id                         = is_trigger_daemon_when_hit_triggered_skill
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_projectile_variation
+|static_stat2_value                      = 187
+|static_stat3_id                         = monster_reverse_point_blank_damage_-%_at_minimum_range
+|static_stat3_value                      = 80
+|static_stat4_id                         = spell_minimum_base_physical_damage
+|static_stat4_value                      = 34
+|static_stat5_id                         = spell_maximum_base_physical_damage
+|static_stat5_value                      = 51
+|static_stat6_id                         = always_pierce
 |static_stat6_value                      = 1
-|static_stat7_id                         = always_pierce
+|static_stat7_id                         = projectiles_nova
 |static_stat7_value                      = 1
-|static_stat8_id                         = is_trigger_daemon_when_hit_triggered_skill
-|static_stat8_value                      = 1
-|stat_text                               = Deals 34 to 51 Physical Damage<br>Fires 5 additional Projectiles<br>Fires Projectiles in a circle<br>Projectiles Pierce all Targets
+|static_stat8_id                         = number_of_additional_projectiles
+|static_stat8_value                      = 5
+|stat_text                               = Deals 34 to 51 Physical Damage<br>Projectiles Pierce all Targets<br>Fires Projectiles in a circle<br>Fires 5 additional Projectiles
 |level1                                  = True
 |level1_level_requirement                = 0
 }}