--- Skill:GSArchnemesisEnergyShieldBurst
+++ Export
@@ -9,21 +9,21 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
+|static_stat1_id                         = is_area_damage
 |static_stat1_value                      = 1
-|static_stat2_id                         = spell_maximum_base_physical_damage
+|static_stat2_id                         = is_trigger_daemon_when_hit_triggered_skill
 |static_stat2_value                      = 1
-|static_stat3_id                         = base_stun_duration_+%
-|static_stat3_value                      = 200
-|static_stat4_id                         = always_stun
+|static_stat3_id                         = spell_minimum_base_physical_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = spell_maximum_base_physical_damage
 |static_stat4_value                      = 1
 |static_stat5_id                         = global_knockback
 |static_stat5_value                      = 1
-|static_stat6_id                         = is_area_damage
-|static_stat6_value                      = 1
-|static_stat7_id                         = is_trigger_daemon_when_hit_triggered_skill
+|static_stat6_id                         = base_stun_duration_+%
+|static_stat6_value                      = 200
+|static_stat7_id                         = always_stun
 |static_stat7_value                      = 1
-|stat_text                               = Deals 1 to 1 Physical Damage<br>200% increased Stun Duration on enemies<br>Stuns Enemies<br>Knocks Enemies Back on Hit
+|stat_text                               = Deals 1 to 1 Physical Damage<br>Knocks Enemies Back on Hit<br>200% increased Stun Duration on enemies<br>Stuns Enemies
 |level1                                  = True
 |level1_level_requirement                = 1
 }}