--- Skill:MinerThrowFire
+++ Export
@@ -16,6 +16,7 @@
 |stat_text                               = Deals (4.6-592.2) Base Fire Damage per second<br>Base duration is (3-4) seconds<br>Deals (6-799) to (8-1066) Fire Damage
 |level1                                  = True
 |level1_level_requirement                = 3
+|level1_duration                         = 3
 |level1_stat_text                        = Deals 4.6 Base Fire Damage per second<br>Base duration is 3 seconds<br>Deals 6 to 8 Fire Damage
 |level1_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level1_stat1_value                      = 277
@@ -27,6 +28,7 @@
 |level1_stat4_value                      = 8
 |level2                                  = True
 |level2_level_requirement                = 5
+|level2_duration                         = 3.1
 |level2_stat_text                        = Deals 5.9 Base Fire Damage per second<br>Base duration is 3.1 seconds<br>Deals 8 to 11 Fire Damage
 |level2_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level2_stat1_value                      = 352
@@ -38,6 +40,7 @@
 |level2_stat4_value                      = 11
 |level3                                  = True
 |level3_level_requirement                = 8
+|level3_duration                         = 3.2
 |level3_stat_text                        = Deals 8.1 Base Fire Damage per second<br>Base duration is 3.2 seconds<br>Deals 11 to 15 Fire Damage
 |level3_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level3_stat1_value                      = 488
@@ -49,6 +52,7 @@
 |level3_stat4_value                      = 15
 |level4                                  = True
 |level4_level_requirement                = 11
+|level4_duration                         = 3.3
 |level4_stat_text                        = Deals 10.9 Base Fire Damage per second<br>Base duration is 3.3 seconds<br>Deals 15 to 20 Fire Damage
 |level4_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level4_stat1_value                      = 656
@@ -60,6 +64,7 @@
 |level4_stat4_value                      = 20
 |level5                                  = True
 |level5_level_requirement                = 15
+|level5_duration                         = 3.4
 |level5_stat_text                        = Deals 15.7 Base Fire Damage per second<br>Base duration is 3.4 seconds<br>Deals 21 to 28 Fire Damage
 |level5_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level5_stat1_value                      = 944
@@ -71,6 +76,7 @@
 |level5_stat4_value                      = 28
 |level6                                  = True
 |level6_level_requirement                = 19
+|level6_duration                         = 3.5
 |level6_stat_text                        = Deals 22 Base Fire Damage per second<br>Base duration is 3.5 seconds<br>Deals 30 to 40 Fire Damage
 |level6_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level6_stat1_value                      = 1322
@@ -82,6 +88,7 @@
 |level6_stat4_value                      = 40
 |level7                                  = True
 |level7_level_requirement                = 23
+|level7_duration                         = 3.6
 |level7_stat_text                        = Deals 30.3 Base Fire Damage per second<br>Base duration is 3.6 seconds<br>Deals 41 to 54 Fire Damage
 |level7_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level7_stat1_value                      = 1816
@@ -93,6 +100,7 @@
 |level7_stat4_value                      = 54
 |level8                                  = True
 |level8_level_requirement                = 27
+|level8_duration                         = 3.7
 |level8_stat_text                        = Deals 41 Base Fire Damage per second<br>Base duration is 3.7 seconds<br>Deals 55 to 74 Fire Damage
 |level8_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level8_stat1_value                      = 2458
@@ -104,6 +112,7 @@
 |level8_stat4_value                      = 74
 |level9                                  = True
 |level9_level_requirement                = 28
+|level9_duration                         = 3.8
 |level9_stat_text                        = Deals 44.1 Base Fire Damage per second<br>Base duration is 3.8 seconds<br>Deals 60 to 79 Fire Damage
 |level9_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level9_stat1_value                      = 2646
@@ -115,6 +124,7 @@
 |level9_stat4_value                      = 79
 |level10                                 = True
 |level10_level_requirement               = 32
+|level10_duration                        = 3.9
 |level10_stat_text                       = Deals 58.8 Base Fire Damage per second<br>Base duration is 3.9 seconds<br>Deals 79 to 106 Fire Damage
 |level10_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level10_stat1_value                     = 3530
@@ -126,6 +136,7 @@
 |level10_stat4_value                     = 106
 |level11                                 = True
 |level11_level_requirement               = 40
+|level11_duration                        = 4
 |level11_stat_text                       = Deals 102 Base Fire Damage per second<br>Base duration is 4 seconds<br>Deals 138 to 184 Fire Damage
 |level11_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level11_stat1_value                     = 6119
@@ -137,6 +148,7 @@
 |level11_stat4_value                     = 184
 |level12                                 = True
 |level12_level_requirement               = 68
+|level12_duration                        = 4
 |level12_stat_text                       = Deals 592.2 Base Fire Damage per second<br>Base duration is 4 seconds<br>Deals 799 to 1066 Fire Damage
 |level12_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level12_stat1_value                     = 35531
