--- Skill:HolyFireElementalFireball
+++ Export
@@ -1,6 +1,6 @@
 {{Skill
 |gem_description                         = Unleashes a ball of fire towards a target which explodes, damaging nearby foes.
-|active_skill_name                       = 
+|active_skill_name                       = Fireball
 |skill_id                                = HolyFireElementalFireball
 |cast_time                               = 0.85
 |required_level                          = 3
@@ -8,10 +8,10 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_maximum_action_distance_+%
-|static_stat1_value                      = -50
-|static_stat2_id                         = base_is_projectile
-|static_stat2_value                      = 1
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_maximum_action_distance_+%
+|static_stat2_value                      = -50
 |stat_text                               = Deals (5-776) to (7-1202) Fire Damage
 |level1                                  = True
 |level1_level_requirement                = 3
