--- Skill:HoundBossVaalMoltenShell
+++ Export
@@ -7,21 +7,22 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_fire_damage
-|static_stat1_value                      = 217
-|static_stat2_id                         = spell_maximum_base_fire_damage
-|static_stat2_value                      = 325
+|static_duration                         = 5
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = molten_shell_explode_each_hit
+|static_stat2_value                      = 1
 |static_stat3_id                         = molten_shell_explosion_cooldown_ms
 |static_stat3_value                      = 500
 |static_stat4_id                         = base_skill_effect_duration
 |static_stat4_value                      = 5000
-|static_stat5_id                         = base_physical_damage_reduction_rating
-|static_stat5_value                      = 50000
-|static_stat6_id                         = molten_shell_explode_each_hit
-|static_stat6_value                      = 1
-|static_stat7_id                         = is_area_damage
-|static_stat7_value                      = 1
-|stat_text                               = Deals 217 to 325 Fire Damage<br>Base duration is 5 seconds<br>50000 additional Armour
+|static_stat5_id                         = spell_minimum_base_fire_damage
+|static_stat5_value                      = 217
+|static_stat6_id                         = spell_maximum_base_fire_damage
+|static_stat6_value                      = 325
+|static_stat7_id                         = base_physical_damage_reduction_rating
+|static_stat7_value                      = 50000
+|stat_text                               = Base duration is 5 seconds<br>Deals 217 to 325 Fire Damage<br>50000 additional Armour
 |level1                                  = True
 |level1_level_requirement                = 45
 }}