--- Skill:BansheeScreechMapBossGeometrySpell
+++ Export
@@ -8,15 +8,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 615
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 923
-|static_stat3_id                         = curse_on_hit_level_temporal_chains
-|static_stat3_value                      = 15
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
-|stat_text                               = Deals 615 to 923 Physical Damage<br>Curse Enemies with Temporal Chains on Hit, with 60% increased Effect
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_physical_damage
+|static_stat2_value                      = 615
+|static_stat3_id                         = spell_maximum_base_physical_damage
+|static_stat3_value                      = 923
+|static_stat4_id                         = curse_on_hit_level_temporal_chains
+|static_stat4_value                      = 15
+|stat_text                               = Deals 615 to 923 Physical Damage<br>Curse Enemies with Temporal Chains on Hit
 |level1                                  = True
 |level1_level_requirement                = 81
 }}