--- Skill:AfflictionPackTeleport
+++ Export
@@ -12,13 +12,13 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = -25
-|static_stat2_id                         = shock_minimum_damage_taken_increase_%
-|static_stat2_value                      = 25
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
 |static_stat3_id                         = always_shock
 |static_stat3_value                      = 1
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
-|stat_text                               = Deals (1-208) to (3-625) Lightning Damage<br>(1-51)% more Cast Speed<br>Shocks from your Hits always increase Damage taken by at least 25%<br>Hits always Shock
+|static_stat4_id                         = shock_minimum_damage_taken_increase_%
+|static_stat4_value                      = 25
+|stat_text                               = Deals (1-208) to (3-625) Lightning Damage<br>Hits always Shock<br>Shocks from your Hits always increase Damage taken by at least 25%<br>(1-51)% more Cast Speed
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_stat_text                        = Deals 1 to 3 Lightning Damage<br>1% more Cast Speed
