--- Skill:CrucibleChannel
+++ Export
@@ -0,0 +1,15 @@
+{{Skill
+|gem_description                         = [DNT] Hold up your weapon and see what happens
+|active_skill_name                       = [DNT] Crucible Channel
+|skill_id                                = CrucibleChannel
+|cast_time                               = 0.5
+|required_level                          = 0
+|static_damage_effectiveness             = 100
+|static_damage_multiplier                = 100
+|static_attack_speed_multiplier          = 100
+|static_stat1_id                         = skill_disabled_unless_in_crucible_zone
+|static_stat1_value                      = 1
+|stat_text                               = 
+|level1                                  = True
+|level1_level_requirement                = 0
+}}