--- Skill:GSConsumePhantasmExplode
+++ Export
@@ -7,14 +7,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage_per_removable_power_charge
-|static_stat1_value                      = 3
-|static_stat2_id                         = spell_maximum_base_lightning_damage_per_removable_power_charge
-|static_stat2_value                      = 5
-|static_stat3_id                         = apply_tangle_tentacle_explosion_on_hit_%_chance
-|static_stat3_value                      = 100
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
+|static_stat1_id                         = apply_tangle_tentacle_explosion_on_hit_%_chance
+|static_stat1_value                      = 100
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = spell_minimum_base_lightning_damage_per_removable_power_charge
+|static_stat3_value                      = 3
+|static_stat4_id                         = spell_maximum_base_lightning_damage_per_removable_power_charge
+|static_stat4_value                      = 5
 |stat_text                               = Deals 3 to 5 base Lightning Damage per Power Charge removed
 |level1                                  = True
 |level1_level_requirement                = 0
