--- Skill:VollConsecrateDaemon
+++ Export
@@ -9,13 +9,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 16
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = -30
-|static_stat2_id                         = life_regeneration_rate_per_minute_%
-|static_stat2_value                      = 180
-|static_stat3_id                         = base_skill_effect_duration
-|static_stat3_value                      = 16000
-|stat_text                               = Regenerate 3% of Life per second<br>Base duration is 16 seconds
+|static_stat2_id                         = base_skill_effect_duration
+|static_stat2_value                      = 16000
+|static_stat3_id                         = life_regeneration_rate_per_minute_%
+|static_stat3_value                      = 180
+|stat_text                               = Base duration is 16 seconds<br>Regenerate 3% of Life per second
 |level1                                  = True
 |level1_level_requirement                = 1
 }}