--- Skill:TrialmasterChaosShrine
+++ Export
@@ -9,23 +9,24 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 717
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 1075
-|static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat3_value                      = -50
-|static_stat4_id                         = orion_circle_maze_number_of_circles
-|static_stat4_value                      = 5
-|static_stat5_id                         = base_skill_effect_duration
-|static_stat5_value                      = 10000
-|static_stat6_id                         = active_skill_base_radius_+
-|static_stat6_value                      = 19
+|static_duration                         = 10
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat2_value                      = -50
+|static_stat3_id                         = orion_circle_maze_number_of_circles
+|static_stat3_value                      = 5
+|static_stat4_id                         = spell_minimum_base_physical_damage
+|static_stat4_value                      = 717
+|static_stat5_id                         = spell_maximum_base_physical_damage
+|static_stat5_value                      = 1075
+|static_stat6_id                         = base_skill_effect_duration
+|static_stat6_value                      = 10000
 |static_stat7_id                         = base_secondary_skill_effect_duration
 |static_stat7_value                      = 1000
-|static_stat8_id                         = is_area_damage
-|static_stat8_value                      = 1
-|stat_text                               = Deals 717 to 1075 Physical Damage<br>Base duration is 10 seconds<br>+19 to radius<br>Base secondary duration is 1 seconds
+|static_stat8_id                         = active_skill_base_radius_+
+|static_stat8_value                      = 19
+|stat_text                               = Deals 717 to 1075 Physical Damage<br>Base duration is 10 seconds<br>Base secondary duration is 1 seconds<br>+19 to radius
 |level1                                  = True
 |level1_level_requirement                = 80
 }}