--- Skill:MonsterGroundHaste
+++ Export
@@ -9,6 +9,7 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 5
 |static_stat1_id                         = action_speed_-%
 |static_stat1_value                      = -30
 |static_stat2_id                         = base_skill_effect_duration
