--- Skill:SkeletonCannonBoneNova
+++ Export
@@ -15,11 +15,11 @@
 |static_stat2_value                      = 30
 |static_stat3_id                         = active_skill_damage_+%_final
 |static_stat3_value                      = 40
-|static_stat4_id                         = number_of_additional_projectiles
-|static_stat4_value                      = 10
-|static_stat5_id                         = projectiles_nova
-|static_stat5_value                      = 1
-|stat_text                               = Deals 40% more Damage<br>Fires 10 additional Projectiles<br>Fires Projectiles in a circle
+|static_stat4_id                         = projectiles_nova
+|static_stat4_value                      = 1
+|static_stat5_id                         = number_of_additional_projectiles
+|static_stat5_value                      = 10
+|stat_text                               = Deals 40% more Damage<br>Fires Projectiles in a circle<br>Fires 10 additional Projectiles
 |level1                                  = True
 |level1_level_requirement                = 1
 }}