--- Skill:SpiderGodActivateLaserDaemon
+++ Export
@@ -9,12 +9,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage
-|static_stat1_value                      = 1177
-|static_stat2_id                         = spell_maximum_base_lightning_damage
-|static_stat2_value                      = 1438
-|static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat3_value                      = -50
+|static_stat1_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat1_value                      = -50
+|static_stat2_id                         = spell_minimum_base_lightning_damage
+|static_stat2_value                      = 1177
+|static_stat3_id                         = spell_maximum_base_lightning_damage
+|static_stat3_value                      = 1438
 |stat_text                               = Deals 1177 to 1438 Lightning Damage
 |level1                                  = True
 |level1_level_requirement                = 57
