--- Skill:DelveAzuriteBulletAbsorb
+++ Export
@@ -8,20 +8,20 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage
-|static_stat1_value                      = 7
-|static_stat2_id                         = spell_maximum_base_lightning_damage
-|static_stat2_value                      = 10
-|static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat3_value                      = -50
-|static_stat4_id                         = skill_lightning_damage_%_to_convert_to_cold
-|static_stat4_value                      = 50
-|static_stat5_id                         = monster_projectile_variation
-|static_stat5_value                      = 59
-|static_stat6_id                         = base_is_projectile
-|static_stat6_value                      = 1
-|static_stat7_id                         = projectile_uses_contact_position
-|static_stat7_value                      = 1
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat2_value                      = -50
+|static_stat3_id                         = monster_projectile_variation
+|static_stat3_value                      = 59
+|static_stat4_id                         = projectile_uses_contact_position
+|static_stat4_value                      = 1
+|static_stat5_id                         = skill_lightning_damage_%_to_convert_to_cold
+|static_stat5_value                      = 50
+|static_stat6_id                         = spell_minimum_base_lightning_damage
+|static_stat6_value                      = 7
+|static_stat7_id                         = spell_maximum_base_lightning_damage
+|static_stat7_value                      = 10
 |stat_text                               = Deals 7 to 10 Lightning Damage
 |level1                                  = True
 |level1_level_requirement                = 0
