--- Skill:ReliquaryMonsterFirestormSanctum
+++ Export
@@ -0,0 +1,36 @@
+{{Skill
+|gem_description                         = Flaming bolts rain down over the targeted area. They explode when landing, dealing damage to nearby enemies.
+|active_skill_name                       = Firestorm
+|skill_id                                = ReliquaryMonsterFirestormSanctum
+|cast_time                               = 1.63
+|required_level                          = 1
+|static_stored_uses                      = 1
+|static_cooldown                         = 6.5
+|static_damage_effectiveness             = 100
+|static_damage_multiplier                = 100
+|static_attack_speed_multiplier          = 100
+|static_duration                         = 1.2
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = 30
+|static_stat2_id                         = base_sanctum_damage
+|static_stat2_value                      = 5
+|static_stat3_id                         = firestorm_base_area_of_effect_+%
+|static_stat3_value                      = -80
+|static_stat4_id                         = is_area_damage
+|static_stat4_value                      = 1
+|static_stat5_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat5_value                      = -30
+|static_stat6_id                         = spell_maximum_action_distance_+%
+|static_stat6_value                      = -50
+|static_stat7_id                         = spell_minimum_base_physical_damage
+|static_stat7_value                      = 2
+|static_stat8_id                         = spell_maximum_base_physical_damage
+|static_stat8_value                      = 2
+|static_stat9_id                         = base_skill_effect_duration
+|static_stat9_value                      = 1200
+|static_stat10_id                        = fire_storm_fireball_delay_ms
+|static_stat10_value                     = 200
+|stat_text                               = Deals 2 to 2 Physical Damage<br>Base duration is 1.2 seconds<br>One impact every 0.2 seconds
+|level1                                  = True
+|level1_level_requirement                = 1
+}}