--- Skill:ShieldCharge
+++ Export
@@ -280,6 +280,8 @@
 |level29_stat2_value                     = 78
 |level30                                 = True
 |level30_level_requirement               = 90
+|level30_damage_effectiveness            = 100
+|level30_damage_multiplier               = 100
 |level30_stat_text                       = 548% more Damage with Hits at Maximum Charge Distance<br>79% increased Movement Speed
 |level30_stat1_id                        = shield_charge_damage_+%_maximum
 |level30_stat1_value                     = 548
