--- Skill:WeaverHide
+++ Export
@@ -14,21 +14,25 @@
 |stat_text                               = Base duration is (20-12) seconds
 |level1                                  = True
 |level1_level_requirement                = 18
+|level1_duration                         = 20
 |level1_stat_text                        = Base duration is 20 seconds
 |level1_stat1_id                         = base_skill_effect_duration
 |level1_stat1_value                      = 20000
 |level2                                  = True
 |level2_level_requirement                = 46
+|level2_duration                         = 8
 |level2_stat_text                        = Base duration is 8 seconds
 |level2_stat1_id                         = base_skill_effect_duration
 |level2_stat1_value                      = 8000
 |level3                                  = True
 |level3_level_requirement                = 61
+|level3_duration                         = 10
 |level3_stat_text                        = Base duration is 10 seconds
 |level3_stat1_id                         = base_skill_effect_duration
 |level3_stat1_value                      = 10000
 |level4                                  = True
 |level4_level_requirement                = 72
+|level4_duration                         = 12
 |level4_stat_text                        = Base duration is 12 seconds
 |level4_stat1_id                         = base_skill_effect_duration
 |level4_stat1_value                      = 12000
