--- Skill:HarbingerBallLightning
+++ Export
@@ -10,14 +10,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage
-|static_stat1_value                      = 2
-|static_stat2_id                         = spell_maximum_base_lightning_damage
-|static_stat2_value                      = 5
-|static_stat3_id                         = base_is_projectile
-|static_stat3_value                      = 1
-|static_stat4_id                         = projectile_uses_contact_position
-|static_stat4_value                      = 1
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = projectile_uses_contact_position
+|static_stat2_value                      = 1
+|static_stat3_id                         = spell_minimum_base_lightning_damage
+|static_stat3_value                      = 2
+|static_stat4_id                         = spell_maximum_base_lightning_damage
+|static_stat4_value                      = 5
 |stat_text                               = Deals 2 to 5 Lightning Damage
 |level1                                  = True
 |level1_level_requirement                = 1
