--- Skill:WaterElementalArcticBreath
+++ Export
@@ -10,10 +10,11 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_skill_effect_duration
-|static_stat1_value                      = 1650
-|static_stat2_id                         = base_is_projectile
-|static_stat2_value                      = 1
+|static_duration                         = 1.65
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = base_skill_effect_duration
+|static_stat2_value                      = 1650
 |stat_text                               = Deals (16-889) to (24-1334) Cold Damage<br>Base duration is 1.65 seconds
 |level1                                  = True
 |level1_level_requirement                = 12
@@ -25,10 +26,10 @@
 |level2                                  = True
 |level2_level_requirement                = 68
 |level2_stat_text                        = Deals 889 to 1334 Cold Damage
-|level2_stat1_id                         = spell_minimum_base_cold_damage
-|level2_stat1_value                      = 889
-|level2_stat2_id                         = spell_maximum_base_cold_damage
-|level2_stat2_value                      = 1334
-|level2_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|level2_stat3_value                      = -25
+|level2_stat1_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|level2_stat1_value                      = -25
+|level2_stat2_id                         = spell_minimum_base_cold_damage
+|level2_stat2_value                      = 889
+|level2_stat3_id                         = spell_maximum_base_cold_damage
+|level2_stat3_value                      = 1334
 }}