--- Skill:ExpeditionMegaSkeletonHeavyMeleeSpiked
+++ Export
@@ -9,16 +9,16 @@
 |static_damage_effectiveness             = 170
 |static_damage_multiplier                = 170
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = skill_physical_damage_%_to_convert_to_cold
-|static_stat1_value                      = 75
-|static_stat2_id                         = melee_range_+
-|static_stat2_value                      = 30
-|static_stat3_id                         = skill_can_fire_arrows
+|static_stat1_id                         = action_attack_or_cast_time_uses_animation_length
+|static_stat1_value                      = 1
+|static_stat2_id                         = skill_can_fire_arrows
+|static_stat2_value                      = 1
+|static_stat3_id                         = skill_can_fire_wand_projectiles
 |static_stat3_value                      = 1
-|static_stat4_id                         = skill_can_fire_wand_projectiles
-|static_stat4_value                      = 1
-|static_stat5_id                         = action_attack_or_cast_time_uses_animation_length
-|static_stat5_value                      = 1
+|static_stat4_id                         = skill_physical_damage_%_to_convert_to_cold
+|static_stat4_value                      = 75
+|static_stat5_id                         = melee_range_+
+|static_stat5_value                      = 30
 |stat_text                               = Deals (-50-65)% more Damage<br>75% of Physical Damage Converted to Cold Damage<br>+30 to Melee Strike Range
 |level1                                  = True
 |level1_level_requirement                = 0
