--- Skill:ElderExpandingNova
+++ Export
@@ -9,11 +9,11 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat1_value                      = -50
-|static_stat2_id                         = is_area_damage
-|static_stat2_value                      = 1
-|stat_text                               = Deals (2259-6517) to (2761-7965) Physical Damage<br>(0-150)% more Cast Speed<br>Skills Repeat an additional (0-2) Times
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat2_value                      = -50
+|stat_text                               = Deals (2259-6517) to (2761-7965) Physical Damage<br>Skills Repeat an additional (0-2) Times<br>(0-150)% more Cast Speed
 |level1                                  = True
 |level1_level_requirement                = 68
 |level1_stat_text                        = Deals 2259 to 2761 Physical Damage
@@ -30,13 +30,13 @@
 |level2_stat2_value                      = 6625
 |level3                                  = True
 |level3_level_requirement                = 85
-|level3_stat_text                        = Deals 6517 to 7965 Physical Damage<br>150% more Cast Speed<br>Skills Repeat an additional 2 Times
+|level3_stat_text                        = Deals 6517 to 7965 Physical Damage<br>Skills Repeat an additional 2 Times<br>150% more Cast Speed
 |level3_stat1_id                         = spell_minimum_base_physical_damage
 |level3_stat1_value                      = 6517
 |level3_stat2_id                         = spell_maximum_base_physical_damage
 |level3_stat2_value                      = 7965
-|level3_stat3_id                         = active_skill_cast_speed_+%_final
-|level3_stat3_value                      = 150
-|level3_stat4_id                         = skill_repeat_count
-|level3_stat4_value                      = 2
+|level3_stat3_id                         = skill_repeat_count
+|level3_stat3_value                      = 2
+|level3_stat4_id                         = active_skill_cast_speed_+%_final
+|level3_stat4_value                      = 150
 }}