--- Skill:MonsterDemonCleave
+++ Export
@@ -9,10 +9,10 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = corrupted_blood_cleave_stacks_to_add
 |static_stat1_value                      = 2
-|static_stat2_id                         = attack_speed_+%
-|static_stat2_value                      = -50
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = attack_speed_+%
+|static_stat3_value                      = -50
 |stat_text                               = Deals (2.5-299.1) Base Physical Damage per second<br>50% reduced Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 3
