--- Skill:AtlasOrionDeathZoneAuraCoreMinion
+++ Export
@@ -7,17 +7,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_physical_damage_to_deal_per_minute
+|static_stat1_id                         = base_lightning_damage_to_deal_per_minute
 |static_stat1_value                      = 53214
 |static_stat2_id                         = action_speed_-%
 |static_stat2_value                      = 25
-|static_stat3_id                         = base_fire_damage_to_deal_per_minute
+|static_stat3_id                         = base_chaos_damage_to_deal_per_minute
 |static_stat3_value                      = 53214
-|static_stat4_id                         = base_chaos_damage_to_deal_per_minute
+|static_stat4_id                         = base_fire_damage_to_deal_per_minute
 |static_stat4_value                      = 53214
-|static_stat5_id                         = base_lightning_damage_to_deal_per_minute
+|static_stat5_id                         = base_physical_damage_to_deal_per_minute
 |static_stat5_value                      = 53214
-|stat_text                               = Deals 886.9 Base Physical Damage per second<br>25% reduced Action Speed<br>Deals 886.9 Base Fire Damage per second<br>Deals 886.9 Base Chaos Damage per second
+|stat_text                               = 25% reduced Action Speed<br>Deals 886.9 Base Chaos Damage per second<br>Deals 886.9 Base Fire Damage per second<br>Deals 886.9 Base Physical Damage per second
 |level1                                  = True
 |level1_level_requirement                = 84
 }}