--- Skill:UniqueSupportCastOnShockedKill
+++ Export
@@ -2,6 +2,7 @@
 |skill_id                                = UniqueSupportCastOnShockedKill
 |support_gem_letter                      = ^
 |required_level                          = 1
+|static_cost_multiplier                  = 100
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
