--- Skill:DoedreBloodProjectileMulti
+++ Export
@@ -7,18 +7,18 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = projectile_minimum_range
-|static_stat1_value                      = 20
-|static_stat2_id                         = projectile_spread_radius
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = projectile_minimum_range
 |static_stat2_value                      = 20
-|static_stat3_id                         = spell_maximum_action_distance_+%
-|static_stat3_value                      = -60
-|static_stat4_id                         = active_skill_cast_speed_+%_final
-|static_stat4_value                      = -15
-|static_stat5_id                         = number_of_additional_projectiles
-|static_stat5_value                      = 2
-|static_stat6_id                         = base_is_projectile
-|static_stat6_value                      = 1
+|static_stat3_id                         = projectile_spread_radius
+|static_stat3_value                      = 20
+|static_stat4_id                         = spell_maximum_action_distance_+%
+|static_stat4_value                      = -60
+|static_stat5_id                         = active_skill_cast_speed_+%_final
+|static_stat5_value                      = -15
+|static_stat6_id                         = number_of_additional_projectiles
+|static_stat6_value                      = 2
 |stat_text                               = Deals (117-418) to (175-628) Physical Damage<br>15% less Cast Speed<br>Fires 2 additional Projectiles
 |level1                                  = True
 |level1_level_requirement                = 40
