--- Skill:BlightTowerAuraWeakenEnemies~buff3
+++ Export
@@ -9,14 +9,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = blight_aura_stat_5
-|static_stat1_value                      = -15
-|static_stat2_id                         = blight_aura_stat_4
+|static_stat1_id                         = blight_aura_stat_1
+|static_stat1_value                      = 25
+|static_stat2_id                         = blight_aura_stat_3
 |static_stat2_value                      = -20
-|static_stat3_id                         = blight_aura_stat_3
+|static_stat3_id                         = blight_aura_stat_4
 |static_stat3_value                      = -20
-|static_stat4_id                         = blight_aura_stat_1
-|static_stat4_value                      = 25
+|static_stat4_id                         = blight_aura_stat_5
+|static_stat4_value                      = -15
 |stat_text                               = 
 |level1                                  = True
 |level1_level_requirement                = 1
