--- Skill:AncestralProtectorBanditMapBoss
+++ Export
@@ -28,7 +28,7 @@
 |static_stat9_value                      = 50
 |static_stat10_id                        = melee_range_+
 |static_stat10_value                     = 16
-|stat_text                               = Totem lasts 12 seconds<br>Summons a Totem which uses this Skill<br>25% more Attack Speed while Totem is Active<br>50% increased Totem Placement speed<br>+16 to Melee Strike Range
+|stat_text                               = Totem lasts 12 seconds<br>Summons a Totem which uses this Skill<br>25% more Attack Speed while Totem is Active<br>50% increased Totem Placement speed<br>+1.6 metres to Melee Strike Range
 |level1                                  = True
 |level1_level_requirement                = 68
 }}