--- Skill:HarvestHammerVineImpactGA
+++ Export
@@ -7,15 +7,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 650
 |static_attack_speed_multiplier          = 58
-|static_stat1_id                         = global_poison_on_hit
+|static_stat1_id                         = is_area_damage
 |static_stat1_value                      = 1
-|static_stat2_id                         = always_stun
+|static_stat2_id                         = is_oshabi_vivid_skill
 |static_stat2_value                      = 1
-|static_stat3_id                         = is_area_damage
+|static_stat3_id                         = always_stun
 |static_stat3_value                      = 1
-|static_stat4_id                         = is_oshabi_vivid_skill
+|static_stat4_id                         = global_poison_on_hit
 |static_stat4_value                      = 1
-|stat_text                               = Poisons Enemies on Hit<br>Stuns Enemies
+|stat_text                               = Stuns Enemies<br>Poisons Enemies on Hit
 |level1                                  = True
 |level1_level_requirement                = 0
 }}