--- Skill:GAHarvestWebMortarImpact
+++ Export
@@ -7,14 +7,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 80
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_skill_effect_duration
-|static_stat1_value                      = 3000
-|static_stat2_id                         = skill_physical_damage_%_to_convert_to_chaos
-|static_stat2_value                      = 15
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
-|static_stat4_id                         = visual_hit_effect_chaos_is_green
-|static_stat4_value                      = 1
+|static_duration                         = 3
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = visual_hit_effect_chaos_is_green
+|static_stat2_value                      = 1
+|static_stat3_id                         = base_skill_effect_duration
+|static_stat3_value                      = 3000
+|static_stat4_id                         = skill_physical_damage_%_to_convert_to_chaos
+|static_stat4_value                      = 15
 |stat_text                               = Base duration is 3 seconds<br>15% of Physical Damage Converted to Chaos Damage
 |level1                                  = True
 |level1_level_requirement                = 0
