--- Skill:LightningGolemArc
+++ Export
@@ -10,6 +10,7 @@
 |static_damage_effectiveness             = 60
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 8
 |static_stat1_id                         = spell_minimum_base_lightning_damage
 |static_stat1_value                      = 1
 |static_stat2_id                         = spell_maximum_base_lightning_damage
