--- Skill:GSCleansingBossLanternExplosionUber
+++ Export
@@ -8,23 +8,22 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_fire_damage
-|static_stat1_value                      = 17
-|static_stat2_id                         = spell_maximum_base_fire_damage
-|static_stat2_value                      = 26
+|static_duration                         = 10
+|static_stat1_id                         = cannot_be_blocked_or_dodged_or_suppressed
+|static_stat1_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
 |static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
 |static_stat3_value                      = -75
 |static_stat4_id                         = voll_slam_damage_+%_final_at_centre
 |static_stat4_value                      = 300
 |static_stat5_id                         = base_skill_effect_duration
 |static_stat5_value                      = 10000
-|static_stat6_id                         = reduce_enemy_dodge_%
-|static_stat6_value                      = 1000
-|static_stat7_id                         = global_reduce_enemy_block_%
-|static_stat7_value                      = 1000
-|static_stat8_id                         = is_area_damage
-|static_stat8_value                      = 1
-|stat_text                               = Deals 17 to 26 Fire Damage<br>Base duration is 10 seconds<br>Enemies have 1000% reduced chance to Dodge Hits from this Skill<br>Enemy Block Chance reduced by 1000% against this Skill
+|static_stat6_id                         = spell_minimum_base_fire_damage
+|static_stat6_value                      = 17
+|static_stat7_id                         = spell_maximum_base_fire_damage
+|static_stat7_value                      = 26
+|stat_text                               = Base duration is 10 seconds<br>Deals 17 to 26 Fire Damage
 |level1                                  = True
 |level1_level_requirement                = 1
 }}