--- Skill:StormWhip
+++ Export
@@ -10,19 +10,20 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 1
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = 30
-|static_stat2_id                         = melee_range_+
-|static_stat2_value                      = 25
-|static_stat3_id                         = active_skill_projectile_damage_+%_final
-|static_stat3_value                      = -40
-|static_stat4_id                         = base_skill_effect_duration
-|static_stat4_value                      = 1000
-|static_stat5_id                         = number_of_additional_projectiles
-|static_stat5_value                      = 7
-|static_stat6_id                         = skill_physical_damage_%_to_convert_to_lightning
-|static_stat6_value                      = 60
-|stat_text                               = +25 to Melee Strike Range<br>Projectiles deal 40% less Damage<br>Base duration is 1 seconds<br>Fires 7 additional Projectiles<br>60% of Physical Damage Converted to Lightning Damage
+|static_stat2_id                         = skill_physical_damage_%_to_convert_to_lightning
+|static_stat2_value                      = 60
+|static_stat3_id                         = base_skill_effect_duration
+|static_stat3_value                      = 1000
+|static_stat4_id                         = number_of_additional_projectiles
+|static_stat4_value                      = 7
+|static_stat5_id                         = melee_range_+
+|static_stat5_value                      = 25
+|static_stat6_id                         = active_skill_projectile_damage_+%_final
+|static_stat6_value                      = -40
+|stat_text                               = 60% of Physical Damage Converted to Lightning Damage<br>Base duration is 1 seconds<br>Fires 7 additional Projectiles<br>+25 to Melee Strike Range<br>40% less Projectile Damage
 |level1                                  = True
 |level1_level_requirement                = 1
 }}