--- Skill:SanctumElreonSmite
+++ Export
@@ -0,0 +1,9 @@
+{{Skill
+|gem_description                         = 
+|active_skill_name                       = 
+|skill_id                                = SanctumElreonSmite
+|cast_time                               = 1.5
+|required_level                          = 1
+|static_damage_effectiveness             = 50
+|static_damage_multiplier                = 50
+}}