--- Skill:MountainGoatmanIceSpear
+++ Export
@@ -18,19 +18,19 @@
 |level1                                  = True
 |level1_level_requirement                = 40
 |level1_stat_text                        = Deals 134 to 201 Cold Damage
-|level1_stat1_id                         = spell_minimum_base_cold_damage
-|level1_stat1_value                      = 134
-|level1_stat2_id                         = spell_maximum_base_cold_damage
-|level1_stat2_value                      = 201
-|level1_stat3_id                         = ice_spear_second_form_damage_+%
-|level1_stat3_value                      = 50
+|level1_stat1_id                         = ice_spear_second_form_damage_+%
+|level1_stat1_value                      = 50
+|level1_stat2_id                         = spell_minimum_base_cold_damage
+|level1_stat2_value                      = 134
+|level1_stat3_id                         = spell_maximum_base_cold_damage
+|level1_stat3_value                      = 201
 |level2                                  = True
 |level2_level_requirement                = 68
 |level2_stat_text                        = Deals 595 to 893 Cold Damage
-|level2_stat1_id                         = spell_minimum_base_cold_damage
-|level2_stat1_value                      = 595
-|level2_stat2_id                         = spell_maximum_base_cold_damage
-|level2_stat2_value                      = 893
-|level2_stat3_id                         = ice_spear_second_form_damage_+%
-|level2_stat3_value                      = 25
+|level2_stat1_id                         = ice_spear_second_form_damage_+%
+|level2_stat1_value                      = 25
+|level2_stat2_id                         = spell_minimum_base_cold_damage
+|level2_stat2_value                      = 595
+|level2_stat3_id                         = spell_maximum_base_cold_damage
+|level2_stat3_value                      = 893
 }}