--- Skill:ZanaRoseScourgeProjectile
+++ Export
@@ -8,15 +8,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_projectile_variation
-|static_stat1_value                      = 80
-|static_stat2_id                         = always_pierce
-|static_stat2_value                      = 1
-|static_stat3_id                         = base_is_projectile
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_projectile_variation
+|static_stat2_value                      = 80
+|static_stat3_id                         = projectile_uses_contact_position
 |static_stat3_value                      = 1
-|static_stat4_id                         = projectile_uses_contact_position
+|static_stat4_id                         = always_pierce
 |static_stat4_value                      = 1
-|stat_text                               = Deals (6-42207) to (9-63311) Physical Damage<br>Deals (250-0)% more Damage<br>Projectiles Pierce all Targets
+|stat_text                               = Deals (6-42207) to (9-63311) Physical Damage<br>Projectiles Pierce all Targets<br>Deals (250-0)% more Damage
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_stat_text                        = Deals 6 to 9 Physical Damage<br>Deals 250% more Damage
