--- Skill:LegionHyrriSnipeClones
+++ Export
@@ -9,19 +9,19 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 200
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = skill_visual_scale_+%
-|static_stat1_value                      = -40
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
 |static_stat2_id                         = monster_projectile_variation
 |static_stat2_value                      = 71
-|static_stat3_id                         = skill_physical_damage_%_to_convert_to_cold
-|static_stat3_value                      = 75
-|static_stat4_id                         = active_skill_attack_speed_+%_final
-|static_stat4_value                      = -59
-|static_stat5_id                         = always_pierce
-|static_stat5_value                      = 1
-|static_stat6_id                         = base_is_projectile
-|static_stat6_value                      = 1
-|stat_text                               = Deals (0-200)% more Damage<br>75% of Physical Damage Converted to Cold Damage<br>59% less Attack Speed<br>Projectiles Pierce all Targets
+|static_stat3_id                         = skill_visual_scale_+%
+|static_stat3_value                      = -40
+|static_stat4_id                         = always_pierce
+|static_stat4_value                      = 1
+|static_stat5_id                         = skill_physical_damage_%_to_convert_to_cold
+|static_stat5_value                      = 75
+|static_stat6_id                         = active_skill_attack_speed_+%_final
+|static_stat6_value                      = -59
+|stat_text                               = Projectiles Pierce all Targets<br>Deals (0-200)% more Damage<br>75% of Physical Damage Converted to Cold Damage<br>59% less Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 1
 |level2                                  = True
