--- Skill:SupportGreaterSpellEcho
+++ Export
@@ -2,6 +2,7 @@
 |skill_id                                = SupportGreaterSpellEcho
 |support_gem_letter                      = h
 |required_level                          = 90
+|static_cost_multiplier                  = 150
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
