--- Skill:BlightTowerAuraBuffTowers3
+++ Export
@@ -9,12 +9,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = blight_aura_stat_3
-|static_stat1_value                      = 3
+|static_stat1_id                         = blight_aura_stat_1
+|static_stat1_value                      = 150
 |static_stat2_id                         = blight_aura_stat_2
 |static_stat2_value                      = 40
-|static_stat3_id                         = blight_aura_stat_1
-|static_stat3_value                      = 150
+|static_stat3_id                         = blight_aura_stat_3
+|static_stat3_value                      = 3
 |stat_text                               = 
 |level1                                  = True
 |level1_level_requirement                = 1
