--- Skill:AncestorLeagueIceCasterFrostblink
+++ Export
@@ -0,0 +1,23 @@
+{{Skill
+|gem_description                         = Teleport to a location, damaging enemies and leaving Chilled ground in an area at both ends of the teleport. Shares a cooldown with other Blink skills.
+|active_skill_name                       = Frostblink
+|skill_id                                = AncestorLeagueIceCasterFrostblink
+|cast_time                               = 0.0
+|required_level                          = 1
+|static_critical_strike_chance           = 5
+|static_damage_effectiveness             = 100
+|static_damage_multiplier                = 100
+|static_attack_speed_multiplier          = 100
+|static_duration                         = 3
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_cold_damage
+|static_stat2_value                      = 6
+|static_stat3_id                         = spell_maximum_base_cold_damage
+|static_stat3_value                      = 9
+|static_stat4_id                         = base_skill_effect_duration
+|static_stat4_value                      = 3000
+|stat_text                               = Deals 6 to 9 Cold Damage<br>Base duration is 3 seconds
+|level1                                  = True
+|level1_level_requirement                = 1
+}}