--- Skill:MonsterFireballGoatmanBoss
+++ Export
@@ -10,15 +10,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_reverse_point_blank_damage_-%_at_minimum_range
-|static_stat1_value                      = 30
-|static_stat2_id                         = base_is_projectile
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = maintain_projectile_direction_when_using_contact_position
 |static_stat2_value                      = 1
-|static_stat3_id                         = use_scaled_contact_offset
-|static_stat3_value                      = 1
+|static_stat3_id                         = monster_reverse_point_blank_damage_-%_at_minimum_range
+|static_stat3_value                      = 30
 |static_stat4_id                         = projectile_uses_contact_position
 |static_stat4_value                      = 1
-|static_stat5_id                         = maintain_projectile_direction_when_using_contact_position
+|static_stat5_id                         = use_scaled_contact_offset
 |static_stat5_value                      = 1
 |stat_text                               = Deals (4-204) to (6-306) Fire Damage<br>Skills Repeat an additional (3-5) Times<br>(75-250)% more Cast Speed<br>Fires (0-4) additional Projectiles
 |level1                                  = True
