--- Skill:WestLightningStormCascade
+++ Export
@@ -10,23 +10,24 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage
-|static_stat1_value                      = 791
-|static_stat2_id                         = spell_maximum_base_lightning_damage
-|static_stat2_value                      = 2372
-|static_stat3_id                         = skill_override_pvp_scaling_time_ms
-|static_stat3_value                      = 450
-|static_stat4_id                         = base_secondary_skill_effect_duration
-|static_stat4_value                      = 500
-|static_stat5_id                         = upheaval_number_of_spikes
-|static_stat5_value                      = 20
-|static_stat6_id                         = fire_storm_fireball_delay_ms
-|static_stat6_value                      = 250
-|static_stat7_id                         = base_skill_effect_duration
-|static_stat7_value                      = 1500
-|static_stat8_id                         = is_area_damage
-|static_stat8_value                      = 1
-|stat_text                               = Deals 791 to 2372 Lightning Damage<br>Base secondary duration is 0.5 seconds<br>Causes 20 Bursts<br>One impact every 0.25 seconds<br>Base duration is 1.5 seconds
+|static_duration                         = 1.5
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = skill_override_pvp_scaling_time_ms
+|static_stat2_value                      = 450
+|static_stat3_id                         = spell_minimum_base_lightning_damage
+|static_stat3_value                      = 791
+|static_stat4_id                         = spell_maximum_base_lightning_damage
+|static_stat4_value                      = 2372
+|static_stat5_id                         = base_skill_effect_duration
+|static_stat5_value                      = 1500
+|static_stat6_id                         = base_secondary_skill_effect_duration
+|static_stat6_value                      = 500
+|static_stat7_id                         = fire_storm_fireball_delay_ms
+|static_stat7_value                      = 250
+|static_stat8_id                         = upheaval_number_of_spikes
+|static_stat8_value                      = 20
+|stat_text                               = Deals 791 to 2372 Lightning Damage<br>Base duration is 1.5 seconds<br>Base secondary duration is 0.5 seconds<br>One impact every 0.25 seconds<br>Causes 20 Bursts
 |level1                                  = True
 |level1_level_requirement                = 79
 }}