--- Skill:StormCloudLightningWarp
+++ Export
@@ -10,14 +10,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage
-|static_stat1_value                      = 277
-|static_stat2_id                         = spell_maximum_base_lightning_damage
-|static_stat2_value                      = 831
-|static_stat3_id                         = skill_effect_duration_+%
-|static_stat3_value                      = -30
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_lightning_damage
+|static_stat2_value                      = 277
+|static_stat3_id                         = spell_maximum_base_lightning_damage
+|static_stat3_value                      = 831
+|static_stat4_id                         = skill_effect_duration_+%
+|static_stat4_value                      = -30
 |stat_text                               = Deals 277 to 831 Lightning Damage<br>30% reduced Skill Effect Duration
 |level1                                  = True
 |level1_level_requirement                = 77
