--- Skill:BrineKingFloodArenaMap
+++ Export
@@ -7,15 +7,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_cold_damage_to_deal_per_minute
-|static_stat1_value                      = 114356
-|static_stat2_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat2_value                      = -50
-|static_stat3_id                         = spell_minimum_base_lightning_damage
-|static_stat3_value                      = 682
-|static_stat4_id                         = spell_maximum_base_lightning_damage
-|static_stat4_value                      = 909
-|stat_text                               = Deals 1905.9 Base Cold Damage per second<br>Deals 682 to 909 Lightning Damage
+|static_stat1_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat1_value                      = -50
+|static_stat2_id                         = spell_minimum_base_lightning_damage
+|static_stat2_value                      = 682
+|static_stat3_id                         = spell_maximum_base_lightning_damage
+|static_stat3_value                      = 909
+|static_stat4_id                         = base_cold_damage_to_deal_per_minute
+|static_stat4_value                      = 114356
+|stat_text                               = Deals 682 to 909 Lightning Damage<br>Deals 1905.9 Base Cold Damage per second
 |level1                                  = True
 |level1_level_requirement                = 50
 }}