--- Skill:IceElementalSpearRepeated
+++ Export
@@ -9,15 +9,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_cold_damage
-|static_stat1_value                      = 5
-|static_stat2_id                         = spell_maximum_base_cold_damage
-|static_stat2_value                      = 7
-|static_stat3_id                         = monster_reverse_point_blank_damage_-%_at_minimum_range
-|static_stat3_value                      = 25
-|static_stat4_id                         = skill_repeat_count
-|static_stat4_value                      = 1
-|static_stat5_id                         = base_is_projectile
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_reverse_point_blank_damage_-%_at_minimum_range
+|static_stat2_value                      = 25
+|static_stat3_id                         = spell_minimum_base_cold_damage
+|static_stat3_value                      = 5
+|static_stat4_id                         = spell_maximum_base_cold_damage
+|static_stat4_value                      = 7
+|static_stat5_id                         = skill_repeat_count
 |static_stat5_value                      = 1
 |stat_text                               = Deals 5 to 7 Cold Damage<br>Skills Repeat an additional Time
 |level1                                  = True
