--- Skill:MonsterShockwaveTotem
+++ Export
@@ -9,11 +9,11 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = -17
-|static_stat2_id                         = global_knockback
+|static_stat2_id                         = base_skill_is_totemified
 |static_stat2_value                      = 1
-|static_stat3_id                         = base_skill_is_totemified
+|static_stat3_id                         = is_area_damage
 |static_stat3_value                      = 1
-|static_stat4_id                         = is_area_damage
+|static_stat4_id                         = global_knockback
 |static_stat4_value                      = 1
 |stat_text                               = Deals (1-61) to (3-114) Physical Damage<br>Knocks Enemies Back on Hit
 |level1                                  = True
