--- Skill:SandstormChaosElemental
+++ Export
@@ -9,14 +9,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_chaos_damage_to_deal_per_minute
-|static_stat1_value                      = 528
-|static_stat2_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat2_value                      = 20
-|static_stat3_id                         = base_skill_effect_duration
-|static_stat3_value                      = 5000
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
+|static_duration                         = 5
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = 20
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = base_chaos_damage_to_deal_per_minute
+|static_stat3_value                      = 528
+|static_stat4_id                         = base_skill_effect_duration
+|static_stat4_value                      = 5000
 |stat_text                               = Deals 8.8 Base Chaos Damage per second<br>Base duration is 5 seconds
 |level1                                  = True
 |level1_level_requirement                = 3
