--- Skill:ThrownWeaponBreachBoss
+++ Export
@@ -8,15 +8,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 110
-|static_stat1_id                         = active_skill_damage_+%_final
-|static_stat1_value                      = -10
-|static_stat2_id                         = attack_speed_+%
-|static_stat2_value                      = 35
-|static_stat3_id                         = spectral_throw_projectile_deceleration_+%
-|static_stat3_value                      = -75
-|static_stat4_id                         = base_is_projectile
-|static_stat4_value                      = 1
-|stat_text                               = Deals 10% less Damage<br>35% increased Attack Speed<br>75% reduced Spectral Throw Projectile Deceleration
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = spectral_throw_projectile_deceleration_+%
+|static_stat2_value                      = -75
+|static_stat3_id                         = active_skill_damage_+%_final
+|static_stat3_value                      = -10
+|static_stat4_id                         = attack_speed_+%
+|static_stat4_value                      = 35
+|stat_text                               = 75% reduced Spectral Throw Projectile Deceleration<br>Deals 10% less Damage<br>35% increased Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 10
 }}