--- Skill:BloodElementalGoreSunder
+++ Export
@@ -12,12 +12,12 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = -50
-|static_stat2_id                         = active_skill_attack_speed_+%_final
-|static_stat2_value                      = -10
-|static_stat3_id                         = shockwave_slam_explosion_damage_+%_final
-|static_stat3_value                      = -80
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = active_skill_attack_speed_+%_final
+|static_stat3_value                      = -10
+|static_stat4_id                         = shockwave_slam_explosion_damage_+%_final
+|static_stat4_value                      = -80
 |stat_text                               = 10% less Attack Speed<br>Shockwaves deal 80% less Damage
 |level1                                  = True
 |level1_level_requirement                = 1
