--- Skill:CockerelMapBossSplitArrow
+++ Export
@@ -8,12 +8,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_damage_+%_final
-|static_stat1_value                      = -20
-|static_stat2_id                         = base_number_of_additional_arrows
-|static_stat2_value                      = 5
-|static_stat3_id                         = skill_can_fire_arrows
-|static_stat3_value                      = 1
+|static_stat1_id                         = skill_can_fire_arrows
+|static_stat1_value                      = 1
+|static_stat2_id                         = active_skill_damage_+%_final
+|static_stat2_value                      = -20
+|static_stat3_id                         = base_number_of_additional_arrows
+|static_stat3_value                      = 5
 |stat_text                               = Deals 20% less Damage<br>Fires 5 additional Arrows
 |level1                                  = True
 |level1_level_requirement                = 66
