--- Skill:IzaroChronosStatueBallLightning
+++ Export
@@ -12,19 +12,19 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = -33
-|static_stat2_id                         = base_projectile_speed_+%
-|static_stat2_value                      = -20
-|static_stat3_id                         = number_of_additional_projectiles
-|static_stat3_value                      = 14
-|static_stat4_id                         = projectiles_nova
+|static_stat2_id                         = base_is_projectile
+|static_stat2_value                      = 1
+|static_stat3_id                         = grant_izaro_buff_on_skill_use
+|static_stat3_value                      = 1
+|static_stat4_id                         = set_user_life_to_1_on_skill_use
 |static_stat4_value                      = 1
-|static_stat5_id                         = base_is_projectile
+|static_stat5_id                         = projectiles_nova
 |static_stat5_value                      = 1
-|static_stat6_id                         = set_user_life_to_1_on_skill_use
-|static_stat6_value                      = 1
-|static_stat7_id                         = grant_izaro_buff_on_skill_use
-|static_stat7_value                      = 1
-|stat_text                               = Deals (1-90) to (3-269) Lightning Damage<br>(4-37)% increased Cast Speed<br>20% reduced Projectile Speed<br>Fires 14 additional Projectiles<br>Fires Projectiles in a circle
+|static_stat6_id                         = base_projectile_speed_+%
+|static_stat6_value                      = -20
+|static_stat7_id                         = number_of_additional_projectiles
+|static_stat7_value                      = 14
+|stat_text                               = Deals (1-90) to (3-269) Lightning Damage<br>Fires Projectiles in a circle<br>(4-37)% increased Cast Speed<br>20% reduced Projectile Speed<br>Fires 14 additional Projectiles
 |level1                                  = True
 |level1_level_requirement                = 3
 |level1_stat_text                        = Deals 1 to 3 Lightning Damage<br>4% increased Cast Speed
