--- Skill:LieutenantRainOfArrows
+++ Export
@@ -10,17 +10,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_chance_to_shock_%
-|static_stat1_value                      = 25
-|static_stat2_id                         = skill_physical_damage_%_to_convert_to_lightning
-|static_stat2_value                      = 25
-|static_stat3_id                         = active_skill_damage_+%_final
-|static_stat3_value                      = 80
-|static_stat4_id                         = base_is_projectile
-|static_stat4_value                      = 1
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
-|stat_text                               = 25% chance to Shock enemies<br>25% of Physical Damage Converted to Lightning Damage<br>Deals 80% more Damage
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = skill_physical_damage_%_to_convert_to_lightning
+|static_stat3_value                      = 25
+|static_stat4_id                         = active_skill_damage_+%_final
+|static_stat4_value                      = 80
+|static_stat5_id                         = base_chance_to_shock_%
+|static_stat5_value                      = 25
+|stat_text                               = 25% of Physical Damage Converted to Lightning Damage<br>Deals 80% more Damage<br>25% chance to Shock enemies
 |level1                                  = True
 |level1_level_requirement                = 7
 |level1_stat1_id                         = active_skill_area_of_effect_radius_+%_final
