--- Skill:BetrayalGraviciusWeaponThrow
+++ Export
@@ -10,20 +10,20 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = 50
-|static_stat2_id                         = number_of_projectiles_override
+|static_stat2_id                         = base_is_projectile
 |static_stat2_value                      = 1
-|static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat3_value                      = -50
-|static_stat4_id                         = skill_physical_damage_%_to_convert_to_fire
-|static_stat4_value                      = 75
-|static_stat5_id                         = is_area_damage
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat4_value                      = -50
+|static_stat5_id                         = number_of_projectiles_override
 |static_stat5_value                      = 1
-|static_stat6_id                         = base_is_projectile
+|static_stat6_id                         = projectile_uses_contact_position
 |static_stat6_value                      = 1
 |static_stat7_id                         = triggered_skill_use_animation_event
 |static_stat7_value                      = 1
-|static_stat8_id                         = projectile_uses_contact_position
-|static_stat8_value                      = 1
+|static_stat8_id                         = skill_physical_damage_%_to_convert_to_fire
+|static_stat8_value                      = 75
 |stat_text                               = Deals (11-2523) to (17-3784) Physical Damage<br>(0-300)% increased Burning Damage<br>75% of Physical Damage Converted to Fire Damage
 |level1                                  = True
 |level1_level_requirement                = 1
