--- Skill:TotemMapBossLightningMortarProjectile
+++ Export
@@ -9,14 +9,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = projectile_speed_variation_+%
-|static_stat1_value                      = 10
-|static_stat2_id                         = projectile_spread_radius
-|static_stat2_value                      = 20
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
-|static_stat4_id                         = base_is_projectile
-|static_stat4_value                      = 1
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = projectile_speed_variation_+%
+|static_stat3_value                      = 10
+|static_stat4_id                         = projectile_spread_radius
+|static_stat4_value                      = 20
 |static_stat5_id                         = projectiles_not_offset
 |static_stat5_value                      = 1
 |stat_text                               = Deals (8-1141) to (12-1711) Lightning Damage<br>Skills Repeat an additional (4-6) Times<br>(80-100)% increased Cast Speed
