--- Skill:ReliquaryMonsterProjectile
+++ Export
@@ -7,19 +7,19 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_maximum_action_distance_+%
-|static_stat1_value                      = -30
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
 |static_stat2_id                         = monster_projectile_variation
 |static_stat2_value                      = 7
-|static_stat3_id                         = number_of_additional_projectiles
-|static_stat3_value                      = 2
-|static_stat4_id                         = active_skill_attack_speed_+%_final
-|static_stat4_value                      = -30
-|static_stat5_id                         = base_projectile_speed_+%
-|static_stat5_value                      = 30
-|static_stat6_id                         = base_is_projectile
-|static_stat6_value                      = 1
-|stat_text                               = Fires 2 additional Projectiles<br>30% less Attack Speed<br>30% increased Projectile Speed
+|static_stat3_id                         = spell_maximum_action_distance_+%
+|static_stat3_value                      = -30
+|static_stat4_id                         = base_projectile_speed_+%
+|static_stat4_value                      = 30
+|static_stat5_id                         = number_of_additional_projectiles
+|static_stat5_value                      = 2
+|static_stat6_id                         = active_skill_attack_speed_+%_final
+|static_stat6_value                      = -30
+|stat_text                               = 30% increased Projectile Speed<br>Fires 2 additional Projectiles<br>30% less Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 1
 }}