--- Skill:BirdAspect
+++ Export
@@ -9,20 +9,21 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 4
 |static_mana_reservation_percent         = 25
-|static_stat1_id                         = base_secondary_skill_effect_duration
+|static_stat1_id                         = base_skill_effect_duration
 |static_stat1_value                      = 4000
-|static_stat2_id                         = base_skill_effect_duration
+|static_stat2_id                         = base_secondary_skill_effect_duration
 |static_stat2_value                      = 4000
-|static_stat3_id                         = minion_movement_speed_+%
+|static_stat3_id                         = base_movement_velocity_+%
 |static_stat3_value                      = 10
-|static_stat4_id                         = base_movement_velocity_+%
+|static_stat4_id                         = minion_movement_speed_+%
 |static_stat4_value                      = 10
-|static_stat5_id                         = minion_chance_to_deal_double_damage_%
+|static_stat5_id                         = chance_to_deal_double_damage_%
 |static_stat5_value                      = 10
-|static_stat6_id                         = chance_to_deal_double_damage_%
+|static_stat6_id                         = minion_chance_to_deal_double_damage_%
 |static_stat6_value                      = 10
-|stat_text                               = Base secondary duration is 4 seconds<br>Base duration is 4 seconds<br>10% increased Minion Movement Speed<br>10% increased Movement Speed<br>Minions gain 10% chance to deal Double Damage<br>10% chance to deal Double Damage
+|stat_text                               = Base duration is 4 seconds<br>Base secondary duration is 4 seconds<br>10% increased Movement Speed<br>10% increased Minion Movement Speed<br>10% chance to deal Double Damage<br>Minions gain 10% chance to deal Double Damage
 |level1                                  = True
 |level1_level_requirement                = 1
 }}