--- Skill:HarvestHammerVineDonutGA3
+++ Export
@@ -7,11 +7,11 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 400
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = global_poison_on_hit
+|static_stat1_id                         = is_area_damage
 |static_stat1_value                      = 1
-|static_stat2_id                         = is_area_damage
+|static_stat2_id                         = is_oshabi_vivid_skill
 |static_stat2_value                      = 1
-|static_stat3_id                         = is_oshabi_vivid_skill
+|static_stat3_id                         = global_poison_on_hit
 |static_stat3_value                      = 1
 |stat_text                               = Poisons Enemies on Hit
 |level1                                  = True
