--- Skill:HarvestRavenBossDashOfDestruction
+++ Export
@@ -9,10 +9,10 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 75
-|static_stat1_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat1_value                      = -25
-|static_stat2_id                         = main_hand_base_maximum_attack_distance
-|static_stat2_value                      = 20
+|static_stat1_id                         = main_hand_base_maximum_attack_distance
+|static_stat1_value                      = 20
+|static_stat2_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat2_value                      = -25
 |static_stat3_id                         = skill_cannot_be_knocked_back
 |static_stat3_value                      = 1
 |static_stat4_id                         = skill_cannot_be_stunned
