--- Skill:GroundSlamSkeletonBossGeofri
+++ Export
@@ -9,17 +9,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_attack_speed_+%_final
-|static_stat1_value                      = -40
-|static_stat2_id                         = active_skill_damage_+%_final
-|static_stat2_value                      = 280
-|static_stat3_id                         = skill_physical_damage_%_to_convert_to_fire
-|static_stat3_value                      = 50
-|static_stat4_id                         = global_cannot_crit
-|static_stat4_value                      = 1
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
-|stat_text                               = 40% less Attack Speed<br>Deals 280% more Damage<br>50% of Physical Damage Converted to Fire Damage<br>Never deal Critical Strikes
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = global_cannot_crit
+|static_stat2_value                      = 1
+|static_stat3_id                         = active_skill_damage_+%_final
+|static_stat3_value                      = 280
+|static_stat4_id                         = skill_physical_damage_%_to_convert_to_fire
+|static_stat4_value                      = 50
+|static_stat5_id                         = active_skill_attack_speed_+%_final
+|static_stat5_value                      = -40
+|stat_text                               = Never deal Critical Strikes<br>Deals 280% more Damage<br>50% of Physical Damage Converted to Fire Damage<br>40% less Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 68
 }}