--- Skill:VaalGuardMeleeLightning
+++ Export
@@ -7,23 +7,23 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 125
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = arrow_projectile_variation
-|static_stat1_value                      = 32
-|static_stat2_id                         = skill_physical_damage_%_to_convert_to_lightning
-|static_stat2_value                      = 50
-|static_stat3_id                         = projectile_chain_from_terrain_chance_%
-|static_stat3_value                      = 35
-|static_stat4_id                         = number_of_chains
+|static_stat1_id                         = action_attack_or_cast_time_uses_animation_length
+|static_stat1_value                      = 1
+|static_stat2_id                         = arrow_projectile_variation
+|static_stat2_value                      = 32
+|static_stat3_id                         = skill_can_fire_arrows
+|static_stat3_value                      = 1
+|static_stat4_id                         = skill_can_fire_wand_projectiles
 |static_stat4_value                      = 1
-|static_stat5_id                         = skill_can_fire_arrows
+|static_stat5_id                         = use_scaled_contact_offset
 |static_stat5_value                      = 1
-|static_stat6_id                         = skill_can_fire_wand_projectiles
-|static_stat6_value                      = 1
-|static_stat7_id                         = action_attack_or_cast_time_uses_animation_length
-|static_stat7_value                      = 1
-|static_stat8_id                         = use_scaled_contact_offset
+|static_stat6_id                         = projectile_chain_from_terrain_chance_%
+|static_stat6_value                      = 35
+|static_stat7_id                         = skill_physical_damage_%_to_convert_to_lightning
+|static_stat7_value                      = 50
+|static_stat8_id                         = number_of_chains
 |static_stat8_value                      = 1
-|stat_text                               = 50% of Physical Damage Converted to Lightning Damage<br>Projectiles have 35% chance to be able to Chain when colliding with terrain<br>Chains +1 Times
+|stat_text                               = Projectiles have 35% chance to be able to Chain when colliding with terrain<br>50% of Physical Damage Converted to Lightning Damage<br>Chains +1 Times
 |level1                                  = True
 |level1_level_requirement                = 1
 }}