--- Skill:DeathExplodeRootSpiderMinion
+++ Export
@@ -9,13 +9,13 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = -43
-|static_stat2_id                         = cast_on_death_%
-|static_stat2_value                      = 100
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
 |static_stat3_id                         = spell_uncastable_if_triggerable
 |static_stat3_value                      = 1
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
-|stat_text                               = Deals (995-2847) to (1327-3795) Fire Damage<br>100% chance to Trigger this Spell on Death<br>You cannot Cast this Spell directly
+|static_stat4_id                         = cast_on_death_%
+|static_stat4_value                      = 100
+|stat_text                               = Deals (995-2847) to (1327-3795) Fire Damage<br>You cannot Cast this Spell directly<br>100% chance to Trigger this Spell on Death
 |level1                                  = True
 |level1_level_requirement                = 53
 |level1_stat_text                        = Deals 995 to 1327 Fire Damage
