--- Skill:FireTailSpark
+++ Export
@@ -7,17 +7,20 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_fire_damage
-|static_stat1_value                      = 6
-|static_stat2_id                         = spell_maximum_base_fire_damage
-|static_stat2_value                      = 9
-|static_stat3_id                         = number_of_additional_projectiles
-|static_stat3_value                      = 8
-|static_stat4_id                         = base_skill_effect_duration
-|static_stat4_value                      = 2000
-|static_stat5_id                         = projectiles_nova
-|static_stat5_value                      = 1
-|stat_text                               = Deals 6 to 9 Fire Damage<br>Fires 8 additional Projectiles<br>Base duration is 2 seconds<br>Fires Projectiles in a circle
+|static_duration                         = 2
+|static_stat1_id                         = base_skill_effect_duration
+|static_stat1_value                      = 2000
+|static_stat2_id                         = spell_minimum_base_fire_damage
+|static_stat2_value                      = 6
+|static_stat3_id                         = spell_maximum_base_fire_damage
+|static_stat3_value                      = 9
+|static_stat4_id                         = projectiles_nova
+|static_stat4_value                      = 1
+|static_stat5_id                         = base_projectile_speed_+%
+|static_stat5_value                      = 25
+|static_stat6_id                         = number_of_additional_projectiles
+|static_stat6_value                      = 8
+|stat_text                               = Base duration is 2 seconds<br>Deals 6 to 9 Fire Damage<br>Fires Projectiles in a circle<br>25% increased Projectile Speed<br>Fires 8 additional Projectiles
 |level1                                  = True
 |level1_level_requirement                = 1
 }}