--- Skill:BlightTowerStunning~buff1
+++ Export
@@ -9,23 +9,24 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
+|static_duration                         = 7.5
+|static_stat1_id                         = is_area_damage
 |static_stat1_value                      = 1
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 2
-|static_stat3_id                         = base_skill_effect_duration
-|static_stat3_value                      = 7500
-|static_stat4_id                         = base_stun_duration_+%
-|static_stat4_value                      = 50
-|static_stat5_id                         = phys_cascade_trap_base_interval_duration_ms
-|static_stat5_value                      = 700
-|static_stat6_id                         = phys_cascade_trap_number_of_cascades
-|static_stat6_value                      = 4
-|static_stat7_id                         = always_stun
-|static_stat7_value                      = 1
-|static_stat8_id                         = is_area_damage
-|static_stat8_value                      = 1
-|stat_text                               = Deals 1 to 2 Physical Damage<br>Base duration is 7.5 seconds<br>50% increased Stun Duration on enemies<br>Releases waves every 0.7 seconds<br>Releases 4 waves<br>Stuns Enemies
+|static_stat2_id                         = spell_minimum_base_physical_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = spell_maximum_base_physical_damage
+|static_stat3_value                      = 2
+|static_stat4_id                         = base_skill_effect_duration
+|static_stat4_value                      = 7500
+|static_stat5_id                         = base_stun_duration_+%
+|static_stat5_value                      = 50
+|static_stat6_id                         = always_stun
+|static_stat6_value                      = 1
+|static_stat7_id                         = phys_cascade_trap_base_interval_duration_ms
+|static_stat7_value                      = 700
+|static_stat8_id                         = phys_cascade_trap_number_of_cascades
+|static_stat8_value                      = 4
+|stat_text                               = Deals 1 to 2 Physical Damage<br>Base duration is 7.5 seconds<br>50% increased Stun Duration on enemies<br>Stuns Enemies<br>Releases waves every 0.7 seconds<br>Releases 4 waves
 |level1                                  = True
 |level1_level_requirement                = 1
 }}