--- Skill:TalismanT2SummonLocustSwarmLightning
+++ Export
@@ -8,15 +8,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = number_of_monsters_to_summon
-|static_stat1_value                      = 1
-|static_stat2_id                         = alternate_minion
-|static_stat2_value                      = 91
-|static_stat3_id                         = summoned_monsters_no_drops_or_experience
+|static_stat1_id                         = alternate_minion
+|static_stat1_value                      = 91
+|static_stat2_id                         = minion_dies_when_parent_dies
+|static_stat2_value                      = 1
+|static_stat3_id                         = number_of_monsters_to_summon
 |static_stat3_value                      = 1
 |static_stat4_id                         = summoned_monsters_are_minions
 |static_stat4_value                      = 1
-|static_stat5_id                         = minion_dies_when_parent_dies
+|static_stat5_id                         = summoned_monsters_no_drops_or_experience
 |static_stat5_value                      = 1
 |stat_text                               = 
 |level1                                  = True
