--- Skill:GladiatorBossLeapSlam
+++ Export
@@ -17,6 +17,8 @@
 |stat_text                               = Deals 25% more Damage
 |level1                                  = True
 |level1_level_requirement                = 38
+|level1_damage_effectiveness             = 100
+|level1_damage_multiplier                = 100
 |level2                                  = True
 |level2_level_requirement                = 56
 |level2_damage_effectiveness             = 140