--- Skill:InsectSpawnerSpit
+++ Export
@@ -7,13 +7,13 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = attack_minimum_added_fire_damage
+|static_stat1_id                         = base_is_projectile
 |static_stat1_value                      = 1
-|static_stat2_id                         = attack_maximum_added_fire_damage
-|static_stat2_value                      = 1
-|static_stat3_id                         = monster_reverse_point_blank_damage_-%_at_minimum_range
-|static_stat3_value                      = 30
-|static_stat4_id                         = base_is_projectile
+|static_stat2_id                         = monster_reverse_point_blank_damage_-%_at_minimum_range
+|static_stat2_value                      = 30
+|static_stat3_id                         = attack_minimum_added_fire_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = attack_maximum_added_fire_damage
 |static_stat4_value                      = 1
 |static_stat5_id                         = attack_minimum_added_physical_damage
 |static_stat5_value                      = 2
