--- Skill:BrineKingCleaveRightMap
+++ Export
@@ -9,11 +9,11 @@
 |static_damage_effectiveness             = 200
 |static_damage_multiplier                = 150
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat1_value                      = -50
-|static_stat2_id                         = global_maim_on_hit
-|static_stat2_value                      = 1
-|static_stat3_id                         = is_area_damage
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat2_value                      = -50
+|static_stat3_id                         = global_maim_on_hit
 |static_stat3_value                      = 1
 |stat_text                               = Maim on Hit
 |level1                                  = True
