--- Skill:BreachBossFrostBlades
+++ Export
@@ -10,15 +10,15 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_damage_+%_final
 |static_stat1_value                      = 100
-|static_stat2_id                         = active_skill_attack_speed_+%_final
-|static_stat2_value                      = 35
-|static_stat3_id                         = number_of_additional_projectiles
-|static_stat3_value                      = 2
-|static_stat4_id                         = melee_weapon_range_+
-|static_stat4_value                      = 14
-|static_stat5_id                         = skill_physical_damage_%_to_convert_to_cold
-|static_stat5_value                      = 50
-|stat_text                               = Deals 100% more Damage<br>35% more Attack Speed<br>Fires 2 additional Projectiles<br>+14 to Melee Strike Range<br>50% of Physical Damage Converted to Cold Damage
+|static_stat2_id                         = number_of_additional_projectiles
+|static_stat2_value                      = 2
+|static_stat3_id                         = skill_physical_damage_%_to_convert_to_cold
+|static_stat3_value                      = 50
+|static_stat4_id                         = active_skill_attack_speed_+%_final
+|static_stat4_value                      = 35
+|static_stat5_id                         = melee_weapon_range_+
+|static_stat5_value                      = 14
+|stat_text                               = Deals 100% more Damage<br>Fires 2 additional Projectiles<br>50% of Physical Damage Converted to Cold Damage<br>35% more Attack Speed<br>+14 to Melee Strike Range
 |level1                                  = True
 |level1_level_requirement                = 1
 }}