--- Skill:BreachFrostBlades
+++ Export
@@ -16,7 +16,7 @@
 |static_stat2_value                      = 50
 |static_stat3_id                         = melee_weapon_range_+
 |static_stat3_value                      = 14
-|stat_text                               = Deals (0-25)% more Damage<br>Fires 2 additional Projectiles<br>50% of Physical Damage Converted to Cold Damage<br>+14 to Melee Strike Range
+|stat_text                               = Deals (0-25)% more Damage<br>Fires 2 additional Projectiles<br>50% of Physical Damage Converted to Cold Damage<br>+1.4 metres to Melee Strike Range
 |level1                                  = True
 |level1_level_requirement                = 1
 |level2                                  = True