--- Skill:AtziriFlameblastRitual
+++ Export
@@ -9,21 +9,21 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_fire_damage
-|static_stat1_value                      = 3
-|static_stat2_id                         = spell_maximum_base_fire_damage
-|static_stat2_value                      = 5
-|static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat3_value                      = -60
-|static_stat4_id                         = flameblast_ailment_damage_+%_final_per_stack
-|static_stat4_value                      = 110
-|static_stat5_id                         = charged_blast_spell_damage_+%_final_per_stack
-|static_stat5_value                      = 110
-|static_stat6_id                         = always_ignite
-|static_stat6_value                      = 1
-|static_stat7_id                         = is_area_damage
-|static_stat7_value                      = 1
-|stat_text                               = Deals 3 to 5 Fire Damage<br>110% more Damage with Ailments for each stage<br>110% more Spell Damage for each stage<br>Hits always Ignite
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat2_value                      = -60
+|static_stat3_id                         = spell_minimum_base_fire_damage
+|static_stat3_value                      = 3
+|static_stat4_id                         = spell_maximum_base_fire_damage
+|static_stat4_value                      = 5
+|static_stat5_id                         = always_ignite
+|static_stat5_value                      = 1
+|static_stat6_id                         = charged_blast_spell_damage_+%_final_per_stack
+|static_stat6_value                      = 110
+|static_stat7_id                         = flameblast_ailment_damage_+%_final_per_stack
+|static_stat7_value                      = 110
+|stat_text                               = Deals 3 to 5 Fire Damage<br>Hits always Ignite<br>110% more Spell Damage for each stage<br>110% more Damage with Ailments for each stage
 |level1                                  = True
 |level1_level_requirement                = 1
 }}