--- Skill:HarbingerLightningMortar
+++ Export
@@ -10,18 +10,18 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = projectile_speed_variation_+%
-|static_stat1_value                      = 10
-|static_stat2_id                         = projectile_spread_radius
-|static_stat2_value                      = 20
-|static_stat3_id                         = number_of_additional_projectiles
-|static_stat3_value                      = 2
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
-|static_stat5_id                         = base_is_projectile
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = projectile_speed_variation_+%
+|static_stat3_value                      = 10
+|static_stat4_id                         = projectile_spread_radius
+|static_stat4_value                      = 20
+|static_stat5_id                         = projectiles_not_offset
 |static_stat5_value                      = 1
-|static_stat6_id                         = projectiles_not_offset
-|static_stat6_value                      = 1
+|static_stat6_id                         = number_of_additional_projectiles
+|static_stat6_value                      = 2
 |stat_text                               = Deals (251-527) to (376-790) Lightning Damage<br>Fires 2 additional Projectiles
 |level1                                  = True
 |level1_level_requirement                = 55
