--- Skill:WbPhaseRun
+++ Export
@@ -9,9 +9,10 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 10
 |static_stat1_id                         = base_skill_effect_duration
 |static_stat1_value                      = 10000
-|stat_text                               = (15-30)% increased Movement Speed<br>(50-70)% more Melee Physical Damage<br>Base duration is 10 seconds
+|stat_text                               = Base duration is 10 seconds<br>(15-30)% increased Movement Speed<br>(50-70)% more Melee Physical Damage
 |level1                                  = True
 |level1_level_requirement                = 15
 |level1_stat_text                        = 15% increased Movement Speed<br>50% more Melee Physical Damage
