--- Skill:SynthesisGolemBossWhipRight
+++ Export
@@ -7,15 +7,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_attack_speed_+%_final
-|static_stat1_value                      = -25
-|static_stat2_id                         = active_skill_damage_+%_final
-|static_stat2_value                      = 120
-|static_stat3_id                         = skill_physical_damage_%_to_convert_to_lightning
-|static_stat3_value                      = 50
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
-|stat_text                               = 25% less Attack Speed<br>Deals 120% more Damage<br>50% of Physical Damage Converted to Lightning Damage
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = skill_physical_damage_%_to_convert_to_lightning
+|static_stat2_value                      = 50
+|static_stat3_id                         = active_skill_damage_+%_final
+|static_stat3_value                      = 120
+|static_stat4_id                         = active_skill_attack_speed_+%_final
+|static_stat4_value                      = -25
+|stat_text                               = 50% of Physical Damage Converted to Lightning Damage<br>Deals 120% more Damage<br>25% less Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 0
 }}