--- Skill:SpikerBossIceCall
+++ Export
@@ -11,20 +11,21 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 2.5
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = -20
-|static_stat2_id                         = cast_on_damage_taken_%
-|static_stat2_value                      = 100
-|static_stat3_id                         = cast_on_damage_taken_threshold
-|static_stat3_value                      = 10
-|static_stat4_id                         = base_skill_effect_duration
-|static_stat4_value                      = 2500
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
-|static_stat6_id                         = base_skill_show_average_damage_instead_of_dps
-|static_stat6_value                      = 1
-|static_stat7_id                         = cast_on_damage_taken_also_tracks_dot
-|static_stat7_value                      = 1
+|static_stat2_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat2_value                      = 1
+|static_stat3_id                         = cast_on_damage_taken_also_tracks_dot
+|static_stat3_value                      = 1
+|static_stat4_id                         = is_area_damage
+|static_stat4_value                      = 1
+|static_stat5_id                         = base_skill_effect_duration
+|static_stat5_value                      = 2500
+|static_stat6_id                         = cast_on_damage_taken_%
+|static_stat6_value                      = 100
+|static_stat7_id                         = cast_on_damage_taken_threshold
+|static_stat7_value                      = 10
 |stat_text                               = Deals (15-741) to (23-1111) Cold Damage<br>Base duration is 2.5 seconds
 |level1                                  = True
 |level1_level_requirement                = 12
