--- Skill:NinjaCopAbyssSlam
+++ Export
@@ -9,17 +9,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat1_value                      = -50
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
 |static_stat2_id                         = main_hand_base_maximum_attack_distance
 |static_stat2_value                      = 50
-|static_stat3_id                         = upheaval_number_of_spikes
-|static_stat3_value                      = 8
+|static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat3_value                      = -50
 |static_stat4_id                         = active_skill_attack_speed_+%_final
 |static_stat4_value                      = -25
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
-|stat_text                               = Deals (-90-800)% more Damage<br>Causes 8 Bursts<br>25% less Attack Speed
+|static_stat5_id                         = upheaval_number_of_spikes
+|static_stat5_value                      = 8
+|stat_text                               = Deals (-90-800)% more Damage<br>25% less Attack Speed<br>Causes 8 Bursts
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_stat_text                        = Deals 90% less Damage
