--- Skill:MonsterFreezingPulse
+++ Export
@@ -10,13 +10,13 @@
 |static_damage_effectiveness             = 125
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_projectile_speed_+%
-|static_stat1_value                      = 50
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
 |static_stat2_id                         = always_pierce
 |static_stat2_value                      = 1
-|static_stat3_id                         = base_is_projectile
-|static_stat3_value                      = 1
-|stat_text                               = Deals (9-812) to (13-1219) Cold Damage<br>50% increased Projectile Speed<br>Projectiles Pierce all Targets
+|static_stat3_id                         = base_projectile_speed_+%
+|static_stat3_value                      = 50
+|stat_text                               = Deals (9-812) to (13-1219) Cold Damage<br>Projectiles Pierce all Targets<br>50% increased Projectile Speed
 |level1                                  = True
 |level1_level_requirement                = 3
 |level1_stat_text                        = Deals 9 to 13 Cold Damage
