--- Skill:TriggeredExplosion
+++ Export
@@ -13,11 +13,11 @@
 |static_stat1_value                      = 170
 |static_stat2_id                         = spell_maximum_base_fire_damage
 |static_stat2_value                      = 255
-|static_stat3_id                         = chance_to_cast_on_kill_%_target_self
-|static_stat3_value                      = 100
-|static_stat4_id                         = spell_uncastable_if_triggerable
-|static_stat4_value                      = 1
-|stat_text                               = Deals 170 to 255 Fire Damage<br>Trigger this Spell on Kill<br>You cannot Cast this Spell directly
+|static_stat3_id                         = spell_uncastable_if_triggerable
+|static_stat3_value                      = 1
+|static_stat4_id                         = chance_to_cast_on_kill_%_target_self
+|static_stat4_value                      = 100
+|stat_text                               = Deals 170 to 255 Fire Damage<br>You cannot Cast this Spell directly<br>Trigger this Spell on Kill
 |level1                                  = True
 |level1_level_requirement                = 60
 }}