--- Skill:LightningGolemArcIzaro
+++ Export
@@ -16,11 +16,11 @@
 |static_stat2_value                      = 11
 |static_stat3_id                         = skill_range_+%
 |static_stat3_value                      = -50
-|static_stat4_id                         = number_of_chains
-|static_stat4_value                      = 1
-|static_stat5_id                         = base_chance_to_shock_%
-|static_stat5_value                      = 10
-|stat_text                               = Deals 4 to 11 Lightning Damage<br>50% reduced Skill Range<br>Chains +1 Times<br>10% chance to Shock enemies
+|static_stat4_id                         = base_chance_to_shock_%
+|static_stat4_value                      = 10
+|static_stat5_id                         = number_of_chains
+|static_stat5_value                      = 1
+|stat_text                               = Deals 4 to 11 Lightning Damage<br>50% reduced Skill Range<br>10% chance to Shock enemies<br>Chains +1 Times
 |level1                                  = True
 |level1_level_requirement                = 3
 }}