--- Skill:ShavronneMinionLightningWhip
+++ Export
@@ -9,13 +9,13 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage
-|static_stat1_value                      = 5
-|static_stat2_id                         = spell_maximum_base_lightning_damage
-|static_stat2_value                      = 16
-|static_stat3_id                         = cannot_stun
-|static_stat3_value                      = 1
-|static_stat4_id                         = is_area_damage
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_lightning_damage
+|static_stat2_value                      = 5
+|static_stat3_id                         = spell_maximum_base_lightning_damage
+|static_stat3_value                      = 16
+|static_stat4_id                         = cannot_stun
 |static_stat4_value                      = 1
 |stat_text                               = Deals 5 to 16 Lightning Damage<br>Your Hits cannot Stun Enemies
 |level1                                  = True
