--- Skill:AfflictionFireSnap
+++ Export
@@ -10,17 +10,18 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_fire_damage
-|static_stat1_value                      = 3
-|static_stat2_id                         = spell_maximum_base_fire_damage
-|static_stat2_value                      = 4
-|static_stat3_id                         = base_skill_effect_duration
-|static_stat3_value                      = 6000
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
-|static_stat5_id                         = base_fire_damage_to_deal_per_minute
-|static_stat5_value                      = 282
-|stat_text                               = Deals 3 to 4 Fire Damage<br>Base duration is 6 seconds<br>Deals 4.7 Base Fire Damage per second
+|static_duration                         = 6
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_fire_damage
+|static_stat2_value                      = 3
+|static_stat3_id                         = spell_maximum_base_fire_damage
+|static_stat3_value                      = 4
+|static_stat4_id                         = base_fire_damage_to_deal_per_minute
+|static_stat4_value                      = 282
+|static_stat5_id                         = base_skill_effect_duration
+|static_stat5_value                      = 6000
+|stat_text                               = Deals 3 to 4 Fire Damage<br>Deals 4.7 Base Fire Damage per second<br>Base duration is 6 seconds
 |level1                                  = True
 |level1_level_requirement                = 0
 }}