--- Skill:SandLeaperLeapSlam
+++ Export
@@ -12,13 +12,13 @@
 |static_stat1_value                      = -25
 |static_stat2_id                         = active_skill_bleeding_damage_+%_final
 |static_stat2_value                      = 115
-|static_stat3_id                         = active_skill_damage_+%_final
-|static_stat3_value                      = 15
-|static_stat4_id                         = global_bleed_on_hit
+|static_stat3_id                         = cast_time_overrides_attack_duration
+|static_stat3_value                      = 1
+|static_stat4_id                         = is_area_damage
 |static_stat4_value                      = 1
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
-|static_stat6_id                         = cast_time_overrides_attack_duration
+|static_stat5_id                         = active_skill_damage_+%_final
+|static_stat5_value                      = 15
+|static_stat6_id                         = global_bleed_on_hit
 |static_stat6_value                      = 1
 |stat_text                               = Deals 15% more Damage<br>Causes Bleeding
 |level1                                  = True
