--- Skill:BloodlinesGhostTotemLightningWarp
+++ Export
@@ -10,12 +10,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = skill_effect_duration_+%
-|static_stat1_value                      = -75
-|static_stat2_id                         = is_area_damage
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = lightning_warp_use_non_screen_shake_effects
 |static_stat2_value                      = 1
-|static_stat3_id                         = lightning_warp_use_non_screen_shake_effects
-|static_stat3_value                      = 1
+|static_stat3_id                         = skill_effect_duration_+%
+|static_stat3_value                      = -75
 |stat_text                               = Deals (5-848) to (14-2543) Lightning Damage<br>(10-43)% increased Cast Speed<br>75% reduced Duration
 |level1                                  = True
 |level1_level_requirement                = 3
