--- Skill:RyslathaStandaloneBurrow
+++ Export
@@ -9,17 +9,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 99.25
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = shield_charge_extra_distance
-|static_stat1_value                      = 45
-|static_stat2_id                         = emerge_speed_+%
-|static_stat2_value                      = 300
-|static_stat3_id                         = global_poison_on_hit
-|static_stat3_value                      = 1
+|static_stat1_id                         = emerge_speed_+%
+|static_stat1_value                      = 300
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = shield_charge_extra_distance
+|static_stat3_value                      = 45
 |static_stat4_id                         = cannot_stun
 |static_stat4_value                      = 1
-|static_stat5_id                         = is_area_damage
+|static_stat5_id                         = global_poison_on_hit
 |static_stat5_value                      = 1
-|stat_text                               = Poisons Enemies on Hit<br>Your Hits cannot Stun Enemies
+|stat_text                               = Your Hits cannot Stun Enemies<br>Poisons Enemies on Hit
 |level1                                  = True
 |level1_level_requirement                = 1
 }}