--- Skill:BreachFlickerStrike
+++ Export
@@ -11,9 +11,9 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = ignores_proximity_shield
 |static_stat1_value                      = 1
-|static_stat2_id                         = summoned_monsters_are_minions
+|static_stat2_id                         = minion_dies_when_parent_dies
 |static_stat2_value                      = 1
-|static_stat3_id                         = minion_dies_when_parent_dies
+|static_stat3_id                         = summoned_monsters_are_minions
 |static_stat3_value                      = 1
 |stat_text                               = 
 |level1                                  = True
