--- Skill:DarkExileThrownWeapon3
+++ Export
@@ -8,21 +8,21 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_maximum_action_distance_+%
-|static_stat1_value                      = -30
-|static_stat2_id                         = attack_speed_+%
-|static_stat2_value                      = 50
-|static_stat3_id                         = base_projectile_speed_+%
-|static_stat3_value                      = 30
-|static_stat4_id                         = number_of_additional_projectiles
-|static_stat4_value                      = 4
-|static_stat5_id                         = projectile_damage_+%
-|static_stat5_value                      = 21
-|static_stat6_id                         = active_skill_damage_+%_final
-|static_stat6_value                      = -25
-|static_stat7_id                         = base_is_projectile
-|static_stat7_value                      = 1
-|stat_text                               = 50% increased Attack Speed<br>30% increased Projectile Speed<br>Fires 4 additional Projectiles<br>21% increased Projectile Damage<br>Deals 25% less Damage
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_maximum_action_distance_+%
+|static_stat2_value                      = -30
+|static_stat3_id                         = active_skill_damage_+%_final
+|static_stat3_value                      = -25
+|static_stat4_id                         = attack_speed_+%
+|static_stat4_value                      = 50
+|static_stat5_id                         = base_projectile_speed_+%
+|static_stat5_value                      = 30
+|static_stat6_id                         = number_of_additional_projectiles
+|static_stat6_value                      = 4
+|static_stat7_id                         = projectile_damage_+%
+|static_stat7_value                      = 21
+|stat_text                               = Deals 25% less Damage<br>50% increased Attack Speed<br>30% increased Projectile Speed<br>Fires 4 additional Projectiles<br>21% increased Projectile Damage
 |level1                                  = True
 |level1_level_requirement                = 73
 }}