--- Skill:SynthesisBossShockRifleSingle
+++ Export
@@ -9,21 +9,21 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
+|static_stat1_id                         = is_area_damage
 |static_stat1_value                      = 1
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 2
-|static_stat3_id                         = active_skill_projectile_damage_+%_final
-|static_stat3_value                      = 625
-|static_stat4_id                         = skill_physical_damage_%_to_convert_to_lightning
-|static_stat4_value                      = 50
-|static_stat5_id                         = base_projectile_speed_+%
+|static_stat2_id                         = projectile_uses_contact_position
+|static_stat2_value                      = 1
+|static_stat3_id                         = spell_minimum_base_physical_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = spell_maximum_base_physical_damage
+|static_stat4_value                      = 2
+|static_stat5_id                         = skill_physical_damage_%_to_convert_to_lightning
 |static_stat5_value                      = 50
-|static_stat6_id                         = projectile_uses_contact_position
-|static_stat6_value                      = 1
-|static_stat7_id                         = is_area_damage
-|static_stat7_value                      = 1
-|stat_text                               = Deals 1 to 2 Physical Damage<br>Projectiles deal 625% more Damage<br>50% of Physical Damage Converted to Lightning Damage<br>50% increased Projectile Speed
+|static_stat6_id                         = base_projectile_speed_+%
+|static_stat6_value                      = 50
+|static_stat7_id                         = active_skill_projectile_damage_+%_final
+|static_stat7_value                      = 625
+|stat_text                               = Deals 1 to 2 Physical Damage<br>50% of Physical Damage Converted to Lightning Damage<br>50% increased Projectile Speed<br>625% more Projectile Damage
 |level1                                  = True
 |level1_level_requirement                = 0
 }}