--- Skill:AtlasEyrieFighterSlash
+++ Export
@@ -7,13 +7,13 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 110
 |static_attack_speed_multiplier          = 50
-|static_stat1_id                         = melee_range_+
-|static_stat1_value                      = 25
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
 |static_stat2_id                         = skill_physical_damage_%_to_convert_to_cold
 |static_stat2_value                      = 75
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
-|stat_text                               = Deals (0-200)% more Damage<br>+25 to Melee Strike Range<br>75% of Physical Damage Converted to Cold Damage
+|static_stat3_id                         = melee_range_+
+|static_stat3_value                      = 25
+|stat_text                               = Deals (0-200)% more Damage<br>75% of Physical Damage Converted to Cold Damage<br>+25 to Melee Strike Range
 |level1                                  = True
 |level1_level_requirement                = 1
 |level2                                  = True
