--- Skill:BreachFlickerStrikeChayula
+++ Export
@@ -13,9 +13,9 @@
 |static_stat1_value                      = 1
 |static_stat2_id                         = ignores_proximity_shield
 |static_stat2_value                      = 1
-|static_stat3_id                         = summoned_monsters_are_minions
+|static_stat3_id                         = minion_dies_when_parent_dies
 |static_stat3_value                      = 1
-|static_stat4_id                         = minion_dies_when_parent_dies
+|static_stat4_id                         = summoned_monsters_are_minions
 |static_stat4_value                      = 1
 |stat_text                               = 
 |level1                                  = True
