--- Skill:AncestralMeleeKnockbackStun
+++ Export
@@ -0,0 +1,21 @@
+{{Skill
+|gem_description                         = Strike your foes down with a powerful blow.
+|active_skill_name                       = Default Attack
+|skill_id                                = AncestralMeleeKnockbackStun
+|cast_time                               = 1.0
+|required_level                          = 0
+|static_damage_effectiveness             = 130
+|static_damage_multiplier                = 100
+|static_attack_speed_multiplier          = 100
+|static_stat1_id                         = action_attack_or_cast_time_uses_animation_length
+|static_stat1_value                      = 1
+|static_stat2_id                         = determine_knockback_direction_from_melee_pattern
+|static_stat2_value                      = 1
+|static_stat3_id                         = global_knockback
+|static_stat3_value                      = 1
+|static_stat4_id                         = base_stun_threshold_reduction_+%
+|static_stat4_value                      = 50
+|stat_text                               = Knocks Enemies Back on Hit<br>50% reduced Enemy Stun Threshold
+|level1                                  = True
+|level1_level_requirement                = 0
+}}