--- Skill:SkeletonFrostBlades
+++ Export
@@ -14,7 +14,7 @@
 |static_stat2_value                      = 40
 |static_stat3_id                         = melee_weapon_range_+
 |static_stat3_value                      = 14
-|stat_text                               = Fires 2 additional Projectiles<br>40% of Physical Damage Converted to Cold Damage<br>+14 to Melee Strike Range
+|stat_text                               = Fires 2 additional Projectiles<br>40% of Physical Damage Converted to Cold Damage<br>+1.4 metres to Melee Strike Range
 |level1                                  = True
 |level1_level_requirement                = 1
 }}