--- Skill:KaomVolcanoMortarElder
+++ Export
@@ -7,23 +7,23 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_fire_damage
-|static_stat1_value                      = 155
-|static_stat2_id                         = spell_maximum_base_fire_damage
-|static_stat2_value                      = 233
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
 |static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
 |static_stat3_value                      = -35
 |static_stat4_id                         = number_of_projectiles_override
 |static_stat4_value                      = 2
-|static_stat5_id                         = skill_range_+%
-|static_stat5_value                      = -50
-|static_stat6_id                         = base_is_projectile
-|static_stat6_value                      = 1
-|static_stat7_id                         = is_area_damage
-|static_stat7_value                      = 1
-|static_stat8_id                         = projectiles_not_offset
-|static_stat8_value                      = 1
-|stat_text                               = Deals 155 to 233 Fire Damage<br>50% reduced Skill Range
+|static_stat5_id                         = projectiles_not_offset
+|static_stat5_value                      = 1
+|static_stat6_id                         = skill_range_+%
+|static_stat6_value                      = -50
+|static_stat7_id                         = spell_minimum_base_fire_damage
+|static_stat7_value                      = 155
+|static_stat8_id                         = spell_maximum_base_fire_damage
+|static_stat8_value                      = 233
+|stat_text                               = 50% reduced Skill Range<br>Deals 155 to 233 Fire Damage
 |level1                                  = True
 |level1_level_requirement                = 38
 }}