--- Skill:DarbelLePageIceStorm
+++ Export
@@ -10,19 +10,20 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = firestorm_drop_ground_ice_duration_ms
-|static_stat1_value                      = 3000
-|static_stat2_id                         = firestorm_base_area_of_effect_+%
-|static_stat2_value                      = -49
-|static_stat3_id                         = base_cast_speed_+%
-|static_stat3_value                      = 35
-|static_stat4_id                         = fire_storm_fireball_delay_ms
-|static_stat4_value                      = 120
-|static_stat5_id                         = base_skill_effect_duration
-|static_stat5_value                      = 3000
-|static_stat6_id                         = is_area_damage
-|static_stat6_value                      = 1
-|stat_text                               = Deals (518-1383) to (777-1) Cold Damage<br>35% increased Cast Speed<br>One impact every 0.12 seconds<br>Base duration is 3 seconds
+|static_duration                         = 3
+|static_stat1_id                         = firestorm_base_area_of_effect_+%
+|static_stat1_value                      = -49
+|static_stat2_id                         = firestorm_drop_ground_ice_duration_ms
+|static_stat2_value                      = 3000
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = base_skill_effect_duration
+|static_stat4_value                      = 3000
+|static_stat5_id                         = base_cast_speed_+%
+|static_stat5_value                      = 35
+|static_stat6_id                         = fire_storm_fireball_delay_ms
+|static_stat6_value                      = 120
+|stat_text                               = Deals (518-1383) to (777-1) Cold Damage<br>Base duration is 3 seconds<br>35% increased Cast Speed<br>One impact every 0.12 seconds
 |level1                                  = True
 |level1_level_requirement                = 68
 |level1_stat_text                        = Deals 518 to 777 Cold Damage
