--- Skill:BetrayalSiegeCannonFire
+++ Export
@@ -13,10 +13,10 @@
 |static_stat1_value                      = 1
 |static_stat2_id                         = betrayal_siege_cannon_beam_damage_+%_final
 |static_stat2_value                      = -90
-|static_stat3_id                         = skill_physical_damage_%_to_convert_to_fire
-|static_stat3_value                      = 50
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = skill_physical_damage_%_to_convert_to_fire
+|static_stat4_value                      = 50
 |stat_text                               = Deals (8-20) to (12-30) Physical Damage<br>Deals (20-0)% less Damage<br>50% of Physical Damage Converted to Fire Damage
 |level1                                  = True
 |level1_level_requirement                = 1
