--- Skill:FrostfireMageFrostStormCall
+++ Export
@@ -7,19 +7,20 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_cold_damage
-|static_stat1_value                      = 14
-|static_stat2_id                         = spell_maximum_base_cold_damage
-|static_stat2_value                      = 21
-|static_stat3_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat3_value                      = -40
-|static_stat4_id                         = base_cast_speed_+%
-|static_stat4_value                      = 100
+|static_duration                         = 3
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = -40
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = spell_minimum_base_cold_damage
+|static_stat3_value                      = 14
+|static_stat4_id                         = spell_maximum_base_cold_damage
+|static_stat4_value                      = 21
 |static_stat5_id                         = base_skill_effect_duration
 |static_stat5_value                      = 3000
-|static_stat6_id                         = is_area_damage
-|static_stat6_value                      = 1
-|stat_text                               = Deals 14 to 21 Cold Damage<br>100% increased Cast Speed<br>Base duration is 3 seconds
+|static_stat6_id                         = base_cast_speed_+%
+|static_stat6_value                      = 100
+|stat_text                               = Deals 14 to 21 Cold Damage<br>Base duration is 3 seconds<br>100% increased Cast Speed
 |level1                                  = True
 |level1_level_requirement                = 3
 }}