--- Skill:DeathExplodeMinerHammerBoss
+++ Export
@@ -7,23 +7,23 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_fire_damage
-|static_stat1_value                      = 53
-|static_stat2_id                         = spell_maximum_base_fire_damage
-|static_stat2_value                      = 79
-|static_stat3_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat3_value                      = 100
-|static_stat4_id                         = active_skill_ignite_duration_+%_final
-|static_stat4_value                      = 200
-|static_stat5_id                         = base_chance_to_ignite_%
-|static_stat5_value                      = 100
-|static_stat6_id                         = cast_on_death_%
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = 100
+|static_stat2_id                         = active_skill_ignite_duration_+%_final
+|static_stat2_value                      = 200
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = spell_minimum_base_fire_damage
+|static_stat4_value                      = 53
+|static_stat5_id                         = spell_maximum_base_fire_damage
+|static_stat5_value                      = 79
+|static_stat6_id                         = base_chance_to_ignite_%
 |static_stat6_value                      = 100
 |static_stat7_id                         = spell_uncastable_if_triggerable
 |static_stat7_value                      = 1
-|static_stat8_id                         = is_area_damage
-|static_stat8_value                      = 1
-|stat_text                               = Deals 53 to 79 Fire Damage<br>100% chance to Ignite enemies<br>100% chance to Trigger this Spell on Death<br>You cannot Cast this Spell directly
+|static_stat8_id                         = cast_on_death_%
+|static_stat8_value                      = 100
+|stat_text                               = Deals 53 to 79 Fire Damage<br>100% chance to Ignite enemies<br>You cannot Cast this Spell directly<br>100% chance to Trigger this Spell on Death
 |level1                                  = True
 |level1_level_requirement                = 3
 }}