--- Skill:CGEConsumePhantasmShockGround
+++ Export
@@ -7,10 +7,11 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_lightning_damage_to_deal_per_minute
-|static_stat1_value                      = 352
-|static_stat2_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat2_value                      = -50
+|static_duration                         = 5
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = -50
+|static_stat2_id                         = base_lightning_damage_to_deal_per_minute
+|static_stat2_value                      = 352
 |static_stat3_id                         = ground_vortex_lightning_art_variation
 |static_stat3_value                      = 2
 |static_stat4_id                         = base_skill_effect_duration
