--- Skill:BestiaryBirdSwarmAttack
+++ Export
@@ -7,19 +7,19 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat1_value                      = -40
-|static_stat2_id                         = active_skill_bleeding_damage_+%_final
-|static_stat2_value                      = 100
-|static_stat3_id                         = geometry_attack_length
-|static_stat3_value                      = 27
-|static_stat4_id                         = active_skill_damage_+%_final
-|static_stat4_value                      = 50
-|static_stat5_id                         = global_bleed_on_hit
-|static_stat5_value                      = 1
-|static_stat6_id                         = keystone_crimson_dance
+|static_stat1_id                         = active_skill_bleeding_damage_+%_final
+|static_stat1_value                      = 100
+|static_stat2_id                         = geometry_attack_length
+|static_stat2_value                      = 27
+|static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat3_value                      = -40
+|static_stat4_id                         = keystone_crimson_dance
+|static_stat4_value                      = 1
+|static_stat5_id                         = active_skill_damage_+%_final
+|static_stat5_value                      = 50
+|static_stat6_id                         = global_bleed_on_hit
 |static_stat6_value                      = 1
-|stat_text                               = Deals 50% more Damage<br>Causes Bleeding<br>Crimson Dance
+|stat_text                               = Crimson Dance<br>Deals 50% more Damage<br>Causes Bleeding
 |level1                                  = True
 |level1_level_requirement                = 0
 }}