--- Skill:RiverBossColdSnapTriggered
+++ Export
@@ -8,17 +8,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 5
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 7
-|static_stat3_id                         = chill_effect_+%
-|static_stat3_value                      = 300
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
+|static_stat1_id                         = active_skill_chill_effect_+%_final
+|static_stat1_value                      = 300
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = spell_minimum_base_physical_damage
+|static_stat3_value                      = 5
+|static_stat4_id                         = spell_maximum_base_physical_damage
+|static_stat4_value                      = 7
 |static_stat5_id                         = base_cold_damage_to_deal_per_minute
 |static_stat5_value                      = 511
-|stat_text                               = Deals 5 to 7 Physical Damage<br>300% increased Effect of Chill<br>Deals 8.5 Base Cold Damage per second
+|stat_text                               = Deals 5 to 7 Physical Damage<br>Deals 8.5 Base Cold Damage per second
 |level1                                  = True
 |level1_level_requirement                = 0
 }}