--- Skill:ExileRangerColdSnap
+++ Export
@@ -9,13 +9,13 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_maximum_action_distance_+%
-|static_stat1_value                      = -50
-|static_stat2_id                         = active_skill_chill_duration_+%_final
-|static_stat2_value                      = 10
-|static_stat3_id                         = never_freeze
-|static_stat3_value                      = 1
-|static_stat4_id                         = is_area_damage
+|static_stat1_id                         = active_skill_chill_duration_+%_final
+|static_stat1_value                      = 10
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = spell_maximum_action_distance_+%
+|static_stat3_value                      = -50
+|static_stat4_id                         = never_freeze
 |static_stat4_value                      = 1
 |stat_text                               = Deals (12-689) to (15-842) Cold Damage<br>(-25-10)% increased Cast Speed<br>Cannot Freeze
 |level1                                  = True
