--- Skill:SummonIceElementalsRareMapBoss
+++ Export
@@ -13,10 +13,10 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = alternate_minion
 |static_stat1_value                      = 32
-|static_stat2_id                         = summoned_monster_rarity
-|static_stat2_value                      = 2
-|static_stat3_id                         = number_of_monsters_to_summon
-|static_stat3_value                      = 3
+|static_stat2_id                         = number_of_monsters_to_summon
+|static_stat2_value                      = 3
+|static_stat3_id                         = summoned_monster_rarity
+|static_stat3_value                      = 2
 |static_stat4_id                         = summoned_monsters_are_minions
 |static_stat4_value                      = 1
 |stat_text                               = 
