--- Skill:AvatarTemporalChainsAura
+++ Export
@@ -24,7 +24,7 @@
 |static_stat7_value                      = 1
 |static_stat8_id                         = temporal_chains_action_speed_+%_vs_rare_or_unique_final
 |static_stat8_value                      = -10
-|stat_text                               = Base duration is 5 seconds<br>+17 to radius<br>50% less Effect of Curse against Players<br>Cursed Normal and Magic Enemies have 20% less Action Speed<br>Other effects on Cursed enemies expire 25% slower<br>This Curse is Cast as an Aura applying to nearby Enemies<br>Cursed Rare and Unique Enemies have 10% less Action Speed
+|stat_text                               = Base duration is 5 seconds<br>+1.7 metres to radius<br>50% less Effect of Curse against Players<br>Cursed Normal and Magic Enemies have 20% less Action Speed<br>Other effects on Cursed enemies expire 25% slower<br>This Curse is Cast as an Aura applying to nearby Enemies<br>Cursed Rare and Unique Enemies have 10% less Action Speed
 |level1                                  = True
 |level1_level_requirement                = 1
 }}