--- Skill:ChestDetonateDead
+++ Export
@@ -9,9 +9,9 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = -18
-|static_stat2_id                         = is_area_damage
+|static_stat2_id                         = detonate_dead_chain_explode
 |static_stat2_value                      = 1
-|static_stat3_id                         = detonate_dead_chain_explode
+|static_stat3_id                         = is_area_damage
 |static_stat3_value                      = 1
 |stat_text                               = Deals (6-783) to (9-1175) Fire Damage<br>Explosion deals base Fire Damage equal to (10-6)% of the corpse's Maximum Life
 |level1                                  = True
