--- Skill:BestiaryDaemonSummonBloodRaven
+++ Export
@@ -9,17 +9,18 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 3
 |static_stat1_id                         = alternate_minion
 |static_stat1_value                      = 1
-|static_stat2_id                         = active_skill_damage_+%_final
-|static_stat2_value                      = -40
-|static_stat3_id                         = base_number_of_raging_spirits_allowed
-|static_stat3_value                      = 50
-|static_stat4_id                         = base_skill_effect_duration
-|static_stat4_value                      = 3000
-|static_stat5_id                         = cannot_stun
-|static_stat5_value                      = 1
-|stat_text                               = Skills Repeat an additional (2-4) Times<br>Deals 40% less Damage<br>Maximum 50 Summoned Raging Spirits<br>Base duration is 3 seconds<br>Your Hits cannot Stun Enemies
+|static_stat2_id                         = cannot_stun
+|static_stat2_value                      = 1
+|static_stat3_id                         = active_skill_damage_+%_final
+|static_stat3_value                      = -40
+|static_stat4_id                         = base_number_of_raging_spirits_allowed
+|static_stat4_value                      = 50
+|static_stat5_id                         = base_skill_effect_duration
+|static_stat5_value                      = 3000
+|stat_text                               = Your Hits cannot Stun Enemies<br>Skills Repeat an additional (2-4) Times<br>Deals 40% less Damage<br>Maximum 50 Summoned Raging Spirits<br>Base duration is 3 seconds
 |level1                                  = True
 |level1_level_requirement                = 4
 |level1_stat_text                        = Skills Repeat an additional 2 Times
