--- Skill:BlightLightningModOnDeathOrbOfStorms
+++ Export
@@ -7,6 +7,7 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 4
 |static_stat1_id                         = spell_minimum_base_lightning_damage
 |static_stat1_value                      = 4
 |static_stat2_id                         = spell_maximum_base_lightning_damage
