--- Skill:ImplicitSupportManaMultiplier
+++ Export
@@ -7,8 +7,11 @@
 |stat_text                               = 
 |level1                                  = True
 |level1_level_requirement                = 0
+|level1_cost_multiplier                  = 96
 |level2                                  = True
 |level2_level_requirement                = 0
+|level2_cost_multiplier                  = 94
 |level3                                  = True
 |level3_level_requirement                = 0
+|level3_cost_multiplier                  = 92
 }}