--- Skill:MapMerveilGeyser
+++ Export
@@ -9,13 +9,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat1_value                      = -35
-|static_stat2_id                         = is_area_damage
-|static_stat2_value                      = 1
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat2_value                      = -35
 |stat_text                               = Deals (578-815) to (867-1223) Cold Damage<br>Base duration is (8-12) seconds
 |level1                                  = True
 |level1_level_requirement                = 66
+|level1_duration                         = 8
 |level1_stat_text                        = Deals 578 to 867 Cold Damage<br>Base duration is 8 seconds
 |level1_stat1_id                         = spell_minimum_base_cold_damage
 |level1_stat1_value                      = 578
@@ -25,6 +26,7 @@
 |level1_stat3_value                      = 8000
 |level2                                  = True
 |level2_level_requirement                = 70
+|level2_duration                         = 10
 |level2_stat_text                        = Deals 687 to 1031 Cold Damage<br>Base duration is 10 seconds
 |level2_stat1_id                         = spell_minimum_base_cold_damage
 |level2_stat1_value                      = 687
@@ -34,6 +36,7 @@
 |level2_stat3_value                      = 10000
 |level3                                  = True
 |level3_level_requirement                = 74
+|level3_duration                         = 12
 |level3_stat_text                        = Deals 815 to 1223 Cold Damage<br>Base duration is 12 seconds
 |level3_stat1_id                         = spell_minimum_base_cold_damage
 |level3_stat1_value                      = 815
