--- Skill:DancingDervishCyclone
+++ Export
@@ -10,10 +10,10 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = cyclone_extra_distance
 |static_stat1_value                      = 25
-|static_stat2_id                         = attack_speed_+%
-|static_stat2_value                      = 20
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = attack_speed_+%
+|static_stat3_value                      = 20
 |stat_text                               = 20% increased Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 1
