--- Skill:CGETarZombieOilGround
+++ Export
@@ -7,19 +7,20 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_fire_damage_to_deal_per_minute
-|static_stat1_value                      = 890
-|static_stat2_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat2_value                      = -30
-|static_stat3_id                         = ground_oil_art_variation
-|static_stat3_value                      = 1
-|static_stat4_id                         = cast_on_death_%
-|static_stat4_value                      = 100
+|static_duration                         = 6
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = -30
+|static_stat2_id                         = ground_oil_art_variation
+|static_stat2_value                      = 1
+|static_stat3_id                         = base_fire_damage_to_deal_per_minute
+|static_stat3_value                      = 890
+|static_stat4_id                         = base_skill_effect_duration
+|static_stat4_value                      = 6000
 |static_stat5_id                         = base_movement_velocity_+%
 |static_stat5_value                      = -20
-|static_stat6_id                         = base_skill_effect_duration
-|static_stat6_value                      = 6000
-|stat_text                               = Deals 14.8 Base Fire Damage per second<br>100% chance to Trigger this Spell on Death<br>20% reduced Movement Speed<br>Base duration is 6 seconds
+|static_stat6_id                         = cast_on_death_%
+|static_stat6_value                      = 100
+|stat_text                               = Deals 14.8 Base Fire Damage per second<br>Base duration is 6 seconds<br>20% reduced Movement Speed<br>100% chance to Trigger this Spell on Death
 |level1                                  = True
 |level1_level_requirement                = 1
 }}