--- Skill:WaterSphereTriggered
+++ Export
@@ -0,0 +1,49 @@
+{{Skill
+|gem_description                         = Create a sphere of water, or move an existing sphere and reset its duration. The sphere continually applies a drenched debuff with a short duration to enemies in its area or those it moves through. You can hit the sphere with other skills to inflict Cold and Lightning Ailments (other than Chill). The sphere pulses when created, at regular intervals while afflicted by an Ailment, and after moving, damaging all drenched enemies in a large area.
+|active_skill_name                       = Hydrosphere
+|skill_id                                = WaterSphereTriggered
+|cast_time                               = 0.6
+|required_level                          = 70
+|static_cost_amounts                     = 20
+|static_cost_types                       = Mana
+|static_stored_uses                      = 1
+|static_cooldown                         = 1
+|static_critical_strike_chance           = 5
+|static_damage_effectiveness             = 110
+|static_damage_multiplier                = 100
+|static_attack_speed_multiplier          = 100
+|static_duration                         = 8
+|static_stat1_id                         = damage_cannot_be_reflected_or_leech_if_used_by_other_object
+|static_stat1_value                      = 1
+|static_stat2_id                         = hydrosphere_ailment_threshold
+|static_stat2_value                      = 103048
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = skill_can_add_multiple_charges_per_action
+|static_stat4_value                      = 1
+|static_stat5_id                         = spell_minimum_base_physical_damage
+|static_stat5_value                      = 510
+|static_stat6_id                         = spell_maximum_base_physical_damage
+|static_stat6_value                      = 765
+|static_stat7_id                         = base_skill_effect_duration
+|static_stat7_value                      = 8000
+|static_stat8_id                         = base_secondary_skill_effect_duration
+|static_stat8_value                      = 4000
+|static_stat9_id                         = hydro_sphere_base_pulse_frequency_ms
+|static_stat9_value                      = 400
+|static_stat10_id                        = active_skill_display_suppress_physical_to_cold_damage_conversion
+|static_stat10_value                     = 1
+|static_stat11_id                        = skill_physical_damage_%_to_convert_to_cold
+|static_stat11_value                     = 100
+|static_stat12_id                        = chance_to_trigger_level_20_hydrosphere_while_channeling_winter_orb_%
+|static_stat12_value                     = 100
+|static_stat13_id                        = hydrosphere_hit_cooldown_ms
+|static_stat13_value                     = 1000
+|static_stat14_id                        = water_sphere_cold_lightning_exposure_%
+|static_stat14_value                     = -10
+|static_stat15_id                        = water_sphere_does_weird_conversion_stuff
+|static_stat15_value                     = 1
+|stat_text                               = Deals 510 to 765 Physical Damage<br>Base duration is 8 seconds<br>Base secondary duration is 4 seconds<br>Pulses every 0.4 seconds while Frozen, Shocked, Brittle or Sapped<br>100% of Physical Damage Converted to Cold Damage<br>This Skill is Triggered every second while you Channel Winter Orb<br>Can only be Hit once every 1 second<br>Drenched Enemies have Cold and Lightning Exposure, applying -10% to Resistances<br>100% of Physical Damage Converted to Cold Damage while Sphere has no Ailments<br>100% of Physical Damage Converted to Lightning Damage while sphere is Shocked or Sapped<br>100% of Physical Damage Converted to Cold Damage while Sphere is Frozen or Brittle
+|level1                                  = True
+|level1_level_requirement                = 70
+}}