--- Skill:PerandusHaste
+++ Export
@@ -13,11 +13,11 @@
 |static_stat1_value                      = 40
 |static_stat2_id                         = base_movement_velocity_+%
 |static_stat2_value                      = 8
-|static_stat3_id                         = cast_speed_+%_granted_from_skill
+|static_stat3_id                         = attack_speed_+%_granted_from_skill
 |static_stat3_value                      = 15
-|static_stat4_id                         = attack_speed_+%_granted_from_skill
+|static_stat4_id                         = cast_speed_+%_granted_from_skill
 |static_stat4_value                      = 15
-|stat_text                               = You and nearby allies gain 8% increased Movement Speed<br>You and nearby allies gain 15% increased Cast Speed<br>You and nearby allies gain 15% increased Attack Speed
+|stat_text                               = You and nearby allies gain 8% increased Movement Speed<br>You and nearby allies gain 15% increased Attack Speed<br>You and nearby allies gain 15% increased Cast Speed
 |level1                                  = True
 |level1_level_requirement                = 1
 }}