--- Skill:SkeletonProjectileLightningIzaroMultipleProjectiles
+++ Export
@@ -8,10 +8,10 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_maximum_action_distance_+%
-|static_stat1_value                      = -60
-|static_stat2_id                         = base_is_projectile
-|static_stat2_value                      = 1
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_maximum_action_distance_+%
+|static_stat2_value                      = -60
 |stat_text                               = Deals (30-210) to (90-630) Lightning Damage<br>Fires (2-4) additional Projectiles
 |level1                                  = True
 |level1_level_requirement                = 33
