--- Skill:AfflictionDemonColdSpearGroundTarget
+++ Export
@@ -7,25 +7,25 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = number_of_projectiles_override
+|static_stat1_id                         = base_deal_no_damage
 |static_stat1_value                      = 1
-|static_stat2_id                         = projectile_minimum_range
-|static_stat2_value                      = 30
+|static_stat2_id                         = base_is_projectile
+|static_stat2_value                      = 1
 |static_stat3_id                         = monster_projectile_variation
 |static_stat3_value                      = 101
-|static_stat4_id                         = active_skill_attack_speed_+%_final
-|static_stat4_value                      = -72
-|static_stat5_id                         = always_pierce
-|static_stat5_value                      = 1
-|static_stat6_id                         = base_is_projectile
+|static_stat4_id                         = number_of_projectiles_override
+|static_stat4_value                      = 1
+|static_stat5_id                         = projectile_minimum_range
+|static_stat5_value                      = 30
+|static_stat6_id                         = projectile_uses_contact_position
 |static_stat6_value                      = 1
-|static_stat7_id                         = projectile_uses_contact_position
+|static_stat7_id                         = use_scaled_contact_offset
 |static_stat7_value                      = 1
-|static_stat8_id                         = use_scaled_contact_offset
+|static_stat8_id                         = always_pierce
 |static_stat8_value                      = 1
-|static_stat9_id                         = base_deal_no_damage
-|static_stat9_value                      = 1
-|stat_text                               = 72% less Attack Speed<br>Projectiles Pierce all Targets
+|static_stat9_id                         = active_skill_attack_speed_+%_final
+|static_stat9_value                      = -72
+|stat_text                               = Projectiles Pierce all Targets<br>72% less Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 1
 }}