--- Skill:MPSAncestralJadeExplosionProj
+++ Export
@@ -0,0 +1,34 @@
+{{Skill
+|gem_description                         = 
+|active_skill_name                       = 
+|skill_id                                = MPSAncestralJadeExplosionProj
+|cast_time                               = 1.33
+|required_level                          = 1
+|static_critical_strike_chance           = 5
+|static_damage_effectiveness             = 100
+|static_damage_multiplier                = 100
+|static_attack_speed_multiplier          = 100
+|static_stat1_id                         = active_skill_projectile_speed_+%_variation_final
+|static_stat1_value                      = 25
+|static_stat2_id                         = base_is_projectile
+|static_stat2_value                      = 1
+|static_stat3_id                         = monster_projectile_variation
+|static_stat3_value                      = 244
+|static_stat4_id                         = projectile_uses_contact_position
+|static_stat4_value                      = 1
+|static_stat5_id                         = spell_minimum_base_physical_damage
+|static_stat5_value                      = 6
+|static_stat6_id                         = spell_maximum_base_physical_damage
+|static_stat6_value                      = 8
+|static_stat7_id                         = projectiles_nova
+|static_stat7_value                      = 1
+|static_stat8_id                         = number_of_additional_projectiles
+|static_stat8_value                      = 10
+|static_stat9_id                         = apply_overpowered_on_enemy_block_reduced_block_and_spell_block_%
+|static_stat9_value                      = -40
+|static_stat10_id                        = projectile_maximum_range_override
+|static_stat10_value                     = 35
+|stat_text                               = Deals 6 to 8 Physical Damage<br>Fires Projectiles in a circle<br>Fires 10 additional Projectiles<br>Overpowered applies --40% chance to Block Attack and Spell Damage<br>Projectile Range is Limited
+|level1                                  = True
+|level1_level_requirement                = 1
+}}