--- Skill:GladiatorBossFireball
+++ Export
@@ -10,13 +10,13 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_maximum_action_distance_+%
-|static_stat1_value                      = -50
-|static_stat2_id                         = number_of_additional_projectiles
-|static_stat2_value                      = 2
-|static_stat3_id                         = base_is_projectile
-|static_stat3_value                      = 1
-|stat_text                               = Deals (2-1051) to (4-1577) Fire Damage<br>(0-100)% chance to Ignite enemies<br>Elemental Ailments caused by this skill spread to other nearby enemies<br>Fires 2 additional Projectiles
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_maximum_action_distance_+%
+|static_stat2_value                      = -50
+|static_stat3_id                         = number_of_additional_projectiles
+|static_stat3_value                      = 2
+|stat_text                               = Deals (2-1051) to (4-1577) Fire Damage<br>(0-100)% chance to Ignite enemies<br>Fires 2 additional Projectiles<br>Elemental Ailments caused by this skill spread to other nearby enemies
 |level1                                  = True
 |level1_level_requirement                = 3
 |level1_stat_text                        = Deals 2 to 4 Fire Damage
