--- Skill:MerveilSeaWitchWave
+++ Export
@@ -8,13 +8,13 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_cold_damage
-|static_stat1_value                      = 58
-|static_stat2_id                         = spell_maximum_base_cold_damage
-|static_stat2_value                      = 86
-|static_stat3_id                         = never_freeze
-|static_stat3_value                      = 1
-|static_stat4_id                         = base_is_projectile
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_cold_damage
+|static_stat2_value                      = 58
+|static_stat3_id                         = spell_maximum_base_cold_damage
+|static_stat3_value                      = 86
+|static_stat4_id                         = never_freeze
 |static_stat4_value                      = 1
 |stat_text                               = Deals 58 to 86 Cold Damage<br>Cannot Freeze
 |level1                                  = True
