--- Skill:AxisShieldCharge
+++ Export
@@ -22,7 +22,7 @@
 |static_stat5_value                      = 120
 |static_stat6_id                         = melee_range_+
 |static_stat6_value                      = -10
-|stat_text                               = Never deal Critical Strikes<br>50% increased Stun Threshold reduction on enemies at Maximum charge distance<br>(70-30)% more Damage with Hits at Maximum Charge Distance<br>120% increased Movement Speed<br>-10 to Melee Strike Range
+|stat_text                               = Never deal Critical Strikes<br>50% increased Stun Threshold reduction on enemies at Maximum charge distance<br>(70-30)% more Damage with Hits at Maximum Charge Distance<br>120% increased Movement Speed<br>-1 metres to Melee Strike Range
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_stat_text                        = 70% more Damage with Hits at Maximum Charge Distance