--- Skill:TriggerCascadeMinionThree
+++ Export
@@ -9,14 +9,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = trigger_skill_restriction
-|static_stat1_value                      = 2
-|static_stat2_id                         = trigger_cascade_A
-|static_stat2_value                      = 60
-|static_stat3_id                         = trigger_cascade_ms_between_spikes
-|static_stat3_value                      = 200
-|static_stat4_id                         = trigger_cascade_number_of_spikes
-|static_stat4_value                      = 3
+|static_stat1_id                         = trigger_cascade_A
+|static_stat1_value                      = 60
+|static_stat2_id                         = trigger_cascade_ms_between_spikes
+|static_stat2_value                      = 200
+|static_stat3_id                         = trigger_cascade_number_of_spikes
+|static_stat3_value                      = 3
+|static_stat4_id                         = trigger_skill_restriction
+|static_stat4_value                      = 2
 |stat_text                               = 
 |level1                                  = True
 |level1_level_requirement                = 1
