--- Skill:ExperimenterBossRaiseZombieInvasion
+++ Export
@@ -15,7 +15,7 @@
 |static_stat3_value                      = 90
 |static_stat4_id                         = active_skill_minion_damage_+%_final
 |static_stat4_value                      = 30
-|stat_text                               = Maximum (3-10) Raised Zombies<br>25% increased Character Size<br>Minions have 90% more Maximum Life<br>Minions deal 30% more Damage
+|stat_text                               = 25% increased Character Size<br>Maximum (3-10) Raised Zombies<br>Minions have 90% more Maximum Life<br>Minions deal 30% more Damage
 |level1                                  = True
 |level1_level_requirement                = 3
 |level1_stat_text                        = Maximum 3 Raised Zombies
