--- Skill:GroundSlamBrute
+++ Export
@@ -12,15 +12,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat1_value                      = -30
-|static_stat2_id                         = active_skill_damage_+%_final
-|static_stat2_value                      = 80
-|static_stat3_id                         = attack_speed_+%
-|static_stat3_value                      = -33
-|static_stat4_id                         = always_stun
-|static_stat4_value                      = 1
-|static_stat5_id                         = is_area_damage
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat2_value                      = -30
+|static_stat3_id                         = active_skill_damage_+%_final
+|static_stat3_value                      = 80
+|static_stat4_id                         = attack_speed_+%
+|static_stat4_value                      = -33
+|static_stat5_id                         = always_stun
 |static_stat5_value                      = 1
 |stat_text                               = Deals 80% more Damage<br>33% reduced Attack Speed<br>Stuns Enemies
 |level1                                  = True
