--- Skill:MonsterFireStormKaomsChest
+++ Export
@@ -10,15 +10,16 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = firestorm_base_area_of_effect_+%
 |static_stat1_value                      = -49
-|static_stat2_id                         = base_chance_to_ignite_%
-|static_stat2_value                      = 25
-|static_stat3_id                         = fire_storm_fireball_delay_ms
-|static_stat3_value                      = 125
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = base_chance_to_ignite_%
+|static_stat3_value                      = 25
+|static_stat4_id                         = fire_storm_fireball_delay_ms
+|static_stat4_value                      = 125
 |stat_text                               = Deals (2-224) to (4-336) Fire Damage<br>Base duration is (1.2-3.6) seconds<br>25% chance to Ignite enemies<br>One impact every 0.12 seconds
 |level1                                  = True
 |level1_level_requirement                = 2
+|level1_duration                         = 1.2
 |level1_stat_text                        = Deals 2 to 4 Fire Damage<br>Base duration is 1.2 seconds
 |level1_stat1_id                         = spell_minimum_base_fire_damage
 |level1_stat1_value                      = 2
@@ -28,6 +29,7 @@
 |level1_stat3_value                      = 1200
 |level2                                  = True
 |level2_level_requirement                = 35
+|level2_duration                         = 2.4
 |level2_stat_text                        = Deals 54 to 80 Fire Damage<br>Base duration is 2.4 seconds
 |level2_stat1_id                         = spell_minimum_base_fire_damage
 |level2_stat1_value                      = 54
@@ -37,6 +39,7 @@
 |level2_stat3_value                      = 2400
 |level3                                  = True
 |level3_level_requirement                = 55
+|level3_duration                         = 3.6
 |level3_stat_text                        = Deals 224 to 336 Fire Damage<br>Base duration is 3.6 seconds
 |level3_stat1_id                         = spell_minimum_base_fire_damage
 |level3_stat1_value                      = 224
