--- Skill:KitavaSlamRightNoConeMapStandalone
+++ Export
@@ -7,19 +7,19 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_damage_+%_final
-|static_stat1_value                      = -53
+|static_stat1_id                         = action_do_not_face_target
+|static_stat1_value                      = 1
 |static_stat2_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat2_value                      = -50
-|static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat3_value                      = -50
-|static_stat4_id                         = global_cannot_crit
-|static_stat4_value                      = 1
-|static_stat5_id                         = is_area_damage
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat4_value                      = -50
+|static_stat5_id                         = global_cannot_crit
 |static_stat5_value                      = 1
-|static_stat6_id                         = action_do_not_face_target
-|static_stat6_value                      = 1
-|stat_text                               = Deals 53% less Damage<br>Never deal Critical Strikes
+|static_stat6_id                         = active_skill_damage_+%_final
+|static_stat6_value                      = -53
+|stat_text                               = Never deal Critical Strikes<br>Deals 53% less Damage
 |level1                                  = True
 |level1_level_requirement                = 68
 |level2                                  = True
