--- Skill:CelestialFlameblastShaperInfluence
+++ Export
@@ -24,7 +24,7 @@
 |static_stat7_value                      = 10
 |static_stat8_id                         = vaal_flameblast_radius_+_per_stage
 |static_stat8_value                      = 3
-|stat_text                               = Deals 4 to 6 Fire Damage<br>13% reduced Cast Speed<br>110% more Spell Damage for each stage<br>110% more Damage with Ailments for each stage<br>10 maximum Stages<br>+3 to Radius for each Stage
+|stat_text                               = Deals 4 to 6 Fire Damage<br>13% reduced Cast Speed<br>110% more Spell Damage for each stage<br>110% more Damage with Ailments for each stage<br>10 maximum Stages<br>+0.3 metres to radius per Stage
 |level1                                  = True
 |level1_level_requirement                = 1
 }}