--- Skill:Restore
+++ Export
@@ -9,9 +9,9 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = restore_skill_es_%
 |static_stat1_value                      = 10
-|static_stat2_id                         = restore_skill_mana_%
+|static_stat2_id                         = restore_skill_life_%
 |static_stat2_value                      = 10
-|static_stat3_id                         = restore_skill_life_%
+|static_stat3_id                         = restore_skill_mana_%
 |static_stat3_value                      = 10
 |stat_text                               = 
 |level1                                  = True
