--- Skill:EmptyActionLegionMultipleShoot
+++ Export
@@ -9,16 +9,16 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_attack_speed_+%_final
-|static_stat1_value                      = -25
-|static_stat2_id                         = skill_cannot_be_stunned
+|static_stat1_id                         = skill_can_fire_arrows
+|static_stat1_value                      = 1
+|static_stat2_id                         = skill_cannot_be_knocked_back
 |static_stat2_value                      = 1
-|static_stat3_id                         = skill_cannot_be_knocked_back
+|static_stat3_id                         = skill_cannot_be_stunned
 |static_stat3_value                      = 1
-|static_stat4_id                         = skill_can_fire_arrows
+|static_stat4_id                         = use_scaled_contact_offset
 |static_stat4_value                      = 1
-|static_stat5_id                         = use_scaled_contact_offset
-|static_stat5_value                      = 1
+|static_stat5_id                         = active_skill_attack_speed_+%_final
+|static_stat5_value                      = -25
 |stat_text                               = 25% less Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 1
