--- Skill:DwellerGroundSlam
+++ Export
@@ -20,7 +20,7 @@
 |static_stat4_value                      = 10
 |static_stat5_id                         = groundslam_damage_to_close_targets_+%_final
 |static_stat5_value                      = 25
-|stat_text                               = 15% reduced Enemy Stun Threshold<br>+10 to radius<br>Deals up to 25% more Damage with Hits to closer targets
+|stat_text                               = 15% reduced Enemy Stun Threshold<br>+1 metre to radius<br>Deals up to 25% more Damage with Hits to closer targets
 |level1                                  = True
 |level1_level_requirement                = 1
 }}