--- Skill:CityStalkerFireNova
+++ Export
@@ -7,12 +7,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_fire_damage
-|static_stat1_value                      = 137
-|static_stat2_id                         = spell_maximum_base_fire_damage
-|static_stat2_value                      = 205
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_fire_damage
+|static_stat2_value                      = 137
+|static_stat3_id                         = spell_maximum_base_fire_damage
+|static_stat3_value                      = 205
 |stat_text                               = Deals 137 to 205 Fire Damage
 |level1                                  = True
 |level1_level_requirement                = 45
