--- Skill:NecromancerElementalWeakness
+++ Export
@@ -9,11 +9,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 6
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = 42
 |static_stat2_id                         = base_skill_effect_duration
 |static_stat2_value                      = 6000
-|stat_text                               = Cursed enemies have -(20-40)% to Elemental Resistances<br>Base duration is 6 seconds
+|stat_text                               = Base duration is 6 seconds<br>Cursed enemies have -(20-40)% to Elemental Resistances
 |level1                                  = True
 |level1_level_requirement                = 10
 |level1_stat_text                        = Cursed enemies have -20% to Elemental Resistances
