--- Skill:FairgravesThrownWeaponMapBoss
+++ Export
@@ -8,17 +8,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat1_value                      = -25
-|static_stat2_id                         = base_projectile_speed_+%
-|static_stat2_value                      = 25
-|static_stat3_id                         = number_of_additional_projectiles
-|static_stat3_value                      = 3
-|static_stat4_id                         = active_skill_damage_+%_final
-|static_stat4_value                      = -47
-|static_stat5_id                         = base_is_projectile
-|static_stat5_value                      = 1
-|stat_text                               = 25% increased Projectile Speed<br>Fires 3 additional Projectiles<br>Deals 47% less Damage
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat2_value                      = -25
+|static_stat3_id                         = active_skill_damage_+%_final
+|static_stat3_value                      = -47
+|static_stat4_id                         = base_projectile_speed_+%
+|static_stat4_value                      = 25
+|static_stat5_id                         = number_of_additional_projectiles
+|static_stat5_value                      = 3
+|stat_text                               = Deals 47% less Damage<br>25% increased Projectile Speed<br>Fires 3 additional Projectiles
 |level1                                  = True
 |level1_level_requirement                = 0
 }}