--- Skill:ElderGuardianWhirlingBladesBarrage
+++ Export
@@ -1,5 +1,5 @@
 {{Skill
-|gem_description                         = Dive through enemies, dealing weapon damage. Only works with daggers, claws and one handed swords. Cannot be supported by Multistrike.
+|gem_description                         = Dive through enemies, dealing weapon damage. If dual wielding attacks with both weapons, dealing the damage of both in one hit. Only works with Daggers, Claws, and One-Handed Swords. Cannot be supported by Multistrike.
 |active_skill_name                       = Whirling Blades
 |item_class_id_restriction               = Dagger, One Hand Sword, Thrusting One Hand Sword, Claw, Rune Dagger
 |skill_id                                = ElderGuardianWhirlingBladesBarrage
@@ -10,17 +10,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_attack_speed_+%_final
-|static_stat1_value                      = -72
-|static_stat2_id                         = global_maim_on_hit
+|static_stat1_id                         = cast_time_overrides_attack_duration
+|static_stat1_value                      = 1
+|static_stat2_id                         = ignores_proximity_shield
 |static_stat2_value                      = 1
-|static_stat3_id                         = global_bleed_on_hit
-|static_stat3_value                      = 1
-|static_stat4_id                         = ignores_proximity_shield
+|static_stat3_id                         = active_skill_attack_speed_+%_final
+|static_stat3_value                      = -72
+|static_stat4_id                         = global_bleed_on_hit
 |static_stat4_value                      = 1
-|static_stat5_id                         = cast_time_overrides_attack_duration
+|static_stat5_id                         = global_maim_on_hit
 |static_stat5_value                      = 1
-|stat_text                               = 72% less Attack Speed<br>Maim on Hit<br>Causes Bleeding
+|stat_text                               = 72% less Attack Speed<br>Causes Bleeding<br>Maim on Hit
 |level1                                  = True
 |level1_level_requirement                = 0
 }}