--- Skill:CageSpiderCycloneTriggerSandstorms
+++ Export
@@ -10,16 +10,16 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = cyclone_extra_distance
 |static_stat1_value                      = 40
-|static_stat2_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat2_value                      = -40
-|static_stat3_id                         = active_skill_damage_+%_final
-|static_stat3_value                      = -60
-|static_stat4_id                         = active_skill_attack_speed_+%_final
-|static_stat4_value                      = 150
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
-|static_stat6_id                         = cyclone_has_triggered_skill
-|static_stat6_value                      = 1
+|static_stat2_id                         = cyclone_has_triggered_skill
+|static_stat2_value                      = 1
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat4_value                      = -40
+|static_stat5_id                         = active_skill_damage_+%_final
+|static_stat5_value                      = -60
+|static_stat6_id                         = active_skill_attack_speed_+%_final
+|static_stat6_value                      = 150
 |stat_text                               = Deals 60% less Damage<br>150% more Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 2
