--- Skill:HeistThugRangedExplosiveArrow
+++ Export
@@ -35,7 +35,7 @@
 |static_stat12_value                     = 3
 |static_stat13_id                        = explosive_arrow_stack_limit
 |static_stat13_value                     = 20
-|stat_text                               = Base duration is 2 seconds<br>Explosion deals 50% less Base Damage<br>Explosion will have 5 to 7 added Fire Damage<br>+2 to Explosion Radius per Explosive Arrow on Target, up to +12<br>Explosion deals 3% more Damage with Hits per Explosive Arrow on Target<br>Maximum 20 Explosive Arrows stuck in an Enemy
+|stat_text                               = Base duration is 2 seconds<br>Explosion deals 50% less Base Damage<br>Explosion will have 5 to 7 added Fire Damage<br>+0.2 metres to Explosion Radius per Explosive Arrow on Target, up to a maximum of +1.2 metres<br>Explosion deals 3% more Damage with Hits per Explosive Arrow on Target<br>Maximum 20 Explosive Arrows stuck in an Enemy
 |level1                                  = True
 |level1_level_requirement                = 1
 }}