--- Skill:DaemonSummonSkeletonDescent2
+++ Export
@@ -7,17 +7,18 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 45
 |static_stat1_id                         = alternate_minion
 |static_stat1_value                      = 1
-|static_stat2_id                         = active_skill_elemental_damage_+%_final
-|static_stat2_value                      = -50
+|static_stat2_id                         = base_number_of_skeletons_allowed
+|static_stat2_value                      = 8
 |static_stat3_id                         = base_skill_effect_duration
 |static_stat3_value                      = 45000
-|static_stat4_id                         = base_number_of_skeletons_allowed
-|static_stat4_value                      = 8
+|static_stat4_id                         = active_skill_elemental_damage_+%_final
+|static_stat4_value                      = -50
 |static_stat5_id                         = number_of_melee_skeletons_to_summon
 |static_stat5_value                      = 3
-|stat_text                               = Minions deal 50% less Elemental Damage<br>Base duration is 45 seconds<br>Maximum 8 Summoned Skeletons<br>Summons 3 Skeleton Warriors
+|stat_text                               = Maximum 8 Summoned Skeletons<br>Base duration is 45 seconds<br>Minions deal 50% less Elemental Damage<br>Summons 3 Skeleton Warriors
 |level1                                  = True
 |level1_level_requirement                = 12
 }}