--- Skill:WestIceNovaDaemon
+++ Export
@@ -8,21 +8,22 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_cold_damage
-|static_stat1_value                      = 639
-|static_stat2_id                         = spell_maximum_base_cold_damage
-|static_stat2_value                      = 998
-|static_stat3_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat3_value                      = 15
-|static_stat4_id                         = base_skill_effect_duration
-|static_stat4_value                      = 1500
-|static_stat5_id                         = never_freeze
+|static_duration                         = 1.5
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = 15
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = spell_minimum_base_cold_damage
+|static_stat3_value                      = 639
+|static_stat4_id                         = spell_maximum_base_cold_damage
+|static_stat4_value                      = 998
+|static_stat5_id                         = cannot_stun
 |static_stat5_value                      = 1
-|static_stat6_id                         = cannot_stun
-|static_stat6_value                      = 1
-|static_stat7_id                         = is_area_damage
+|static_stat6_id                         = base_skill_effect_duration
+|static_stat6_value                      = 1500
+|static_stat7_id                         = never_freeze
 |static_stat7_value                      = 1
-|stat_text                               = Deals 639 to 998 Cold Damage<br>Base duration is 1.5 seconds<br>Cannot Freeze<br>Your Hits cannot Stun Enemies
+|stat_text                               = Deals 639 to 998 Cold Damage<br>Your Hits cannot Stun Enemies<br>Base duration is 1.5 seconds<br>Cannot Freeze
 |level1                                  = True
 |level1_level_requirement                = 77
 }}