--- Skill:MPWHeistMilitaryMinibossProjCleave
+++ Export
@@ -7,21 +7,21 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 150
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_projectile_variation
-|static_stat1_value                      = 122
-|static_stat2_id                         = skill_physical_damage_%_to_convert_to_fire
-|static_stat2_value                      = 50
-|static_stat3_id                         = number_of_additional_projectiles
-|static_stat3_value                      = 2
-|static_stat4_id                         = base_is_projectile
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = maintain_projectile_direction_when_using_contact_position
+|static_stat2_value                      = 1
+|static_stat3_id                         = monster_projectile_variation
+|static_stat3_value                      = 122
+|static_stat4_id                         = projectile_uses_contact_position
 |static_stat4_value                      = 1
 |static_stat5_id                         = use_scaled_contact_offset
 |static_stat5_value                      = 1
-|static_stat6_id                         = projectile_uses_contact_position
-|static_stat6_value                      = 1
-|static_stat7_id                         = maintain_projectile_direction_when_using_contact_position
-|static_stat7_value                      = 1
-|stat_text                               = 50% of Physical Damage Converted to Fire Damage<br>Fires 2 additional Projectiles
+|static_stat6_id                         = number_of_additional_projectiles
+|static_stat6_value                      = 2
+|static_stat7_id                         = skill_physical_damage_%_to_convert_to_fire
+|static_stat7_value                      = 50
+|stat_text                               = Fires 2 additional Projectiles<br>50% of Physical Damage Converted to Fire Damage
 |level1                                  = True
 |level1_level_requirement                = 1
 }}