--- Skill:RhoaTableChargeTaunt
+++ Export
@@ -10,15 +10,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = shield_charge_damage_+%_maximum
-|static_stat1_value                      = 50
-|static_stat2_id                         = shield_charge_stun_duration_+%_maximum
-|static_stat2_value                      = 50
-|static_stat3_id                         = global_cannot_crit
-|static_stat3_value                      = 1
-|static_stat4_id                         = ignores_proximity_shield
-|static_stat4_value                      = 1
-|stat_text                               = 50% more Damage with Hits at Maximum Charge Distance<br>50% increased Stun Duration on enemies at Maximum charge distance<br>Never deal Critical Strikes
+|static_stat1_id                         = ignores_proximity_shield
+|static_stat1_value                      = 1
+|static_stat2_id                         = global_cannot_crit
+|static_stat2_value                      = 1
+|static_stat3_id                         = shield_charge_stun_duration_+%_maximum
+|static_stat3_value                      = 50
+|static_stat4_id                         = shield_charge_damage_+%_maximum
+|static_stat4_value                      = 50
+|stat_text                               = Never deal Critical Strikes<br>50% increased Stun Duration on enemies at Maximum charge distance<br>50% more Damage with Hits at Maximum Charge Distance
 |level1                                  = True
 |level1_level_requirement                = 1
 }}