--- Skill:UtulaFireSwing
+++ Export
@@ -0,0 +1,21 @@
+{{Skill
+|gem_description                         = Strike your foes down with a powerful blow.
+|active_skill_name                       = Default Attack
+|skill_id                                = UtulaFireSwing
+|cast_time                               = 1.0
+|required_level                          = 0
+|static_stored_uses                      = 1
+|static_cooldown                         = 10
+|static_damage_effectiveness             = 100
+|static_damage_multiplier                = 130
+|static_attack_speed_multiplier          = 100
+|static_stat1_id                         = action_attack_or_cast_time_uses_animation_length
+|static_stat1_value                      = 1
+|static_stat2_id                         = main_hand_base_maximum_attack_distance
+|static_stat2_value                      = 30
+|static_stat3_id                         = skill_physical_damage_%_to_convert_to_fire
+|static_stat3_value                      = 40
+|stat_text                               = 40% of Physical Damage Converted to Fire Damage
+|level1                                  = True
+|level1_level_requirement                = 0
+}}