--- Skill:PhysicalProjectileRetaliation
+++ Export
@@ -7,13 +7,13 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_reverse_point_blank_damage_-%_at_minimum_range
-|static_stat1_value                      = 30
-|static_stat2_id                         = cannot_stun
-|static_stat2_value                      = 1
-|static_stat3_id                         = is_trigger_daemon_when_hit_triggered_skill
+|static_stat1_id                         = is_trigger_daemon_when_hit_triggered_skill
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_reverse_point_blank_damage_-%_at_minimum_range
+|static_stat2_value                      = 30
+|static_stat3_id                         = cannot_stun
 |static_stat3_value                      = 1
-|stat_text                               = Deals (3-310) to (5-465) Physical Damage<br>(70-0)% reduced Projectile Speed<br>Your Hits cannot Stun Enemies
+|stat_text                               = Deals (3-310) to (5-465) Physical Damage<br>Your Hits cannot Stun Enemies<br>(70-0)% reduced Projectile Speed
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_stat_text                        = Deals 3 to 5 Physical Damage<br>70% reduced Projectile Speed
