--- Skill:MapDemonSuicideExplosion
+++ Export
@@ -7,10 +7,10 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = track_which_enemies_hit_index
-|static_stat1_value                      = 19
-|static_stat2_id                         = is_area_damage
-|static_stat2_value                      = 1
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = track_which_enemies_hit_index
+|static_stat2_value                      = 19
 |stat_text                               = Deals (19-26) to (28-39) Lightning Damage
 |level1                                  = True
 |level1_level_requirement                = 3
