--- Skill:ExileStaticStrike
+++ Export
@@ -8,13 +8,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 150
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = static_strike_explosion_damage_+%_final
-|static_stat1_value                      = -20
+|static_duration                         = 0.5
+|static_stat1_id                         = skill_physical_damage_%_to_convert_to_lightning
+|static_stat1_value                      = 100
 |static_stat2_id                         = base_skill_effect_duration
 |static_stat2_value                      = 500
-|static_stat3_id                         = skill_physical_damage_%_to_convert_to_lightning
-|static_stat3_value                      = 100
-|stat_text                               = Explosion deals 20% less Damage with Hits and Ailments<br>Base duration is 0.5 seconds<br>100% of Physical Damage Converted to Lightning Damage
+|static_stat3_id                         = static_strike_explosion_damage_+%_final
+|static_stat3_value                      = -20
+|stat_text                               = 100% of Physical Damage Converted to Lightning Damage<br>Base duration is 0.5 seconds<br>Explosion deals 20% less Damage with Hits and Ailments
 |level1                                  = True
 |level1_level_requirement                = 1
 }}