--- Skill:DoedreBubbleBurstPus
+++ Export
@@ -8,14 +8,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = skill_cold_damage_%_to_convert_to_chaos
-|static_stat1_value                      = 50
-|static_stat2_id                         = cluster_burst_spawn_amount
-|static_stat2_value                      = 4
-|static_stat3_id                         = is_area_damage
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = skill_cold_damage_%_to_convert_to_chaos
+|static_stat2_value                      = 50
+|static_stat3_id                         = visual_hit_effect_chaos_is_green
 |static_stat3_value                      = 1
-|static_stat4_id                         = visual_hit_effect_chaos_is_green
-|static_stat4_value                      = 1
+|static_stat4_id                         = cluster_burst_spawn_amount
+|static_stat4_value                      = 4
 |stat_text                               = Deals (11-321) to (17-482) Cold Damage<br>Creates 4 explosions
 |level1                                  = True
 |level1_level_requirement                = 1
