--- Skill:DeceleratingProjectileTora
+++ Export
@@ -7,17 +7,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 125
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = number_of_projectiles_override
+|static_stat1_id                         = base_is_projectile
 |static_stat1_value                      = 1
 |static_stat2_id                         = decelerating_projectile_speed_variation_+%
 |static_stat2_value                      = 150
-|static_stat3_id                         = skill_physical_damage_%_to_convert_to_chaos
-|static_stat3_value                      = 20
-|static_stat4_id                         = global_poison_on_hit
+|static_stat3_id                         = number_of_projectiles_override
+|static_stat3_value                      = 1
+|static_stat4_id                         = visual_hit_effect_chaos_is_green
 |static_stat4_value                      = 1
-|static_stat5_id                         = base_is_projectile
-|static_stat5_value                      = 1
-|static_stat6_id                         = visual_hit_effect_chaos_is_green
+|static_stat5_id                         = skill_physical_damage_%_to_convert_to_chaos
+|static_stat5_value                      = 20
+|static_stat6_id                         = global_poison_on_hit
 |static_stat6_value                      = 1
 |stat_text                               = 20% of Physical Damage Converted to Chaos Damage<br>Poisons Enemies on Hit
 |level1                                  = True
