--- Skill:AfflictionatorSummonMagicMonsters
+++ Export
@@ -9,10 +9,10 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = summon_monster_radius
-|static_stat1_value                      = 30
-|static_stat2_id                         = number_of_packs_to_summon
-|static_stat2_value                      = 1
+|static_stat1_id                         = number_of_packs_to_summon
+|static_stat1_value                      = 1
+|static_stat2_id                         = summon_monster_radius
+|static_stat2_value                      = 30
 |static_stat3_id                         = summoned_monster_rarity
 |static_stat3_value                      = 1
 |static_stat4_id                         = summoned_monsters_are_minions
