--- Skill:CorpseWalk
+++ Export
@@ -1,5 +1,5 @@
 {{Skill
-|gem_description                         = Spawns corpses around you while you move.
+|gem_description                         = Spawns corpses around you while you move. The corpses will be chosen from the monsters in the current area and any Spectres that have existed in this instance.
 |active_skill_name                       = Corpse Walk
 |skill_id                                = CorpseWalk
 |cast_time                               = 1.0
@@ -7,13 +7,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = cast_on_gain_skill
+|static_stat1_id                         = spell_uncastable_if_triggerable
 |static_stat1_value                      = 1
-|static_stat2_id                         = spell_uncastable_if_triggerable
-|static_stat2_value                      = 1
-|static_stat3_id                         = no_cost
+|static_stat2_id                         = desecrate_corpse_level
+|static_stat2_value                      = 100
+|static_stat3_id                         = cast_on_gain_skill
 |static_stat3_value                      = 1
-|stat_text                               = This Spell is Triggered when Gained<br>You cannot Cast this Spell directly<br>This Skill Costs nothing
+|static_stat4_id                         = no_cost
+|static_stat4_value                      = 1
+|stat_text                               = You cannot Cast this Spell directly<br>Spawns corpses up to Level 100<br>This Spell is Triggered when Gained<br>This Skill Costs nothing
 |level1                                  = True
 |level1_level_requirement                = 60
 }}