--- Skill:DoedreBloodProjectile
+++ Export
@@ -7,14 +7,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = projectile_minimum_range
-|static_stat1_value                      = 5
-|static_stat2_id                         = projectile_spread_radius
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = projectile_minimum_range
 |static_stat2_value                      = 5
-|static_stat3_id                         = spell_maximum_action_distance_+%
-|static_stat3_value                      = -60
-|static_stat4_id                         = base_is_projectile
-|static_stat4_value                      = 1
+|static_stat3_id                         = projectile_spread_radius
+|static_stat3_value                      = 5
+|static_stat4_id                         = spell_maximum_action_distance_+%
+|static_stat4_value                      = -60
 |stat_text                               = Deals (175-628) to (263-941) Physical Damage
 |level1                                  = True
 |level1_level_requirement                = 40
