--- Skill:EDASanctumTriggeredRainOfArrows
+++ Export
@@ -0,0 +1,21 @@
+{{Skill
+|gem_description                         = 
+|active_skill_name                       = 
+|skill_id                                = EDASanctumTriggeredRainOfArrows
+|cast_time                               = 1.0
+|required_level                          = 1
+|static_damage_effectiveness             = 100
+|static_damage_multiplier                = 95
+|static_attack_speed_multiplier          = 100
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = 50
+|static_stat2_id                         = base_is_projectile
+|static_stat2_value                      = 1
+|static_stat3_id                         = base_sanctum_damage
+|static_stat3_value                      = 10
+|static_stat4_id                         = skill_physical_damage_%_to_convert_to_cold
+|static_stat4_value                      = 30
+|stat_text                               = 30% of Physical Damage Converted to Cold Damage
+|level1                                  = True
+|level1_level_requirement                = 1
+}}