--- Skill:IntimidayingCryTriggered
+++ Export
@@ -11,27 +11,28 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_skill_effect_duration
-|static_stat1_value                      = 4000
-|static_stat2_id                         = skill_empowers_next_x_melee_attacks
-|static_stat2_value                      = 2
-|static_stat3_id                         = intimidating_cry_enemy_phys_reduction_%_penalty_vs_hit_per_5_MP
-|static_stat3_value                      = 5
-|static_stat4_id                         = warcry_count_power_from_enemies
-|static_stat4_value                      = 1
-|static_stat5_id                         = intimidating_cry_empowerd_attacks_deal_double_damage_display
+|static_duration                         = 4
+|static_stat1_id                         = base_deal_no_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = cannot_cancel_skill_before_contact_point
+|static_stat2_value                      = 1
+|static_stat3_id                         = use_intimidating_cry_buff_visual_for_intimidate
+|static_stat3_value                      = 1
+|static_stat4_id                         = base_skill_effect_duration
+|static_stat4_value                      = 4000
+|static_stat5_id                         = enemies_taunted_by_your_warcies_are_intimidated
 |static_stat5_value                      = 1
-|static_stat6_id                         = enemies_taunted_by_your_warcies_are_intimidated
+|static_stat6_id                         = skill_has_trigger_from_unique_item
 |static_stat6_value                      = 1
-|static_stat7_id                         = skill_has_trigger_from_unique_item
+|static_stat7_id                         = warcry_count_power_from_enemies
 |static_stat7_value                      = 1
-|static_stat8_id                         = base_deal_no_damage
-|static_stat8_value                      = 1
-|static_stat9_id                         = cannot_cancel_skill_before_contact_point
-|static_stat9_value                      = 1
-|static_stat10_id                        = use_intimidating_cry_buff_visual_for_intimidate
+|static_stat8_id                         = intimidating_cry_enemy_phys_reduction_%_penalty_vs_hit_per_5_MP
+|static_stat8_value                      = 5
+|static_stat9_id                         = skill_empowers_next_x_melee_attacks
+|static_stat9_value                      = 2
+|static_stat10_id                        = intimidating_cry_empowerd_attacks_deal_double_damage_display
 |static_stat10_value                     = 1
-|stat_text                               = Base duration is 4 seconds<br>Exerts the next 2 Melee Attacks you perform<br>Buff causes your Hits to Overwhelm 5% Physical Damage Reduction per 5 Power, up to 30%<br>Counts total Power of Enemies in Range<br>Exerted Attacks deal Double Damage<br>Enemies Taunted by this Warcry are Intimidated<br>This Skill is Triggered by a Unique Item
+|stat_text                               = Base duration is 4 seconds<br>Enemies Taunted by this Warcry are Intimidated<br>This Skill is Triggered by a Unique Item<br>Counts total Power of Enemies in Range<br>Buff causes your Hits to Overwhelm 5% Physical Damage Reduction per 5 Power, up to 30%<br>Exerts the next 2 Melee Attacks you perform<br>Exerted Attacks deal Double Damage
 |level1                                  = True
 |level1_level_requirement                = 10
 }}