--- Skill:AfflictionOnDeathFirestorm
+++ Export
@@ -8,21 +8,22 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = fire_storm_fireball_delay_ms
-|static_stat1_value                      = 250
+|static_duration                         = 0.9
+|static_stat1_id                         = firestorm_avoid_unwalkable_terrain
+|static_stat1_value                      = 1
 |static_stat2_id                         = firestorm_base_area_of_effect_+%
 |static_stat2_value                      = -75
-|static_stat3_id                         = base_skill_effect_duration
-|static_stat3_value                      = 900
-|static_stat4_id                         = skill_physical_damage_%_to_convert_to_fire
-|static_stat4_value                      = 75
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
-|static_stat6_id                         = firestorm_avoid_unwalkable_terrain
-|static_stat6_value                      = 1
-|static_stat7_id                         = firestorm_use_server_effects
-|static_stat7_value                      = 1
-|stat_text                               = Deals (4-560) to (7-840) Physical Damage<br>One impact every 0.25 seconds<br>Deals (-30-129)% more Damage<br>Base duration is 0.9 seconds<br>75% of Physical Damage Converted to Fire Damage
+|static_stat3_id                         = firestorm_use_server_effects
+|static_stat3_value                      = 1
+|static_stat4_id                         = is_area_damage
+|static_stat4_value                      = 1
+|static_stat5_id                         = base_skill_effect_duration
+|static_stat5_value                      = 900
+|static_stat6_id                         = fire_storm_fireball_delay_ms
+|static_stat6_value                      = 250
+|static_stat7_id                         = skill_physical_damage_%_to_convert_to_fire
+|static_stat7_value                      = 75
+|stat_text                               = Deals (4-560) to (7-840) Physical Damage<br>Deals (-30-129)% more Damage<br>Base duration is 0.9 seconds<br>One impact every 0.25 seconds<br>75% of Physical Damage Converted to Fire Damage
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_stat_text                        = Deals 4 to 7 Physical Damage<br>Deals 30% less Damage
