--- Skill:BrutusGroundSlam
+++ Export
@@ -9,13 +9,13 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_damage_+%_final
-|static_stat1_value                      = 120
-|static_stat2_id                         = always_stun
-|static_stat2_value                      = 1
-|static_stat3_id                         = is_area_damage
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = active_skill_damage_+%_final
+|static_stat2_value                      = 120
+|static_stat3_id                         = always_stun
 |static_stat3_value                      = 1
-|stat_text                               = (20-46)% less Attack Speed<br>Deals 120% more Damage<br>Stuns Enemies
+|stat_text                               = Deals 120% more Damage<br>Stuns Enemies<br>(20-46)% less Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 9
 |level1_stat_text                        = 20% less Attack Speed
@@ -34,8 +34,8 @@
 |level4                                  = True
 |level4_level_requirement                = 66
 |level4_stat_text                        = 46% less Attack Speed
-|level4_stat1_id                         = active_skill_attack_speed_+%_final
-|level4_stat1_value                      = -46
-|level4_stat2_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|level4_stat2_value                      = -30
+|level4_stat1_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|level4_stat1_value                      = -30
+|level4_stat2_id                         = active_skill_attack_speed_+%_final
+|level4_stat2_value                      = -46
 }}