--- Skill:IceNovaOnDeathWaterElemental
+++ Export
@@ -10,19 +10,19 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = 15
-|static_stat2_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat2_value                      = -25
-|static_stat3_id                         = cast_on_death_%
-|static_stat3_value                      = 100
-|static_stat4_id                         = spell_uncastable_if_triggerable
-|static_stat4_value                      = 1
-|static_stat5_id                         = spell_only_castable_on_death
+|static_stat2_id                         = force_lite_skill_effects
+|static_stat2_value                      = 1
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat4_value                      = -25
+|static_stat5_id                         = spell_uncastable_if_triggerable
 |static_stat5_value                      = 1
-|static_stat6_id                         = is_area_damage
+|static_stat6_id                         = spell_only_castable_on_death
 |static_stat6_value                      = 1
-|static_stat7_id                         = force_lite_skill_effects
-|static_stat7_value                      = 1
-|stat_text                               = Deals (8-782) to (12-1173) Cold Damage<br>100% chance to Trigger this Spell on Death<br>You cannot Cast this Spell directly
+|static_stat7_id                         = cast_on_death_%
+|static_stat7_value                      = 100
+|stat_text                               = Deals (8-782) to (12-1173) Cold Damage<br>You cannot Cast this Spell directly<br>100% chance to Trigger this Spell on Death
 |level1                                  = True
 |level1_level_requirement                = 4
 |level1_stat_text                        = Deals 8 to 12 Cold Damage
