--- Skill:IzaroDaemonBladefall
+++ Export
@@ -11,12 +11,12 @@
 |static_damage_effectiveness             = 90
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = bladefall_number_of_volleys
-|static_stat1_value                      = 5
-|static_stat2_id                         = bladefall_damage_per_stage_+%_final
-|static_stat2_value                      = -5
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = bladefall_number_of_volleys
+|static_stat2_value                      = 5
+|static_stat3_id                         = bladefall_damage_per_stage_+%_final
+|static_stat3_value                      = -5
 |stat_text                               = Deals (16-1521) to (23-2282) Physical Damage<br>5 Volleys<br>5% less Damage per Volley
 |level1                                  = True
 |level1_level_requirement                = 12
