--- Skill:AncestralTawhanukuSoulRend
+++ Export
@@ -0,0 +1,24 @@
+{{Skill
+|gem_description                         = 
+|active_skill_name                       = 
+|skill_id                                = AncestralTawhanukuSoulRend
+|cast_time                               = 1.0
+|required_level                          = 0
+|static_damage_effectiveness             = 100
+|static_damage_multiplier                = 100
+|static_attack_speed_multiplier          = 100
+|static_duration                         = 1
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = -30
+|static_stat2_id                         = active_skill_projectile_speed_+%_variation_final
+|static_stat2_value                      = 25
+|static_stat3_id                         = base_is_projectile
+|static_stat3_value                      = 1
+|static_stat4_id                         = base_chaos_damage_to_deal_per_minute
+|static_stat4_value                      = 84
+|static_stat5_id                         = base_skill_effect_duration
+|static_stat5_value                      = 1000
+|stat_text                               = Deals 1.4 Base Chaos Damage per second<br>Base duration is 1 seconds
+|level1                                  = True
+|level1_level_requirement                = 0
+}}