--- Skill:ChattersIceCrash
+++ Export
@@ -12,17 +12,17 @@
 |static_attack_speed_multiplier          = 85
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = -20
-|static_stat2_id                         = base_chance_to_freeze_%
-|static_stat2_value                      = 25
-|static_stat3_id                         = ice_crash_third_hit_damage_+%_final
-|static_stat3_value                      = -30
-|static_stat4_id                         = ice_crash_second_hit_damage_+%_final
-|static_stat4_value                      = -15
-|static_stat5_id                         = skill_physical_damage_%_to_convert_to_cold
-|static_stat5_value                      = 100
-|static_stat6_id                         = is_area_damage
-|static_stat6_value                      = 1
-|stat_text                               = 25% chance to Freeze enemies<br>Third Stage deals 30% less Damage<br>Second Stage deals 15% less Damage<br>100% of Physical Damage Converted to Cold Damage
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = base_chance_to_freeze_%
+|static_stat3_value                      = 25
+|static_stat4_id                         = skill_physical_damage_%_to_convert_to_cold
+|static_stat4_value                      = 100
+|static_stat5_id                         = ice_crash_second_hit_damage_+%_final
+|static_stat5_value                      = -15
+|static_stat6_id                         = ice_crash_third_hit_damage_+%_final
+|static_stat6_value                      = -30
+|stat_text                               = 25% chance to Freeze enemies<br>100% of Physical Damage Converted to Cold Damage<br>Second Stage deals 15% less Damage<br>Third Stage deals 30% less Damage
 |level1                                  = True
 |level1_level_requirement                = 1
 }}