--- Skill:BoneGolemMultiAttack
+++ Export
@@ -7,10 +7,10 @@
 |static_damage_effectiveness             = 135
 |static_damage_multiplier                = 135
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = bone_golem_damage_+%_final_after_third_combo_index
-|static_stat1_value                      = 20
-|static_stat2_id                         = bone_golem_attack_speed_+%_final_after_third_combo_index
-|static_stat2_value                      = 200
+|static_stat1_id                         = bone_golem_attack_speed_+%_final_after_third_combo_index
+|static_stat1_value                      = 200
+|static_stat2_id                         = bone_golem_damage_+%_final_after_third_combo_index
+|static_stat2_value                      = 20
 |stat_text                               = 
 |level1                                  = True
 |level1_level_requirement                = 1
