--- Skill:HeistTwinsWhirlDashTriggeredVin
+++ Export
@@ -11,10 +11,10 @@
 |static_stat1_value                      = 100
 |static_stat2_id                         = chill_minimum_slow_%_from_skill
 |static_stat2_value                      = 30
-|static_stat3_id                         = skill_physical_damage_%_to_convert_to_cold
-|static_stat3_value                      = 50
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = skill_physical_damage_%_to_convert_to_cold
+|static_stat4_value                      = 50
 |stat_text                               = 50% of Physical Damage Converted to Cold Damage
 |level1                                  = True
 |level1_level_requirement                = 0
