--- Skill:BetrayalMassSkeletonRainOfArrows
+++ Export
@@ -18,7 +18,7 @@
 |static_stat3_value                      = 1
 |static_stat4_id                         = melee_range_+
 |static_stat4_value                      = 50
-|stat_text                               = +50 to Melee Strike Range
+|stat_text                               = +5 metres to Melee Strike Range
 |level1                                  = True
 |level1_level_requirement                = 1
 }}