--- Skill:HolyElderSpectralRuckusTriggerSlam
+++ Export
@@ -9,12 +9,12 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = 150
-|static_stat2_id                         = active_skill_damage_+%_final
-|static_stat2_value                      = 100
-|static_stat3_id                         = is_area_damage
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = thrown_weapon_on_return_trigger_id
 |static_stat3_value                      = 1
-|static_stat4_id                         = thrown_weapon_on_return_trigger_id
-|static_stat4_value                      = 1
+|static_stat4_id                         = active_skill_damage_+%_final
+|static_stat4_value                      = 100
 |stat_text                               = Deals 100% more Damage
 |level1                                  = True
 |level1_level_requirement                = 1
