--- Skill:MPSUltimatumSnakeProjectileBasic
+++ Export
@@ -8,27 +8,27 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage
-|static_stat1_value                      = 2
-|static_stat2_id                         = spell_maximum_base_lightning_damage
-|static_stat2_value                      = 3
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_projectile_variation
+|static_stat2_value                      = 182
 |static_stat3_id                         = number_of_projectiles_override
 |static_stat3_value                      = 1
-|static_stat4_id                         = monster_projectile_variation
-|static_stat4_value                      = 182
-|static_stat5_id                         = skill_lightning_damage_%_to_convert_to_chaos
-|static_stat5_value                      = 50
-|static_stat6_id                         = always_pierce
+|static_stat4_id                         = projectile_uses_contact_position
+|static_stat4_value                      = 1
+|static_stat5_id                         = use_scaled_contact_offset
+|static_stat5_value                      = 1
+|static_stat6_id                         = visual_hit_effect_chaos_is_green
 |static_stat6_value                      = 1
-|static_stat7_id                         = base_is_projectile
-|static_stat7_value                      = 1
-|static_stat8_id                         = projectile_uses_contact_position
-|static_stat8_value                      = 1
-|static_stat9_id                         = use_scaled_contact_offset
+|static_stat7_id                         = spell_minimum_base_lightning_damage
+|static_stat7_value                      = 2
+|static_stat8_id                         = spell_maximum_base_lightning_damage
+|static_stat8_value                      = 3
+|static_stat9_id                         = always_pierce
 |static_stat9_value                      = 1
-|static_stat10_id                        = visual_hit_effect_chaos_is_green
-|static_stat10_value                     = 1
-|stat_text                               = Deals 2 to 3 Lightning Damage<br>50% of Lightning Damage Converted to Chaos Damage<br>Projectiles Pierce all Targets
+|static_stat10_id                        = skill_lightning_damage_%_to_convert_to_chaos
+|static_stat10_value                     = 50
+|stat_text                               = Deals 2 to 3 Lightning Damage<br>Projectiles Pierce all Targets<br>50% of Lightning Damage Converted to Chaos Damage
 |level1                                  = True
 |level1_level_requirement                = 0
 }}