--- Skill:DemonLightningWarp
+++ Export
@@ -9,12 +9,12 @@
 |static_damage_effectiveness             = 50
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = skill_effect_duration_+%
-|static_stat1_value                      = -50
-|static_stat2_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat1_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat1_value                      = 1
+|static_stat2_id                         = is_area_damage
 |static_stat2_value                      = 1
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
+|static_stat3_id                         = skill_effect_duration_+%
+|static_stat3_value                      = -50
 |stat_text                               = Deals (5-489) to (14-1466) Lightning Damage<br>50% reduced Duration
 |level1                                  = True
 |level1_level_requirement                = 3
