--- Skill:CatarinaCarpetMortar
+++ Export
@@ -9,10 +9,10 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 9
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 14
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
 |static_stat3_id                         = mortar_distance_in_front_of_target
 |static_stat3_value                      = 15
 |static_stat4_id                         = projectile_distance_variance
@@ -21,17 +21,17 @@
 |static_stat5_value                      = 20
 |static_stat6_id                         = projectile_spread_radius_per_additional_projectile
 |static_stat6_value                      = 10
-|static_stat7_id                         = skill_physical_damage_%_to_convert_to_fire
-|static_stat7_value                      = 25
-|static_stat8_id                         = number_of_additional_projectiles
-|static_stat8_value                      = 1
-|static_stat9_id                         = base_is_projectile
-|static_stat9_value                      = 1
-|static_stat10_id                        = is_area_damage
+|static_stat7_id                         = projectile_uses_contact_position
+|static_stat7_value                      = 1
+|static_stat8_id                         = spell_minimum_base_physical_damage
+|static_stat8_value                      = 9
+|static_stat9_id                         = spell_maximum_base_physical_damage
+|static_stat9_value                      = 14
+|static_stat10_id                        = number_of_additional_projectiles
 |static_stat10_value                     = 1
-|static_stat11_id                        = projectile_uses_contact_position
-|static_stat11_value                     = 1
-|stat_text                               = Deals 9 to 14 Physical Damage<br>25% of Physical Damage Converted to Fire Damage<br>Fires an additional Projectile
+|static_stat11_id                        = skill_physical_damage_%_to_convert_to_fire
+|static_stat11_value                     = 25
+|stat_text                               = Deals 9 to 14 Physical Damage<br>Fires an additional Projectile<br>25% of Physical Damage Converted to Fire Damage
 |level1                                  = True
 |level1_level_requirement                = 1
 }}