--- Skill:MavenBrainProjectileBarrageExplode
+++ Export
@@ -8,14 +8,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_cold_damage
-|static_stat1_value                      = 7
-|static_stat2_id                         = spell_maximum_base_cold_damage
-|static_stat2_value                      = 10
-|static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat3_value                      = -25
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat2_value                      = -25
+|static_stat3_id                         = spell_minimum_base_cold_damage
+|static_stat3_value                      = 7
+|static_stat4_id                         = spell_maximum_base_cold_damage
+|static_stat4_value                      = 10
 |stat_text                               = Deals 7 to 10 Cold Damage
 |level1                                  = True
 |level1_level_requirement                = 0
