--- Skill:MetamorphosisClusterBurst
+++ Export
@@ -12,12 +12,12 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = -29
-|static_stat2_id                         = cluster_burst_spawn_amount
-|static_stat2_value                      = 3
-|static_stat3_id                         = base_is_projectile
+|static_stat2_id                         = base_is_projectile
+|static_stat2_value                      = 1
+|static_stat3_id                         = skill_can_fire_wand_projectiles
 |static_stat3_value                      = 1
-|static_stat4_id                         = skill_can_fire_wand_projectiles
-|static_stat4_value                      = 1
+|static_stat4_id                         = cluster_burst_spawn_amount
+|static_stat4_value                      = 3
 |stat_text                               = Creates 3 explosions
 |level1                                  = True
 |level1_level_requirement                = 2
