--- Skill:AfflictionDoodadSummonPackRare2
+++ Export
@@ -7,13 +7,13 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = summoned_monster_rarity
-|static_stat1_value                      = 2
-|static_stat2_id                         = summon_specific_monsters_radius_+%
-|static_stat2_value                      = -60
-|static_stat3_id                         = alternate_minion
-|static_stat3_value                      = 482
-|static_stat4_id                         = number_of_monsters_to_summon
+|static_stat1_id                         = alternate_minion
+|static_stat1_value                      = 482
+|static_stat2_id                         = number_of_monsters_to_summon
+|static_stat2_value                      = 2
+|static_stat3_id                         = summon_specific_monsters_radius_+%
+|static_stat3_value                      = -60
+|static_stat4_id                         = summoned_monster_rarity
 |static_stat4_value                      = 2
 |stat_text                               = 
 |level1                                  = True
