--- Skill:MonsterHeavyStrikeUnique
+++ Export
@@ -8,17 +8,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 75
-|static_stat1_id                         = active_skill_damage_+%_final
-|static_stat1_value                      = 50
-|static_stat2_id                         = physical_damage_+%
-|static_stat2_value                      = 15
-|static_stat3_id                         = base_stun_threshold_reduction_+%
-|static_stat3_value                      = 250
-|static_stat4_id                         = global_knockback
-|static_stat4_value                      = 1
-|static_stat5_id                         = action_attack_or_cast_time_uses_animation_length
-|static_stat5_value                      = 1
-|stat_text                               = Deals 50% more Damage<br>15% increased Physical Damage<br>250% reduced Enemy Stun Threshold<br>Knocks Enemies Back on Hit
+|static_stat1_id                         = action_attack_or_cast_time_uses_animation_length
+|static_stat1_value                      = 1
+|static_stat2_id                         = global_knockback
+|static_stat2_value                      = 1
+|static_stat3_id                         = active_skill_damage_+%_final
+|static_stat3_value                      = 50
+|static_stat4_id                         = base_stun_threshold_reduction_+%
+|static_stat4_value                      = 250
+|static_stat5_id                         = physical_damage_+%
+|static_stat5_value                      = 15
+|stat_text                               = Knocks Enemies Back on Hit<br>Deals 50% more Damage<br>250% reduced Enemy Stun Threshold<br>15% increased Physical Damage
 |level1                                  = True
 |level1_level_requirement                = 1
 }}