--- Skill:GSReaperWraithCoffinPush
+++ Export
@@ -7,17 +7,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 4
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 6
-|static_stat3_id                         = skill_physical_damage_%_to_convert_to_chaos
-|static_stat3_value                      = 25
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_physical_damage
+|static_stat2_value                      = 4
+|static_stat3_id                         = spell_maximum_base_physical_damage
+|static_stat3_value                      = 6
 |static_stat4_id                         = global_knockback
 |static_stat4_value                      = 1
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
-|stat_text                               = Deals 4 to 6 Physical Damage<br>25% of Physical Damage Converted to Chaos Damage<br>Knocks Enemies Back on Hit
+|static_stat5_id                         = skill_physical_damage_%_to_convert_to_chaos
+|static_stat5_value                      = 25
+|stat_text                               = Deals 4 to 6 Physical Damage<br>Knocks Enemies Back on Hit<br>25% of Physical Damage Converted to Chaos Damage
 |level1                                  = True
 |level1_level_requirement                = 0
 }}