--- Skill:ElderTentacleMinionProjectileEpic
+++ Export
@@ -10,16 +10,16 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = damage_vs_tiki_totems_+%
-|static_stat1_value                      = 50
-|static_stat2_id                         = skill_visual_scale_+%
-|static_stat2_value                      = 100
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = damage_vs_tiki_totems_+%
+|static_stat2_value                      = 50
 |static_stat3_id                         = monster_projectile_variation
 |static_stat3_value                      = 48
-|static_stat4_id                         = base_is_projectile
+|static_stat4_id                         = projectile_uses_contact_position
 |static_stat4_value                      = 1
-|static_stat5_id                         = projectile_uses_contact_position
-|static_stat5_value                      = 1
+|static_stat5_id                         = skill_visual_scale_+%
+|static_stat5_value                      = 100
 |stat_text                               = Deals (2510-6022) to (3765-9034) Lightning Damage
 |level1                                  = True
 |level1_level_requirement                = 68
