--- Skill:GARootSpiderSpitImpact
+++ Export
@@ -6,13 +6,13 @@
 |required_level                          = 1
 |static_damage_effectiveness             = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = skill_physical_damage_%_to_convert_to_fire
-|static_stat1_value                      = 60
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
 |static_stat2_id                         = always_stun
 |static_stat2_value                      = 1
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
-|stat_text                               = 60% of Physical Damage Converted to Fire Damage<br>Stuns Enemies
+|static_stat3_id                         = skill_physical_damage_%_to_convert_to_fire
+|static_stat3_value                      = 60
+|stat_text                               = Stuns Enemies<br>60% of Physical Damage Converted to Fire Damage
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_damage_multiplier                = 600
