--- Skill:CameraLensExpandingDonut
+++ Export
@@ -10,17 +10,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_cold_damage
-|static_stat1_value                      = 5078
-|static_stat2_id                         = spell_maximum_base_cold_damage
-|static_stat2_value                      = 7617
-|static_stat3_id                         = base_secondary_skill_effect_duration
-|static_stat3_value                      = 250
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
-|static_stat5_id                         = base_cold_damage_to_deal_per_minute
-|static_stat5_value                      = 264485
-|stat_text                               = Deals 5078 to 7617 Cold Damage<br>Base secondary duration is 0.25 seconds<br>Deals 4408.1 Base Cold Damage per second
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_cold_damage
+|static_stat2_value                      = 5078
+|static_stat3_id                         = spell_maximum_base_cold_damage
+|static_stat3_value                      = 7617
+|static_stat4_id                         = base_cold_damage_to_deal_per_minute
+|static_stat4_value                      = 264485
+|static_stat5_id                         = base_secondary_skill_effect_duration
+|static_stat5_value                      = 250
+|stat_text                               = Deals 5078 to 7617 Cold Damage<br>Deals 4408.1 Base Cold Damage per second<br>Base secondary duration is 0.25 seconds
 |level1                                  = True
 |level1_level_requirement                = 75
 }}