--- Skill:GemlingCascade2
+++ Export
@@ -10,20 +10,20 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 7
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 11
-|static_stat3_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat3_value                      = -25
-|static_stat4_id                         = active_skill_base_radius_+
-|static_stat4_value                      = 3
-|static_stat5_id                         = skill_physical_damage_%_to_convert_to_cold
-|static_stat5_value                      = 100
-|static_stat6_id                         = upheaval_number_of_spikes
-|static_stat6_value                      = 6
-|static_stat7_id                         = is_area_damage
-|static_stat7_value                      = 1
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = -25
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = spell_minimum_base_physical_damage
+|static_stat3_value                      = 7
+|static_stat4_id                         = spell_maximum_base_physical_damage
+|static_stat4_value                      = 11
+|static_stat5_id                         = active_skill_base_radius_+
+|static_stat5_value                      = 3
+|static_stat6_id                         = skill_physical_damage_%_to_convert_to_cold
+|static_stat6_value                      = 100
+|static_stat7_id                         = upheaval_number_of_spikes
+|static_stat7_value                      = 6
 |stat_text                               = Deals 7 to 11 Physical Damage<br>+3 to radius<br>100% of Physical Damage Converted to Cold Damage<br>Causes 6 Bursts
 |level1                                  = True
 |level1_level_requirement                = 1
