--- Skill:AtlasInvadersConsumeFodder2Arc
+++ Export
@@ -10,25 +10,25 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage
-|static_stat1_value                      = 4
-|static_stat2_id                         = spell_maximum_base_lightning_damage
-|static_stat2_value                      = 12
+|static_stat1_id                         = arc_chain_distance
+|static_stat1_value                      = 35
+|static_stat2_id                         = arc_enhanced_behaviour
+|static_stat2_value                      = 1
 |static_stat3_id                         = arc_to_minion_monster_variety_hash
 |static_stat3_value                      = 5571
-|static_stat4_id                         = spell_maximum_action_distance_+%
-|static_stat4_value                      = -60
-|static_stat5_id                         = arc_chain_distance
-|static_stat5_value                      = 35
-|static_stat6_id                         = number_of_chains
-|static_stat6_value                      = 7
-|static_stat7_id                         = base_chance_to_shock_%
-|static_stat7_value                      = 20
-|static_stat8_id                         = arc_enhanced_behaviour
-|static_stat8_value                      = 1
-|static_stat9_id                         = disable_visual_hit_effect
-|static_stat9_value                      = 1
-|stat_text                               = Deals 4 to 12 Lightning Damage<br>Chains +7 Times<br>20% chance to Shock enemies
+|static_stat4_id                         = disable_visual_hit_effect
+|static_stat4_value                      = 1
+|static_stat5_id                         = spell_maximum_action_distance_+%
+|static_stat5_value                      = -60
+|static_stat6_id                         = spell_minimum_base_lightning_damage
+|static_stat6_value                      = 4
+|static_stat7_id                         = spell_maximum_base_lightning_damage
+|static_stat7_value                      = 12
+|static_stat8_id                         = base_chance_to_shock_%
+|static_stat8_value                      = 20
+|static_stat9_id                         = number_of_chains
+|static_stat9_value                      = 7
+|stat_text                               = Deals 4 to 12 Lightning Damage<br>20% chance to Shock enemies<br>Chains +7 Times
 |level1                                  = True
 |level1_level_requirement                = 1
 }}