--- Skill:GladiatorBossEliteVaalGlacialHammer
+++ Export
@@ -26,6 +26,8 @@
 |stat_text                               = Deals 50% more Damage<br>Base duration is 4 seconds<br>20% chance to Freeze enemies<br>29% increased Physical Damage<br>50% of Physical Damage Converted to Cold Damage
 |level1                                  = True
 |level1_level_requirement                = 38
+|level1_damage_effectiveness             = 100
+|level1_damage_multiplier                = 100
 |level2                                  = True
 |level2_level_requirement                = 56
 |level2_damage_effectiveness             = 140