--- Skill:GoreShockwave
+++ Export
@@ -17,11 +17,9 @@
 |level1_level_requirement                = 1
 |level1_damage_effectiveness             = 160
 |level1_damage_multiplier                = 160
-|level1_stat_text                        = +1 to radius<br>0% increased Cooldown Recovery Rate
+|level1_stat_text                        = +1 to radius
 |level1_stat1_id                         = active_skill_base_radius_+
 |level1_stat1_value                      = 1
-|level1_stat2_id                         = base_cooldown_speed_+%
-|level1_stat2_value                      = 0
 |level2                                  = True
 |level2_level_requirement                = 1
 |level2_damage_effectiveness             = 181