--- Skill:HeraldOfAgonyMinionTailSpikeFaithGuard
+++ Export
@@ -7,14 +7,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_projectile_variation
-|static_stat1_value                      = 41
-|static_stat2_id                         = number_of_additional_projectiles
-|static_stat2_value                      = 4
-|static_stat3_id                         = base_is_projectile
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_projectile_variation
+|static_stat2_value                      = 41
+|static_stat3_id                         = visual_hit_effect_chaos_is_green
 |static_stat3_value                      = 1
-|static_stat4_id                         = visual_hit_effect_chaos_is_green
-|static_stat4_value                      = 1
+|static_stat4_id                         = number_of_additional_projectiles
+|static_stat4_value                      = 4
 |stat_text                               = Fires 4 additional Projectiles
 |level1                                  = True
 |level1_level_requirement                = 1
