--- Skill:OtherworldlyThrallGeometryProjectileRightAhead
+++ Export
@@ -7,19 +7,19 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage
-|static_stat1_value                      = 6
-|static_stat2_id                         = spell_maximum_base_lightning_damage
-|static_stat2_value                      = 12
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_reverse_point_blank_damage_-%_at_minimum_range
+|static_stat2_value                      = 30
 |static_stat3_id                         = number_of_projectiles_override
 |static_stat3_value                      = 1
-|static_stat4_id                         = monster_reverse_point_blank_damage_-%_at_minimum_range
-|static_stat4_value                      = 30
-|static_stat5_id                         = always_pierce
-|static_stat5_value                      = 1
-|static_stat6_id                         = base_is_projectile
-|static_stat6_value                      = 1
-|static_stat7_id                         = projectile_uses_contact_position
+|static_stat4_id                         = projectile_uses_contact_position
+|static_stat4_value                      = 1
+|static_stat5_id                         = spell_minimum_base_lightning_damage
+|static_stat5_value                      = 6
+|static_stat6_id                         = spell_maximum_base_lightning_damage
+|static_stat6_value                      = 12
+|static_stat7_id                         = always_pierce
 |static_stat7_value                      = 1
 |stat_text                               = Deals 6 to 12 Lightning Damage<br>Projectiles Pierce all Targets
 |level1                                  = True
