--- Skill:EmberSeaWitch
+++ Export
@@ -1,5 +1,5 @@
 {{Skill
-|gem_description                         = Fires projectiles which fly in an arc, exploding on impact with either enemies or the ground near where you target. Targeting farther away causes the projectiles to spread out and land over a wider area.
+|gem_description                         = Fires projectiles which fly in an arc, dealing area damage on impact with either enemies or the ground near where you target. Targeting farther away causes the projectiles to spread out and land over a wider area.
 |active_skill_name                       = Blazing Salvo
 |skill_id                                = EmberSeaWitch
 |cast_time                               = 1.333
@@ -8,15 +8,15 @@
 |static_damage_effectiveness             = 40
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_fire_damage
-|static_stat1_value                      = 4
-|static_stat2_id                         = spell_maximum_base_fire_damage
-|static_stat2_value                      = 6
-|static_stat3_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat3_value                      = -70
-|static_stat4_id                         = number_of_additional_projectiles
-|static_stat4_value                      = 1
-|static_stat5_id                         = base_is_projectile
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = -70
+|static_stat2_id                         = base_is_projectile
+|static_stat2_value                      = 1
+|static_stat3_id                         = spell_minimum_base_fire_damage
+|static_stat3_value                      = 4
+|static_stat4_id                         = spell_maximum_base_fire_damage
+|static_stat4_value                      = 6
+|static_stat5_id                         = number_of_additional_projectiles
 |static_stat5_value                      = 1
 |stat_text                               = Deals 4 to 6 Fire Damage<br>Fires an additional Projectile
 |level1                                  = True
