--- Skill:AtlasExileOrionSpammableBeamBlastMedium
+++ Export
@@ -10,12 +10,12 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = 50
-|static_stat2_id                         = skill_physical_damage_%_to_convert_to_lightning
-|static_stat2_value                      = 30
-|static_stat3_id                         = skill_physical_damage_%_to_convert_to_fire
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = skill_physical_damage_%_to_convert_to_lightning
 |static_stat3_value                      = 30
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
+|static_stat4_id                         = skill_physical_damage_%_to_convert_to_fire
+|static_stat4_value                      = 30
 |stat_text                               = Deals (8-2278) to (12-3416) Physical Damage<br>30% of Physical Damage Converted to Lightning Damage<br>30% of Physical Damage Converted to Fire Damage
 |level1                                  = True
 |level1_level_requirement                = 1
