--- Skill:BeyondDemonFrostCall
+++ Export
@@ -14,6 +14,7 @@
 |stat_text                               = Deals (7-1239) to (10-1859) Cold Damage<br>Base duration is (2.5-1.75) seconds<br>(75-50)% reduced Cast Speed
 |level1                                  = True
 |level1_level_requirement                = 3
+|level1_duration                         = 2.5
 |level1_stat_text                        = Deals 7 to 10 Cold Damage<br>Base duration is 2.5 seconds<br>75% reduced Cast Speed
 |level1_stat1_id                         = spell_minimum_base_cold_damage
 |level1_stat1_value                      = 7
@@ -25,6 +26,7 @@
 |level1_stat4_value                      = -75
 |level2                                  = True
 |level2_level_requirement                = 51
+|level2_duration                         = 2
 |level2_stat_text                        = Deals 345 to 517 Cold Damage<br>Base duration is 2 seconds<br>60% reduced Cast Speed
 |level2_stat1_id                         = spell_minimum_base_cold_damage
 |level2_stat1_value                      = 345
@@ -36,6 +38,7 @@
 |level2_stat4_value                      = -60
 |level3                                  = True
 |level3_level_requirement                = 71
+|level3_duration                         = 1.75
 |level3_stat_text                        = Deals 1239 to 1859 Cold Damage<br>Base duration is 1.75 seconds<br>50% reduced Cast Speed
 |level3_stat1_id                         = spell_minimum_base_cold_damage
 |level3_stat1_value                      = 1239
