--- Skill:MartyrExplosion
+++ Export
@@ -7,12 +7,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = suicide_explosion_%_of_default_monster_life_to_deal_as_fire
+|static_stat1_id                         = is_area_damage
 |static_stat1_value                      = 1
-|static_stat2_id                         = cast_on_death_%
-|static_stat2_value                      = 100
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
+|static_stat2_id                         = suicide_explosion_%_of_default_monster_life_to_deal_as_fire
+|static_stat2_value                      = 1
+|static_stat3_id                         = cast_on_death_%
+|static_stat3_value                      = 100
 |stat_text                               = 100% chance to Trigger this Spell on Death
 |level1                                  = True
 |level1_level_requirement                = 1
