--- Skill:MetamorphosisEtherealKnivesSpiral
+++ Export
@@ -7,18 +7,18 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 2
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 2
-|static_stat3_id                         = projectile_spiral_nova_time_ms
-|static_stat3_value                      = 700
-|static_stat4_id                         = base_number_of_projectiles_in_spiral_nova
-|static_stat4_value                      = 13
-|static_stat5_id                         = projectile_spiral_nova_angle
-|static_stat5_value                      = 300
-|static_stat6_id                         = base_is_projectile
-|static_stat6_value                      = 1
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = projectile_spiral_nova_time_ms
+|static_stat2_value                      = 700
+|static_stat3_id                         = spell_minimum_base_physical_damage
+|static_stat3_value                      = 2
+|static_stat4_id                         = spell_maximum_base_physical_damage
+|static_stat4_value                      = 2
+|static_stat5_id                         = base_number_of_projectiles_in_spiral_nova
+|static_stat5_value                      = 13
+|static_stat6_id                         = projectile_spiral_nova_angle
+|static_stat6_value                      = 300
 |stat_text                               = Deals 2 to 2 Physical Damage<br>Fires 13 Projectiles in a spiral
 |level1                                  = True
 |level1_level_requirement                = 1
