--- Skill:GroundSlamMiner
+++ Export
@@ -10,13 +10,13 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_attack_speed_+%_final
-|static_stat1_value                      = -20
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
 |static_stat2_id                         = base_stun_threshold_reduction_+%
 |static_stat2_value                      = 35
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
-|stat_text                               = Deals (50-58)% more Damage<br>20% less Attack Speed<br>35% reduced Enemy Stun Threshold
+|static_stat3_id                         = active_skill_attack_speed_+%_final
+|static_stat3_value                      = -20
+|stat_text                               = Deals (50-58)% more Damage<br>35% reduced Enemy Stun Threshold<br>20% less Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 2
 |level1_stat_text                        = Deals 50% more Damage
