--- Skill:HailrakeGlacialHammer
+++ Export
@@ -8,16 +8,16 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 75
-|static_stat1_id                         = active_skill_freeze_duration_+%_final
-|static_stat1_value                      = 100
-|static_stat2_id                         = active_skill_damage_+%_final
-|static_stat2_value                      = 20
-|static_stat3_id                         = base_chance_to_freeze_%
-|static_stat3_value                      = 30
-|static_stat4_id                         = skill_physical_damage_%_to_convert_to_cold
-|static_stat4_value                      = 50
-|static_stat5_id                         = action_attack_or_cast_time_uses_animation_length
-|static_stat5_value                      = 1
+|static_stat1_id                         = action_attack_or_cast_time_uses_animation_length
+|static_stat1_value                      = 1
+|static_stat2_id                         = active_skill_freeze_duration_+%_final
+|static_stat2_value                      = 100
+|static_stat3_id                         = active_skill_damage_+%_final
+|static_stat3_value                      = 20
+|static_stat4_id                         = base_chance_to_freeze_%
+|static_stat4_value                      = 30
+|static_stat5_id                         = skill_physical_damage_%_to_convert_to_cold
+|static_stat5_value                      = 50
 |stat_text                               = Deals 20% more Damage<br>30% chance to Freeze enemies<br>50% of Physical Damage Converted to Cold Damage
 |level1                                  = True
 |level1_level_requirement                = 1
