--- Skill:BreachCleave
+++ Export
@@ -24,7 +24,7 @@
 |static_stat7_value                      = 50
 |static_stat8_id                         = melee_range_+
 |static_stat8_value                      = 4
-|stat_text                               = When Dual Wielding, Deals 60% Damage from each Weapon combined<br>29% increased Physical Damage<br>50% of Physical Damage Converted to Fire Damage<br>+4 to Melee Strike Range
+|stat_text                               = When Dual Wielding, Deals 60% Damage from each Weapon combined<br>29% increased Physical Damage<br>50% of Physical Damage Converted to Fire Damage<br>+0.4 metres to Melee Strike Range
 |level1                                  = True
 |level1_level_requirement                = 1
 }}