--- Skill:RainOfArrowsOnyxOblivionVaalSummonedSkeleton
+++ Export
@@ -10,12 +10,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_damage_+%_final
-|static_stat1_value                      = -20
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
 |static_stat2_id                         = is_area_damage
 |static_stat2_value                      = 1
-|static_stat3_id                         = base_is_projectile
-|static_stat3_value                      = 1
+|static_stat3_id                         = active_skill_damage_+%_final
+|static_stat3_value                      = -20
 |stat_text                               = Deals 20% less Damage
 |level1                                  = True
 |level1_level_requirement                = 1
