--- Skill:BreachCleaveBoss
+++ Export
@@ -12,13 +12,13 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = 35
-|static_stat2_id                         = skill_physical_damage_%_to_convert_to_fire
-|static_stat2_value                      = 50
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
 |static_stat3_id                         = active_skill_damage_+%_final
 |static_stat3_value                      = 200
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
-|stat_text                               = 50% of Physical Damage Converted to Fire Damage<br>Deals 200% more Damage
+|static_stat4_id                         = skill_physical_damage_%_to_convert_to_fire
+|static_stat4_value                      = 50
+|stat_text                               = Deals 200% more Damage<br>50% of Physical Damage Converted to Fire Damage
 |level1                                  = True
 |level1_level_requirement                = 1
 }}