--- Skill:BeyondDemonSplitArrow
+++ Export
@@ -8,15 +8,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_damage_+%_final
-|static_stat1_value                      = -20
-|static_stat2_id                         = physical_damage_+%
-|static_stat2_value                      = 18
+|static_stat1_id                         = skill_can_fire_arrows
+|static_stat1_value                      = 1
+|static_stat2_id                         = active_skill_damage_+%_final
+|static_stat2_value                      = -20
 |static_stat3_id                         = base_number_of_additional_arrows
 |static_stat3_value                      = 4
-|static_stat4_id                         = skill_can_fire_arrows
-|static_stat4_value                      = 1
-|stat_text                               = Deals 20% less Damage<br>18% increased Physical Damage<br>Fires 4 additional Arrows
+|static_stat4_id                         = physical_damage_+%
+|static_stat4_value                      = 18
+|stat_text                               = Deals 20% less Damage<br>Fires 4 additional Arrows<br>18% increased Physical Damage
 |level1                                  = True
 |level1_level_requirement                = 3
 }}