--- Skill:SanctumHellscapeMeteor
+++ Export
@@ -0,0 +1,33 @@
+{{Skill
+|gem_description                         = Flaming bolts rain down over the targeted area. They explode when landing, dealing damage to nearby enemies.
+|active_skill_name                       = Firestorm
+|skill_id                                = SanctumHellscapeMeteor
+|cast_time                               = 1.0
+|required_level                          = 1
+|static_stored_uses                      = 3
+|static_cooldown                         = 4
+|static_critical_strike_chance           = 5
+|static_damage_effectiveness             = 100
+|static_damage_multiplier                = 100
+|static_attack_speed_multiplier          = 100
+|static_duration                         = 0.25
+|static_stat1_id                         = base_sanctum_damage
+|static_stat1_value                      = 25
+|static_stat2_id                         = firestorm_base_area_of_effect_+%
+|static_stat2_value                      = -50
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = base_skill_effect_duration
+|static_stat4_value                      = 250
+|static_stat5_id                         = spell_minimum_base_fire_damage
+|static_stat5_value                      = 4
+|static_stat6_id                         = spell_maximum_base_fire_damage
+|static_stat6_value                      = 6
+|static_stat7_id                         = active_skill_area_of_effect_+%_final
+|static_stat7_value                      = 100
+|static_stat8_id                         = fire_storm_fireball_delay_ms
+|static_stat8_value                      = 500
+|stat_text                               = Base duration is 0.25 seconds<br>Deals 4 to 6 Fire Damage<br>100% more Area of Effect<br>One impact every 0.5 seconds
+|level1                                  = True
+|level1_level_requirement                = 1
+}}