--- Skill:DarkExileBallLightning
+++ Export
@@ -7,23 +7,23 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage
-|static_stat1_value                      = 315
-|static_stat2_id                         = spell_maximum_base_lightning_damage
-|static_stat2_value                      = 585
-|static_stat3_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat3_value                      = -47
-|static_stat4_id                         = monster_reverse_point_blank_damage_-%_at_minimum_range
-|static_stat4_value                      = 50
-|static_stat5_id                         = number_of_additional_projectiles
-|static_stat5_value                      = 2
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = -47
+|static_stat2_id                         = base_is_projectile
+|static_stat2_value                      = 1
+|static_stat3_id                         = monster_reverse_point_blank_damage_-%_at_minimum_range
+|static_stat3_value                      = 50
+|static_stat4_id                         = spell_minimum_base_lightning_damage
+|static_stat4_value                      = 315
+|static_stat5_id                         = spell_maximum_base_lightning_damage
+|static_stat5_value                      = 585
 |static_stat6_id                         = base_cast_speed_+%
 |static_stat6_value                      = 33
 |static_stat7_id                         = base_projectile_speed_+%
 |static_stat7_value                      = 40
-|static_stat8_id                         = base_is_projectile
-|static_stat8_value                      = 1
-|stat_text                               = Deals 315 to 585 Lightning Damage<br>Fires 2 additional Projectiles<br>33% increased Cast Speed<br>40% increased Projectile Speed
+|static_stat8_id                         = number_of_additional_projectiles
+|static_stat8_value                      = 2
+|stat_text                               = Deals 315 to 585 Lightning Damage<br>33% increased Cast Speed<br>40% increased Projectile Speed<br>Fires 2 additional Projectiles
 |level1                                  = True
 |level1_level_requirement                = 73
 }}