--- Skill:UniqueSupportLifeLeech
+++ Export
@@ -2,6 +2,7 @@
 |skill_id                                = UniqueSupportLifeLeech
 |support_gem_letter                      = l
 |required_level                          = 62
+|static_cost_multiplier                  = 120
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
