--- Skill:HarbingerChillArcUber
+++ Export
@@ -11,11 +11,11 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_chill_duration_+%_final
 |static_stat1_value                      = 100
-|static_stat2_id                         = chill_effect_+%
+|static_stat2_id                         = active_skill_chill_effect_+%_final
 |static_stat2_value                      = 40
 |static_stat3_id                         = number_of_chains
 |static_stat3_value                      = 10
-|stat_text                               = 40% increased Effect of Chill<br>Chains +10 Times
+|stat_text                               = Chains +10 Times
 |level1                                  = True
 |level1_level_requirement                = 1
 }}