--- Skill:ElderDoomProjectile1
+++ Export
@@ -11,9 +11,9 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = base_is_projectile
 |static_stat1_value                      = 1
-|static_stat2_id                         = projectile_uses_contact_position
+|static_stat2_id                         = projectile_uses_contact_direction
 |static_stat2_value                      = 1
-|static_stat3_id                         = projectile_uses_contact_direction
+|static_stat3_id                         = projectile_uses_contact_position
 |static_stat3_value                      = 1
 |stat_text                               = Deals (8-5266) to (13-7899) Cold Damage<br>(0-50)% more Cast Speed
 |level1                                  = True
@@ -26,12 +26,12 @@
 |level2                                  = True
 |level2_level_requirement                = 85
 |level2_stat_text                        = Deals 5266 to 7899 Cold Damage<br>50% more Cast Speed
-|level2_stat1_id                         = spell_minimum_base_cold_damage
-|level2_stat1_value                      = 5266
-|level2_stat2_id                         = spell_maximum_base_cold_damage
-|level2_stat2_value                      = 7899
-|level2_stat3_id                         = active_skill_cast_speed_+%_final
-|level2_stat3_value                      = 50
-|level2_stat4_id                         = active_skill_area_of_effect_radius_+%_final
+|level2_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|level2_stat1_value                      = 50
+|level2_stat2_id                         = spell_minimum_base_cold_damage
+|level2_stat2_value                      = 5266
+|level2_stat3_id                         = spell_maximum_base_cold_damage
+|level2_stat3_value                      = 7899
+|level2_stat4_id                         = active_skill_cast_speed_+%_final
 |level2_stat4_value                      = 50
 }}