--- Skill:JunTriggeredBloodRage
+++ Export
@@ -11,23 +11,24 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 7
 |static_stat1_id                         = skill_level
 |static_stat1_value                      = 1
-|static_stat2_id                         = chance_to_cast_on_kill_%_target_self
-|static_stat2_value                      = 10
-|static_stat3_id                         = base_skill_effect_duration
-|static_stat3_value                      = 7000
-|static_stat4_id                         = add_frenzy_charge_on_kill_%_chance
-|static_stat4_value                      = 25
-|static_stat5_id                         = base_physical_damage_%_of_maximum_energy_shield_to_deal_per_minute
+|static_stat2_id                         = base_skill_effect_duration
+|static_stat2_value                      = 7000
+|static_stat3_id                         = attack_speed_+%_granted_from_skill
+|static_stat3_value                      = 5
+|static_stat4_id                         = life_leech_from_physical_attack_damage_permyriad
+|static_stat4_value                      = 120
+|static_stat5_id                         = base_physical_damage_%_of_maximum_life_to_deal_per_minute
 |static_stat5_value                      = 240
-|static_stat6_id                         = base_physical_damage_%_of_maximum_life_to_deal_per_minute
+|static_stat6_id                         = base_physical_damage_%_of_maximum_energy_shield_to_deal_per_minute
 |static_stat6_value                      = 240
-|static_stat7_id                         = life_leech_from_physical_attack_damage_permyriad
-|static_stat7_value                      = 120
-|static_stat8_id                         = attack_speed_+%_granted_from_skill
-|static_stat8_value                      = 5
-|stat_text                               = 10% chance to Trigger this Spell on Kill<br>Base duration is 7 seconds<br>25% chance to gain a Frenzy Charge on Kill<br>You take 4% of your Maximum Energy Shield per second as Physical Damage<br>You take 4% of your Maximum Life per second as Physical Damage<br>1.2% of Attack Physical Damage Leeched as Life<br>5% increased Attack Speed
+|static_stat7_id                         = add_frenzy_charge_on_kill_%_chance
+|static_stat7_value                      = 25
+|static_stat8_id                         = chance_to_cast_on_kill_%_target_self
+|static_stat8_value                      = 10
+|stat_text                               = Base duration is 7 seconds<br>Grants 5% increased Attack Speed<br>1.2% of Attack Physical Damage Leeched as Life<br>You take 4% of your Maximum Life per second as Physical Damage<br>You take 4% of your Maximum Energy Shield per second as Physical Damage<br>25% chance to gain a Frenzy Charge on Kill<br>10% chance to Trigger this Spell on Kill
 |level1                                  = True
 |level1_level_requirement                = 16
 }}