--- Skill:BetrayalRikerBombShot
+++ Export
@@ -9,18 +9,18 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_attack_speed_+%_final
-|static_stat1_value                      = -25
-|static_stat2_id                         = base_is_projectile
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = is_area_damage
 |static_stat2_value                      = 1
 |static_stat3_id                         = projectile_uses_contact_position
 |static_stat3_value                      = 1
-|static_stat4_id                         = is_area_damage
+|static_stat4_id                         = skill_cannot_be_knocked_back
 |static_stat4_value                      = 1
 |static_stat5_id                         = skill_cannot_be_stunned
 |static_stat5_value                      = 1
-|static_stat6_id                         = skill_cannot_be_knocked_back
-|static_stat6_value                      = 1
+|static_stat6_id                         = active_skill_attack_speed_+%_final
+|static_stat6_value                      = -25
 |stat_text                               = Deals (65-50)% less Damage<br>25% less Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 1
