--- Skill:ShaperQuestSpaceFrostboltRuckus
+++ Export
@@ -10,19 +10,19 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = projectile_spiral_nova_time_ms
-|static_stat1_value                      = 1200
-|static_stat2_id                         = base_number_of_projectiles_in_spiral_nova
-|static_stat2_value                      = 6
-|static_stat3_id                         = projectile_spiral_nova_angle
-|static_stat3_value                      = 100
-|static_stat4_id                         = always_pierce
-|static_stat4_value                      = 1
-|static_stat5_id                         = projectile_spiral_nova_both_directions
-|static_stat5_value                      = 1
-|static_stat6_id                         = base_is_projectile
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = projectile_spiral_nova_time_ms
+|static_stat2_value                      = 1200
+|static_stat3_id                         = always_pierce
+|static_stat3_value                      = 1
+|static_stat4_id                         = base_number_of_projectiles_in_spiral_nova
+|static_stat4_value                      = 6
+|static_stat5_id                         = projectile_spiral_nova_angle
+|static_stat5_value                      = 100
+|static_stat6_id                         = projectile_spiral_nova_both_directions
 |static_stat6_value                      = 1
-|stat_text                               = Deals (2390-3146) to (3584-1) Cold Damage<br>Fires 6 Projectiles in a spiral<br>Projectiles Pierce all Targets
+|stat_text                               = Deals (2390-3146) to (3584-1) Cold Damage<br>Projectiles Pierce all Targets<br>Fires 6 Projectiles in a spiral
 |level1                                  = True
 |level1_level_requirement                = 75
 |level1_stat_text                        = Deals 2390 to 3584 Cold Damage
