--- Skill:FlaskChargeNova
+++ Export
@@ -9,12 +9,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = remove_%_of_targets_current_flask_charges_on_hit
-|static_stat1_value                      = 25
+|static_stat1_id                         = base_deal_no_damage
+|static_stat1_value                      = 1
 |static_stat2_id                         = is_area_damage
 |static_stat2_value                      = 1
-|static_stat3_id                         = base_deal_no_damage
-|static_stat3_value                      = 1
+|static_stat3_id                         = remove_%_of_targets_current_flask_charges_on_hit
+|static_stat3_value                      = 25
 |stat_text                               = 
 |level1                                  = True
 |level1_level_requirement                = 1
