--- Skill:LbEdgeTrap
+++ Export
@@ -7,27 +7,25 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = bleed_damage_+%_vs_player_minions_final
-|static_stat1_value                      = -90
-|static_stat2_id                         = active_skill_bleeding_damage_+%_final
-|static_stat2_value                      = -45
-|static_stat3_id                         = damage_+%_vs_players
-|static_stat3_value                      = 20
-|static_stat4_id                         = %_of_es_to_deal_as_damage
-|static_stat4_value                      = 15
-|static_stat5_id                         = %_of_life_to_deal_as_damage
-|static_stat5_value                      = 25
-|static_stat6_id                         = base_bleed_duration_+%
-|static_stat6_value                      = -40
-|static_stat7_id                         = reduce_enemy_dodge_%
-|static_stat7_value                      = 100
-|static_stat8_id                         = global_reduce_enemy_block_%
-|static_stat8_value                      = 100
-|static_stat9_id                         = global_always_hit
+|static_stat1_id                         = %_of_es_to_deal_as_damage
+|static_stat1_value                      = 15
+|static_stat2_id                         = %_of_life_to_deal_as_damage
+|static_stat2_value                      = 25
+|static_stat3_id                         = active_skill_bleeding_damage_+%_final
+|static_stat3_value                      = -45
+|static_stat4_id                         = bleed_damage_+%_vs_player_minions_final
+|static_stat4_value                      = -90
+|static_stat5_id                         = cannot_be_blocked_or_dodged_or_suppressed
+|static_stat5_value                      = 1
+|static_stat6_id                         = damage_+%_vs_players
+|static_stat6_value                      = 20
+|static_stat7_id                         = base_bleed_duration_+%
+|static_stat7_value                      = -40
+|static_stat8_id                         = global_always_hit
+|static_stat8_value                      = 1
+|static_stat9_id                         = global_bleed_on_hit
 |static_stat9_value                      = 1
-|static_stat10_id                        = global_bleed_on_hit
-|static_stat10_value                     = 1
-|stat_text                               = 40% reduced Bleeding Duration<br>Enemies have 100% reduced chance to Dodge Hits from this Skill<br>Enemy Block Chance reduced by 100% against this Skill<br>Can't be Evaded<br>Causes Bleeding
+|stat_text                               = 40% reduced Bleeding Duration<br>Can't be Evaded<br>Causes Bleeding
 |level1                                  = True
 |level1_level_requirement                = 1
 }}