--- Skill:GABloodChieftainSlamRuin
+++ Export
@@ -11,11 +11,11 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = -35
-|static_stat2_id                         = is_area_damage
+|static_stat2_id                         = apply_ultimatum_ruin_on_hit
 |static_stat2_value                      = 1
-|static_stat3_id                         = apply_ultimatum_ruin_on_hit
+|static_stat3_id                         = cast_time_overrides_attack_duration
 |static_stat3_value                      = 1
-|static_stat4_id                         = cast_time_overrides_attack_duration
+|static_stat4_id                         = is_area_damage
 |static_stat4_value                      = 1
 |stat_text                               = Deals (0-50)% more Damage
 |level1                                  = True
