--- Skill:DisintegrateSeawitchBoss
+++ Export
@@ -24,7 +24,7 @@
 |static_stat7_value                      = 2
 |static_stat8_id                         = disintegrate_damage_+%_final_per_intensity
 |static_stat8_value                      = 40
-|stat_text                               = Deals 4 to 7 Lightning Damage<br>This Spell loses 1 Intensity every 0.25 seconds while moving, or immediately if you teleport<br>This Spell gains Intensity when you Cast it, to a maximum of 3<br>+2 to Beam Width Per Intensity<br>33% less branching angle per Intensity<br>40% more Damage with Hits and Ailments per Intensity
+|stat_text                               = Deals 4 to 7 Lightning Damage<br>This Spell loses 1 Intensity every 0.25 seconds while moving, or immediately if you teleport<br>This Spell gains Intensity when you Cast it, to a maximum of 3<br>+0.2 metres to Beam Width Per Intensity<br>33% less branching angle per Intensity<br>40% more Damage with Hits and Ailments per Intensity
 |level1                                  = True
 |level1_level_requirement                = 1
 }}