--- Skill:KitavaDemonXMortarMetamorph
+++ Export
@@ -11,16 +11,16 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = 100
-|static_stat2_id                         = projectile_spread_radius_per_additional_projectile
-|static_stat2_value                      = 125
-|static_stat3_id                         = projectile_minimum_range
-|static_stat3_value                      = 10
-|static_stat4_id                         = spell_maximum_action_distance_+%
-|static_stat4_value                      = 500
-|static_stat5_id                         = base_is_projectile
-|static_stat5_value                      = 1
-|static_stat6_id                         = is_area_damage
-|static_stat6_value                      = 1
+|static_stat2_id                         = base_is_projectile
+|static_stat2_value                      = 1
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = projectile_minimum_range
+|static_stat4_value                      = 10
+|static_stat5_id                         = projectile_spread_radius_per_additional_projectile
+|static_stat5_value                      = 125
+|static_stat6_id                         = spell_maximum_action_distance_+%
+|static_stat6_value                      = 500
 |stat_text                               = Deals (11-531) to (16-786) Physical Damage
 |level1                                  = True
 |level1_level_requirement                = 1
