--- Skill:BookArc
+++ Export
@@ -8,12 +8,12 @@
 |static_damage_effectiveness             = 50
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage
-|static_stat1_value                      = 3
-|static_stat2_id                         = spell_maximum_base_lightning_damage
-|static_stat2_value                      = 9
-|static_stat3_id                         = spell_maximum_action_distance_+%
-|static_stat3_value                      = -70
+|static_stat1_id                         = spell_maximum_action_distance_+%
+|static_stat1_value                      = -70
+|static_stat2_id                         = spell_minimum_base_lightning_damage
+|static_stat2_value                      = 3
+|static_stat3_id                         = spell_maximum_base_lightning_damage
+|static_stat3_value                      = 9
 |static_stat4_id                         = base_chance_to_shock_%
 |static_stat4_value                      = 10
 |stat_text                               = Deals 3 to 9 Lightning Damage<br>10% chance to Shock enemies
