--- Skill:MaligaroEtherealKnives
+++ Export
@@ -8,9 +8,9 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_projectile_variation
+|static_stat1_id                         = base_is_projectile
 |static_stat1_value                      = 1
-|static_stat2_id                         = base_is_projectile
+|static_stat2_id                         = monster_projectile_variation
 |static_stat2_value                      = 1
 |stat_text                               = Deals (75-413) to (118-650) Physical Damage<br>Fires (6-9) additional Projectiles
 |level1                                  = True
