--- Skill:BeyondDemonTornadoShot
+++ Export
@@ -8,12 +8,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = tornado_shot_num_of_secondary_projectiles
-|static_stat1_value                      = 3
-|static_stat2_id                         = base_is_projectile
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = skill_can_fire_arrows
 |static_stat2_value                      = 1
-|static_stat3_id                         = skill_can_fire_arrows
-|static_stat3_value                      = 1
+|static_stat3_id                         = tornado_shot_num_of_secondary_projectiles
+|static_stat3_value                      = 3
 |stat_text                               = Deals (-10-30)% less Damage<br>Fires (0-2) additional Projectiles<br>Fires 3 secondary Projectiles
 |level1                                  = True
 |level1_level_requirement                = 1
