--- Skill:GAVoranaComboLeftSlash
+++ Export
@@ -7,11 +7,11 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 60
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = global_knockback
+|static_stat1_id                         = determine_knockback_direction_from_melee_pattern
 |static_stat1_value                      = 1
 |static_stat2_id                         = is_area_damage
 |static_stat2_value                      = 1
-|static_stat3_id                         = determine_knockback_direction_from_melee_pattern
+|static_stat3_id                         = global_knockback
 |static_stat3_value                      = 1
 |stat_text                               = Knocks Enemies Back on Hit
 |level1                                  = True
