--- Skill:MonsterDetonateDeadAreaAliraMaps
+++ Export
@@ -10,21 +10,22 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_fire_damage
-|static_stat1_value                      = 8
-|static_stat2_id                         = spell_maximum_base_fire_damage
-|static_stat2_value                      = 12
-|static_stat3_id                         = upheaval_number_of_spikes
-|static_stat3_value                      = 6
-|static_stat4_id                         = base_secondary_skill_effect_duration
-|static_stat4_value                      = 500
-|static_stat5_id                         = base_skill_effect_duration
-|static_stat5_value                      = 1500
+|static_duration                         = 1.5
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_fire_damage
+|static_stat2_value                      = 8
+|static_stat3_id                         = spell_maximum_base_fire_damage
+|static_stat3_value                      = 12
+|static_stat4_id                         = base_skill_effect_duration
+|static_stat4_value                      = 1500
+|static_stat5_id                         = base_secondary_skill_effect_duration
+|static_stat5_value                      = 500
 |static_stat6_id                         = corpse_explosion_monster_life_%
 |static_stat6_value                      = 1
-|static_stat7_id                         = is_area_damage
-|static_stat7_value                      = 1
-|stat_text                               = Deals 8 to 12 Fire Damage<br>Causes 6 Bursts<br>Base secondary duration is 0.5 seconds<br>Base duration is 1.5 seconds<br>Explosion deals base Fire Damage equal to 1% of the corpse's Maximum Life
+|static_stat7_id                         = upheaval_number_of_spikes
+|static_stat7_value                      = 6
+|stat_text                               = Deals 8 to 12 Fire Damage<br>Base duration is 1.5 seconds<br>Base secondary duration is 0.5 seconds<br>Explosion deals base Fire Damage equal to 1% of the corpse's Maximum Life<br>Causes 6 Bursts
 |level1                                  = True
 |level1_level_requirement                = 1
 }}