--- Skill:ProphecyFrostWall
+++ Export
@@ -11,10 +11,11 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = wall_expand_delay_ms
 |static_stat1_value                      = 150
-|stat_text                               = Deals (2-35) to (2-52) Cold Damage<br>Base duration is (1.6-3.45) seconds<br>Wall will be (20-54) units long<br>0.15 seconds between appearance of Wall sections
+|stat_text                               = Deals (2-35) to (2-52) Cold Damage<br>Base duration is (1.6-3.45) seconds<br>0.15 seconds between appearance of Wall sections<br>Wall will be (20-54) units long
 |level1                                  = True
 |level1_level_requirement                = 4
 |level1_cooldown                         = 8
+|level1_duration                         = 1.6
 |level1_stat_text                        = Deals 2 to 2 Cold Damage<br>Base duration is 1.6 seconds<br>Wall will be 20 units long
 |level1_stat1_id                         = spell_minimum_base_cold_damage
 |level1_stat1_value                      = 2
@@ -27,6 +28,7 @@
 |level2                                  = True
 |level2_level_requirement                = 8
 |level2_cooldown                         = 8
+|level2_duration                         = 1.65
 |level2_stat_text                        = Deals 2 to 3 Cold Damage<br>Base duration is 1.65 seconds<br>Wall will be 20 units long
 |level2_stat1_id                         = spell_minimum_base_cold_damage
 |level2_stat1_value                      = 2
@@ -39,6 +41,7 @@
 |level3                                  = True
 |level3_level_requirement                = 11
 |level3_cooldown                         = 8
+|level3_duration                         = 1.7
 |level3_stat_text                        = Deals 3 to 4 Cold Damage<br>Base duration is 1.7 seconds<br>Wall will be 22 units long
 |level3_stat1_id                         = spell_minimum_base_cold_damage
 |level3_stat1_value                      = 3
@@ -51,6 +54,7 @@
 |level4                                  = True
 |level4_level_requirement                = 15
 |level4_cooldown                         = 8
+|level4_duration                         = 1.75
 |level4_stat_text                        = Deals 3 to 5 Cold Damage<br>Base duration is 1.75 seconds<br>Wall will be 22 units long
 |level4_stat1_id                         = spell_minimum_base_cold_damage
 |level4_stat1_value                      = 3
@@ -63,6 +67,7 @@
 |level5                                  = True
 |level5_level_requirement                = 20
 |level5_cooldown                         = 8
+|level5_duration                         = 1.8
 |level5_stat_text                        = Deals 4 to 6 Cold Damage<br>Base duration is 1.8 seconds<br>Wall will be 24 units long
 |level5_stat1_id                         = spell_minimum_base_cold_damage
 |level5_stat1_value                      = 4
@@ -75,6 +80,7 @@
 |level6                                  = True
 |level6_level_requirement                = 25
 |level6_cooldown                         = 8
+|level6_duration                         = 1.85
 |level6_stat_text                        = Deals 5 to 8 Cold Damage<br>Base duration is 1.85 seconds<br>Wall will be 24 units long
 |level6_stat1_id                         = spell_minimum_base_cold_damage
 |level6_stat1_value                      = 5
@@ -87,6 +93,7 @@
 |level7                                  = True
 |level7_level_requirement                = 29
 |level7_cooldown                         = 8
+|level7_duration                         = 1.9
 |level7_stat_text                        = Deals 6 to 10 Cold Damage<br>Base duration is 1.9 seconds<br>Wall will be 26 units long
 |level7_stat1_id                         = spell_minimum_base_cold_damage
 |level7_stat1_value                      = 6
@@ -99,6 +106,7 @@
 |level8                                  = True
 |level8_level_requirement                = 31
 |level8_cooldown                         = 8
+|level8_duration                         = 1.95
 |level8_stat_text                        = Deals 7 to 10 Cold Damage<br>Base duration is 1.95 seconds<br>Wall will be 26 units long
 |level8_stat1_id                         = spell_minimum_base_cold_damage
 |level8_stat1_value                      = 7
@@ -111,6 +119,7 @@
 |level9                                  = True
 |level9_level_requirement                = 32
 |level9_cooldown                         = 8
+|level9_duration                         = 2
 |level9_stat_text                        = Deals 7 to 11 Cold Damage<br>Base duration is 2 seconds<br>Wall will be 28 units long
 |level9_stat1_id                         = spell_minimum_base_cold_damage
 |level9_stat1_value                      = 7
@@ -123,6 +132,7 @@
 |level10                                 = True
 |level10_level_requirement               = 34
 |level10_cooldown                        = 8
+|level10_duration                        = 2.05
 |level10_stat_text                       = Deals 8 to 12 Cold Damage<br>Base duration is 2.05 seconds<br>Wall will be 28 units long
 |level10_stat1_id                        = spell_minimum_base_cold_damage
 |level10_stat1_value                     = 8
@@ -135,6 +145,7 @@
 |level11                                 = True
 |level11_level_requirement               = 37
 |level11_cooldown                        = 7
+|level11_duration                        = 2.1
 |level11_stat_text                       = Deals 9 to 13 Cold Damage<br>Base duration is 2.1 seconds<br>Wall will be 30 units long
 |level11_stat1_id                        = spell_minimum_base_cold_damage
 |level11_stat1_value                     = 9
@@ -147,6 +158,7 @@
 |level12                                 = True
 |level12_level_requirement               = 40
 |level12_cooldown                        = 7
+|level12_duration                        = 2.15
 |level12_stat_text                       = Deals 10 to 15 Cold Damage<br>Base duration is 2.15 seconds<br>Wall will be 30 units long
 |level12_stat1_id                        = spell_minimum_base_cold_damage
 |level12_stat1_value                     = 10
@@ -159,6 +171,7 @@
 |level13                                 = True
 |level13_level_requirement               = 42
 |level13_cooldown                        = 7
+|level13_duration                        = 2.2
 |level13_stat_text                       = Deals 10 to 16 Cold Damage<br>Base duration is 2.2 seconds<br>Wall will be 32 units long
 |level13_stat1_id                        = spell_minimum_base_cold_damage
 |level13_stat1_value                     = 10
@@ -171,6 +184,7 @@
 |level14                                 = True
 |level14_level_requirement               = 45
 |level14_cooldown                        = 7
+|level14_duration                        = 2.25
 |level14_stat_text                       = Deals 12 to 17 Cold Damage<br>Base duration is 2.25 seconds<br>Wall will be 32 units long
 |level14_stat1_id                        = spell_minimum_base_cold_damage
 |level14_stat1_value                     = 12
@@ -183,6 +197,7 @@
 |level15                                 = True
 |level15_level_requirement               = 50
 |level15_cooldown                        = 7
+|level15_duration                        = 2.3
 |level15_stat_text                       = Deals 14 to 20 Cold Damage<br>Base duration is 2.3 seconds<br>Wall will be 34 units long
 |level15_stat1_id                        = spell_minimum_base_cold_damage
 |level15_stat1_value                     = 14
@@ -195,6 +210,7 @@
 |level16                                 = True
 |level16_level_requirement               = 51
 |level16_cooldown                        = 7
+|level16_duration                        = 2.35
 |level16_stat_text                       = Deals 14 to 21 Cold Damage<br>Base duration is 2.35 seconds<br>Wall will be 34 units long
 |level16_stat1_id                        = spell_minimum_base_cold_damage
 |level16_stat1_value                     = 14
@@ -207,6 +223,7 @@
 |level17                                 = True
 |level17_level_requirement               = 52
 |level17_cooldown                        = 7
+|level17_duration                        = 2.4
 |level17_stat_text                       = Deals 15 to 22 Cold Damage<br>Base duration is 2.4 seconds<br>Wall will be 36 units long
 |level17_stat1_id                        = spell_minimum_base_cold_damage
 |level17_stat1_value                     = 15
@@ -219,6 +236,7 @@
 |level18                                 = True
 |level18_level_requirement               = 54
 |level18_cooldown                        = 7
+|level18_duration                        = 2.45
 |level18_stat_text                       = Deals 16 to 23 Cold Damage<br>Base duration is 2.45 seconds<br>Wall will be 36 units long
 |level18_stat1_id                        = spell_minimum_base_cold_damage
 |level18_stat1_value                     = 16
@@ -231,6 +249,7 @@
 |level19                                 = True
 |level19_level_requirement               = 58
 |level19_cooldown                        = 7
+|level19_duration                        = 2.5
 |level19_stat_text                       = Deals 18 to 26 Cold Damage<br>Base duration is 2.5 seconds<br>Wall will be 38 units long
 |level19_stat1_id                        = spell_minimum_base_cold_damage
 |level19_stat1_value                     = 18
@@ -243,6 +262,7 @@
 |level20                                 = True
 |level20_level_requirement               = 66
 |level20_cooldown                        = 7
+|level20_duration                        = 2.55
 |level20_stat_text                       = Deals 22 to 33 Cold Damage<br>Base duration is 2.55 seconds<br>Wall will be 38 units long
 |level20_stat1_id                        = spell_minimum_base_cold_damage
 |level20_stat1_value                     = 22
@@ -255,6 +275,7 @@
 |level21                                 = True
 |level21_level_requirement               = 67
 |level21_cooldown                        = 6
+|level21_duration                        = 2.6
 |level21_stat_text                       = Deals 23 to 34 Cold Damage<br>Base duration is 2.6 seconds<br>Wall will be 40 units long
 |level21_stat1_id                        = spell_minimum_base_cold_damage
 |level21_stat1_value                     = 23
@@ -267,6 +288,7 @@
 |level22                                 = True
 |level22_level_requirement               = 68
 |level22_cooldown                        = 6
+|level22_duration                        = 2.65
 |level22_stat_text                       = Deals 24 to 35 Cold Damage<br>Base duration is 2.65 seconds<br>Wall will be 40 units long
 |level22_stat1_id                        = spell_minimum_base_cold_damage
 |level22_stat1_value                     = 24
@@ -279,6 +301,7 @@
 |level23                                 = True
 |level23_level_requirement               = 69
 |level23_cooldown                        = 6
+|level23_duration                        = 2.7
 |level23_stat_text                       = Deals 24 to 36 Cold Damage<br>Base duration is 2.7 seconds<br>Wall will be 42 units long
 |level23_stat1_id                        = spell_minimum_base_cold_damage
 |level23_stat1_value                     = 24
@@ -291,6 +314,7 @@
 |level24                                 = True
 |level24_level_requirement               = 70
 |level24_cooldown                        = 6
+|level24_duration                        = 2.75
 |level24_stat_text                       = Deals 25 to 37 Cold Damage<br>Base duration is 2.75 seconds<br>Wall will be 42 units long
 |level24_stat1_id                        = spell_minimum_base_cold_damage
 |level24_stat1_value                     = 25
@@ -303,6 +327,7 @@
 |level25                                 = True
 |level25_level_requirement               = 71
 |level25_cooldown                        = 6
+|level25_duration                        = 2.8
 |level25_stat_text                       = Deals 26 to 39 Cold Damage<br>Base duration is 2.8 seconds<br>Wall will be 44 units long
 |level25_stat1_id                        = spell_minimum_base_cold_damage
 |level25_stat1_value                     = 26
@@ -315,6 +340,7 @@
 |level26                                 = True
 |level26_level_requirement               = 72
 |level26_cooldown                        = 6
+|level26_duration                        = 2.85
 |level26_stat_text                       = Deals 26 to 40 Cold Damage<br>Base duration is 2.85 seconds<br>Wall will be 44 units long
 |level26_stat1_id                        = spell_minimum_base_cold_damage
 |level26_stat1_value                     = 26
@@ -327,6 +353,7 @@
 |level27                                 = True
 |level27_level_requirement               = 73
 |level27_cooldown                        = 6
+|level27_duration                        = 2.9
 |level27_stat_text                       = Deals 27 to 41 Cold Damage<br>Base duration is 2.9 seconds<br>Wall will be 46 units long
 |level27_stat1_id                        = spell_minimum_base_cold_damage
 |level27_stat1_value                     = 27
@@ -339,6 +366,7 @@
 |level28                                 = True
 |level28_level_requirement               = 74
 |level28_cooldown                        = 6
+|level28_duration                        = 2.95
 |level28_stat_text                       = Deals 28 to 42 Cold Damage<br>Base duration is 2.95 seconds<br>Wall will be 46 units long
 |level28_stat1_id                        = spell_minimum_base_cold_damage
 |level28_stat1_value                     = 28
@@ -351,6 +379,7 @@
 |level29                                 = True
 |level29_level_requirement               = 75
 |level29_cooldown                        = 6
+|level29_duration                        = 3
 |level29_stat_text                       = Deals 29 to 43 Cold Damage<br>Base duration is 3 seconds<br>Wall will be 48 units long
 |level29_stat1_id                        = spell_minimum_base_cold_damage
 |level29_stat1_value                     = 29
@@ -363,6 +392,7 @@
 |level30                                 = True
 |level30_level_requirement               = 76
 |level30_cooldown                        = 6
+|level30_duration                        = 3.05
 |level30_stat_text                       = Deals 30 to 44 Cold Damage<br>Base duration is 3.05 seconds<br>Wall will be 48 units long
 |level30_stat1_id                        = spell_minimum_base_cold_damage
 |level30_stat1_value                     = 30
@@ -375,6 +405,7 @@
 |level31                                 = True
 |level31_level_requirement               = 77
 |level31_cooldown                        = 5
+|level31_duration                        = 3.1
 |level31_stat_text                       = Deals 30 to 46 Cold Damage<br>Base duration is 3.1 seconds<br>Wall will be 50 units long
 |level31_stat1_id                        = spell_minimum_base_cold_damage
 |level31_stat1_value                     = 30
@@ -387,6 +418,7 @@
 |level32                                 = True
 |level32_level_requirement               = 78
 |level32_cooldown                        = 5
+|level32_duration                        = 3.15
 |level32_stat_text                       = Deals 31 to 47 Cold Damage<br>Base duration is 3.15 seconds<br>Wall will be 50 units long
 |level32_stat1_id                        = spell_minimum_base_cold_damage
 |level32_stat1_value                     = 31
@@ -399,6 +431,7 @@
 |level33                                 = True
 |level33_level_requirement               = 79
 |level33_cooldown                        = 5
+|level33_duration                        = 3.2
 |level33_stat_text                       = Deals 32 to 48 Cold Damage<br>Base duration is 3.2 seconds<br>Wall will be 52 units long
 |level33_stat1_id                        = spell_minimum_base_cold_damage
 |level33_stat1_value                     = 32
@@ -411,6 +444,7 @@
 |level34                                 = True
 |level34_level_requirement               = 80
 |level34_cooldown                        = 5
+|level34_duration                        = 3.25
 |level34_stat_text                       = Deals 33 to 49 Cold Damage<br>Base duration is 3.25 seconds<br>Wall will be 52 units long
 |level34_stat1_id                        = spell_minimum_base_cold_damage
 |level34_stat1_value                     = 33
@@ -423,6 +457,7 @@
 |level35                                 = True
 |level35_level_requirement               = 81
 |level35_cooldown                        = 5
+|level35_duration                        = 3.3
 |level35_stat_text                       = Deals 34 to 51 Cold Damage<br>Base duration is 3.3 seconds<br>Wall will be 54 units long
 |level35_stat1_id                        = spell_minimum_base_cold_damage
 |level35_stat1_value                     = 34
@@ -435,6 +470,7 @@
 |level36                                 = True
 |level36_level_requirement               = 82
 |level36_cooldown                        = 5
+|level36_duration                        = 3.45
 |level36_stat_text                       = Deals 35 to 52 Cold Damage<br>Base duration is 3.45 seconds<br>Wall will be 54 units long
 |level36_stat1_id                        = spell_minimum_base_cold_damage
 |level36_stat1_value                     = 35
