--- Skill:EssenceSummonPanickedMossMonster
+++ Export
@@ -7,13 +7,13 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = number_of_monsters_to_summon
-|static_stat1_value                      = 1
-|static_stat2_id                         = alternate_minion
-|static_stat2_value                      = 164
-|static_stat3_id                         = cast_on_death_%
+|static_stat1_id                         = alternate_minion
+|static_stat1_value                      = 164
+|static_stat2_id                         = number_of_monsters_to_summon
+|static_stat2_value                      = 1
+|static_stat3_id                         = summoned_monsters_no_drops_or_experience
 |static_stat3_value                      = 1
-|static_stat4_id                         = summoned_monsters_no_drops_or_experience
+|static_stat4_id                         = cast_on_death_%
 |static_stat4_value                      = 1
 |stat_text                               = 1% chance to Trigger this Spell on Death
 |level1                                  = True
