--- Skill:WbLightningEliteProjectileNova
+++ Export
@@ -10,21 +10,21 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage
-|static_stat1_value                      = 4
-|static_stat2_id                         = spell_maximum_base_lightning_damage
-|static_stat2_value                      = 12
-|static_stat3_id                         = monster_projectile_variation
-|static_stat3_value                      = 2
-|static_stat4_id                         = spell_maximum_action_distance_+%
-|static_stat4_value                      = -60
-|static_stat5_id                         = number_of_additional_projectiles
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_projectile_variation
+|static_stat2_value                      = 2
+|static_stat3_id                         = spell_maximum_action_distance_+%
+|static_stat3_value                      = -60
+|static_stat4_id                         = spell_minimum_base_lightning_damage
+|static_stat4_value                      = 4
+|static_stat5_id                         = spell_maximum_base_lightning_damage
 |static_stat5_value                      = 12
 |static_stat6_id                         = projectiles_nova
 |static_stat6_value                      = 1
-|static_stat7_id                         = base_is_projectile
-|static_stat7_value                      = 1
-|stat_text                               = Deals 4 to 12 Lightning Damage<br>Fires 12 additional Projectiles<br>Fires Projectiles in a circle
+|static_stat7_id                         = number_of_additional_projectiles
+|static_stat7_value                      = 12
+|stat_text                               = Deals 4 to 12 Lightning Damage<br>Fires Projectiles in a circle<br>Fires 12 additional Projectiles
 |level1                                  = True
 |level1_level_requirement                = 3
 }}