--- Skill:AliraVolatileDeadBlessing
+++ Export
@@ -8,19 +8,19 @@
 |static_damage_effectiveness             = 90
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_fire_damage
-|static_stat1_value                      = 207
-|static_stat2_id                         = spell_maximum_base_fire_damage
-|static_stat2_value                      = 310
-|static_stat3_id                         = volatile_dead_max_cores_allowed
-|static_stat3_value                      = 60
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_fire_damage
+|static_stat2_value                      = 207
+|static_stat3_id                         = spell_maximum_base_fire_damage
+|static_stat3_value                      = 310
 |static_stat4_id                         = corpse_explosion_monster_life_%
 |static_stat4_value                      = 1
 |static_stat5_id                         = volatile_dead_base_number_of_corpses_to_consume
 |static_stat5_value                      = 5
-|static_stat6_id                         = is_area_damage
-|static_stat6_value                      = 1
-|stat_text                               = Deals 207 to 310 Fire Damage<br>Maximum of 60 Orbs at a time<br>Explosion deals base Fire Damage equal to 1% of the corpse's Maximum Life<br>Consumes up to 5 corpses
+|static_stat6_id                         = volatile_dead_max_cores_allowed
+|static_stat6_value                      = 60
+|stat_text                               = Deals 207 to 310 Fire Damage<br>Explosion deals base Fire Damage equal to 1% of the corpse's Maximum Life<br>Consumes up to 5 corpses<br>Maximum of 60 Orbs at a time
 |level1                                  = True
 |level1_level_requirement                = 50
 }}