--- Skill:GSThauamturgistBasicProjectileImpact
+++ Export
@@ -8,14 +8,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_chaos_damage
-|static_stat1_value                      = 2
-|static_stat2_id                         = spell_maximum_base_chaos_damage
-|static_stat2_value                      = 3
-|static_stat3_id                         = base_chance_to_poison_on_hit_%
-|static_stat3_value                      = 50
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_chaos_damage
+|static_stat2_value                      = 2
+|static_stat3_id                         = spell_maximum_base_chaos_damage
+|static_stat3_value                      = 3
+|static_stat4_id                         = base_chance_to_poison_on_hit_%
+|static_stat4_value                      = 50
 |stat_text                               = Deals 2 to 3 Chaos Damage<br>50% chance to Poison on Hit
 |level1                                  = True
 |level1_level_requirement                = 1
