--- Skill:CGEMavenOrbChilledGround
+++ Export
@@ -7,13 +7,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_cold_damage_to_deal_per_minute
-|static_stat1_value                      = 1554
-|static_stat2_id                         = ground_maelstrom_art_variation
-|static_stat2_value                      = 8
-|static_stat3_id                         = base_skill_effect_duration
-|static_stat3_value                      = 7500
-|stat_text                               = Deals 25.9 Base Cold Damage per second<br>Base duration is 7.5 seconds
+|static_duration                         = 7.5
+|static_stat1_id                         = ground_maelstrom_art_variation
+|static_stat1_value                      = 8
+|static_stat2_id                         = base_skill_effect_duration
+|static_stat2_value                      = 7500
+|static_stat3_id                         = base_cold_damage_to_deal_per_minute
+|static_stat3_value                      = 1554
+|stat_text                               = Base duration is 7.5 seconds<br>Deals 25.9 Base Cold Damage per second
 |level1                                  = True
 |level1_level_requirement                = 1
 }}