--- Skill:BreachCleave
+++ Export
@@ -8,23 +8,23 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat1_value                      = -20
-|static_stat2_id                         = melee_range_+
-|static_stat2_value                      = 4
-|static_stat3_id                         = skill_physical_damage_%_to_convert_to_fire
-|static_stat3_value                      = 50
-|static_stat4_id                         = physical_damage_+%
-|static_stat4_value                      = 29
-|static_stat5_id                         = cleave_damage_+%_final_while_dual_wielding
+|static_stat1_id                         = action_attack_or_cast_time_uses_animation_length
+|static_stat1_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat3_value                      = -20
+|static_stat4_id                         = skill_double_hits_when_dual_wielding
+|static_stat4_value                      = 1
+|static_stat5_id                         = active_skill_merged_damage_+%_final_while_dual_wielding
 |static_stat5_value                      = -40
-|static_stat6_id                         = is_area_damage
-|static_stat6_value                      = 1
-|static_stat7_id                         = skill_double_hits_when_dual_wielding
-|static_stat7_value                      = 1
-|static_stat8_id                         = action_attack_or_cast_time_uses_animation_length
-|static_stat8_value                      = 1
-|stat_text                               = +4 to Melee Strike Range<br>50% of Physical Damage Converted to Fire Damage<br>29% increased Physical Damage<br>When Dual Wielding, Deals 60% Damage from each Weapon combined
+|static_stat6_id                         = physical_damage_+%
+|static_stat6_value                      = 29
+|static_stat7_id                         = skill_physical_damage_%_to_convert_to_fire
+|static_stat7_value                      = 50
+|static_stat8_id                         = melee_range_+
+|static_stat8_value                      = 4
+|stat_text                               = When Dual Wielding, Deals 60% Damage from each Weapon combined<br>29% increased Physical Damage<br>50% of Physical Damage Converted to Fire Damage<br>+4 to Melee Strike Range
 |level1                                  = True
 |level1_level_requirement                = 1
 }}