--- Skill:ChestSummonTormentContinous
+++ Export
@@ -7,23 +7,23 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = summon_specific_monsters_radius_+%
-|static_stat1_value                      = -95
-|static_stat2_id                         = alternate_minion
-|static_stat2_value                      = 133
-|static_stat3_id                         = number_of_monsters_to_summon
+|static_stat1_id                         = alternate_minion
+|static_stat1_value                      = 133
+|static_stat2_id                         = number_of_monsters_to_summon
+|static_stat2_value                      = 1
+|static_stat3_id                         = summon_specific_monsters_cannot_fail
 |static_stat3_value                      = 1
-|static_stat4_id                         = active_skill_cast_speed_+%_final
-|static_stat4_value                      = -67
-|static_stat5_id                         = skill_repeat_count
-|static_stat5_value                      = 4
+|static_stat4_id                         = summon_specific_monsters_radius_+%
+|static_stat4_value                      = -95
+|static_stat5_id                         = summoned_monster_types_arent_duplicated
+|static_stat5_value                      = 1
 |static_stat6_id                         = summoned_monsters_are_minions
 |static_stat6_value                      = 1
-|static_stat7_id                         = summoned_monster_types_arent_duplicated
-|static_stat7_value                      = 1
-|static_stat8_id                         = summon_specific_monsters_cannot_fail
-|static_stat8_value                      = 1
-|stat_text                               = 67% less Cast Speed<br>Skills Repeat an additional 4 Times
+|static_stat7_id                         = skill_repeat_count
+|static_stat7_value                      = 4
+|static_stat8_id                         = active_skill_cast_speed_+%_final
+|static_stat8_value                      = -67
+|stat_text                               = Skills Repeat an additional 4 Times<br>67% less Cast Speed
 |level1                                  = True
 |level1_level_requirement                = 1
 }}