--- Skill:ProphecySummonRagingSpirit
+++ Export
@@ -7,19 +7,20 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 5
 |static_stat1_id                         = alternate_minion
 |static_stat1_value                      = 1
-|static_stat2_id                         = active_skill_cast_speed_+%_final
-|static_stat2_value                      = 100
-|static_stat3_id                         = skill_repeat_count
-|static_stat3_value                      = 1
-|static_stat4_id                         = base_cast_speed_+%
-|static_stat4_value                      = -10
-|static_stat5_id                         = base_number_of_raging_spirits_allowed
-|static_stat5_value                      = 50
-|static_stat6_id                         = base_skill_effect_duration
-|static_stat6_value                      = 5000
-|stat_text                               = Deals (25-42)% less Damage<br>100% more Cast Speed<br>Skills Repeat an additional Time<br>10% reduced Cast Speed<br>Maximum 50 Summoned Raging Spirits<br>Base duration is 5 seconds
+|static_stat2_id                         = skill_repeat_count
+|static_stat2_value                      = 1
+|static_stat3_id                         = base_number_of_raging_spirits_allowed
+|static_stat3_value                      = 50
+|static_stat4_id                         = base_skill_effect_duration
+|static_stat4_value                      = 5000
+|static_stat5_id                         = active_skill_cast_speed_+%_final
+|static_stat5_value                      = 100
+|static_stat6_id                         = base_cast_speed_+%
+|static_stat6_value                      = -10
+|stat_text                               = Skills Repeat an additional Time<br>Deals (25-42)% less Damage<br>Maximum 50 Summoned Raging Spirits<br>Base duration is 5 seconds<br>100% more Cast Speed<br>10% reduced Cast Speed
 |level1                                  = True
 |level1_level_requirement                = 3
 |level1_stat_text                        = Deals 25% less Damage
