--- Skill:CGEDoomBossBurrowGooErupt
+++ Export
@@ -11,10 +11,10 @@
 |static_stat1_value                      = -40
 |static_stat2_id                         = ground_tar_art_variation
 |static_stat2_value                      = 5
-|static_stat3_id                         = base_movement_velocity_+%
-|static_stat3_value                      = -70
-|static_stat4_id                         = infinite_skill_effect_duration
-|static_stat4_value                      = 1
+|static_stat3_id                         = infinite_skill_effect_duration
+|static_stat3_value                      = 1
+|static_stat4_id                         = base_movement_velocity_+%
+|static_stat4_value                      = -70
 |stat_text                               = 70% reduced Movement Speed
 |level1                                  = True
 |level1_level_requirement                = 1
