--- Skill:ExileWhirlingBlades
+++ Export
@@ -10,14 +10,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_damage_+%_final
-|static_stat1_value                      = -30
-|static_stat2_id                         = physical_damage_+%
-|static_stat2_value                      = 6
-|static_stat3_id                         = ignores_proximity_shield
-|static_stat3_value                      = 1
-|static_stat4_id                         = base_skill_show_average_damage_instead_of_dps
-|static_stat4_value                      = 1
+|static_stat1_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat1_value                      = 1
+|static_stat2_id                         = ignores_proximity_shield
+|static_stat2_value                      = 1
+|static_stat3_id                         = active_skill_damage_+%_final
+|static_stat3_value                      = -30
+|static_stat4_id                         = physical_damage_+%
+|static_stat4_value                      = 6
 |stat_text                               = Deals 30% less Damage<br>6% increased Physical Damage
 |level1                                  = True
 |level1_level_requirement                = 0
