--- Skill:HarvestScorpionBurrow
+++ Export
@@ -11,14 +11,14 @@
 |static_attack_speed_multiplier          = 160
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = -45
-|static_stat2_id                         = shield_charge_extra_distance
-|static_stat2_value                      = 30
-|static_stat3_id                         = emerge_speed_+%
-|static_stat3_value                      = 200
-|static_stat4_id                         = active_skill_damage_+%_final
-|static_stat4_value                      = -50
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
+|static_stat2_id                         = emerge_speed_+%
+|static_stat2_value                      = 200
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = shield_charge_extra_distance
+|static_stat4_value                      = 30
+|static_stat5_id                         = active_skill_damage_+%_final
+|static_stat5_value                      = -50
 |stat_text                               = Deals 50% less Damage
 |level1                                  = True
 |level1_level_requirement                = 1
