--- Skill:MachinariumFireBarrageFar
+++ Export
@@ -8,27 +8,28 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_fire_damage
-|static_stat1_value                      = 2714
-|static_stat2_id                         = spell_maximum_base_fire_damage
-|static_stat2_value                      = 4071
+|static_duration                         = 8
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
 |static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
 |static_stat3_value                      = -25
 |static_stat4_id                         = mortar_projectile_distance_override
 |static_stat4_value                      = 40
-|static_stat5_id                         = base_skill_effect_duration
-|static_stat5_value                      = 8000
-|static_stat6_id                         = is_area_damage
+|static_stat5_id                         = projectile_uses_contact_direction
+|static_stat5_value                      = 1
+|static_stat6_id                         = projectile_uses_contact_position
 |static_stat6_value                      = 1
-|static_stat7_id                         = base_is_projectile
+|static_stat7_id                         = triggered_skill_use_animation_event
 |static_stat7_value                      = 1
-|static_stat8_id                         = projectile_uses_contact_direction
-|static_stat8_value                      = 1
-|static_stat9_id                         = projectile_uses_contact_position
-|static_stat9_value                      = 1
-|static_stat10_id                        = triggered_skill_use_animation_event
-|static_stat10_value                     = 1
-|stat_text                               = Deals 2714 to 4071 Fire Damage<br>Base duration is 8 seconds
+|static_stat8_id                         = base_skill_effect_duration
+|static_stat8_value                      = 8000
+|static_stat9_id                         = spell_minimum_base_fire_damage
+|static_stat9_value                      = 2714
+|static_stat10_id                        = spell_maximum_base_fire_damage
+|static_stat10_value                     = 4071
+|stat_text                               = Base duration is 8 seconds<br>Deals 2714 to 4071 Fire Damage
 |level1                                  = True
 |level1_level_requirement                = 80
 }}