--- Skill:ProphecyRighteousFire
+++ Export
@@ -7,10 +7,10 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_fire_damage_to_deal_per_minute
-|static_stat1_value                      = 144
-|static_stat2_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat2_value                      = -27
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = -27
+|static_stat2_id                         = base_fire_damage_to_deal_per_minute
+|static_stat2_value                      = 144
 |stat_text                               = Deals 2.4 Base Fire Damage per second
 |level1                                  = True
 |level1_level_requirement                = 1
