--- Skill:SpiderBarrelCoverInSpiders
+++ Export
@@ -7,6 +7,7 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 4
 |static_stat1_id                         = covered_in_spiders_attack_and_cast_speed_+%_final
 |static_stat1_value                      = -20
 |static_stat2_id                         = covered_in_spiders_damage_+%_final
