--- Skill:LegionTemplarGeneralSwipeRight
+++ Export
@@ -7,13 +7,13 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 75
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat1_value                      = -25
-|static_stat2_id                         = skill_physical_damage_%_to_convert_to_lightning
-|static_stat2_value                      = 75
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
-|stat_text                               = Deals (0-200)% more Damage<br>75% of Physical Damage Converted to Lightning Damage
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat2_value                      = -25
+|static_stat3_id                         = skill_physical_damage_%_to_convert_to_lightning
+|static_stat3_value                      = 75
+|stat_text                               = 75% of Physical Damage Converted to Lightning Damage<br>Deals (0-200)% more Damage
 |level1                                  = True
 |level1_level_requirement                = 1
 |level2                                  = True
