--- Skill:HolyElderDoubleSlam2
+++ Export
@@ -9,12 +9,12 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = 300
-|static_stat2_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat2_value                      = -50
-|static_stat3_id                         = active_skill_attack_speed_+%_final
-|static_stat3_value                      = -55
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat3_value                      = -50
+|static_stat4_id                         = active_skill_attack_speed_+%_final
+|static_stat4_value                      = -55
 |stat_text                               = 55% less Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 1
