--- Skill:EssenceBloodProjectileNova
+++ Export
@@ -7,25 +7,25 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 8
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 12
-|static_stat3_id                         = corrupted_blood_on_hit_%_average_damage_to_deal_per_minute_per_stack
-|static_stat3_value                      = 130
-|static_stat4_id                         = corrupted_blood_on_hit_duration
-|static_stat4_value                      = 2000
-|static_stat5_id                         = monster_projectile_variation
-|static_stat5_value                      = 13
-|static_stat6_id                         = number_of_additional_projectiles
-|static_stat6_value                      = 4
-|static_stat7_id                         = base_projectile_speed_+%
-|static_stat7_value                      = -75
-|static_stat8_id                         = projectiles_nova
-|static_stat8_value                      = 1
-|static_stat9_id                         = projectiles_not_offset
-|static_stat9_value                      = 1
-|stat_text                               = Deals 8 to 12 Physical Damage<br>Fires 4 additional Projectiles<br>75% reduced Projectile Speed<br>Fires Projectiles in a circle
+|static_stat1_id                         = corrupted_blood_on_hit_%_average_damage_to_deal_per_minute_per_stack
+|static_stat1_value                      = 130
+|static_stat2_id                         = corrupted_blood_on_hit_duration
+|static_stat2_value                      = 2000
+|static_stat3_id                         = monster_projectile_variation
+|static_stat3_value                      = 13
+|static_stat4_id                         = projectiles_not_offset
+|static_stat4_value                      = 1
+|static_stat5_id                         = spell_minimum_base_physical_damage
+|static_stat5_value                      = 8
+|static_stat6_id                         = spell_maximum_base_physical_damage
+|static_stat6_value                      = 12
+|static_stat7_id                         = projectiles_nova
+|static_stat7_value                      = 1
+|static_stat8_id                         = base_projectile_speed_+%
+|static_stat8_value                      = -75
+|static_stat9_id                         = number_of_additional_projectiles
+|static_stat9_value                      = 4
+|stat_text                               = Deals 8 to 12 Physical Damage<br>Fires Projectiles in a circle<br>75% reduced Projectile Speed<br>Fires 4 additional Projectiles
 |level1                                  = True
 |level1_level_requirement                = 3
 }}