--- Skill:VoidShot
+++ Export
@@ -8,21 +8,21 @@
 |static_damage_effectiveness             = 65
 |static_damage_multiplier                = 65
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_area_damage_+%_final
-|static_stat1_value                      = 25
-|static_stat2_id                         = skill_physical_damage_%_to_convert_to_cold
-|static_stat2_value                      = 50
-|static_stat3_id                         = trigger_on_skill_use_%_if_you_have_a_void_arrow
-|static_stat3_value                      = 100
-|static_stat4_id                         = attack_unusable_if_triggerable
-|static_stat4_value                      = 1
-|static_stat5_id                         = base_is_projectile
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat2_value                      = 1
+|static_stat3_id                         = skill_can_fire_arrows
+|static_stat3_value                      = 1
+|static_stat4_id                         = skill_physical_damage_%_to_convert_to_cold
+|static_stat4_value                      = 50
+|static_stat5_id                         = attack_unusable_if_triggerable
 |static_stat5_value                      = 1
-|static_stat6_id                         = skill_can_fire_arrows
-|static_stat6_value                      = 1
-|static_stat7_id                         = base_skill_show_average_damage_instead_of_dps
-|static_stat7_value                      = 1
-|stat_text                               = 25% more Area Damage<br>50% of Physical Damage Converted to Cold Damage<br>Consume a Void Charge to Trigger this Skill when you fire Arrows with a Non-Triggered Skill<br>You cannot use this Attack directly
+|static_stat6_id                         = active_skill_area_damage_+%_final
+|static_stat6_value                      = 25
+|static_stat7_id                         = trigger_on_skill_use_%_if_you_have_a_void_arrow
+|static_stat7_value                      = 100
+|stat_text                               = 50% of Physical Damage Converted to Cold Damage<br>You cannot use this Attack directly<br>25% more Area Damage<br>Consume a Void Charge to Trigger this Skill when you fire Arrows with a Non-Triggered Skill
 |level1                                  = True
 |level1_level_requirement                = 70
 }}