--- Skill:BreachBossLightningWhip
+++ Export
@@ -8,15 +8,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage
-|static_stat1_value                      = 9
-|static_stat2_id                         = spell_maximum_base_lightning_damage
-|static_stat2_value                      = 26
-|static_stat3_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat3_value                      = 120
-|static_stat4_id                         = always_shock
-|static_stat4_value                      = 1
-|static_stat5_id                         = is_area_damage
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = 120
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = spell_minimum_base_lightning_damage
+|static_stat3_value                      = 9
+|static_stat4_id                         = spell_maximum_base_lightning_damage
+|static_stat4_value                      = 26
+|static_stat5_id                         = always_shock
 |static_stat5_value                      = 1
 |stat_text                               = Deals 9 to 26 Lightning Damage<br>Hits always Shock
 |level1                                  = True
