--- Skill:GSSanctumVortexDash
+++ Export
@@ -0,0 +1,22 @@
+{{Skill
+|gem_description                         = 
+|active_skill_name                       = 
+|skill_id                                = GSSanctumVortexDash
+|cast_time                               = 1.0
+|required_level                          = 0
+|static_critical_strike_chance           = 5
+|static_damage_effectiveness             = 100
+|static_damage_multiplier                = 100
+|static_attack_speed_multiplier          = 100
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_physical_damage
+|static_stat2_value                      = 3
+|static_stat3_id                         = spell_maximum_base_physical_damage
+|static_stat3_value                      = 5
+|static_stat4_id                         = skill_physical_damage_%_to_convert_to_chaos
+|static_stat4_value                      = 25
+|stat_text                               = Deals 3 to 5 Physical Damage<br>25% of Physical Damage Converted to Chaos Damage
+|level1                                  = True
+|level1_level_requirement                = 0
+}}