--- Skill:ExileStaticStrikeKitava
+++ Export
@@ -8,6 +8,7 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 1
 |static_stat1_id                         = base_skill_effect_duration
 |static_stat1_value                      = 1000
 |stat_text                               = Base duration is 1 seconds
