--- Skill:SwampWarp
+++ Export
@@ -11,7 +11,7 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_base_radius_+
 |static_stat1_value                      = 26
-|stat_text                               = +26 to radius
+|stat_text                               = +2.6 metres to radius
 |level1                                  = True
 |level1_level_requirement                = 1
 }}