--- Skill:ChestSynthesisSummonNormalPack
+++ Export
@@ -7,10 +7,10 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = summon_monster_radius
-|static_stat1_value                      = 30
-|static_stat2_id                         = number_of_packs_to_summon
-|static_stat2_value                      = 3
+|static_stat1_id                         = number_of_packs_to_summon
+|static_stat1_value                      = 3
+|static_stat2_id                         = summon_monster_radius
+|static_stat2_value                      = 30
 |stat_text                               = 
 |level1                                  = True
 |level1_level_requirement                = 1
