--- Skill:InfernalCryOnHitExplosion
+++ Export
@@ -31,6 +31,8 @@
 |stat_text                               = 60% of Physical Damage Converted to Fire Damage<br>This Skill is Triggered by Attacks Exerted by Infernal Cry
 |level1                                  = True
 |level1_level_requirement                = 24
+|level1_damage_effectiveness             = 100
+|level1_damage_multiplier                = 100
 |level2                                  = True
 |level2_level_requirement                = 27
 |level2_damage_effectiveness             = 104