--- Skill:LegionMarakethGeneralScreechSnipe
+++ Export
@@ -9,21 +9,21 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = projectile_minimum_range
-|static_stat1_value                      = 30
-|static_stat2_id                         = skill_physical_damage_%_to_convert_to_lightning
-|static_stat2_value                      = 75
-|static_stat3_id                         = active_skill_attack_speed_+%_final
-|static_stat3_value                      = -25
-|static_stat4_id                         = always_pierce
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = projectile_minimum_range
+|static_stat2_value                      = 30
+|static_stat3_id                         = projectile_uses_contact_position
+|static_stat3_value                      = 1
+|static_stat4_id                         = use_scaled_contact_offset
 |static_stat4_value                      = 1
-|static_stat5_id                         = base_is_projectile
+|static_stat5_id                         = always_pierce
 |static_stat5_value                      = 1
-|static_stat6_id                         = projectile_uses_contact_position
-|static_stat6_value                      = 1
-|static_stat7_id                         = use_scaled_contact_offset
-|static_stat7_value                      = 1
-|stat_text                               = Deals (3-701) to (5-1051) Physical Damage<br>Deals (0-200)% more Damage<br>75% of Physical Damage Converted to Lightning Damage<br>25% less Attack Speed<br>Projectiles Pierce all Targets
+|static_stat6_id                         = skill_physical_damage_%_to_convert_to_lightning
+|static_stat6_value                      = 75
+|static_stat7_id                         = active_skill_attack_speed_+%_final
+|static_stat7_value                      = -25
+|stat_text                               = Deals (3-701) to (5-1051) Physical Damage<br>Projectiles Pierce all Targets<br>75% of Physical Damage Converted to Lightning Damage<br>Deals (0-200)% more Damage<br>25% less Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_stat_text                        = Deals 3 to 5 Physical Damage
