--- Skill:StormBurst
+++ Export
@@ -9,14 +9,14 @@
 |static_damage_effectiveness             = 40
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = number_of_additional_projectiles
+|static_stat1_id                         = base_is_projectile
 |static_stat1_value                      = 1
-|static_stat2_id                         = active_skill_area_damage_+%_final
-|static_stat2_value                      = 65
-|static_stat3_id                         = base_is_projectile
+|static_stat2_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat2_value                      = 1
+|static_stat3_id                         = number_of_additional_projectiles
 |static_stat3_value                      = 1
-|static_stat4_id                         = base_skill_show_average_damage_instead_of_dps
-|static_stat4_value                      = 1
+|static_stat4_id                         = active_skill_area_damage_+%_final
+|static_stat4_value                      = 65
 |stat_text                               = Deals (31-521) to (47-782) Lightning Damage<br>Fires an additional Projectile<br>65% more Area Damage
 |level1                                  = True
 |level1_level_requirement                = 28
