--- Skill:SpiderAspect
+++ Export
@@ -9,16 +9,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 6
 |static_mana_reservation_percent         = 25
 |static_stat1_id                         = spider_aspect_web_interval_ms
 |static_stat1_value                      = 1000
-|static_stat2_id                         = spider_aspect_max_web_count
-|static_stat2_value                      = 3
-|static_stat3_id                         = base_skill_effect_duration
-|static_stat3_value                      = 6000
-|static_stat4_id                         = base_damage_taken_+%
-|static_stat4_value                      = 5
-|stat_text                               = Can inflict up to 3 Spider's Webs on an Enemy<br>Base duration is 6 seconds<br>5% increased Damage taken
+|static_stat2_id                         = base_skill_effect_duration
+|static_stat2_value                      = 6000
+|static_stat3_id                         = base_damage_taken_+%
+|static_stat3_value                      = 5
+|static_stat4_id                         = spider_aspect_max_web_count
+|static_stat4_value                      = 3
+|stat_text                               = Base duration is 6 seconds<br>5% increased Damage taken<br>Can inflict up to 3 Spider's Webs on an Enemy
 |level1                                  = True
 |level1_level_requirement                = 1
 }}