--- Skill:EAAVaalGuardBarrageRuin
+++ Export
@@ -9,9 +9,9 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = cast_time_overrides_attack_duration
+|static_stat1_id                         = apply_ultimatum_ruin_on_hit
 |static_stat1_value                      = 1
-|static_stat2_id                         = apply_ultimatum_ruin_on_hit
+|static_stat2_id                         = cast_time_overrides_attack_duration
 |static_stat2_value                      = 1
 |stat_text                               = 
 |level1                                  = True
