--- Skill:ExileFlickerStrikeKitava
+++ Export
@@ -10,15 +10,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_damage_+%_final
-|static_stat1_value                      = -30
-|static_stat2_id                         = base_melee_attack_repeat_count
-|static_stat2_value                      = 2
-|static_stat3_id                         = active_skill_attack_speed_+%_final
-|static_stat3_value                      = 100
-|static_stat4_id                         = ignores_proximity_shield
-|static_stat4_value                      = 1
-|stat_text                               = Deals 30% less Damage<br>Supported Skills Repeat 2 additional times<br>100% more Attack Speed
+|static_stat1_id                         = ignores_proximity_shield
+|static_stat1_value                      = 1
+|static_stat2_id                         = active_skill_damage_+%_final
+|static_stat2_value                      = -30
+|static_stat3_id                         = base_melee_attack_repeat_count
+|static_stat3_value                      = 2
+|static_stat4_id                         = active_skill_attack_speed_+%_final
+|static_stat4_value                      = 100
+|stat_text                               = Deals 30% less Damage<br>This Attack Repeats 2 additional times<br>100% more Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 1
 }}