--- Skill:ArchnemesisPunishment
+++ Export
@@ -1,5 +1,5 @@
 {{Skill
-|gem_description                         = Curses all targets in an area, causing them to be debilitated when they hit enemies and increasing damage they take while on low life. Some overkill damage from a killing blow on a cursed enemy will be reflected to surrounding targets.
+|gem_description                         = Curses all targets in an area, causing them to be debilitated when they hit enemies and increasing damage they take while on low life.
 |active_skill_name                       = Punishment
 |skill_id                                = ArchnemesisPunishment
 |cast_time                               = 1.0
@@ -9,13 +9,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = debilitate_self_for_x_milliseconds_on_hit
-|static_stat1_value                      = 2000
+|static_duration                         = 5
+|static_stat1_id                         = base_skill_effect_duration
+|static_stat1_value                      = 5000
 |static_stat2_id                         = damage_taken_+%_on_low_life
 |static_stat2_value                      = 70
-|static_stat3_id                         = base_skill_effect_duration
-|static_stat3_value                      = 5000
-|stat_text                               = Cursed Enemies are Debilitated for 2 Seconds when they Hit<br>Cursed Enemies take 70% increased Damage while on Low Life<br>Base duration is 5 seconds
+|static_stat3_id                         = debilitate_self_for_x_milliseconds_on_hit
+|static_stat3_value                      = 2000
+|stat_text                               = Base duration is 5 seconds<br>Cursed Enemies take 70% increased Damage while on Low Life<br>Cursed Enemies are Debilitated for 2 Seconds when they Hit
 |level1                                  = True
 |level1_level_requirement                = 1
 }}