--- Skill:HeistFreidrichMinionSuicideExplode
+++ Export
@@ -7,14 +7,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 11
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 16
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
-|static_stat4_id                         = grant_kill_to_target_when_exploding_self
-|static_stat4_value                      = 1
+|static_stat1_id                         = grant_kill_to_target_when_exploding_self
+|static_stat1_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = spell_minimum_base_physical_damage
+|static_stat3_value                      = 11
+|static_stat4_id                         = spell_maximum_base_physical_damage
+|static_stat4_value                      = 16
 |stat_text                               = Deals 11 to 16 Physical Damage
 |level1                                  = True
 |level1_level_requirement                = 1
