--- Skill:RotateDaemonObsidian5
+++ Export
@@ -9,16 +9,16 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = move_daemon_y_offset
-|static_stat1_value                      = 15
-|static_stat2_id                         = move_daemon_pivot_variation
-|static_stat2_value                      = 1
-|static_stat3_id                         = move_daemon_end_rotation
-|static_stat3_value                      = 350
-|static_stat4_id                         = move_daemon_movement_speed
-|static_stat4_value                      = 30
-|static_stat5_id                         = move_daemon_number_of_daemons_to_spawn
-|static_stat5_value                      = 1
+|static_stat1_id                         = move_daemon_end_rotation
+|static_stat1_value                      = 350
+|static_stat2_id                         = move_daemon_movement_speed
+|static_stat2_value                      = 30
+|static_stat3_id                         = move_daemon_number_of_daemons_to_spawn
+|static_stat3_value                      = 1
+|static_stat4_id                         = move_daemon_pivot_variation
+|static_stat4_value                      = 1
+|static_stat5_id                         = move_daemon_y_offset
+|static_stat5_value                      = 15
 |stat_text                               = 
 |level1                                  = True
 |level1_level_requirement                = 1
