--- Skill:MonsterFlickerStrikeExtraRange
+++ Export
@@ -24,7 +24,7 @@
 |static_stat5_value                      = 10
 |static_stat6_id                         = melee_range_+
 |static_stat6_value                      = 5
-|stat_text                               = Deals 10% more Damage<br>30% increased Attack Speed<br>10% increased Attack Speed per Frenzy Charge<br>+5 to Melee Strike Range
+|stat_text                               = Deals 10% more Damage<br>30% increased Attack Speed<br>10% increased Attack Speed per Frenzy Charge<br>+0.5 metres to Melee Strike Range
 |level1                                  = True
 |level1_level_requirement                = 13
 }}