--- Skill:JaguarDemonDash
+++ Export
@@ -13,7 +13,7 @@
 |static_stat1_value                      = 1
 |static_stat2_id                         = melee_range_+
 |static_stat2_value                      = 50
-|stat_text                               = +50 to Melee Strike Range
+|stat_text                               = +5 metres to Melee Strike Range
 |level1                                  = True
 |level1_level_requirement                = 0
 }}