--- Skill:TemplarSlaveDriverStormCallLong
+++ Export
@@ -8,21 +8,22 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage
-|static_stat1_value                      = 148
-|static_stat2_id                         = spell_maximum_base_lightning_damage
-|static_stat2_value                      = 445
+|static_duration                         = 8
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = 20
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
 |static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
 |static_stat3_value                      = -25
-|static_stat4_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat4_value                      = 20
-|static_stat5_id                         = base_cast_speed_+%
-|static_stat5_value                      = -55
+|static_stat4_id                         = spell_minimum_base_lightning_damage
+|static_stat4_value                      = 148
+|static_stat5_id                         = spell_maximum_base_lightning_damage
+|static_stat5_value                      = 445
 |static_stat6_id                         = base_skill_effect_duration
 |static_stat6_value                      = 8000
-|static_stat7_id                         = is_area_damage
-|static_stat7_value                      = 1
-|stat_text                               = Deals 148 to 445 Lightning Damage<br>55% reduced Cast Speed<br>Base duration is 8 seconds
+|static_stat7_id                         = base_cast_speed_+%
+|static_stat7_value                      = -55
+|stat_text                               = Deals 148 to 445 Lightning Damage<br>Base duration is 8 seconds<br>55% reduced Cast Speed
 |level1                                  = True
 |level1_level_requirement                = 40
 }}