--- Skill:MeleeSpiralStrMissionTotem
+++ Export
@@ -11,12 +11,12 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = projectile_spiral_nova_time_ms
 |static_stat1_value                      = 1600
-|static_stat2_id                         = base_number_of_projectiles_in_spiral_nova
-|static_stat2_value                      = 15
-|static_stat3_id                         = projectile_spiral_nova_angle
-|static_stat3_value                      = 3200
-|static_stat4_id                         = skill_can_fire_wand_projectiles
-|static_stat4_value                      = 1
+|static_stat2_id                         = skill_can_fire_wand_projectiles
+|static_stat2_value                      = 1
+|static_stat3_id                         = base_number_of_projectiles_in_spiral_nova
+|static_stat3_value                      = 15
+|static_stat4_id                         = projectile_spiral_nova_angle
+|static_stat4_value                      = 3200
 |stat_text                               = Fires 15 Projectiles in a spiral
 |level1                                  = True
 |level1_level_requirement                = 1
