--- Skill:DawnFireBeamDivineMap
+++ Export
@@ -9,13 +9,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 1.5
 |static_stat1_id                         = base_fire_damage_to_deal_per_minute
 |static_stat1_value                      = 53144
-|static_stat2_id                         = fire_beam_additional_stack_damage_+%_final
-|static_stat2_value                      = -70
-|static_stat3_id                         = base_skill_effect_duration
-|static_stat3_value                      = 1500
-|stat_text                               = Deals 885.7 Base Fire Damage per second<br>Additional Debuff stages add 30% of Damage<br>Base duration is 1.5 seconds
+|static_stat2_id                         = base_skill_effect_duration
+|static_stat2_value                      = 1500
+|static_stat3_id                         = fire_beam_additional_stack_damage_+%_final
+|static_stat3_value                      = -70
+|stat_text                               = Deals 885.7 Base Fire Damage per second<br>Base duration is 1.5 seconds<br>Additional Debuff stages add 30% of Damage
 |level1                                  = True
 |level1_level_requirement                = 76
 }}