--- Skill:VaalGuardExplosiveArrowAtAnimationSpeed
+++ Export
@@ -9,21 +9,22 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 115
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = minimum_fire_damage_per_fuse_arrow_orb
-|static_stat1_value                      = 6
-|static_stat2_id                         = maximum_fire_damage_per_fuse_arrow_orb
-|static_stat2_value                      = 9
-|static_stat3_id                         = fuse_arrow_explosion_radius_+_per_fuse_arrow_orb
-|static_stat3_value                      = 2
+|static_duration                         = 1
+|static_stat1_id                         = action_attack_or_cast_time_uses_animation_length
+|static_stat1_value                      = 1
+|static_stat2_id                         = base_is_projectile
+|static_stat2_value                      = 1
+|static_stat3_id                         = skill_can_fire_arrows
+|static_stat3_value                      = 1
 |static_stat4_id                         = base_skill_effect_duration
 |static_stat4_value                      = 1000
-|static_stat5_id                         = action_attack_or_cast_time_uses_animation_length
-|static_stat5_value                      = 1
-|static_stat6_id                         = skill_can_fire_arrows
-|static_stat6_value                      = 1
-|static_stat7_id                         = base_is_projectile
-|static_stat7_value                      = 1
-|stat_text                               = Explosion deals 6 to 9 Base Fire Damage per Fuse Charge<br>+2 to Explosion Radius per Explosive Arrow on Target<br>Base duration is 1 seconds
+|static_stat5_id                         = minimum_fire_damage_per_fuse_arrow_orb
+|static_stat5_value                      = 6
+|static_stat6_id                         = maximum_fire_damage_per_fuse_arrow_orb
+|static_stat6_value                      = 9
+|static_stat7_id                         = fuse_arrow_explosion_radius_+_per_fuse_arrow_orb
+|static_stat7_value                      = 2
+|stat_text                               = Base duration is 1 seconds<br>Explosion deals 6 to 9 Base Fire Damage per Fuse Charge<br>+2 to Explosion Radius per Explosive Arrow on Target
 |level1                                  = True
 |level1_level_requirement                = 1
 }}