--- Skill:ProphecyVaalFlameBlast
+++ Export
@@ -10,17 +10,17 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = 50
-|static_stat2_id                         = flameblast_ailment_damage_+%_final_per_stack
-|static_stat2_value                      = 110
-|static_stat3_id                         = base_chance_to_ignite_%
-|static_stat3_value                      = 100
-|static_stat4_id                         = base_cast_speed_+%
-|static_stat4_value                      = 35
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = base_cast_speed_+%
+|static_stat3_value                      = 35
+|static_stat4_id                         = base_chance_to_ignite_%
+|static_stat4_value                      = 100
 |static_stat5_id                         = charged_blast_spell_damage_+%_final_per_stack
 |static_stat5_value                      = 110
-|static_stat6_id                         = is_area_damage
-|static_stat6_value                      = 1
-|stat_text                               = Deals (4-152) to (6-227) Fire Damage<br>110% more Damage with Ailments for each stage<br>100% chance to Ignite enemies<br>35% increased Cast Speed<br>110% more Spell Damage for each stage
+|static_stat6_id                         = flameblast_ailment_damage_+%_final_per_stack
+|static_stat6_value                      = 110
+|stat_text                               = Deals (4-152) to (6-227) Fire Damage<br>35% increased Cast Speed<br>100% chance to Ignite enemies<br>110% more Spell Damage for each stage<br>110% more Damage with Ailments for each stage
 |level1                                  = True
 |level1_level_requirement                = 3
 |level1_cooldown                         = 7
