--- Skill:DoomArrow
+++ Export
@@ -7,16 +7,16 @@
 |required_level                          = 28
 |static_cost_types                       = Mana
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = doom_arrow_area_of_effect_+%_final_per_frenzy_charge
-|static_stat1_value                      = 25
-|static_stat2_id                         = doom_arrow_number_of_arrows
-|static_stat2_value                      = 5
-|static_stat3_id                         = add_frenzy_charge_on_skill_hit_%
-|static_stat3_value                      = 8
-|static_stat4_id                         = base_is_projectile
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = doom_arrow_area_of_effect_+%_final_per_frenzy_charge
+|static_stat2_value                      = 25
+|static_stat3_id                         = doom_arrow_number_of_arrows
+|static_stat3_value                      = 5
+|static_stat4_id                         = is_area_damage
 |static_stat4_value                      = 1
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
+|static_stat5_id                         = add_frenzy_charge_on_skill_hit_%
+|static_stat5_value                      = 8
 |stat_text                               = 8% chance to gain a Frenzy Charge on Hit
 |level1                                  = True
 |level1_level_requirement                = 28
