--- Skill:LightningGolemMapBossArc
+++ Export
@@ -10,12 +10,13 @@
 |static_damage_effectiveness             = 60
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage
-|static_stat1_value                      = 196
-|static_stat2_id                         = spell_maximum_base_lightning_damage
-|static_stat2_value                      = 587
-|static_stat3_id                         = alternate_minion
-|static_stat3_value                      = 1
+|static_duration                         = 8
+|static_stat1_id                         = alternate_minion
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_lightning_damage
+|static_stat2_value                      = 196
+|static_stat3_id                         = spell_maximum_base_lightning_damage
+|static_stat3_value                      = 587
 |static_stat4_id                         = base_skill_effect_duration
 |static_stat4_value                      = 8000
 |stat_text                               = Deals 196 to 587 Lightning Damage<br>Base duration is 8 seconds
