--- Skill:AssassinsMarkTriggered
+++ Export
@@ -10,13 +10,13 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = chance_to_grant_power_charge_on_death_%
-|static_stat1_value                      = 100
-|static_stat2_id                         = enemy_additional_critical_strike_chance_against_self
-|static_stat2_value                      = 150
-|static_stat3_id                         = trigger_on_hit_against_rare_or_unique_if_no_marked_enemy
-|static_stat3_value                      = 1
-|static_stat4_id                         = base_deal_no_damage
+|static_stat1_id                         = base_deal_no_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = chance_to_grant_power_charge_on_death_%
+|static_stat2_value                      = 100
+|static_stat3_id                         = enemy_additional_critical_strike_chance_against_self
+|static_stat3_value                      = 150
+|static_stat4_id                         = trigger_on_hit_against_rare_or_unique_if_no_marked_enemy
 |static_stat4_value                      = 1
 |stat_text                               = Hits have +1.5% additional Critical Strike Chance against you<br>Trigger this Skill when you Hit a Rare or Unique Enemy and have no Mark
 |level1                                  = True
