--- Skill:WbColdLeaderIcestorm
+++ Export
@@ -1,6 +1,6 @@
 {{Skill
 |gem_description                         = Flaming bolts rain down over the targeted area. They explode when landing, dealing damage to nearby enemies.
-|active_skill_name                       = 
+|active_skill_name                       = Firestorm
 |skill_id                                = WbColdLeaderIcestorm
 |cast_time                               = 0.9
 |required_level                          = 3
@@ -9,19 +9,20 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = firestorm_drop_ground_ice_duration_ms
-|static_stat1_value                      = 3000
-|static_stat2_id                         = firestorm_base_area_of_effect_+%
-|static_stat2_value                      = 300
-|static_stat3_id                         = fire_storm_fireball_delay_ms
-|static_stat3_value                      = 100
-|static_stat4_id                         = base_skill_effect_duration
-|static_stat4_value                      = 6000
-|static_stat5_id                         = cannot_stun
-|static_stat5_value                      = 1
-|static_stat6_id                         = is_area_damage
-|static_stat6_value                      = 1
-|stat_text                               = Deals (1-2) to 2 Cold Damage<br>One impact every 0.1 seconds<br>Base duration is 6 seconds<br>Your Hits cannot Stun Enemies
+|static_duration                         = 6
+|static_stat1_id                         = firestorm_base_area_of_effect_+%
+|static_stat1_value                      = 300
+|static_stat2_id                         = firestorm_drop_ground_ice_duration_ms
+|static_stat2_value                      = 3000
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = cannot_stun
+|static_stat4_value                      = 1
+|static_stat5_id                         = base_skill_effect_duration
+|static_stat5_value                      = 6000
+|static_stat6_id                         = fire_storm_fireball_delay_ms
+|static_stat6_value                      = 100
+|stat_text                               = Deals (1-2) to 2 Cold Damage<br>Your Hits cannot Stun Enemies<br>Base duration is 6 seconds<br>One impact every 0.1 seconds
 |level1                                  = True
 |level1_level_requirement                = 3
 |level1_stat_text                        = Deals 1 to 2 Cold Damage
