--- Skill:MerveilDaughterMoltenStrike
+++ Export
@@ -1,5 +1,5 @@
 {{Skill
-|gem_description                         = Infuses your melee weapon with molten energies to attack with physical and fire damage. This attack causes balls of molten magma to launch forth from the enemies you hit, divided amongst all enemies hit by the strike. These will explode, causing AoE attack damage to enemies where they land.
+|gem_description                         = Infuses your melee weapon with molten energies to attack with physical and fire damage. This attack causes balls of molten magma to launch forth from the enemies you hit, divided amongst all enemies hit by the strike. These will deal area attack damage to enemies where they land.
 |active_skill_name                       = Molten Strike
 |item_class_id_restriction               = Claw, Dagger, One Hand Sword, Thrusting One Hand Sword, One Hand Axe, One Hand Mace, Sceptre, Staff, Two Hand Sword, Two Hand Axe, Two Hand Mace, Warstaff, Rune Dagger
 |skill_id                                = MerveilDaughterMoltenStrike
@@ -8,13 +8,13 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 75
-|static_stat1_id                         = base_projectile_speed_+%
-|static_stat1_value                      = -25
-|static_stat2_id                         = skill_physical_damage_%_to_convert_to_fire
-|static_stat2_value                      = 60
-|static_stat3_id                         = action_attack_or_cast_time_uses_animation_length
-|static_stat3_value                      = 1
-|stat_text                               = Fires (2-4) additional Projectiles<br>Deals (0-20)% less Damage<br>Projectiles deal (0-30)% less Damage<br>Skills Repeat an additional (0-2) Times<br>(0-100)% more Attack Speed<br>25% reduced Projectile Speed<br>60% of Physical Damage Converted to Fire Damage
+|static_stat1_id                         = action_attack_or_cast_time_uses_animation_length
+|static_stat1_value                      = 1
+|static_stat2_id                         = base_projectile_speed_+%
+|static_stat2_value                      = -25
+|static_stat3_id                         = skill_physical_damage_%_to_convert_to_fire
+|static_stat3_value                      = 60
+|stat_text                               = Skills Repeat an additional (0-2) Times<br>Deals (0-20)% less Damage<br>25% reduced Projectile Speed<br>Fires (2-4) additional Projectiles<br>60% of Physical Damage Converted to Fire Damage<br>(0-100)% more Attack Speed<br>(0-30)% less Projectile Damage
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_stat_text                        = Fires 2 additional Projectiles
@@ -22,15 +22,15 @@
 |level1_stat1_value                      = 2
 |level2                                  = True
 |level2_level_requirement                = 66
-|level2_stat_text                        = Fires 4 additional Projectiles<br>Deals 20% less Damage<br>Projectiles deal 30% less Damage<br>Skills Repeat an additional 2 Times<br>100% more Attack Speed
-|level2_stat1_id                         = number_of_additional_projectiles
-|level2_stat1_value                      = 4
+|level2_stat_text                        = Skills Repeat an additional 2 Times<br>Deals 20% less Damage<br>Fires 4 additional Projectiles<br>100% more Attack Speed<br>30% less Projectile Damage
+|level2_stat1_id                         = skill_repeat_count
+|level2_stat1_value                      = 2
 |level2_stat2_id                         = active_skill_damage_+%_final
 |level2_stat2_value                      = -20
-|level2_stat3_id                         = active_skill_projectile_damage_+%_final
-|level2_stat3_value                      = -30
-|level2_stat4_id                         = skill_repeat_count
-|level2_stat4_value                      = 2
-|level2_stat5_id                         = active_skill_attack_speed_+%_final
-|level2_stat5_value                      = 100
+|level2_stat3_id                         = number_of_additional_projectiles
+|level2_stat3_value                      = 4
+|level2_stat4_id                         = active_skill_attack_speed_+%_final
+|level2_stat4_value                      = 100
+|level2_stat5_id                         = active_skill_projectile_damage_+%_final
+|level2_stat5_value                      = -30
 }}