--- Skill:PhysicalReflectMortar
+++ Export
@@ -11,18 +11,18 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = 10
-|static_stat2_id                         = projectile_minimum_range
-|static_stat2_value                      = 20
-|static_stat3_id                         = trigger_daemon_when_hit_triggered_skill_chance_%
-|static_stat3_value                      = 100
-|static_stat4_id                         = projectile_spread_radius
-|static_stat4_value                      = 5
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
-|static_stat6_id                         = base_is_projectile
-|static_stat6_value                      = 1
-|static_stat7_id                         = is_trigger_daemon_when_hit_triggered_skill
-|static_stat7_value                      = 1
+|static_stat2_id                         = base_is_projectile
+|static_stat2_value                      = 1
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = is_trigger_daemon_when_hit_triggered_skill
+|static_stat4_value                      = 1
+|static_stat5_id                         = projectile_minimum_range
+|static_stat5_value                      = 20
+|static_stat6_id                         = projectile_spread_radius
+|static_stat6_value                      = 5
+|static_stat7_id                         = trigger_daemon_when_hit_triggered_skill_chance_%
+|static_stat7_value                      = 100
 |stat_text                               = Deals (2-1352) to (3-2028) Physical Damage
 |level1                                  = True
 |level1_level_requirement                = 1
