--- Skill:MassReviveUndeadDelveRhoa
+++ Export
@@ -0,0 +1,21 @@
+{{Skill
+|gem_description                         = 
+|active_skill_name                       = 
+|skill_id                                = MassReviveUndeadDelveRhoa
+|cast_time                               = 6.0
+|required_level                          = 0
+|static_stored_uses                      = 1
+|static_cooldown                         = 7
+|static_damage_effectiveness             = 100
+|static_damage_multiplier                = 100
+|static_attack_speed_multiplier          = 100
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = -75
+|static_stat2_id                         = necromancer_additional_rarity_levels_can_be_raised
+|static_stat2_value                      = 2
+|static_stat3_id                         = base_cast_speed_+%
+|static_stat3_value                      = 200
+|stat_text                               = 200% increased Cast Speed
+|level1                                  = True
+|level1_level_requirement                = 0
+}}