--- Skill:WbHeavyStrike
+++ Export
@@ -8,15 +8,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_attack_speed_+%_final
-|static_stat1_value                      = 25
+|static_stat1_id                         = global_knockback
+|static_stat1_value                      = 1
 |static_stat2_id                         = active_skill_damage_+%_final
 |static_stat2_value                      = 10
 |static_stat3_id                         = base_stun_threshold_reduction_+%
 |static_stat3_value                      = 30
-|static_stat4_id                         = global_knockback
-|static_stat4_value                      = 1
-|stat_text                               = 25% more Attack Speed<br>Deals 10% more Damage<br>30% reduced Enemy Stun Threshold<br>Knocks Enemies Back on Hit
+|static_stat4_id                         = active_skill_attack_speed_+%_final
+|static_stat4_value                      = 25
+|stat_text                               = Knocks Enemies Back on Hit<br>Deals 10% more Damage<br>30% reduced Enemy Stun Threshold<br>25% more Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 1
 }}