--- Skill:ImmortalCallOak
+++ Export
@@ -8,9 +8,10 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 1
 |static_stat1_id                         = base_skill_effect_duration
 |static_stat1_value                      = 1000
-|stat_text                               = +(0.5-0.8) seconds to Base Duration per Endurance Charge removed<br>Base duration is 1 seconds
+|stat_text                               = Base duration is 1 seconds<br>+(0.5-0.8) seconds to Base Duration per Endurance Charge removed
 |level1                                  = True
 |level1_level_requirement                = 19
 |level1_cost_amounts                     = 17
