--- Skill:ChaosArchitectProjectileNova
+++ Export
@@ -8,19 +8,19 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_chaos_damage
-|static_stat1_value                      = 5
-|static_stat2_id                         = spell_maximum_base_chaos_damage
-|static_stat2_value                      = 7
-|static_stat3_id                         = monster_projectile_variation
-|static_stat3_value                      = 56
-|static_stat4_id                         = number_of_additional_projectiles
-|static_stat4_value                      = 10
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_projectile_variation
+|static_stat2_value                      = 56
+|static_stat3_id                         = spell_minimum_base_chaos_damage
+|static_stat3_value                      = 5
+|static_stat4_id                         = spell_maximum_base_chaos_damage
+|static_stat4_value                      = 7
 |static_stat5_id                         = projectiles_nova
 |static_stat5_value                      = 1
-|static_stat6_id                         = base_is_projectile
-|static_stat6_value                      = 1
-|stat_text                               = Deals 5 to 7 Chaos Damage<br>Fires 10 additional Projectiles<br>Fires Projectiles in a circle
+|static_stat6_id                         = number_of_additional_projectiles
+|static_stat6_value                      = 10
+|stat_text                               = Deals 5 to 7 Chaos Damage<br>Fires Projectiles in a circle<br>Fires 10 additional Projectiles
 |level1                                  = True
 |level1_level_requirement                = 1
 }}