--- Skill:ReliquaryMonsterFirestorm
+++ Export
@@ -9,23 +9,24 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 3
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 5
-|static_stat3_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat3_value                      = 30
+|static_duration                         = 1.2
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = 30
+|static_stat2_id                         = firestorm_base_area_of_effect_+%
+|static_stat2_value                      = -55
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
 |static_stat4_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
 |static_stat4_value                      = -30
-|static_stat5_id                         = firestorm_base_area_of_effect_+%
-|static_stat5_value                      = -55
-|static_stat6_id                         = fire_storm_fireball_delay_ms
-|static_stat6_value                      = 300
+|static_stat5_id                         = spell_minimum_base_physical_damage
+|static_stat5_value                      = 3
+|static_stat6_id                         = spell_maximum_base_physical_damage
+|static_stat6_value                      = 5
 |static_stat7_id                         = base_skill_effect_duration
 |static_stat7_value                      = 1200
-|static_stat8_id                         = is_area_damage
-|static_stat8_value                      = 1
-|stat_text                               = Deals 3 to 5 Physical Damage<br>One impact every 0.3 seconds<br>Base duration is 1.2 seconds
+|static_stat8_id                         = fire_storm_fireball_delay_ms
+|static_stat8_value                      = 300
+|stat_text                               = Deals 3 to 5 Physical Damage<br>Base duration is 1.2 seconds<br>One impact every 0.3 seconds
 |level1                                  = True
 |level1_level_requirement                = 1
 }}