--- Skill:DelveAbyssLichArmLaser
+++ Export
@@ -8,21 +8,21 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 877
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 1315
-|static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat3_value                      = -75
-|static_stat4_id                         = skill_physical_damage_%_to_convert_to_chaos
-|static_stat4_value                      = 20
-|static_stat5_id                         = skill_physical_damage_%_to_convert_to_lightning
-|static_stat5_value                      = 80
-|static_stat6_id                         = cannot_stun
-|static_stat6_value                      = 1
-|static_stat7_id                         = is_area_damage
-|static_stat7_value                      = 1
-|stat_text                               = Deals 877 to 1315 Physical Damage<br>20% of Physical Damage Converted to Chaos Damage<br>80% of Physical Damage Converted to Lightning Damage<br>Your Hits cannot Stun Enemies
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat2_value                      = -75
+|static_stat3_id                         = spell_minimum_base_physical_damage
+|static_stat3_value                      = 877
+|static_stat4_id                         = spell_maximum_base_physical_damage
+|static_stat4_value                      = 1315
+|static_stat5_id                         = cannot_stun
+|static_stat5_value                      = 1
+|static_stat6_id                         = skill_physical_damage_%_to_convert_to_lightning
+|static_stat6_value                      = 80
+|static_stat7_id                         = skill_physical_damage_%_to_convert_to_chaos
+|static_stat7_value                      = 20
+|stat_text                               = Deals 877 to 1315 Physical Damage<br>Your Hits cannot Stun Enemies<br>80% of Physical Damage Converted to Lightning Damage<br>20% of Physical Damage Converted to Chaos Damage
 |level1                                  = True
 |level1_level_requirement                = 83
 }}