--- Skill:MinerThrowFireSpectre
+++ Export
@@ -16,6 +16,7 @@
 |stat_text                               = Deals (2.8-387.1) Base Fire Damage per second<br>Base duration is (2-4) seconds<br>Deals (1-174) to (2-261) Fire Damage
 |level1                                  = True
 |level1_level_requirement                = 3
+|level1_duration                         = 2
 |level1_stat_text                        = Deals 2.8 Base Fire Damage per second<br>Base duration is 2 seconds<br>Deals 1 to 2 Fire Damage
 |level1_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level1_stat1_value                      = 170
@@ -27,6 +28,7 @@
 |level1_stat4_value                      = 2
 |level2                                  = True
 |level2_level_requirement                = 5
+|level2_duration                         = 2.1
 |level2_stat_text                        = Deals 3.5 Base Fire Damage per second<br>Base duration is 2.1 seconds<br>Deals 2 to 2 Fire Damage
 |level2_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level2_stat1_value                      = 212
@@ -38,6 +40,7 @@
 |level2_stat4_value                      = 2
 |level3                                  = True
 |level3_level_requirement                = 8
+|level3_duration                         = 2.2
 |level3_stat_text                        = Deals 4.8 Base Fire Damage per second<br>Base duration is 2.2 seconds<br>Deals 2 to 3 Fire Damage
 |level3_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level3_stat1_value                      = 287
@@ -49,6 +52,7 @@
 |level3_stat4_value                      = 3
 |level4                                  = True
 |level4_level_requirement                = 11
+|level4_duration                         = 2.3
 |level4_stat_text                        = Deals 6.3 Base Fire Damage per second<br>Base duration is 2.3 seconds<br>Deals 3 to 4 Fire Damage
 |level4_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level4_stat1_value                      = 378
@@ -60,6 +64,7 @@
 |level4_stat4_value                      = 4
 |level5                                  = True
 |level5_level_requirement                = 15
+|level5_duration                         = 2.4
 |level5_stat_text                        = Deals 8.8 Base Fire Damage per second<br>Base duration is 2.4 seconds<br>Deals 4 to 6 Fire Damage
 |level5_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level5_stat1_value                      = 527
@@ -71,6 +76,7 @@
 |level5_stat4_value                      = 6
 |level6                                  = True
 |level6_level_requirement                = 19
+|level6_duration                         = 2.5
 |level6_stat_text                        = Deals 11.9 Base Fire Damage per second<br>Base duration is 2.5 seconds<br>Deals 5 to 8 Fire Damage
 |level6_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level6_stat1_value                      = 716
@@ -82,6 +88,7 @@
 |level6_stat4_value                      = 8
 |level7                                  = True
 |level7_level_requirement                = 23
+|level7_duration                         = 2.6
 |level7_stat_text                        = Deals 15.9 Base Fire Damage per second<br>Base duration is 2.6 seconds<br>Deals 7 to 11 Fire Damage
 |level7_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level7_stat1_value                      = 954
@@ -93,6 +100,7 @@
 |level7_stat4_value                      = 11
 |level8                                  = True
 |level8_level_requirement                = 27
+|level8_duration                         = 2.7
 |level8_stat_text                        = Deals 20.9 Base Fire Damage per second<br>Base duration is 2.7 seconds<br>Deals 9 to 14 Fire Damage
 |level8_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level8_stat1_value                      = 1252
@@ -104,6 +112,7 @@
 |level8_stat4_value                      = 14
 |level9                                  = True
 |level9_level_requirement                = 28
+|level9_duration                         = 2.8
 |level9_stat_text                        = Deals 22.3 Base Fire Damage per second<br>Base duration is 2.8 seconds<br>Deals 10 to 15 Fire Damage
 |level9_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level9_stat1_value                      = 1337
@@ -115,6 +124,7 @@
 |level9_stat4_value                      = 15
 |level10                                 = True
 |level10_level_requirement               = 32
+|level10_duration                        = 2.9
 |level10_stat_text                       = Deals 28.8 Base Fire Damage per second<br>Base duration is 2.9 seconds<br>Deals 13 to 19 Fire Damage
 |level10_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level10_stat1_value                     = 1730
@@ -126,6 +136,7 @@
 |level10_stat4_value                     = 19
 |level11                                 = True
 |level11_level_requirement               = 40
+|level11_duration                        = 3
 |level11_stat_text                       = Deals 47 Base Fire Damage per second<br>Base duration is 3 seconds<br>Deals 21 to 32 Fire Damage
 |level11_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level11_stat1_value                     = 2821
@@ -137,6 +148,7 @@
 |level11_stat4_value                     = 32
 |level12                                 = True
 |level12_level_requirement               = 79
+|level12_duration                        = 4
 |level12_stat_text                       = Deals 387.1 Base Fire Damage per second<br>Base duration is 4 seconds<br>Deals 174 to 261 Fire Damage
 |level12_stat1_id                        = base_fire_damage_to_deal_per_minute
 |level12_stat1_value                     = 23226
