--- Skill:ViciousHexExplosionRoyale
+++ Export
@@ -1,5 +1,5 @@
 {{Skill
-|gem_description                         = Deals chaos damage in an area based on the amount of Doom on the triggering Hex.
+|gem_description                         = Deals chaos damage in an area.
 |active_skill_name                       = Doom Blast
 |skill_id                                = ViciousHexExplosionRoyale
 |cast_time                               = 1.0
@@ -10,13 +10,11 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = doom_blast_hit_damage_+%_final_per_5_doom_on_hex
-|static_stat1_value                      = 60
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
 |static_stat2_id                         = triggered_vicious_hex_explosion
 |static_stat2_value                      = 1
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
-|stat_text                               = Deals (4-54) to (6-81) Chaos Damage<br>60% more Damage with Hits per 5 Doom on Hex<br>Trigger This Skill when a Hex from Supported Skills ends
+|stat_text                               = Deals (4-54) to (6-81) Chaos Damage<br>Trigger This Skill when a Hex from Supported Skills ends
 |level1                                  = True
 |level1_level_requirement                = 4
 |level1_stat_text                        = Deals 4 to 6 Chaos Damage
