--- Skill:ExileWitchSummonRagingSpirit
+++ Export
@@ -7,6 +7,7 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 8
 |static_stat1_id                         = alternate_minion
 |static_stat1_value                      = 1
 |static_stat2_id                         = skill_repeat_count
@@ -15,7 +16,7 @@
 |static_stat3_value                      = 50
 |static_stat4_id                         = base_skill_effect_duration
 |static_stat4_value                      = 8000
-|stat_text                               = (40-60)% increased Cast Speed<br>Skills Repeat an additional 2 Times<br>Maximum 50 Summoned Raging Spirits<br>Base duration is 8 seconds
+|stat_text                               = Skills Repeat an additional 2 Times<br>Maximum 50 Summoned Raging Spirits<br>Base duration is 8 seconds<br>(40-60)% increased Cast Speed
 |level1                                  = True
 |level1_level_requirement                = 4
 |level1_stat_text                        = 40% increased Cast Speed
