--- Skill:EnchantmentOfBladesOnHit4
+++ Export
@@ -10,23 +10,23 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 1833
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 2903
-|static_stat3_id                         = number_of_additional_projectiles
-|static_stat3_value                      = 2
-|static_stat4_id                         = cast_on_hit_%
-|static_stat4_value                      = 100
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_physical_damage
+|static_stat2_value                      = 1833
+|static_stat3_id                         = spell_maximum_base_physical_damage
+|static_stat3_value                      = 2903
+|static_stat4_id                         = number_of_additional_projectiles
+|static_stat4_value                      = 2
 |static_stat5_id                         = number_of_chains
 |static_stat5_value                      = 4
 |static_stat6_id                         = spell_uncastable_if_triggerable
 |static_stat6_value                      = 1
-|static_stat7_id                         = damage_cannot_be_reflected
-|static_stat7_value                      = 1
-|static_stat8_id                         = base_is_projectile
+|static_stat7_id                         = cast_on_hit_%
+|static_stat7_value                      = 100
+|static_stat8_id                         = damage_cannot_be_reflected
 |static_stat8_value                      = 1
-|stat_text                               = Deals 1833 to 2903 Physical Damage<br>Fires 2 additional Projectiles<br>Trigger this Spell when you Hit<br>Chains +4 Times<br>You cannot Cast this Spell directly<br>Damage cannot be Reflected
+|stat_text                               = Deals 1833 to 2903 Physical Damage<br>Fires 2 additional Projectiles<br>Chains +4 Times<br>You cannot Cast this Spell directly<br>Trigger this Spell when you Hit<br>Damage cannot be Reflected
 |level1                                  = True
 |level1_level_requirement                = 75
 }}