--- Skill:InfernalCryOnHitExplosion
+++ Export
@@ -22,17 +22,15 @@
 |quality_type4_stat1_id                  = cover_in_ash_on_hit_%
 |quality_type4_stat1_value               = 1000
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = skill_physical_damage_%_to_convert_to_fire
-|static_stat1_value                      = 60
-|static_stat2_id                         = triggered_by_infernal_cry
-|static_stat2_value                      = 1
-|static_stat3_id                         = is_area_damage
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = skill_physical_damage_%_to_convert_to_fire
+|static_stat2_value                      = 60
+|static_stat3_id                         = triggered_by_infernal_cry
 |static_stat3_value                      = 1
 |stat_text                               = 60% of Physical Damage Converted to Fire Damage<br>This Skill is Triggered by Attacks Exerted by Infernal Cry
 |level1                                  = True
 |level1_level_requirement                = 24
-|level1_damage_effectiveness             = 100
-|level1_damage_multiplier                = 100
 |level2                                  = True
 |level2_level_requirement                = 27
 |level2_damage_effectiveness             = 104
