--- Skill:CockerelMapBossVaalRainOfArrows
+++ Export
@@ -10,15 +10,16 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_skill_effect_duration
-|static_stat1_value                      = 1500
-|static_stat2_id                         = no_movement_speed
+|static_duration                         = 1.5
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = is_area_damage
 |static_stat2_value                      = 1
-|static_stat3_id                         = base_is_projectile
+|static_stat3_id                         = rain_of_arrows_pin
 |static_stat3_value                      = 1
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
-|static_stat5_id                         = rain_of_arrows_pin
+|static_stat4_id                         = base_skill_effect_duration
+|static_stat4_value                      = 1500
+|static_stat5_id                         = no_movement_speed
 |static_stat5_value                      = 1
 |stat_text                               = Base duration is 1.5 seconds<br>Locks enemy in place
 |level1                                  = True
