--- Skill:WbFrostBlades
+++ Export
@@ -19,6 +19,8 @@
 |stat_text                               = Fires 2 additional Projectiles<br>40% of Physical Damage Converted to Cold Damage<br>+14 to Melee Strike Range
 |level1                                  = True
 |level1_level_requirement                = 1
+|level1_damage_effectiveness             = 100
+|level1_damage_multiplier                = 100
 |level2                                  = True
 |level2_level_requirement                = 4
 |level2_damage_effectiveness             = 101