--- Skill:RockGolemSlamTriggered5
+++ Export
@@ -9,12 +9,12 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = 180
-|static_stat2_id                         = chance_to_taunt_on_hit_%
-|static_stat2_value                      = 33
-|static_stat3_id                         = active_skill_attack_speed_+%_final
-|static_stat3_value                      = -20
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = chance_to_taunt_on_hit_%
+|static_stat3_value                      = 33
+|static_stat4_id                         = active_skill_attack_speed_+%_final
+|static_stat4_value                      = -20
 |stat_text                               = 33% chance to Taunt on Hit<br>20% less Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 1
