--- Skill:GoatmanStormCallBossPrisonersGate
+++ Export
@@ -10,13 +10,14 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = -31
-|static_stat2_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat2_value                      = -25
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat3_value                      = -25
 |stat_text                               = Deals (12-225) to (35-674) Lightning Damage<br>Base duration is (2-1.5) seconds<br>(40-20)% reduced Cast Speed
 |level1                                  = True
 |level1_level_requirement                = 10
+|level1_duration                         = 2
 |level1_stat_text                        = Deals 12 to 35 Lightning Damage<br>Base duration is 2 seconds<br>40% reduced Cast Speed
 |level1_stat1_id                         = spell_minimum_base_lightning_damage
 |level1_stat1_value                      = 12
@@ -28,6 +29,7 @@
 |level1_stat4_value                      = -40
 |level2                                  = True
 |level2_level_requirement                = 43
+|level2_duration                         = 1.75
 |level2_stat_text                        = Deals 105 to 316 Lightning Damage<br>Base duration is 1.75 seconds<br>20% reduced Cast Speed
 |level2_stat1_id                         = spell_minimum_base_lightning_damage
 |level2_stat1_value                      = 105
@@ -39,6 +41,7 @@
 |level2_stat4_value                      = -20
 |level3                                  = True
 |level3_level_requirement                = 57
+|level3_duration                         = 1.5
 |level3_stat_text                        = Deals 225 to 674 Lightning Damage<br>Base duration is 1.5 seconds<br>20% reduced Cast Speed
 |level3_stat1_id                         = spell_minimum_base_lightning_damage
 |level3_stat1_value                      = 225
