--- Skill:ScorpionBurrow
+++ Export
@@ -13,10 +13,10 @@
 |static_stat1_value                      = -70
 |static_stat2_id                         = emerge_speed_+%
 |static_stat2_value                      = 100
-|static_stat3_id                         = active_skill_damage_+%_final
-|static_stat3_value                      = -33
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = active_skill_damage_+%_final
+|static_stat4_value                      = -33
 |stat_text                               = Deals 33% less Damage
 |level1                                  = True
 |level1_level_requirement                = 1
