--- Skill:RigwaldWeaponThrow
+++ Export
@@ -17,6 +17,8 @@
 |stat_text                               = 25% increased Projectile Speed<br>25% increased Accuracy Rating
 |level1                                  = True
 |level1_level_requirement                = 60
+|level1_damage_effectiveness             = 100
+|level1_damage_multiplier                = 100
 |level2                                  = True
 |level2_level_requirement                = 70
 |level2_damage_effectiveness             = 120
