--- Skill:DelveSummonRareUndying
+++ Export
@@ -9,16 +9,16 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = remove_parent_life_on_death_permyriad
-|static_stat1_value                      = 3500
-|static_stat2_id                         = summoned_monster_rarity
-|static_stat2_value                      = 2
-|static_stat3_id                         = summon_specific_monsters_in_front_offset
-|static_stat3_value                      = 198
-|static_stat4_id                         = number_of_monsters_to_summon
-|static_stat4_value                      = 1
-|static_stat5_id                         = alternate_minion
-|static_stat5_value                      = 286
+|static_stat1_id                         = alternate_minion
+|static_stat1_value                      = 286
+|static_stat2_id                         = number_of_monsters_to_summon
+|static_stat2_value                      = 1
+|static_stat3_id                         = remove_parent_life_on_death_permyriad
+|static_stat3_value                      = 3500
+|static_stat4_id                         = summon_specific_monsters_in_front_offset
+|static_stat4_value                      = 198
+|static_stat5_id                         = summoned_monster_rarity
+|static_stat5_value                      = 2
 |static_stat6_id                         = summoned_monsters_are_minions
 |static_stat6_value                      = 1
 |stat_text                               = 
