--- Skill:CGEAncestralChilledGround
+++ Export
@@ -0,0 +1,24 @@
+{{Skill
+|gem_description                         = 
+|active_skill_name                       = 
+|skill_id                                = CGEAncestralChilledGround
+|cast_time                               = 1.0
+|required_level                          = 0
+|static_stored_uses                      = 1
+|static_cooldown                         = 10
+|static_damage_effectiveness             = 100
+|static_damage_multiplier                = 100
+|static_attack_speed_multiplier          = 100
+|static_duration                         = 10
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = -10
+|static_stat2_id                         = active_skill_chill_effect_+%_final
+|static_stat2_value                      = 200
+|static_stat3_id                         = spell_maximum_action_distance_+%
+|static_stat3_value                      = -40
+|static_stat4_id                         = base_skill_effect_duration
+|static_stat4_value                      = 10000
+|stat_text                               = Base duration is 10 seconds
+|level1                                  = True
+|level1_level_requirement                = 0
+}}