--- Skill:MachinariumElementalHitCold
+++ Export
@@ -7,21 +7,21 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 150
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = chance_to_freeze_shock_ignite_%
-|static_stat1_value                      = 25
-|static_stat2_id                         = elemental_hit_cannot_roll_fire_damage
+|static_stat1_id                         = action_attack_or_cast_time_uses_animation_length
+|static_stat1_value                      = 1
+|static_stat2_id                         = is_area_damage
 |static_stat2_value                      = 1
-|static_stat3_id                         = elemental_hit_cannot_roll_lightning_damage
+|static_stat3_id                         = skill_can_fire_arrows
 |static_stat3_value                      = 1
-|static_stat4_id                         = is_area_damage
+|static_stat4_id                         = skill_can_fire_wand_projectiles
 |static_stat4_value                      = 1
-|static_stat5_id                         = skill_can_fire_arrows
+|static_stat5_id                         = elemental_hit_cannot_roll_fire_damage
 |static_stat5_value                      = 1
-|static_stat6_id                         = skill_can_fire_wand_projectiles
+|static_stat6_id                         = elemental_hit_cannot_roll_lightning_damage
 |static_stat6_value                      = 1
-|static_stat7_id                         = action_attack_or_cast_time_uses_animation_length
-|static_stat7_value                      = 1
-|stat_text                               = 25% chance to Freeze, Shock and Ignite<br>Prismatic Skills cannot choose Fire<br>Prismatic Skills cannot choose Lightning
+|static_stat7_id                         = chance_to_freeze_shock_ignite_%
+|static_stat7_value                      = 25
+|stat_text                               = Prismatic Skills cannot choose Fire<br>Prismatic Skills cannot choose Lightning<br>25% chance to Freeze, Shock and Ignite
 |level1                                  = True
 |level1_level_requirement                = 1
 }}