--- Skill:IncaBossRocksTriggered
+++ Export
@@ -7,16 +7,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 0.9
 |static_stat1_id                         = firestorm_base_area_of_effect_+%
 |static_stat1_value                      = -50
-|static_stat2_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat2_value                      = -25
-|static_stat3_id                         = base_skill_effect_duration
-|static_stat3_value                      = 900
-|static_stat4_id                         = fire_storm_fireball_delay_ms
-|static_stat4_value                      = 300
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat3_value                      = -25
+|static_stat4_id                         = base_skill_effect_duration
+|static_stat4_value                      = 900
+|static_stat5_id                         = fire_storm_fireball_delay_ms
+|static_stat5_value                      = 300
 |stat_text                               = Deals (22-295) to (34-442) Physical Damage<br>Base duration is 0.9 seconds<br>One impact every 0.3 seconds
 |level1                                  = True
 |level1_level_requirement                = 23
