--- Skill:SeawitchFrostbolt
+++ Export
@@ -10,9 +10,9 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = always_pierce
+|static_stat1_id                         = base_is_projectile
 |static_stat1_value                      = 1
-|static_stat2_id                         = base_is_projectile
+|static_stat2_id                         = always_pierce
 |static_stat2_value                      = 1
 |stat_text                               = Deals (8-889) to (12-1337) Cold Damage<br>Projectiles Pierce all Targets
 |level1                                  = True
