--- Skill:MonsterLeapSlamCausticCloudPlaguewing
+++ Export
@@ -10,21 +10,22 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 75
-|static_stat1_id                         = leap_slam_base_caustic_cloud_damage_to_deal_per_minute
-|static_stat1_value                      = 35
+|static_duration                         = 9
+|static_stat1_id                         = cast_time_overrides_attack_duration
+|static_stat1_value                      = 1
 |static_stat2_id                         = ground_caustic_art_variation
 |static_stat2_value                      = 2
-|static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat3_value                      = -30
-|static_stat4_id                         = base_skill_effect_duration
-|static_stat4_value                      = 9000
-|static_stat5_id                         = active_skill_damage_+%_final
-|static_stat5_value                      = 50
-|static_stat6_id                         = is_area_damage
-|static_stat6_value                      = 1
-|static_stat7_id                         = cast_time_overrides_attack_duration
-|static_stat7_value                      = 1
-|stat_text                               = Base duration is 9 seconds<br>Deals 50% more Damage
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = leap_slam_base_caustic_cloud_damage_to_deal_per_minute
+|static_stat4_value                      = 35
+|static_stat5_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat5_value                      = -30
+|static_stat6_id                         = active_skill_damage_+%_final
+|static_stat6_value                      = 50
+|static_stat7_id                         = base_skill_effect_duration
+|static_stat7_value                      = 9000
+|stat_text                               = Deals 50% more Damage<br>Base duration is 9 seconds
 |level1                                  = True
 |level1_level_requirement                = 1
 }}