--- Skill:ProphecyVaalSummonZombies
+++ Export
@@ -17,6 +17,8 @@
 |stat_text                               = Maximum (29-50) Summoned Skeletons<br>Base duration is 40 seconds<br>(0-80)% increased Minion Movement Speed<br>Summons (11-19) Skeleton Warriors<br>Summons (1-6) Skeleton Archers<br>Summons (0-3) Skeleton Mages<br>Summons 1 Skeleton General
 |level1                                  = True
 |level1_level_requirement                = 3
+|level1_damage_effectiveness             = 100
+|level1_damage_multiplier                = 100
 |level1_stat_text                        = Maximum 29 Summoned Skeletons<br>Summons 11 Skeleton Warriors<br>Summons 1 Skeleton Archer
 |level1_stat1_id                         = base_number_of_skeletons_allowed
 |level1_stat1_value                      = 29
@@ -26,6 +28,8 @@
 |level1_stat3_value                      = 1
 |level2                                  = True
 |level2_level_requirement                = 5
+|level2_damage_effectiveness             = 100
+|level2_damage_multiplier                = 100
 |level2_stat_text                        = Maximum 30 Summoned Skeletons<br>2% increased Minion Movement Speed<br>Summons 12 Skeleton Warriors<br>Summons 1 Skeleton Archer
 |level2_stat1_id                         = base_number_of_skeletons_allowed
 |level2_stat1_value                      = 30
@@ -37,6 +41,8 @@
 |level2_stat4_value                      = 1
 |level3                                  = True
 |level3_level_requirement                = 8
+|level3_damage_effectiveness             = 100
+|level3_damage_multiplier                = 100
 |level3_stat_text                        = Maximum 31 Summoned Skeletons<br>4% increased Minion Movement Speed<br>Summons 13 Skeleton Warriors<br>Summons 1 Skeleton Archer
 |level3_stat1_id                         = base_number_of_skeletons_allowed
 |level3_stat1_value                      = 31
@@ -48,6 +54,8 @@
 |level3_stat4_value                      = 1
 |level4                                  = True
 |level4_level_requirement                = 10
+|level4_damage_effectiveness             = 100
+|level4_damage_multiplier                = 100
 |level4_stat_text                        = Maximum 32 Summoned Skeletons<br>6% increased Minion Movement Speed<br>Summons 14 Skeleton Warriors<br>Summons 2 Skeleton Archers
 |level4_stat1_id                         = base_number_of_skeletons_allowed
 |level4_stat1_value                      = 32
@@ -59,6 +67,8 @@
 |level4_stat4_value                      = 2
 |level5                                  = True
 |level5_level_requirement                = 13
+|level5_damage_effectiveness             = 100
+|level5_damage_multiplier                = 100
 |level5_stat_text                        = Maximum 33 Summoned Skeletons<br>8% increased Minion Movement Speed<br>Summons 15 Skeleton Warriors<br>Summons 2 Skeleton Archers
 |level5_stat1_id                         = base_number_of_skeletons_allowed
 |level5_stat1_value                      = 33
@@ -70,6 +80,8 @@
 |level5_stat4_value                      = 2
 |level6                                  = True
 |level6_level_requirement                = 16
+|level6_damage_effectiveness             = 100
+|level6_damage_multiplier                = 100
 |level6_stat_text                        = Maximum 34 Summoned Skeletons<br>10% increased Minion Movement Speed<br>Summons 15 Skeleton Warriors<br>Summons 3 Skeleton Archers
 |level6_stat1_id                         = base_number_of_skeletons_allowed
 |level6_stat1_value                      = 34
@@ -81,6 +93,8 @@
 |level6_stat4_value                      = 3
 |level7                                  = True
 |level7_level_requirement                = 19
+|level7_damage_effectiveness             = 100
+|level7_damage_multiplier                = 100
 |level7_stat_text                        = Maximum 35 Summoned Skeletons<br>12% increased Minion Movement Speed<br>Summons 16 Skeleton Warriors<br>Summons 3 Skeleton Archers
 |level7_stat1_id                         = base_number_of_skeletons_allowed
 |level7_stat1_value                      = 35
@@ -92,6 +106,8 @@
 |level7_stat4_value                      = 3
 |level8                                  = True
 |level8_level_requirement                = 22
+|level8_damage_effectiveness             = 100
+|level8_damage_multiplier                = 100
 |level8_stat_text                        = Maximum 36 Summoned Skeletons<br>14% increased Minion Movement Speed<br>Summons 16 Skeleton Warriors<br>Summons 4 Skeleton Archers
 |level8_stat1_id                         = base_number_of_skeletons_allowed
 |level8_stat1_value                      = 36
@@ -103,6 +119,8 @@
 |level8_stat4_value                      = 4
 |level9                                  = True
 |level9_level_requirement                = 25
+|level9_damage_effectiveness             = 100
+|level9_damage_multiplier                = 100
 |level9_stat_text                        = Maximum 37 Summoned Skeletons<br>16% increased Minion Movement Speed<br>Summons 17 Skeleton Warriors<br>Summons 4 Skeleton Archers
 |level9_stat1_id                         = base_number_of_skeletons_allowed
 |level9_stat1_value                      = 37
@@ -114,6 +132,8 @@
 |level9_stat4_value                      = 4
 |level10                                 = True
 |level10_level_requirement               = 28
+|level10_damage_effectiveness            = 100
+|level10_damage_multiplier               = 100
 |level10_stat_text                       = Maximum 38 Summoned Skeletons<br>18% increased Minion Movement Speed<br>Summons 17 Skeleton Warriors<br>Summons 5 Skeleton Archers
 |level10_stat1_id                        = base_number_of_skeletons_allowed
 |level10_stat1_value                     = 38
@@ -125,6 +145,8 @@
 |level10_stat4_value                     = 5
 |level11                                 = True
 |level11_level_requirement               = 32
+|level11_damage_effectiveness            = 100
+|level11_damage_multiplier               = 100
 |level11_stat_text                       = Maximum 39 Summoned Skeletons<br>20% increased Minion Movement Speed<br>Summons 18 Skeleton Warriors<br>Summons 5 Skeleton Archers
 |level11_stat1_id                        = base_number_of_skeletons_allowed
 |level11_stat1_value                     = 39
@@ -136,6 +158,8 @@
 |level11_stat4_value                     = 5
 |level12                                 = True
 |level12_level_requirement               = 36
+|level12_damage_effectiveness            = 100
+|level12_damage_multiplier               = 100
 |level12_stat_text                       = Maximum 40 Summoned Skeletons<br>22% increased Minion Movement Speed<br>Summons 18 Skeleton Warriors<br>Summons 5 Skeleton Archers<br>Summons 1 Skeleton Mage
 |level12_stat1_id                        = base_number_of_skeletons_allowed
 |level12_stat1_value                     = 40
@@ -149,6 +173,8 @@
 |level12_stat5_value                     = 1
 |level13                                 = True
 |level13_level_requirement               = 40
+|level13_damage_effectiveness            = 100
+|level13_damage_multiplier               = 100
 |level13_stat_text                       = Maximum 41 Summoned Skeletons<br>24% increased Minion Movement Speed<br>Summons 18 Skeleton Warriors<br>Summons 6 Skeleton Archers<br>Summons 1 Skeleton Mage
 |level13_stat1_id                        = base_number_of_skeletons_allowed
 |level13_stat1_value                     = 41
@@ -162,6 +188,8 @@
 |level13_stat5_value                     = 1
 |level14                                 = True
 |level14_level_requirement               = 44
+|level14_damage_effectiveness            = 100
+|level14_damage_multiplier               = 100
 |level14_stat_text                       = Maximum 42 Summoned Skeletons<br>26% increased Minion Movement Speed<br>Summons 18 Skeleton Warriors<br>Summons 6 Skeleton Archers<br>Summons 2 Skeleton Mages
 |level14_stat1_id                        = base_number_of_skeletons_allowed
 |level14_stat1_value                     = 42
@@ -175,6 +203,8 @@
 |level14_stat5_value                     = 2
 |level15                                 = True
 |level15_level_requirement               = 47
+|level15_damage_effectiveness            = 100
+|level15_damage_multiplier               = 100
 |level15_stat_text                       = Maximum 43 Summoned Skeletons<br>28% increased Minion Movement Speed<br>Summons 19 Skeleton Warriors<br>Summons 6 Skeleton Archers<br>Summons 2 Skeleton Mages
 |level15_stat1_id                        = base_number_of_skeletons_allowed
 |level15_stat1_value                     = 43
@@ -188,6 +218,8 @@
 |level15_stat5_value                     = 2
 |level16                                 = True
 |level16_level_requirement               = 50
+|level16_damage_effectiveness            = 100
+|level16_damage_multiplier               = 100
 |level16_stat_text                       = Maximum 44 Summoned Skeletons<br>30% increased Minion Movement Speed<br>Summons 19 Skeleton Warriors<br>Summons 6 Skeleton Archers<br>Summons 3 Skeleton Mages
 |level16_stat1_id                        = base_number_of_skeletons_allowed
 |level16_stat1_value                     = 44
@@ -201,6 +233,8 @@
 |level16_stat5_value                     = 3
 |level17                                 = True
 |level17_level_requirement               = 53
+|level17_damage_effectiveness            = 100
+|level17_damage_multiplier               = 100
 |level17_stat_text                       = Maximum 45 Summoned Skeletons<br>32% increased Minion Movement Speed<br>Summons 19 Skeleton Warriors<br>Summons 6 Skeleton Archers<br>Summons 3 Skeleton Mages
 |level17_stat1_id                        = base_number_of_skeletons_allowed
 |level17_stat1_value                     = 45