--- Skill:TormentDaemonPoisonNova
+++ Export
@@ -8,19 +8,19 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_reverse_point_blank_damage_-%_at_minimum_range
-|static_stat1_value                      = 30
-|static_stat2_id                         = monster_projectile_variation
-|static_stat2_value                      = 5
-|static_stat3_id                         = number_of_additional_projectiles
-|static_stat3_value                      = 15
+|static_stat1_id                         = monster_projectile_variation
+|static_stat1_value                      = 5
+|static_stat2_id                         = monster_reverse_point_blank_damage_-%_at_minimum_range
+|static_stat2_value                      = 30
+|static_stat3_id                         = projectiles_nova
+|static_stat3_value                      = 1
 |static_stat4_id                         = base_projectile_speed_+%
 |static_stat4_value                      = -35
-|static_stat5_id                         = global_poison_on_hit
-|static_stat5_value                      = 1
-|static_stat6_id                         = projectiles_nova
+|static_stat5_id                         = number_of_additional_projectiles
+|static_stat5_value                      = 15
+|static_stat6_id                         = global_poison_on_hit
 |static_stat6_value                      = 1
-|stat_text                               = Deals (3-452) to (5-679) Chaos Damage<br>Fires 15 additional Projectiles<br>35% reduced Projectile Speed<br>Poisons Enemies on Hit<br>Fires Projectiles in a circle
+|stat_text                               = Deals (3-452) to (5-679) Chaos Damage<br>Fires Projectiles in a circle<br>35% reduced Projectile Speed<br>Fires 15 additional Projectiles<br>Poisons Enemies on Hit
 |level1                                  = True
 |level1_level_requirement                = 3
 |level1_cooldown                         = 12
