--- Skill:BellyTotemEssenceDrain
+++ Export
@@ -9,17 +9,18 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_chaos_damage_to_deal_per_minute
-|static_stat1_value                      = 173
-|static_stat2_id                         = base_skill_effect_duration
-|static_stat2_value                      = 2000
-|static_stat3_id                         = base_is_projectile
-|static_stat3_value                      = 1
-|static_stat4_id                         = spell_minimum_base_chaos_damage
-|static_stat4_value                      = 1
-|static_stat5_id                         = spell_maximum_base_chaos_damage
-|static_stat5_value                      = 2
-|stat_text                               = Deals 2.9 Base Chaos Damage per second<br>Base duration is 2 seconds<br>Deals 1 to 2 Chaos Damage
+|static_duration                         = 2
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_chaos_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = spell_maximum_base_chaos_damage
+|static_stat3_value                      = 2
+|static_stat4_id                         = base_chaos_damage_to_deal_per_minute
+|static_stat4_value                      = 173
+|static_stat5_id                         = base_skill_effect_duration
+|static_stat5_value                      = 2000
+|stat_text                               = Deals 1 to 2 Chaos Damage<br>Deals 2.9 Base Chaos Damage per second<br>Base duration is 2 seconds
 |level1                                  = True
 |level1_level_requirement                = 1
 }}