--- Skill:BullUpheaval
+++ Export
@@ -9,14 +9,14 @@
 |static_damage_effectiveness             = 80
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_damage_+%_final
-|static_stat1_value                      = -50
-|static_stat2_id                         = active_skill_attack_speed_+%_final
-|static_stat2_value                      = -14
-|static_stat3_id                         = upheaval_number_of_spikes
-|static_stat3_value                      = 12
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = active_skill_damage_+%_final
+|static_stat2_value                      = -50
+|static_stat3_id                         = active_skill_attack_speed_+%_final
+|static_stat3_value                      = -14
+|static_stat4_id                         = upheaval_number_of_spikes
+|static_stat4_value                      = 12
 |stat_text                               = Deals 50% less Damage<br>14% less Attack Speed<br>Causes 12 Bursts
 |level1                                  = True
 |level1_level_requirement                = 0
