--- Skill:DancingDervishCycloneSlow
+++ Export
@@ -10,13 +10,13 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = cyclone_extra_distance
 |static_stat1_value                      = 25
-|static_stat2_id                         = cyclone_movement_speed_+%_final
-|static_stat2_value                      = -50
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
 |static_stat3_id                         = attack_speed_+%
 |static_stat3_value                      = 20
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
-|stat_text                               = 50% less Movement Speed<br>20% increased Attack Speed
+|static_stat4_id                         = cyclone_movement_speed_+%_final
+|static_stat4_value                      = -50
+|stat_text                               = 20% increased Attack Speed<br>50% less Movement Speed
 |level1                                  = True
 |level1_level_requirement                = 1
 }}