--- Skill:MonsterBarrageBossBlackwood
+++ Export
@@ -1,5 +1,5 @@
 {{Skill
-|gem_description                         = After a short preparation time, you attack repeatedly with a ranged weapon. These attacks have a small randomised spread. Only works with Bows and Wands.
+|gem_description                         = After a short preparation time, you attack repeatedly with a ranged weapon. These attacks have a small randomised spread. Only works with Bows and Wands. This skill cannot be Triggered.
 |active_skill_name                       = Barrage
 |item_class_id_restriction               = Bow, Wand
 |skill_id                                = MonsterBarrageBossBlackwood
@@ -12,19 +12,19 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = monster_reverse_point_blank_damage_-%_at_minimum_range
 |static_stat1_value                      = 30
-|static_stat2_id                         = active_skill_damage_+%_final
-|static_stat2_value                      = -60
-|static_stat3_id                         = projectiles_barrage
+|static_stat2_id                         = projectile_uses_contact_position
+|static_stat2_value                      = 1
+|static_stat3_id                         = skill_can_fire_arrows
 |static_stat3_value                      = 1
-|static_stat4_id                         = skill_can_fire_arrows
+|static_stat4_id                         = skill_can_fire_wand_projectiles
 |static_stat4_value                      = 1
-|static_stat5_id                         = skill_can_fire_wand_projectiles
+|static_stat5_id                         = use_scaled_contact_offset
 |static_stat5_value                      = 1
-|static_stat6_id                         = use_scaled_contact_offset
-|static_stat6_value                      = 1
-|static_stat7_id                         = projectile_uses_contact_position
+|static_stat6_id                         = active_skill_damage_+%_final
+|static_stat6_value                      = -60
+|static_stat7_id                         = projectiles_barrage
 |static_stat7_value                      = 1
-|stat_text                               = Fires (3-5) additional Projectiles<br>(0-30)% more Attack Speed<br>Deals 60% less Damage<br>Fires Projectiles sequentially
+|stat_text                               = Deals 60% less Damage<br>Fires Projectiles sequentially<br>Fires (3-5) additional Projectiles<br>(0-30)% more Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 15
 |level1_stat_text                        = Fires 3 additional Projectiles
