--- Skill:GroundEffectsSlamDockworkerChampion
+++ Export
@@ -9,17 +9,18 @@
 |static_damage_effectiveness             = 250
 |static_damage_multiplier                = 150
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_cold_damage_to_deal_per_minute
-|static_stat1_value                      = 618
-|static_stat2_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat2_value                      = -50
+|static_duration                         = 4
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = -50
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
 |static_stat3_id                         = base_skill_effect_duration
 |static_stat3_value                      = 4000
-|static_stat4_id                         = active_skill_attack_speed_+%_final
-|static_stat4_value                      = -40
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
-|stat_text                               = Deals 10.3 Base Cold Damage per second<br>Base duration is 4 seconds<br>40% less Attack Speed
+|static_stat4_id                         = base_cold_damage_to_deal_per_minute
+|static_stat4_value                      = 618
+|static_stat5_id                         = active_skill_attack_speed_+%_final
+|static_stat5_value                      = -40
+|stat_text                               = Base duration is 4 seconds<br>Deals 10.3 Base Cold Damage per second<br>40% less Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 1
 }}