--- Skill:VastiriBossSpectralThrow1
+++ Export
@@ -8,12 +8,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spectral_throw_deceleration_override
-|static_stat1_value                      = 55
-|static_stat2_id                         = active_skill_damage_+%_final
-|static_stat2_value                      = -52
-|static_stat3_id                         = base_is_projectile
-|static_stat3_value                      = 1
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = spectral_throw_deceleration_override
+|static_stat2_value                      = 55
+|static_stat3_id                         = active_skill_damage_+%_final
+|static_stat3_value                      = -52
 |stat_text                               = Deals 52% less Damage
 |level1                                  = True
 |level1_level_requirement                = 1
