--- Skill:GladiatorBossCausticArrowTrap
+++ Export
@@ -36,6 +36,8 @@
 |stat_text                               = Trap lasts 30 seconds<br>Deals (2-198.8) Base Chaos Damage per second<br>Base duration is 4 seconds<br>Modifiers to Projectile Damage apply to this Skill's Damage Over Time effect<br>Gain 25% of your Physical Damage as Extra Chaos Damage
 |level1                                  = True
 |level1_level_requirement                = 3
+|level1_damage_effectiveness             = 100
+|level1_damage_multiplier                = 100
 |level1_stat_text                        = Deals 2 Base Chaos Damage per second
 |level1_stat1_id                         = base_chaos_damage_to_deal_per_minute
 |level1_stat1_value                      = 120
