--- Skill:DarkExileExplosiveArrow
+++ Export
@@ -31,7 +31,7 @@
 |static_stat9_value                      = 3
 |static_stat10_id                        = active_skill_attack_speed_+%_final
 |static_stat10_value                     = 150
-|stat_text                               = Skills Repeat an additional 6 Times<br>Deals 10% less Attack Damage<br>Base duration is 2 seconds<br>Fires 5 additional Projectiles<br>Explosion deals 653 to 980 Base Fire Damage per Fuse Charge<br>+3 to Explosion Radius per Explosive Arrow on Target<br>150% more Attack Speed
+|stat_text                               = Skills Repeat an additional 6 Times<br>Deals 10% less Attack Damage<br>Base duration is 2 seconds<br>Fires 5 additional Projectiles<br>Explosion deals 653 to 980 Base Fire Damage per Fuse Charge<br>+0.3 metres to Explosion Radius per Explosive Arrow on Target<br>150% more Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 73
 }}