--- Skill:DelveGolemProjectileGeodite2
+++ Export
@@ -8,16 +8,16 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 8
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 12
-|static_stat3_id                         = monster_projectile_variation
-|static_stat3_value                      = 71
-|static_stat4_id                         = base_is_projectile
-|static_stat4_value                      = 1
-|static_stat5_id                         = projectile_uses_contact_position
-|static_stat5_value                      = 1
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_projectile_variation
+|static_stat2_value                      = 71
+|static_stat3_id                         = projectile_uses_contact_position
+|static_stat3_value                      = 1
+|static_stat4_id                         = spell_minimum_base_physical_damage
+|static_stat4_value                      = 8
+|static_stat5_id                         = spell_maximum_base_physical_damage
+|static_stat5_value                      = 12
 |stat_text                               = Deals 8 to 12 Physical Damage
 |level1                                  = True
 |level1_level_requirement                = 1
