--- Skill:SkeletonProjectileFireInvasion
+++ Export
@@ -8,12 +8,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_maximum_action_distance_+%
-|static_stat1_value                      = -60
-|static_stat2_id                         = base_is_projectile
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = projectile_uses_contact_position
 |static_stat2_value                      = 1
-|static_stat3_id                         = projectile_uses_contact_position
-|static_stat3_value                      = 1
+|static_stat3_id                         = spell_maximum_action_distance_+%
+|static_stat3_value                      = -60
 |static_stat4_id                         = use_scaled_contact_offset
 |static_stat4_value                      = 1
 |stat_text                               = Deals (4-504) to (6-756) Fire Damage<br>(10-51)% increased Cast Speed<br>Fires (0-10) additional Projectiles
