--- Skill:AfflictionMonsterFireSlash
+++ Export
@@ -7,15 +7,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 150
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = skill_physical_damage_%_to_convert_to_fire
-|static_stat1_value                      = 75
-|static_stat2_id                         = always_ignite
+|static_stat1_id                         = cast_time_overrides_attack_duration
+|static_stat1_value                      = 1
+|static_stat2_id                         = is_area_damage
 |static_stat2_value                      = 1
-|static_stat3_id                         = is_area_damage
+|static_stat3_id                         = always_ignite
 |static_stat3_value                      = 1
-|static_stat4_id                         = cast_time_overrides_attack_duration
-|static_stat4_value                      = 1
-|stat_text                               = Deals (0-200)% more Damage<br>(0-400)% more Damage with Ignite<br>75% of Physical Damage Converted to Fire Damage<br>Hits always Ignite
+|static_stat4_id                         = skill_physical_damage_%_to_convert_to_fire
+|static_stat4_value                      = 75
+|stat_text                               = Hits always Ignite<br>Deals (0-200)% more Damage<br>75% of Physical Damage Converted to Fire Damage<br>(0-400)% more Damage with Ignite
 |level1                                  = True
 |level1_level_requirement                = 1
 |level2                                  = True
