--- Skill:VikingUpheaval
+++ Export
@@ -9,15 +9,16 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
 |static_stat1_value                      = -25
-|static_stat2_id                         = skill_physical_damage_%_to_convert_to_fire
-|static_stat2_value                      = 50
-|static_stat3_id                         = upheaval_number_of_spikes
-|static_stat3_value                      = 10
-|static_stat4_id                         = global_cannot_crit
-|static_stat4_value                      = 1
-|stat_text                               = Deals (2.7-320.6) Base Fire Damage per second<br>Base duration is (5-20) seconds<br>50% of Physical Damage Converted to Fire Damage<br>Causes 10 Bursts<br>Never deal Critical Strikes
+|static_stat2_id                         = global_cannot_crit
+|static_stat2_value                      = 1
+|static_stat3_id                         = skill_physical_damage_%_to_convert_to_fire
+|static_stat3_value                      = 50
+|static_stat4_id                         = upheaval_number_of_spikes
+|static_stat4_value                      = 10
+|stat_text                               = Never deal Critical Strikes<br>Deals (2.7-320.6) Base Fire Damage per second<br>Base duration is (5-20) seconds<br>50% of Physical Damage Converted to Fire Damage<br>Causes 10 Bursts
 |level1                                  = True
 |level1_level_requirement                = 4
+|level1_duration                         = 5
 |level1_stat_text                        = Deals 2.7 Base Fire Damage per second<br>Base duration is 5 seconds
 |level1_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level1_stat1_value                      = 160
@@ -25,6 +26,7 @@
 |level1_stat2_value                      = 5000
 |level2                                  = True
 |level2_level_requirement                = 50
+|level2_duration                         = 8
 |level2_stat_text                        = Deals 108.9 Base Fire Damage per second<br>Base duration is 8 seconds
 |level2_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level2_stat1_value                      = 6534
@@ -32,6 +34,7 @@
 |level2_stat2_value                      = 8000
 |level3                                  = True
 |level3_level_requirement                = 67
+|level3_duration                         = 20
 |level3_stat_text                        = Deals 320.6 Base Fire Damage per second<br>Base duration is 20 seconds
 |level3_stat1_id                         = base_fire_damage_to_deal_per_minute
 |level3_stat1_value                      = 19234
