--- Skill:GASanctumSentinelMaceHeavySlam
+++ Export
@@ -0,0 +1,29 @@
+{{Skill
+|gem_description                         = 
+|active_skill_name                       = 
+|skill_id                                = GASanctumSentinelMaceHeavySlam
+|cast_time                               = 4.0
+|required_level                          = 0
+|static_stored_uses                      = 1
+|static_cooldown                         = 12
+|static_damage_effectiveness             = 100
+|static_damage_multiplier                = 150
+|static_attack_speed_multiplier          = 75
+|static_stat1_id                         = action_attack_or_cast_time_uses_animation_length
+|static_stat1_value                      = 1
+|static_stat2_id                         = base_knockback_speed_+%
+|static_stat2_value                      = 100
+|static_stat3_id                         = base_sanctum_damage
+|static_stat3_value                      = 60
+|static_stat4_id                         = is_area_damage
+|static_stat4_value                      = 1
+|static_stat5_id                         = global_knockback
+|static_stat5_value                      = 1
+|static_stat6_id                         = always_stun
+|static_stat6_value                      = 1
+|static_stat7_id                         = melee_range_+
+|static_stat7_value                      = 20
+|stat_text                               = Knocks Enemies Back on Hit<br>Stuns Enemies<br>+20 to Melee Strike Range
+|level1                                  = True
+|level1_level_requirement                = 0
+}}