--- Skill:UltimatumGuardProjectileImpactRuin
+++ Export
@@ -9,13 +9,13 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = -25
-|static_stat2_id                         = skill_physical_damage_%_to_convert_to_lightning
-|static_stat2_value                      = 60
+|static_stat2_id                         = apply_ultimatum_ruin_on_hit
+|static_stat2_value                      = 1
 |static_stat3_id                         = is_area_damage
 |static_stat3_value                      = 1
-|static_stat4_id                         = apply_ultimatum_ruin_on_hit
-|static_stat4_value                      = 1
-|stat_text                               = Deals (30-200)% more Damage<br>60% of Physical Damage Converted to Lightning Damage
+|static_stat4_id                         = skill_physical_damage_%_to_convert_to_lightning
+|static_stat4_value                      = 60
+|stat_text                               = 60% of Physical Damage Converted to Lightning Damage<br>Deals (30-200)% more Damage
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_stat_text                        = Deals 30% more Damage
