--- Skill:AxisOrbOfStorms
+++ Export
@@ -9,15 +9,16 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = storm_cloud_charged_damage_+%_final
-|static_stat1_value                      = 10
-|static_stat2_id                         = orb_of_storms_base_bolt_frequency_ms
-|static_stat2_value                      = 1000
+|static_duration                         = 6
+|static_stat1_id                         = storm_cloud_destroy_when_caster_dies
+|static_stat1_value                      = 1
+|static_stat2_id                         = storm_cloud_charged_damage_+%_final
+|static_stat2_value                      = 10
 |static_stat3_id                         = base_skill_effect_duration
 |static_stat3_value                      = 6000
-|static_stat4_id                         = storm_cloud_destroy_when_caster_dies
-|static_stat4_value                      = 1
-|stat_text                               = Deals (4-337) to (12-1011) Lightning Damage<br>Beams Split towards (3-8) additional targets<br>Charged Storm Cloud deals 10 more Damage<br>Strikes every 1 second, and when you use a Lightning Skill near the Orb<br>Base duration is 6 seconds
+|static_stat4_id                         = orb_of_storms_base_bolt_frequency_ms
+|static_stat4_value                      = 1000
+|stat_text                               = Deals (4-337) to (12-1011) Lightning Damage<br>Charged Storm Cloud deals 10 more Damage<br>Base duration is 6 seconds<br>Strikes every 1 second, and when you use a Lightning Skill near the Orb<br>Beams Split towards (3-8) additional targets
 |level1                                  = True
 |level1_level_requirement                = 4
 |level1_stat_text                        = Deals 4 to 12 Lightning Damage<br>Beams Split towards 3 additional targets
