--- Skill:IncursionChaosGroundSlam
+++ Export
@@ -9,13 +9,13 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 150
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_attack_speed_+%_final
-|static_stat1_value                      = -10
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
 |static_stat2_id                         = skill_physical_damage_%_to_convert_to_chaos
 |static_stat2_value                      = 25
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
-|stat_text                               = 10% less Attack Speed<br>25% of Physical Damage Converted to Chaos Damage
+|static_stat3_id                         = active_skill_attack_speed_+%_final
+|static_stat3_value                      = -10
+|stat_text                               = 25% of Physical Damage Converted to Chaos Damage<br>10% less Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 0
 }}