--- Skill:HeistTibbsMelee1
+++ Export
@@ -11,19 +11,19 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = base_knockback_speed_+%
 |static_stat1_value                      = 300
-|static_stat2_id                         = knockback_distance_+%
-|static_stat2_value                      = 100
-|static_stat3_id                         = global_knockback
+|static_stat2_id                         = cast_time_overrides_attack_duration
+|static_stat2_value                      = 1
+|static_stat3_id                         = determine_knockback_direction_from_melee_pattern
 |static_stat3_value                      = 1
 |static_stat4_id                         = skill_can_fire_arrows
 |static_stat4_value                      = 1
 |static_stat5_id                         = skill_can_fire_wand_projectiles
 |static_stat5_value                      = 1
-|static_stat6_id                         = determine_knockback_direction_from_melee_pattern
+|static_stat6_id                         = global_knockback
 |static_stat6_value                      = 1
-|static_stat7_id                         = cast_time_overrides_attack_duration
-|static_stat7_value                      = 1
-|stat_text                               = Deals (-50-400)% more Damage<br>100% increased Knockback Distance<br>Knocks Enemies Back on Hit
+|static_stat7_id                         = knockback_distance_+%
+|static_stat7_value                      = 100
+|stat_text                               = Knocks Enemies Back on Hit<br>Deals (-50-400)% more Damage<br>100% increased Knockback Distance
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_stat_text                        = Deals 50% less Damage
