--- Skill:VilentaSunderMap
+++ Export
@@ -11,12 +11,12 @@
 |static_stat1_value                      = -50
 |static_stat2_id                         = firestorm_base_area_of_effect_+%
 |static_stat2_value                      = 80
-|static_stat3_id                         = active_skill_attack_speed_+%_final
-|static_stat3_value                      = 5
-|static_stat4_id                         = shockwave_slam_explosion_damage_+%_final
-|static_stat4_value                      = -85
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = active_skill_attack_speed_+%_final
+|static_stat4_value                      = 5
+|static_stat5_id                         = shockwave_slam_explosion_damage_+%_final
+|static_stat5_value                      = -85
 |stat_text                               = Deals (67-0)% less Damage<br>5% more Attack Speed<br>Shockwaves deal 85% less Damage
 |level1                                  = True
 |level1_level_requirement                = 68
