--- Skill:AbyssalCryTriggered
+++ Export
@@ -11,23 +11,24 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_skill_effect_duration
-|static_stat1_value                      = 6000
-|static_stat2_id                         = abyssal_cry_%_max_life_as_chaos_on_death
-|static_stat2_value                      = 8
-|static_stat3_id                         = base_movement_velocity_+%
-|static_stat3_value                      = -20
-|static_stat4_id                         = abyssal_cry_movement_velocity_+%_per_one_hundred_nearby_enemies
-|static_stat4_value                      = -60
+|static_duration                         = 6
+|static_stat1_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat1_value                      = 1
+|static_stat2_id                         = display_skill_deals_secondary_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = base_skill_effect_duration
+|static_stat3_value                      = 6000
+|static_stat4_id                         = base_movement_velocity_+%
+|static_stat4_value                      = -20
 |static_stat5_id                         = damage_cannot_be_reflected
 |static_stat5_value                      = 1
-|static_stat6_id                         = skill_has_trigger_from_unique_item
-|static_stat6_value                      = 1
-|static_stat7_id                         = base_skill_show_average_damage_instead_of_dps
-|static_stat7_value                      = 1
-|static_stat8_id                         = display_skill_deals_secondary_damage
+|static_stat6_id                         = abyssal_cry_movement_velocity_+%_per_one_hundred_nearby_enemies
+|static_stat6_value                      = -60
+|static_stat7_id                         = abyssal_cry_%_max_life_as_chaos_on_death
+|static_stat7_value                      = 8
+|static_stat8_id                         = skill_has_trigger_from_unique_item
 |static_stat8_value                      = 1
-|stat_text                               = Base duration is 6 seconds<br>Explosion deals Chaos Damage equal to 8% of the Monster's maximum Life<br>20% reduced Movement Speed<br>0.6% reduced Movement Speed per Nearby Enemy<br>Damage cannot be Reflected<br>This Skill is Triggered by a Unique Item
+|stat_text                               = Base duration is 6 seconds<br>20% reduced Movement Speed<br>Damage cannot be Reflected<br>0.6% reduced Movement Speed per Nearby Enemy<br>Explosion deals Chaos Damage equal to 8% of the Monster's maximum Life<br>This Skill is Triggered by a Unique Item
 |level1                                  = True
 |level1_level_requirement                = 34
 }}