--- Skill:SummonSilverOrbOnDeath
+++ Export
@@ -7,21 +7,21 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = summon_specific_monsters_radius_+%
-|static_stat1_value                      = -95
-|static_stat2_id                         = alternate_minion
-|static_stat2_value                      = 244
+|static_stat1_id                         = alternate_minion
+|static_stat1_value                      = 244
+|static_stat2_id                         = monster_no_drops_or_experience
+|static_stat2_value                      = 1
 |static_stat3_id                         = number_of_monsters_to_summon
 |static_stat3_value                      = 1
-|static_stat4_id                         = cast_on_death_%
-|static_stat4_value                      = 25
-|static_stat5_id                         = spell_uncastable_if_triggerable
+|static_stat4_id                         = summon_specific_monsters_radius_+%
+|static_stat4_value                      = -95
+|static_stat5_id                         = summoned_monsters_are_minions
 |static_stat5_value                      = 1
-|static_stat6_id                         = summoned_monsters_are_minions
+|static_stat6_id                         = spell_uncastable_if_triggerable
 |static_stat6_value                      = 1
-|static_stat7_id                         = monster_no_drops_or_experience
-|static_stat7_value                      = 1
-|stat_text                               = 25% chance to Trigger this Spell on Death<br>You cannot Cast this Spell directly
+|static_stat7_id                         = cast_on_death_%
+|static_stat7_value                      = 25
+|stat_text                               = You cannot Cast this Spell directly<br>25% chance to Trigger this Spell on Death
 |level1                                  = True
 |level1_level_requirement                = 1
 }}