--- Skill:GAOlrothGroundSlamInitial
+++ Export
@@ -9,25 +9,23 @@
 |static_damage_effectiveness             = 604
 |static_damage_multiplier                = 604
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat1_value                      = -50
-|static_stat2_id                         = reduce_enemy_dodge_%
-|static_stat2_value                      = 50
-|static_stat3_id                         = global_reduce_enemy_block_%
-|static_stat3_value                      = 50
-|static_stat4_id                         = melee_range_+
-|static_stat4_value                      = 12
-|static_stat5_id                         = is_area_damage
+|static_stat1_id                         = action_attack_or_cast_time_uses_animation_length
+|static_stat1_value                      = 1
+|static_stat2_id                         = cannot_be_blocked_or_dodged_or_suppressed
+|static_stat2_value                      = 1
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat4_value                      = -50
+|static_stat5_id                         = skill_cannot_be_interrupted
 |static_stat5_value                      = 1
-|static_stat6_id                         = action_attack_or_cast_time_uses_animation_length
+|static_stat6_id                         = skill_cannot_be_knocked_back
 |static_stat6_value                      = 1
-|static_stat7_id                         = skill_cannot_be_interrupted
+|static_stat7_id                         = skill_cannot_be_stunned
 |static_stat7_value                      = 1
-|static_stat8_id                         = skill_cannot_be_stunned
-|static_stat8_value                      = 1
-|static_stat9_id                         = skill_cannot_be_knocked_back
-|static_stat9_value                      = 1
-|stat_text                               = Enemies have 50% reduced chance to Dodge Hits from this Skill<br>Enemy Block Chance reduced by 50% against this Skill<br>+12 to Melee Strike Range
+|static_stat8_id                         = melee_range_+
+|static_stat8_value                      = 12
+|stat_text                               = +12 to Melee Strike Range
 |level1                                  = True
 |level1_level_requirement                = 0
 }}