--- Skill:SpiderPoisonProjectileChampion
+++ Export
@@ -9,12 +9,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_chaos_damage
-|static_stat1_value                      = 2
-|static_stat2_id                         = spell_maximum_base_chaos_damage
-|static_stat2_value                      = 3
-|static_stat3_id                         = monster_projectile_variation
-|static_stat3_value                      = 5
+|static_stat1_id                         = monster_projectile_variation
+|static_stat1_value                      = 5
+|static_stat2_id                         = spell_minimum_base_chaos_damage
+|static_stat2_value                      = 2
+|static_stat3_id                         = spell_maximum_base_chaos_damage
+|static_stat3_value                      = 3
 |static_stat4_id                         = base_projectile_speed_+%
 |static_stat4_value                      = -20
 |static_stat5_id                         = global_poison_on_hit
