--- Skill:RampageDaemonAnimateWeapon4
+++ Export
@@ -7,19 +7,20 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = number_of_animated_two_handed_swords_to_summon
-|static_stat1_value                      = 5
+|static_duration                         = 37.5
+|static_stat1_id                         = emerge_speed_+%
+|static_stat1_value                      = 100
 |static_stat2_id                         = number_of_animated_bows_to_summon
 |static_stat2_value                      = 5
 |static_stat3_id                         = number_of_animated_daggers_to_summon
 |static_stat3_value                      = 5
-|static_stat4_id                         = emerge_speed_+%
-|static_stat4_value                      = 100
-|static_stat5_id                         = active_skill_attack_speed_+%_final
-|static_stat5_value                      = 100
-|static_stat6_id                         = base_skill_effect_duration
-|static_stat6_value                      = 37500
-|stat_text                               = 100% more Attack Speed<br>Base duration is 37.5 seconds
+|static_stat4_id                         = number_of_animated_two_handed_swords_to_summon
+|static_stat4_value                      = 5
+|static_stat5_id                         = base_skill_effect_duration
+|static_stat5_value                      = 37500
+|static_stat6_id                         = active_skill_attack_speed_+%_final
+|static_stat6_value                      = 100
+|stat_text                               = Base duration is 37.5 seconds<br>100% more Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 1
 }}