--- Skill:TriggerSummonedAncestorRakiata
+++ Export
@@ -0,0 +1,32 @@
+{{Skill
+|gem_description                         = Summons an immortal chieftain minion of the Tasalio tribe to fight alongside you for a duration. This minion grants you a buff which prevents your action speed being lowered. Shares a Cooldown with other Spirit Chieftain Skills.
+|active_skill_name                       = Summon Spirit of Rakiata
+|skill_id                                = TriggerSummonedAncestorRakiata
+|cast_time                               = 1.0
+|required_level                          = 0
+|static_stored_uses                      = 1
+|static_damage_effectiveness             = 100
+|static_damage_multiplier                = 100
+|static_attack_speed_multiplier          = 100
+|static_duration                         = 12
+|static_stat1_id                         = base_skill_effect_duration
+|static_stat1_value                      = 12000
+|static_stat2_id                         = ancestral_buff_action_speed_+%_minimum_value
+|static_stat2_value                      = 10
+|static_stat3_id                         = ancestral_embrace_effect_+%_per_tasalio_tattoo
+|static_stat3_value                      = 20
+|static_stat4_id                         = display_this_skill_cooldown_does_not_recover_during_buff
+|static_stat4_value                      = 1
+|static_stat5_id                         = minion_actor_level_is_user_level_up_to_maximum
+|static_stat5_value                      = 85
+|stat_text                               = Base duration is 12 seconds<br>(0-10)% chance to Trigger this Spell on Kill<br>Buff causes your Action Speed to always be at least 110% of base value<br>20% increased Buff Effect per Allocated Tasalio Tattoo<br>This Skill's Cooldown does not recover during its effect<br>Minions' Level is equal to yours, up to a maximum of 85
+|level1                                  = True
+|level1_level_requirement                = 0
+|level1_cooldown                         = 80
+|level2                                  = True
+|level2_level_requirement                = 0
+|level2_cooldown                         = 20
+|level2_stat_text                        = 10% chance to Trigger this Spell on Kill
+|level2_stat1_id                         = chance_to_cast_on_kill_%
+|level2_stat1_value                      = 10
+}}