--- Skill:HellionCausticCloudGroundEffect
+++ Export
@@ -7,13 +7,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 7.5
 |static_stat1_id                         = base_chaos_damage_to_deal_per_minute
 |static_stat1_value                      = 110
-|static_stat2_id                         = cast_on_death_%
-|static_stat2_value                      = 100
-|static_stat3_id                         = base_skill_effect_duration
-|static_stat3_value                      = 7500
-|stat_text                               = Deals 1.8 Base Chaos Damage per second<br>100% chance to Trigger this Spell on Death<br>Base duration is 7.5 seconds
+|static_stat2_id                         = base_skill_effect_duration
+|static_stat2_value                      = 7500
+|static_stat3_id                         = cast_on_death_%
+|static_stat3_value                      = 100
+|stat_text                               = Deals 1.8 Base Chaos Damage per second<br>Base duration is 7.5 seconds<br>100% chance to Trigger this Spell on Death
 |level1                                  = True
 |level1_level_requirement                = 3
 }}