--- Skill:GeneralsCrySupport
+++ Export
@@ -1,20 +1,21 @@
 {{Skill
 |skill_id                                = GeneralsCrySupport
 |required_level                          = 24
+|static_cost_multiplier                  = 100
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = number_of_warcries_exerting_this_action
+|static_stat1_id                         = base_damage_not_from_skill_user
 |static_stat1_value                      = 1
-|static_stat2_id                         = triggered_by_spiritual_cry
+|static_stat2_id                         = force_lite_skill_effects
 |static_stat2_value                      = 1
 |static_stat3_id                         = no_spirit_strikes
 |static_stat3_value                      = 1
-|static_stat4_id                         = force_lite_skill_effects
+|static_stat4_id                         = number_of_warcries_exerting_this_action
 |static_stat4_value                      = 1
-|static_stat5_id                         = base_damage_not_from_skill_user
+|static_stat5_id                         = triggered_by_spiritual_cry
 |static_stat5_value                      = 1
-|stat_text                               = Supported Skills deal (45-28)% less Damage<br>Attacks with Supported Skills count as Exerted<br>Supported Skills are used by Mirage Warriors Summoned by General's Cry<br>Supported Skills count as Triggered<br>Supported Strike Skills cannot target additional Enemies
+|stat_text                               = Supported Strike Skills cannot target additional Enemies<br>Attacks with Supported Skills count as Exerted<br>Supported Skills deal (45-28)% less Damage<br>Supported Skills are used by Mirage Warriors Summoned by General's Cry<br>Supported Skills count as Triggered
 |level1                                  = True
 |level1_level_requirement                = 24
 |level1_stat_text                        = Supported Skills deal 45% less Damage
