--- Skill:VaalGuardShrapnelShot
+++ Export
@@ -8,21 +8,21 @@
 |static_damage_effectiveness             = 65
 |static_damage_multiplier                = 80
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = number_of_additional_projectiles
-|static_stat1_value                      = 2
-|static_stat2_id                         = skill_physical_damage_%_to_convert_to_lightning
-|static_stat2_value                      = 50
-|static_stat3_id                         = base_arrows_always_pierce
+|static_stat1_id                         = action_attack_or_cast_time_uses_animation_length
+|static_stat1_value                      = 1
+|static_stat2_id                         = base_is_projectile
+|static_stat2_value                      = 1
+|static_stat3_id                         = skill_can_fire_arrows
 |static_stat3_value                      = 1
-|static_stat4_id                         = additive_arrow_speed_modifiers_apply_to_area_of_effect
+|static_stat4_id                         = base_arrows_always_pierce
 |static_stat4_value                      = 1
-|static_stat5_id                         = base_is_projectile
-|static_stat5_value                      = 1
-|static_stat6_id                         = skill_can_fire_arrows
-|static_stat6_value                      = 1
-|static_stat7_id                         = action_attack_or_cast_time_uses_animation_length
+|static_stat5_id                         = skill_physical_damage_%_to_convert_to_lightning
+|static_stat5_value                      = 50
+|static_stat6_id                         = number_of_additional_projectiles
+|static_stat6_value                      = 2
+|static_stat7_id                         = additive_arrow_speed_modifiers_apply_to_area_of_effect
 |static_stat7_value                      = 1
-|stat_text                               = Fires 2 additional Projectiles<br>50% of Physical Damage Converted to Lightning Damage<br>Arrows Pierce all Targets<br>Increases and Reductions to Projectile Speed also apply to<br>this Skill's Area of Effect
+|stat_text                               = Arrows Pierce all Targets<br>50% of Physical Damage Converted to Lightning Damage<br>Fires 2 additional Projectiles<br>Increases and Reductions to Projectile Speed also apply to<br>this Skill's Area of Effect
 |level1                                  = True
 |level1_level_requirement                = 1
 }}