--- Skill:SummonBeastialSnake
+++ Export
@@ -1,5 +1,5 @@
 {{Skill
-|gem_description                         = Summons a Bestial Snake. The Bestial Snake fires powerful chaos projectile attacks. This minion's hits cannot be evaded.
+|gem_description                         = Summons a Bestial Snake. The Bestial Snake fires powerful chaos projectile attacks. This minion's hits cannot be evaded. Enemies will not engage this minion directly, and they are immune to damage.
 |active_skill_name                       = Summon Bestial Snake
 |skill_id                                = SummonBeastialSnake
 |cast_time                               = 1.0
@@ -13,14 +13,16 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = alternate_minion
 |static_stat1_value                      = 2
-|static_stat2_id                         = display_minion_monster_level
+|static_stat2_id                         = base_display_minion_actor_level
 |static_stat2_value                      = 70
 |static_stat3_id                         = display_minion_monster_type
 |static_stat3_value                      = 14
-|static_stat4_id                         = minion_maximum_life_+%
-|static_stat4_value                      = 30
-|static_stat5_id                         = base_number_of_beast_minions_allowed
-|static_stat5_value                      = 1
+|static_stat4_id                         = infinite_minion_duration
+|static_stat4_value                      = 1
+|static_stat5_id                         = minion_maximum_life_+%
+|static_stat5_value                      = 30
+|static_stat6_id                         = base_number_of_beast_minions_allowed
+|static_stat6_value                      = 1
 |stat_text                               = 30% increased Minion Maximum Life<br>Maximum 1 Summoned Bestial Minion
 |level1                                  = True
 |level1_level_requirement                = 70
