--- Skill:HandfingerThrownWeaponBossNotFar
+++ Export
@@ -8,15 +8,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spectral_throw_deceleration_override
-|static_stat1_value                      = 135
-|static_stat2_id                         = base_projectile_speed_+%
-|static_stat2_value                      = 50
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = spectral_throw_deceleration_override
+|static_stat2_value                      = 135
 |static_stat3_id                         = active_skill_damage_+%_final
 |static_stat3_value                      = -30
-|static_stat4_id                         = base_is_projectile
-|static_stat4_value                      = 1
-|stat_text                               = 50% increased Projectile Speed<br>Deals 30% less Damage
+|static_stat4_id                         = base_projectile_speed_+%
+|static_stat4_value                      = 50
+|stat_text                               = Deals 30% less Damage<br>50% increased Projectile Speed
 |level1                                  = True
 |level1_level_requirement                = 1
 }}