--- Skill:SlavedriverFlameWhip
+++ Export
@@ -1,6 +1,6 @@
 {{Skill
 |gem_description                         = Strikes enemies in front of you with a surge of flame. Burning enemies are dealt more damage. If you hit an ignited enemy, will create burning ground under them. Your damage modifiers don't apply to this burning ground.
-|active_skill_name                       = 
+|active_skill_name                       = Flame Surge
 |skill_id                                = SlavedriverFlameWhip
 |cast_time                               = 0.5
 |required_level                          = 1
@@ -8,16 +8,16 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage
-|static_stat1_value                      = 5
-|static_stat2_id                         = spell_maximum_base_lightning_damage
-|static_stat2_value                      = 15
-|static_stat3_id                         = base_cast_speed_+%
-|static_stat3_value                      = -65
-|static_stat4_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat4_value                      = 50
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = 33
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = spell_minimum_base_lightning_damage
+|static_stat3_value                      = 5
+|static_stat4_id                         = spell_maximum_base_lightning_damage
+|static_stat4_value                      = 15
+|static_stat5_id                         = base_cast_speed_+%
+|static_stat5_value                      = -65
 |stat_text                               = Deals 5 to 15 Lightning Damage<br>65% reduced Cast Speed
 |level1                                  = True
 |level1_level_requirement                = 1
