--- Skill:GladiatorBossEliteGlacialHammer
+++ Export
@@ -15,11 +15,15 @@
 |stat_text                               = Deals 10% more Damage<br>(25-35)% chance to Freeze enemies<br>(0-10)% increased Physical Damage<br>50% of Physical Damage Converted to Cold Damage<br>(0-25)% more Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 4
+|level1_damage_effectiveness             = 100
+|level1_damage_multiplier                = 100
 |level1_stat_text                        = 25% chance to Freeze enemies
 |level1_stat1_id                         = base_chance_to_freeze_%
 |level1_stat1_value                      = 25
 |level2                                  = True
 |level2_level_requirement                = 11
+|level2_damage_effectiveness             = 100
+|level2_damage_multiplier                = 100
 |level2_stat_text                        = 25% chance to Freeze enemies<br>1% increased Physical Damage
 |level2_stat1_id                         = base_chance_to_freeze_%
 |level2_stat1_value                      = 25
@@ -27,6 +31,8 @@
 |level2_stat2_value                      = 1
 |level3                                  = True
 |level3_level_requirement                = 20
+|level3_damage_effectiveness             = 100
+|level3_damage_multiplier                = 100
 |level3_stat_text                        = 25% chance to Freeze enemies<br>1% increased Physical Damage
 |level3_stat1_id                         = base_chance_to_freeze_%
 |level3_stat1_value                      = 25
@@ -34,6 +40,8 @@
 |level3_stat2_value                      = 1
 |level4                                  = True
 |level4_level_requirement                = 29
+|level4_damage_effectiveness             = 100
+|level4_damage_multiplier                = 100
 |level4_stat_text                        = 25% chance to Freeze enemies<br>2% increased Physical Damage<br>5% more Attack Speed
 |level4_stat1_id                         = base_chance_to_freeze_%
 |level4_stat1_value                      = 25
@@ -43,6 +51,8 @@
 |level4_stat3_value                      = 5
 |level5                                  = True
 |level5_level_requirement                = 32
+|level5_damage_effectiveness             = 100
+|level5_damage_multiplier                = 100
 |level5_stat_text                        = 25% chance to Freeze enemies<br>2% increased Physical Damage<br>5% more Attack Speed
 |level5_stat1_id                         = base_chance_to_freeze_%
 |level5_stat1_value                      = 25
@@ -52,6 +62,8 @@
 |level5_stat3_value                      = 5
 |level6                                  = True
 |level6_level_requirement                = 37
+|level6_damage_effectiveness             = 100
+|level6_damage_multiplier                = 100
 |level6_stat_text                        = 25% chance to Freeze enemies<br>3% increased Physical Damage<br>5% more Attack Speed
 |level6_stat1_id                         = base_chance_to_freeze_%
 |level6_stat1_value                      = 25
@@ -61,6 +73,8 @@
 |level6_stat3_value                      = 5
 |level7                                  = True
 |level7_level_requirement                = 42
+|level7_damage_effectiveness             = 100
+|level7_damage_multiplier                = 100
 |level7_stat_text                        = 25% chance to Freeze enemies<br>4% increased Physical Damage<br>10% more Attack Speed
 |level7_stat1_id                         = base_chance_to_freeze_%
 |level7_stat1_value                      = 25
@@ -70,6 +84,8 @@
 |level7_stat3_value                      = 10
 |level8                                  = True
 |level8_level_requirement                = 50
+|level8_damage_effectiveness             = 100
+|level8_damage_multiplier                = 100
 |level8_stat_text                        = 25% chance to Freeze enemies<br>4% increased Physical Damage<br>10% more Attack Speed
 |level8_stat1_id                         = base_chance_to_freeze_%
 |level8_stat1_value                      = 25
@@ -79,6 +95,8 @@
 |level8_stat3_value                      = 10
 |level9                                  = True
 |level9_level_requirement                = 52
+|level9_damage_effectiveness             = 100
+|level9_damage_multiplier                = 100
 |level9_stat_text                        = 35% chance to Freeze enemies<br>5% increased Physical Damage<br>10% more Attack Speed
 |level9_stat1_id                         = base_chance_to_freeze_%
 |level9_stat1_value                      = 35
