--- Skill:StormCascadeTriggered
+++ Export
@@ -22,7 +22,7 @@
 |static_stat5_value                      = 100
 |static_stat6_id                         = upheaval_number_of_spikes
 |static_stat6_value                      = 5
-|stat_text                               = Deals (48-1042) to (72-1563) Physical Damage<br>100% of Physical Damage Converted to Lightning Damage<br>+3 to radius<br>You cannot Cast this Spell directly<br>Trigger this Spell when you Attack<br>Causes 5 Bursts
+|stat_text                               = Deals (48-1042) to (72-1563) Physical Damage<br>100% of Physical Damage Converted to Lightning Damage<br>+0.3 metres to radius<br>You cannot Cast this Spell directly<br>Trigger this Spell when you Attack<br>Causes 5 Bursts
 |level1                                  = True
 |level1_level_requirement                = 28
 |level1_stat_text                        = Deals 48 to 72 Physical Damage