--- Skill:HarbingerSkyBeam
+++ Export
@@ -9,12 +9,13 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 3
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = -80
-|static_stat2_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat2_value                      = -60
-|static_stat3_id                         = ground_fire_art_variation
-|static_stat3_value                      = 5
+|static_stat2_id                         = ground_fire_art_variation
+|static_stat2_value                      = 5
+|static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat3_value                      = -60
 |static_stat4_id                         = base_skill_effect_duration
 |static_stat4_value                      = 3000
 |stat_text                               = Deals (7.2-1467.8) Base Fire Damage per second<br>Base duration is 3 seconds
