--- Skill:DaressoDodgeRight
+++ Export
@@ -11,10 +11,10 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = monster_dodge_distance
-|static_stat1_value                      = 25
-|static_stat2_id                         = monster_dodge_direction
-|static_stat2_value                      = 1
+|static_stat1_id                         = monster_dodge_direction
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_dodge_distance
+|static_stat2_value                      = 25
 |stat_text                               = 
 |level1                                  = True
 |level1_level_requirement                = 1
