--- Skill:LegionMarakethRiderQuicksand
+++ Export
@@ -7,6 +7,7 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 5
 |static_stat1_id                         = base_skill_effect_duration
 |static_stat1_value                      = 5000
 |stat_text                               = Base duration is 5 seconds
