--- Skill:AnimatedThrownWeaponShaperInfluence
+++ Export
@@ -10,14 +10,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 48
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spectral_throw_projectile_scale_+%
-|static_stat1_value                      = 100
-|static_stat2_id                         = alternate_minion
+|static_stat1_id                         = alternate_minion
+|static_stat1_value                      = 1
+|static_stat2_id                         = base_is_projectile
 |static_stat2_value                      = 1
-|static_stat3_id                         = base_projectile_speed_+%
-|static_stat3_value                      = 40
-|static_stat4_id                         = base_is_projectile
-|static_stat4_value                      = 1
+|static_stat3_id                         = spectral_throw_projectile_scale_+%
+|static_stat3_value                      = 100
+|static_stat4_id                         = base_projectile_speed_+%
+|static_stat4_value                      = 40
 |stat_text                               = 40% increased Projectile Speed
 |level1                                  = True
 |level1_level_requirement                = 1
