--- Skill:AxisCasterArc
+++ Export
@@ -11,11 +11,11 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = skill_range_+%
 |static_stat1_value                      = -50
-|static_stat2_id                         = number_of_chains
-|static_stat2_value                      = 1
-|static_stat3_id                         = base_chance_to_shock_%
-|static_stat3_value                      = 10
-|stat_text                               = Deals (2-288) to (7-864) Lightning Damage<br>50% reduced Skill Range<br>Chains +1 Times<br>10% chance to Shock enemies
+|static_stat2_id                         = base_chance_to_shock_%
+|static_stat2_value                      = 10
+|static_stat3_id                         = number_of_chains
+|static_stat3_value                      = 1
+|stat_text                               = Deals (2-288) to (7-864) Lightning Damage<br>50% reduced Skill Range<br>10% chance to Shock enemies<br>Chains +1 Times
 |level1                                  = True
 |level1_level_requirement                = 3
 |level1_cost_amounts                     = 20
