--- Skill:ExileShieldChargeWeak
+++ Export
@@ -8,16 +8,16 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 50
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = shield_charge_extra_distance
-|static_stat1_value                      = 10
-|static_stat2_id                         = attack_speed_+%
+|static_stat1_id                         = ignores_proximity_shield
+|static_stat1_value                      = 1
+|static_stat2_id                         = shield_charge_extra_distance
 |static_stat2_value                      = 10
-|static_stat3_id                         = shield_charge_damage_+%_maximum
-|static_stat3_value                      = 25
-|static_stat4_id                         = base_movement_velocity_+%
-|static_stat4_value                      = 50
-|static_stat5_id                         = ignores_proximity_shield
-|static_stat5_value                      = 1
+|static_stat3_id                         = attack_speed_+%
+|static_stat3_value                      = 10
+|static_stat4_id                         = shield_charge_damage_+%_maximum
+|static_stat4_value                      = 25
+|static_stat5_id                         = base_movement_velocity_+%
+|static_stat5_value                      = 50
 |stat_text                               = 10% increased Attack Speed<br>25% more Damage with Hits at Maximum Charge Distance<br>50% increased Movement Speed
 |level1                                  = True
 |level1_level_requirement                = 1
