--- Skill:ProphecyFireUpheaval
+++ Export
@@ -7,19 +7,20 @@
 |static_damage_effectiveness             = 70
 |static_damage_multiplier                = 70
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_fire_damage_to_deal_per_minute
-|static_stat1_value                      = 141
-|static_stat2_id                         = active_skill_attack_speed_+%_final
-|static_stat2_value                      = -40
+|static_duration                         = 2
+|static_stat1_id                         = global_cannot_crit
+|static_stat1_value                      = 1
+|static_stat2_id                         = base_fire_damage_to_deal_per_minute
+|static_stat2_value                      = 141
 |static_stat3_id                         = base_skill_effect_duration
 |static_stat3_value                      = 2000
 |static_stat4_id                         = skill_physical_damage_%_to_convert_to_fire
 |static_stat4_value                      = 50
-|static_stat5_id                         = upheaval_number_of_spikes
-|static_stat5_value                      = 12
-|static_stat6_id                         = global_cannot_crit
-|static_stat6_value                      = 1
-|stat_text                               = Deals 2.4 Base Fire Damage per second<br>40% less Attack Speed<br>Base duration is 2 seconds<br>50% of Physical Damage Converted to Fire Damage<br>Causes 12 Bursts<br>Never deal Critical Strikes
+|static_stat5_id                         = active_skill_attack_speed_+%_final
+|static_stat5_value                      = -40
+|static_stat6_id                         = upheaval_number_of_spikes
+|static_stat6_value                      = 12
+|stat_text                               = Never deal Critical Strikes<br>Deals 2.4 Base Fire Damage per second<br>Base duration is 2 seconds<br>50% of Physical Damage Converted to Fire Damage<br>40% less Attack Speed<br>Causes 12 Bursts
 |level1                                  = True
 |level1_level_requirement                = 3
 }}