--- Skill:MapBossBowFlameDash
+++ Export
@@ -9,11 +9,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_skill_effect_duration
-|static_stat1_value                      = 12000
-|static_stat2_id                         = is_area_damage
-|static_stat2_value                      = 1
-|stat_text                               = Deals 3 to (4-5) Fire Damage<br>Base duration is 12 seconds<br>Deals (5.2-6.5) Base Fire Damage per second
+|static_duration                         = 12
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = base_skill_effect_duration
+|static_stat2_value                      = 12000
+|stat_text                               = Deals 3 to (4-5) Fire Damage<br>Deals (5.2-6.5) Base Fire Damage per second<br>Base duration is 12 seconds
 |level1                                  = True
 |level1_level_requirement                = 3
 |level1_cooldown                         = 4.75
