--- Skill:IzaroDaemonIncinerate
+++ Export
@@ -9,23 +9,23 @@
 |static_damage_effectiveness             = 30
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_fire_damage
-|static_stat1_value                      = 2
-|static_stat2_id                         = spell_maximum_base_fire_damage
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = skill_repeat_count
 |static_stat2_value                      = 3
-|static_stat3_id                         = skill_repeat_count
-|static_stat3_value                      = 3
-|static_stat4_id                         = number_of_additional_projectiles
-|static_stat4_value                      = 3
-|static_stat5_id                         = base_projectile_speed_+%
-|static_stat5_value                      = 40
-|static_stat6_id                         = flamethrower_damage_+%_per_stage_final
-|static_stat6_value                      = 100
-|static_stat7_id                         = always_pierce
-|static_stat7_value                      = 1
-|static_stat8_id                         = base_is_projectile
-|static_stat8_value                      = 1
-|stat_text                               = Deals 2 to 3 Fire Damage<br>Skills Repeat an additional 3 Times<br>Fires 3 additional Projectiles<br>40% increased Projectile Speed<br>100% more Damage with Hits and Ailments for each stage<br>Projectiles Pierce all Targets
+|static_stat3_id                         = always_pierce
+|static_stat3_value                      = 1
+|static_stat4_id                         = spell_minimum_base_fire_damage
+|static_stat4_value                      = 2
+|static_stat5_id                         = spell_maximum_base_fire_damage
+|static_stat5_value                      = 3
+|static_stat6_id                         = base_projectile_speed_+%
+|static_stat6_value                      = 40
+|static_stat7_id                         = number_of_additional_projectiles
+|static_stat7_value                      = 3
+|static_stat8_id                         = flamethrower_damage_+%_per_stage_final
+|static_stat8_value                      = 100
+|stat_text                               = Skills Repeat an additional 3 Times<br>Projectiles Pierce all Targets<br>Deals 2 to 3 Fire Damage<br>40% increased Projectile Speed<br>Fires 3 additional Projectiles<br>100% more Damage with Hits and Ailments for each stage
 |level1                                  = True
 |level1_level_requirement                = 3
 }}