--- Skill:TalismanT1SiegeBallista
+++ Export
@@ -171,6 +171,7 @@
 |level18_stat2_value                     = 3
 |level19                                 = True
 |level19_level_requirement               = 64
+|level19_damage_effectiveness            = 100
 |level19_damage_multiplier               = 100.44
 |level19_stat1_id                        = base_active_skill_totem_level
 |level19_stat1_value                     = 67