--- Skill:ElementalStrikeVaalColossus
+++ Export
@@ -8,17 +8,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = fixed_projectile_spread
-|static_stat1_value                      = 70
-|static_stat2_id                         = number_of_chains
-|static_stat2_value                      = 2
-|static_stat3_id                         = elemental_strike_physical_damage_%_to_convert
-|static_stat3_value                      = 60
-|static_stat4_id                         = always_pierce
-|static_stat4_value                      = 1
-|static_stat5_id                         = action_attack_or_cast_time_uses_animation_length
-|static_stat5_value                      = 1
-|stat_text                               = Chains +2 Times<br>60% of Physical Damage Converted to Fire, Cold or Lightning Damage<br>Projectiles Pierce all Targets
+|static_stat1_id                         = action_attack_or_cast_time_uses_animation_length
+|static_stat1_value                      = 1
+|static_stat2_id                         = fixed_projectile_spread
+|static_stat2_value                      = 70
+|static_stat3_id                         = always_pierce
+|static_stat3_value                      = 1
+|static_stat4_id                         = number_of_chains
+|static_stat4_value                      = 2
+|static_stat5_id                         = elemental_strike_physical_damage_%_to_convert
+|static_stat5_value                      = 60
+|stat_text                               = Projectiles Pierce all Targets<br>Chains +2 Times<br>60% of Physical Damage Converted to Fire, Cold or Lightning Damage
 |level1                                  = True
 |level1_level_requirement                = 1
 }}