--- Skill:GSConsumeBossDisintegrateBeam
+++ Export
@@ -10,19 +10,19 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage
-|static_stat1_value                      = 12
-|static_stat2_id                         = spell_maximum_base_lightning_damage
-|static_stat2_value                      = 35
-|static_stat3_id                         = shock_minimum_damage_taken_increase_%
-|static_stat3_value                      = 30
-|static_stat4_id                         = always_shock
-|static_stat4_value                      = 1
-|static_stat5_id                         = is_area_damage
+|static_stat1_id                         = action_attack_or_cast_time_uses_animation_length
+|static_stat1_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = spell_minimum_base_lightning_damage
+|static_stat3_value                      = 12
+|static_stat4_id                         = spell_maximum_base_lightning_damage
+|static_stat4_value                      = 35
+|static_stat5_id                         = always_shock
 |static_stat5_value                      = 1
-|static_stat6_id                         = action_attack_or_cast_time_uses_animation_length
-|static_stat6_value                      = 1
-|stat_text                               = Deals 12 to 35 Lightning Damage<br>Shocks from your Hits always increase Damage taken by at least 30%<br>Hits always Shock
+|static_stat6_id                         = shock_minimum_damage_taken_increase_%
+|static_stat6_value                      = 30
+|stat_text                               = Deals 12 to 35 Lightning Damage<br>Hits always Shock<br>Shocks from your Hits always increase Damage taken by at least 30%
 |level1                                  = True
 |level1_level_requirement                = 1
 }}