--- Skill:SpawnBloodMeteors
+++ Export
@@ -9,16 +9,16 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 705
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 1058
-|static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat3_value                      = -25
-|static_stat4_id                         = number_of_blood_meteors_to_create
-|static_stat4_value                      = 20
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat2_value                      = -25
+|static_stat3_id                         = number_of_blood_meteors_to_create
+|static_stat3_value                      = 20
+|static_stat4_id                         = spell_minimum_base_physical_damage
+|static_stat4_value                      = 705
+|static_stat5_id                         = spell_maximum_base_physical_damage
+|static_stat5_value                      = 1058
 |stat_text                               = Deals 705 to 1058 Physical Damage
 |level1                                  = True
 |level1_level_requirement                = 80
