--- Skill:FireEnrage
+++ Export
@@ -9,29 +9,30 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = attack_speed_+%
-|static_stat1_value                      = 50
-|static_stat2_id                         = base_movement_velocity_+%
+|static_duration                         = 5
+|static_stat1_id                         = base_skill_effect_duration
+|static_stat1_value                      = 5000
+|static_stat2_id                         = attack_speed_+%
 |static_stat2_value                      = 50
-|static_stat3_id                         = base_skill_effect_duration
-|static_stat3_value                      = 5000
-|stat_text                               = Gain (100-200)% of your Physical Damage as Extra Fire Damage<br>50% increased Attack Speed<br>50% increased Movement Speed<br>Base duration is 5 seconds
+|static_stat3_id                         = base_movement_velocity_+%
+|static_stat3_value                      = 50
+|stat_text                               = Base duration is 5 seconds<br>50% increased Attack Speed<br>50% increased Movement Speed<br>Gain (100-200)% of your Physical Damage as Extra Fire Damage
 |level1                                  = True
 |level1_level_requirement                = 70
 |level1_stat_text                        = Gain 100% of your Physical Damage as Extra Fire Damage
-|level1_stat1_id                         = physical_damage_%_to_add_as_fire
-|level1_stat1_value                      = 100
+|level1_stat1_id                         = maximum_fire_damage_to_return_when_hit
+|level1_stat1_value                      = 620
 |level1_stat2_id                         = minimum_fire_damage_to_return_when_hit
 |level1_stat2_value                      = 413
-|level1_stat3_id                         = maximum_fire_damage_to_return_when_hit
-|level1_stat3_value                      = 620
+|level1_stat3_id                         = physical_damage_%_to_add_as_fire
+|level1_stat3_value                      = 100
 |level2                                  = True
 |level2_level_requirement                = 80
 |level2_stat_text                        = Gain 200% of your Physical Damage as Extra Fire Damage
-|level2_stat1_id                         = physical_damage_%_to_add_as_fire
-|level2_stat1_value                      = 200
+|level2_stat1_id                         = maximum_fire_damage_to_return_when_hit
+|level2_stat1_value                      = 974
 |level2_stat2_id                         = minimum_fire_damage_to_return_when_hit
 |level2_stat2_value                      = 650
-|level2_stat3_id                         = maximum_fire_damage_to_return_when_hit
-|level2_stat3_value                      = 974
+|level2_stat3_id                         = physical_damage_%_to_add_as_fire
+|level2_stat3_value                      = 200
 }}