--- Skill:MonsterCausticArrow
+++ Export
@@ -15,18 +15,19 @@
 |static_stat2_value                      = 2
 |static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
 |static_stat3_value                      = -25
-|static_stat4_id                         = physical_damage_%_to_add_as_chaos
-|static_stat4_value                      = 34
-|static_stat5_id                         = projectile_damage_modifiers_apply_to_skill_dot
+|static_stat4_id                         = skill_can_fire_arrows
+|static_stat4_value                      = 1
+|static_stat5_id                         = visual_hit_effect_chaos_is_green
 |static_stat5_value                      = 1
-|static_stat6_id                         = visual_hit_effect_chaos_is_green
+|static_stat6_id                         = projectile_damage_modifiers_apply_to_skill_dot
 |static_stat6_value                      = 1
-|static_stat7_id                         = skill_can_fire_arrows
-|static_stat7_value                      = 1
-|stat_text                               = Deals (1.5-219.3) Base Chaos Damage per second<br>Base duration is (1.2-3) seconds<br>Gain 34% of your Physical Damage as Extra Chaos Damage<br>Modifiers to Projectile Damage apply to this Skill's Damage Over Time effect
+|static_stat7_id                         = physical_damage_%_to_add_as_chaos
+|static_stat7_value                      = 34
+|stat_text                               = Deals (1.5-219.3) Base Chaos Damage per second<br>Base duration is (1.2-3) seconds<br>Modifiers to Projectile Damage apply to this Skill's Damage Over Time effect<br>Gain 34% of your Physical Damage as Extra Chaos Damage
 |level1                                  = True
 |level1_level_requirement                = 3
 |level1_cost_amounts                     = 10
+|level1_duration                         = 1.2
 |level1_stat_text                        = Deals 1.5 Base Chaos Damage per second<br>Base duration is 1.2 seconds
 |level1_stat1_id                         = base_chaos_damage_to_deal_per_minute
 |level1_stat1_value                      = 89
@@ -35,6 +36,7 @@
 |level2                                  = True
 |level2_level_requirement                = 12
 |level2_cost_amounts                     = 9
+|level2_duration                         = 1.2
 |level2_stat_text                        = Deals 3.6 Base Chaos Damage per second<br>Base duration is 1.2 seconds
 |level2_stat1_id                         = base_chaos_damage_to_deal_per_minute
 |level2_stat1_value                      = 215
@@ -43,6 +45,7 @@
 |level3                                  = True
 |level3_level_requirement                = 26
 |level3_cost_amounts                     = 8
+|level3_duration                         = 1.2
 |level3_stat_text                        = Deals 10.2 Base Chaos Damage per second<br>Base duration is 1.2 seconds
 |level3_stat1_id                         = base_chaos_damage_to_deal_per_minute
 |level3_stat1_value                      = 611
@@ -51,6 +54,7 @@
 |level4                                  = True
 |level4_level_requirement                = 67
 |level4_cost_amounts                     = 8
+|level4_duration                         = 1.2
 |level4_stat_text                        = Deals 109.1 Base Chaos Damage per second<br>Base duration is 1.2 seconds
 |level4_stat1_id                         = base_chaos_damage_to_deal_per_minute
 |level4_stat1_value                      = 6546
@@ -59,6 +63,7 @@
 |level5                                  = True
 |level5_level_requirement                = 68
 |level5_cost_amounts                     = 8
+|level5_duration                         = 2.4
 |level5_stat_text                        = Deals 160.9 Base Chaos Damage per second<br>Base duration is 2.4 seconds
 |level5_stat1_id                         = base_chaos_damage_to_deal_per_minute
 |level5_stat1_value                      = 9655
@@ -67,6 +72,7 @@
 |level6                                  = True
 |level6_level_requirement                = 69
 |level6_cost_amounts                     = 8
+|level6_duration                         = 2.5
 |level6_stat_text                        = Deals 169.5 Base Chaos Damage per second<br>Base duration is 2.5 seconds
 |level6_stat1_id                         = base_chaos_damage_to_deal_per_minute
 |level6_stat1_value                      = 10170
@@ -75,6 +81,7 @@
 |level7                                  = True
 |level7_level_requirement                = 70
 |level7_cost_amounts                     = 8
+|level7_duration                         = 2.6
 |level7_stat_text                        = Deals 178.5 Base Chaos Damage per second<br>Base duration is 2.6 seconds
 |level7_stat1_id                         = base_chaos_damage_to_deal_per_minute
 |level7_stat1_value                      = 10711
@@ -83,6 +90,7 @@
 |level8                                  = True
 |level8_level_requirement                = 71
 |level8_cost_amounts                     = 8
+|level8_duration                         = 2.7
 |level8_stat_text                        = Deals 188 Base Chaos Damage per second<br>Base duration is 2.7 seconds
 |level8_stat1_id                         = base_chaos_damage_to_deal_per_minute
 |level8_stat1_value                      = 11279
@@ -91,6 +99,7 @@
 |level9                                  = True
 |level9_level_requirement                = 72
 |level9_cost_amounts                     = 8
+|level9_duration                         = 2.8
 |level9_stat_text                        = Deals 197.9 Base Chaos Damage per second<br>Base duration is 2.8 seconds
 |level9_stat1_id                         = base_chaos_damage_to_deal_per_minute
 |level9_stat1_value                      = 11876
@@ -99,6 +108,7 @@
 |level10                                 = True
 |level10_level_requirement               = 73
 |level10_cost_amounts                    = 8
+|level10_duration                        = 2.9
 |level10_stat_text                       = Deals 208.4 Base Chaos Damage per second<br>Base duration is 2.9 seconds
 |level10_stat1_id                        = base_chaos_damage_to_deal_per_minute
 |level10_stat1_value                     = 12502
@@ -107,6 +117,7 @@
 |level11                                 = True
 |level11_level_requirement               = 74
 |level11_cost_amounts                    = 8
+|level11_duration                        = 3
 |level11_stat_text                       = Deals 219.3 Base Chaos Damage per second<br>Base duration is 3 seconds
 |level11_stat1_id                        = base_chaos_damage_to_deal_per_minute
 |level11_stat1_value                     = 13159
