--- Skill:SkeletonSoldierTornadoShotBlood
+++ Export
@@ -10,17 +10,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = tornado_shot_num_of_secondary_projectiles
-|static_stat1_value                      = 3
-|static_stat2_id                         = number_of_additional_projectiles
-|static_stat2_value                      = 2
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = skill_can_fire_arrows
+|static_stat2_value                      = 1
 |static_stat3_id                         = active_skill_damage_+%_final
 |static_stat3_value                      = -50
-|static_stat4_id                         = base_is_projectile
-|static_stat4_value                      = 1
-|static_stat5_id                         = skill_can_fire_arrows
-|static_stat5_value                      = 1
-|stat_text                               = Fires 3 secondary Projectiles<br>Fires 2 additional Projectiles<br>Deals 50% less Damage
+|static_stat4_id                         = number_of_additional_projectiles
+|static_stat4_value                      = 2
+|static_stat5_id                         = tornado_shot_num_of_secondary_projectiles
+|static_stat5_value                      = 3
+|stat_text                               = Deals 50% less Damage<br>Fires 2 additional Projectiles<br>Fires 3 secondary Projectiles
 |level1                                  = True
 |level1_level_requirement                = 2
 |level2                                  = True
