--- Skill:SanctumGuardConeArrowLightning
+++ Export
@@ -0,0 +1,9 @@
+{{Skill
+|gem_description                         = 
+|active_skill_name                       = 
+|skill_id                                = SanctumGuardConeArrowLightning
+|cast_time                               = 1.0
+|required_level                          = 0
+|static_damage_effectiveness             = 100
+|static_damage_multiplier                = 100
+}}