--- Skill:GAVaalDominationLargeSlamCelestial
+++ Export
@@ -0,0 +1,19 @@
+{{Skill
+|gem_description                         = 
+|active_skill_name                       = 
+|skill_id                                = GAVaalDominationLargeSlamCelestial
+|cast_time                               = 1.0
+|required_level                          = 0
+|static_stored_uses                      = 1
+|static_cooldown                         = 7
+|static_damage_effectiveness             = 350
+|static_damage_multiplier                = 350
+|static_attack_speed_multiplier          = 23
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = melee_range_+
+|static_stat2_value                      = 15
+|stat_text                               = +15 to Melee Strike Range
+|level1                                  = True
+|level1_level_requirement                = 0
+}}