--- Skill:SolarisSuperSkyBeam
+++ Export
@@ -7,14 +7,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_fire_damage_to_deal_per_minute
-|static_stat1_value                      = 950
-|static_stat2_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat2_value                      = -45
+|static_duration                         = 2
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = -45
+|static_stat2_id                         = ground_fire_art_variation
+|static_stat2_value                      = 4
 |static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
 |static_stat3_value                      = -60
-|static_stat4_id                         = ground_fire_art_variation
-|static_stat4_value                      = 4
+|static_stat4_id                         = base_fire_damage_to_deal_per_minute
+|static_stat4_value                      = 950
 |static_stat5_id                         = base_skill_effect_duration
 |static_stat5_value                      = 2000
 |stat_text                               = Deals 15.8 Base Fire Damage per second<br>Base duration is 2 seconds
