--- Skill:TalismanRigwaldSummonDrainEssenceTotem
+++ Export
@@ -11,9 +11,9 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = alternate_minion
 |static_stat1_value                      = 74
-|static_stat2_id                         = number_of_monsters_to_summon
+|static_stat2_id                         = minion_dies_when_parent_dies
 |static_stat2_value                      = 1
-|static_stat3_id                         = minion_dies_when_parent_dies
+|static_stat3_id                         = number_of_monsters_to_summon
 |static_stat3_value                      = 1
 |static_stat4_id                         = summoned_monsters_are_minions
 |static_stat4_value                      = 1
