--- Skill:MonsterDualStrikePitbullBleed
+++ Export
@@ -12,12 +12,12 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_bleeding_damage_+%_final
 |static_stat1_value                      = -44
-|static_stat2_id                         = active_skill_attack_speed_+%_final
-|static_stat2_value                      = -25
-|static_stat3_id                         = bleed_on_hit_with_attacks_%
-|static_stat3_value                      = 100
-|static_stat4_id                         = skill_double_hits_when_dual_wielding
-|static_stat4_value                      = 1
+|static_stat2_id                         = skill_double_hits_when_dual_wielding
+|static_stat2_value                      = 1
+|static_stat3_id                         = active_skill_attack_speed_+%_final
+|static_stat3_value                      = -25
+|static_stat4_id                         = bleed_on_hit_with_attacks_%
+|static_stat4_value                      = 100
 |stat_text                               = 25% less Attack Speed<br>Causes Bleeding
 |level1                                  = True
 |level1_level_requirement                = 1
