--- Skill:TalismanBossCastGroundEffectCaustic
+++ Export
@@ -9,15 +9,16 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_chaos_damage_to_deal_per_minute
-|static_stat1_value                      = 111
-|static_stat2_id                         = cast_on_damage_taken_%
-|static_stat2_value                      = 33
-|static_stat3_id                         = cast_on_damage_taken_threshold
-|static_stat3_value                      = 1
-|static_stat4_id                         = base_skill_effect_duration
-|static_stat4_value                      = 6000
-|static_stat5_id                         = cast_on_damage_taken_also_tracks_dot
+|static_duration                         = 6
+|static_stat1_id                         = cast_on_damage_taken_also_tracks_dot
+|static_stat1_value                      = 1
+|static_stat2_id                         = base_chaos_damage_to_deal_per_minute
+|static_stat2_value                      = 111
+|static_stat3_id                         = base_skill_effect_duration
+|static_stat3_value                      = 6000
+|static_stat4_id                         = cast_on_damage_taken_%
+|static_stat4_value                      = 33
+|static_stat5_id                         = cast_on_damage_taken_threshold
 |static_stat5_value                      = 1
 |stat_text                               = Deals 1.9 Base Chaos Damage per second<br>Base duration is 6 seconds
 |level1                                  = True
