--- Skill:SummonGuardianRelic
+++ Export
@@ -0,0 +1,26 @@
+{{Skill
+|gem_description                         = Summons a Relic of a random element that stays near you. Depending on the element chosen, the relic minion will have an Anger, Hatred, or Wrath aura. These relics explode when they die, dealing elemental damage to enemies around them. If you already have a relic of the chosen element, its duration will be refreshed instead of summoning a new one.
+|active_skill_name                       = Summon Elemental Relic
+|skill_id                                = SummonGuardianRelic
+|cast_time                               = 1.0
+|required_level                          = 4
+|static_stored_uses                      = 1
+|static_cooldown                         = 0.3
+|static_damage_effectiveness             = 100
+|static_damage_multiplier                = 100
+|static_attack_speed_multiplier          = 100
+|static_duration                         = 5
+|static_stat1_id                         = display_minion_monster_type
+|static_stat1_value                      = 24
+|static_stat2_id                         = base_skill_effect_duration
+|static_stat2_value                      = 5000
+|static_stat3_id                         = guardian_relic_explode_on_death_for_%_life_as_element_damage
+|static_stat3_value                      = 100
+|static_stat4_id                         = minion_actor_level_is_user_level_up_to_maximum
+|static_stat4_value                      = 85
+|static_stat5_id                         = skill_triggered_by_nearby_allies_kill_or_hit_rare_unique_%_chance
+|static_stat5_value                      = 25
+|stat_text                               = Base duration is 5 seconds<br>Minions Explode on Death, dealing 100% of their maximum Life as Damage of their chosen Element<br>Minions' Level is equal to yours, up to a maximum of 85<br>25% chance to Trigger this Skill when you or a nearby ally Kill an Enemy or Hit a Rare or Unique Enemy
+|level1                                  = True
+|level1_level_requirement                = 4
+}}