--- Skill:LeapSlamHardMode
+++ Export
@@ -21,6 +21,8 @@
 |stat_text                               = (20-54)% increased Stun Duration against Enemies that are on Full Life<br>+0.55 seconds to Attack Time<br>Damaging Hits always Stun Enemies that are on Full Life
 |level1                                  = True
 |level1_level_requirement                = 10
+|level1_damage_effectiveness             = 100
+|level1_damage_multiplier                = 100
 |level1_stat_text                        = 20% increased Stun Duration against Enemies that are on Full Life
 |level1_stat1_id                         = stun_duration_+%_vs_enemies_that_are_on_full_life
 |level1_stat1_value                      = 20