--- Skill:SynthesisHoundSlam
+++ Export
@@ -9,15 +9,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 200
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_attack_speed_+%_final
-|static_stat1_value                      = -22
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
 |static_stat2_id                         = skill_physical_damage_%_to_convert_to_lightning
 |static_stat2_value                      = 100
-|static_stat3_id                         = melee_range_+
-|static_stat3_value                      = 20
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
-|stat_text                               = Deals (0-400)% more Damage<br>22% less Attack Speed<br>100% of Physical Damage Converted to Lightning Damage<br>+20 to Melee Strike Range
+|static_stat3_id                         = active_skill_attack_speed_+%_final
+|static_stat3_value                      = -22
+|static_stat4_id                         = melee_range_+
+|static_stat4_value                      = 20
+|stat_text                               = 100% of Physical Damage Converted to Lightning Damage<br>Deals (0-400)% more Damage<br>22% less Attack Speed<br>+20 to Melee Strike Range
 |level1                                  = True
 |level1_level_requirement                = 1
 |level2                                  = True
