--- Skill:AtlasOrionDeathZoneShockwave8
+++ Export
@@ -12,13 +12,13 @@
 |static_stat1_value                      = 350
 |static_stat2_id                         = base_knockback_speed_+%
 |static_stat2_value                      = 100
-|static_stat3_id                         = knockback_distance_+%
-|static_stat3_value                      = 100
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
 |static_stat4_id                         = global_knockback
 |static_stat4_value                      = 1
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
-|stat_text                               = 100% increased Knockback Distance<br>Knocks Enemies Back on Hit
+|static_stat5_id                         = knockback_distance_+%
+|static_stat5_value                      = 100
+|stat_text                               = Knocks Enemies Back on Hit<br>100% increased Knockback Distance
 |level1                                  = True
 |level1_level_requirement                = 0
 }}