--- Skill:SupportCursePillarTriggerCursesLessCurseEffect
+++ Export
@@ -2,15 +2,16 @@
 |skill_id                                = SupportCursePillarTriggerCursesLessCurseEffect
 |support_gem_letter                      = !
 |required_level                          = 70
+|static_cost_multiplier                  = 100
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = curse_pillar_curse_effect_+%_final
-|static_stat1_value                      = -20
-|static_stat2_id                         = number_of_additional_curses_allowed
-|static_stat2_value                      = 5
-|static_stat3_id                         = skill_has_trigger_from_unique_item
-|static_stat3_value                      = 1
+|static_stat1_id                         = skill_has_trigger_from_unique_item
+|static_stat1_value                      = 1
+|static_stat2_id                         = curse_pillar_curse_effect_+%_final
+|static_stat2_value                      = -20
+|static_stat3_id                         = number_of_additional_curses_allowed
+|static_stat3_value                      = 5
 |stat_text                               = 20% less Effect of your Curses<br>Supported Skills can apply 5 additional Curses
 |level1                                  = True
 |level1_level_requirement                = 70
