--- Skill:RitualEnergyShieldRechargeBuff
+++ Export
@@ -9,13 +9,14 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
+|static_duration                         = 2
 |static_stat1_id                         = base_skill_effect_duration
 |static_stat1_value                      = 2000
-|static_stat2_id                         = energy_shield_recharge_rate_+%
-|static_stat2_value                      = 300
-|static_stat3_id                         = energy_shield_delay_-%
-|static_stat3_value                      = 800
-|stat_text                               = Base duration is 2 seconds<br>300% increased Energy Shield Recharge rate<br>800% faster start of Energy Shield Recharge
+|static_stat2_id                         = energy_shield_delay_-%
+|static_stat2_value                      = 800
+|static_stat3_id                         = energy_shield_recharge_rate_+%
+|static_stat3_value                      = 300
+|stat_text                               = Base duration is 2 seconds<br>800% faster start of Energy Shield Recharge<br>300% increased Energy Shield Recharge Rate
 |level1                                  = True
 |level1_level_requirement                = 1
 }}