--- Skill:HarvestOshabiVineCascadeGS2
+++ Export
@@ -9,19 +9,19 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = -25
-|static_stat2_id                         = base_chance_to_freeze_%
-|static_stat2_value                      = 33
-|static_stat3_id                         = base_chance_to_shock_%
-|static_stat3_value                      = 33
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = is_oshabi_primal_skill
+|static_stat3_value                      = 1
 |static_stat4_id                         = skill_physical_damage_%_to_convert_to_lightning
 |static_stat4_value                      = 25
-|static_stat5_id                         = skill_physical_damage_%_to_convert_to_cold
-|static_stat5_value                      = 25
-|static_stat6_id                         = is_area_damage
-|static_stat6_value                      = 1
-|static_stat7_id                         = is_oshabi_primal_skill
-|static_stat7_value                      = 1
-|stat_text                               = 33% chance to Freeze enemies<br>33% chance to Shock enemies<br>25% of Physical Damage Converted to Lightning Damage<br>25% of Physical Damage Converted to Cold Damage
+|static_stat5_id                         = base_chance_to_freeze_%
+|static_stat5_value                      = 33
+|static_stat6_id                         = base_chance_to_shock_%
+|static_stat6_value                      = 33
+|static_stat7_id                         = skill_physical_damage_%_to_convert_to_cold
+|static_stat7_value                      = 25
+|stat_text                               = 25% of Physical Damage Converted to Lightning Damage<br>33% chance to Freeze enemies<br>33% chance to Shock enemies<br>25% of Physical Damage Converted to Cold Damage
 |level1                                  = True
 |level1_level_requirement                = 83
 }}