--- Skill:DelveCaveDwellerBoltTriggered
+++ Export
@@ -7,21 +7,22 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage
-|static_stat1_value                      = 4
-|static_stat2_id                         = spell_maximum_base_lightning_damage
-|static_stat2_value                      = 8
-|static_stat3_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat3_value                      = -50
-|static_stat4_id                         = base_skill_effect_duration
-|static_stat4_value                      = 260
-|static_stat5_id                         = skill_range_+%
-|static_stat5_value                      = 100
-|static_stat6_id                         = is_area_damage
-|static_stat6_value                      = 1
-|static_stat7_id                         = cast_on_trigger_cascade_event
-|static_stat7_value                      = 1
-|stat_text                               = Deals 4 to 8 Lightning Damage<br>Base duration is 0.26 seconds<br>100% increased Skill Range
+|static_duration                         = 0.26
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = -50
+|static_stat2_id                         = cast_on_trigger_cascade_event
+|static_stat2_value                      = 1
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = spell_minimum_base_lightning_damage
+|static_stat4_value                      = 4
+|static_stat5_id                         = spell_maximum_base_lightning_damage
+|static_stat5_value                      = 8
+|static_stat6_id                         = skill_range_+%
+|static_stat6_value                      = 100
+|static_stat7_id                         = base_skill_effect_duration
+|static_stat7_value                      = 260
+|stat_text                               = Deals 4 to 8 Lightning Damage<br>100% increased Skill Range<br>Base duration is 0.26 seconds
 |level1                                  = True
 |level1_level_requirement                = 1
 }}