--- Skill:SynthesisGuardian2FrontalDonut4
+++ Export
@@ -7,12 +7,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_damage_+%_final
-|static_stat1_value                      = 300
-|static_stat2_id                         = skill_physical_damage_%_to_convert_to_fire
-|static_stat2_value                      = 75
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = active_skill_damage_+%_final
+|static_stat2_value                      = 300
+|static_stat3_id                         = skill_physical_damage_%_to_convert_to_fire
+|static_stat3_value                      = 75
 |stat_text                               = Deals 300% more Damage<br>75% of Physical Damage Converted to Fire Damage
 |level1                                  = True
 |level1_level_requirement                = 0
