--- Skill:DoedreSummonStonePillarArea
+++ Export
@@ -9,12 +9,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = summon_specific_monsters_radius_+%
-|static_stat1_value                      = -90
+|static_stat1_id                         = alternate_minion
+|static_stat1_value                      = 259
 |static_stat2_id                         = number_of_monsters_to_summon
 |static_stat2_value                      = 1
-|static_stat3_id                         = alternate_minion
-|static_stat3_value                      = 259
+|static_stat3_id                         = summon_specific_monsters_radius_+%
+|static_stat3_value                      = -90
 |stat_text                               = 
 |level1                                  = True
 |level1_level_requirement                = 1
