--- Skill:LbRetractableTrap
+++ Export
@@ -7,21 +7,19 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = bleed_damage_+%_vs_player_minions_final
-|static_stat1_value                      = -90
-|static_stat2_id                         = damage_+%_vs_players
-|static_stat2_value                      = 16
-|static_stat3_id                         = %_of_es_to_deal_as_damage
-|static_stat3_value                      = 15
-|static_stat4_id                         = %_of_life_to_deal_as_damage
-|static_stat4_value                      = 25
-|static_stat5_id                         = reduce_enemy_dodge_%
-|static_stat5_value                      = 100
-|static_stat6_id                         = global_reduce_enemy_block_%
-|static_stat6_value                      = 100
-|static_stat7_id                         = global_always_hit
-|static_stat7_value                      = 1
-|stat_text                               = Enemies have 100% reduced chance to Dodge Hits from this Skill<br>Enemy Block Chance reduced by 100% against this Skill<br>Can't be Evaded
+|static_stat1_id                         = %_of_es_to_deal_as_damage
+|static_stat1_value                      = 15
+|static_stat2_id                         = %_of_life_to_deal_as_damage
+|static_stat2_value                      = 25
+|static_stat3_id                         = bleed_damage_+%_vs_player_minions_final
+|static_stat3_value                      = -90
+|static_stat4_id                         = cannot_be_blocked_or_dodged_or_suppressed
+|static_stat4_value                      = 1
+|static_stat5_id                         = damage_+%_vs_players
+|static_stat5_value                      = 16
+|static_stat6_id                         = global_always_hit
+|static_stat6_value                      = 1
+|stat_text                               = Can't be Evaded
 |level1                                  = True
 |level1_level_requirement                = 1
 }}