--- Skill:SpikerSpineDeathNovaChromeMaps
+++ Export
@@ -9,21 +9,21 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = attack_on_death_%
 |static_stat1_value                      = 70
-|static_stat2_id                         = spell_maximum_action_distance_+%
-|static_stat2_value                      = -20
+|static_stat2_id                         = base_is_projectile
+|static_stat2_value                      = 1
 |static_stat3_id                         = monster_projectile_variation
 |static_stat3_value                      = 25
-|static_stat4_id                         = active_skill_damage_+%_final
-|static_stat4_value                      = 75
-|static_stat5_id                         = number_of_additional_projectiles
-|static_stat5_value                      = 11
-|static_stat6_id                         = projectiles_nova
-|static_stat6_value                      = 1
-|static_stat7_id                         = base_is_projectile
+|static_stat4_id                         = skill_unusable
+|static_stat4_value                      = 1
+|static_stat5_id                         = spell_maximum_action_distance_+%
+|static_stat5_value                      = -20
+|static_stat6_id                         = active_skill_damage_+%_final
+|static_stat6_value                      = 75
+|static_stat7_id                         = projectiles_nova
 |static_stat7_value                      = 1
-|static_stat8_id                         = skill_unusable
-|static_stat8_value                      = 1
-|stat_text                               = Deals 75% more Damage<br>Fires 11 additional Projectiles<br>Fires Projectiles in a circle
+|static_stat8_id                         = number_of_additional_projectiles
+|static_stat8_value                      = 11
+|stat_text                               = Deals 75% more Damage<br>Fires Projectiles in a circle<br>Fires 11 additional Projectiles
 |level1                                  = True
 |level1_level_requirement                = 68
 }}