--- Skill:MMSHellscapeFleshPustule
+++ Export
@@ -8,25 +8,25 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 5
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 8
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
 |static_stat3_id                         = projectile_speed_is_divided_by_%_of_scale
 |static_stat3_value                      = 200
 |static_stat4_id                         = projectile_spread_radius
 |static_stat4_value                      = 5
-|static_stat5_id                         = number_of_additional_projectiles
+|static_stat5_id                         = projectile_uses_contact_position
 |static_stat5_value                      = 1
-|static_stat6_id                         = projectiles_nova
-|static_stat6_value                      = 1
-|static_stat7_id                         = is_area_damage
-|static_stat7_value                      = 1
-|static_stat8_id                         = base_is_projectile
+|static_stat6_id                         = spell_minimum_base_physical_damage
+|static_stat6_value                      = 5
+|static_stat7_id                         = spell_maximum_base_physical_damage
+|static_stat7_value                      = 8
+|static_stat8_id                         = projectiles_nova
 |static_stat8_value                      = 1
-|static_stat9_id                         = projectile_uses_contact_position
+|static_stat9_id                         = number_of_additional_projectiles
 |static_stat9_value                      = 1
-|stat_text                               = Deals 5 to 8 Physical Damage<br>Fires an additional Projectile<br>Fires Projectiles in a circle
+|stat_text                               = Deals 5 to 8 Physical Damage<br>Fires Projectiles in a circle<br>Fires an additional Projectile
 |level1                                  = True
 |level1_level_requirement                = 0
 }}