--- Skill:HeistAdmiralDashThrustTriggered
+++ Export
@@ -9,13 +9,13 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = 50
-|static_stat2_id                         = skill_physical_damage_%_to_convert_to_cold
-|static_stat2_value                      = 100
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
 |static_stat3_id                         = always_freeze
 |static_stat3_value                      = 1
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
-|stat_text                               = Deals (0-300)% more Damage<br>100% of Physical Damage Converted to Cold Damage<br>Always Freezes enemies
+|static_stat4_id                         = skill_physical_damage_%_to_convert_to_cold
+|static_stat4_value                      = 100
+|stat_text                               = Deals (0-300)% more Damage<br>Always Freezes enemies<br>100% of Physical Damage Converted to Cold Damage
 |level1                                  = True
 |level1_level_requirement                = 0
 |level2                                  = True
