--- Skill:AtlasAdjudicatorRockBarrage10
+++ Export
@@ -7,13 +7,13 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 200
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = always_pierce
+|static_stat1_id                         = base_is_projectile
 |static_stat1_value                      = 1
-|static_stat2_id                         = global_bleed_on_hit
+|static_stat2_id                         = projectile_uses_contact_position
 |static_stat2_value                      = 1
-|static_stat3_id                         = base_is_projectile
+|static_stat3_id                         = always_pierce
 |static_stat3_value                      = 1
-|static_stat4_id                         = projectile_uses_contact_position
+|static_stat4_id                         = global_bleed_on_hit
 |static_stat4_value                      = 1
 |stat_text                               = Projectiles Pierce all Targets<br>Causes Bleeding
 |level1                                  = True
