--- Skill:EmptyActionVaalSergeantMortarSpam
+++ Export
@@ -7,12 +7,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 75
-|static_stat1_id                         = melee_range_+
-|static_stat1_value                      = 70
+|static_stat1_id                         = skill_cannot_be_knocked_back
+|static_stat1_value                      = 1
 |static_stat2_id                         = skill_cannot_be_stunned
 |static_stat2_value                      = 1
-|static_stat3_id                         = skill_cannot_be_knocked_back
-|static_stat3_value                      = 1
+|static_stat3_id                         = melee_range_+
+|static_stat3_value                      = 70
 |stat_text                               = +70 to Melee Strike Range
 |level1                                  = True
 |level1_level_requirement                = 1
