--- Skill:LegionEmpireSpearCombo1
+++ Export
@@ -7,9 +7,9 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 125
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = global_maim_on_hit
+|static_stat1_id                         = is_area_damage
 |static_stat1_value                      = 1
-|static_stat2_id                         = is_area_damage
+|static_stat2_id                         = global_maim_on_hit
 |static_stat2_value                      = 1
 |stat_text                               = Maim on Hit
 |level1                                  = True
