--- Skill:AtlasEyrieSpawnFrozenHalfPipe
+++ Export
@@ -7,11 +7,11 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = always_freeze
+|static_stat1_id                         = is_area_damage
 |static_stat1_value                      = 1
-|static_stat2_id                         = is_area_damage
+|static_stat2_id                         = always_freeze
 |static_stat2_value                      = 1
-|stat_text                               = Deals (465-1000) to (698-1500) Cold Damage<br>(0-200)% increased Freeze Duration on enemies<br>Always Freezes enemies
+|stat_text                               = Deals (465-1000) to (698-1500) Cold Damage<br>Always Freezes enemies<br>(0-200)% increased Freeze Duration on enemies
 |level1                                  = True
 |level1_level_requirement                = 68
 |level1_stat_text                        = Deals 465 to 698 Cold Damage
