--- Skill:RampageDaemonCombinedNova
+++ Export
@@ -8,25 +8,25 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 11
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 16
-|static_stat3_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat3_value                      = 15
-|static_stat4_id                         = skill_repeat_count
-|static_stat4_value                      = 8
-|static_stat5_id                         = explode_on_kill_%_fire_damage_to_deal
-|static_stat5_value                      = 25
-|static_stat6_id                         = ice_nova_radius_+%_per_repeat
-|static_stat6_value                      = -5
-|static_stat7_id                         = ice_nova_number_of_repeats
-|static_stat7_value                      = 20
-|static_stat8_id                         = always_stun
-|static_stat8_value                      = 1
-|static_stat9_id                         = is_area_damage
-|static_stat9_value                      = 1
-|stat_text                               = Deals 11 to 16 Physical Damage<br>Skills Repeat an additional 8 Times<br>Enemies Killed with Attack or Spell Hits Explode, dealing 25% of their Life as Fire Damage<br>5% reduced Area of Effect each repeat<br>Causes smaller novas up to 20 times on enemies hit<br>Stuns Enemies
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = 15
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = spell_minimum_base_physical_damage
+|static_stat3_value                      = 11
+|static_stat4_id                         = spell_maximum_base_physical_damage
+|static_stat4_value                      = 16
+|static_stat5_id                         = skill_repeat_count
+|static_stat5_value                      = 8
+|static_stat6_id                         = explode_on_kill_%_fire_damage_to_deal
+|static_stat6_value                      = 25
+|static_stat7_id                         = always_stun
+|static_stat7_value                      = 1
+|static_stat8_id                         = ice_nova_number_of_repeats
+|static_stat8_value                      = 20
+|static_stat9_id                         = ice_nova_radius_+%_per_repeat
+|static_stat9_value                      = -5
+|stat_text                               = Deals 11 to 16 Physical Damage<br>Skills Repeat an additional 8 Times<br>Enemies Killed with Attack or Spell Hits Explode, dealing 25% of their Life as Fire Damage<br>Stuns Enemies<br>Causes smaller novas up to 20 times on enemies hit<br>5% reduced Area of Effect each repeat
 |level1                                  = True
 |level1_level_requirement                = 2
 }}