--- Skill:SummonRigwaldsPack
+++ Export
@@ -22,7 +22,7 @@
 |static_stat6_value                      = 1
 |static_stat7_id                         = modifiers_to_claw_critical_strike_multiplier_apply_minion_critical_strike_multiplier
 |static_stat7_value                      = 1
-|stat_text                               = Base duration is 30 seconds<br>You cannot Cast this Spell directly<br>Modifiers to Claw Critical Strike Chance also apply to Minions<br>Modifiers to Claw Critical Strike Multiplier also apply to Minions<br>Buff grants (3-8) to (6-16) Added Attack Physical Damage
+|stat_text                               = Base duration is 30 seconds<br>You cannot Cast this Spell directly<br>Trigger this Spell on Kill<br>Modifiers to Claw Critical Strike Chance also apply to Minions<br>Modifiers to Claw Critical Strike Multiplier also apply to Minions<br>Buff grants (3-8) to (6-16) Added Attack Physical Damage
 |level1                                  = True
 |level1_level_requirement                = 55
 |level1_stat_text                        = Trigger this Spell on Kill<br>Buff grants 3 to 6 Added Attack Physical Damage
@@ -35,10 +35,8 @@
 |level2                                  = True
 |level2_level_requirement                = 78
 |level2_stat_text                        = Buff grants 8 to 16 Added Attack Physical Damage
-|level2_stat1_id                         = chance_to_cast_on_kill_%_target_self <!--dummy text added manually -->
-|level2_stat1_value                      = 0
-|level2_stat2_id                         = spectral_wolf_grants_attack_minimum_added_physical_damage
-|level2_stat2_value                      = 8
-|level2_stat3_id                         = spectral_wolf_grants_attack_maximum_added_physical_damage
-|level2_stat3_value                      = 16
+|level2_stat1_id                         = spectral_wolf_grants_attack_minimum_added_physical_damage
+|level2_stat1_value                      = 8
+|level2_stat2_id                         = spectral_wolf_grants_attack_maximum_added_physical_damage
+|level2_stat2_value                      = 16
 }}