--- Skill:SnakeProjectileSlow
+++ Export
@@ -7,19 +7,19 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_maximum_action_distance_+%
-|static_stat1_value                      = -40
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
 |static_stat2_id                         = monster_projectile_variation
 |static_stat2_value                      = 1
-|static_stat3_id                         = base_poison_damage_+%
-|static_stat3_value                      = 30
+|static_stat3_id                         = spell_maximum_action_distance_+%
+|static_stat3_value                      = -40
 |static_stat4_id                         = active_skill_damage_+%_final
 |static_stat4_value                      = 100
 |static_stat5_id                         = base_chance_to_poison_on_hit_%
 |static_stat5_value                      = 100
-|static_stat6_id                         = base_is_projectile
-|static_stat6_value                      = 1
-|stat_text                               = 30% increased Damage with Poison<br>Deals 100% more Damage<br>Always Poison on Hit
+|static_stat6_id                         = base_poison_damage_+%
+|static_stat6_value                      = 30
+|stat_text                               = Deals 100% more Damage<br>Always Poison on Hit<br>30% increased Damage with Poison
 |level1                                  = True
 |level1_level_requirement                = 1
 }}