--- Skill:IntMissionDaemonVaalSummonSkeletonsCaster
+++ Export
@@ -8,11 +8,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = base_skill_effect_duration
-|static_stat1_value                      = 20000
-|static_stat2_id                         = base_number_of_skeletons_allowed
-|static_stat2_value                      = 60
-|stat_text                               = Base duration is 20 seconds<br>Maximum 60 Summoned Skeletons
+|static_duration                         = 20
+|static_stat1_id                         = base_number_of_skeletons_allowed
+|static_stat1_value                      = 60
+|static_stat2_id                         = base_skill_effect_duration
+|static_stat2_value                      = 20000
+|stat_text                               = Maximum 60 Summoned Skeletons<br>Base duration is 20 seconds
 |level1                                  = True
 |level1_level_requirement                = 4
 |level1_cooldown                         = 50
