--- Skill:WaterElementalFrostbite
+++ Export
@@ -19,6 +19,7 @@
 |level1_cost_multiplier                  = 100
 |level1_stored_uses                      = 1
 |level1_cooldown                         = 10
+|level1_attack_speed_multiplier          = 100
 |level1_stat_text                        = Cursed enemies have -20% to Cold Resistance
 |level1_stat1_id                         = base_cold_damage_resistance_%
 |level1_stat1_value                      = -20