--- Skill:AfflictionDemonTeleportAttackTriggeredCone
+++ Export
@@ -13,11 +13,11 @@
 |static_stat2_value                      = 4000
 |static_stat3_id                         = corrupted_blood_on_hit_num_stacks
 |static_stat3_value                      = 1
-|static_stat4_id                         = global_knockback
+|static_stat4_id                         = determine_knockback_direction_from_melee_pattern
 |static_stat4_value                      = 1
 |static_stat5_id                         = is_area_damage
 |static_stat5_value                      = 1
-|static_stat6_id                         = determine_knockback_direction_from_melee_pattern
+|static_stat6_id                         = global_knockback
 |static_stat6_value                      = 1
 |stat_text                               = Knocks Enemies Back on Hit
 |level1                                  = True
