--- Skill:BreachBoneMortar
+++ Export
@@ -10,11 +10,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 41
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 62
-|static_stat3_id                         = expend_consume_charges_on_skill_use
+|static_duration                         = 10
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = expend_consume_charges_on_skill_use
+|static_stat2_value                      = 1
+|static_stat3_id                         = is_area_damage
 |static_stat3_value                      = 1
 |static_stat4_id                         = mortar_cone_angle
 |static_stat4_value                      = 120
@@ -22,14 +23,14 @@
 |static_stat5_value                      = 217
 |static_stat6_id                         = projectile_spread_radius
 |static_stat6_value                      = 543
-|static_stat7_id                         = base_skill_effect_duration
-|static_stat7_value                      = 10000
-|static_stat8_id                         = number_of_additional_projectiles
-|static_stat8_value                      = 12
-|static_stat9_id                         = is_area_damage
-|static_stat9_value                      = 1
-|static_stat10_id                        = base_is_projectile
-|static_stat10_value                     = 1
+|static_stat7_id                         = spell_minimum_base_physical_damage
+|static_stat7_value                      = 41
+|static_stat8_id                         = spell_maximum_base_physical_damage
+|static_stat8_value                      = 62
+|static_stat9_id                         = base_skill_effect_duration
+|static_stat9_value                      = 10000
+|static_stat10_id                        = number_of_additional_projectiles
+|static_stat10_value                     = 12
 |stat_text                               = Deals 41 to 62 Physical Damage<br>Base duration is 10 seconds<br>Fires 12 additional Projectiles
 |level1                                  = True
 |level1_level_requirement                = 38
