--- Skill:WbLeaderLightningStrike2
+++ Export
@@ -8,19 +8,19 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_damage_+%_final
-|static_stat1_value                      = 20
-|static_stat2_id                         = active_skill_projectile_damage_+%_final
-|static_stat2_value                      = -30
-|static_stat3_id                         = number_of_additional_projectiles
-|static_stat3_value                      = 7
-|static_stat4_id                         = active_skill_attack_damage_+%_final
-|static_stat4_value                      = 30
-|static_stat5_id                         = skill_physical_damage_%_to_convert_to_lightning
-|static_stat5_value                      = 50
-|static_stat6_id                         = show_number_of_projectiles
-|static_stat6_value                      = 1
-|stat_text                               = Deals 20% more Damage<br>Projectiles deal 30% less Damage<br>Fires 7 additional Projectiles<br>Deals 30% more Attack Damage<br>50% of Physical Damage Converted to Lightning Damage
+|static_stat1_id                         = show_number_of_projectiles
+|static_stat1_value                      = 1
+|static_stat2_id                         = skill_physical_damage_%_to_convert_to_lightning
+|static_stat2_value                      = 50
+|static_stat3_id                         = active_skill_attack_damage_+%_final
+|static_stat3_value                      = 30
+|static_stat4_id                         = active_skill_damage_+%_final
+|static_stat4_value                      = 20
+|static_stat5_id                         = number_of_additional_projectiles
+|static_stat5_value                      = 7
+|static_stat6_id                         = active_skill_projectile_damage_+%_final
+|static_stat6_value                      = -30
+|stat_text                               = 50% of Physical Damage Converted to Lightning Damage<br>Deals 30% more Attack Damage<br>Deals 20% more Damage<br>Fires 7 additional Projectiles<br>30% less Projectile Damage
 |level1                                  = True
 |level1_level_requirement                = 4
 |level2                                  = True
