--- Skill:AfflictionPizzaSlamUnique
+++ Export
@@ -7,23 +7,23 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 5
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 7
-|static_stat3_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat3_value                      = -20
-|static_stat4_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
-|static_stat4_value                      = 50
-|static_stat5_id                         = chance_to_unnerve_on_hit_%
-|static_stat5_value                      = 100
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = -20
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
+|static_stat3_value                      = 50
+|static_stat4_id                         = spell_minimum_base_physical_damage
+|static_stat4_value                      = 5
+|static_stat5_id                         = spell_maximum_base_physical_damage
+|static_stat5_value                      = 7
 |static_stat6_id                         = melee_range_+
 |static_stat6_value                      = 100
-|static_stat7_id                         = upheaval_number_of_spikes
-|static_stat7_value                      = 15
-|static_stat8_id                         = is_area_damage
-|static_stat8_value                      = 1
-|stat_text                               = Deals 5 to 7 Physical Damage<br>Unnerve Enemies for 4 seconds on Hit<br>+100 to Melee Strike Range<br>Causes 15 Bursts
+|static_stat7_id                         = chance_to_unnerve_on_hit_%
+|static_stat7_value                      = 100
+|static_stat8_id                         = upheaval_number_of_spikes
+|static_stat8_value                      = 15
+|stat_text                               = Deals 5 to 7 Physical Damage<br>+100 to Melee Strike Range<br>Unnerve Enemies for 4 seconds on Hit<br>Causes 15 Bursts
 |level1                                  = True
 |level1_level_requirement                = 1
 }}