--- Skill:EnchantmentOfInfernoOnKill2
+++ Export
@@ -10,25 +10,26 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_fire_damage
-|static_stat1_value                      = 722
-|static_stat2_id                         = spell_maximum_base_fire_damage
-|static_stat2_value                      = 1107
+|static_duration                         = 2
+|static_stat1_id                         = base_skill_show_average_damage_instead_of_dps
+|static_stat1_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
 |static_stat3_id                         = skill_override_pvp_scaling_time_ms
 |static_stat3_value                      = 450
-|static_stat4_id                         = chance_to_cast_on_kill_%_target_self
-|static_stat4_value                      = 100
-|static_stat5_id                         = fire_storm_fireball_delay_ms
-|static_stat5_value                      = 100
-|static_stat6_id                         = base_skill_effect_duration
-|static_stat6_value                      = 2000
-|static_stat7_id                         = damage_cannot_be_reflected
-|static_stat7_value                      = 1
-|static_stat8_id                         = base_skill_show_average_damage_instead_of_dps
-|static_stat8_value                      = 1
-|static_stat9_id                         = is_area_damage
+|static_stat4_id                         = base_skill_effect_duration
+|static_stat4_value                      = 2000
+|static_stat5_id                         = spell_minimum_base_fire_damage
+|static_stat5_value                      = 722
+|static_stat6_id                         = spell_maximum_base_fire_damage
+|static_stat6_value                      = 1107
+|static_stat7_id                         = fire_storm_fireball_delay_ms
+|static_stat7_value                      = 100
+|static_stat8_id                         = chance_to_cast_on_kill_%_target_self
+|static_stat8_value                      = 100
+|static_stat9_id                         = damage_cannot_be_reflected
 |static_stat9_value                      = 1
-|stat_text                               = Deals 722 to 1107 Fire Damage<br>Trigger this Spell on Kill<br>One impact every 0.1 seconds<br>Base duration is 2 seconds<br>Damage cannot be Reflected
+|stat_text                               = Base duration is 2 seconds<br>Deals 722 to 1107 Fire Damage<br>One impact every 0.1 seconds<br>Trigger this Spell on Kill<br>Damage cannot be Reflected
 |level1                                  = True
 |level1_level_requirement                = 53
 }}