--- Skill:InvisibleDaemonShieldCharge
+++ Export
@@ -8,16 +8,16 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = shield_charge_end_slide_distance
-|static_stat1_value                      = 50
-|static_stat2_id                         = shield_charge_extra_distance
+|static_stat1_id                         = shield_charge_continue_past_target
+|static_stat1_value                      = 1
+|static_stat2_id                         = shield_charge_end_slide_distance
 |static_stat2_value                      = 50
-|static_stat3_id                         = base_movement_velocity_+%
+|static_stat3_id                         = shield_charge_extra_distance
 |static_stat3_value                      = 50
 |static_stat4_id                         = shield_charge_remove_stun_and_knockback
 |static_stat4_value                      = 1
-|static_stat5_id                         = shield_charge_continue_past_target
-|static_stat5_value                      = 1
+|static_stat5_id                         = base_movement_velocity_+%
+|static_stat5_value                      = 50
 |stat_text                               = 50% increased Movement Speed
 |level1                                  = True
 |level1_level_requirement                = 1
