--- Skill:SabreSpiderDualStrike
+++ Export
@@ -17,7 +17,7 @@
 |static_stat3_value                      = 60
 |static_stat4_id                         = melee_range_+
 |static_stat4_value                      = 6
-|stat_text                               = 60% of Physical Damage Converted to Chaos Damage<br>+6 to Melee Strike Range
+|stat_text                               = 60% of Physical Damage Converted to Chaos Damage<br>+0.6 metres to Melee Strike Range
 |level1                                  = True
 |level1_level_requirement                = 1
 }}