--- Skill:SummonRadiantSentinel
+++ Export
@@ -21,7 +21,7 @@
 |static_stat4_value                      = 1800
 |static_stat5_id                         = radiant_sentinel_minion_burning_effect_radius
 |static_stat5_value                      = 45
-|stat_text                               = Base duration is 20 seconds<br>Minion's Level is equal to yours, up to a maximum of 85<br>Minion deals Fire Damage equal to 30% of its maximum Life per Second to Enemies in a Base radius of 45
+|stat_text                               = Base duration is 20 seconds<br>Minion's Level is equal to yours, up to a maximum of 85<br>Minion deals Fire Damage equal to 30% of its maximum Life per Second to Enemies in a Base radius of 4.5 metres
 |level1                                  = True
 |level1_level_requirement                = 12
 }}