--- Skill:DroolmawDetonateDead
+++ Export
@@ -10,18 +10,18 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = secondary_minimum_base_fire_damage
-|static_stat1_value                      = 7
-|static_stat2_id                         = secondary_maximum_base_fire_damage
-|static_stat2_value                      = 10
-|static_stat3_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat3_value                      = -18
-|static_stat4_id                         = corpse_explosion_monster_life_%
-|static_stat4_value                      = 4
-|static_stat5_id                         = is_area_damage
-|static_stat5_value                      = 1
-|static_stat6_id                         = corpse_skill_has_triggers
-|static_stat6_value                      = 1
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = -18
+|static_stat2_id                         = corpse_skill_has_triggers
+|static_stat2_value                      = 1
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = secondary_minimum_base_fire_damage
+|static_stat4_value                      = 7
+|static_stat5_id                         = secondary_maximum_base_fire_damage
+|static_stat5_value                      = 10
+|static_stat6_id                         = corpse_explosion_monster_life_%
+|static_stat6_value                      = 4
 |stat_text                               = Deals 7 to 10 Fire Damage<br>Explosion deals base Fire Damage equal to 4% of the corpse's Maximum Life
 |level1                                  = True
 |level1_level_requirement                = 1
