--- Skill:TheFaunDaemonLavaGeyser
+++ Export
@@ -9,14 +9,15 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_fire_damage
-|static_stat1_value                      = 22
-|static_stat2_id                         = spell_maximum_base_fire_damage
-|static_stat2_value                      = 33
-|static_stat3_id                         = base_skill_effect_duration
-|static_stat3_value                      = 12000
-|static_stat4_id                         = is_area_damage
-|static_stat4_value                      = 1
+|static_duration                         = 12
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_fire_damage
+|static_stat2_value                      = 22
+|static_stat3_id                         = spell_maximum_base_fire_damage
+|static_stat3_value                      = 33
+|static_stat4_id                         = base_skill_effect_duration
+|static_stat4_value                      = 12000
 |stat_text                               = Deals 22 to 33 Fire Damage<br>Base duration is 12 seconds
 |level1                                  = True
 |level1_level_requirement                = 7
