--- Skill:GSHeistScienceLabradorExplosionGreen
+++ Export
@@ -8,12 +8,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage
-|static_stat1_value                      = 8
-|static_stat2_id                         = spell_maximum_base_lightning_damage
-|static_stat2_value                      = 23
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = spell_minimum_base_lightning_damage
+|static_stat2_value                      = 8
+|static_stat3_id                         = spell_maximum_base_lightning_damage
+|static_stat3_value                      = 23
 |stat_text                               = Deals 8 to 23 Lightning Damage
 |level1                                  = True
 |level1_level_requirement                = 1
