--- Skill:SanctumMiniFlamenado
+++ Export
@@ -0,0 +1,34 @@
+{{Skill
+|gem_description                         = Launches unpredictable sparks that move randomly until they hit an enemy or expire.
+|active_skill_name                       = Spark
+|skill_id                                = SanctumMiniFlamenado
+|cast_time                               = 2.0
+|required_level                          = 1
+|static_damage_effectiveness             = 100
+|static_damage_multiplier                = 100
+|static_attack_speed_multiplier          = 100
+|static_duration                         = 3
+|static_stat1_id                         = base_is_projectile
+|static_stat1_value                      = 1
+|static_stat2_id                         = base_sanctum_damage
+|static_stat2_value                      = 3
+|static_stat3_id                         = monster_projectile_variation
+|static_stat3_value                      = 4
+|static_stat4_id                         = projectiles_not_offset
+|static_stat4_value                      = 1
+|static_stat5_id                         = skill_visual_scale_+%
+|static_stat5_value                      = -50
+|static_stat6_id                         = base_skill_effect_duration
+|static_stat6_value                      = 3000
+|static_stat7_id                         = spell_minimum_base_fire_damage
+|static_stat7_value                      = 1
+|static_stat8_id                         = spell_maximum_base_fire_damage
+|static_stat8_value                      = 1
+|static_stat9_id                         = base_projectile_speed_+%
+|static_stat9_value                      = -60
+|static_stat10_id                        = number_of_additional_projectiles
+|static_stat10_value                     = 10
+|stat_text                               = Base duration is 3 seconds<br>Deals 1 to 1 Fire Damage<br>60% reduced Projectile Speed<br>Fires 10 additional Projectiles
+|level1                                  = True
+|level1_level_requirement                = 1
+}}