--- Skill:AncestralStunnerEarthquake
+++ Export
@@ -7,8 +7,8 @@
 |required_level                          = 0
 |static_stored_uses                      = 1
 |static_cooldown                         = 10
-|static_damage_effectiveness             = 110
-|static_damage_multiplier                = 200
+|static_damage_effectiveness             = 100
+|static_damage_multiplier                = 150
 |static_attack_speed_multiplier          = 100
 |static_duration                         = 2.5
 |static_stat1_id                         = action_attack_or_cast_time_uses_animation_length