--- Skill:AvatarSummonFrostBombDaemon
+++ Export
@@ -7,12 +7,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = number_of_monsters_to_summon
-|static_stat1_value                      = 1
-|static_stat2_id                         = alternate_minion
-|static_stat2_value                      = 224
-|static_stat3_id                         = cast_at_teleport_end_%
-|static_stat3_value                      = 100
+|static_stat1_id                         = alternate_minion
+|static_stat1_value                      = 224
+|static_stat2_id                         = cast_at_teleport_end_%
+|static_stat2_value                      = 100
+|static_stat3_id                         = number_of_monsters_to_summon
+|static_stat3_value                      = 1
 |static_stat4_id                         = summoned_monsters_are_parented_to_my_target
 |static_stat4_value                      = 1
 |stat_text                               = 
