--- Skill:WbCleave
+++ Export
@@ -8,12 +8,12 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = cleave_damage_+%_final_while_dual_wielding
-|static_stat1_value                      = -40
-|static_stat2_id                         = is_area_damage
+|static_stat1_id                         = is_area_damage
+|static_stat1_value                      = 1
+|static_stat2_id                         = skill_double_hits_when_dual_wielding
 |static_stat2_value                      = 1
-|static_stat3_id                         = skill_double_hits_when_dual_wielding
-|static_stat3_value                      = 1
+|static_stat3_id                         = active_skill_merged_damage_+%_final_while_dual_wielding
+|static_stat3_value                      = -40
 |stat_text                               = When Dual Wielding, Deals 60% Damage from each Weapon combined
 |level1                                  = True
 |level1_level_requirement                = 1
