--- Skill:MonsterLightningStrikeNovaBoss
+++ Export
@@ -8,17 +8,17 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = active_skill_damage_+%_final
-|static_stat1_value                      = 20
-|static_stat2_id                         = number_of_additional_projectiles
-|static_stat2_value                      = 12
-|static_stat3_id                         = active_skill_attack_damage_+%_final
-|static_stat3_value                      = 30
-|static_stat4_id                         = skill_physical_damage_%_to_convert_to_lightning
-|static_stat4_value                      = 50
-|static_stat5_id                         = projectiles_nova
-|static_stat5_value                      = 1
-|stat_text                               = (7-21)% increased Physical Damage<br>Deals 20% more Damage<br>Fires 12 additional Projectiles<br>Deals 30% more Attack Damage<br>50% of Physical Damage Converted to Lightning Damage<br>Fires Projectiles in a circle
+|static_stat1_id                         = skill_physical_damage_%_to_convert_to_lightning
+|static_stat1_value                      = 50
+|static_stat2_id                         = active_skill_attack_damage_+%_final
+|static_stat2_value                      = 30
+|static_stat3_id                         = active_skill_damage_+%_final
+|static_stat3_value                      = 20
+|static_stat4_id                         = projectiles_nova
+|static_stat4_value                      = 1
+|static_stat5_id                         = number_of_additional_projectiles
+|static_stat5_value                      = 12
+|stat_text                               = 50% of Physical Damage Converted to Lightning Damage<br>Deals 30% more Attack Damage<br>Deals 20% more Damage<br>Fires Projectiles in a circle<br>Fires 12 additional Projectiles<br>(7-21)% increased Physical Damage
 |level1                                  = True
 |level1_level_requirement                = 1
 |level1_stat_text                        = 7% increased Physical Damage
