--- Skill:KaomLeapSlam
+++ Export
@@ -12,19 +12,19 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = 40
-|static_stat2_id                         = active_skill_attack_speed_+%_final
-|static_stat2_value                      = -53
-|static_stat3_id                         = skill_physical_damage_%_to_convert_to_fire
-|static_stat3_value                      = 100
-|static_stat4_id                         = base_chance_to_ignite_%
-|static_stat4_value                      = 100
-|static_stat5_id                         = active_skill_damage_+%_final
-|static_stat5_value                      = 50
-|static_stat6_id                         = is_area_damage
-|static_stat6_value                      = 1
-|static_stat7_id                         = cast_time_overrides_attack_duration
-|static_stat7_value                      = 1
-|stat_text                               = 53% less Attack Speed<br>100% of Physical Damage Converted to Fire Damage<br>100% chance to Ignite enemies<br>Deals 50% more Damage
+|static_stat2_id                         = cast_time_overrides_attack_duration
+|static_stat2_value                      = 1
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = active_skill_damage_+%_final
+|static_stat4_value                      = 50
+|static_stat5_id                         = base_chance_to_ignite_%
+|static_stat5_value                      = 100
+|static_stat6_id                         = skill_physical_damage_%_to_convert_to_fire
+|static_stat6_value                      = 100
+|static_stat7_id                         = active_skill_attack_speed_+%_final
+|static_stat7_value                      = -53
+|stat_text                               = Deals 50% more Damage<br>100% chance to Ignite enemies<br>100% of Physical Damage Converted to Fire Damage<br>53% less Attack Speed
 |level1                                  = True
 |level1_level_requirement                = 2
 }}