--- Skill:ExileVaalStormCall
+++ Export
@@ -11,13 +11,14 @@
 |static_attack_speed_multiplier          = 100
 |static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
 |static_stat1_value                      = -13
-|static_stat2_id                         = vaal_storm_call_base_delay_ms
-|static_stat2_value                      = 250
-|static_stat3_id                         = is_area_damage
-|static_stat3_value                      = 1
+|static_stat2_id                         = is_area_damage
+|static_stat2_value                      = 1
+|static_stat3_id                         = vaal_storm_call_base_delay_ms
+|static_stat3_value                      = 250
 |stat_text                               = Deals (4-386) to (12-1157) Lightning Damage<br>Base duration is (3-5.9) seconds<br>Strikes an Enemy every 0.25 seconds
 |level1                                  = True
 |level1_level_requirement                = 4
+|level1_duration                         = 3
 |level1_stat_text                        = Deals 4 to 12 Lightning Damage<br>Base duration is 3 seconds
 |level1_stat1_id                         = spell_minimum_base_lightning_damage
 |level1_stat1_value                      = 4
@@ -27,6 +28,7 @@
 |level1_stat3_value                      = 3000
 |level2                                  = True
 |level2_level_requirement                = 6
+|level2_duration                         = 3.1
 |level2_stat_text                        = Deals 5 to 14 Lightning Damage<br>Base duration is 3.1 seconds
 |level2_stat1_id                         = spell_minimum_base_lightning_damage
 |level2_stat1_value                      = 5
@@ -36,6 +38,7 @@
 |level2_stat3_value                      = 3100
 |level3                                  = True
 |level3_level_requirement                = 9
+|level3_duration                         = 3.2
 |level3_stat_text                        = Deals 6 to 19 Lightning Damage<br>Base duration is 3.2 seconds
 |level3_stat1_id                         = spell_minimum_base_lightning_damage
 |level3_stat1_value                      = 6
@@ -45,6 +48,7 @@
 |level3_stat3_value                      = 3200
 |level4                                  = True
 |level4_level_requirement                = 12
+|level4_duration                         = 3.3
 |level4_stat_text                        = Deals 8 to 24 Lightning Damage<br>Base duration is 3.3 seconds
 |level4_stat1_id                         = spell_minimum_base_lightning_damage
 |level4_stat1_value                      = 8
@@ -54,6 +58,7 @@
 |level4_stat3_value                      = 3300
 |level5                                  = True
 |level5_level_requirement                = 15
+|level5_duration                         = 3.4
 |level5_stat_text                        = Deals 10 to 30 Lightning Damage<br>Base duration is 3.4 seconds
 |level5_stat1_id                         = spell_minimum_base_lightning_damage
 |level5_stat1_value                      = 10
@@ -63,6 +68,7 @@
 |level5_stat3_value                      = 3400
 |level6                                  = True
 |level6_level_requirement                = 19
+|level6_duration                         = 3.5
 |level6_stat_text                        = Deals 13 to 40 Lightning Damage<br>Base duration is 3.5 seconds
 |level6_stat1_id                         = spell_minimum_base_lightning_damage
 |level6_stat1_value                      = 13
@@ -72,6 +78,7 @@
 |level6_stat3_value                      = 3500
 |level7                                  = True
 |level7_level_requirement                = 23
+|level7_duration                         = 3.6
 |level7_stat_text                        = Deals 17 to 51 Lightning Damage<br>Base duration is 3.6 seconds
 |level7_stat1_id                         = spell_minimum_base_lightning_damage
 |level7_stat1_value                      = 17
@@ -81,6 +88,7 @@
 |level7_stat3_value                      = 3600
 |level8                                  = True
 |level8_level_requirement                = 28
+|level8_duration                         = 3.7
 |level8_stat_text                        = Deals 23 to 69 Lightning Damage<br>Base duration is 3.7 seconds
 |level8_stat1_id                         = spell_minimum_base_lightning_damage
 |level8_stat1_value                      = 23
@@ -90,6 +98,7 @@
 |level8_stat3_value                      = 3700
 |level9                                  = True
 |level9_level_requirement                = 33
+|level9_duration                         = 3.8
 |level9_stat_text                        = Deals 30 to 91 Lightning Damage<br>Base duration is 3.8 seconds
 |level9_stat1_id                         = spell_minimum_base_lightning_damage
 |level9_stat1_value                      = 30
@@ -99,6 +108,7 @@
 |level9_stat3_value                      = 3800
 |level10                                 = True
 |level10_level_requirement               = 39
+|level10_duration                        = 3.9
 |level10_stat_text                       = Deals 42 to 125 Lightning Damage<br>Base duration is 3.9 seconds
 |level10_stat1_id                        = spell_minimum_base_lightning_damage
 |level10_stat1_value                     = 42
@@ -108,6 +118,7 @@
 |level10_stat3_value                     = 3900
 |level11                                 = True
 |level11_level_requirement               = 43
+|level11_duration                        = 4
 |level11_stat_text                       = Deals 51 to 153 Lightning Damage<br>Base duration is 4 seconds
 |level11_stat1_id                        = spell_minimum_base_lightning_damage
 |level11_stat1_value                     = 51
@@ -117,6 +128,7 @@
 |level11_stat3_value                     = 4000
 |level12                                 = True
 |level12_level_requirement               = 46
+|level12_duration                        = 4.1
 |level12_stat_text                       = Deals 59 to 178 Lightning Damage<br>Base duration is 4.1 seconds
 |level12_stat1_id                        = spell_minimum_base_lightning_damage
 |level12_stat1_value                     = 59
@@ -126,6 +138,7 @@
 |level12_stat3_value                     = 4100
 |level13                                 = True
 |level13_level_requirement               = 49
+|level13_duration                        = 4.2
 |level13_stat_text                       = Deals 69 to 206 Lightning Damage<br>Base duration is 4.2 seconds
 |level13_stat1_id                        = spell_minimum_base_lightning_damage
 |level13_stat1_value                     = 69
@@ -135,6 +148,7 @@
 |level13_stat3_value                     = 4200
 |level14                                 = True
 |level14_level_requirement               = 52
+|level14_duration                        = 4.3
 |level14_stat_text                       = Deals 79 to 238 Lightning Damage<br>Base duration is 4.3 seconds
 |level14_stat1_id                        = spell_minimum_base_lightning_damage
 |level14_stat1_value                     = 79
@@ -144,6 +158,7 @@
 |level14_stat3_value                     = 4300
 |level15                                 = True
 |level15_level_requirement               = 55
+|level15_duration                        = 4.4
 |level15_stat_text                       = Deals 91 to 274 Lightning Damage<br>Base duration is 4.4 seconds
 |level15_stat1_id                        = spell_minimum_base_lightning_damage
 |level15_stat1_value                     = 91
@@ -153,6 +168,7 @@
 |level15_stat3_value                     = 4400
 |level16                                 = True
 |level16_level_requirement               = 58
+|level16_duration                        = 4.5
 |level16_stat_text                       = Deals 105 to 315 Lightning Damage<br>Base duration is 4.5 seconds
 |level16_stat1_id                        = spell_minimum_base_lightning_damage
 |level16_stat1_value                     = 105
@@ -162,6 +178,7 @@
 |level16_stat3_value                     = 4500
 |level17                                 = True
 |level17_level_requirement               = 61
+|level17_duration                        = 4.6
 |level17_stat_text                       = Deals 120 to 361 Lightning Damage<br>Base duration is 4.6 seconds
 |level17_stat1_id                        = spell_minimum_base_lightning_damage
 |level17_stat1_value                     = 120
@@ -171,6 +188,7 @@
 |level17_stat3_value                     = 4600
 |level18                                 = True
 |level18_level_requirement               = 64
+|level18_duration                        = 4.7
 |level18_stat_text                       = Deals 138 to 413 Lightning Damage<br>Base duration is 4.7 seconds
 |level18_stat1_id                        = spell_minimum_base_lightning_damage
 |level18_stat1_value                     = 138
@@ -180,6 +198,7 @@
 |level18_stat3_value                     = 4700
 |level19                                 = True
 |level19_level_requirement               = 66
+|level19_duration                        = 4.8
 |level19_stat_text                       = Deals 150 to 451 Lightning Damage<br>Base duration is 4.8 seconds
 |level19_stat1_id                        = spell_minimum_base_lightning_damage
 |level19_stat1_value                     = 150
@@ -189,6 +208,7 @@
 |level19_stat3_value                     = 4800
 |level20                                 = True
 |level20_level_requirement               = 67
+|level20_duration                        = 4.85
 |level20_stat_text                       = Deals 157 to 472 Lightning Damage<br>Base duration is 4.85 seconds
 |level20_stat1_id                        = spell_minimum_base_lightning_damage
 |level20_stat1_value                     = 157
@@ -198,6 +218,7 @@
 |level20_stat3_value                     = 4850
 |level21                                 = True
 |level21_level_requirement               = 68
+|level21_duration                        = 4.9
 |level21_stat_text                       = Deals 164 to 493 Lightning Damage<br>Base duration is 4.9 seconds
 |level21_stat1_id                        = spell_minimum_base_lightning_damage
 |level21_stat1_value                     = 164
@@ -207,6 +228,7 @@
 |level21_stat3_value                     = 4900
 |level22                                 = True
 |level22_level_requirement               = 69
+|level22_duration                        = 4.95
 |level22_stat_text                       = Deals 172 to 515 Lightning Damage<br>Base duration is 4.95 seconds
 |level22_stat1_id                        = spell_minimum_base_lightning_damage
 |level22_stat1_value                     = 172
@@ -216,6 +238,7 @@
 |level22_stat3_value                     = 4950
 |level23                                 = True
 |level23_level_requirement               = 70
+|level23_duration                        = 5
 |level23_stat_text                       = Deals 179 to 538 Lightning Damage<br>Base duration is 5 seconds
 |level23_stat1_id                        = spell_minimum_base_lightning_damage
 |level23_stat1_value                     = 179
@@ -225,6 +248,7 @@
 |level23_stat3_value                     = 5000
 |level24                                 = True
 |level24_level_requirement               = 71
+|level24_duration                        = 5.05
 |level24_stat_text                       = Deals 188 to 563 Lightning Damage<br>Base duration is 5.05 seconds
 |level24_stat1_id                        = spell_minimum_base_lightning_damage
 |level24_stat1_value                     = 188
@@ -234,6 +258,7 @@
 |level24_stat3_value                     = 5050
 |level25                                 = True
 |level25_level_requirement               = 72
+|level25_duration                        = 5.1
 |level25_stat_text                       = Deals 196 to 588 Lightning Damage<br>Base duration is 5.1 seconds
 |level25_stat1_id                        = spell_minimum_base_lightning_damage
 |level25_stat1_value                     = 196
@@ -243,6 +268,7 @@
 |level25_stat3_value                     = 5100
 |level26                                 = True
 |level26_level_requirement               = 73
+|level26_duration                        = 5.15
 |level26_stat_text                       = Deals 205 to 614 Lightning Damage<br>Base duration is 5.15 seconds
 |level26_stat1_id                        = spell_minimum_base_lightning_damage
 |level26_stat1_value                     = 205
@@ -252,6 +278,7 @@
 |level26_stat3_value                     = 5150
 |level27                                 = True
 |level27_level_requirement               = 74
+|level27_duration                        = 5.2
 |level27_stat_text                       = Deals 214 to 641 Lightning Damage<br>Base duration is 5.2 seconds
 |level27_stat1_id                        = spell_minimum_base_lightning_damage
 |level27_stat1_value                     = 214
@@ -261,6 +288,7 @@
 |level27_stat3_value                     = 5200
 |level28                                 = True
 |level28_level_requirement               = 75
+|level28_duration                        = 5.25
 |level28_stat_text                       = Deals 223 to 669 Lightning Damage<br>Base duration is 5.25 seconds
 |level28_stat1_id                        = spell_minimum_base_lightning_damage
 |level28_stat1_value                     = 223
@@ -270,6 +298,7 @@
 |level28_stat3_value                     = 5250
 |level29                                 = True
 |level29_level_requirement               = 76
+|level29_duration                        = 5.3
 |level29_stat_text                       = Deals 233 to 698 Lightning Damage<br>Base duration is 5.3 seconds
 |level29_stat1_id                        = spell_minimum_base_lightning_damage
 |level29_stat1_value                     = 233
@@ -279,6 +308,7 @@
 |level29_stat3_value                     = 5300
 |level30                                 = True
 |level30_level_requirement               = 77
+|level30_duration                        = 5.35
 |level30_stat_text                       = Deals 243 to 729 Lightning Damage<br>Base duration is 5.35 seconds
 |level30_stat1_id                        = spell_minimum_base_lightning_damage
 |level30_stat1_value                     = 243
@@ -288,6 +318,7 @@
 |level30_stat3_value                     = 5350
 |level31                                 = True
 |level31_level_requirement               = 78
+|level31_duration                        = 5.4
 |level31_stat_text                       = Deals 253 to 760 Lightning Damage<br>Base duration is 5.4 seconds
 |level31_stat1_id                        = spell_minimum_base_lightning_damage
 |level31_stat1_value                     = 253
@@ -297,6 +328,7 @@
 |level31_stat3_value                     = 5400
 |level32                                 = True
 |level32_level_requirement               = 79
+|level32_duration                        = 5.45
 |level32_stat_text                       = Deals 265 to 794 Lightning Damage<br>Base duration is 5.45 seconds
 |level32_stat1_id                        = spell_minimum_base_lightning_damage
 |level32_stat1_value                     = 265
@@ -306,6 +338,7 @@
 |level32_stat3_value                     = 5450
 |level33                                 = True
 |level33_level_requirement               = 80
+|level33_duration                        = 5.5
 |level33_stat_text                       = Deals 276 to 828 Lightning Damage<br>Base duration is 5.5 seconds
 |level33_stat1_id                        = spell_minimum_base_lightning_damage
 |level33_stat1_value                     = 276
@@ -315,6 +348,7 @@
 |level33_stat3_value                     = 5500
 |level34                                 = True
 |level34_level_requirement               = 81
+|level34_duration                        = 5.55
 |level34_stat_text                       = Deals 288 to 864 Lightning Damage<br>Base duration is 5.55 seconds
 |level34_stat1_id                        = spell_minimum_base_lightning_damage
 |level34_stat1_value                     = 288
@@ -324,6 +358,7 @@
 |level34_stat3_value                     = 5550
 |level35                                 = True
 |level35_level_requirement               = 82
+|level35_duration                        = 5.6
 |level35_stat_text                       = Deals 300 to 901 Lightning Damage<br>Base duration is 5.6 seconds
 |level35_stat1_id                        = spell_minimum_base_lightning_damage
 |level35_stat1_value                     = 300
@@ -333,6 +368,7 @@
 |level35_stat3_value                     = 5600
 |level36                                 = True
 |level36_level_requirement               = 84
+|level36_duration                        = 5.7
 |level36_stat_text                       = Deals 327 to 980 Lightning Damage<br>Base duration is 5.7 seconds
 |level36_stat1_id                        = spell_minimum_base_lightning_damage
 |level36_stat1_value                     = 327
@@ -342,6 +378,7 @@
 |level36_stat3_value                     = 5700
 |level37                                 = True
 |level37_level_requirement               = 86
+|level37_duration                        = 5.8
 |level37_stat_text                       = Deals 355 to 1065 Lightning Damage<br>Base duration is 5.8 seconds
 |level37_stat1_id                        = spell_minimum_base_lightning_damage
 |level37_stat1_value                     = 355
@@ -351,6 +388,7 @@
 |level37_stat3_value                     = 5800
 |level38                                 = True
 |level38_level_requirement               = 88
+|level38_duration                        = 5.9
 |level38_stat_text                       = Deals 386 to 1157 Lightning Damage<br>Base duration is 5.9 seconds
 |level38_stat1_id                        = spell_minimum_base_lightning_damage
 |level38_stat1_value                     = 386
