--- Skill:BestiaryBloodRavenCorpseEruption
+++ Export
@@ -1,5 +1,5 @@
 {{Skill
-|gem_description                         = A targeted corpse explodes, dealing area damage and turning into a volcanic geyser, which will repeatedly unleash projectiles sequentially over the surrounding area for a duration. The explosion of the corpse is not affected by modifiers to spell damage, and cannot be reflected.
+|gem_description                         = A targeted corpse explodes, dealing area damage and turning into a volcanic geyser, which will repeatedly unleash exploding projectiles sequentially over the surrounding area for a duration. The explosion of the corpse is not affected by modifiers to spell damage, and cannot be reflected.
 |active_skill_name                       = Cremation
 |skill_id                                = BestiaryBloodRavenCorpseEruption
 |cast_time                               = 0.5
@@ -7,27 +7,28 @@
 |static_damage_effectiveness             = 100
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_physical_damage
-|static_stat1_value                      = 6
-|static_stat2_id                         = spell_maximum_base_physical_damage
-|static_stat2_value                      = 9
-|static_stat3_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat3_value                      = -20
+|static_duration                         = 4
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = -20
+|static_stat2_id                         = base_is_projectile
+|static_stat2_value                      = 1
+|static_stat3_id                         = is_area_damage
+|static_stat3_value                      = 1
 |static_stat4_id                         = monster_penalty_against_minions_damage_+%_final_vs_player_minions
 |static_stat4_value                      = -80
-|static_stat5_id                         = cremation_base_fires_projectile_every_x_ms
-|static_stat5_value                      = 1000
-|static_stat6_id                         = number_of_additional_projectiles
-|static_stat6_value                      = 2
-|static_stat7_id                         = corpse_erruption_base_maximum_number_of_geyers
-|static_stat7_value                      = 1
+|static_stat5_id                         = spell_minimum_base_physical_damage
+|static_stat5_value                      = 6
+|static_stat6_id                         = spell_maximum_base_physical_damage
+|static_stat6_value                      = 9
+|static_stat7_id                         = cremation_base_fires_projectile_every_x_ms
+|static_stat7_value                      = 1000
 |static_stat8_id                         = base_skill_effect_duration
 |static_stat8_value                      = 4000
-|static_stat9_id                         = is_area_damage
+|static_stat9_id                         = corpse_erruption_base_maximum_number_of_geyers
 |static_stat9_value                      = 1
-|static_stat10_id                        = base_is_projectile
-|static_stat10_value                     = 1
-|stat_text                               = Deals 6 to 9 Physical Damage<br>Fires Projectiles every 1 seconds<br>Fires 2 additional Projectiles<br>Maximum of 1 Geysers at a time<br>Base duration is 4 seconds
+|static_stat10_id                        = number_of_additional_projectiles
+|static_stat10_value                     = 2
+|stat_text                               = Deals 6 to 9 Physical Damage<br>Fires Projectiles every 1 seconds<br>Base duration is 4 seconds<br>Maximum of 1 Geysers at a time<br>Fires 2 additional Projectiles
 |level1                                  = True
 |level1_level_requirement                = 1
 }}