--- Skill:DexMissionBallLightningOnDeath
+++ Export
@@ -7,21 +7,21 @@
 |static_damage_effectiveness             = 20
 |static_damage_multiplier                = 100
 |static_attack_speed_multiplier          = 100
-|static_stat1_id                         = spell_minimum_base_lightning_damage
-|static_stat1_value                      = 1
-|static_stat2_id                         = spell_maximum_base_lightning_damage
-|static_stat2_value                      = 2
-|static_stat3_id                         = active_skill_area_of_effect_radius_+%_final
-|static_stat3_value                      = -33
-|static_stat4_id                         = cast_on_death_%
-|static_stat4_value                      = 100
+|static_stat1_id                         = active_skill_area_of_effect_radius_+%_final
+|static_stat1_value                      = -33
+|static_stat2_id                         = base_is_projectile
+|static_stat2_value                      = 1
+|static_stat3_id                         = spell_minimum_base_lightning_damage
+|static_stat3_value                      = 1
+|static_stat4_id                         = spell_maximum_base_lightning_damage
+|static_stat4_value                      = 2
 |static_stat5_id                         = base_projectile_speed_+%
 |static_stat5_value                      = -200
 |static_stat6_id                         = spell_uncastable_if_triggerable
 |static_stat6_value                      = 1
-|static_stat7_id                         = base_is_projectile
-|static_stat7_value                      = 1
-|stat_text                               = Deals 1 to 2 Lightning Damage<br>100% chance to Trigger this Spell on Death<br>200% reduced Projectile Speed<br>You cannot Cast this Spell directly
+|static_stat7_id                         = cast_on_death_%
+|static_stat7_value                      = 100
+|stat_text                               = Deals 1 to 2 Lightning Damage<br>200% reduced Projectile Speed<br>You cannot Cast this Spell directly<br>100% chance to Trigger this Spell on Death
 |level1                                  = True
 |level1_level_requirement                = 3
 }}